<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8"/>
<style>body{background-color:white;}</style>
<script src="redUruguay_files/htmlwidgets-1.5.4/htmlwidgets.js"></script>
<link href="redUruguay_files/vis-9.1.0/vis-network.min.css" rel="stylesheet" />
<script src="redUruguay_files/vis-9.1.0/vis-network.min.js"></script>
<script src="redUruguay_files/visNetwork-binding-2.1.2/visNetwork.js"></script>
  <title>visNetwork</title>
</head>
<body>
<div id="htmlwidget_container">
  <div id="htmlwidget-40b5eba84db10dfbcbe9" style="width:960px;height:500px;" class="visNetwork html-widget"></div>
</div>
<script type="application/json" data-for="htmlwidget-40b5eba84db10dfbcbe9">{"x":{"nodes":{"id":["per_57219793915","per_57212764664","per_36464380700","per_56436897300","per_57456534500","per_57189387436","per_55280003800","per_53985251000","per_10440626500","per_55743415000","per_6507604844","per_54403612000","per_7103353204","per_6603463277","per_53985361900","per_55979915700","per_56741425700","per_57190007914","per_35739135100","per_24512267300","per_57211533183","per_17342030700","per_8543458200","per_39962198000","per_8542323900","per_56190665700","per_22954214400","per_36665830700","per_55940741800","per_57212211925","per_57250988600","per_56194926500","per_6603929800","per_55726786800","per_52564389200","per_6602636501","per_14423907700","per_7004144747","per_37034834100","per_23024872800","per_55276541900","per_57299589400","per_35107535800","per_57222077797","per_57224946340","per_8647872300","per_11839066900","per_57225087680","per_18634327300","per_57192692601","per_23093709400","per_57224577809","per_57206898463","per_36550241400","per_25027075800","per_15057981500","per_35267867000","per_55832083700","per_6701767937","per_55654724000","per_56210777300","per_7006077782","per_26433451900","per_57221371525","per_6508090491","per_23488048300","per_36997206000","per_36997017700","per_57202985610","per_7101928929","per_7004526393","per_57193317190","per_7005713737","per_57208315990","per_36925959500","per_57196479751","per_7006691882","per_55250349200","per_6602160256","per_6701907779","per_57218268471","per_50262130700","per_6602356437","per_37034854100","per_56400915100","per_56685832700","per_9432632800","per_36785334500","per_57216353981","per_23767166000","per_57216338926","per_57021871900","per_57205611884","per_6603223065","per_6602981109","per_36562060300","per_55841158700","per_55166465600","per_56123759300","per_15127074300","per_36823419900","per_57212795362","per_57204186756","per_55540217900","per_7005952768","per_57203635258","per_57208318231","per_6504054963","per_24331023000","per_56162108500","per_57207945523","per_24723823500","per_6507184057","per_8855704500","per_57194586621","per_16751030800","per_6504749273","per_28367582700","per_36674748500","per_41862438300","per_57204391106","per_57204393725","per_56205553600","per_16426432300","per_57202946021","per_23502406500","per_16031671800","per_56417516200","per_55891701000","per_41661641800","per_55375454200","per_7005385351","per_24068102500","per_6507277353","per_57793531800","per_8982534700","per_21743627000","per_16444345400","per_55954974200","per_36244466900","per_6602445638","per_21740696500","per_6507071133","per_55328949500","per_7401971740","per_6602674368","per_24070523500","per_55375874000","per_56748010200","per_35099343500","per_56433512900","per_54406796800","per_6603171524","per_13407411800","per_6506385182","per_55342418800","per_36128589400","per_6506958322","per_54584939400","per_15070249500","per_36521591600","per_6602843376","per_25931363700","per_7202755185","per_18041904100","per_55394472800","per_8862840800","per_35195259300","per_36104233000","per_55851946982","per_36104759600","per_36104785900","per_55996060000","per_36997190700","per_36997336200","per_6507291476","per_22981409500","per_23024205100","per_23024124800","per_56055484200","per_57192872701","per_8976045400","pap_57219793915::Age-specific rate of severe and critical SARS-CoV-2 infections estimated with multi-country seroprevalence studies","pap_57212764664::Origin and spreading of canine morbillivirus in South America","pap_54403612000::Wool scoured colour: Heritability, genetic and phenotypic correlations with wool traits in Corriedale sheep","pap_56741425700::Assessment of Real-Time Quaking-Induced Conversion (RT-QuIC) Assay, Immunohistochemistry and ELISA for Detection of Chronic Wasting Disease under Field Conditions in White-Tailed Deer: A Bayesian Approach","pap_57224533812::Colonization rather than fragmentation explains the geographical distribution and diversification of treefrogs endemic to Brazilian shield sky islands","pap_35739135100::Performance of bovine genital campylobacteriosis diagnostic tests in bulls from Uruguay: a Bayesian latent class model approach","pap_24512267300::Not the same: phylogenetic relationships and ecological niche comparisons between two different forms of Aglaoctenus lagotis from Argentina and Uruguay","pap_57217136035::Whole genomic comparative analysis of Streptococcus pneumoniae serotype 1 isolates causing invasive and non-invasive infections among children under 5 years in Casablanca, Morocco","pap_39962198000::Ultraconserved Elements Improve the Resolution of Difficult Nodes within the Rapid Radiation of Neotropical Sigmodontine Rodents (Cricetidae: Sigmodontinae)","pap_8542323900::Characterization of “candidatus ehrlichia pampeana” in haemaphysalis juxtakochi ticks and gray brocket deer (Mazama gouazoubira) from uruguay","pap_56190665700::An Hepatozoon americanum-like protozoan in crab-eating (Cerdocyon thous) and grey pampean (Lycalopex gymnocercus) foxes from Uruguay","pap_56194926500::Genomic footprints of Quaternary colonization and population expansion in the Patagonian-Fuegian region rules out a separate southern refugium in Tierra del Fuego","pap_55726786800::Automatic calibration and uncertainty quantification in waves dynamical downscaling","pap_55909075500::Sediment grain size determines microplastic exposure landscapes for sandy beach macroinfauna","pap_37034834100::Emergence and spread of a b.1.1.28-derived p.6 lineage with q675h and q677h spike mutations in uruguay","pap_25824960900::Quaternary climatic fluctuations influence the demographic history of two species of sky-island endemic amphibians in the Neotropics","pap_25821559500::Building the GLENCOE Platform -Grasslands LENding eConomic and ecOsystems sErvices","pap_57193920068::Population structure and gene flow of Geoffroy's cat (Leopardus geoffroyi) in the Uruguayan Savanna ecoregion","pap_57224577809::Water-quality data imputation with a high percentage of missing values: A machine learning approach","pap_6701767937::A new early warning indicator of tree species crashes from effective intraspecific interactions in tropical forests","pap_55654724000::The inner ear anatomy of glyptodonts and pampatheres (Xenarthra, Cingulata): Functional and phylogenetic implications","pap_6701767937::A method for predicting species trajectories tested with trees in barro colorado tropical forest","pap_6603547554::Markets as ecological networks: Inferring interactions and identifying communities","pap_57216650158::Riparian forest subsidises the biomass of fish in a recently formed subtropical reservoir","pap_57221371525::Adaptive evolution of β-globin gene in subterranean in South America octodontid rodents","pap_56927960700::Inclusion of bioclimatic variables in genetic evaluations of dairy cattle","pap_36997206000::Applying Bayesian Networks to help Physicians Diagnose Respiratory Diseases in the context of COVID-19 Pandemic","pap_8542323900::Molecular characterization of novel Ehrlichia genotypes in Ixodes auritulus from Uruguay","pap_57202985610::Development of a Population Pharmacokinetic Model for Cyclosporine from Therapeutic Drug Monitoring Data","pap_56927960700::Selection for test-day milk yield and thermotolerance in brazilian holstein cattle","pap_57208315990::Validation of a multiplex PCR assay to detect Babesia spp. and Anaplasma marginale in cattle in Uruguay in the absence of a gold standard test","pap_57219778546::Genetic parameters for production traits, resistance and resilience to Nematode parasites under different worm burden challenges in Corriedale sheep","pap_35228674100::Evidence of increasing diversification of emerging Severe Acute Respiratory Syndrome Coronavirus 2 strains","pap_57218268471::Is it possible to accurately estimate lactation curve parameters in extensive beef production systems?","pap_37034854100::An evolutionary insight into emerging Ebolavirus strains isolated in Africa","pap_57218192116::Low additive genetic variation in a trait under selection in domesticated rice","pap_55841211100::Single-step genomic evaluations from theory to practice: using snp chips and sequence data in blupf90","pap_57193259331::Historical origins and zoonotic potential of avian influenza virus H9N2 in Tunisia revealed by Bayesian analysis and molecular characterization","pap_45861549800::Multilocus phylogeny of Paratelmatobiinae (Anura: Leptodactylidae) reveals strong spatial structure and previously unknown diversity in the Atlantic Forest hotspot","pap_56685832700::Predicting cyanobacterial biovolume from water temperature and conductivity using a Bayesian compound Poisson-Gamma model","pap_56805256800::Gametes Simulator: A multilocus genotype simulator to analyze genetic structure in outbreeding diploid species","pap_57215014126::Using approximate Bayesian inference for a \"steps and turns\" continuous-time random walk observed at regular time intervals","pap_57205611884::Fraud Detection in Electric Power Distribution: An Approach That Maximizes the Economic Return","pap_56741425700::Association between results of diagnostic tests for bovine tuberculosis and Johne's disease in cattle","pap_57202058591::The discovery of Neotropical Lepidosira (Collembola, Entomobryidae) and its systematic position","pap_7202016735::APPARENT BIAS: WHAT DOES ATTITUDE POLARIZATION SHOW?","pap_36562060300::The impact of anthropogenic food subsidies on a generalist seabird during nestling growth","pap_57193920068::Conservation planning in Uruguay based on small felids (Leopardus spp.) as umbrella species","pap_57211321476::Species diversity and distribution of the genus Colpomenia (Scytosiphonaceae, Phaeophyceae) along the coast of China","pap_56123759300::Coalescent-based species delimitation in the sand lizards of the Liolaemus wiegmannii complex (Squamata: Liolaemidae)","pap_56741425700::Modeling the Accuracy of Two in-vitro Bovine Tuberculosis Tests Using a Bayesian Approach","pap_55540217900::Millennium-length precipitation reconstruction over south-eastern asia: A pseudo-proxy approach","pap_26657911800::Detection of presumed genes encoding beta-lactamases by sequence based screening of metagenomes derived from Antarctic microbial mats","pap_57196479751::Probability of Rhipicephalus microplus introduction into farms by cattle movement using a Bayesian Belief Network","pap_56210777300::Phylogeny, Macroevolutionary Trends and Historical Biogeography of Sloths: Insights From a Bayesian Morphological Clock Analysis","pap_57195593253::Cryptic diversity in Brazilian endemic monkey frogs (Hylidae, Phyllomedusinae, Pithecopus) revealed by multispecies coalescent and integrative approaches","pap_6602206179::Joint Minimization of Monitoring Cost and Delay in Overlay Networks: Optimal Policies with a Markovian Approach","pap_7003534815::Genetic and pathogenic diversity of Colletotrichum species associated with apple diseases in southern Brazil and Uruguay","pap_57210218727::A hybrid swarm of manatees along the guianas coastline, a peculiar environment under the influence of the Amazon River plume","pap_57021871900::Fully Bayesian analysis of allele-specific RNA-seq data","pap_57207945523::A historical perspective of niche differentiation between two top predators in the Uruguayan coastal area","pap_57197386412::Impact of Pleistocene geoclimatic events on the genetic structure in mid-latitude South American plants: Insights from the phylogeography of Turnera sidoides complex (Passifloraceae, Turneroideae)","pap_57194586621::An evolutionary insight into zika virus strains isolated in the Latin American region","pap_28367582700::Mouth gape determines the response of marine top predators to long-term fishery-induced changes in food web structure","pap_56205553600::Inferring population dynamics of HIV-1 subtype C epidemics in Eastern Africa and Southern Brazil applying different Bayesian phylodynamics approaches","pap_57042629400::Phylodynamics of Merkel-cell polyomavirus and human polyomavirus 6: A long-term history with humans","pap_57202946021::Action semantics at the bottom of the brain: Insights from dysplastic cerebellar gangliocytoma","pap_56805256800::Genetic structure, core collection, and regeneration quality in white dent corn landraces","pap_7006290311::Prediction of complex traits: Robust alternatives to best linear unbiased prediction","pap_57195056016::Human Bocavirus: Detection, Quantification and Molecular Characterization in Sewage and Surface Waters in Uruguay","pap_14015150800::The HIV-1 subtype B epidemic in French guiana and Suriname is driven by ongoing transmissions of pandemic and non-pandemic lineages","pap_54987316200::Kelps' long-distance dispersal: Role of Ecological/Oceanographic processes and implications to marine forest conservation","pap_36718043400::Joint denoising and decompression: A patch-based Bayesian approach","pap_57206396817::Geographic distribution modeling of the margay (Leopardus wiedii) and jaguarundi (Puma yagouaroundi): A comparative assessment","pap_55777005500::Latent class models for Echinococcus multilocularis diagnosis in foxes in Switzerland in the absence of a gold standard","pap_41661641800::Historical isolation and contemporary gene flow drive population diversity of the brown alga Sargassum thunbergii along the coast of China","pap_43861064400::Evidence of increasing diversification of Zika virus strains isolated in the American continent","pap_23484882900::Amphitropical disjunctions in new world menthinae: Three pliocene dispersals to South America following late miocene dispersal to North America from the old world","pap_24068102500::The disjunct pattern of the Neotropical harvestman Discocyrtus dilatatus (Gonyleptidae) explained by climate-driven range shifts in the Quaternary: Paleodistributional and molecular evidence","pap_32067654500::Tracing the origin of the NS1 A188V substitution responsible for recent enhancement of ZIKA virus asian genotype infectivity","pap_57190007914::Evolutionarily significant units of the critically endangered leaf frog Pithecopus ayeaye (Anura, Phyllomedusidae) are not effectively preserved by the Brazilian protected areas network","pap_6508090491::Molecular adaptive convergence in the α-globin in subterranean octodontid rodents","pap_55485032800::Description of a new soft tick species (Acari: Argasidae: Ornithodoros) associated with stream-breeding frogs (Anura: Cycloramphidae: Cycloramphus) in Brazil","pap_45861485800::A new species of Deltamys Thomas, 1917 (Rodentia: Cricetidae) endemic to the southern Brazilian Araucaria Forest and notes on the expanded phylogeographic scenario of D. kempi","pap_55819830400::Uncertainty Quantification in a Patient-Specific One-Dimensional Arterial Network Model: EnKF-Based Inflow Estimator","pap_36674748500::Trophic relationships in apex predators in an estuary system: A multiple-method approximation","pap_56205553600::Phylodynamics of the major HIV-1 CRF02_AG African lineages and its global dissemination","pap_24723823500::Trophic niche shifts driven by phytoplankton in sandy beach ecosystems","pap_16444345400::Modeling a production scale milk drying process: Parameter estimation, uncertainty and sensitivity analysis","pap_23024872800::Bayesian coalescent inference reveals high evolutionary rates and diversification of Zika virus populations","pap_12042289100::New insights into the hepatitis E virus genotype 3 phylodynamics and evolutionary history","pap_56452472600::Weighting strategies for single-step genomic BLUP: An iterative approach for accurate calculation of GEBV and GWAS","pap_36244466900::Threatened amphibians of Uruguay: Effectiveness of protected areas face of climate change","pap_56340718000::Hidden Markov models for gene sequence classification: Classifying the VSG gene in the Trypanosoma brucei genome","pap_56520190900::Genetic structure of Colletotrichum fructicola associated to apple bitter rot and Glomerella leaf spot in Southern Brazil and Uruguay","pap_15724490400::Trophic Dynamics of the Cape Stumpnose (Rhabdosargus holubi, Sparidae) Across Three Adjacent Aquatic Habitats","pap_21741325600::Genetic parameters for carcass traits and body weight using a Bayesian approach in the Canchim cattle","pap_17342030700::Imprints of multiple glacial refugia in the Pyrenees revealed by phylogeography and palaeodistribution modelling of an endemic spider","pap_7004217649::Genetic relationships among American donkey populations: Insights into the process of colonization","pap_36778318500::Genetic relationships between Atlantic and Pacific populations of the notothenioid fish Eleginops maclovinus: The footprints of Quaternary glaciations in Patagonia","pap_55620407900::Genotype by environment interaction for tick resistance of Hereford and Braford beef cattle using reaction norm models","pap_57189904568::Genetic parameters of objectionable fibers and of their associations with fleece traits in Corriedale sheep","pap_55726786800::Automatic calibration of a wave model with an evolutionary Bayesian method","pap_37086878000::Phylogenetic relationships between tuco-tucos (Ctenomys, Rodentia) of the Corrientes group and the C. pearsoni complex","pap_28367582700::Stable isotopes reveal long-Term fidelity to foraging grounds in the galapagos sea lion zalophus wollebaeki)","pap_45861485800::Genetic diversity of the swamp rat in South America: Population expansion after transgressive-regressive marine events in the Late Quaternary","pap_24068102500::More on the mesopotamian-yungas disjunction in subtropical and temperate argentina: Bioclimatic distribution models of the harvestman discocyrtus dilatatus (opiliones: Gonyleptidae)","pap_56400915100::Genetic diversification of local onion populations under different production systems in Uruguay","pap_6507604844::Molecular phylogeography of canine distemper virus: Geographic origin and global spreading","pap_55250349200::Description of the growth curve for Angus pasture-fed cows under extensive systems","pap_55375454200::An evolutionary insight into Newcastle disease viruses isolated in Antarctica","pap_55192958000::Phylogenetic analysis of Hepatitis B virus genotypes circulating in different risk groups of Panama, evidence of the introduction of genotype A2 in the country","pap_56748010200::Phylogenetic studies of the three rna silencing suppressor genes of south american ctv isolates reveal the circulation of a novel genetic lineage","pap_7004290258::Status, population trend and genetic structure of South American fur seals, Arctocephalus australis, in southwestern Atlantic waters","pap_7005873546::Genomic prediction for tick resistance in braford and hereford cattle","pap_28367582700::Sexual foraging segregation in South American sea lions increases during the pre-breeding period in the Río de la Plata plume","pap_7202016735::Does the better-than-average effect show that people are overconfident?: Two experiments","pap_56433512900::Temperament and its heritability in Corriedale and Merino lambs","pap_55308459300::Bayesian age and growth analysis of the shortfin mako shark Isurus oxyrinchus in the Western South Atlantic Ocean using a flexible model","pap_56205553600::Phylodynamics of major HIV-1 subtype B pandemic clades circuLating in Latin America","pap_17342030700::Endemism and evolution in the littoral woodlouse Halophiloscia Verhoeff, 1908 (Crustacea, Isopoda, Oniscidea) from the Canary Islands: implications for conservation policies","pap_7003385168::A phylogeny of the Gochnatieae: Understanding a critically placed tribe in the Compositae","pap_32067654500::Spatiotemporal dynamics of the HIV-1 subtype G epidemic in West and Central Africa","pap_26026180300::Climate change and the distribution of neotropical red-bellied toads (Melanophryniscus, Anura, Amphibia): How to prioritize species and populations?","pap_27067620300::Temporal dynamics of norovirus GII.4 variants in Brazil between 2004 and 2012","pap_55841211100::Methods for genomic evaluation of a relatively small genotyped dairy population and effect of genotyped cow information in multiparity analyses","pap_7006293432::Thermal tolerances in rodents: Species that evolved in cold climates exhibit a wider thermoneutral zone","pap_55413829400::Diversification in the south american pampas: The genetic and morphological variation of the widespread petunia axillaris complex (solanaceae)","pap_30067502200::Comparative population genetics and evolutionary history of two commonly misidentified billfishes of management and conservation concern","pap_36100371800::Genetic diversity and population structure of the New World screwworm fly from the Amazon region of Brazil","pap_8543458200::Applying spatial analysis of genetic and environmental data to predict connection corridors to the New World screwworm populations in South America","pap_37119917600::A multilocus perspective on the phylogenetic relationships and diversification of rodents of the tribe Abrotrichini (Cricetidae: Sigmodontinae)","pap_55945072500::Single Step, a general approach for genomic selection","pap_55342533900::Functional diversification within a predatory species","pap_7005722174::New records and distribution modeling of Gryne orensis (Sørensen) (Opiliones: Cosmetidae) support the Mesopotamian-Yungas disjunction in subtropical Argentina","pap_36674748500::Bias in diet determination: Incorporating traditional methods in Bayesian mixing models","pap_8543458200::The Phylogeographic History of the New World Screwworm Fly, Inferred by Approximate Bayesian Computation Analysis","pap_8647872300::Estimates of genetic parameters for worm resistance, wool and growth traits in Merino sheep of Uruguay","pap_32067654500::Tracing the Origin and Northward Dissemination Dynamics of HIV-1 Subtype C in Brazil","pap_36823419900::Quaternary range and demographic expansion of Liolaemus darwinii (Squamata: Liolaemidae) in the Monte Desert of Central Argentina using Bayesian phylogeography and ecological niche modelling","pap_55778646000::Molecular characterisation of hepatitis B virus in the resident Chinese population in Panama City","pap_36128589400::Evolution of optimal codon choices in the family Enterobacteriaceae","pap_54584939400::Fine-Scale Population Structure of Blue Whale Wintering Aggregations in the Gulf of California","pap_39962198000::Dating an impressive Neotropical radiation: Molecular time estimates for the Sigmodontinae (Rodentia) provide insights into its historical biogeography","pap_15070187100::Bayesian coalescent analysis of pandemic H1N1 influenza A virus circulating in the South American region","pap_25931363700::Exploring polymorphisms in B-DNA helical conformations","pap_36823419900::Species delimitation with abc and other coalescent-based methods: A test of accuracy with simulations and an empirical example with lizards of the liolaemus darwinii complex (Squamata: Liolaemidae)","pap_7202755185::Patterns of population differentiation in annual killifishes from the Paraná-Uruguay-La Plata Basin: The role of vicariance and dispersal","pap_55250349200::Genetic parameters for nematode resistance in periparturient ewes and post-weaning lambs in Uruguayan Merino sheep","pap_54410107400::Genetic variability and demographic history of Heliothis virescens (Lepidoptera: Noctuidae) populations from Brazil inferred by mtDNA sequences","pap_36778318500::Phylogeography of the sub-Antarctic notothenioid fish Eleginops maclovinus: Evidence of population expansion","pap_57548899500::Diversification of the American bulb-bearing Oxalis (Oxalidaceae): Dispersal to North America and modification of the tristylous breeding system","pap_7005385351::Phylogenetics of the chiliotrichum group (Compositae: Astereae): The story of the fascinating radiation in the paleate Astereae genera from southern South America","pap_14033086500::Protected areas do not fulfil the wintering habitat needs of the trans-Saharan migratory Montagu's harrier","pap_54584939400::Population genetic structure and social kin associations of franciscana dolphin, pontoporia blainvillei","pap_56172545600::Serological monitoring of selected avian pathogens of commercial poultry farms in Uruguay","pap_56172545600::Assessing the seroprevalence for abortigenic diseases of ovine herds based on diagnostic tests applied to individual-animals in Uruguay","pap_6508090491::The evolution of mitochondrial genomes in subterranean caviomorph rodents: Adaptation against a background of purifying selection","pap_7202016735::Apparent overconfidence","pap_36997206000::Comparing Bayesian inference and case-based reasoning as support techniques in the diagnosis of Acute Bacterial Meningitis","pap_39962198000::Species groups and the evolutionary diversification of tuco-tucos, genus Ctenomys (Rodentia: Ctenomyidae)","pap_57218505091::Genome-wide marker-assisted selection combining all pedigree phenotypic information with genotypic data in one step: An example using broiler chickens","pap_36997336200::Evaluation of case based reasoning for clinical decision support systems applied to acute meningitis diagnose","pap_55979915700::Phenotypic and genetic characterization of novel somatic cell count traits from weekly or monthly observations","pap_20535566900::Evolution of Dengue virus type 3 genotype III in Venezuela: Diversification, rates and population dynamics","pap_28367582700::Change in the foraging strategy of female South American sea lions (Carnivora: Pinnipedia) after parturition","pap_14015150800::Phylodynamics of HIV-1 Circulating Recombinant Forms 12_BF and 38_BF in Argentina and Uruguay","pap_15760008700::Dispersal and population structure at different spatial scales in the subterranean rodent Ctenomys australis","pap_37079657700::Genetic history of hepatitis C virus in Venezuela: High diversity and long time of evolution of HCV genotype 2","pap_55921976700::L2-Boosting algorithm applied to high-dimensional problems in genomic selection","pap_56172545600::Assessing the seroprevalence against avian pneumovirus and Ornithobacterium rhinotracheale in broilers in Uruguay","pap_36104233000::Flock-level seroprevalence against avian pneumovirus amongst uruguayan broiler chickens","pap_56172545600::Flock-level seroprevalence against Ornithobacterium rhinotracheale among broilers in Uruguay","pap_15070187100::Modeling gene sequences over time in 2009 H1N1 Influenza A Virus populations","pap_16426432300::Bayesian coalescent inference reveals high evolutionary rates and expansion of Norovirus populations","pap_55893081600::Predicting quantitative traits with regression models for dense molecular markers and pedigree","pap_23024872800::Modeling gene sequence changes over time in type 3 dengue viruses from Ecuador","pap_57204215827::Predictive ability of direct genomic values for lifetime net merit of Holstein sires using selected subsets of single nucleotide polymorphism markers","pap_56172545600::Monitoring the infectious disease status of dairy cattle herds in Uruguay: A random effects hierarchical model analysis","pap_7202755185::Phylogeography of the Southwestern Atlantic menhaden genus Brevoortia (Clupeidae, Alosinae)","pap_24068102500::Phylogeography of the Neotropical catfish Pimelodus albicans (Siluriformes: Pimelodidae) from río de la Plata basin, South America, and conservation remarks","pap_6603463277::A comparison between Poisson and zero-inflated Poisson regression models with an application to number of black spots in Corriedale sheep","pap_6504749273::Bayesian coalescent inference of hepatitis A virus populations: Evolutionary rates and patterns","pap_55979915700::Genetic variability in calving success in Aberdeen Angus cows under extensive recording","pap_23492809900::Genetic and environment factors on probability of early pregnancy of Caracu cattle","pap_6603663336::Phylogeography of the deer mouse (Peromyscus maniculatus) provides a predictive framework for research on hantaviruses","pap_6603463277::Inferring parameters shaping amino acid usage in prokaryotic genomes via Bayesian MCMC methods","pap_55979915700::Model comparison for genetic evaluation of milk yield in Uruguayan holsteins","pap_55979915700::Evaluation of extent and amount of heterogeneous variance for milk yield in Uruguayan Holsteins","pap_35581971400::Assessing relationships between genetic evaluations using robust regression with an application to holsteins in uruguay","pap_7004526393::Bioequivalence study of theophylline considering Michaelis-Menten kinetics"],"label":["Herrera-Esposito, Daniel","Fuques E.","Tomás G.","Grecco S.","Condon E.","Techera C.","Marandino A.","Sarute N.","Benech A.","Pérez R.","Panzera Y.","Neimaur K.","Kremer R.","Naya H.","Sienra I.","Urioste J.I.","Picasso-Risso, Catalina","de Magalhães, Rafael Félix","Mederos, America","González, Macarena","Kacevas, Nadia","Bidegaray-Batista, Leticia","Fresia, Pablo","Parada, Andrés","Félix, María Laura","Carvalho, Luis Andrés","Queirolo, Diego","Armúa-Fernández, María Teresa","Venzal, José Manuel","Bazzano, Valentin","da Costa, Anthony","Giorello, Facundo M.","Lessa, Enrique P.","Alonso, Rodrigo","Solari, Sebastián","Lercari, Diego","Celentano, Eleonora","Defeo, Omar","Rego, Natalia","Fajardo, Alvaro","Fernández-Calero, Tamara","Noya, Veronica","Machado, Matias R.","Brandes, Mariana","Arce, Rodrigo","Ciappesoni, Gabriel","Soutullo, Álvaro","Hernández, Daniel","González, Susana","Bartesaghi, Lucía","Cosse, Mariana","Rodríguez, Rafael","Pastorini, Marcos","Etcheverry, Lorena","Chreties, Christian","Fossati, Mónica","Castro, Alberto","Gorgoglione, Angela","Fort, Hugo","Tambusso, P. Sebastián","Varela, Luciano","Fariña, Richard A.","González-Bergonzoni, Iván","Pejo, Mariana","Tomasco, Ivanna H.","Aguilar, Ignacio","Edye, Ernesto Ocampo","Delgado, Silvia Herrera","Umpiérrez, Martín","Ibarra, Manuel","Fagiolino, Pietro","Vázquez, Marta","Maldonado, Cecilia","Parodi, Pablo","Rivero, Rodolfo","Miraballes, Cecilia","Riet-Correa, Franklin","Goldberg, Virginia","Colina, Rodney","Cristina, Juan","Iewdiukow M.","Lema O.M.","Quintans G.","Pereira-Gomez, Marianoel","Monteverde, Eliana","Haakonsson, Signe","Pérez, María del Carmen","Bonilla, Sylvia","Fernández, Peter","Galván, Guillermo A.","Priano, Federico Condón","Alvarez-Castro, Ignacio","Massaferro, Pablo","Alvarez-Valin, Fernando","Dubra, Juan","Lenzi, Javier","Machín, Emanuel","González E.M.","Villamil, Joaquín","Maneyro, Raúl","Camargo, Arley","Nunez, Alvaro","Suanes, Alejandra","Talento, Stefanie","Romero, Héctor","Saporiti, Tatiana","Lara, Stephanie","Belzarena, Pablo","Alaniz S.","Mondino P.","Artecona, Florencia","Bergamino, Leandro","Szteren, Diana","Speranza, Pablo R.","Simón, Diego","Moreno, Pilar","Moratorio, Gonzalo","Drago, Massimiliano","Franco-Trecu, Valentina","Segura, Angel M.","Valdivia, Meica","González, Enrique M.","Mir, Daiana","Victoria, Matías","Cervetto, Sabrina","Scarabino, Fabrizio","Ortega, Leonardo","Fabiano, Graciela","Muse, Pablo","Li, Jing-Jing","Soñora, Martín","Bonifacino, Jose M.","Vergara, Julia","Bia, Daniel","German, Yanina Zócalo","Dimitriadis, Caterina","Passadore, Cecilia","Ferrari A.","Musto, Héctor","Toranza, Carolina","Brazeiro, Alejandro","Espasandín A.C.","Llambí S.","Iraola, Gregorio","Hernández, Martín","Ravagnolo O.","Echeverría, Natalia","Comas, Victoria","Benítez-Galeano, María José","Sapriza, F. G. Riet","Zambra N.","Gimeno D.","Van Lier E.","Domingo, Andrés","Naya, Daniel E.","Duarte, Alejandro","Iriarte, Andrés","Castro-Sowinski, Susana","Costa-Urrutia, Paula","Coppola, Leticia","Ramas, Viviana","Chiparelli, Hector","Dans, Pablo D.","García, Graciela","Gutiérrez, Verónica","Loureiro, Marcelo","Vaio, Magdalena","Abud, Carolina","Giossa G.","Rodríguez G.","Trenchi G.","Trenchi H.","Nuñez R.","MacEiras, Mariana","Maurente, Cecilia","Ruchansky, Dora","Recarey, Ricardo","D' Andrea, Lucía","Alvarez, Macarena","Marmo F.","Guarino H.","Piovani, Rosina","Age-specific rate of severe...","Origin and spreading of...","Wool scoured colour: Heritability,...","Assessment of Real-Time Quaking-Induced...","Colonization rather than fragmentation...","Performance of bovine genital...","Not the same: phylogenetic...","Whole genomic comparative analysis...","Ultraconserved Elements Improve the...","Characterization of “candidatus ehrlichia...","An Hepatozoon americanum-like protozoan...","Genomic footprints of Quaternary...","Automatic calibration and uncertainty...","Sediment grain size determines...","Emergence and spread of...","Quaternary climatic fluctuations influence...","Building the GLENCOE Platform...","Population structure and gene...","Water-quality data imputation with...","A new early warning...","The inner ear anatomy...","A method for predicting...","Markets as ecological networks:...","Riparian forest subsidises the...","Adaptive evolution of β-globin...","Inclusion of bioclimatic variables...","Applying Bayesian Networks to...","Molecular characterization of novel...","Development of a Population...","Selection for test-day milk...","Validation of a multiplex...","Genetic parameters for production...","Evidence of increasing diversification...","Is it possible to...","An evolutionary insight into...","Low additive genetic variation...","Single-step genomic evaluations from...","Historical origins and zoonotic...","Multilocus phylogeny of Paratelmatobiinae...","Predicting cyanobacterial biovolume from...","Gametes Simulator: A multilocus...","Using approximate Bayesian inference...","Fraud Detection in Electric...","Association between results of...","The discovery of Neotropical...","APPARENT BIAS: WHAT DOES...","The impact of anthropogenic...","Conservation planning in Uruguay...","Species diversity and distribution...","Coalescent-based species delimitation in...","Modeling the Accuracy of...","Millennium-length precipitation reconstruction over...","Detection of presumed genes...","Probability of Rhipicephalus microplus...","Phylogeny, Macroevolutionary Trends and...","Cryptic diversity in Brazilian...","Joint Minimization of Monitoring...","Genetic and pathogenic diversity...","A hybrid swarm of...","Fully Bayesian analysis of...","A historical perspective of...","Impact of Pleistocene geoclimatic...","An evolutionary insight into...","Mouth gape determines the...","Inferring population dynamics of...","Phylodynamics of Merkel-cell polyomavirus...","Action semantics at the...","Genetic structure, core collection,...","Prediction of complex traits:...","Human Bocavirus: Detection, Quantification...","The HIV-1 subtype B...","Kelps' long-distance dispersal: Role...","Joint denoising and decompression:...","Geographic distribution modeling of...","Latent class models for...","Historical isolation and contemporary...","Evidence of increasing diversification...","Amphitropical disjunctions in new...","The disjunct pattern of...","Tracing the origin of...","Evolutionarily significant units of...","Molecular adaptive convergence in...","Description of a new...","A new species of...","Uncertainty Quantification in a...","Trophic relationships in apex...","Phylodynamics of the major...","Trophic niche shifts driven...","Modeling a production scale...","Bayesian coalescent inference reveals...","New insights into the...","Weighting strategies for single-step...","Threatened amphibians of Uruguay:...","Hidden Markov models for...","Genetic structure of Colletotrichum...","Trophic Dynamics of the...","Genetic parameters for carcass...","Imprints of multiple glacial...","Genetic relationships among American...","Genetic relationships between Atlantic...","Genotype by environment interaction...","Genetic parameters of objectionable...","Automatic calibration of a...","Phylogenetic relationships between tuco-tucos...","Stable isotopes reveal long-Term...","Genetic diversity of the...","More on the mesopotamian-yungas...","Genetic diversification of local...","Molecular phylogeography of canine...","Description of the growth...","An evolutionary insight into...","Phylogenetic analysis of Hepatitis...","Phylogenetic studies of the...","Status, population trend and...","Genomic prediction for tick...","Sexual foraging segregation in...","Does the better-than-average effect...","Temperament and its heritability...","Bayesian age and growth...","Phylodynamics of major HIV-1...","Endemism and evolution in...","A phylogeny of the...","Spatiotemporal dynamics of the...","Climate change and the...","Temporal dynamics of norovirus...","Methods for genomic evaluation...","Thermal tolerances in rodents:...","Diversification in the south...","Comparative population genetics and...","Genetic diversity and population...","Applying spatial analysis of...","A multilocus perspective on...","Single Step, a general...","Functional diversification within a...","New records and distribution...","Bias in diet determination:...","The Phylogeographic History of...","Estimates of genetic parameters...","Tracing the Origin and...","Quaternary range and demographic...","Molecular characterisation of hepatitis...","Evolution of optimal codon...","Fine-Scale Population Structure of...","Dating an impressive Neotropical...","Bayesian coalescent analysis of...","Exploring polymorphisms in B-DNA...","Species delimitation with abc...","Patterns of population differentiation...","Genetic parameters for nematode...","Genetic variability and demographic...","Phylogeography of the sub-Antarctic...","Diversification of the American...","Phylogenetics of the chiliotrichum...","Protected areas do not...","Population genetic structure and...","Serological monitoring of selected...","Assessing the seroprevalence for...","The evolution of mitochondrial...","Apparent overconfidence...","Comparing Bayesian inference and...","Species groups and the...","Genome-wide marker-assisted selection combining...","Evaluation of case based...","Phenotypic and genetic characterization...","Evolution of Dengue virus...","Change in the foraging...","Phylodynamics of HIV-1 Circulating...","Dispersal and population structure...","Genetic history of hepatitis...","L2-Boosting algorithm applied to...","Assessing the seroprevalence against...","Flock-level seroprevalence against avian...","Flock-level seroprevalence against Ornithobacterium...","Modeling gene sequences over...","Bayesian coalescent inference reveals...","Predicting quantitative traits with...","Modeling gene sequence changes...","Predictive ability of direct...","Monitoring the infectious disease...","Phylogeography of the Southwestern...","Phylogeography of the Neotropical...","A comparison between Poisson...","Bayesian coalescent inference of...","Genetic variability in calving...","Genetic and environment factors...","Phylogeography of the deer...","Inferring parameters shaping amino...","Model comparison for genetic...","Evaluation of extent and...","Assessing relationships between genetic...","Bioequivalence study of theophylline..."],"paises":["Uruguay","Uruguay","Uruguay","Uruguay","Uruguay","Uruguay","Uruguay","Uruguay","Uruguay","Uruguay","Uruguay","Uruguay","United Kingdom; Uruguay","United Kingdom; United States; Uruguay","United Kingdom; Uruguay","United Kingdom; Uruguay; Sweden","United States; Uruguay","United Kingdom; Uruguay; Brazil","Uruguay","Uruguay","Uruguay","Uruguay; Spain","United Kingdom; Uruguay; Brazil","Chile; Uruguay","United Kingdom; Uruguay","Uruguay; New Zealand","United Kingdom; Uruguay","United Kingdom; Uruguay","United Kingdom; Uruguay","Uruguay","Uruguay","Uruguay","United Kingdom; Uruguay","Uruguay","Uruguay","Uruguay","Uruguay","Uruguay","Uruguay","United Kingdom; Uruguay","Uruguay","Uruguay","Uruguay","Uruguay","Uruguay","United Kingdom; Uruguay","Uruguay; Spain","Uruguay","Uruguay","Uruguay","Uruguay","Uruguay","Uruguay","Uruguay","Uruguay","Uruguay","Uruguay","Uruguay","United Kingdom; Uruguay","United Kingdom; Uruguay","Uruguay","Uruguay","Uruguay","Uruguay","United Kingdom; Uruguay","United Kingdom; Uruguay","United Kingdom; Uruguay","United Kingdom; Uruguay","Uruguay","Uruguay","Uruguay","Uruguay","Uruguay","United Kingdom; Uruguay","Uruguay","United Kingdom; Uruguay","Uruguay","United Kingdom; Uruguay","United Kingdom; Uruguay","United Kingdom; Uruguay","Uruguay","Uruguay","Uruguay","Uruguay","United States; Uruguay","Uruguay","United Kingdom; Uruguay","Uruguay","Uruguay","Uruguay","Uruguay","Uruguay","Uruguay","Uruguay","United Kingdom; Uruguay","United States; Uruguay","Uruguay","United Kingdom; Uruguay","Uruguay","Uruguay","Argentina; Uruguay","Uruguay","Uruguay","Uruguay; Germany","United Kingdom; Uruguay","Uruguay","Uruguay","Uruguay","Uruguay","United Kingdom; Uruguay","Uruguay","United Kingdom; Uruguay","Uruguay","United Kingdom; Uruguay","Uruguay","United Kingdom; Uruguay","United Kingdom; Uruguay","United Kingdom; Uruguay; Ecuador; Spain","United Kingdom; Uruguay","Uruguay","Uruguay","Uruguay","United Kingdom; Uruguay; Brazil","Uruguay; Brazil","Argentina; Uruguay","Uruguay","Uruguay","Uruguay","Uruguay","Uruguay","United Kingdom; Uruguay","Uruguay","Argentina; Uruguay","Uruguay","Uruguay","Uruguay","Australia; Uruguay","Uruguay","United Kingdom; Uruguay","Uruguay","Uruguay","Uruguay","Uruguay","Uruguay","Uruguay","Uruguay","Uruguay","Uruguay","Uruguay","Colombia; Uruguay","Uruguay","Uruguay","Uruguay","United Kingdom; Uruguay","Uruguay","United Kingdom; Uruguay","Uruguay","Uruguay","Uruguay; Mexico","Uruguay","Uruguay","United Kingdom; Uruguay","Uruguay; Spain","United Kingdom; Uruguay","United Kingdom; Uruguay","Uruguay","Uruguay; Brazil","Uruguay","Argentina; Uruguay","Argentina; Uruguay","Argentina; Uruguay","Argentina; Uruguay","Argentina; Uruguay","Uruguay","Uruguay","Uruguay","Uruguay","Uruguay","Uruguay","Argentina; Uruguay","Argentina; Uruguay","Uruguay",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null],"afiliaciones":["Centro Interdisciplinario de Ciencia de Datos y Aprendizaje Automático, Universidad de la República, Montevideo, Uruguay | Laboratorio de Neurociencias, Facultad de Ciencias, Universidad de la República","Sección Genética Evolutiva, Departamento de Biología Animal, Instituto de Biología, Facultad de Ciencias, Universidad de la República, Iguá 4225, 11400, Montevideo, Uruguay","Sección Genética Evolutiva, Departamento de Biología Animal, Instituto de Biología, Facultad de Ciencias, Universidad de la República, Iguá 4225, 11400, Montevideo, Uruguay","Sección Genética Evolutiva, Departamento de Biología Animal, Instituto de Biología, Facultad de Ciencias, Universidad de la República, Iguá 4225, 11400, Montevideo, Uruguay","Sección Genética Evolutiva, Departamento de Biología Animal, Instituto de Biología, Facultad de Ciencias, Universidad de la República, Iguá 4225, 11400, Montevideo, Uruguay","Sección Genética Evolutiva, Departamento de Biología Animal, Instituto de Biología, Facultad de Ciencias, Universidad de la República, Iguá 4225, 11400, Montevideo, Uruguay","Sección Genética Evolutiva, Departamento de Biología Animal, Instituto de Biología, Facultad de Ciencias, Universidad de la República, Iguá 4225, 11400, Montevideo, Uruguay","Sección Genética Evolutiva, Departamento de Biología Animal, Instituto de Biología, Facultad de Ciencias, Universidad de la República, Iguá 4225, 11400, Montevideo, Uruguay | Sección Genética Evolutiva, Instituto de Biología, Facultad de Ciencias, Universidad de la República, Montevideo, Uruguay","Unidad de Clínica y Hospital Veterinario, Facultad de Veterinaria, Universidad de la República, Montevideo, Uruguay","Sección Genética Evolutiva, Departamento de Biología Animal, Instituto de Biología, Facultad de Ciencias, Universidad de la República, Iguá 4225, 11400, Montevideo, Uruguay | Sección Genética Evolutiva, Instituto de Biología, Facultad de Ciencias, Universidad de la República, Montevideo, Uruguay","Sección Genética Evolutiva, Departamento de Biología Animal, Instituto de Biología, Facultad de Ciencias, Universidad de la República, Iguá 4225, 11400, Montevideo, Uruguay | Sección Genética Evolutiva, Instituto de Biología, Facultad de Ciencias, Universidad de la República, Montevideo, Uruguay","Departamento Ovinos, Lanas y Caprinos, Facultad de Veterinaria, Universidad de la República, Montevideo, 11600, Uruguay | Unidad académica Ovinos, Lanas y Caprinos, Departamento de Producción Animal y Salud de los Sistemas Productivos, Facultad de Veterinaria, Universidad de la República, Ruta 1, km 42,5- Libertad, San José, Uruguay","| Departamento de Ovinos Y Lanas, Facultad de Veterinaria, Montevideo 11600, Av. Lasplaces 1550, Uruguay | Unidad académica Ovinos, Lanas y Caprinos, Departamento de Producción Animal y Salud de los Sistemas Productivos, Facultad de Veterinaria, Universidad de la República, Ruta 1, km 42,5- Libertad, San José, Uruguay","| Unidad de Bioinformática, Institut Pasteur de Montevideo, Mataojo 2020, Montevideo 11400, Uruguay | Unidad de Bioinformática, Institut Pasteur de Montevideo, 11400 Montevideo, Mataojo 2020, Uruguay | Departamento de Producción Animal Y Pasturas, Facultad de Agronomía, Universidad de la República, Montevideo, Uruguay | Unidad de Bioinformática, Institut Pasteur de Montevideo, Mataojo 2020, Montevideo, Uruguay | Department of Animal Sciences, University of Wisconsin-Madison, Madison, WI 53706, United States | Unidad de Bioinformática, Instituto Pasteur de Montevideo | Bioinformatics Unit, Institut Pasteur, 11400 Montevideo, Uruguay | Laboratorio de Organización Y Evolución del Genoma, Departamento de Biología Celular Y Molecular, Facultad de Ciencias | Departamento de Producción Animal Y Pasturas, Facultad de Agronomía, Universidad de la República, 12900 Montevideo, Garzón 780, Uruguay | Institut Pasteur de Montevideo | Bioinformatics Unit, Institut Pasteur de Montevideo, Mataojo 2020, Montevideo, 11400, Uruguay | Departamento de Producción Animal Y Pasturas, Facultad de Agronomía | Departmento de Producción Animal y Pasturas, Facultad de Agronomía, Universidad de la República, Av. Gral. Eugenio Garzón 780, Montevideo, 12900, Uruguay | Department of Animal Sciences, University of Wisconsin-Madison, United States | Departamento de Producción Animal y Pasturas, Facultad de Agronomía | Departamento de Producción Animal Y Pasturas, Facultad de Agronomía, Montevideo 12900, Av. Garzón 780, Uruguay","| Departamento Ovinos, Lanas y Caprinos, Facultad de Veterinaria, Universidad de la República, Montevideo, 11600, Uruguay","| Departamento de Producción Animal Y Pasturas, Facultad de Agronomía, Universidad de la República, 12900 Montevideo, Garzón 780, Uruguay | Cancer Genetics Building, St. James' University Hospital, Leeds LS9 7TF, Beckett Street, United Kingdom | Departamento de Producción Animal Y Pasturas, Facultad de Agronomía | Departamento de Producción Animal y Pasturas, Facultad de Agronomía | Department of Animal Breeding and Genetics, Swedish University of Agricultural Sciences, S-75007 Uppsala, PO Box 7023, Sweden | Departamento de Producción Animal Y Pasturas, Facultad de Agronomía, Montevideo 12900, Av. Garzón 780, Uruguay | Departamento de Producción Animal y Pasturas, Fac. de Agronomía, UDELAR, 12900 Montevideo, Garzón 780, Uruguay","Minnesota Center for Prion Research and Outreach, University of Minnesota, Saint Paul, 55108, MN, United States | Department of Veterinary Population Medicine, University of Minnesota, Saint Paul, 55108, MN, United States | Department of Veterinary Population Medicine, University of Minnesota, Saint Paul, MN, United States | Facultad de Veterinaria, Universidad de la Republica, Montevideo, 11200, Uruguay | Facultad de Veterinaria, Universidad de la Republica, Montevideo, Uruguay","| Programa de Pós-Graduação em Zoologia, Universidade Federal de Minas Gerais | Programa de Pós-Graduação em Zoologia, Instituto de Ciências Biológicas, Universidade Federal de Minas Gerais, Avenida Antônio Carlos, 6627, Belo Horizonte, 31270-010, MG, Brazil | Programa de Pós-Graduação em Zoologia, Instituto de Ciências Biológicas, Universidade Federal de Minas Gerais – UFMG, Minas Gerais, Belo Horizonte, Brazil | Departamento de Zoologia, ICB, Universidade Federal de Minas Gerais, Av. Antônio Carlos, 6627, Belo Horizonte, 31270-901, MG, Brazil | Departamento de Ciências Naturais, campus Dom Bosco, Universidade Federal de São João del-Rei, São João del-Rei, Brazil | Departamento de Ciências Naturais (DCNAT), Campus Dom Bosco, Universidade Federal de São João del-Rei (UFSJ), Praça Dom Helvécio, 74, São João del-Rei, 36301-160, MG, Brazil | Departamento de Ciências Naturais, Campus Dom Bosco, Universidade Federal de São João del-Rei, São João del Rei, 36301-160, MG, Brazil | Programa de Desarrollo Universitario, Centro Universitario de Rivera, Universidad de la República – UdelaR, Rivera, Uruguay","Instituto Nacional de Investigación Agropecuaria (INIA), Programa Nacional Producción de Carne y Lana, Ruta 5 Km 386, Tacuarembó, 45000, Uruguay","Departamento de Ecología y Biología Evolutiva, Instituto de Investigaciones Biológicas Clemente Estable (IIBCE), Ministerio de Educación y Cultura (MEC), Avenida Italia 3318, Montevideo, PC 11600, Uruguay","Departamento de Ecología y Biología Evolutiva, Instituto de Investigaciones Biológicas Clemente Estable (IIBCE), Ministerio de Educación y Cultura (MEC), Avenida Italia 3318, Montevideo, PC 11600, Uruguay | Departamento de Biodiversidad y Genética, Instituto de Investigaciones Biológicas Clemente Estable (IIBCE), Ministerio de Educación y Cultura (MEC), Avenida Italia 3318, Montevideo, PC 11600, Uruguay","Departamento de Biodiversidad y Genética, Instituto de Investigaciones Biológicas Clemente Estable (IIBCE), Ministerio de Educación y Cultura (MEC), Avenida Italia 3318, Montevideo, PC 11600, Uruguay | Departament de Biologia Animal, Facultat de Biologia, Universitat de Barcelona, Av. Diagonal 643, Barcelona, 08028, Spain | Laboratorio de Etologíıa, Ecologíıa y Evoluciíon, Instituto de Investigaciones Biolíogicas Clemente Estable, Montevideo, Uruguay | Institut de Recerca de la Biodiversitat, Universitat de Barcelona, Av. Diagonal 643, Barcelona, 08028, Spain | Institut de Recerca de la Biodiversitat & Departament de Biologia Animal, Universitat de Barcelona, Barcelona, Spain | Laboratorio de Etología, Ecología y Evolución, Instituto de Investigaciones Biológicas Clemente Estable, Avenida Italia 3318, Montevideo, 11600, Uruguay","| Institut Pasteur de Montevideo, Microbial Genomics Laboratory, Montevideo, Uruguay | Unidad de Bioinform atica, Institut Pasteur de Montevideo | Departamento de Entomologia e Acarologia, Escola Superior de Agricultura 'Luiz de Queiroz', USP, Piracicaba, SP 13418-900, Avenida Pádua Dias, 11, Brazil | Institut Pasteur de Montevideo, Pasteur + INIA Joint Unit (UMPI), Montevideo, Uruguay | Unidad de Bioinformatica, Institut Pasteur de Montevideo, Mataojo, 2020, Montevideo, Uruguay","Laboratorio de Evolución, Facultad de Ciencias, Universidad de la República | Microbial Genomics Laboratory, Institut Pasteur Montevideo, Mataojo 2020, Montevideo, 11400, Uruguay | Departamento de Ecología, Facultad de Ciencias Biológicas, Pontificia Universidad Católica de Chile, Santiago 6513677, Avda. Libertador Bernardo O'Higgins 340, Chile | Departamento de Ecología, Center for Advanced Studies in Ecology and Biodiversity, Pontificia Universidad Católica, Santiago, Chile | Instituto de Ciencias Ambientales y Evolutivas, Facultad de Ciencias, Universidad Austral de Chile, Campus Isla Teja s/n, Valdivia, 5090000, Chile","| Laboratorio de Vectores y Enfermedades Transmitidas, Departamento de Ciencias Biológicas, CENUR Litoral Norte—Salto, Universidad de la República, Rivera 1350, Salto, 50000, Uruguay | Laboratorio de Vectores Y Enfermedades Transmitidas, Departamento de Ciencias Biológicas, CENUR Litoral Norte, Universidad de La República, Rivera 1350, Salto, CP 50000, Uruguay","AgResearch, Grasslands Research Centre, Private Bag 11008, Palmerston North, 4442, New Zealand | Laboratorio de Vectores Y Enfermedades Transmitidas, Departamento de Ciencias Biológicas, CENUR Litoral Norte, Universidad de La República, Rivera 1350, Salto, CP 50000, Uruguay","| CENUR Noreste, Universidad de La República, Ituzaingó 667, Rivera, CP 40000, Uruguay | Centro Universitario de Rivera, Universidad de la República","| Laboratorio de Vectores y enfermedades transmitidas, Facultad de Veterinaria, CENUR Litoral Norte, Universidad de la República | Laboratorio de Vectores y Enfermedades Transmitidas, Facultad de Veterinaria, CENUR Litoral Norte - Salto, Universidad de la República, Rivera 1350, Salto, 50000, Uruguay | Laboratorio de Vectores Y Enfermedades Transmitidas, Departamento de Ciencias Biológicas, CENUR Litoral Norte, Universidad de La República, Rivera 1350, Salto, CP 50000, Uruguay | Institute of Parasitology, Vetsuisse Faculty, University of Zurich","| Laboratorio de Vectores y Enfermedades Transmitidas, Departamento de Ciencias Biológicas, CENUR Litoral Norte—Salto, Universidad de la República, Rivera 1350, Salto, 50000, Uruguay | Laboratorio de Vectores y enfermedades transmitidas, Facultad de Veterinaria, CENUR Litoral Norte, Universidad de la República | Laboratorio de Vectores Y Enfermedades Transmitidas, Departamento de Ciencias Biológicas, CENUR Litoral Norte, Universidad de La República, Rivera 1350, Salto, CP 50000, Uruguay","Laboratorio de Vectores Y Enfermedades Transmitidas, Departamento de Ciencias Biológicas, CENUR Litoral Norte, Universidad de La República, Rivera 1350, Salto, CP 50000, Uruguay","Laboratorio de Vectores Y Enfermedades Transmitidas, Departamento de Ciencias Biológicas, CENUR Litoral Norte, Universidad de La República, Rivera 1350, Salto, CP 50000, Uruguay","Espacio de Biología Vegetal del Noreste, Centro Universitario de Tacuarembó, Universidad de la República, Tacuarembó, Uruguay | Departamento de Ecología y Evolución, Facultad de Ciencias, Universidad de la República, Montevideo, Uruguay","| Laboratorio de Evolución, Facultad de Ciencias, Universidad de la República | Laboratorio de Evolución, Facultad de Ciencias, Universidad de la Repúplica, Montevideo 11400, Iguá 4225, Uruguay | Departamento de Ecología y Evolución, Facultad de Ciencias, Universidad de la República, Montevideo, Uruguay | Departamento de Ecología y Evolución Facultad de Ciencias, Universidad de la República, 11400 Montevideo, Uruguay","Instituto de Mecánica de Los Fluidos e Ingeniería Ambiental, Facultad de Ingeniería, Universidad de la República, Julio Herrera y Reissig 565, Montevideo, 11300, Uruguay | Universidad de la República, Facultad de Ingeniería Instituto de, Mecánica de los Fluidos e Ingeniería Ambiental, Julio Herrera y Reissig, 565, CP, Montevideo, 11300, Uruguay","Instituto de Mecánica de Los Fluidos e Ingeniería Ambiental, Facultad de Ingeniería, Universidad de la República, Julio Herrera y Reissig 565, Montevideo, 11300, Uruguay | Universidad de la República, Facultad de Ingeniería Instituto de, Mecánica de los Fluidos e Ingeniería Ambiental, Julio Herrera y Reissig, 565, CP, Montevideo, 11300, Uruguay","Centro Universitario Regional Este (CURE), Universidad de la República | UNDECIMAR, Departamento de Ecología y Evolución, Facultad de Ciencias, Universidad de La República, Iguá 4225, Montevideo, 11400, Uruguay | UNDECIMAR, Facultad de Ciencias, Iguá 4225, Montevideo, 11400, Uruguay","UNDECIMAR, Departamento de Ecología y Evolución, Facultad de Ciencias, Universidad de La República, Iguá 4225, Montevideo, 11400, Uruguay","Centro Universitario Regional Este (CURE), Universidad de la República | UNDECIMAR, Departamento de Ecología y Evolución, Facultad de Ciencias, Universidad de La República, Iguá 4225, Montevideo, 11400, Uruguay | UNDECIMAR, Facultad de Ciencias, Iguá 4225, Montevideo, 11400, Uruguay","Bioinformatics Unit, Institut Pasteur de Montevideo, Montevideo, 11400, Uruguay | Bioinformatics Unit, Institut Pasteur de Montevideo, Mataojo 2020, Montevideo, 11400, Uruguay","| Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, Universidad de la República | Facultad de Ciencias, Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Universidad de la Republica | Laboratorio de Evolución Experimental de Virus, Institut Pasteur de Montevideo | Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, 11400 Montevideo, Igua 4225, Uruguay | Laboratorio de Virología Molecular, Facultad de Ciencias, Universidad de la República | Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, Universidad de la República, Iguá 4225, Montevideo, 11400, Uruguay | Laboratorio de Virologia Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, Universidad de la Republica, Igua 4225, Montevideo, 11400, Uruguay | Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, Igua 4225, 11400 Montevideo, Uruguay","Department of Exact and Natural Sciences, Universidad Católica del Uruguay | Bioinformatics Unit, Institut Pasteur de Montevideo, Montevideo, 11400, Uruguay","Laboratorio de Biología Molecular, Sanatorio Americano, Montevideo, 11600, Uruguay","Protein Engineering, Institut Pasteur de Montevideo, Montevideo, 11400, Uruguay","Bioinformatics Unit, Institut Pasteur de Montevideo, Montevideo, 11400, Uruguay","Laboratorio de Evolución Experimental de Virus, Institut Pasteur de Montevideo | Laboratorio de Biología Molecular, Sanatorio Americano, Montevideo, 11600, Uruguay | Laboratorio de Virología Molecular, Facultad de Ciencias, Universidad de la República","| Instituto Nacional de Investigación Agropecuaria, INIA Las Brujas, Rincón del Colorado, Canelones, Ruta 48, km 10, Uruguay | Instituto Nacional de Investigación Agropecuaria, INIA Las Brujas, Ruta 48, km 10, Rincón del Colorado, Canelones, 90200, Uruguay","Estación Biológica Terra Natura, CIBIO, Universidad de Alicante, Apdo. Correos 99, E-03080 Alicante, Campus San Vicente del Raspeig, s/n, Spain | Área de Biodiversidad y Conservación, Museo Nacional de Historia Natural, CP 11000 Montevideo, Casilla de Correo 399, Uruguay | Departamento de Ecología y Gestión Ambiental, Centro Universitario Regional del Este, Universidad de la República, Calle Tacuarembó esquina Bvar. Artigas, Maldonado, 20000, Uruguay | Laboratorio de Etología, Ecología and Evolución, Instituto de Investigaciones Biológicas Clemente Estable, CP 11600 Montevideo, Avenida Italia 3318, Uruguay","Laboratorio de Control Ambiental, Polo Educativo Tecnológico Arrayanes, Camino de los Arrayanes km 7, Piriápolis, 20200, Uruguay","Departamento de Biodiversidad y Genética, Instituto de Investigaciones Biológicas Clemente Estable, Ministerio de Educación y Cultura, Avenida Italia 3318, Montevideo, 11600, Uruguay","Departamento de Biodiversidad y Genética, Instituto de Investigaciones Biológicas Clemente Estable, Ministerio de Educación y Cultura, Avenida Italia 3318, Montevideo, 11600, Uruguay","Departamento de Biodiversidad y Genética, Instituto de Investigaciones Biológicas Clemente Estable, Ministerio de Educación y Cultura, Avenida Italia 3318, Montevideo, 11600, Uruguay","Instituto de Mecánica de los Fluidos e Ingeniería Ambiental (IMFIA), Facultad de Ingeniería, Universidad de la República, Montevideo, 11300, Uruguay","Instituto de Computación (InCo), Facultad de Ingeniería, Universidad de la República, Montevideo, 11300, Uruguay","Instituto de Computación (InCo), Facultad de Ingeniería, Universidad de la República, Montevideo, 11300, Uruguay","Instituto de Mecánica de los Fluidos e Ingeniería Ambiental (IMFIA), Facultad de Ingeniería, Universidad de la República, Montevideo, 11300, Uruguay","Instituto de Mecánica de los Fluidos e Ingeniería Ambiental (IMFIA), Facultad de Ingeniería, Universidad de la República, Montevideo, 11300, Uruguay","Instituto de Computación (InCo), Facultad de Ingeniería, Universidad de la República, Montevideo, 11300, Uruguay","Instituto de Mecánica de los Fluidos e Ingeniería Ambiental (IMFIA), Facultad de Ingeniería, Universidad de la República, Montevideo, 11300, Uruguay","| Institute of Physics, Faculty of Science, Universidad de la República, Iguá 4225, Montevideo, 11400, Uruguay | Institute of Physics, Faculty of Science, Universidad de la República","| Departamento de Paleontología, Facultad de Ciencias, Universidad de La República, Iguá 4225, Montevideo, 11400, Uruguay | Servicio Académico Universitario y Centro de Estudio Paleontológicos (SAUCE-P), Universidad de La República, Departamento de Canelones, Santa Isabel S/n, Sauce, 91500, Uruguay","Servicio Académico Universitario y Centro de Estudio Paleontológicos (SAUCE-P), Universidad de La República, Departamento de Canelones, Santa Isabel S/n, Sauce, 91500, Uruguay | Departamento de Paleontología, Facultad de Ciencias, Universidad de la República, Iguá 4225, Montevideo, 11400, Uruguay | Departamento de Paleontología, Facultad de Ciencias, Universidad de La República, Iguá 4225, Montevideo, 11400, Uruguay","Servicio Académico Universitario y Centro de Estudio Paleontológicos (SAUCE-P), Universidad de La República, Departamento de Canelones, Santa Isabel S/n, Sauce, 91500, Uruguay | Departamento de Paleontología, Facultad de Ciencias, Universidad de la República, Iguá 4225, Montevideo, 11400, Uruguay | Departamento de Paleontología, Facultad de Ciencias, Universidad de La República, Iguá 4225, Montevideo, 11400, Uruguay","Departamento del Agua, CENUR Litoral Norte Universidad de la República | Departamento de Ecología y Biología Evolutiva, Instituto de Investigaciones Biológicas Clemente Estable, Montevideo, Uruguay | Departamento de Ecología y Biología Evolutiva, Instituto de Investigaciones Biológicas Clemente Estable, MEC, Montevideo, Uruguay | Departamento del Agua CENUR Ltoral Norte sede Paysandú, Universidad de la República, Ruta 3 Km 363 EEMAC, Paysandú, 60000, Uruguay","Departamento de Ecología y Evolución, Facultad de Ciencias, Universidad de la República, Montevideo, Uruguay","| Departamento de Ecología y Evolución, Facultad de Ciencias, Universidad de la República, Montevideo 11400, Iguá 4225, Uruguay | Departamento de Ecología y Evolución, Facultad de Ciencias, Universidad de la República, Iguá 4225, Montevideo, 11400, Uruguay | Departamento de Ecología y Evolución, Facultad de Ciencias, Universidad de la República, Montevideo, Uruguay","| Department of Animal Breeding, Instituto Nacional de Investigación Agropecuaria | Instituto Nacional de Investigacion Agropecuaria, Las Brujas 90200, Uruguay | Instituto Nacional de Investigación Agropecuaria (INIA), Montevideo, 11500, Uruguay | Department of Animal and Dairy Science, University of Georgia | Instituto Nacional de Investigación Agropecuaria (INIA) Las Brujas | National Agricultural Research Institute, Las Brujas, Uruguay | Instituto Nacional de Investigación Agropecuaria (INIA), Rincón del Colorado, CA, 90200, Uruguay | Instituto Nacional de Investigación Agropecuaria, Las Brujas 90200, Uruguay","| Departamento de Informática y Ciencias de la Computación, Universidad Católica Del Uruguay, CP 11600, Montevideo, Uruguay","| Clínica Pediátrica, Hospital Central FFAA, CP 11600, Montevideo, Uruguay | Centro de Referencia Nacional de VIH-SIDA, Hospital Pereira-Rossell","Pharmaceutical Sciences Department, Faculty of Chemistry, Universidad de la República, Uruguay, General Flores, Montevideo, 2124, Uruguay","Pharmaceutical Sciences Department, Faculty of Chemistry, Universidad de la República, Uruguay, General Flores, Montevideo, 2124, Uruguay","Catedra de Farmacologia/Biofarmacia, Facultad de Quimica, 11800 Montevideo, CC 1157, Uruguay | Pharmaceutical Sciences Department, Faculty of Chemistry, Universidad de la República, Uruguay, General Flores, Montevideo, 2124, Uruguay","Pharmaceutical Sciences Department, Faculty of Chemistry, Universidad de la República, Uruguay, General Flores, Montevideo, 2124, Uruguay","Pharmaceutical Sciences Department, Faculty of Chemistry, Universidad de la República, Uruguay, General Flores, Montevideo, 2124, Uruguay","| Facultad de Veterinaria, Universidad de la Republica, Alberto Lasplaces 1550, Montevideo, 11600, CP, Uruguay | Instituto Nacional de Investigación Agropecuaria (INIA), Plataforma de Salud Animal, Estación Experimental INIA Tacuarembó, Ruta 5 km 386, Tacuarembó, 45000, CP, Uruguay","Laboratorio Regional Noroeste “Miguel C. Rubino”, División de Laboratorios Veterinarios “Miguel C. Rubino”, Paysandú, Uruguay","| Facultad de Veterinaria, Universidad de la Republica, Alberto Lasplaces 1550, Montevideo, 11600, CP, Uruguay | Instituto Nacional de Investigación Agropecuaria (INIA), Plataforma de Salud Animal, Estación Experimental INIA Tacuarembó, Ruta 5 km 386, Tacuarembó, 45000, CP, Uruguay","Facultad de Veterinaria, Universidad de la Republica, Alberto Lasplaces 1550, Montevideo, 11600, CP, Uruguay | Instituto Nacional de Investigación Agropecuaria, Plataforma de Salud Animal, Estación Experimental INIA, Tacuarembó, Uruguay | Instituto Nacional de Investigación Agropecuaria (INIA), Plataforma de Salud Animal, Estación Experimental INIA Tacuarembó, Ruta 5 km 386, Tacuarembó, 45000, CP, Uruguay","| Instituto Nacional de Investigación Agropecuaria (INIA) Las Brujas, Uruguay | Instituto Nacional de Investigación Agropecuaria, INIA Las Brujas, Rincón del Colorado, Canelones, Ruta 48, km 10, Uruguay | Instituto Nacional de Investigación Agropecuaria, INIA Las Brujas, Ruta 48, km 10, Rincón del Colorado, Canelones, 90200, Uruguay","| Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, 11400 Montevideo, Iguá 4225, Uruguay | Laboratorio de Virología Molecular, Centro Universitario Regional Noroeste (CENUR Noroeste), Universidad de la Republica (UdelaR) | Laboratorio de Virología Molecular, CENUR Litoral Norte, Sede Salto, Universidad de la República, Uruguay | Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, 11400 Montevideo, Igua 4225, Uruguay","| Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, 11400 Montevideo, Iguá 4225, Uruguay | Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, Universidad de la República | Facultad de Ciencias, Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Universidad de la Republica | Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, 11400 Montevideo, Igua 4225, Uruguay | Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, Universidad de la República, Iguá 4225, Montevideo, 11400, Uruguay | Laboratorio de Virologia Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, Universidad de la Republica, Igua 4225, Montevideo, 11400, Uruguay | Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias Universidad de la República, Igua 4225, 11400 Montevideo, Uruguay | Molecular Virology Laboratory, CIN, Facultad de Ciencias, Universid ad de la Republica","Instituto Nacional de Investigación Agropecuaria (INIA), Programa Producción de Carne y Lana, Estación Experimental INIA Treinta y Tres, Ruta 8 km 281, 33000, Treinta y Tres, Uruguay","Instituto Nacional de Investigación Agropecuaria (INIA), Programa Producción de Carne y Lana, Estación Experimental INIA Treinta y Tres, Ruta 8 km 281, 33000, Treinta y Tres, Uruguay","Instituto Nacional de Investigación Agropecuaria (INIA), Programa Producción de Carne y Lana, Estación Experimental INIA Treinta y Tres, Ruta 8 km 281, 33000, Treinta y Tres, Uruguay","Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, Universidad de la República | Laboratorio de Inmunovirología, Institut Pasteur de Montevideo, Montevideo, Uruguay","Section of Plant Breeding and Genetics, Cornell University, Ithaca, NY, United States | Laboratory of Plant Evolution and Domestication, Department of Plant Biology, Universidad de la República, Av. E. Garzón 780, Montevideo, CP12900, Uruguay","Physiology and Ecology Phytoplankton Group, CSIC 1176, Uruguay | Sección Limnología, Instituto de Ecología y Ciencias Ambientales, Facultad de Ciencias, Universidad de la República, Iguá 4225, Montevideo, 11400, Uruguay","| Physiology and Ecology Phytoplankton Group, CSIC 1176, Uruguay","Physiology and Ecology Phytoplankton Group, CSIC 1176, Uruguay | Sección Limnología, Instituto de Ecología y Ciencias Ambientales, Facultad de Ciencias, Universidad de la República","Instituto Nacional de Investigación Agropecuaria, Colonia, 70000, Uruguay","Dep. de Producción Vegetal, Centro Regional Sur (CRS), Facultad de Agronomía, Univ. de la República, Progreso, Canelones, Uruguay | Department of Plant Production, Centro Regional sur (CRS), Universidad de la República, Camino Folle km 36 Progreso, Uruguay | Universidad de la República, Facultad de Agronomía, Departamento de Producción Vegetal, Centro Regional Sur, Canelones, 90100, Uruguay","Instituto Nacional de Investigación Agropecuaria, Unidad de Semillas y Recursos Fitogenéticos, Estación Experimental La Estanzuela, Colonia, 70000, Uruguay","Universidad de la República | Instituto de Estadística, Universidad de la República, Montevideo, Uruguay","Universidad de la República, Montevideo, 11200, Uruguay","Sección Biomatemática-Facultad de Ciencias, Universidad de la República | Sección Biomatemática, Unidad de Genómica Evolutiva, Facultad de Ciencias, Universidad de la República, Montevideo, Uruguay","| Dept. of Economics, Universidad de Montevideo, Montevideo, CP 11400, Prudencio de Pena 2440, Uruguay | Universidad de Montevideo","Purdue University, Department of Forestry and Natural Resources, 715 West State Street, West Lafayette, 47907, IN, United States | Purdue University, Ecological Sciences and Engineering Program, 715 West State Street, West Lafayette, 47907, IN, United States | Centro de Investigación y Conservación Marina - CICMAR, Avenida Giannattasio Km 30.5, Canelones, 15008, Uruguay","Facultad de Ciencias, Universidad de la República, Iguá 4225, Montevideo, 11400, Uruguay","| Museo Nacional de Historia Natural, 25 de mayo 582, Montevideo, CEP 11000, Uruguay","Laboratorio de Sistemática e Historia Natural de Vertebrados, Instituto de Ecología y Ciencias Ambientales, Facultad de Ciencias, UdelaR, Iguá 4225, Montevideo, Uruguay","Laboratorio de Sistemática e Historia Natural de Vertebrados IECA, Facultad de Ciencias, Universidad de la República | Laboratorio de Sistemática e Historia Natural de Vertebrados, Instituto de Ecología y Ciencias Ambientales, Facultad de Ciencias, UdelaR, Iguá 4225, Montevideo, Uruguay | Laboratorio de Sistemática e Historia Natural de Vertebrados, Facultad de Ciencias, Universidad de la República","Department of Biology and Monte L. Bean Museum, Brigham Young University | Unidad de Diversidad Sistemática y Evoluciõn, Centro Nacional Patagõnico, Consejo Nacional de Investigaciones Científicas y Técnicas, U9120ACD Puerto Madryn Chubut, Boulevard Almirante Brown 2915, Argentina | Programa de Desarrollo Universitario, Centro Universitario de Rivera, Universidad de la República – UdelaR, Rivera, Uruguay | Centro Universitario de Rivera, Universidad de la República","División Laboratorios Veterinarios “Miguel C. Rubino”, Ministerio de Ganadería, Agricultura y Pesca, Montevideo, Uruguay","División Laboratorios Veterinarios “Miguel C. Rubino”, Ministerio de Ganadería, Agricultura y Pesca, Montevideo, Uruguay","Department of Geography, Climatology, Climate Dynamics and Climate Change, Justus Liebig University of Giessen, Giessen, Germany | Physics Institute, Science Faculty, Universidad de la República, Montevideo, Uruguay","| Departamento de Ecología Y Evolucion, Facultad de Ciencias-CURE, Universidad de la Republica, Montevideo, Uruguay | Laboratorio de Organización Y Evolución del Genoma, Departamento de Biología Celular Y Molecular, Facultad de Ciencias | Laboratorio de Organización y Evolución del Genoma, UDELAR | Escuela Universitaria de Tecnología Médica, Facultad de Medicina | Genome Organization and Evolution Laboratory, Ecology and Evolution Department, Faculty of Sciences, UdelaR, Montevideo, 11400, Uruguay | Laboratorio de Organizacion y Evolución del Genoma, Sección Biomatemáticas, Universidad de la República, Montevideo, Uruguay","Facultad de Veterinaria, Universidad de la Republica, Alberto Lasplaces 1550, Montevideo, 11600, CP, Uruguay | Instituto Nacional de Investigación Agropecuaria (INIA), Plataforma de Salud Animal, Estación Experimental INIA Tacuarembó, Ruta 5 km 386, Tacuarembó, 45000, CP, Uruguay","Facultad de Veterinaria, Universidad de la Republica, Alberto Lasplaces 1550, Montevideo, 11600, CP, Uruguay","Facultad de Ingeniería, Universidad de la República, Montevideo, Uruguay","Department of Plant Protection, Faculty of Agronomy, University of the Republic, Av. Garz on 780, Montevideo, CP 12900, Uruguay | Department of Plant Protection, Faculty of Agronomy, University of the Republic, Av. Garzón 780, Montevideo, CP 12900, Uruguay","| Department of Plant Protection, Faculty of Agronomy, University of the Republic, Av. Garz on 780, Montevideo, CP 12900, Uruguay","Laboratorio de Zoología Vertebrados, Dpto de Ecología y Evolución, Facultad de Ciencias, Universidad de la República, Iguá 4225, Montevideo, 11400, Uruguay","| Centro Universitario Regional Este (CURE), Universidad de la República, Rutas 9 y 15 s/n, Rocha, Uruguay | Centro Universitario Regional Este (CURE), Universidad de la República | Department of Zoology and Entomology, Rhodes University","Dpto. de Ecología y Evolución, Facultad de Ciencias, Universidad de la República, Iguá 4225, Montevideo, 11400, Uruguay | Laboratorio de Zoología Vertebrados, Dpto de Ecología y Evolución, Facultad de Ciencias, Universidad de la República","| Departamento de Biología Vegetal, Facultad de Agronomía, Universidad de la República, Av. E. Garzón 780, Montevideo, 12900, Uruguay | Laboratory of Plant Evolution and Domestication, Department of Plant Biology, Universidad de la República, Av. E. Garzón 780, Montevideo, CP12900, Uruguay","Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, Universidad de la República, Iguá 4225, Montevideo, 11400, Uruguay | Laboratorio de Organización y Evolución del Genoma, Unidad de Genómica Evolutiva, Facultad de Ciencias, Universidad de la República, Iguá 4225, Montevideo, 11400, Uruguay","| Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, Universidad de la República, Iguá 4225, Montevideo, 11400, Uruguay | Molecular Virology Laboratory, CIN, Facultad de Ciencias, Universid ad de la Republica","| Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, 11400 Montevideo, Iguá 4225, Uruguay | Unidad de Biofísica de Proteínas, Institut Pasteur de Montevideo | Departamento de Técnicas, Nucleares Aplicadas, Facultad de Ciencias, Centro de Investigaciones Nucleares, Universidad de la República | Unidad de Biofísica de Proteínas, Institut Pasteur-Montevideo, 11400 Montevideo, Mataojo 2020, Uruguay | Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, 11400 Montevideo, Igua 4225, Uruguay | Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, Universidad de la República, Iguá 4225, Montevideo, 11400, Uruguay | Viral Populations and Pathogenesis laboratory. Institut Pasteur | Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias Universidad de la República, Igua 4225, 11400 Montevideo, Uruguay | Laboratorio de Inmunovirología, Institut Pasteur de Montevideo, Mataojo 2020, Montevideo, 11400, Uruguay | Molecular Virology Laboratory, CIN, Facultad de Ciencias, Universid ad de la Republica","| Departamento de Ecología y Evolución, Centro Universitario Regional Este (CURE), Universidad de la República, C/Tacuarembó s/n, Maldonado, 20000, Uruguay | IRBio and Department of Evolutive Biology, Ecology and Environmental Sciences, University of Barcelona, Av. Diagonal 643, Barcelona, 08028, Spain | Department of Animal Biology, Faculty of Biology, University of Barcelona | Programa PROMETEO-SENESCYT, Secretaría de Educación Superior, Ciencia, Tecnología e Innovación, Quito, Ecuador | Department of Ecology and Evolution, Centro Universitario Regional Este (CURE), University of the Republic (UdeLaR), Maldonado, Uruguay","| Proyecto Franciscana, Sección Etología, Facultad de Ciencias, Universidad de la República, Iguá 4225, Montevideo, 11400, Uruguay | Departamento de Ecología y Evolución, Facultad de Ciencias, Universidad de la República, Iguá 4225, Montevideo, 11400, Uruguay | Department of Ecology and Evolution, Faculty of Sciences, Universidad de la República, Iguá 4225, Montevideo, 11400, Uruguay | Department of Ecology and Evolution, Faculty of Sciences, University of the Republic (UdeLaR)","National Museum of Natural History (MNHN), 25 de Mayo 582, Montevideo, 11000, Uruguay | Modelization and Analysis of Natural Resources (MAREN), Centro Universitario Regional Este (CURE), Universidad de la República","National Museum of Natural History (MNHN), 25 de Mayo 582, Montevideo, 11000, Uruguay","National Museum of Natural History (MNHN), 25 de Mayo 582, Montevideo, 11000, Uruguay","| Departamento de Ecología y Evolución, Facultad de Ciencias, Universidad de la República, Montevideo, Uruguay | Laboratório de AIDS and Imunologia Molecular, Instituto Oswaldo Cruz, FIOCRUZ | Fundação Oswaldo Cruz-Fiocruz, Instituto Oswaldo Cruz, Laboratório de AIDS e Imunologia Molecular, Rio de Janeiro, RJ, Brazil | Laboratorio de AIDS and Imunologia Molecular, Instituto Oswaldo Cruz, Av. Brasil 4365, Rio de Janeiro, 21045900, Brazil","Department of Biological Sciences, Laboratory of Molecular Virology, CENUR Litoral Norte, Sede Salto, Universidad de la República | Universidad de la República, Laboratorio de Virología Molecular - Regional Norte | Laboratório de Virologia Comparada e Ambiental, Instituto Oswaldo Cruz, FIOCRUZ, Manguinhos, 21040-360 Rio de Janeiro, RJ, Av. Brasil 4365, Brazil | Laboratorio de Virología Molecular, CENUR Litoral Norte, Sede Salto, Universidad de la República, Uruguay","Laboratory of Experimental Psychology and Neuroscience, Institute of Cognitive and Translational Neuroscience, INECO Foundation, Favaloro University, Buenos Aires, Argentina | Departamento de Educación Física y Salud, Instituto Superior de Educación Física, Universidad de la República, Montevideo, Uruguay","Centro Universitario Regional del Este (CURE), Universidad de la República, Intersección Ruta 9 y Ruta 15, Rocha, 27000, Uruguay","Dirección Nacional de Recursos Acuáticos (DINARA), Montevideo, 11200, Uruguay","Dirección Nacional de Recursos Acuáticos (DINARA), Montevideo, 11200, Uruguay","Facultad de Ingeniería, Universidad de la República, Uruguay","Unidad de Bioinform atica, Institut Pasteur de Montevideo, Mataojo, Montevideo, 2020, Uruguay","| Laboratorio de Virologia Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, Universidad de la Republica, Igua 4225, Montevideo, 11400, Uruguay | Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias Universidad de la República, Igua 4225, 11400 Montevideo, Uruguay | Facultad de Ciencias, Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Universidad de la Republica","Laboratorio de Botánica, Facultad de Agronomía, Universidad de la República, Montevideo, Casilla de Correos 1238, Uruguay | Laboratorio de Botánica, Universidad de la República, Montevideo, Casilla de Correos 1238, Uruguay | Laboratorio de Botánica, Facultad de Agronomía, Casilla de Correos 1238, Montevideo, Uruguay","Sección Genética Evolutiva, Facultad de Ciencias, UdelaR, CP.11400 Montevideo, Iguá 4225, Uruguay | Instituto de Diversidad y Ecología Animal (IDEA), CONICET-Universidad Nacional de Córdoba, and Cátedra de Diversidad Animal i, Facultad de Ciencias Exactas, Físicas y Naturales, X5000JJC Córdoba, Av. Vélez Sarsfield 299, Argentina | CONICET-Cátedra de Diversidad Animal I, FCEFyN, UNC, Argentina | Cátedra de Diversidad Animal I, Facultad de Ciencias Exactas, Físicas y Naturales, U. N. C., Córdoba, Argentina | Sección Genética Evolutiva, Facultad de Ciencias, Montevideo, Uruguay | Instituto de Diversidad y Ecología Animal (IDEA), CONICET – Universidad Nacional de Córdoba, Cátedra de Diversidad Animal I, Físicas y Naturales, U.N.C., Av. Vélez Sarsfield 299, Córdoba, X5000JJC, Argentina | Instituto de Diversidad y Ecología Animal (IDEA), CONICET-Universidad Nacional de Córdoba | Sección Genética Evolutiva, Facultad de Ciencias, Universidad de la República, Montevideo, Iguá 4225, CP 11400, Uruguay","Department of Physiology, Universidad de la República, Montevideo, 11800, Uruguay","Department of Physiology, Universidad de la República, Montevideo, 11800, Uruguay","Proyecto Franciscana, Sección Etología, Facultad de Ciencias, Universidad de la República, Iguá 4225, Montevideo, 11400, Uruguay","Proyecto Franciscana, Sección Etología, Facultad de Ciencias, Universidad de la República, Iguá 4225, Montevideo, 11400, Uruguay | Cetacean Ecology, Behaviour and Evolution Lab, School of Biological Sciences, Flinders University, Sturt Road, Bedford Park, 5042, South Australia, Australia","Chemical and Process Systems Engineering Group, Chemical Engineering Inst. - Engineering School, Universidad de la República, Julio Herrera y Reissig 565, Montevideo, CP 11300, Uruguay","| Laboratorio de Organización y Evolución del Genoma, Instituto de Biología, Facultad de Ciencias, 11400 Montevideo, Iguá 4225, Uruguay | Laboratorio de Organización Y Evolución del Genoma, Departamento de Biología Celular Y Molecular, Facultad de Ciencias | Laboratorio de Organización y Evolución del Genoma, UDELAR","Grupo Biodiversidad y Ecología de la Conservación IECA, Facultad de Ciencias, Universidad de la República, CP: 11600, Uruguay | Centro Interdisciplinario de Respuesta al Cambio y la Variabilidad Climática, Espacio Interdisciplinario, Universidad de la República | Laboratorio de Sistemática e Historia Natural de Vertebrados IECA, Facultad de Ciencias, Universidad de la República, Uruguay","Grupo Biodiversidad y Ecología de la Conservación IECA, Facultad de Ciencias, Universidad de la República, CP: 11600, Uruguay","Department of Animal and Forage Production Science, University of the Republic-School of Agronomy-EEMAC, Paysandú, Uruguay | Departamento de Producción Animal Y Pasturas, Universidad de La República (UDELAR), Paysandú, Uruguay","Instituto de Producción Animal, Área Genética, Facultad de Veterinaria, Universidad de la República (UdelaR), Montevideo, Uruguay","Sección Genética Evolutiva, Instituto de Biología, Facultad de Ciencias, Universidad de la República, Montevideo, Uruguay | Unidad de Bioinformática, Institut Pasteur Montevideo, Montevideo, Uruguay","Sección Genética Evolutiva, Instituto de Biología, Facultad de Ciencias, Universidad de la República, Montevideo, Uruguay","Instituto Nacional de Investigación Agropecuaria, INIA Las Brujas, Ruta 48, km 10, Rincón del Colorado, Canelones, CP 90200, Uruguay","Laboratorio de Virologia Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, Universidad de la Republica, Igua 4225, Montevideo, 11400, Uruguay","Laboratorio de Virologia Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, Universidad de la Republica, Igua 4225, Montevideo, 11400, Uruguay | Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias Universidad de la República, Igua 4225, 11400 Montevideo, Uruguay","Laboratorio de Virología Molecular, Centro Universitario Regional Noroeste (CENUR Noroeste), Universidad de la Republica (UdelaR), Rivera 1350, Salto, 50000, Uruguay","Proyecto Pinnípedos, Sección Etología, Facultad de Ciencias, Universidad de la República, Iguá 4225, Montevideo, 11400, Uruguay | Departamento Ciencias Biológicas, Facultad de Ciencias, Universidad de los Andes, Carrera 1E No 18A - 10, Bogotá, Colombia","Departamento de Producción Animal y Pasturas, Facultad de Agronomía, Universidad de la República, Av. E. Garzón 780, Montevideo, 12900, Uruguay","Secretariado Uruguayo de la Lana, Rambla Baltasar Brum 3764 | Secretariado Uruguayo de la Lana, Rambla Baltasar Brum 3764, Montevideo, 11800, Uruguay","Departamento de Producción Animal y Pasturas, Facultad de Agronomía, Universidad de la República, Av. E. Garzón 780, Montevideo, 12900, Uruguay | Estación Experimental Facultad de Agronomía Salto, Ruta 31 Km 21, Salto, 50000, Uruguay","| Departamento de Recursos Pelágicos, Dirección Nacional de Recursos Acuáticos (DINARA), Constituyente 1497, Montevideo, CP 11200, Uruguay","Departamento de Ecología Y Evolución, Facultad de Ciencias, Universidad de la República, Montevideo, Uruguay","| Sección Zoología Vertebrados, Facultad de Ciencias, Montevideo, Uruguay","Laboratorio de Organización y Evolución del Genoma, UDELAR, 11400 Montevideo, Iguá 4225, Uruguay | Laboratorio de Evolución, UDELAR, 11400 Montevideo, Iguá 4225, Uruguay | Área Genética, Depto. de Genética y Mejora Animal, UDELAR, CP 11600, Montevideo, Av. A. Lasplaces 1550, Uruguay","Sección Bioquímica y Biología Molecular, UDELAR, 11400 Montevideo, Iguá 4225, Uruguay","Departamento de Ecología y Evolución Facultad de Ciencias, Universidad de la República, 11400 Montevideo, Uruguay | Franciscana Project/Cetaceos Uruguay, Sección de Etología, Facultad de Ciencias, Universidad de la República, 11400 Montevideo, Uruguay | Laboratorio de Ecología Molecular, Facultad de Ciencias Marinas, Universidad Autónoma de Baja California, Baja California, Mexico","Centro Nacional de Referencia de Influenza, Departamento de Laboratorios de Salud Pública, Ministerio de Salud Pública, 11200 Montevideo, Alfredo Navarro 3051 acceso norte, Uruguay","Centro Nacional de Referencia de Influenza, Departamento de Laboratorios de Salud Pública, Ministerio de Salud Pública, 11200 Montevideo, Alfredo Navarro 3051 acceso norte, Uruguay","| Centro Nacional de Referencia de Influenza, Departamento de Laboratorios de Salud Pública, Ministerio de Salud Pública, 11200 Montevideo, Alfredo Navarro 3051 acceso norte, Uruguay","Institut Pasteur de Montevideo, Montevideo 11400, Mataojo 2020, Uruguay | Joint IRB-BSC Program on Computational Biology, Institute for Research in Biomedicine (IRB), Parc Cientific de Barcelona, Barcelona 08028, Josep Samitier 1-5, Spain","| Sección Genética Evolutiva, Facultad de Ciencias, Montevideo, Uruguay | Sección Genética Evolutiva, Facultad de Ciencias, Universidad de la República, Montevideo, Iguá 4225, CP 11400, Uruguay","| Sección Genética Evolutiva, Facultad de Ciencias, UdelaR, CP.11400 Montevideo, Iguá 4225, Uruguay","Sección Zoología Vertebrados, Facultad de Ciencias, Montevideo, Uruguay | Departamento de Ictiología, Museo Nacional de Historia Natural","Department of Botany, Center of Biological Sciences, Federal University of Pernambuco, Recife, Pernambuco, Rua Nelson Chaves s/n 50670-420, Brazil | Department of Plant Biology, College of Agronomy, University of the Republic, CP 12900 Montevideo, Avenida Eugenio Garzón 780, Uruguay","Departamento de Ecología y Evolución Facultad de Ciencias, Universidad de la República, 11400 Montevideo, Uruguay | Franciscana Project/Cetaceos Uruguay, Sección de Etología, Facultad de Ciencias, Universidad de la República","Facultad de Ciencias Veterinarias, Universidad Nacional de La Plata, La Plata, B1900AVW, Buenos Aires, Avda. 60 y 118, Argentina | Área de Patología y Productión Avícola, Facultad de Veterinaria, Universidad de la República, Montevideo, Uruguay | Universidad de la República | Laboratorio de Diagnóstico de Enfermedades de las Aves y los Pilíferos, Facultad de Ciencias Veterinarias, Universidad Nacional de La Plata | Área de Patologia y Producción Avícola, Facultad de Veterinaria, Universidad de la República, Montevideo, Uruguay | Area de Patologia y Produccion Avicola, Facultad de Veterinaria, Universidad de la Republica, Montevideo, Uruguay","Facultad de Ciencias Veterinarias, Universidad Nacional de La Plata, La Plata, B1900AVW, Buenos Aires, Avda. 60 y 118, Argentina | Área de Patología y Productión Avícola, Facultad de Veterinaria, Universidad de la República, Montevideo, Uruguay | Universidad de la República | Laboratorio de Diagnóstico de Enfermedades de las Aves y los Pilíferos, Facultad de Ciencias Veterinarias, Universidad Nacional de La Plata | Área de Patologia y Producción Avícola, Facultad de Veterinaria, Universidad de la República, Montevideo, Uruguay | Area de Patologia y Produccion Avicola, Facultad de Veterinaria, Universidad de la Republica, Montevideo, Uruguay","Facultad de Ciencias Veterinarias, Universidad Nacional de La Plata, La Plata, B1900AVW, Buenos Aires, Avda. 60 y 118, Argentina | Área de Patología y Productión Avícola, Facultad de Veterinaria, Universidad de la República, Montevideo, Uruguay | Universidad de la República | Laboratorio de Diagnóstico de Enfermedades de las Aves y los Pilíferos, Facultad de Ciencias Veterinarias, Universidad Nacional de La Plata | Área de Patologia y Producción Avícola, Facultad de Veterinaria, Universidad de la República, Montevideo, Uruguay | Area de Patologia y Produccion Avicola, Facultad de Veterinaria, Universidad de la Republica, Montevideo, Uruguay","Facultad de Ciencias Veterinarias, Universidad Nacional de La Plata, La Plata, B1900AVW, Buenos Aires, Avda. 60 y 118, Argentina | Universidad de la República | Laboratorio de Diagnóstico de Enfermedades de las Aves y los Pilíferos, Facultad de Ciencias Veterinarias, Universidad Nacional de La Plata | Área de Patologia y Producción Avícola, Facultad de Veterinaria, Universidad de la República, Montevideo, Uruguay | Area de Patologia y Produccion Avicola, Facultad de Veterinaria, Universidad de la Republica, Montevideo, Uruguay","Universidad de la República, Montevideo, Uruguay | Facultad de Ciencias Veterinarias, Universidad Nacional de La Plata, La Plata, B1900AVW, Buenos Aires, Avda. 60 y 118, Argentina","Departamento de Informática y Ciencias de la Computación, Universidad Católica Del Uruguay, CP 11600, Montevideo, Uruguay","Facultad de Ingeniería y Tecnologías, Universidad Católica del Uruguay, Montevideo, Uruguay | Departamento de Informática y Ciencias de la Computación, Universidad Católica Del Uruguay, CP 11600, Montevideo, Uruguay","Laboratorio Nacional de Referencia VIH-SIDA, Servicio Nacional de Laboratorios de Salud Publica - MSP, Montevideo, Uruguay","Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, 11400 Montevideo, Igua 4225, Uruguay","Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, 11400 Montevideo, Igua 4225, Uruguay","Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, 11400 Montevideo, Igua 4225, Uruguay","Facultad de Veterinaria, Universidad de la República, C.P. 11600, Montevideo, Avda. Alberto Lasplaces 1620, Uruguay | PROVETSUR, Facultad de Ciencias Veterinarias, Universidad Nacional de La Plata, La Plata, B1900AVW, Buenos Aires, Avda. 60 y 118, Argentina","Facultad de Veterinaria, Universidad de la República, C.P. 11600, Montevideo, Avda. Alberto Lasplaces 1620, Uruguay | PROVETSUR, Facultad de Ciencias Veterinarias, Universidad Nacional de La Plata, La Plata, B1900AVW, Buenos Aires, Avda. 60 y 118, Argentina","Laboratorio de Organización y Evolución del Genoma, Instituto de Biología, Facultad de Ciencias, 11400 Montevideo, Iguá 4225, Uruguay",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null],"papers":[1,1,1,1,1,1,1,2,1,2,2,2,3,11,2,9,3,6,1,1,1,3,7,3,3,3,3,5,5,1,1,1,10,2,2,2,1,2,2,9,1,1,1,1,1,4,2,1,1,1,1,1,1,1,1,1,1,1,3,2,2,2,2,1,4,10,3,3,1,1,2,1,1,2,1,2,2,4,7,15,1,1,1,1,2,1,1,1,1,3,1,2,1,2,3,1,1,3,1,3,4,1,1,1,6,1,1,1,2,2,1,3,2,3,1,3,7,6,6,1,1,1,8,4,1,1,1,1,1,1,4,3,6,1,1,1,1,1,4,1,1,2,1,1,1,1,1,2,1,1,1,2,1,2,1,2,1,1,2,1,1,2,1,3,2,1,1,1,4,4,4,4,1,1,2,1,1,1,1,1,1,1,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null],"contacto":["True","False","False","False","False","False","False","False","False","False","True","True","False","True","False","True","True","True","True","True","False","True","True","True","True","True","True","True","True","False","False","True","True","True","False","False","False","False","True","True","True","True","True","True","True","True","False","False","False","False","True","True","True","True","True","True","True","True","True","True","True","False","True","False","True","True","True","True","True","True","True","True","True","True","False","True","False","True","True","True","True","False","False","False","False","True","False","False","False","False","False","True","True","True","True","True","False","True","True","False","True","False","False","True","False","False","False","True","False","False","False","True","True","True","True","True","True","True","True","False","False","False","True","True","False","True","True","True","False","False","False","True","True","True","True","False","False","True","True","True","False","False","False","False","False","False","False","False","True","False","True","False","False","False","False","False","False","False","True","False","False","True","False","True","False","False","False","False","False","False","False","False","False","True","True","True","False","False","False","False","False","False",null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null],"type":[true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false],"title":["<b>Author:<\/b> Herrera-Esposito, Daniel<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Centro Interdisciplinario de Ciencia de Datos y Aprendizaje Automático, Universidad de la República, Montevideo, Uruguay <br>- Laboratorio de Neurociencias, Facultad de Ciencias, Universidad de la República","<b>Author:<\/b> Fuques E.<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Sección Genética Evolutiva, Departamento de Biología Animal, Instituto de Biología, Facultad de Ciencias, Universidad de la República, Iguá 4225, 11400, Montevideo, Uruguay","<b>Author:<\/b> Tomás G.<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Sección Genética Evolutiva, Departamento de Biología Animal, Instituto de Biología, Facultad de Ciencias, Universidad de la República, Iguá 4225, 11400, Montevideo, Uruguay","<b>Author:<\/b> Grecco S.<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Sección Genética Evolutiva, Departamento de Biología Animal, Instituto de Biología, Facultad de Ciencias, Universidad de la República, Iguá 4225, 11400, Montevideo, Uruguay","<b>Author:<\/b> Condon E.<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Sección Genética Evolutiva, Departamento de Biología Animal, Instituto de Biología, Facultad de Ciencias, Universidad de la República, Iguá 4225, 11400, Montevideo, Uruguay","<b>Author:<\/b> Techera C.<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Sección Genética Evolutiva, Departamento de Biología Animal, Instituto de Biología, Facultad de Ciencias, Universidad de la República, Iguá 4225, 11400, Montevideo, Uruguay","<b>Author:<\/b> Marandino A.<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Sección Genética Evolutiva, Departamento de Biología Animal, Instituto de Biología, Facultad de Ciencias, Universidad de la República, Iguá 4225, 11400, Montevideo, Uruguay","<b>Author:<\/b> Sarute N.<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Sección Genética Evolutiva, Departamento de Biología Animal, Instituto de Biología, Facultad de Ciencias, Universidad de la República, Iguá 4225, 11400, Montevideo, Uruguay <br>- Sección Genética Evolutiva, Instituto de Biología, Facultad de Ciencias, Universidad de la República, Montevideo, Uruguay","<b>Author:<\/b> Benech A.<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Unidad de Clínica y Hospital Veterinario, Facultad de Veterinaria, Universidad de la República, Montevideo, Uruguay","<b>Author:<\/b> Pérez R.<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Sección Genética Evolutiva, Departamento de Biología Animal, Instituto de Biología, Facultad de Ciencias, Universidad de la República, Iguá 4225, 11400, Montevideo, Uruguay <br>- Sección Genética Evolutiva, Instituto de Biología, Facultad de Ciencias, Universidad de la República, Montevideo, Uruguay","<b>Author:<\/b> Panzera Y.<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Sección Genética Evolutiva, Departamento de Biología Animal, Instituto de Biología, Facultad de Ciencias, Universidad de la República, Iguá 4225, 11400, Montevideo, Uruguay <br>- Sección Genética Evolutiva, Instituto de Biología, Facultad de Ciencias, Universidad de la República, Montevideo, Uruguay","<b>Author:<\/b> Neimaur K.<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Departamento Ovinos, Lanas y Caprinos, Facultad de Veterinaria, Universidad de la República, Montevideo, 11600, Uruguay <br>- Unidad académica Ovinos, Lanas y Caprinos, Departamento de Producción Animal y Salud de los Sistemas Productivos, Facultad de Veterinaria, Universidad de la República, Ruta 1, km 42,5- Libertad, San José, Uruguay","<b>Author:<\/b> Kremer R.<br><b>Countries:<\/b> United Kingdom; Uruguay<br><b>Affiliations:<\/b><br>- <br>- Departamento de Ovinos Y Lanas, Facultad de Veterinaria, Montevideo 11600, Av. Lasplaces 1550, Uruguay <br>- Unidad académica Ovinos, Lanas y Caprinos, Departamento de Producción Animal y Salud de los Sistemas Productivos, Facultad de Veterinaria, Universidad de la República, Ruta 1, km 42,5- Libertad, San José, Uruguay","<b>Author:<\/b> Naya H.<br><b>Countries:<\/b> United Kingdom; United States; Uruguay<br><b>Affiliations:<\/b><br>- <br>- Unidad de Bioinformática, Institut Pasteur de Montevideo, Mataojo 2020, Montevideo 11400, Uruguay <br>- Unidad de Bioinformática, Institut Pasteur de Montevideo, 11400 Montevideo, Mataojo 2020, Uruguay <br>- Departamento de Producción Animal Y Pasturas, Facultad de Agronomía, Universidad de la República, Montevideo, Uruguay <br>- Unidad de Bioinformática, Institut Pasteur de Montevideo, Mataojo 2020, Montevideo, Uruguay <br>- Department of Animal Sciences, University of Wisconsin-Madison, Madison, WI 53706, United States <br>- Unidad de Bioinformática, Instituto Pasteur de Montevideo <br>- Bioinformatics Unit, Institut Pasteur, 11400 Montevideo, Uruguay <br>- Laboratorio de Organización Y Evolución del Genoma, Departamento de Biología Celular Y Molecular, Facultad de Ciencias <br>- Departamento de Producción Animal Y Pasturas, Facultad de Agronomía, Universidad de la República, 12900 Montevideo, Garzón 780, Uruguay <br>- Institut Pasteur de Montevideo <br>- Bioinformatics Unit, Institut Pasteur de Montevideo, Mataojo 2020, Montevideo, 11400, Uruguay <br>- Departamento de Producción Animal Y Pasturas, Facultad de Agronomía <br>- Departmento de Producción Animal y Pasturas, Facultad de Agronomía, Universidad de la República, Av. Gral. Eugenio Garzón 780, Montevideo, 12900, Uruguay <br>- Department of Animal Sciences, University of Wisconsin-Madison, United States <br>- Departamento de Producción Animal y Pasturas, Facultad de Agronomía <br>- Departamento de Producción Animal Y Pasturas, Facultad de Agronomía, Montevideo 12900, Av. Garzón 780, Uruguay","<b>Author:<\/b> Sienra I.<br><b>Countries:<\/b> United Kingdom; Uruguay<br><b>Affiliations:<\/b><br>- <br>- Departamento Ovinos, Lanas y Caprinos, Facultad de Veterinaria, Universidad de la República, Montevideo, 11600, Uruguay","<b>Author:<\/b> Urioste J.I.<br><b>Countries:<\/b> United Kingdom; Uruguay; Sweden<br><b>Affiliations:<\/b><br>- <br>- Departamento de Producción Animal Y Pasturas, Facultad de Agronomía, Universidad de la República, 12900 Montevideo, Garzón 780, Uruguay <br>- Cancer Genetics Building, St. James' University Hospital, Leeds LS9 7TF, Beckett Street, United Kingdom <br>- Departamento de Producción Animal Y Pasturas, Facultad de Agronomía <br>- Departamento de Producción Animal y Pasturas, Facultad de Agronomía <br>- Department of Animal Breeding and Genetics, Swedish University of Agricultural Sciences, S-75007 Uppsala, PO Box 7023, Sweden <br>- Departamento de Producción Animal Y Pasturas, Facultad de Agronomía, Montevideo 12900, Av. Garzón 780, Uruguay <br>- Departamento de Producción Animal y Pasturas, Fac. de Agronomía, UDELAR, 12900 Montevideo, Garzón 780, Uruguay","<b>Author:<\/b> Picasso-Risso, Catalina<br><b>Countries:<\/b> United States; Uruguay<br><b>Affiliations:<\/b><br>- Minnesota Center for Prion Research and Outreach, University of Minnesota, Saint Paul, 55108, MN, United States <br>- Department of Veterinary Population Medicine, University of Minnesota, Saint Paul, 55108, MN, United States <br>- Department of Veterinary Population Medicine, University of Minnesota, Saint Paul, MN, United States <br>- Facultad de Veterinaria, Universidad de la Republica, Montevideo, 11200, Uruguay <br>- Facultad de Veterinaria, Universidad de la Republica, Montevideo, Uruguay","<b>Author:<\/b> de Magalhães, Rafael Félix<br><b>Countries:<\/b> United Kingdom; Uruguay; Brazil<br><b>Affiliations:<\/b><br>- <br>- Programa de Pós-Graduação em Zoologia, Universidade Federal de Minas Gerais <br>- Programa de Pós-Graduação em Zoologia, Instituto de Ciências Biológicas, Universidade Federal de Minas Gerais, Avenida Antônio Carlos, 6627, Belo Horizonte, 31270-010, MG, Brazil <br>- Programa de Pós-Graduação em Zoologia, Instituto de Ciências Biológicas, Universidade Federal de Minas Gerais – UFMG, Minas Gerais, Belo Horizonte, Brazil <br>- Departamento de Zoologia, ICB, Universidade Federal de Minas Gerais, Av. Antônio Carlos, 6627, Belo Horizonte, 31270-901, MG, Brazil <br>- Departamento de Ciências Naturais, campus Dom Bosco, Universidade Federal de São João del-Rei, São João del-Rei, Brazil <br>- Departamento de Ciências Naturais (DCNAT), Campus Dom Bosco, Universidade Federal de São João del-Rei (UFSJ), Praça Dom Helvécio, 74, São João del-Rei, 36301-160, MG, Brazil <br>- Departamento de Ciências Naturais, Campus Dom Bosco, Universidade Federal de São João del-Rei, São João del Rei, 36301-160, MG, Brazil <br>- Programa de Desarrollo Universitario, Centro Universitario de Rivera, Universidad de la República – UdelaR, Rivera, Uruguay","<b>Author:<\/b> Mederos, America<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Instituto Nacional de Investigación Agropecuaria (INIA), Programa Nacional Producción de Carne y Lana, Ruta 5 Km 386, Tacuarembó, 45000, Uruguay","<b>Author:<\/b> González, Macarena<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Departamento de Ecología y Biología Evolutiva, Instituto de Investigaciones Biológicas Clemente Estable (IIBCE), Ministerio de Educación y Cultura (MEC), Avenida Italia 3318, Montevideo, PC 11600, Uruguay","<b>Author:<\/b> Kacevas, Nadia<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Departamento de Ecología y Biología Evolutiva, Instituto de Investigaciones Biológicas Clemente Estable (IIBCE), Ministerio de Educación y Cultura (MEC), Avenida Italia 3318, Montevideo, PC 11600, Uruguay <br>- Departamento de Biodiversidad y Genética, Instituto de Investigaciones Biológicas Clemente Estable (IIBCE), Ministerio de Educación y Cultura (MEC), Avenida Italia 3318, Montevideo, PC 11600, Uruguay","<b>Author:<\/b> Bidegaray-Batista, Leticia<br><b>Countries:<\/b> Uruguay; Spain<br><b>Affiliations:<\/b><br>- Departamento de Biodiversidad y Genética, Instituto de Investigaciones Biológicas Clemente Estable (IIBCE), Ministerio de Educación y Cultura (MEC), Avenida Italia 3318, Montevideo, PC 11600, Uruguay <br>- Departament de Biologia Animal, Facultat de Biologia, Universitat de Barcelona, Av. Diagonal 643, Barcelona, 08028, Spain <br>- Laboratorio de Etologíıa, Ecologíıa y Evoluciíon, Instituto de Investigaciones Biolíogicas Clemente Estable, Montevideo, Uruguay <br>- Institut de Recerca de la Biodiversitat, Universitat de Barcelona, Av. Diagonal 643, Barcelona, 08028, Spain <br>- Institut de Recerca de la Biodiversitat & Departament de Biologia Animal, Universitat de Barcelona, Barcelona, Spain <br>- Laboratorio de Etología, Ecología y Evolución, Instituto de Investigaciones Biológicas Clemente Estable, Avenida Italia 3318, Montevideo, 11600, Uruguay","<b>Author:<\/b> Fresia, Pablo<br><b>Countries:<\/b> United Kingdom; Uruguay; Brazil<br><b>Affiliations:<\/b><br>- <br>- Institut Pasteur de Montevideo, Microbial Genomics Laboratory, Montevideo, Uruguay <br>- Unidad de Bioinform atica, Institut Pasteur de Montevideo <br>- Departamento de Entomologia e Acarologia, Escola Superior de Agricultura 'Luiz de Queiroz', USP, Piracicaba, SP 13418-900, Avenida Pádua Dias, 11, Brazil <br>- Institut Pasteur de Montevideo, Pasteur + INIA Joint Unit (UMPI), Montevideo, Uruguay <br>- Unidad de Bioinformatica, Institut Pasteur de Montevideo, Mataojo, 2020, Montevideo, Uruguay","<b>Author:<\/b> Parada, Andrés<br><b>Countries:<\/b> Chile; Uruguay<br><b>Affiliations:<\/b><br>- Laboratorio de Evolución, Facultad de Ciencias, Universidad de la República <br>- Microbial Genomics Laboratory, Institut Pasteur Montevideo, Mataojo 2020, Montevideo, 11400, Uruguay <br>- Departamento de Ecología, Facultad de Ciencias Biológicas, Pontificia Universidad Católica de Chile, Santiago 6513677, Avda. Libertador Bernardo O'Higgins 340, Chile <br>- Departamento de Ecología, Center for Advanced Studies in Ecology and Biodiversity, Pontificia Universidad Católica, Santiago, Chile <br>- Instituto de Ciencias Ambientales y Evolutivas, Facultad de Ciencias, Universidad Austral de Chile, Campus Isla Teja s/n, Valdivia, 5090000, Chile","<b>Author:<\/b> Félix, María Laura<br><b>Countries:<\/b> United Kingdom; Uruguay<br><b>Affiliations:<\/b><br>- <br>- Laboratorio de Vectores y Enfermedades Transmitidas, Departamento de Ciencias Biológicas, CENUR Litoral Norte—Salto, Universidad de la República, Rivera 1350, Salto, 50000, Uruguay <br>- Laboratorio de Vectores Y Enfermedades Transmitidas, Departamento de Ciencias Biológicas, CENUR Litoral Norte, Universidad de La República, Rivera 1350, Salto, CP 50000, Uruguay","<b>Author:<\/b> Carvalho, Luis Andrés<br><b>Countries:<\/b> Uruguay; New Zealand<br><b>Affiliations:<\/b><br>- AgResearch, Grasslands Research Centre, Private Bag 11008, Palmerston North, 4442, New Zealand <br>- Laboratorio de Vectores Y Enfermedades Transmitidas, Departamento de Ciencias Biológicas, CENUR Litoral Norte, Universidad de La República, Rivera 1350, Salto, CP 50000, Uruguay","<b>Author:<\/b> Queirolo, Diego<br><b>Countries:<\/b> United Kingdom; Uruguay<br><b>Affiliations:<\/b><br>- <br>- CENUR Noreste, Universidad de La República, Ituzaingó 667, Rivera, CP 40000, Uruguay <br>- Centro Universitario de Rivera, Universidad de la República","<b>Author:<\/b> Armúa-Fernández, María Teresa<br><b>Countries:<\/b> United Kingdom; Uruguay<br><b>Affiliations:<\/b><br>- <br>- Laboratorio de Vectores y enfermedades transmitidas, Facultad de Veterinaria, CENUR Litoral Norte, Universidad de la República <br>- Laboratorio de Vectores y Enfermedades Transmitidas, Facultad de Veterinaria, CENUR Litoral Norte - Salto, Universidad de la República, Rivera 1350, Salto, 50000, Uruguay <br>- Laboratorio de Vectores Y Enfermedades Transmitidas, Departamento de Ciencias Biológicas, CENUR Litoral Norte, Universidad de La República, Rivera 1350, Salto, CP 50000, Uruguay <br>- Institute of Parasitology, Vetsuisse Faculty, University of Zurich","<b>Author:<\/b> Venzal, José Manuel<br><b>Countries:<\/b> United Kingdom; Uruguay<br><b>Affiliations:<\/b><br>- <br>- Laboratorio de Vectores y Enfermedades Transmitidas, Departamento de Ciencias Biológicas, CENUR Litoral Norte—Salto, Universidad de la República, Rivera 1350, Salto, 50000, Uruguay <br>- Laboratorio de Vectores y enfermedades transmitidas, Facultad de Veterinaria, CENUR Litoral Norte, Universidad de la República <br>- Laboratorio de Vectores Y Enfermedades Transmitidas, Departamento de Ciencias Biológicas, CENUR Litoral Norte, Universidad de La República, Rivera 1350, Salto, CP 50000, Uruguay","<b>Author:<\/b> Bazzano, Valentin<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Laboratorio de Vectores Y Enfermedades Transmitidas, Departamento de Ciencias Biológicas, CENUR Litoral Norte, Universidad de La República, Rivera 1350, Salto, CP 50000, Uruguay","<b>Author:<\/b> da Costa, Anthony<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Laboratorio de Vectores Y Enfermedades Transmitidas, Departamento de Ciencias Biológicas, CENUR Litoral Norte, Universidad de La República, Rivera 1350, Salto, CP 50000, Uruguay","<b>Author:<\/b> Giorello, Facundo M.<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Espacio de Biología Vegetal del Noreste, Centro Universitario de Tacuarembó, Universidad de la República, Tacuarembó, Uruguay <br>- Departamento de Ecología y Evolución, Facultad de Ciencias, Universidad de la República, Montevideo, Uruguay","<b>Author:<\/b> Lessa, Enrique P.<br><b>Countries:<\/b> United Kingdom; Uruguay<br><b>Affiliations:<\/b><br>- <br>- Laboratorio de Evolución, Facultad de Ciencias, Universidad de la República <br>- Laboratorio de Evolución, Facultad de Ciencias, Universidad de la Repúplica, Montevideo 11400, Iguá 4225, Uruguay <br>- Departamento de Ecología y Evolución, Facultad de Ciencias, Universidad de la República, Montevideo, Uruguay <br>- Departamento de Ecología y Evolución Facultad de Ciencias, Universidad de la República, 11400 Montevideo, Uruguay","<b>Author:<\/b> Alonso, Rodrigo<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Instituto de Mecánica de Los Fluidos e Ingeniería Ambiental, Facultad de Ingeniería, Universidad de la República, Julio Herrera y Reissig 565, Montevideo, 11300, Uruguay <br>- Universidad de la República, Facultad de Ingeniería Instituto de, Mecánica de los Fluidos e Ingeniería Ambiental, Julio Herrera y Reissig, 565, CP, Montevideo, 11300, Uruguay","<b>Author:<\/b> Solari, Sebastián<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Instituto de Mecánica de Los Fluidos e Ingeniería Ambiental, Facultad de Ingeniería, Universidad de la República, Julio Herrera y Reissig 565, Montevideo, 11300, Uruguay <br>- Universidad de la República, Facultad de Ingeniería Instituto de, Mecánica de los Fluidos e Ingeniería Ambiental, Julio Herrera y Reissig, 565, CP, Montevideo, 11300, Uruguay","<b>Author:<\/b> Lercari, Diego<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Centro Universitario Regional Este (CURE), Universidad de la República <br>- UNDECIMAR, Departamento de Ecología y Evolución, Facultad de Ciencias, Universidad de La República, Iguá 4225, Montevideo, 11400, Uruguay <br>- UNDECIMAR, Facultad de Ciencias, Iguá 4225, Montevideo, 11400, Uruguay","<b>Author:<\/b> Celentano, Eleonora<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- UNDECIMAR, Departamento de Ecología y Evolución, Facultad de Ciencias, Universidad de La República, Iguá 4225, Montevideo, 11400, Uruguay","<b>Author:<\/b> Defeo, Omar<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Centro Universitario Regional Este (CURE), Universidad de la República <br>- UNDECIMAR, Departamento de Ecología y Evolución, Facultad de Ciencias, Universidad de La República, Iguá 4225, Montevideo, 11400, Uruguay <br>- UNDECIMAR, Facultad de Ciencias, Iguá 4225, Montevideo, 11400, Uruguay","<b>Author:<\/b> Rego, Natalia<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Bioinformatics Unit, Institut Pasteur de Montevideo, Montevideo, 11400, Uruguay <br>- Bioinformatics Unit, Institut Pasteur de Montevideo, Mataojo 2020, Montevideo, 11400, Uruguay","<b>Author:<\/b> Fajardo, Alvaro<br><b>Countries:<\/b> United Kingdom; Uruguay<br><b>Affiliations:<\/b><br>- <br>- Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, Universidad de la República <br>- Facultad de Ciencias, Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Universidad de la Republica <br>- Laboratorio de Evolución Experimental de Virus, Institut Pasteur de Montevideo <br>- Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, 11400 Montevideo, Igua 4225, Uruguay <br>- Laboratorio de Virología Molecular, Facultad de Ciencias, Universidad de la República <br>- Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, Universidad de la República, Iguá 4225, Montevideo, 11400, Uruguay <br>- Laboratorio de Virologia Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, Universidad de la Republica, Igua 4225, Montevideo, 11400, Uruguay <br>- Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, Igua 4225, 11400 Montevideo, Uruguay","<b>Author:<\/b> Fernández-Calero, Tamara<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Department of Exact and Natural Sciences, Universidad Católica del Uruguay <br>- Bioinformatics Unit, Institut Pasteur de Montevideo, Montevideo, 11400, Uruguay","<b>Author:<\/b> Noya, Veronica<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Laboratorio de Biología Molecular, Sanatorio Americano, Montevideo, 11600, Uruguay","<b>Author:<\/b> Machado, Matias R.<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Protein Engineering, Institut Pasteur de Montevideo, Montevideo, 11400, Uruguay","<b>Author:<\/b> Brandes, Mariana<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Bioinformatics Unit, Institut Pasteur de Montevideo, Montevideo, 11400, Uruguay","<b>Author:<\/b> Arce, Rodrigo<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Laboratorio de Evolución Experimental de Virus, Institut Pasteur de Montevideo <br>- Laboratorio de Biología Molecular, Sanatorio Americano, Montevideo, 11600, Uruguay <br>- Laboratorio de Virología Molecular, Facultad de Ciencias, Universidad de la República","<b>Author:<\/b> Ciappesoni, Gabriel<br><b>Countries:<\/b> United Kingdom; Uruguay<br><b>Affiliations:<\/b><br>- <br>- Instituto Nacional de Investigación Agropecuaria, INIA Las Brujas, Rincón del Colorado, Canelones, Ruta 48, km 10, Uruguay <br>- Instituto Nacional de Investigación Agropecuaria, INIA Las Brujas, Ruta 48, km 10, Rincón del Colorado, Canelones, 90200, Uruguay","<b>Author:<\/b> Soutullo, Álvaro<br><b>Countries:<\/b> Uruguay; Spain<br><b>Affiliations:<\/b><br>- Estación Biológica Terra Natura, CIBIO, Universidad de Alicante, Apdo. Correos 99, E-03080 Alicante, Campus San Vicente del Raspeig, s/n, Spain <br>- Área de Biodiversidad y Conservación, Museo Nacional de Historia Natural, CP 11000 Montevideo, Casilla de Correo 399, Uruguay <br>- Departamento de Ecología y Gestión Ambiental, Centro Universitario Regional del Este, Universidad de la República, Calle Tacuarembó esquina Bvar. Artigas, Maldonado, 20000, Uruguay <br>- Laboratorio de Etología, Ecología and Evolución, Instituto de Investigaciones Biológicas Clemente Estable, CP 11600 Montevideo, Avenida Italia 3318, Uruguay","<b>Author:<\/b> Hernández, Daniel<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Laboratorio de Control Ambiental, Polo Educativo Tecnológico Arrayanes, Camino de los Arrayanes km 7, Piriápolis, 20200, Uruguay","<b>Author:<\/b> González, Susana<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Departamento de Biodiversidad y Genética, Instituto de Investigaciones Biológicas Clemente Estable, Ministerio de Educación y Cultura, Avenida Italia 3318, Montevideo, 11600, Uruguay","<b>Author:<\/b> Bartesaghi, Lucía<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Departamento de Biodiversidad y Genética, Instituto de Investigaciones Biológicas Clemente Estable, Ministerio de Educación y Cultura, Avenida Italia 3318, Montevideo, 11600, Uruguay","<b>Author:<\/b> Cosse, Mariana<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Departamento de Biodiversidad y Genética, Instituto de Investigaciones Biológicas Clemente Estable, Ministerio de Educación y Cultura, Avenida Italia 3318, Montevideo, 11600, Uruguay","<b>Author:<\/b> Rodríguez, Rafael<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Instituto de Mecánica de los Fluidos e Ingeniería Ambiental (IMFIA), Facultad de Ingeniería, Universidad de la República, Montevideo, 11300, Uruguay","<b>Author:<\/b> Pastorini, Marcos<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Instituto de Computación (InCo), Facultad de Ingeniería, Universidad de la República, Montevideo, 11300, Uruguay","<b>Author:<\/b> Etcheverry, Lorena<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Instituto de Computación (InCo), Facultad de Ingeniería, Universidad de la República, Montevideo, 11300, Uruguay","<b>Author:<\/b> Chreties, Christian<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Instituto de Mecánica de los Fluidos e Ingeniería Ambiental (IMFIA), Facultad de Ingeniería, Universidad de la República, Montevideo, 11300, Uruguay","<b>Author:<\/b> Fossati, Mónica<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Instituto de Mecánica de los Fluidos e Ingeniería Ambiental (IMFIA), Facultad de Ingeniería, Universidad de la República, Montevideo, 11300, Uruguay","<b>Author:<\/b> Castro, Alberto<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Instituto de Computación (InCo), Facultad de Ingeniería, Universidad de la República, Montevideo, 11300, Uruguay","<b>Author:<\/b> Gorgoglione, Angela<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Instituto de Mecánica de los Fluidos e Ingeniería Ambiental (IMFIA), Facultad de Ingeniería, Universidad de la República, Montevideo, 11300, Uruguay","<b>Author:<\/b> Fort, Hugo<br><b>Countries:<\/b> United Kingdom; Uruguay<br><b>Affiliations:<\/b><br>- <br>- Institute of Physics, Faculty of Science, Universidad de la República, Iguá 4225, Montevideo, 11400, Uruguay <br>- Institute of Physics, Faculty of Science, Universidad de la República","<b>Author:<\/b> Tambusso, P. Sebastián<br><b>Countries:<\/b> United Kingdom; Uruguay<br><b>Affiliations:<\/b><br>- <br>- Departamento de Paleontología, Facultad de Ciencias, Universidad de La República, Iguá 4225, Montevideo, 11400, Uruguay <br>- Servicio Académico Universitario y Centro de Estudio Paleontológicos (SAUCE-P), Universidad de La República, Departamento de Canelones, Santa Isabel S/n, Sauce, 91500, Uruguay","<b>Author:<\/b> Varela, Luciano<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Servicio Académico Universitario y Centro de Estudio Paleontológicos (SAUCE-P), Universidad de La República, Departamento de Canelones, Santa Isabel S/n, Sauce, 91500, Uruguay <br>- Departamento de Paleontología, Facultad de Ciencias, Universidad de la República, Iguá 4225, Montevideo, 11400, Uruguay <br>- Departamento de Paleontología, Facultad de Ciencias, Universidad de La República, Iguá 4225, Montevideo, 11400, Uruguay","<b>Author:<\/b> Fariña, Richard A.<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Servicio Académico Universitario y Centro de Estudio Paleontológicos (SAUCE-P), Universidad de La República, Departamento de Canelones, Santa Isabel S/n, Sauce, 91500, Uruguay <br>- Departamento de Paleontología, Facultad de Ciencias, Universidad de la República, Iguá 4225, Montevideo, 11400, Uruguay <br>- Departamento de Paleontología, Facultad de Ciencias, Universidad de La República, Iguá 4225, Montevideo, 11400, Uruguay","<b>Author:<\/b> González-Bergonzoni, Iván<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Departamento del Agua, CENUR Litoral Norte Universidad de la República <br>- Departamento de Ecología y Biología Evolutiva, Instituto de Investigaciones Biológicas Clemente Estable, Montevideo, Uruguay <br>- Departamento de Ecología y Biología Evolutiva, Instituto de Investigaciones Biológicas Clemente Estable, MEC, Montevideo, Uruguay <br>- Departamento del Agua CENUR Ltoral Norte sede Paysandú, Universidad de la República, Ruta 3 Km 363 EEMAC, Paysandú, 60000, Uruguay","<b>Author:<\/b> Pejo, Mariana<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Departamento de Ecología y Evolución, Facultad de Ciencias, Universidad de la República, Montevideo, Uruguay","<b>Author:<\/b> Tomasco, Ivanna H.<br><b>Countries:<\/b> United Kingdom; Uruguay<br><b>Affiliations:<\/b><br>- <br>- Departamento de Ecología y Evolución, Facultad de Ciencias, Universidad de la República, Montevideo 11400, Iguá 4225, Uruguay <br>- Departamento de Ecología y Evolución, Facultad de Ciencias, Universidad de la República, Iguá 4225, Montevideo, 11400, Uruguay <br>- Departamento de Ecología y Evolución, Facultad de Ciencias, Universidad de la República, Montevideo, Uruguay","<b>Author:<\/b> Aguilar, Ignacio<br><b>Countries:<\/b> United Kingdom; Uruguay<br><b>Affiliations:<\/b><br>- <br>- Department of Animal Breeding, Instituto Nacional de Investigación Agropecuaria <br>- Instituto Nacional de Investigacion Agropecuaria, Las Brujas 90200, Uruguay <br>- Instituto Nacional de Investigación Agropecuaria (INIA), Montevideo, 11500, Uruguay <br>- Department of Animal and Dairy Science, University of Georgia <br>- Instituto Nacional de Investigación Agropecuaria (INIA) Las Brujas <br>- National Agricultural Research Institute, Las Brujas, Uruguay <br>- Instituto Nacional de Investigación Agropecuaria (INIA), Rincón del Colorado, CA, 90200, Uruguay <br>- Instituto Nacional de Investigación Agropecuaria, Las Brujas 90200, Uruguay","<b>Author:<\/b> Edye, Ernesto Ocampo<br><b>Countries:<\/b> United Kingdom; Uruguay<br><b>Affiliations:<\/b><br>- <br>- Departamento de Informática y Ciencias de la Computación, Universidad Católica Del Uruguay, CP 11600, Montevideo, Uruguay","<b>Author:<\/b> Delgado, Silvia Herrera<br><b>Countries:<\/b> United Kingdom; Uruguay<br><b>Affiliations:<\/b><br>- <br>- Clínica Pediátrica, Hospital Central FFAA, CP 11600, Montevideo, Uruguay <br>- Centro de Referencia Nacional de VIH-SIDA, Hospital Pereira-Rossell","<b>Author:<\/b> Umpiérrez, Martín<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Pharmaceutical Sciences Department, Faculty of Chemistry, Universidad de la República, Uruguay, General Flores, Montevideo, 2124, Uruguay","<b>Author:<\/b> Ibarra, Manuel<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Pharmaceutical Sciences Department, Faculty of Chemistry, Universidad de la República, Uruguay, General Flores, Montevideo, 2124, Uruguay","<b>Author:<\/b> Fagiolino, Pietro<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Catedra de Farmacologia/Biofarmacia, Facultad de Quimica, 11800 Montevideo, CC 1157, Uruguay <br>- Pharmaceutical Sciences Department, Faculty of Chemistry, Universidad de la República, Uruguay, General Flores, Montevideo, 2124, Uruguay","<b>Author:<\/b> Vázquez, Marta<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Pharmaceutical Sciences Department, Faculty of Chemistry, Universidad de la República, Uruguay, General Flores, Montevideo, 2124, Uruguay","<b>Author:<\/b> Maldonado, Cecilia<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Pharmaceutical Sciences Department, Faculty of Chemistry, Universidad de la República, Uruguay, General Flores, Montevideo, 2124, Uruguay","<b>Author:<\/b> Parodi, Pablo<br><b>Countries:<\/b> United Kingdom; Uruguay<br><b>Affiliations:<\/b><br>- <br>- Facultad de Veterinaria, Universidad de la Republica, Alberto Lasplaces 1550, Montevideo, 11600, CP, Uruguay <br>- Instituto Nacional de Investigación Agropecuaria (INIA), Plataforma de Salud Animal, Estación Experimental INIA Tacuarembó, Ruta 5 km 386, Tacuarembó, 45000, CP, Uruguay","<b>Author:<\/b> Rivero, Rodolfo<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Laboratorio Regional Noroeste “Miguel C. Rubino”, División de Laboratorios Veterinarios “Miguel C. Rubino”, Paysandú, Uruguay","<b>Author:<\/b> Miraballes, Cecilia<br><b>Countries:<\/b> United Kingdom; Uruguay<br><b>Affiliations:<\/b><br>- <br>- Facultad de Veterinaria, Universidad de la Republica, Alberto Lasplaces 1550, Montevideo, 11600, CP, Uruguay <br>- Instituto Nacional de Investigación Agropecuaria (INIA), Plataforma de Salud Animal, Estación Experimental INIA Tacuarembó, Ruta 5 km 386, Tacuarembó, 45000, CP, Uruguay","<b>Author:<\/b> Riet-Correa, Franklin<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Facultad de Veterinaria, Universidad de la Republica, Alberto Lasplaces 1550, Montevideo, 11600, CP, Uruguay <br>- Instituto Nacional de Investigación Agropecuaria, Plataforma de Salud Animal, Estación Experimental INIA, Tacuarembó, Uruguay <br>- Instituto Nacional de Investigación Agropecuaria (INIA), Plataforma de Salud Animal, Estación Experimental INIA Tacuarembó, Ruta 5 km 386, Tacuarembó, 45000, CP, Uruguay","<b>Author:<\/b> Goldberg, Virginia<br><b>Countries:<\/b> United Kingdom; Uruguay<br><b>Affiliations:<\/b><br>- <br>- Instituto Nacional de Investigación Agropecuaria (INIA) Las Brujas, Uruguay <br>- Instituto Nacional de Investigación Agropecuaria, INIA Las Brujas, Rincón del Colorado, Canelones, Ruta 48, km 10, Uruguay <br>- Instituto Nacional de Investigación Agropecuaria, INIA Las Brujas, Ruta 48, km 10, Rincón del Colorado, Canelones, 90200, Uruguay","<b>Author:<\/b> Colina, Rodney<br><b>Countries:<\/b> United Kingdom; Uruguay<br><b>Affiliations:<\/b><br>- <br>- Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, 11400 Montevideo, Iguá 4225, Uruguay <br>- Laboratorio de Virología Molecular, Centro Universitario Regional Noroeste (CENUR Noroeste), Universidad de la Republica (UdelaR) <br>- Laboratorio de Virología Molecular, CENUR Litoral Norte, Sede Salto, Universidad de la República, Uruguay <br>- Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, 11400 Montevideo, Igua 4225, Uruguay","<b>Author:<\/b> Cristina, Juan<br><b>Countries:<\/b> United Kingdom; Uruguay<br><b>Affiliations:<\/b><br>- <br>- Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, 11400 Montevideo, Iguá 4225, Uruguay <br>- Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, Universidad de la República <br>- Facultad de Ciencias, Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Universidad de la Republica <br>- Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, 11400 Montevideo, Igua 4225, Uruguay <br>- Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, Universidad de la República, Iguá 4225, Montevideo, 11400, Uruguay <br>- Laboratorio de Virologia Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, Universidad de la Republica, Igua 4225, Montevideo, 11400, Uruguay <br>- Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias Universidad de la República, Igua 4225, 11400 Montevideo, Uruguay <br>- Molecular Virology Laboratory, CIN, Facultad de Ciencias, Universid ad de la Republica","<b>Author:<\/b> Iewdiukow M.<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Instituto Nacional de Investigación Agropecuaria (INIA), Programa Producción de Carne y Lana, Estación Experimental INIA Treinta y Tres, Ruta 8 km 281, 33000, Treinta y Tres, Uruguay","<b>Author:<\/b> Lema O.M.<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Instituto Nacional de Investigación Agropecuaria (INIA), Programa Producción de Carne y Lana, Estación Experimental INIA Treinta y Tres, Ruta 8 km 281, 33000, Treinta y Tres, Uruguay","<b>Author:<\/b> Quintans G.<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Instituto Nacional de Investigación Agropecuaria (INIA), Programa Producción de Carne y Lana, Estación Experimental INIA Treinta y Tres, Ruta 8 km 281, 33000, Treinta y Tres, Uruguay","<b>Author:<\/b> Pereira-Gomez, Marianoel<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, Universidad de la República <br>- Laboratorio de Inmunovirología, Institut Pasteur de Montevideo, Montevideo, Uruguay","<b>Author:<\/b> Monteverde, Eliana<br><b>Countries:<\/b> United States; Uruguay<br><b>Affiliations:<\/b><br>- Section of Plant Breeding and Genetics, Cornell University, Ithaca, NY, United States <br>- Laboratory of Plant Evolution and Domestication, Department of Plant Biology, Universidad de la República, Av. E. Garzón 780, Montevideo, CP12900, Uruguay","<b>Author:<\/b> Haakonsson, Signe<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Physiology and Ecology Phytoplankton Group, CSIC 1176, Uruguay <br>- Sección Limnología, Instituto de Ecología y Ciencias Ambientales, Facultad de Ciencias, Universidad de la República, Iguá 4225, Montevideo, 11400, Uruguay","<b>Author:<\/b> Pérez, María del Carmen<br><b>Countries:<\/b> United Kingdom; Uruguay<br><b>Affiliations:<\/b><br>- <br>- Physiology and Ecology Phytoplankton Group, CSIC 1176, Uruguay","<b>Author:<\/b> Bonilla, Sylvia<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Physiology and Ecology Phytoplankton Group, CSIC 1176, Uruguay <br>- Sección Limnología, Instituto de Ecología y Ciencias Ambientales, Facultad de Ciencias, Universidad de la República","<b>Author:<\/b> Fernández, Peter<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Instituto Nacional de Investigación Agropecuaria, Colonia, 70000, Uruguay","<b>Author:<\/b> Galván, Guillermo A.<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Dep. de Producción Vegetal, Centro Regional Sur (CRS), Facultad de Agronomía, Univ. de la República, Progreso, Canelones, Uruguay <br>- Department of Plant Production, Centro Regional sur (CRS), Universidad de la República, Camino Folle km 36 Progreso, Uruguay <br>- Universidad de la República, Facultad de Agronomía, Departamento de Producción Vegetal, Centro Regional Sur, Canelones, 90100, Uruguay","<b>Author:<\/b> Priano, Federico Condón<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Instituto Nacional de Investigación Agropecuaria, Unidad de Semillas y Recursos Fitogenéticos, Estación Experimental La Estanzuela, Colonia, 70000, Uruguay","<b>Author:<\/b> Alvarez-Castro, Ignacio<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Universidad de la República <br>- Instituto de Estadística, Universidad de la República, Montevideo, Uruguay","<b>Author:<\/b> Massaferro, Pablo<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Universidad de la República, Montevideo, 11200, Uruguay","<b>Author:<\/b> Alvarez-Valin, Fernando<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Sección Biomatemática-Facultad de Ciencias, Universidad de la República <br>- Sección Biomatemática, Unidad de Genómica Evolutiva, Facultad de Ciencias, Universidad de la República, Montevideo, Uruguay","<b>Author:<\/b> Dubra, Juan<br><b>Countries:<\/b> United Kingdom; Uruguay<br><b>Affiliations:<\/b><br>- <br>- Dept. of Economics, Universidad de Montevideo, Montevideo, CP 11400, Prudencio de Pena 2440, Uruguay <br>- Universidad de Montevideo","<b>Author:<\/b> Lenzi, Javier<br><b>Countries:<\/b> United States; Uruguay<br><b>Affiliations:<\/b><br>- Purdue University, Department of Forestry and Natural Resources, 715 West State Street, West Lafayette, 47907, IN, United States <br>- Purdue University, Ecological Sciences and Engineering Program, 715 West State Street, West Lafayette, 47907, IN, United States <br>- Centro de Investigación y Conservación Marina - CICMAR, Avenida Giannattasio Km 30.5, Canelones, 15008, Uruguay","<b>Author:<\/b> Machín, Emanuel<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Facultad de Ciencias, Universidad de la República, Iguá 4225, Montevideo, 11400, Uruguay","<b>Author:<\/b> González E.M.<br><b>Countries:<\/b> United Kingdom; Uruguay<br><b>Affiliations:<\/b><br>- <br>- Museo Nacional de Historia Natural, 25 de mayo 582, Montevideo, CEP 11000, Uruguay","<b>Author:<\/b> Villamil, Joaquín<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Laboratorio de Sistemática e Historia Natural de Vertebrados, Instituto de Ecología y Ciencias Ambientales, Facultad de Ciencias, UdelaR, Iguá 4225, Montevideo, Uruguay","<b>Author:<\/b> Maneyro, Raúl<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Laboratorio de Sistemática e Historia Natural de Vertebrados IECA, Facultad de Ciencias, Universidad de la República <br>- Laboratorio de Sistemática e Historia Natural de Vertebrados, Instituto de Ecología y Ciencias Ambientales, Facultad de Ciencias, UdelaR, Iguá 4225, Montevideo, Uruguay <br>- Laboratorio de Sistemática e Historia Natural de Vertebrados, Facultad de Ciencias, Universidad de la República","<b>Author:<\/b> Camargo, Arley<br><b>Countries:<\/b> Argentina; Uruguay<br><b>Affiliations:<\/b><br>- Department of Biology and Monte L. Bean Museum, Brigham Young University <br>- Unidad de Diversidad Sistemática y Evoluciõn, Centro Nacional Patagõnico, Consejo Nacional de Investigaciones Científicas y Técnicas, U9120ACD Puerto Madryn Chubut, Boulevard Almirante Brown 2915, Argentina <br>- Programa de Desarrollo Universitario, Centro Universitario de Rivera, Universidad de la República – UdelaR, Rivera, Uruguay <br>- Centro Universitario de Rivera, Universidad de la República","<b>Author:<\/b> Nunez, Alvaro<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- División Laboratorios Veterinarios “Miguel C. Rubino”, Ministerio de Ganadería, Agricultura y Pesca, Montevideo, Uruguay","<b>Author:<\/b> Suanes, Alejandra<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- División Laboratorios Veterinarios “Miguel C. Rubino”, Ministerio de Ganadería, Agricultura y Pesca, Montevideo, Uruguay","<b>Author:<\/b> Talento, Stefanie<br><b>Countries:<\/b> Uruguay; Germany<br><b>Affiliations:<\/b><br>- Department of Geography, Climatology, Climate Dynamics and Climate Change, Justus Liebig University of Giessen, Giessen, Germany <br>- Physics Institute, Science Faculty, Universidad de la República, Montevideo, Uruguay","<b>Author:<\/b> Romero, Héctor<br><b>Countries:<\/b> United Kingdom; Uruguay<br><b>Affiliations:<\/b><br>- <br>- Departamento de Ecología Y Evolucion, Facultad de Ciencias-CURE, Universidad de la Republica, Montevideo, Uruguay <br>- Laboratorio de Organización Y Evolución del Genoma, Departamento de Biología Celular Y Molecular, Facultad de Ciencias <br>- Laboratorio de Organización y Evolución del Genoma, UDELAR <br>- Escuela Universitaria de Tecnología Médica, Facultad de Medicina <br>- Genome Organization and Evolution Laboratory, Ecology and Evolution Department, Faculty of Sciences, UdelaR, Montevideo, 11400, Uruguay <br>- Laboratorio de Organizacion y Evolución del Genoma, Sección Biomatemáticas, Universidad de la República, Montevideo, Uruguay","<b>Author:<\/b> Saporiti, Tatiana<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Facultad de Veterinaria, Universidad de la Republica, Alberto Lasplaces 1550, Montevideo, 11600, CP, Uruguay <br>- Instituto Nacional de Investigación Agropecuaria (INIA), Plataforma de Salud Animal, Estación Experimental INIA Tacuarembó, Ruta 5 km 386, Tacuarembó, 45000, CP, Uruguay","<b>Author:<\/b> Lara, Stephanie<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Facultad de Veterinaria, Universidad de la Republica, Alberto Lasplaces 1550, Montevideo, 11600, CP, Uruguay","<b>Author:<\/b> Belzarena, Pablo<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Facultad de Ingeniería, Universidad de la República, Montevideo, Uruguay","<b>Author:<\/b> Alaniz S.<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Department of Plant Protection, Faculty of Agronomy, University of the Republic, Av. Garz on 780, Montevideo, CP 12900, Uruguay <br>- Department of Plant Protection, Faculty of Agronomy, University of the Republic, Av. Garzón 780, Montevideo, CP 12900, Uruguay","<b>Author:<\/b> Mondino P.<br><b>Countries:<\/b> United Kingdom; Uruguay<br><b>Affiliations:<\/b><br>- <br>- Department of Plant Protection, Faculty of Agronomy, University of the Republic, Av. Garz on 780, Montevideo, CP 12900, Uruguay","<b>Author:<\/b> Artecona, Florencia<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Laboratorio de Zoología Vertebrados, Dpto de Ecología y Evolución, Facultad de Ciencias, Universidad de la República, Iguá 4225, Montevideo, 11400, Uruguay","<b>Author:<\/b> Bergamino, Leandro<br><b>Countries:<\/b> United Kingdom; Uruguay<br><b>Affiliations:<\/b><br>- <br>- Centro Universitario Regional Este (CURE), Universidad de la República, Rutas 9 y 15 s/n, Rocha, Uruguay <br>- Centro Universitario Regional Este (CURE), Universidad de la República <br>- Department of Zoology and Entomology, Rhodes University","<b>Author:<\/b> Szteren, Diana<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Dpto. de Ecología y Evolución, Facultad de Ciencias, Universidad de la República, Iguá 4225, Montevideo, 11400, Uruguay <br>- Laboratorio de Zoología Vertebrados, Dpto de Ecología y Evolución, Facultad de Ciencias, Universidad de la República","<b>Author:<\/b> Speranza, Pablo R.<br><b>Countries:<\/b> United Kingdom; Uruguay<br><b>Affiliations:<\/b><br>- <br>- Departamento de Biología Vegetal, Facultad de Agronomía, Universidad de la República, Av. E. Garzón 780, Montevideo, 12900, Uruguay <br>- Laboratory of Plant Evolution and Domestication, Department of Plant Biology, Universidad de la República, Av. E. Garzón 780, Montevideo, CP12900, Uruguay","<b>Author:<\/b> Simón, Diego<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, Universidad de la República, Iguá 4225, Montevideo, 11400, Uruguay <br>- Laboratorio de Organización y Evolución del Genoma, Unidad de Genómica Evolutiva, Facultad de Ciencias, Universidad de la República, Iguá 4225, Montevideo, 11400, Uruguay","<b>Author:<\/b> Moreno, Pilar<br><b>Countries:<\/b> United Kingdom; Uruguay<br><b>Affiliations:<\/b><br>- <br>- Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, Universidad de la República, Iguá 4225, Montevideo, 11400, Uruguay <br>- Molecular Virology Laboratory, CIN, Facultad de Ciencias, Universid ad de la Republica","<b>Author:<\/b> Moratorio, Gonzalo<br><b>Countries:<\/b> United Kingdom; Uruguay<br><b>Affiliations:<\/b><br>- <br>- Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, 11400 Montevideo, Iguá 4225, Uruguay <br>- Unidad de Biofísica de Proteínas, Institut Pasteur de Montevideo <br>- Departamento de Técnicas, Nucleares Aplicadas, Facultad de Ciencias, Centro de Investigaciones Nucleares, Universidad de la República <br>- Unidad de Biofísica de Proteínas, Institut Pasteur-Montevideo, 11400 Montevideo, Mataojo 2020, Uruguay <br>- Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, 11400 Montevideo, Igua 4225, Uruguay <br>- Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, Universidad de la República, Iguá 4225, Montevideo, 11400, Uruguay <br>- Viral Populations and Pathogenesis laboratory. Institut Pasteur <br>- Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias Universidad de la República, Igua 4225, 11400 Montevideo, Uruguay <br>- Laboratorio de Inmunovirología, Institut Pasteur de Montevideo, Mataojo 2020, Montevideo, 11400, Uruguay <br>- Molecular Virology Laboratory, CIN, Facultad de Ciencias, Universid ad de la Republica","<b>Author:<\/b> Drago, Massimiliano<br><b>Countries:<\/b> United Kingdom; Uruguay; Ecuador; Spain<br><b>Affiliations:<\/b><br>- <br>- Departamento de Ecología y Evolución, Centro Universitario Regional Este (CURE), Universidad de la República, C/Tacuarembó s/n, Maldonado, 20000, Uruguay <br>- IRBio and Department of Evolutive Biology, Ecology and Environmental Sciences, University of Barcelona, Av. Diagonal 643, Barcelona, 08028, Spain <br>- Department of Animal Biology, Faculty of Biology, University of Barcelona <br>- Programa PROMETEO-SENESCYT, Secretaría de Educación Superior, Ciencia, Tecnología e Innovación, Quito, Ecuador <br>- Department of Ecology and Evolution, Centro Universitario Regional Este (CURE), University of the Republic (UdeLaR), Maldonado, Uruguay","<b>Author:<\/b> Franco-Trecu, Valentina<br><b>Countries:<\/b> United Kingdom; Uruguay<br><b>Affiliations:<\/b><br>- <br>- Proyecto Franciscana, Sección Etología, Facultad de Ciencias, Universidad de la República, Iguá 4225, Montevideo, 11400, Uruguay <br>- Departamento de Ecología y Evolución, Facultad de Ciencias, Universidad de la República, Iguá 4225, Montevideo, 11400, Uruguay <br>- Department of Ecology and Evolution, Faculty of Sciences, Universidad de la República, Iguá 4225, Montevideo, 11400, Uruguay <br>- Department of Ecology and Evolution, Faculty of Sciences, University of the Republic (UdeLaR)","<b>Author:<\/b> Segura, Angel M.<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- National Museum of Natural History (MNHN), 25 de Mayo 582, Montevideo, 11000, Uruguay <br>- Modelization and Analysis of Natural Resources (MAREN), Centro Universitario Regional Este (CURE), Universidad de la República","<b>Author:<\/b> Valdivia, Meica<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- National Museum of Natural History (MNHN), 25 de Mayo 582, Montevideo, 11000, Uruguay","<b>Author:<\/b> González, Enrique M.<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- National Museum of Natural History (MNHN), 25 de Mayo 582, Montevideo, 11000, Uruguay","<b>Author:<\/b> Mir, Daiana<br><b>Countries:<\/b> United Kingdom; Uruguay; Brazil<br><b>Affiliations:<\/b><br>- <br>- Departamento de Ecología y Evolución, Facultad de Ciencias, Universidad de la República, Montevideo, Uruguay <br>- Laboratório de AIDS and Imunologia Molecular, Instituto Oswaldo Cruz, FIOCRUZ <br>- Fundação Oswaldo Cruz-Fiocruz, Instituto Oswaldo Cruz, Laboratório de AIDS e Imunologia Molecular, Rio de Janeiro, RJ, Brazil <br>- Laboratorio de AIDS and Imunologia Molecular, Instituto Oswaldo Cruz, Av. Brasil 4365, Rio de Janeiro, 21045900, Brazil","<b>Author:<\/b> Victoria, Matías<br><b>Countries:<\/b> Uruguay; Brazil<br><b>Affiliations:<\/b><br>- Department of Biological Sciences, Laboratory of Molecular Virology, CENUR Litoral Norte, Sede Salto, Universidad de la República <br>- Universidad de la República, Laboratorio de Virología Molecular - Regional Norte <br>- Laboratório de Virologia Comparada e Ambiental, Instituto Oswaldo Cruz, FIOCRUZ, Manguinhos, 21040-360 Rio de Janeiro, RJ, Av. Brasil 4365, Brazil <br>- Laboratorio de Virología Molecular, CENUR Litoral Norte, Sede Salto, Universidad de la República, Uruguay","<b>Author:<\/b> Cervetto, Sabrina<br><b>Countries:<\/b> Argentina; Uruguay<br><b>Affiliations:<\/b><br>- Laboratory of Experimental Psychology and Neuroscience, Institute of Cognitive and Translational Neuroscience, INECO Foundation, Favaloro University, Buenos Aires, Argentina <br>- Departamento de Educación Física y Salud, Instituto Superior de Educación Física, Universidad de la República, Montevideo, Uruguay","<b>Author:<\/b> Scarabino, Fabrizio<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Centro Universitario Regional del Este (CURE), Universidad de la República, Intersección Ruta 9 y Ruta 15, Rocha, 27000, Uruguay","<b>Author:<\/b> Ortega, Leonardo<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Dirección Nacional de Recursos Acuáticos (DINARA), Montevideo, 11200, Uruguay","<b>Author:<\/b> Fabiano, Graciela<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Dirección Nacional de Recursos Acuáticos (DINARA), Montevideo, 11200, Uruguay","<b>Author:<\/b> Muse, Pablo<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Facultad de Ingeniería, Universidad de la República, Uruguay","<b>Author:<\/b> Li, Jing-Jing<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Unidad de Bioinform atica, Institut Pasteur de Montevideo, Mataojo, Montevideo, 2020, Uruguay","<b>Author:<\/b> Soñora, Martín<br><b>Countries:<\/b> United Kingdom; Uruguay<br><b>Affiliations:<\/b><br>- <br>- Laboratorio de Virologia Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, Universidad de la Republica, Igua 4225, Montevideo, 11400, Uruguay <br>- Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias Universidad de la República, Igua 4225, 11400 Montevideo, Uruguay <br>- Facultad de Ciencias, Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Universidad de la Republica","<b>Author:<\/b> Bonifacino, Jose M.<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Laboratorio de Botánica, Facultad de Agronomía, Universidad de la República, Montevideo, Casilla de Correos 1238, Uruguay <br>- Laboratorio de Botánica, Universidad de la República, Montevideo, Casilla de Correos 1238, Uruguay <br>- Laboratorio de Botánica, Facultad de Agronomía, Casilla de Correos 1238, Montevideo, Uruguay","<b>Author:<\/b> Vergara, Julia<br><b>Countries:<\/b> Argentina; Uruguay<br><b>Affiliations:<\/b><br>- Sección Genética Evolutiva, Facultad de Ciencias, UdelaR, CP.11400 Montevideo, Iguá 4225, Uruguay <br>- Instituto de Diversidad y Ecología Animal (IDEA), CONICET-Universidad Nacional de Córdoba, and Cátedra de Diversidad Animal i, Facultad de Ciencias Exactas, Físicas y Naturales, X5000JJC Córdoba, Av. Vélez Sarsfield 299, Argentina <br>- CONICET-Cátedra de Diversidad Animal I, FCEFyN, UNC, Argentina <br>- Cátedra de Diversidad Animal I, Facultad de Ciencias Exactas, Físicas y Naturales, U. N. C., Córdoba, Argentina <br>- Sección Genética Evolutiva, Facultad de Ciencias, Montevideo, Uruguay <br>- Instituto de Diversidad y Ecología Animal (IDEA), CONICET – Universidad Nacional de Córdoba, Cátedra de Diversidad Animal I, Físicas y Naturales, U.N.C., Av. Vélez Sarsfield 299, Córdoba, X5000JJC, Argentina <br>- Instituto de Diversidad y Ecología Animal (IDEA), CONICET-Universidad Nacional de Córdoba <br>- Sección Genética Evolutiva, Facultad de Ciencias, Universidad de la República, Montevideo, Iguá 4225, CP 11400, Uruguay","<b>Author:<\/b> Bia, Daniel<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Department of Physiology, Universidad de la República, Montevideo, 11800, Uruguay","<b>Author:<\/b> German, Yanina Zócalo<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Department of Physiology, Universidad de la República, Montevideo, 11800, Uruguay","<b>Author:<\/b> Dimitriadis, Caterina<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Proyecto Franciscana, Sección Etología, Facultad de Ciencias, Universidad de la República, Iguá 4225, Montevideo, 11400, Uruguay","<b>Author:<\/b> Passadore, Cecilia<br><b>Countries:<\/b> Australia; Uruguay<br><b>Affiliations:<\/b><br>- Proyecto Franciscana, Sección Etología, Facultad de Ciencias, Universidad de la República, Iguá 4225, Montevideo, 11400, Uruguay <br>- Cetacean Ecology, Behaviour and Evolution Lab, School of Biological Sciences, Flinders University, Sturt Road, Bedford Park, 5042, South Australia, Australia","<b>Author:<\/b> Ferrari A.<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Chemical and Process Systems Engineering Group, Chemical Engineering Inst. - Engineering School, Universidad de la República, Julio Herrera y Reissig 565, Montevideo, CP 11300, Uruguay","<b>Author:<\/b> Musto, Héctor<br><b>Countries:<\/b> United Kingdom; Uruguay<br><b>Affiliations:<\/b><br>- <br>- Laboratorio de Organización y Evolución del Genoma, Instituto de Biología, Facultad de Ciencias, 11400 Montevideo, Iguá 4225, Uruguay <br>- Laboratorio de Organización Y Evolución del Genoma, Departamento de Biología Celular Y Molecular, Facultad de Ciencias <br>- Laboratorio de Organización y Evolución del Genoma, UDELAR","<b>Author:<\/b> Toranza, Carolina<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Grupo Biodiversidad y Ecología de la Conservación IECA, Facultad de Ciencias, Universidad de la República, CP: 11600, Uruguay <br>- Centro Interdisciplinario de Respuesta al Cambio y la Variabilidad Climática, Espacio Interdisciplinario, Universidad de la República <br>- Laboratorio de Sistemática e Historia Natural de Vertebrados IECA, Facultad de Ciencias, Universidad de la República, Uruguay","<b>Author:<\/b> Brazeiro, Alejandro<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Grupo Biodiversidad y Ecología de la Conservación IECA, Facultad de Ciencias, Universidad de la República, CP: 11600, Uruguay","<b>Author:<\/b> Espasandín A.C.<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Department of Animal and Forage Production Science, University of the Republic-School of Agronomy-EEMAC, Paysandú, Uruguay <br>- Departamento de Producción Animal Y Pasturas, Universidad de La República (UDELAR), Paysandú, Uruguay","<b>Author:<\/b> Llambí S.<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Instituto de Producción Animal, Área Genética, Facultad de Veterinaria, Universidad de la República (UdelaR), Montevideo, Uruguay","<b>Author:<\/b> Iraola, Gregorio<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Sección Genética Evolutiva, Instituto de Biología, Facultad de Ciencias, Universidad de la República, Montevideo, Uruguay <br>- Unidad de Bioinformática, Institut Pasteur Montevideo, Montevideo, Uruguay","<b>Author:<\/b> Hernández, Martín<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Sección Genética Evolutiva, Instituto de Biología, Facultad de Ciencias, Universidad de la República, Montevideo, Uruguay","<b>Author:<\/b> Ravagnolo O.<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Instituto Nacional de Investigación Agropecuaria, INIA Las Brujas, Ruta 48, km 10, Rincón del Colorado, Canelones, CP 90200, Uruguay","<b>Author:<\/b> Echeverría, Natalia<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Laboratorio de Virologia Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, Universidad de la Republica, Igua 4225, Montevideo, 11400, Uruguay","<b>Author:<\/b> Comas, Victoria<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Laboratorio de Virologia Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, Universidad de la Republica, Igua 4225, Montevideo, 11400, Uruguay <br>- Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias Universidad de la República, Igua 4225, 11400 Montevideo, Uruguay","<b>Author:<\/b> Benítez-Galeano, María José<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Laboratorio de Virología Molecular, Centro Universitario Regional Noroeste (CENUR Noroeste), Universidad de la Republica (UdelaR), Rivera 1350, Salto, 50000, Uruguay","<b>Author:<\/b> Sapriza, F. G. Riet<br><b>Countries:<\/b> Colombia; Uruguay<br><b>Affiliations:<\/b><br>- Proyecto Pinnípedos, Sección Etología, Facultad de Ciencias, Universidad de la República, Iguá 4225, Montevideo, 11400, Uruguay <br>- Departamento Ciencias Biológicas, Facultad de Ciencias, Universidad de los Andes, Carrera 1E No 18A - 10, Bogotá, Colombia","<b>Author:<\/b> Zambra N.<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Departamento de Producción Animal y Pasturas, Facultad de Agronomía, Universidad de la República, Av. E. Garzón 780, Montevideo, 12900, Uruguay","<b>Author:<\/b> Gimeno D.<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Secretariado Uruguayo de la Lana, Rambla Baltasar Brum 3764 <br>- Secretariado Uruguayo de la Lana, Rambla Baltasar Brum 3764, Montevideo, 11800, Uruguay","<b>Author:<\/b> Van Lier E.<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Departamento de Producción Animal y Pasturas, Facultad de Agronomía, Universidad de la República, Av. E. Garzón 780, Montevideo, 12900, Uruguay <br>- Estación Experimental Facultad de Agronomía Salto, Ruta 31 Km 21, Salto, 50000, Uruguay","<b>Author:<\/b> Domingo, Andrés<br><b>Countries:<\/b> United Kingdom; Uruguay<br><b>Affiliations:<\/b><br>- <br>- Departamento de Recursos Pelágicos, Dirección Nacional de Recursos Acuáticos (DINARA), Constituyente 1497, Montevideo, CP 11200, Uruguay","<b>Author:<\/b> Naya, Daniel E.<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Departamento de Ecología Y Evolución, Facultad de Ciencias, Universidad de la República, Montevideo, Uruguay","<b>Author:<\/b> Duarte, Alejandro<br><b>Countries:<\/b> United Kingdom; Uruguay<br><b>Affiliations:<\/b><br>- <br>- Sección Zoología Vertebrados, Facultad de Ciencias, Montevideo, Uruguay","<b>Author:<\/b> Iriarte, Andrés<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Laboratorio de Organización y Evolución del Genoma, UDELAR, 11400 Montevideo, Iguá 4225, Uruguay <br>- Laboratorio de Evolución, UDELAR, 11400 Montevideo, Iguá 4225, Uruguay <br>- Área Genética, Depto. de Genética y Mejora Animal, UDELAR, CP 11600, Montevideo, Av. A. Lasplaces 1550, Uruguay","<b>Author:<\/b> Castro-Sowinski, Susana<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Sección Bioquímica y Biología Molecular, UDELAR, 11400 Montevideo, Iguá 4225, Uruguay","<b>Author:<\/b> Costa-Urrutia, Paula<br><b>Countries:<\/b> Uruguay; Mexico<br><b>Affiliations:<\/b><br>- Departamento de Ecología y Evolución Facultad de Ciencias, Universidad de la República, 11400 Montevideo, Uruguay <br>- Franciscana Project/Cetaceos Uruguay, Sección de Etología, Facultad de Ciencias, Universidad de la República, 11400 Montevideo, Uruguay <br>- Laboratorio de Ecología Molecular, Facultad de Ciencias Marinas, Universidad Autónoma de Baja California, Baja California, Mexico","<b>Author:<\/b> Coppola, Leticia<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Centro Nacional de Referencia de Influenza, Departamento de Laboratorios de Salud Pública, Ministerio de Salud Pública, 11200 Montevideo, Alfredo Navarro 3051 acceso norte, Uruguay","<b>Author:<\/b> Ramas, Viviana<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Centro Nacional de Referencia de Influenza, Departamento de Laboratorios de Salud Pública, Ministerio de Salud Pública, 11200 Montevideo, Alfredo Navarro 3051 acceso norte, Uruguay","<b>Author:<\/b> Chiparelli, Hector<br><b>Countries:<\/b> United Kingdom; Uruguay<br><b>Affiliations:<\/b><br>- <br>- Centro Nacional de Referencia de Influenza, Departamento de Laboratorios de Salud Pública, Ministerio de Salud Pública, 11200 Montevideo, Alfredo Navarro 3051 acceso norte, Uruguay","<b>Author:<\/b> Dans, Pablo D.<br><b>Countries:<\/b> Uruguay; Spain<br><b>Affiliations:<\/b><br>- Institut Pasteur de Montevideo, Montevideo 11400, Mataojo 2020, Uruguay <br>- Joint IRB-BSC Program on Computational Biology, Institute for Research in Biomedicine (IRB), Parc Cientific de Barcelona, Barcelona 08028, Josep Samitier 1-5, Spain","<b>Author:<\/b> García, Graciela<br><b>Countries:<\/b> United Kingdom; Uruguay<br><b>Affiliations:<\/b><br>- <br>- Sección Genética Evolutiva, Facultad de Ciencias, Montevideo, Uruguay <br>- Sección Genética Evolutiva, Facultad de Ciencias, Universidad de la República, Montevideo, Iguá 4225, CP 11400, Uruguay","<b>Author:<\/b> Gutiérrez, Verónica<br><b>Countries:<\/b> United Kingdom; Uruguay<br><b>Affiliations:<\/b><br>- <br>- Sección Genética Evolutiva, Facultad de Ciencias, UdelaR, CP.11400 Montevideo, Iguá 4225, Uruguay","<b>Author:<\/b> Loureiro, Marcelo<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Sección Zoología Vertebrados, Facultad de Ciencias, Montevideo, Uruguay <br>- Departamento de Ictiología, Museo Nacional de Historia Natural","<b>Author:<\/b> Vaio, Magdalena<br><b>Countries:<\/b> Uruguay; Brazil<br><b>Affiliations:<\/b><br>- Department of Botany, Center of Biological Sciences, Federal University of Pernambuco, Recife, Pernambuco, Rua Nelson Chaves s/n 50670-420, Brazil <br>- Department of Plant Biology, College of Agronomy, University of the Republic, CP 12900 Montevideo, Avenida Eugenio Garzón 780, Uruguay","<b>Author:<\/b> Abud, Carolina<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Departamento de Ecología y Evolución Facultad de Ciencias, Universidad de la República, 11400 Montevideo, Uruguay <br>- Franciscana Project/Cetaceos Uruguay, Sección de Etología, Facultad de Ciencias, Universidad de la República","<b>Author:<\/b> Giossa G.<br><b>Countries:<\/b> Argentina; Uruguay<br><b>Affiliations:<\/b><br>- Facultad de Ciencias Veterinarias, Universidad Nacional de La Plata, La Plata, B1900AVW, Buenos Aires, Avda. 60 y 118, Argentina <br>- Área de Patología y Productión Avícola, Facultad de Veterinaria, Universidad de la República, Montevideo, Uruguay <br>- Universidad de la República <br>- Laboratorio de Diagnóstico de Enfermedades de las Aves y los Pilíferos, Facultad de Ciencias Veterinarias, Universidad Nacional de La Plata <br>- Área de Patologia y Producción Avícola, Facultad de Veterinaria, Universidad de la República, Montevideo, Uruguay <br>- Area de Patologia y Produccion Avicola, Facultad de Veterinaria, Universidad de la Republica, Montevideo, Uruguay","<b>Author:<\/b> Rodríguez G.<br><b>Countries:<\/b> Argentina; Uruguay<br><b>Affiliations:<\/b><br>- Facultad de Ciencias Veterinarias, Universidad Nacional de La Plata, La Plata, B1900AVW, Buenos Aires, Avda. 60 y 118, Argentina <br>- Área de Patología y Productión Avícola, Facultad de Veterinaria, Universidad de la República, Montevideo, Uruguay <br>- Universidad de la República <br>- Laboratorio de Diagnóstico de Enfermedades de las Aves y los Pilíferos, Facultad de Ciencias Veterinarias, Universidad Nacional de La Plata <br>- Área de Patologia y Producción Avícola, Facultad de Veterinaria, Universidad de la República, Montevideo, Uruguay <br>- Area de Patologia y Produccion Avicola, Facultad de Veterinaria, Universidad de la Republica, Montevideo, Uruguay","<b>Author:<\/b> Trenchi G.<br><b>Countries:<\/b> Argentina; Uruguay<br><b>Affiliations:<\/b><br>- Facultad de Ciencias Veterinarias, Universidad Nacional de La Plata, La Plata, B1900AVW, Buenos Aires, Avda. 60 y 118, Argentina <br>- Área de Patología y Productión Avícola, Facultad de Veterinaria, Universidad de la República, Montevideo, Uruguay <br>- Universidad de la República <br>- Laboratorio de Diagnóstico de Enfermedades de las Aves y los Pilíferos, Facultad de Ciencias Veterinarias, Universidad Nacional de La Plata <br>- Área de Patologia y Producción Avícola, Facultad de Veterinaria, Universidad de la República, Montevideo, Uruguay <br>- Area de Patologia y Produccion Avicola, Facultad de Veterinaria, Universidad de la Republica, Montevideo, Uruguay","<b>Author:<\/b> Trenchi H.<br><b>Countries:<\/b> Argentina; Uruguay<br><b>Affiliations:<\/b><br>- Facultad de Ciencias Veterinarias, Universidad Nacional de La Plata, La Plata, B1900AVW, Buenos Aires, Avda. 60 y 118, Argentina <br>- Universidad de la República <br>- Laboratorio de Diagnóstico de Enfermedades de las Aves y los Pilíferos, Facultad de Ciencias Veterinarias, Universidad Nacional de La Plata <br>- Área de Patologia y Producción Avícola, Facultad de Veterinaria, Universidad de la República, Montevideo, Uruguay <br>- Area de Patologia y Produccion Avicola, Facultad de Veterinaria, Universidad de la Republica, Montevideo, Uruguay","<b>Author:<\/b> Nuñez R.<br><b>Countries:<\/b> Argentina; Uruguay<br><b>Affiliations:<\/b><br>- Universidad de la República, Montevideo, Uruguay <br>- Facultad de Ciencias Veterinarias, Universidad Nacional de La Plata, La Plata, B1900AVW, Buenos Aires, Avda. 60 y 118, Argentina","<b>Author:<\/b> MacEiras, Mariana<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Departamento de Informática y Ciencias de la Computación, Universidad Católica Del Uruguay, CP 11600, Montevideo, Uruguay","<b>Author:<\/b> Maurente, Cecilia<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Facultad de Ingeniería y Tecnologías, Universidad Católica del Uruguay, Montevideo, Uruguay <br>- Departamento de Informática y Ciencias de la Computación, Universidad Católica Del Uruguay, CP 11600, Montevideo, Uruguay","<b>Author:<\/b> Ruchansky, Dora<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Laboratorio Nacional de Referencia VIH-SIDA, Servicio Nacional de Laboratorios de Salud Publica - MSP, Montevideo, Uruguay","<b>Author:<\/b> Recarey, Ricardo<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, 11400 Montevideo, Igua 4225, Uruguay","<b>Author:<\/b> D' Andrea, Lucía<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, 11400 Montevideo, Igua 4225, Uruguay","<b>Author:<\/b> Alvarez, Macarena<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Laboratorio de Virología Molecular, Centro de Investigaciones Nucleares, Facultad de Ciencias, 11400 Montevideo, Igua 4225, Uruguay","<b>Author:<\/b> Marmo F.<br><b>Countries:<\/b> Argentina; Uruguay<br><b>Affiliations:<\/b><br>- Facultad de Veterinaria, Universidad de la República, C.P. 11600, Montevideo, Avda. Alberto Lasplaces 1620, Uruguay <br>- PROVETSUR, Facultad de Ciencias Veterinarias, Universidad Nacional de La Plata, La Plata, B1900AVW, Buenos Aires, Avda. 60 y 118, Argentina","<b>Author:<\/b> Guarino H.<br><b>Countries:<\/b> Argentina; Uruguay<br><b>Affiliations:<\/b><br>- Facultad de Veterinaria, Universidad de la República, C.P. 11600, Montevideo, Avda. Alberto Lasplaces 1620, Uruguay <br>- PROVETSUR, Facultad de Ciencias Veterinarias, Universidad Nacional de La Plata, La Plata, B1900AVW, Buenos Aires, Avda. 60 y 118, Argentina","<b>Author:<\/b> Piovani, Rosina<br><b>Countries:<\/b> Uruguay<br><b>Affiliations:<\/b><br>- Laboratorio de Organización y Evolución del Genoma, Instituto de Biología, Facultad de Ciencias, 11400 Montevideo, Iguá 4225, Uruguay","<b>Age-specific rate of severe and critical SARS-CoV-2 infections estimated with multi-country seroprevalence studies<\/b><br><br>Background: Knowing the age-specific rates at which individuals infected with SARS-CoV-2 develop severe and critical disease is essential for designing public policy, for infectious disease modeling, and for individual risk evaluation. Methods: In this study, we present the first estimates of these rates using multi-country serology studies, and public data on hospital admissions and mortality from early to mid-2020. We combine these under a Bayesian framework that accounts for the high heterogeneity between data sources and their respective uncertainties. We also validate our results using an indirect method based on infection fatality rates and hospital mortality data. Results: Our results show that the risk of severe and critical disease increases exponentially with age, but much less steeply than the risk of fatal illness. We also show that our results are consistent across several robustness checks. Conclusion: A complete evaluation of the risks of SARS-CoV-2 for health must take non-fatal disease outcomes into account, particularly in young populations where they can be 2 orders of magnitude more frequent than deaths. © 2022, The Author(s).","<b>Origin and spreading of canine morbillivirus in South America<\/b><br><br>Canine distemper virus (CDV) is a Morbillivirus (Canine morbillivirus) that greatly impacts domestic and wildlife carnivores worldwide. The CDV RNA genome has high genetic variability, evidenced by several lineages that follow a global geographic pattern. The evolutionary trajectories and population dynamics of CDV lineages are still unclear and debatable, particularly in South America, where relatively few sequences are available. We performed phylogenetic and Bayesian analyses using an updated dataset of the highly variable hemagglutinin (H) gene, including seven South American countries. The time to the most recent common ancestor (tMRCA) of the current CDV lineages was dated to the early 1900s in North America. Maximum likelihood and Bayesian maximum clade credibility phylogenies showed similar topologies with two main branches (L1 and L2) corresponding to the NA1 lineage (L1) and the remaining lineages worldwide (L2). The four circulating lineages in South America (EU1/SA1, SA2, SA3, NA4/SA4) arose from independent migration events from North America and Europe. North American strains colonized most northern South American countries via Ecuador and then Colombia and Peru, originating the SA3 and NA4/SA4 lineages during their spread. The entry and expansion in the southern part of South America (Argentina, Brazil, Chile, and Uruguay) occurred through three independent migration events and gave rise to the EU1/SA1 and SA2 lineages. South American lineages have specific combinations of amino acids under positive selection that constitute signatures of taxonomic and evolutionary relevance. Our findings provide a comprehensive scenario for the origin and migration routes of Canine morbillivirus in South America and highlight the importance of phylodynamics in understanding the geographic patterns of modern genetic variability. © 2022 Elsevier B.V.","<b>Wool scoured colour: Heritability, genetic and phenotypic correlations with wool traits in Corriedale sheep<\/b><br><br>Scoured wool colour determines the potenti al colour range and dye-ability of wool tops. Improvement of scoured wool colour of fleeces from Corriedale sheep through multi-trait breeding programmes is currently not possible due to limited existing information on its heritability and genetic correlations with other fleece traits. The present study aimed to determine genetic parameters of scoured wool colour traits, with emphasis on their genetic and phenotypic correlations with objectively and visually assessed fleece traits. Data from 1181 hoggets of two experimental flocks were used. Heritabilities were estimated for scoured wool yellowness (Y-Z) and scoured wool brightness (Y), greasy fleece weight (GFW) and clean fleece weight (CFW), wool yield (WY), mean fibre diameter (MFD), coefficient of variation of fibre diameter (CVD), staple length (SL), staple strength (SS) and a range of visual wool traits (greasy wool colour, wool character, handle, staple formation, dust penetration and cross-linking). Trivariate statistical models under a Bayesian setting were used, including the environmental effects of flock, sex, litter size and climatic variables (accumulated precipitation, maximum temperature and relative humidity) and the animal genetic effect, Y-Z was moderately heritable (0.22 ± 0.06) and should respond to selection. Its genetic correlations with GFW, CFW and SL were significant, positive (unfavourable) and moderate (0.43 ± 0.15, 0.56 ± 0.14, and 0.44 ± 0.14, respectively). There was a positive (favourable) association between MFD and Y-Z (0.25 ± 0.17). The genetic correlation between Y-Z and greasy wool colour was 0.70 ± 0.13, although this trait showed a low heritability value (0.13 ± 0.04). Wool brightness showed low heritability (0.10 ± 0.04) and only appeared to be genetically correlated with SL and wool character (0.56 ± 0.17 and 0.39 ± 0.22, respectively). Phenotypic correlations of Y-Z and Y with wool production and quality traits were all less than 0.20 in size. Overall, this study reports new genetic parameters for wool colour and their associations with fleece traits in Corriedale sheep. Our findings suggest that it could be possible to select against wool yellowness, with some caution, given the possible detrimental effects on economically important wool traits, especially fleece weight and staple length. More investigation is warranted. Indirect selection via greasy wool colour is an alternative approach, but the responses would be slower. Easy and cheap measurement should be balanced against the size of the response from direct selection versus indirect selection. © 2022 Elsevier B.V.","<b>Assessment of Real-Time Quaking-Induced Conversion (RT-QuIC) Assay, Immunohistochemistry and ELISA for Detection of Chronic Wasting Disease under Field Conditions in White-Tailed Deer: A Bayesian Approach<\/b><br><br>Chronic wasting disease (CWD) is a transmissible prion disease of the cervidae family. ELISA and IHC tests performed postmortem on the medial retropharyngeal lymph nodes (RPLN) or obex are considered diagnostic gold standards for prion detection. However, differences in CWD transmission, stage of infection, pathogenesis, and strain can limit performance. To overcome these uncertainties, we used Bayesian statistics to assess the accuracy of RT-QuIC, an increasingly used prion amplification assay, to diagnose CWD on tonsil (TLN), parotid (PLN) and submandibular lymph nodes (SMLN), and ELISA/IHC on RPLN of white-tailed deer (WTD) sampled from Minnesota. Dichotomous RT-QuIC and ELISA/IHC results from wild (n = 61) and captive (n = 46) WTD were analyzed with two-dependent-test, one-population models. RT-QuIC performed on TLN and SMLN of the wild WTD population had similar sensitivity (median range (MR): 92.2–95.1) to ELISA/IHC on RPLN (MR: 91.1–92.3). Slightly lower (4–7%) sensitivity estimates were obtained from farmed animal and PLN models. RT-QuIC specificity estimates were high (MR: 94.5–98.5%) and similar to ELISA/IHC estimates (MR: 95.7–97.6%) in all models. This study offers new insights on RT-QuIC and ELISA/IHC performance at the population level and under field conditions, an important step in CWD diagnosis and management. © 2022 by the authors. Licensee MDPI, Basel, Switzerland.","<b>Colonization rather than fragmentation explains the geographical distribution and diversification of treefrogs endemic to Brazilian shield sky islands<\/b><br><br>Aim: Geographical patterns of montane biodiversity worldwide are related to biotic and abiotic factors, such as historical climate dynamics and species dispersal capabilities, which affect the biota from population to community levels. Understanding of processes related to population diversification in extra-Andean Neotropical mountains remains largely unknown. Here, we tested how colonization of new areas as opposed to fragmentation of geographical ranges influenced the distribution and diversification of frogs from Brazilian mountains. Location: Espinhaço Range, the largest extra-Andean mountain range in South America. Taxon: Ledge treefrog Bokermannohyla saxicola. Methods: We used multilocus DNA and morphometric data throughout the species' distribution to delimit lineages, infer phylogenetic relationships and estimate divergence times. We used ecological niche modelling (ENM) and approximate Bayesian computation (ABC) to reconstruct changes of geographical distribution and population sizes for testing alternative hypotheses of diversification concerning fragmentation and colonization. Results: We found four evolutionary significant lineages that diverged from the Pliocene to the Early Pleistocene. All lineages were validated by DNA and morphometric data by independent and joint analyses. ENM showed that climatic fluctuations might have influenced the species' distribution, whereas ABC model selection further supports associated demographic changes. Taken together, a Pleistocene jump-dispersal scenario best explains the diversification of the Bokermannohyla saxicola lineages. Main conclusions: Endemism in the Espinhaço Range is usually explained by ancient diversification, associated with long-term isolation of climatic stable areas following climate-driven habitat fragmentation. Our results challenge this general view, indicating recent diversification, habitat expansion and colonization of new areas as important processes explaining the current distribution and genetic diversity of B. saxicola. © 2022 John Wiley & Sons Ltd.","<b>Performance of bovine genital campylobacteriosis diagnostic tests in bulls from Uruguay: a Bayesian latent class model approach<\/b><br><br>The sensitivity (Se) and specificity (Sp) of three diagnostic tests for the detection of Campylobacter fetus venerealis (Cfv) using field samples were estimated using a Bayesian latent class model (BLCM), accounting for the absence of a gold standard. The tests included in this study were direct immunofluorescence antibody test (IFAT), polymerase chain reaction (PCR), and real-time PCR (RT-PCR). Twelve farms from two different populations were selected and bull prepuce samples were collected. The IFAT was performed according to the OIE Manual. The conventional PCR was performed as multiplex, targeting the gene nahE for C. fetus species identification and insertion element ISCfe1 for Cfv identification. The RT-PCR was performed as uniplex: one targeting the gene nahE for C. fetus and the other targeting the insertion ISCfe1 (ISC2) for Cfv. Results from the BLCM showed a median Se of 11.7% (Bayesian credibility interval (BCI): 1.93–29.79%), 53.7% (BCI: 23.1–95.0%), and 36.1% (BCI: 14.5–71.7%) for IFAT, PCR, and RT-PCR respectively. The Sp were 94.5% (BCI: 90.1–97.9%), 97.0% (BCI: 92.9–99.3%), and 98.4% (BCI: 95.3–99.7%) for IFAT, PCR, and RT-PCR respectively. The correlation between PCR and RT-PCR was positive and low in samples from both sampled population (0.63% vs 8.47%). These results suggest that diagnostic sensitivity of the studied tests is lower using field samples than using pure Cfv strains. © 2021, The Author(s), under exclusive licence to Springer Nature B.V.","<b>Not the same: phylogenetic relationships and ecological niche comparisons between two different forms of Aglaoctenus lagotis from Argentina and Uruguay<\/b><br><br>Species are the fundamental category and the key to formulate conservation efforts. DNA and ecological niche modeling have become valuable tools for species delimitation. Wolf spiders include few web-living species, such as Aglaoctenus lagotis (Holmberg, 1876), a priority species for conservation in Uruguay. Behavioral and body coloration patterns of this species have allowed us to distinguish two groups (forms I and II). Here, we combine information from gene trees and multispecies coalescent analyses on mitochondrial (cox1, 12S, 16S + L1 + nad1) and nuclear (intron tif5A) DNA sequences, as well as from ecological niches comparisons, in order to clarify their taxonomic identity. We worked with localities in Uruguay and Argentina, including sympatric and allopatric areas. Gene trees were inferred with Maximum Likelihood, Bayesian, and statistical parsimony analyses. Molecular species delimitation analyses were conducted, and the species tree and divergence times were co-estimated. Characterization and comparison of the climatic requirements of both forms throughout annual and sexual periods were analyzed. Species delimitation and species tree analyses recovered three main lineages (Form I, Form IIa, and Form IIb). Form I is restricted to Uruguay and is closely related and sympatric with Form IIa. Form IIb is located in Argentina and in the Uruguayan west coast, generating a sympatric area of the three forms. Regarding to the sexual climatic niche, the three main lineages differ and do not overlap. Our results support the existence of more than one lineage within what is nowadays Aglaoctenus lagotis. Possible evolving processes explaining this scenario and the conservation consequences are discussed. © 2022, Gesellschaft für Biologische Systematik.","<b>Whole genomic comparative analysis of Streptococcus pneumoniae serotype 1 isolates causing invasive and non-invasive infections among children under 5 years in Casablanca, Morocco<\/b><br><br>Background: Streptococcus pneumoniae serotype 1 remains a leading cause of invasive pneumococcal diseases, even in countries with PCV-10/PCV-13 vaccine implementation. The main objective of this study, which is part of the Pneumococcal African Genome project (PAGe), was to determine the phylogenetic relationships of serotype 1 isolates recovered from children patients in Casablanca (Morocco), compared to these from other African countries; and to investigate the contribution of accessory genes and recombination events to the genetic diversity of this serotype. Results: The genome average size of the six-pneumococcus serotype 1 from Casablanca was 2,227,119 bp, and the average content of coding sequences was 2113, ranging from 2041 to 2161. Pangenome analysis of the 80 genomes used in this study revealed 1685 core genes and 1805 accessory genes. The phylogenetic tree based on core genes and the hierarchical bayesian clustering analysis revealed five sublineages with a phylogeographic structure by country. The Moroccan strains cluster in two different lineages, the five invasive strains clusters altogether in a divergent clade distantly related to the non-invasive strain, that cluster with all the serotype 1 genomes from Africa. Conclusions: The whole genome sequencing provides increased resolution analysis of the highly virulent serotype 1 in Casablanca, Morocco. Our results are concordant with previous works, showing that the phylogeography of S. pneumoniae serotype 1 is structured by country, and despite the small size (six isolates) of the Moroccan sample, our analysis shows the genetic cohesion of the Moroccan invasive isolates. © 2021, The Author(s).","<b>Ultraconserved Elements Improve the Resolution of Difficult Nodes within the Rapid Radiation of Neotropical Sigmodontine Rodents (Cricetidae: Sigmodontinae)<\/b><br><br>Sigmodontine rodents (Cricetidae, Sigmodontinae) represent the second largest muroid subfamily and the most species-rich group of New World mammals, encompassing above 410 living species and ca. 87 genera. Even with advances on the clarification of sigmodontine phylogenetic relationships that have been made recently, the phylogenetic relationships among the 12 main groups of genera (i.e., tribes) remain poorly resolved, in particular among those forming the large clade Oryzomyalia. This pattern has been interpreted as consequence of a rapid radiation upon the group entrance into South America. Here, we attempted to resolve phylogenetic relationships within Sigmodontinae using target capture and high-throughput sequencing of ultraconserved elements (UCEs). We enriched and sequenced UCEs for 56 individuals and collected data from four already available genomes. Analyses of distinct data sets, based on the capture of 4634 loci, resulted in a highly resolved phylogeny consistent across different methods. Coalescent species-tree-based approaches, concatenated matrices, and Bayesian analyses recovered similar topologies that were congruent at the resolution of difficult nodes. We recovered good support for the intertribal relationships within Oryzomyalia; for instance, the tribe Oryzomyini appears as the sister taxa of the remaining oryzomyalid tribes. The estimates of divergence times agree with the results of previous studies. We inferred the crown age of the sigmodontine rodents at the end of the Middle Miocene, while the main lineages of Oryzomyalia appear to have radiated in a short interval during the Late Miocene. Thus, the collection of a genomic-scale data set with a wide taxonomic sampling provided resolution for the first time of the relationships among the main lineages of Sigmodontinae. We expect the phylogeny presented here will become the backbone for future systematic and evolutionary studies of the group. © 2021 The Author(s). Published by Oxford University Press, on behalf of the Society of Systematic Biologists. All rights reserved.","<b>Characterization of “candidatus ehrlichia pampeana” in haemaphysalis juxtakochi ticks and gray brocket deer (Mazama gouazoubira) from uruguay<\/b><br><br>Human ehrlichiosis are scantily documented in Uruguay. The aim of this study was to investigate the presence of Ehrlichia spp. in Haemaphysalis juxtakochi and in a gray brocket deer (Mazama gouazoubira) from Uruguay. The presence of Ehrlichia DNA was investigated in free-living H. juxtakochi in five localities of southeast and northeast Uruguay, as well as blood, spleen, and ticks retrieved from a M. gouazoubira. Ehrlichia spp. DNA was detected in six out of 99 tick pools from vegetation, in the spleen of M. gouazoubira, and in one out of five pools of ticks feeding on this cervid. Bayesian inference analyses for three loci (16S rRNA, dsb, and groEL) revealed the presence of a new rickettsial organism, named herein as “Candidatus Ehrlichia pampeana”. This new detected Ehrlichia is phylogenetically related to those found in ticks from Asia, as well as Ehrlichia ewingii from USA and Cameroon. Although the potential pathogenicity of “Ca. E. pampeana” for humans is currently unknown, some eco-epidemiological factors may be relevant to its possible pathogenic role, namely: (i) the phylogenetic closeness with the zoonotic agent E. ewingii, (ii) the evidence of H. juxtakochi parasitizing humans, and (iii) the importance of cervids as reservoirs for zoonotic Ehrlichia spp. The molecular detection of “Ca. E. pampeana” represents the third Ehrlichia genotype described in Uruguay. © 2021 by the authors. Licensee MDPI, Basel, Switzerland.","<b>An Hepatozoon americanum-like protozoan in crab-eating (Cerdocyon thous) and grey pampean (Lycalopex gymnocercus) foxes from Uruguay<\/b><br><br>In South America, apicomplexan parasites of the genus Hepatozoon have been sporadically detected in mammals. Previous studies in wild canids from Brazil and Argentina demonstrated infections by species genetically related to Hepatozoon americanum. The aim of the present work was to detect the presence of Hepatozoon in road-killed foxes encountered in Uruguayan highways. Blood samples from 45 crab-eating (Cerdocyon thous) and 32 grey pampean (Lycalopex gymnocercus) foxes were analyzed by PCR for Hepatozoon 18S rRNA gene. Eight foxes (10.4%) were found to be infected with an H. americanum-like protozoan, an Hepatozoon closely related to H. americanum. Bayesian and maximum-likelihood phylogenetic analyses revealed that the sequences obtained in this study cluster with H. americanum from the United States, and with an H. americanum-like species from dog and foxes from Brazil and Argentina. In the Unites States, H. americanum causes severe disease in dogs. In addition to this, an increasing habitat overlap between dogs and foxes makes the presence of H. americanum-like protozoan in foxes acquires veterinary relevance. This work represents the first report of L. gymnocercus infected with an H. americanum-like protozoan, and of wild canids infected with Hepatozoon in Uruguay. © 2021, The Author(s), under exclusive licence to Springer-Verlag GmbH Germany, part of Springer Nature.","<b>Genomic footprints of Quaternary colonization and population expansion in the Patagonian-Fuegian region rules out a separate southern refugium in Tierra del Fuego<\/b><br><br>Aim: During the glacial phases, the Andean slopes of southern South America were covered by ice sheets, causing the isolation of many species in glacial refugia. This work aimed to test the number and putative location of glacial refugia in southern South America and establish the sources, timing and direction of post-glacial colonization events. Location: Southern South America, Patagonia. Taxon: Olive mouse (Abrothrix olivacea). Methods: Using exome capture, we genotyped 7339 SNPs in 172 specimens of A. olivacea collected at 21 localities distributed across the steppe, grasslands, and forests in the Patagonian-Fuegian region and seven specimens from two other species of Abrothrix. Based on the effective migration rate, principal component analysis, and admixture composition, we first study the population structure of A. olivacea. The source and direction of colonization events were inferred using the directionality index and standard population genetic analyses. To study the role of the glacial phases, we performed a bayesian demographic analysis to infer divergences times and ancestral population sizes. Results: Population genomic analyses based on SNPs of more than 7000 exons identified three Patagonian genetic clusters: Valdivian forests, continental Magellan forests, and a combination of continental open biomes and Tierra del Fuego. The directionality index supports a single source in northwestern forests, in line with a Valdivian forest refugium, from which the post-glacial colonization may have started. The effective diversity rate, the onset of population expansions and divergence times also support a Valdivian forest refugium and rule out a separate southern refugium in Tierra del Fuego. Main conclusions: Our results support a coastal refugium located in or near the previously proposed Valdivian refugium and reject in situ differentiation in Tierra del Fuego. More in general, our demographical analysis reveals the role of the Last Glacial Maximum in affecting the population dynamics (i.e., contraction and expansion) of A. olivacea and in fostering the geographical diversification of populations. © 2021 John Wiley & Sons Ltd","<b>Automatic calibration and uncertainty quantification in waves dynamical downscaling<\/b><br><br>Automatic calibration of numerical models and systematic quantification of the uncertainties associated with their use is not yet a widespread practice in coastal engineering. This paper proposes the use of a Bayesian algorithm for the automatic calibration and uncertainty quantification of a wave model, in the framework of the dynamical downscaling of off-shore waves to a nearshore project site, where a set of wave measurements is available. A spectral error was defined and used for the definition of the likelihood function used by the algorithm; this allows for improvements in terms of the wave spectra and not only in terms of a restricted set of wave parameter. In addition to the calibration of the model parameters, the methodology also addresses errors coming from boundary conditions; to this end it distinguishes between different wave systems and a set of parameters are defined for the correction of each wave system, reducing the errors introduced into the model by the boundary conditions. A case study in the South Atlantic coast showed the ability of the methodology to calibrate the model, resulting in simulations that properly fit the available measurement, while also providing an estimation of the uncertainties associated with the model results that can be straightforwardly used for probabilistic analysis in the coastal environment. © 2021 Elsevier B.V.","<b>Sediment grain size determines microplastic exposure landscapes for sandy beach macroinfauna<\/b><br><br>Despite the global occurrence of microplastic contamination on sandy beaches, evidence of microplastic distribution within beaches remains contradictory. When conflicting evidence is used to inform sampling surveys, it increases uncertainty in resulting data. Moreover, it hampers spatially explicit risk characterization of microplastic pollution to intertidal fauna. We aimed to guide sampling designs for microplastic monitoring on beaches, and to quantify macroinfauna exposure to microplastics. Microplastic abundance, quantified between 5 mm–66 μm, lacked a significant zonation across the top sediment layer of sub-terrestrial, upper and lower midlittoral, and swash zones at two sites with varying anthropogenic influence on a microtidal dissipative beach in Uruguay. Microplastic abundance decreased exponentially with increasing grain size, as revealed by Bayesian Poisson regression, although the decrease was less steep compared to prior knowledge regarding sediment – plastic interactions obtained for large (millimeter-sized) industrial pellets. Significant differences in microplastic contamination between the two sites with varying anthropogenic influence likely related to their proximity to a freshwater canal. Corresponding field measurements of body burdens of fibers and irregular particles were significantly lower for the polychaete Euzonus (Thoracophelia) furcifera, despite its preference for finer sediments with higher microplastic loads, compared to the isopods Excirolana braziliensis and Excirolana armata. Results provide critical insights toward representative sampling of microplastics within beach sites. Specifically, we caution against sampling limited to the drift line, and instead recommend: 1) reporting beach morphodynamic characteristics; 2) using clearly defined, ecologically-informed zonation schemes; and 3) accounting for sediment grain size as a covariate to normalize among reported contamination levels. The results contribute valuable baseline data toward realistic exposure landscapes relative to the sediment grain size preferences of macroinfauna, needed to inform laboratory experiments. © 2021 The Author(s)","<b>Emergence and spread of a b.1.1.28-derived p.6 lineage with q675h and q677h spike mutations in uruguay<\/b><br><br>Uruguay controlled the viral dissemination during the first nine months of the SARS-CoV-2 pandemic. Unfortunately, towards the end of 2020, the number of daily new cases exponentially in-creased. Herein, we analyzed the country-wide genetic diversity of SARS-CoV-2 between November 2020 and April 2021. We identified that the most prevalent viral variant during the first epidemic wave in Uruguay (December 2020–February 2021) was a B.1.1.28 sublineage carrying Spike mutations Q675H + Q677H, now designated as P.6, followed by lineages P.2 and P.7. P.6 probably arose around November 2020, in Montevideo, Uruguay’s capital department, and rapidly spread to other depart-ments, with evidence of further local transmission clusters; it also spread sporadically to the USA and Spain. The more efficient dissemination of lineage P.6 with respect to P.2 and P.7 and the presence of mutations (Q675H and Q677H) in the proximity of the key cleavage site at the S1/S2 boundary suggest that P.6 may be more transmissible than other lineages co-circulating in Uruguay. Although P.6 was replaced by the variant of concern (VOC) P.1 as the predominant lineage in Uruguay since April 2021, the monitoring of the concurrent emergence of Q675H + Q677H in VOCs should be of worldwide interest. © 2021 by the authors. Licensee MDPI, Basel, Switzerland.","<b>Quaternary climatic fluctuations influence the demographic history of two species of sky-island endemic amphibians in the Neotropics<\/b><br><br>We evaluated the role of Quaternary climatic fluctuations on the demographic history and population structure of amphibian species endemic to the ‘campo rupestre’ in the Neotropics, evaluating their distributional shifts, demographic changes, and lineage formation from the end of Pleistocene to present. We chose two anurans endemic to the high-elevation ‘campo rupestre’ in the Espinhaço Range (ER) in northeastern and southeastern Brazil (Bokermannohyla alvarengai and Bokermannohyla oxente), as models to test the role of Quaternary climatic fluctuations over their distribution range in this region. We collected tissue samples throughout their distribution range and used statistical phylogeography to examine processes of divergence and population demography. We generated spatial–temporal reconstructions using Bayesian inference in a coalescent framework in combination with hind-cast projections of species distribution models (SDMs). We also used the results and literature information to test alternative diversification scenarios via approximate Bayesian computation (ABC). Our results show that Quaternary climatic fluctuations influenced the geographic ranges of both species showing population expansion during the last glacial maximum (LGM) and range contraction during interglacial periods, as inferred from selected ABC models and from past projections of SDMs. We recovered Pleistocene diversification for both species occuring in distinctly unique periods for each taxon. An older and range-restricted lineage was recovered in a geographically isolated geological massif, deserving conservation and further taxonomic study. The diversification and distribution of these amphibian species endemic to the Neotropical ‘campo rupestre’ were influenced by Quaternary climatic fluctuations. The expansion of cold adapted species restricted to higher elevations during glacial periods and their concomitant retraction during interglacial periods may have been crucial for producing patterns of species richness and endemism along elevation gradients in tropical and subtropical domains. Such processes may influence the evolution of the biota distributed in heterogeneous landscapes with varied topography. © 2021 Elsevier Inc.","<b>Building the GLENCOE Platform -Grasslands LENding eConomic and ecOsystems sErvices<\/b><br><br>To feed the rising population whilst also preserving ecosystem functions, creative solutions are needed for the ecological intensification of natural grassland-based livestock systems. In Uruguay, natural grasslands are the main nutritional resource for livestock production. In these ecosystems, cattle and sheep graze together all the year round, and grasslands are frequently heavily grazed. Considerable research has been generated concerning grassland management, but there is still no knowledge about the impact of decision rules that supports management actions on long-term ecosystem functioning, at the system level. To meet this deficit, a participatory working group of farmers, researchers, and consultants have developed the GLENCOE platform. This platform is a large-scale facility, supported by INIA-Uruguay, designed to answer the following question: How to intensify the grazing management to improve the sustainability of livestock systems based on natural grasslands? To build the platform three steps were followed: (I) definition of the research problem using a problem tree analysis; (ii) conceptualization of the platform and the design of the grazing systems to be evaluated; and, (iii) spatial allocation of the grazing systems according to the variability of soil, slopes, and seasonal dynamic of vegetation indexes. These criteria were considered across farmlets that were equivalent in the initial stage, allowing causal inferences for the systems trajectories on productive and environmental traits. The platform is composed of three independent farmlets of 50 ha each, where multiparous Hereford cows and Merinos wethers co-graze under three grazing management systems. Each farmlet is managed according to different spatio-temporal decisions of the specific management of vegetation communities, grazing methods, and the stockpile of forage that is allowed by the number of the existing paddocks. Farmlet-1; comprises less decisions (2 paddocks), Farmlet-2; intermediate (8 paddocks), and Farmlet-3; high level of decisions (32 paddocks). This innovative platform will be used as a participatory and interdisciplinary space for research and co-learning of management on processes that can only be observed in long-term evaluations, and at farmlet scale. We expect that this new approach will contribute to the developement and implemention of sustainable grazing management systems in Uruguay. © Copyright © 2021 Devincenzi, Jaurena, Durante, Savian, Ciappesoni, Navajas, Ciganda, Lattanzi and Paruelo.","<b>Population structure and gene flow of Geoffroy's cat (Leopardus geoffroyi) in the Uruguayan Savanna ecoregion<\/b><br><br>Felids are among the species most threatened by habitat fragmentation resulting from land-use change. In the Uruguayan Savanna ecoregion, about 30% of natural habitats have been lost, large felids have been eradicated from most of the region, and the impact of anthropogenic threats over the smaller species that remain is unknown. To develop management strategies, it is important to enhance knowledge about species population structure and landscape connectivity, particularly when land-use change will continue and intensify in the next years. In this study, we evaluate the population structure and gene flow of Geoffroy's cat in the Uruguayan Savanna ecoregion. We generated a matrix of 11 microsatellite loci for 70 individuals. Based on Bayesian approaches we found that within the Uruguayan Savanna, Geoffroy's cat shows high levels of genetic variability and no population structure. However, we observed genetic differences between individuals from the Uruguayan Savanna and those from the contiguous ecoregion, the Argentinian Humid Pampa. Four first-generation migrants from Humid Pampa were identified in the Uruguayan Savanna, suggesting a stronger gene flow in the west-east direction. We detected a past bottleneck followed by a subsequent recovery in Geoffroy's cat populations in both ecoregions. These results lay the groundwork to understand the population dynamics and conservation status of Geoffroy's cat in the Uruguayan Savanna ecoregion, and provide baseline data to establish population monitoring. © 2021 The Author(s).","<b>Water-quality data imputation with a high percentage of missing values: A machine learning approach<\/b><br><br>The monitoring of surface-water quality followed by water-quality modeling and analysis are essential for generating effective strategies in surface-water-resource management. However, worldwide, particularly in developing countries, water-quality studies are limited due to the lack of a complete and reliable dataset of surface-water-quality variables. In this context, several statistical and machine-learning models were assessed for imputing water-quality data at six monitoring stations located in the Santa Lucía Chico river (Uruguay), a mixed lotic and lentic river system. The challenge of this study is represented by the high percentage of missing data (between 50% and 70%) and the high temporal and spatial variability that characterizes the water-quality variables. The competing algorithms implement univariate and multivariate imputation methods (inverse distance weighting (IDW), Random Forest Regressor (RFR), Ridge (R), Bayesian Ridge (BR), AdaBoost (AB), Hubber Regressor (HR), Support Vector Regressor (SVR) and K-nearest neighbors Regressor (KNNR)). According to the results, more than 76% of the imputation outcomes are considered ‘satisfactory’ (NSE > 0.45). The imputation performance shows better results at the monitoring stations located inside the reservoir than those positioned along the mainstream. IDW was the model with the best imputation results, followed by RFR, HR and SVR. The approach proposed in this study is expected to aid water-resource researchers and managers in augmenting water-quality datasets and overcoming the missing data issue to increase the number of future studies related to the water-quality matter. © 2021 by the authors. Licensee MDPI, Basel, Switzerland.","<b>A new early warning indicator of tree species crashes from effective intraspecific interactions in tropical forests<\/b><br><br>The vulnerability of species richness to several factors like, climate change, habitat fragmentation, resource exploitation, etc., poses a challenge to conservation biologists and agencies working to sustain the ecosystem services. Hence, there is a clear need for early warning indicators of species loss generated from empirical data. The tree community of the long-term 50-hectare plot on Barro Colorado Island (BCI), Panama, is one of the most intensively studied in the world. This plot was established in 1981 and fully censused in 1982 then every 5 years from 1985 through 2015. This extensive dataset reveals that some tree species suffered steep population declines. Here we propose an early warning indicator of such tree population crashes and test it against the BCI dataset. The spatial covariance matrices, Cij, of the 20 most abundant tree species in BCI allow us to compute, via MaxEnt, the effective interaction matrices, Jij, among these species for the eight censuses available from 1982 to 2015. For each species i and each census c, the absolute value of the intraspecific competition coefficients Jii(c) are much larger than those of the interspecific interaction coefficients Jij(c) with i ≠ j. We show that this result can be derived from a similar empirical relationship observed for the matrices Cii(c). Our main finding is that for those tree species that suffered steep population declines (of at least 50%), across the eight tree censuses, the drop of Jii is always steeper and occurs before the drop of the corresponding species abundance Ni. Indeed, such sharp declines in Jii occur between 5 and 15 years in advance than comparable declines for Ni, and thus they serve as early warnings of impending population busts. Furthermore, this drop of Jii is linked to the anomalous variance, which is a known early warning of incoming catastrophic shifts. © 2021 The Author(s)","<b>The inner ear anatomy of glyptodonts and pampatheres (Xenarthra, Cingulata): Functional and phylogenetic implications<\/b><br><br>In recent years, the increase in studies on the inner ear anatomy of xenarthrans provided new insights regarding some locomotor and phylogenetic aspects. These works have begun to include fossil specimens, although so far, only two extinct representatives of the suborder Folivora (ground sloths) have been analyzed. In the present study, we present the first reconstruction and description of the bony labyrinth of five extinct specimens of the suborder Cingulata, the glyptodonts Glyptodon, Doedicurus, Panochthus, and Pseudoplohophorus, as well as the pampathere Holmesina, in order to analyze their morphology and evaluate functional and phylogenetic aspects. The anatomical and morphometric comparison shows that the inner ear of glyptodonts is largely similar to that of modern armadillos, particularly sharing many characteristics with Chlamyphorus. While the inner ear of Holmesina is similar to that of glyptodonts, it presents some characteristics that make it more similar to extant armadillos. Regarding some functional aspects inferred from the anatomy of the semicircular canals, glyptodonts would have had low levels of agility, although similar to most extant armadillos, and low head angular velocity. On the other hand, Holmesina would have had an agility level similar to that of the giant armadillo Priodontes, although it would also present low head angular velocity as seen in most extant armadillos. Finally, preliminary parsimony and Bayesian phylogenetic inference analyses based on a reduced set of inner ear characters showed interesting results, with glyptodonts and Holmesina recovered as closely related to Chlamyphorus. These results closely resemble the recent topologies obtained using ancient DNA and markedly contrast with the topologies obtained using more traditional morphological characters, highlighting the importance of inner ear data for the inference of phylogenetic relationships in cases where DNA preservation in fossil taxa is unaccessible. © 2021 Elsevier Ltd","<b>A method for predicting species trajectories tested with trees in barro colorado tropical forest<\/b><br><br>The ability to predict changes in the abundances of the species in ecological communities is essential for sustainable management, biodiversity conservation, and community restoration. We propose a framework to predict such changes. We test our method, which uses the linear Lotka-Volterra equations (LLVE) as well as other empirical predictors (linear least squares regression, quadratic extrapolation, simple exponential smoothing), against the measured abundances of trees from the long-term 50-ha plot on Barro Colorado Island (BCI), along eight censuses. To obtain the parameters of the LLVE -the intrinsic growth rate r and the carrying capacity K of each species and the interspecific interaction matrix A- we first estimate A through the Maximum Entropy (MaxEnt) method. Next, using A as input, we fit r and K. Then, feeding the LLVE with these parameters, we obtain predicted species trajectories along censuses. Since for this particular community the interspecific interaction coefficients are much smaller than the intraspecific ones, keeping only intraspecific competition is enough to predict the evolution of the abundances of several tree species, i.e. the LLVE reduce to a set of uncoupled logistic equations. However, this simplification is not a requirement of the method. We define P-values to establish when the predicted trajectory for a species is statistically significant; this is crucial in determining the set of species over which a particular predictor can be meaningfully applied. To illustrate a possible application of the method, we present our predictions for the abundances of tree species for the currently underway BCI 2020 census, which provide warnings regarding species that are likely to experience important population loss. © 2021 Elsevier B.V.","<b>Markets as ecological networks: Inferring interactions and identifying communities<\/b><br><br>Financial markets are paradigmatic examples of complex systems and have been compared to ecological networks in which different species (firms) interact and co-evolve. A central object governing species dynamics in ecology is the community matrix, whose elements are closely related to pairwise interspecific interaction coefficients. Using this ecological analogy we propose a method, based on the Maximum Entropy (MaxEnt) principle, that allows us to infer candidates for an economic community matrix from time series data of market values. To assess the usefulness of this picture, we construct community matrices for a set of companies belonging to the Fortune 500 list and perform a community analysis on the resultant networks. This analysis shows these networks to strongly reflect the known industry groupings of the firms. We conclude therefore that our community matrices capture non-Trivial information about the interaction of firms, not immediately apparent from the covariance of market values. We anticipate our approach being useful in elucidating further aspects of market structure, as well as forming the basis of forecasting market dynamics. © 2021 The Author(s). Published by Oxford University Press.","<b>Riparian forest subsidises the biomass of fish in a recently formed subtropical reservoir<\/b><br><br>Riparian forest may represent a key subsidy to aquatic food webs. While most research has been done in running waters, the origin of resources subsidising biomass and the role of riparian coverage in food webs remains largely unstudied in reservoirs. This research evaluated the role of forest riparian zones as food web subsidies in a recently formed reservoir of Ijuí River, Brazil. The diet of fish and the origin of carbon and nitrogen fuelling fish biomass was compared between littoral habitats with riparian forests and of open canopy. Sampling for stable isotopes of fish and basal resources and the application of bayesian mixing models showed that most of the fish production originates from terrestrial carbon in this recently formed reservoir. Moreover, an increasing terrestrial support to the biomass was found towards the riparian forest areas (subsidising c.a. 70% vs. around 57% of fish biomass in littoral areas of riparian forest and open canopy, respectively).This study remarked the role of riparian forest areas as energetic subsidises to aquatic ecosystems. Management actions in newly formed reservoirs should include preserving riparian forests to avoid losing natural ecosystem subsidies. © 2020 John Wiley & Sons A/S. Published by John Wiley & Sons Ltd","<b>Adaptive evolution of β-globin gene in subterranean in South America octodontid rodents<\/b><br><br>The convergent evolution of subterranean rodents is an excellent model to study how natural selection operates and the genetic bases of these adaptations, but the study on the different taxa has been very uneven and still insufficient. In the octodontoid caviomorph rodent superfamily there are two independent lineages where they have recently evolved into totally underground lifestyles: the genera Ctenomys (tuco-tucos) and Spalacopus (coruro). The underground habitat is characterized by an hypoxic and hypercapnic atmosphere, thus gas exchange is one of the most important challenges for these animals. The invasion of the underground niche could have modified the selective regimes of proteins involved in the respiration and transport of O2 of these rodents, positively selecting mutations of higher affinity for O2. Here we examine the sequence variation in the beta globin gene in these two lineages, within a robust phylogenetic context. Using different approaches (classical and Bayesian maximum likelihood (PAML/Datamonkey) and alternatives methods (TreeSAAP)) we found at least three sites with evidence of positive selection in underground lineages, especially the basal branch that leads to the Octodontidae family and the branch that leads to the coruro, suggesting some adaptive changes to the underground life. We also found a convergence with another underground rodent, which cannot be identified by the above methods. © 2020","<b>Inclusion of bioclimatic variables in genetic evaluations of dairy cattle<\/b><br><br>Objective: Considering the importance of dairy farming and the negative effects of heat stress, more tolerant genotypes need to be identified. The objective of this study was to investigate the effect of heat stress via temperature-humidity index (THI) and diurnal temperature variation (DTV) in the genetic evaluations for daily milk yield of Holstein dairy cattle, using random regression models. Methods: The data comprised 94,549 test-day records of 11,294 first parity Holstein cows from Brazil, collected from 1997 to 2013, and bioclimatic data (THI and DTV) from 18 weather stations. Least square linear regression models were used to determine the THI and DTV thresholds for milk yield losses caused by heat stress. In addition to the standard model (SM, without bioclimatic variables), THI and DTV were combined in various ways and tested for different days, totaling 41 models. Results: The THI and DTV thresholds for milk yield losses was THI = 74 (-0.106 kg/d/THI) and DTV = 13 (-0.045 kg/d/DTV). The model that included THI and DTV as fixed effects, considering the two-day average, presented better fit (-2logL, Akaike information criterion, and Bayesian information criterion). The estimated breeding values (EBVs) and the reliabilities of the EBVs improved when using this model. Conclusion: Sires are re-ranking when heat stress indicators are included in the model. Genetic evaluation using the mean of two days of THI and DTV as fixed effect, improved EBVs and EBVs reliability. © 2021 by Animal Bioscience.","<b>Applying Bayesian Networks to help Physicians Diagnose Respiratory Diseases in the context of COVID-19 Pandemic<\/b><br><br>The differential diagnosis of respiratory diseases is usually a challenge for medical specialists in the first line of care, increased under the current COVID-19 pandemic. A Clinical Decision Support System-CDSS-is being developed using Bayesian Networks-BNs-to help physicians diagnose respiratory diseases, including those related to COVID-19. Network structure has been elicited from expert physicians, and network parameters (diseases prevalence, symptoms, findings, and lab results conditional probabilities) were extracted from relevant bibliography or currently standard global information sources. The CDSS is being tested using case studies taken from real situations, provided and validated by physicians. The resulting system demonstrates the suitability and flexibility of BNs for diagnosis support and healthcare training. © 2021 IEEE.","<b>Molecular characterization of novel Ehrlichia genotypes in Ixodes auritulus from Uruguay<\/b><br><br>Ehrlichia are small intracellular Gram-negative bacteria transmitted by ticks. These microorganisms cause ehrlichiosis, a complex of life-threatening emerging zoonoses and diseases of global veterinary relevance. The aim of this study was to investigate the presence of Ehrlichia in free-living Ixodes auritulus collected in Uruguay. Ticks were collected from vegetation in five localities from the southeast and northeast of the country between 2014 and 2017. Detection of Ehrlichia DNA was performed in pools of adults or nymphs grouped according to the collection site and date. A total of 1,548 I. auritulus ticks were collected in four of the five locations sampled. Fragments of three loci (16S rRNA, dsb and groEL) were obtained by PCR, and phylogenies inferred using Bayesian inference analysis for each gene independently. DNA of Ehrlichia spp. was found in 15 out of 42 tick pools. Based on the topology of the phylogenetic trees, our sequences represent two novel genotypes for the genus named as Ehrlichia sp. Serrana and Ehrlichia sp. Laguna Negra. Both genotypes were closely related to Ehrlichia sp. Magellanica, a species detected in Ixodes uriae and Magellanic penguins. Considering that all stages of I. auritulus and I. uriae are parasites of birds, their phylogenetic relationships, and common eco–epidemiological profiles, it is reasonable to state that these genotypes of Ehrlichia spp. may represent a natural group likely associated with birds. Our results constitute the first characterization of Ehrlichia spp. in Uruguay. Future studies on birds reported as hosts for I. auritulus are needed to further understand the epidemiological cycles of both Ehrlichia genotypes in the country. Finally, I. auritulus does not feed on humans, so the two Ehrlichia species reported herein might have no implications in human health. © 2021 The Author(s)","<b>Development of a Population Pharmacokinetic Model for Cyclosporine from Therapeutic Drug Monitoring Data<\/b><br><br>Aim. To develop a population pharmacokinetic model for Uruguayan patients under treatment with cyclosporine (CsA) that can be applied to TDM. Patients and Methods. A total of 53 patients under treatment with CsA were included. 37 patients with at least one pharmacokinetic profile described with four samples were considered for model building, while the remaining 16 were considered for the assessments of predictive performances. Pharmacokinetic parameter estimation was performed using a nonlinear mixed effect modelling implemented in the Monolix® software (version 2019R1, Lixoft, France); meanwhile, simulations were performed in R v.3.6.0 with the mlxR package. Results. A two-compartment model with a first-order disposition model including lag time was used as a structural model. The final model was internally validated using prediction corrected visual predictive check (pcVPC) and other graphical diagnostics. A total of 621 CsA steady-state concentrations were analyzed for model development. Population estimates for the absorption constant (ka) and lag time were 0.523 h-1 and 0.512 h, respectively; apparent clearance (CL/F) was 30.3 L/h (relative standard error RSE±8.25%) with an interindividual variability of 39.8% and interoccasion variability of 38.0%; meanwhile, apparent clearance of distribution (Q/F) was 17.0 L/h (RSE±12.1%) with and interindividual variability of 53.2%. The covariate analysis identified creatinine clearance (ClCrea) as an individual factor influencing the Cl of CsA. The predictive capacity of the population model was demonstrated to be effective since predictions made for new patients were accurate for C1 and C2 (MPPEs below 50%). Bayesian forecasting improved significantly in the second and third occasions. Conclusion. A population pharmacokinetic model was developed to reasonably estimate the individual cyclosporine clearance for patients. Hence, it can be utilized to individualize CsA doses for prompt and adequate achievement of target blood concentrations of CsA. © 2021 Martín Umpiérrez et al.","<b>Selection for test-day milk yield and thermotolerance in brazilian holstein cattle<\/b><br><br>Intense selection for milk yield has increased environmental sensitivity in animals, and currently, heat stress is an expensive problem in dairy farming. The objectives were to identify the best model for characterizing environmental sensitivity in Holstein cattle, using the test-day milk yield (TDMY) combined with the temperature–humidity index (THI), and identify sires genetically superior for heat-stress (HS) tolerance and milk yield, through random regression. The data comprised 94,549 TDMYs of 11,294 first-parity Holstein cows in Brazil, collected from 1997 to 2013. The yield data were fitted to Legendre orthogonal polynomials, linear splines and the Wilmink function. The THI (the average of two days before the dairy control) was used as an environmental gradient. An animal model that fitted production using a Legendre polynomials of quartic order for the days in milk and quadratic equations for the THI presented a better quality of fit (Akaike’s information criterion (AIC) and Bayesian information criterion (BIC)). The Spearman correlation coefficient of greatest impact was 0.54, between the top 1% for TDMY and top 1% for HS. Only 9% of the sires showed plasticity and an aptitude for joint selection. Thus, despite the small population fraction allowed for joint selection, sufficient genetic variability for selecting more resilient sires was found, which promoted concomitant genetic gains in milk yield and thermotolerance. © 2021 by the authors. Licensee MDPI, Basel, Switzerland.","<b>Validation of a multiplex PCR assay to detect Babesia spp. and Anaplasma marginale in cattle in Uruguay in the absence of a gold standard test<\/b><br><br>Detection of bovine Babesia spp. and Anaplasma marginale is based on the reading of Giemsa-stained blood or organ smears, which can have low sensitivity. Our aim was to improve the detection of bovine Babesia spp. and A. marginale by validating a multiplex PCR (mPCR). We used 466 samples of blood and/or organs of animals with signs and presumptive autopsy findings of babesiosis or anaplasmosis. The primers in our mPCR amplified the rap-1a gene region of Babesia bovis and B. bigemina, and the msp-5 region of A. marginale. We used a Bayesian model with a non-informative priori distribution for the prevalence estimate and informative priori distribution for estimation of sensitivity and specificity. The sensitivity and specificity for smear detection of Babesia spp. were 68.6% and 99.1%, and for A. marginale 85.6% and 98.8%, respectively. Sensitivity and specificity for mPCR detection for Babesia spp. were 94.2% and 97.1%, and for A. marginale 95.2% and 92.7%, respectively. Our mPCR had good accuracy in detecting Babesia spp. and A. marginale, and would be a reliable test for veterinarians to choose the correct treatment for each agent. © 2020 The Author(s).","<b>Genetic parameters for production traits, resistance and resilience to Nematode parasites under different worm burden challenges in Corriedale sheep<\/b><br><br>Characterized as the most important gastrointestinal parasite (GIP) in the world, Haemonchus contortus is also the most predominant and pathogenic genus in Uruguay, causing high morbidity and mortality rates in sheep flocks. For approximately 26 years resistance to GIP has been included in the National Genetic Evaluation (NGE) of Corriedale and Merino breeds, using Faecal Egg Count (FEC) as the selection criterion. The aim of the present study was to estimate genetic parameters for FEC, post-weaning Body Weight (BW), Packed Cell Volume (PCV), FAMACHA© and Body Condition Score (BCS), to determine the possibility to include these traits in the NGE of GIP resistance and resilience, as complementary selection criteria and to investigate the sensitivity of breeding values to different environmental worm burden challenges. Data were collected on 19,510 lambs from 2000 to 2009 in 24 Corriedale studs that participated in the NGE of sheep in Uruguay. (Co)variances components were estimated using a multi-trait animal model performing a Bayesian analysis. Posterior means estimated for heritability (h2) were of low to moderate magnitude (between 0.10 and 0.33) for the traits analysed. Genetic correlations (rg) were not significantly different from zero for BW- Loge (FEC + 100), BW-FAMACHA©, BW-PCV, Loge (FEC + 100)-PCV, BCS-FAMACHA©, moderate favourable rg for Loge (FEC + 100)-FAMACHA and PCV-BCS (0.55, 0.41) and high favourable rg for BW-BCS (0.66). These results indicate the possibility to complement selection programs incorporating BW, Loge (FEC + 100), FAMACHA©, PCV and BCS in order to select resistant and resilient animal to GIP. Furthermore, genetic parameters for BW and Loge (FEC + 100) to different environmental worm burden challenge was investigated. Two contemporary groups of low worm environment (Loge (FEC + 100) below 6.40) and a high worm environment (Loge (FEC + 100) above 6.75) were considered. In this model, h2 estimated for BW and Loge (FEC + 100) were found to be not statistically different between low and high environmental worm burden, furthermore, high correlations between traits at both environments were also presented. © 2020 Elsevier B.V.","<b>Evidence of increasing diversification of emerging Severe Acute Respiratory Syndrome Coronavirus 2 strains<\/b><br><br>On 30th January 2020, an outbreak of atypical pneumonia caused by a novel betacoronavirus, named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), was declared a public health emergency of international concern by the World Health Organization. For this reason, a detailed evolutionary analysis of SARS-CoV-2 strains currently circulating in different geographic regions of the world was performed. A compositional analysis as well as a Bayesian coalescent analysis of complete genome sequences of SARS-CoV-2 strains recently isolated in Europe, North America, South America, and Asia was performed. The results of these studies revealed a diversification of SARS-CoV-2 strains in three different genetic clades. Co-circulation of different clades in different countries, as well as different genetic lineages within different clades were observed. The time of the most recent common ancestor was established to be around 1st November 2019. A mean rate of evolution of 6.57 × 10−4 substitutions per site per year was found. A significant migration rate per genetic lineage per year from Europe to South America was also observed. The results of these studies revealed an increasing diversification of SARS-CoV-2 strains. High evolutionary rates and fast population growth characterizes the population dynamics of SARS-CoV-2 strains. © 2020 Wiley Periodicals LLC","<b>Is it possible to accurately estimate lactation curve parameters in extensive beef production systems?<\/b><br><br>Objective: The objective was to characterize the lactation curve applying 3 different models using multiparous grazing beef cows. Materials and Methods: Milk production data from 99 British crossbred multiparous cows grazing native pastures were analyzed. Lactation was assessed 15 d postpartum and then monthly until weaning (180 d postpartum) using a milking machine after an oxytocin injection. Total milk production and lactation curve were characterized using Wood (WD) and Wilmink (WIL) models, and both were compared with spline functions. Comparison was made applying adjusted coefficient of determination (R2adj) and MSE. Results and Discussion: Cubic splines with 5 equally spaced knots (CS5) presented the best adjustment (lowest Akaike information criterion and Bayesian information criterion). The R2adj values were 0.55, 0.54, and 0.53 (the greater the better) and MSE values were 2.54, 2.59, and 2.47 (the lower the better) for WD, WIL, and CS5, respectively. Estimated milk production for the lactation period was 1,277, 1,255, and 1,195 kg for WD, WIL, and CS5, respectively. Milk peak was predicted to happen at 32, 25, and 36 d postpartum, with a production of 8.74, 8.21, and 8.40 kg for WD, WIL, and CS5, respectively. No differences were evident in the lactation curves (95% CI). Implications and Applications: The proposed method and frequency used to assess grazing beef cattle milk production accurately estimate the lactation curve. The Wood model, used worldwide, was a precise estimator of the lactation curve, which in our case was verified applying splines. These results provide key information to calculate grazing beef cow requirements. © 2020 American Registry of Professional Animal Scientists","<b>An evolutionary insight into emerging Ebolavirus strains isolated in Africa<\/b><br><br>Abstract: On July 19, 2019, the World Health Organization declared the current Ebolavirus (EBOV) outbreak in Congo Democratic Republic (COD) a public health emergency of international concern. To address the potential threat of EBOV evolution outpacing antibody treatment and vaccine efforts, a detailed evolutionary analysis of EBOV strains circulating in different African countries was performed. Genome composition of EBOV strains was studied using multivariate statistical analysis. To investigate the patterns of evolution of EBOV strains, a Bayesian Markov Chain Monte Carlo approach was used. Two different genetic lineages, with a distinct genome composition gave rise to the recent EBOV outbreaks in central and western Africa. Strains isolated in COD in 2018 fall into two different genetic clusters, according to their geographical location of isolation. Different amino acid substitutions among strains from these two clusters have been found, particularly in NP, GP, and L proteins. Significant differences in codon and amino acid usage among clusters were found. Strains isolated in COD in 2018 belong to two distinct genetic clusters, with distinct codon and amino acid usage. Geographical diversity plays an important role in shaping the molecular evolution of EBOV populations. © 2019 Wiley Periodicals, Inc.","<b>Low additive genetic variation in a trait under selection in domesticated rice<\/b><br><br>Quantitative traits are important targets of both natural and artificial selection. The genetic architecture of these traits and its change during the adaptive process is thus of fundamental interest. The fate of the additive effects of variants underlying a trait receives particular attention because they constitute the genetic variation component that is transferred from parents to offspring and thus governs the response to selection. While estimation of this component of phenotypic variation is challenging, the increasing availability of dense molecular markers puts it within reach. Inbred plant species offer an additional advantage because phenotypes of genetically identical individuals can be measured in replicate. This makes it possible to estimate marker effects separately from the contribution of the genetic background not captured by genotyped loci. We focused on root growth in domesticated rice, Oryza sativa, under normal and aluminum (Al) stress conditions, a trait under recent selection because it correlates with survival under drought. A dense single nucleotide polymorphism (SNP) map is available for all accessions studied. Taking advantage of this map and a set of Bayesian models, we assessed additive marker effects. While total genetic variation accounted for a large proportion of phenotypic variance, marker effects contributed little information, particularly in the Al-tolerant tropical japonica population of rice. We were unable to identify any loci associated with root growth in this population. Models estimating the aggregate effects of all measured genotypes likewise produced low estimates of marker heritability and were unable to predict total genetic values accurately. Our results support the long-standing conjecture that additive genetic variation is depleted in traits under selection. We further provide evidence that this depletion is due to the prevalence of low-frequency alleles that underlie the trait. Copyright © 2020 Karavolias et al.","<b>Single-step genomic evaluations from theory to practice: using snp chips and sequence data in blupf90<\/b><br><br>Single-step genomic evaluation became a standard procedure in livestock breeding, and the main reason is the ability to combine all pedigree, phenotypes, and genotypes available into one single evaluation, without the need of post-analysis processing. Therefore, the incorporation of data on genotyped and non-genotyped animals in this method is straightforward. Since 2009, two main implementations of single-step were proposed. One is called single-step genomic best linear unbiased prediction (ssGBLUP) and uses single nucleotide polymorphism (SNP) to construct the genomic relationship matrix; the other is the single-step Bayesian regression (ssBR), which is a marker effect model. Under the same assumptions, both models are equivalent. In this review, we focus solely on ssGBLUP. The implementation of ssGBLUP into the BLUPF90 software suite was done in 2009, and since then, several changes were made to make ssGBLUP flexible to any model, number of traits, number of phenotypes, and number of genotyped animals. Single-step GBLUP from the BLUPF90 software suite has been used for genomic evaluations worldwide. In this review, we will show theoretical developments and numerical examples of ssGBLUP using SNP data from regular chips to sequence data. © 2020 by the authors. Licensee MDPI, Basel, Switzerland.","<b>Historical origins and zoonotic potential of avian influenza virus H9N2 in Tunisia revealed by Bayesian analysis and molecular characterization<\/b><br><br>During 2009–2012, several outbreaks of avian influenza virus H9N2 were reported in Tunisian poultry. The circulating strains carried in their hemagglutinins the human-like marker 226L, which is known to be important for avian-to-human viral transmission. To investigate the origins and zoonotic potential of the Tunisian H9N2 viruses, five new isolates were identified during 2012–2016 and their whole genomes were sequenced. Bayesian-based phylogeny showed that the HA, NA, M and NP segments belong to the G1-like lineage. The PB1, PB2, PA and NS segments appeared to have undergone multiple intersubtype reassortments and to be only distantly related to all of the Eurasian lineages (G1-like, Y280-like and Korean-like). The spatiotemporal dynamic of virus spread revealed that the H9N2 virus was transferred to Tunisia from the UAE through Asian and European pathways. As indicated by Bayesian analysis of host traits, ducks and terrestrial birds played an important role in virus transmission to Tunisia. The subtype phylodynamics showed that the history of the PB1 and PB2 segments was marked by intersubtype reassortments with H4N6, H10N4 and H2N2 subtypes. Most of these transitions between locations, hosts and subtypes were statistically supported (BF > 3) and not influenced by sampling bias. Evidence of genetic evolution was observed in the predicted amino acid sequences of the viral proteins of recent Tunisian H9N2 viruses, which were characterized by the acquisition of new mutations involved in virus adaptation to avian and mammalian hosts and amantadine resistance. This study is the first comprehensive analysis of the evolutionary history of Tunisian H9N2 viruses and highlights the zoonotic risk associated with their circulation in poultry, indicating the need for continuous surveillance of their molecular evolution. © 2020, Springer-Verlag GmbH Austria, part of Springer Nature.","<b>Multilocus phylogeny of Paratelmatobiinae (Anura: Leptodactylidae) reveals strong spatial structure and previously unknown diversity in the Atlantic Forest hotspot<\/b><br><br>The Brazilian Atlantic Forest harbors high levels of anuran diversity and endemism, including several taxa restricted to small geographic ranges. Here, we provide a multilocus phylogeny for Paratelmatobiinae, a leptodactylid subfamily composed of small-ranged species distributed in the Brazilian Atlantic Forest and in the campo rupestre ecosystem. We performed Bayesian inference and maximum likelihood analyses using three mitochondrial and five nuclear markers, and a matrix comprising a broad taxonomic sampling. We then delimitated independently evolving lineages within the group. We recovered Paratelmatobiinae and each of its four genera as monophyletic and robustly supported. Five putatively new species included in our analyses were unambiguously supported in the phylogenetic trees and delimitation analyses. We also recovered other deeply divergent and geographically structured lineages within the four genera of Paratelmatobiinae. Our estimation of divergence times indicates that diversification in the subfamily began in the Eocene and continued until the Pleistocene. We discuss possible scenarios of diversification for the four genera of Paratelmatobiinae, and outline the implications of our findings for taxonomy and conservation. © 2020 Elsevier Inc.","<b>Predicting cyanobacterial biovolume from water temperature and conductivity using a Bayesian compound Poisson-Gamma model<\/b><br><br>Eutrophication and climate change scenarios engender the need to develop good predictive models for harmful cyanobacterial blooms (CyanoHABs). Nevertheless, modeling cyanobacterial biomass is a challenging task due to strongly skewed distributions that include many absences as well as extreme values (dense blooms). Most modeling approaches alter the natural distribution of the data by splitting them into zeros (absences) and positive values, assuming that different processes underlie these two components. Our objectives were (1) to develop a probabilistic model relating cyanobacterial biovolume to environmental variables in the Río de la Plata Estuary (35°S, 56°W, n = 205 observations) considering all biovolume values (zeros and positive biomass) as part of the same process; and (2) to use the model to predict cyanobacterial biovolume under different risk level scenarios using water temperature and conductivity as explanatory variables. We developed a compound Poisson-Gamma (CPG) regression model, an approach that has not previously been used for modeling phytoplankton biovolume, within a Bayesian hierarchical framework. Posterior predictive checks showed that the fitted model had a good overall fit to the observed cyanobacterial biovolume and to more specific features of the data, such as the proportion of samples crossing three threshold risk levels (0.2, 1 and 2 mm³ L−1) at different water temperatures and conductivities. The CPG model highlights the strong control of cyanobacterial biovolume by nonlinear and interactive effects of water temperature and conductivity. The highest probability of crossing the three biovolume levels occurred at 22.2 °C and at the lowest observed conductivity (∼0.1 mS cm−1). Cross-validation of the fitted model using out-of-sample observations (n = 72) showed the model's potential to be used in situ, as it enabled prediction of cyanobacterial biomass based on two readily measured variables (temperature and conductivity), making it an interesting tool for early alert systems and management strategies. Furthermore, this novel application demonstrates the potential of the Bayesian CPG approach for predicting cyanobacterial dynamics in response to environmental change. © 2020 Elsevier Ltd","<b>Gametes Simulator: A multilocus genotype simulator to analyze genetic structure in outbreeding diploid species<\/b><br><br>Bulk sampling and subsequent DNA fingerprinting are applied to obtain estimated allelic frequency data and unveil population structure for outbreeding species. We developed ‘Gametes Simulator’, a routine to simulate gametes from allelic frequencies of unlinked loci, which enables the analysis of population structure through Bayesian analysis. Based on the allelic frequencies in the populations, the software simulates the alleles per population in the right proportions and assigns one allele per marker to each gamete following Mendel’s laws that govern gametogenesis. © 2020, Brazilian Society of Plant Breeding. All rights reserved.","<b>Using approximate Bayesian inference for a \"steps and turns\" continuous-time random walk observed at regular time intervals<\/b><br><br>The study of animal movement is challenging because movement is a process modulated by many factors acting at different spatial and temporal scales. In order to describe and analyse animal movement, several models have been proposed which differ primarily in the temporal conceptualization, namely continuous and discrete time formulations. Naturally, animal movement occurs in continuous time but we tend to observe it at fixed time intervals. To account for the temporal mismatch between observations and movement decisions, we used a state-space model where movement decisions (steps and turns) are made in continuous time. That is, at any time there is a nonzero probability of making a change in movement direction. The movement process is then observed at regular time intervals. As the likelihood function of this state-space model turned out to be intractable yet simulating data is straightforward, we conduct inference using different variations of Approximate Bayesian Computation (ABC). We explore the applicability of this approach as a function of the discrepancy between the temporal scale of the observations and that of the movement process in a simulation study. Simulation results suggest that the model parameters can be recovered if the observation time scale is moderately close to the average time between changes in movement direction. Good estimates were obtained when the scale of observation was up to five times that of the scale of changes in direction. We demonstrate the application of this model to a trajectory of a sheep that was reconstructed in high resolution using information from magnetometer and GPS devices. The state-space model used here allowed us to connect the scales of the observations and movement decisions in an intuitive and easy to interpret way. Our findings underscore the idea that the time scale at which animal movement decisions are made needs to be considered when designing data collection protocols. In principle, ABC methods allow to make inferences about movement processes defined in continuous time but in terms of easily interpreted steps and turns. © 2020 Ruiz-Suarez et al.","<b>Fraud Detection in Electric Power Distribution: An Approach That Maximizes the Economic Return<\/b><br><br>The detection of non-technical losses (NTL) is a very important economic issue for power utilities. Diverse machine learning strategies have been proposed to support electric power companies tackling this problem. Methods performance is often measured using standard cost-insensitive metrics, such as the accuracy, true positive ratio, AUC, or F1. In contrast, we propose to design a NTL detection solution that maximizes the effective economic return. To that end, both the income recovered and the inspection cost are considered. Furthermore, the proposed framework can be used to design the infrastructure of the division in charge of performing customers inspections. Then, assisting not only short term decisions, e.g., which customer should be inspected first, but also the elaboration of long term strategies, e.g., planning of NTL company budget. The problem is formulated in a Bayesian risk framework. Experimental validation is presented using a large dataset of real users from the Uruguayan utility. The results obtained show that the proposed method can boost companies profit and provide a highly efficient and realistic countermeasure to NTL. Moreover, the proposed pipeline is general and can be easily adapted to other practical problems. © 1969-2012 IEEE.","<b>Association between results of diagnostic tests for bovine tuberculosis and Johne's disease in cattle<\/b><br><br>Background Bovine tuberculosis (bTB) diagnosis is impaired by numerous factors including cross-reactivity with Mycobacterium avium subspecies paratuberculosis, which causes Johne's disease (JD). In addition, the effect of repeated bTB-intradermal testing on the performance of JD diagnostic tests is not fully understood. This study aimed to evaluate the impact of repeated bTB-intradermal tests under field conditions in Spain on the JD serological status of cattle. Methods bTB-positive herds (n=264) from Castilla-y-Leon region were selected and matched with officially tuberculosis-free control herds. The association between JD and bTB status at the herd level was assessed using conditional logistic regression and, in herds with both JD-positive and bTB-positive animals, a Bayesian hierarchical mixed-effect model was used for individual-level analysis. Results A significantly higher risk of being JD positive (OR: 1.48; 95 per cent CI: 1.01 to 2.15) was found for bTB-positive herds compared with controls. Individual results indicated that cattle tested more than three times per year, within the last 90 days and more than 12 months were more likely to be JD positive. A skin test-related boost in antibody response could be the cause of an apparent increase of the sensitivity of the JD-absorbed ELISA. Conclusion The results demonstrate the interaction between bTB repeated testing and JD individual and herd-level results and this improved knowledge will facilitate the design of more effective control programmes in herds coinfected with two of the most important endemic diseases affecting cattle in Spain. © 2019 British Veterinary Association. No commercial re-use. See rights and permissions. Published by BMJ.","<b>The discovery of Neotropical Lepidosira (Collembola, Entomobryidae) and its systematic position<\/b><br><br>We herein present the first reliable record of Lepidosira from Neotropical Region. Lepidosira neotropicalis sp. n. from Brazil is described and illustrated in detail, including its complete mitochondrial genome. We perform a Bayesian phylogenetic analysis to place the new species within the Entomobryidae, and at the same time to test previous contrasting hypotheses on Lepidosira position within the Entomobryinae versus Seirinae for the first time. Phylogenetic analyses were based on one mitochondrial and two nuclear genes, Cytochrome Oxidase subunit I, 18S ribosomal RNA and 28S ribosomal RNA, respectively. Lepidosira neotropicalis sp. n. resembles L. sundana Yoshii and Suhardjono and L. nigropunctata (Nguyen) in dorsal chaetotaxy of abdominal segments I and II, but differs from all other species by the combination of head (dorsally and ventrally) and dorsal trunk chaetotaxy, plus empodial complex morphology. Our phylogenetic analyses support the placement of Lepidosira within Entomobryinae, as the sister group of Lepidocyrtoides. Overall, our revision enables a more objective diagnosis to Lepidosira and suggests that the genus is in need of a full revision due to its variable morphology, and lack of data needed to evaluate its monophyly. Finally, we provide an identification key for Neotropical genera of Entomobryinae. © 2019 Royal Swedish Academy of Sciences","<b>APPARENT BIAS: WHAT DOES ATTITUDE POLARIZATION SHOW?<\/b><br><br>Many, though not all, experiments have found that exposing groups of subjects who disagree to the same evidence may cause their initial attitudes to strengthen and move further apart, or polarize. Some have concluded that findings of attitude polarization show that people process information in a biased manner so as to support their initial views. We argue that, on the contrary, polarization is often what we should expect to find in an unbiased Bayesian population, in the context of experiments that find polarization. © (2019) by the Economics Department of the University of Pennsylvania and the Osaka University Institute of Social and Economic Research Association","<b>The impact of anthropogenic food subsidies on a generalist seabird during nestling growth<\/b><br><br>Anthropogenic food subsidies, such as refuse, are an important driver of animal population changes and gulls heavily forage on this food source. Foraging on refuse during the rearing period could affect the acquisition of resources with potential demographic consequences. Using conventional diet analysis and stable isotopes of δ13C and δ15N of blood of Kelp Gull (Larus dominicanus) nestlings, we studied the variation of the chick growth in response to foraging on refuse on a reproductive colony in the Rio de la Plata Estuary in Uruguay. Using Bayesian mixing models on isotopic data, we estimated the proportion and variation of natural food and refuse in the diet of nestlings. Then, we modelled the variation between the mean posterior densities of the food sources and their standard deviation with the nestling morphometric measurements of different sizes. We found that refuse was gradually delivered to Kelp Gull nestlings during the chick rearing period. Additionally, variation of refuse incorporated into nestling tissues increased with nestlings' size. We propose that parents use more isotopically unique food sources during the nestling growth thereby increasing isotopic diversity. This study highlights the need to improve the current waste management system, which is being reviewed in Uruguay. We believe that decision makers should consider the results of this study, which show that refuse is directly impacting coastal ecosystems through mechanisms poorly explored by the environmental sciences. © 2019 Elsevier B.V.","<b>Conservation planning in Uruguay based on small felids (Leopardus spp.) as umbrella species<\/b><br><br>Felids biogeographic patterns in Uruguay are poorly understood despite the importance of fringe populations of species in conservation. We determined suitability areas, analyzed species richness and assessed the role of Uruguayan protected areas in the conservation of small felids: margay, pampas cat, and Geoffroy’s cat. We applied umbrella-species approach to select conservation areas based on habitat suitability for these species. Habitat suitability models were developed with Maxent, 218 presence records and 21 bioclimatic and topographic variables. Maps show margay in the east, pampas cat in the entire country except the north and Geoffroy’s cat in the entire country. Highest richness was found in the east and the lowest in the north. Protected areas cover 1% of each species distribution range and although most of them provide suitable habitat for at least one species, none of them has adequate surface alone to sustain viable populations. This is the first study that addresses Uruguayan felid distribution based on objective methodologies and a broad set of presence records. Based on information generated, we selected six zones to focus efforts in research and biodiversity conservation. The approach proved useful to guide management plans and conservation decisions and can be extrapolated to other species and countries. © 2019, © 2019 Informa UK Limited, trading as Taylor & Francis Group.","<b>Species diversity and distribution of the genus Colpomenia (Scytosiphonaceae, Phaeophyceae) along the coast of China<\/b><br><br>The marine brown algal genus Colpomenia has a worldwide distribution, with five species reported in Korea and Japan. However, no studies to date attempted to identify the number of species and geographical distribution of Colpomenia along Chinese coast. To fill the biodiversity knowledge gap, we analyzed 63 mitochondrial cox3 and 62 mitochondrial atp6 sequences of Colpomenia specimens collected from 30 localities along the Chinese coast. Maximum likelihood and Bayesian inference trees suggest the presence of at least three Colpomenia species (i.e., C. peregrina, C. claytoniae, and C. sinuosa) in China. C. peregrina and C. claytoniae are documented for the first time. C. sinuosa was only found in the South China Sea and its distribution didn’t overlap with that of C. peregrina which was found in the Yellow-Bohai Sea and the East China Sea. C. claytoniae appears to be confined to three isolated islands in the East and the South China Sea, where it occurs in sympatry with, respectively, C. peregrina and C. sinuosa. Future study can focus on comparing eco-physiological differences of Colpomenia species in response to environmental variables and exploring possible genetic hybridization / introgression at inter-specific contact zones. © 2019 The Korean Society of Phycology.","<b>Coalescent-based species delimitation in the sand lizards of the Liolaemus wiegmannii complex (Squamata: Liolaemidae)<\/b><br><br>Coalescent-based algorithms coupled with the access to genome-wide data have become powerful tools for assessing questions on recent or rapid diversification, as well as delineating species boundaries in the absence of reciprocal monophyly. In southern South America, the diversification of Liolaemus lizards during the Pleistocene is well documented and has been attributed to the climatic changes that characterized this recent period of time. Past climatic changes had harsh effects at extreme latitudes, including Patagonia, but habitat changes at intermediate latitudes of South America have also been recorded, including expansion of sand fields over northern Patagonia and Pampas). In this work, we apply a coalescent-based approach to study the diversification of the Liolaemus wiegmannii species complex, a morphologically conservative clade that inhabits sandy soils across northwest and south-central Argentina, and the south shores of Uruguay. Using four standard sequence markers (mitochondrial DNA and three nuclear loci) along with ddRADseq data we inferred species limits and a time-calibrated species tree for the L. wiegmannii complex in order to evaluate the influence of Quaternary sand expansion/retraction cycles on diversification. We also evaluated the evolutionary independence of the recently described L. gardeli and inferred its phylogenetic position relative to L. wiegmannii. We find strong evidence for six allopatric candidate species within L. wiegmannii, which diversified during the Pleistocene. The Great Patagonian Glaciation (∼1 million years before present) likely split the species complex into two main groups: one composed of lineages associated with sub-Andean sedimentary formations, and the other mostly related to sand fields in the Pampas and northern Patagonia. We hypothesize that early speciation within L. wiegmannii was influenced by the expansion of sand dunes throughout central Argentina and Pampas. Finally, L. gardeli is supported as a distinct lineage nested within the L. wiegmannii complex. © 2019 Elsevier Inc.","<b>Modeling the Accuracy of Two in-vitro Bovine Tuberculosis Tests Using a Bayesian Approach<\/b><br><br>Accuracy of new or alternative diagnostic tests is typically estimated in relation to a well-standardized reference test referred to as a gold standard. However, for bovine tuberculosis (bTB), a chronic disease of cattle, affecting animal and public health, no reliable gold standard is available. In this context, latent-class models implemented using a Bayesian approach can help to assess the accuracy of diagnostic tests incorporating previous knowledge on test performance and disease prevalence. In Uruguay, bTB-prevalence has increased in the past decades partially because of the limited accuracy of the diagnostic strategy in place, based on intradermal testing (caudal fold test, CFT, for screening and comparative cervical test, CCT, for confirmation) and slaughter of reactors. Here, we evaluated the performance of two alternative bTB-diagnostic tools, the interferon-gamma assay, IGRA, and the enzyme-linked immunosorbent assay (ELISA), which had never been used in Uruguay in the absence of a gold standard. In order to do so animals from two heavily infected dairy herds and tested with CFT-CCT were also analyzed with the IGRA using two antigens (study 1) and the ELISA (study 2). The accuracy of the IGRA and ELISA was assessed fitting two latent-class models: a two test-one population model (LCA-a) based on the analysis of CFT/CFT-CCT test results and one in-vitro test (IGRA/ELISA), and a one test-one population model (LCA-b) using the IGRA or ELISA information in which the prevalence was modeled using information from the skin tests. Posterior estimates for model LCA-a suggested that IGRA was as sensitive (75–78%) as the CFT and more sensitive than the serial use of CFT-CCT. Its specificity (90–96%) was superior to the one for the CFT and equivalent to the use of CFT-CCT. Estimates from LCA-b models consistently yielded lower posterior Se estimates for the IGRA but similar results for its Sp. Estimates for the Se (52% 95%PPI:44.41-71.28) and the Sp (92% 95%PPI:78.63–98.76) of the ELISA were however similar regardless of the model used. These results suggest that the incorporation of IGRA for detection of bTB in highly infected herds could be a useful tool to improve the sensitivity of the bTB-control in Uruguay. © Copyright © 2019 Picasso-Risso, Perez, Gil, Nunez, Salaberry, Suanes and Alvarez.","<b>Millennium-length precipitation reconstruction over south-eastern asia: A pseudo-proxy approach<\/b><br><br>Quantifying precipitation variability beyond the instrumental period is essential for putting current and future fluctuations into long-Term perspective and providing a test bed for evaluating climate simulations. For south-eastern Asia such quantifications are scarce and millennium-long attempts are still missing. In this study we take a pseudo-proxy approach to evaluate the potential for generating summer precipitation reconstructions over south-eastern Asia during the past millennium. The ability of a series of novel Bayesian approaches to generate reconstructions at either annual or decadal resolutions and under diverse scenarios of pseudo-proxy records noise is analysed and compared to the classic analogue method. We find that for all the algorithms and resolutions a high density of pseudo-proxy information is a necessary but not sufficient condition for a successful reconstruction. Among the selected algorithms, the Bayesian techniques perform generally better than the analogue method, the difference in abilities being highest over the semi-Arid areas and in the decadal-resolution framework. The superiority of the Bayesian schemes indicates that directly modelling the space and time precipitation field variability is more appropriate than just relying on a pool of observational-based analogues in which certain precipitation regimes might be absent. Using a pseudo-proxy network with locations and noise levels similar to the ones found in the real world, we conclude that performing a millennium-long precipitation reconstruction over south-eastern Asia is feasible as the Bayesian schemes provide skilful results over most of the target area. © Author(s) 2019. This work is distributed under the Creative Commons Attribution 4.0 License.","<b>Detection of presumed genes encoding beta-lactamases by sequence based screening of metagenomes derived from Antarctic microbial mats<\/b><br><br>Analysis of environmental samples for bacterial antibiotic resistance genes may have different objectives and analysis strategies. In some cases, the purpose was to study diversity and evolution of genes that could be grouped within a mechanism of antibiotic resistance. Different protocols have been designed for detection and confirmation that a functional gene was found. In this study, we present a sequence-based screening of candidate genes encoding beta-lactamases in 14 metagenomes of Antarctic microbial mats. The samples were obtained from different sites, representing diverse biogeographic regions of maritime and continental Antarctica. A protocol was designed based on generation of Hidden Markov Models from the four beta-lactamase classes by Ambler classification, using sequences from the Comprehensive Antibiotic Resistance Database (CARD). The models were used as queries for metagenome analysis and recovered contigs were subsequently annotated using RAST. According to our analysis, 14 metagenomes analyzed contain A, B and C beta-lactamase genes. Class D genes, however, were identified in 11 metagenomes. The most abundant was class C (46.8%), followed by classes B (35.5%), A (14.2%) and D (3.5%). A considerable number of sequences formed clusters which included, in some cases, contigs from different metagenomes. These assemblies are clearly separated from reference clusters, previously identified using CARD beta-lactamase sequences. While bacterial antibiotic resistance is a major challenge of public health worldwide, our results suggest that environmental diversity of beta-lactamase genes is higher than that currently reported, although this should be complemented with gene function analysis. [Figure not available: see fulltext.]. © 2019, Higher Education Press and Springer-Verlag GmbH Germany, part of Springer Nature.","<b>Probability of Rhipicephalus microplus introduction into farms by cattle movement using a Bayesian Belief Network<\/b><br><br>Attempts to eliminate Rhipicephalus microplus from Uruguay have been unsuccessful, and, currently, the country is divided into two areas: a tick-free area and a tick-infested area. In the tick-infested area, different farms face different situations. Some farms are in regions where, due to environmental conditions or a lack of infrastructure, it is difficult to eliminate R. microplus, and the only option is to control it. In contrast, other farms can attempt complete removal. Before deciding whether a farmer should attempt to eliminate R. microplus, the probability of reintroduction must be evaluated. The objective of this study was to develop a probabilistic model based on a Bayesian Belief Network (BBN) to assess the likelihood of a farm becoming infested with R. microplus via the introduction of tick-infested cattle. Only the tick-infested area was considered in the development of this model. Nine variables related to environmental conditions and biosecurity measures, with a focus on cattle movement, were considered. Three different sources of data were used to populate the BBN model: data from the literature; a representative national survey from 2016; and a survey developed to identify biosecurity practices on farms. Model sensitivity and specificity were assessed, and an overall accuracy of 92% was obtained. The model was applied to 33 farms located in the tick-infested area. For one farm, the probability of introduction of R. microplus was 1%; for three farms, the probability was between 21% and 34%; for seven farms, it was between 66% and 76%; and for 22 farms, the probability was greater than 83%. This model was useful for estimating the probability of the introduction of R. microplus into farms, making it possible to assess the impact that the evaluated biosecurity measures have on the probability of introduction and, thus, guiding more objective decision making about the control or elimination of R. microplus from farms. © 2019 Elsevier GmbH","<b>Phylogeny, Macroevolutionary Trends and Historical Biogeography of Sloths: Insights From a Bayesian Morphological Clock Analysis<\/b><br><br>Sloths, like other xenarthrans, are an extremely interesting group of mammals that, after a long history of evolution and diversification in South America, became established on islands in the Caribbean and later reached North America during the Great American Biotic Interchange. In all three regions, they were part of the impressive Pleistocene megafauna. Most taxa became extinct and only two small, distantly related tree-dwelling genera survived. Here, we incorporate several recently described genera of sloths into an assembled morphological data supermatrix and apply Bayesian inference, using phylogenetic and morphological clock methods, to 64 sloth genera. Thus, we investigate the evolution of the group in terms of the timing of divergence of different lineages and their diversity, morphological disparity and biogeographical history. The phylogeny obtained supports the existence of the commonly recognized clades for the group. Our results provide divergence time estimates for the major clades within Folivora that could not be dated with molecular methods. Lineage diversity shows an early increase, reaching a peak in the Early Miocene followed by a major drop at the end of the Santacrucian (Early Miocene). A second peak in the Late Miocene was also followed by a major drop at the end of the Huayquerian (Late Miocene). Both events show differential impact at the family level. After that, a slight Plio-Pleistocene decline was observed before the marked drop with the extinction at the end of the Pleistocene. Phenotypic evolutionary rates were high during the early history of the clade, mainly associated with Mylodontidae, but rapidly decreased to lower values around 25 Ma, whereas Megalonychidae had lower rates at the beginning followed by a steady increase, peaking during the Late Miocene and the Pliocene. Morphological disparity showed a similar trend, with an early increase, followed by a slowly increasing phase through the Late Oligocene and Early Miocene, and ending with another increase beginning at the middle of the Miocene. Biogeographic analysis showed southern South America as the most probable area of origin of the clade and the main region in which the early diversification events took place. Both Megatheriinae and Nothrotheriinae basal nodes were strongly correlated with Andean uplift events, whereas the early history of Mylodontidae is closely associated with southern South America and also shows an early occupation of the northern regions. Within Megalonychidae, our results show Choloepus as a descendant of an island dispersing ancestor and a probable re-ingression to South America by a clade that originated in Central or North America. © The Author(s) 2018.","<b>Cryptic diversity in Brazilian endemic monkey frogs (Hylidae, Phyllomedusinae, Pithecopus) revealed by multispecies coalescent and integrative approaches<\/b><br><br>Even though Brazil is the world leader in amphibian diversity, a significant part of its richness remains unknown or hidden under cryptic taxa. Here, we used model-based species delimitation in an integrative taxonomic approach, by gathering molecular and morphometric data to assess cryptic taxa within the monkey frogs Pithecopus rohdei, from the Atlantic Forest, and P. megacephalus, from campos rupestres ecosystem. We sampled one mitochondrial, five nuclear loci, and 18 morphometric variables. Using species-delimitation methods with genetic and morphometric data, we recovered five divergent lineages within P. rohdei and no cryptic lineages were recovered for P. megacephalus. Morphometric data show differentiation only for one of the candidate species revealed by the delimitation approaches, suggesting that individuals from Doce River basin constitute a putative species for formal taxonomic description. The time-calibrated mtDNA tree shows that P. rohdei complex lineages began to diverge in late Miocene. However, dates from the multilocus species tree are more recent, occurring in Pleistocene, and suggesting their persistence in refuges of forest and sky islands within the Atlantic Forest biome. The existence of cryptic taxa within P. rohdei is, therefore, relevant for planning conservation strategies for this species complex in the Atlantic Forest biodiversity hotspot. © 2018 Elsevier Inc.","<b>Joint Minimization of Monitoring Cost and Delay in Overlay Networks: Optimal Policies with a Markovian Approach<\/b><br><br>Continuous monitoring of network resources enables to make more-informed resource allocation decisions but incurs overheads. We investigate the trade-off between monitoring costs and benefits of accurate state information for a routing problem. In our approach link delays are modeled by Markov chains or hidden Markov models. The current delay information on a link can be obtained by actively monitoring this link at a fixed cost. At each time slot, the decision maker chooses to monitor a subset of links with the objective of minimizing a linear combination of long-run average delay and monitoring costs. This decision problem is modeled as a Markov decision process whose solution is computed numerically. In addition, in simple settings we prove that immediate monitoring cost and delay minimization leads to a threshold policy on a filter which sums up information from past measurements. The lightweight method as well as the optimal policy are tested on several use-cases. We demonstrate on an overlay of 30 nodes of RIPE Atlas that we obtain delay values close to the performance of the always best path with an extremely low monitoring effort when delays between nodes are modeled with hierarchical Dirichlet process hidden Markov models. © 2018, Springer Science+Business Media, LLC, part of Springer Nature.","<b>Genetic and pathogenic diversity of Colletotrichum species associated with apple diseases in southern Brazil and Uruguay<\/b><br><br>Two distinct diseases can be caused by Colletotrichum spp. on apple (Malus domestica); apple bitter rot (ABR) and Glomerella leaf spot (GLS). ABR is an old fruit disease occurring in most countries where apple is commercially cultivated. In contrast, GLS was first observed in the USA in 1970, and since then, it has been reported as an emerging leaf disease in different countries, such as Brazil in 1988 and China in 2012. In Brazil, GLS-outbreaks have become more frequent in the last years, making this disease one of the most important constraints for further cultivation of susceptible apples descending from 'Golden Delicious', like 'Gala'. Symptoms of GLS begin with small reddish-purple leaf spots about two days after pathogen infection, evolving into larger irregular necrotic lesions a few days later. Leaves usually turn yellow and fall off resulting in severe tree defoliation. Interestingly, whereas most isolates show organ specialization causing either ABR or GLS, only a few of them can provoke both diseases. Both ABR and GLS occur in Brazil, but only the former in Uruguay. Multilocus sequence analysis allowed the identification of three species causing ABR and GLS in Brazil, Colletotrichum fructicola, C. karstii, and C. nymphaeae, and four species causing ABR in Uruguay, C. fructicola, C. theobromicola, C. melonis, and C. paranaense. As C. fructicola is the most frequent species in both countries, genetic structure of isolates was determined using ISSR-based analysis and vegetative compatibility groups (VCGs). Both populations showed similar levels of Nei's diversity, and Bayesian cluster analysis inferred two genetic clusters correlated with the geographical origin of isolates. Moreover, all Uruguayan isolates were grouped into a unique VCG, while Brazilian isolates were grouped into four VCGs. C. fructicola isolates causing ABR and GLS in southern Brazil revealed to be genetically distinct and arose independently from the Uruguayan population. © 2019 International Society for Horticultural Science. All rights reserved.","<b>A hybrid swarm of manatees along the guianas coastline, a peculiar environment under the influence of the Amazon River plume<\/b><br><br>The West Indian (Trichechus manatus) and Amazonian (T. inunguis) manatees have a sympatric occurrence at the mouth of the Amazon River. A result of this interspecific encounter is the occurrence of hybrids, which are frequently found along the coasts of Amapá state in Brazil, French Guiana and Guyana. Here we present new genetic evidence indicating the occurrence of a hybrid swarm along the Guianas Shield coastline, which is an interspecific hybrid zone that also separates T. manatus populations located east (Brazil) and west (Caribbean, Gulf of Mexico, Florida and Antilles). In addition, we suggest that this hybrid population occupies a peculiar mangrove-rich environment under strong influence of the Amazon River plume, which requires an independent management and should be considered a special conservation area. © 2019, Academia Brasileira de Ciencias. All rights reserved.","<b>Fully Bayesian analysis of allele-specific RNA-seq data<\/b><br><br>Diploid organisms have two copies of each gene, called alleles, that can be separately transcribed. The RNA abundance associated to any particular allele is known as allele-specific expression (ASE). When two alleles have polymorphisms in transcribed regions, ASE can be studied using RNA-seq read count data. ASE has characteristics different from the regular RNA-seq expression: ASE cannot be assessed for every gene, measures of ASE can be biased towards one of the alleles (reference allele), and ASE provides two measures of expression for a single gene for each biological samples with leads to additional complications for single-gene models. We present statistical methods for modeling ASE and detecting genes with differential allelic expression. We propose a hierarchical, overdispersed, count regression model to deal with ASE counts. The model accommodates gene-specific overdispersion, has an internal measure of the reference allele bias, and uses random effects to model the gene-specific regression parameters. Fully Bayesian inference is obtained using the fbseq package that implements a parallel strategy to make the computational times reasonable. Simulation and real data analysis suggest the proposed model is a practical and powerful tool for the study of differential ASE. © 2019 the Author(s),","<b>A historical perspective of niche differentiation between two top predators in the Uruguayan coastal area<\/b><br><br>Context: As top predators, marine mammals play a key role consuming in different trophic levels and the trophic niche characterization may help to understand how species utilize and share resources. On the coast of the Río de la Plata and the South-west Atlantic, the South American sea lion (Otaria flavescens) and the franciscana dolphin (Pontoporia blainvillei) are two important predators. Aims: The present study investigated potential trophic overlap of both species by measuring stable carbon (δ13C) and nitrogen (δ15N) isotopes over two periods: historical (1959-79) and recent (2002-15) on the Uruguayan coast. Methods: Bone samples of P. blainvillei and O. flavescens were used to determine the isotopic niche using the Stable Isotope Bayesian Ellipses in R (SIBER) analysis. Key results: The isotopic niche did not overlap between species in any period. δ15N was higher in O. flavescens in both periods (20.29‰ ± 0.73 in the historical and 19.95‰ ± 1.0 in the recent period), indicating that it feeds at a higher trophic level than P. blainvillei. The δ13C was also significantly higher in O. flavescens than in P. blainvillei during the two periods (O. flavescens: -11.43 ± 0.6‰ historic, -12.72 ± 0.4‰ recent, and P. blainvillei: -12.69 ± 1.1‰ historic, -13.84 ± 1.3‰ recent). The isotopic niche areas of P. blainvillei in recent and historic periods confirmed they forage in 2 distinct environments, marine and estuarine, with low isotopic overlap. This overlap was higher in the recent period. Conclusions and Implications: O. flavescens and both P. blainvillei groups were segregated in both periods, with a higher overlap in the recent. These species appear to reduce competition by using different resources in the same coastal habitat. O. flavescens preferentially feeds on benthic fish and showed wider trophic amplitude in both periods, whereas P. blainvillei has a more coastal-pelagic diet and included a greater variability of resources in its diet. The differences between species trophic niches can still be detected after both marine mammals species abundance has declined and after the development of fisheries. © 2019 CSIRO.","<b>Impact of Pleistocene geoclimatic events on the genetic structure in mid-latitude South American plants: Insights from the phylogeography of Turnera sidoides complex (Passifloraceae, Turneroideae)<\/b><br><br>Aiming to shed light on the influence of Pleistocene geoclimatic events on the current genetic patterns in the flora of the South American Chaqueño Phytogeograpic Domain, we performed phylogeographic and Bayesian ancestral area reconstruction analyses based on plastid DNA sequences of Turnera sidoides. Lineages started to diversify 2.02 Mya, forming two major clades, one comprising extant populations from the western boundary and the other including populations from the eastern boundary of the distribution of T. sidoides. Major divergences within clades occurred at two years 1.44 and 1.72 Mya. Ancestral area reconstruction analyses suggest a western origin of T. sidoides in the Peripampasic Arc. Our results support the hypothesis that Pleistocene geoclimatic events in the Chaqueño Domain played an important role in shaping genetic variation. The establishment of the Arid Diagonal acted as a barrier to gene flow between the western and eastern populations during the Pleistocene Patagonian Glaciations. After this period, the orogeny of the region and Pleistocene climatic oscillations played a role in the divergence of isolated ancestral populations followed by migrations to the Chaco-Pampa plain. © 2018 The Linnean Society of London","<b>An evolutionary insight into zika virus strains isolated in the Latin American region<\/b><br><br>Zika virus (ZIKV) is an emerging pathogen member of the Flaviviridae family. ZIKV has spread rapidly in the Latin American region, causing hundreds of thousands of cases of ZIKV disease, as well as microcephaly in congenital infections. Detailed studies on the pattern of evolution of ZIKV strains have been extremely important to our understanding of viral survival, fitness, and evasion of the host’s immune system. For these reasons, we performed a comprehensive phylogenetic analysis of ZIKV strains recently isolated in the Americas. The results of these studies revealed evidence of diversification of ZIKV strains circulating in the Latin American region into at least five different genetic clusters. This diversification was also reflected in the different trends in dinucleotide bias and codon usage variation. Amino acid substitutions were found in E and prM proteins of the ZIKV strains isolated in this region, revealing the presence of novel genetic variants circulating in Latin America. © 2018 by the authors. Licensee MDPI, Basel, Switzerland.","<b>Mouth gape determines the response of marine top predators to long-term fishery-induced changes in food web structure<\/b><br><br>Here, we analyse changes throughout time in the isotopic niche of the Franciscana dolphin (Pontoporia blainvillei), the South American fur seal (Arctocephalus australis) and the South American sea lion (Otaria flavescens) from the Río de la Plata estuary and adjacent Atlantic Ocean to test the hypothesis that fishing may modify the diet of small-gape predators by reducing the average size of prey. The overall evidence, from stable isotope and stomach contents analyses, reveals major changes in resource partitioning between the three predators considered, mainly because of an increased access of Franciscana dolphins to juvenile demersal fishes. These results are consistent with the changes in the length distribution of demersal fish species resulting from fishing and suggest that Franciscana dolphin has been the most benefited species of the three marine mammal species considered because of its intermediate mouth gape. In conclusion, the impact of fishing on marine mammals goes beyond the simple reduction in prey biomass and is highly dependent on the mouth gape of the species involved. © 2018, The Author(s).","<b>Inferring population dynamics of HIV-1 subtype C epidemics in Eastern Africa and Southern Brazil applying different Bayesian phylodynamics approaches<\/b><br><br>The subtype C Eastern Africa clade (CEA), a particularly successful HIV-1 subtype C lineage, has seeded several sub-epidemics in Eastern African countries and Southern Brazil during the 1960s and 1970s. Here, we characterized the past population dynamics of the major CEA sub-epidemics in Eastern Africa and Brazil by using Bayesian phylodynamic approaches based on coalescent and birth-death models. All phylodynamic models support similar epidemic dynamics and exponential growth rates until roughly the mid-1980s for all the CEA sub-epidemics. Divergent growth patterns, however, were supported afterwards. The Bayesian skygrid coalescent model (BSKG) and the birth-death skyline model (BDSKY) supported longer exponential growth phases than the Bayesian skyline coalescent model (BSKL). The BDSKY model uncovers patterns of a recent decline for the CEA sub-epidemics in Burundi/Rwanda and Tanzania (Re < 1) and a recent growth for Southern Brazil (Re > 1); whereas coalescent models infer an epidemic stabilization. To the contrary, the BSKG model captured a decline of Ethiopian CEA sub-epidemic between the mid-1990s and mid-2000s that was not uncovered by the BDSKY model. These results underscore that the joint use of different phylodynamic approaches may yield complementary insights into the past HIV population dynamics. © 2018 The Author(s).","<b>Phylodynamics of Merkel-cell polyomavirus and human polyomavirus 6: A long-term history with humans<\/b><br><br>New human polyomaviruses have been described in the last years, including the Merkel-cell polyomavirus (MCPyV; Human polyomavirus 5) and the Human polyomavirus 6 (HPyV6). Although their infection is usually asymptomatic, in immunocompromised host can cause life-threatening pathologies, such as the Merkel cell carcinoma, an aggressive skin neoplasia associated to the MCPyV. Despite being prevalent viruses in population, epidemiological data from South America are scarce, as well as the characterization of the viral types circulating and their origin. The aims of this work were to describe MCPyV and HPyV6 from environmental samples with different geographical origin and to analyze their phylogenetic and evolutionary histories, particularly for MCPyV. Partial and complete genome sequences were obtained from sewage samples from Argentina, Uruguay and Spain. A total number of 87 sequences were obtained for MCPyV and 33 for HPyV6. Phylogenetic analysis showed that MCPyV sequences distributed according to their geographic origin in Europe/North America, Africa, Asia, South America and Oceania groups, suggesting that viral diversification might have followed human migrations across the globe. In particular, viruses from Argentina associated with Europe/North America and South America genotypes, whereas those from Uruguay and Spain also grouped with Africa genotype, reflecting the origin of the current population in each country, which could arrive not only during ancient human migration but also during recent migratory events. In addition, the South American group presented a high level of clusterization, showing internal clusters that could be related to specific locations, such as French Guiana and Brazil or the Southern region into South America, such as Argentina and Uruguay, suggesting a long term evolutionary process in the region. Additionally, in this work, we carried out the first analysis about the evolutionary history of MCPyV trough the integration of phylogenetic, epidemiological and historical data. Since a strong association is observed between the phylogenetic relationships and the origin of the sampled population, this analysis was based on the hypothesis of co-divergence between the virus and human populations. This analysis resulted in a substitution rate of 5.1 × 10−8 s/s/y (∼5.1% of divergence per million years) for the complete genome of MCPyV, which is in the range of those estimated for other double-stranded DNA viruses. Regarding HPyV6, a South American group with clusterization was observed (sequences from Uruguay). Meanwhile, sequences from Argentina grouped with European ones (France and Spain) and remained separated from those isolated in China, USA or Australia. The analysis of viruses from the environment allowed us to deep characterize prevalent infections in different geographic regions, reveling that viruses circulating in each population reflected its origin and that there are specific lineages associated with South America. © 2018 Elsevier Inc.","<b>Action semantics at the bottom of the brain: Insights from dysplastic cerebellar gangliocytoma<\/b><br><br>Recent embodied cognition research shows that access to action verbs in shallow-processing tasks becomes selectively compromised upon atrophy of the cerebellum, a critical motor region. Here we assessed whether cerebellar damage also disturbs explicit semantic processing of action pictures and its integration with ongoing motor responses. We evaluated a cognitively preserved 33-year-old man with severe dysplastic cerebellar gangliocytoma (Lhermitte-Duclos disease), encompassing most of the right cerebellum and the posterior part of the left cerebellum. The patient and eight healthy controls completed two semantic association tasks (involving pictures of objects and actions, respectively) that required motor responses. Accuracy results via Crawford's modified t-tests revealed that the patient was selectively impaired in action association. Moreover, reaction-time analysis through Crawford's Revised Standardized Difference Test showed that, while processing of action concepts involved slower manual responses in controls, no such effect was observed in the patient, suggesting that motor-semantic integration dynamics may be compromised following cerebellar damage. Notably, a Bayesian Test for a Deficit allowing for Covariates revealed that these patterns remained after covarying for executive performance, indicating that they were not secondary to extra-linguistic impairments. Taken together, our results extend incipient findings on the embodied functions of the cerebellum, offering unprecedented evidence of its crucial role in processing non-verbal action meanings and integrating them with concomitant movements. These findings illuminate the relatively unexplored semantic functions of this region while calling for extensions of motor cognition models. © 2018 Cervetto, Abrevaya, Martorell Caro, Kozono, Muñoz, Ferrari, Sedeño, Ibáñez and García.","<b>Genetic structure, core collection, and regeneration quality in white dent corn landraces<\/b><br><br>Genetic diversity and structure of plant germplasm collections, frequently determined by molecular markers, can be used to assist breeding programs, to validate core collections determined by other methodologies, to identify priority accessions for conservation, and to confirm genetic integrity after regeneration. This research aimed to study the genetic diversity and structure of the Uruguayan white dent maize collection, to evaluate the genetic representativeness of its core collection (previously defined by phenotypic traits), and to confirm the genetic integrity of seven regenerations made in Mexico and Uruguay in comparison with the original accessions. Ninety accessions were fingerprinted using 26 simple sequence repeat (SSR) markers. Genetic structure was analyzed by Ward clustering, canonical analysis, and a Bayesian approach based on allelic frequencies. All SSR markers were polymorphic with a mean number of alleles (A) of 7.43, an effective allele number (Ae) of 3.04, and expected heterozygosity of 0.579. The genetic variation between accessions was 0.251, and variation within accessions was 0.749. Four genetic groups were obtained using the three approaches. The core collection represented the structure of the whole collection because the four genetic groups were proportionally represented. The genetic diversity in the core collection did not differ from the entire collection in A, Ae, expected heterozygosity (He) percentage of polymorphic loci (%P), and expected heterozygosity within accessions (Hs). Most regenerations (9 out of 14) preserved the genetic integrity of original accessions, whereas in other cases, either new or lost alleles caused genetic differences. The analysis of genetic structure and diversity of germplasm collections, in combination with morphological characterizations, helps to define ex situ conservation strategies and usage in breeding programs. © Crop Science Society of America | 5585 Guilford Rd., Madison, WI 53711 USA All rights reserved.","<b>Prediction of complex traits: Robust alternatives to best linear unbiased prediction<\/b><br><br>A widely used method for prediction of complex traits in animal and plant breeding is \"genomic best linear unbiased prediction\" (GBLUP). In a quantitative genetics setting, BLUP is a linear regression of phenotypes on a pedigree or on a genomic relationship matrix, depending on the type of input information available. Normality of the distributions of random effects and of model residuals is not required for BLUP but a Gaussian assumption is made implicitly. A potential downside is that Gaussian linear regressions are sensitive to outliers, genetic or environmental in origin. We present simple (relative to a fully Bayesian analysis) to implement robust alternatives to BLUP using a linear model with residual t or Laplace distributions instead of a Gaussian one, and evaluate the methods with milk yield records on Italian Brown Swiss cattle, grain yield data in inbred wheat lines, and using three traits measured on accessions of Arabidopsis thaliana. The methods do not use Markov chain Monte Carlo sampling and model hyper-parameters, viewed here as regularization \"knobs,\" are tuned via some cross-validation. Uncertainty of predictions are evaluated by employing bootstrapping or by random reconstruction of training and testing sets. It was found (e.g., test-day milk yield in cows, flowering time and FRIGIDA expression in Arabidopsis) that the best predictions were often those obtained with the robust methods. The results obtained are encouraging and stimulate further investigation and generalization. © 2018 Gianola, Cecchinato, Naya and Schön.","<b>Human Bocavirus: Detection, Quantification and Molecular Characterization in Sewage and Surface Waters in Uruguay<\/b><br><br>Human bocavirus (HBoV) infections are related to respiratory and gastroenteric diseases. The aim of this study was to investigate the presence of HBoV in both sewage and surface waters in Uruguay. Sixty-eight sewage samples from the cities of Salto, Paysandú, Bella Unión, Fray Bentos, Treinta y Tres and Melo and 36 surface water samples from the cities of Salto, Florida and Santa Lucía were studied. HBoV was screened by multiplex qPCR for the detection of the four subtypes, followed by monoplex qPCRs for the independent quantification of each subtype. A qualitative PCR followed by DNA sequencing and phylogenetic analysis was carried out for molecular characterization of HBoV strains. HBoV was present in a high frequency (69%) in sewage and only one positive sample (3%) was found in surface water. Concerning sewage samples, HBoV1 was detected in 11 (23%) out of the 47 positives samples, with a mean concentration of 8.2 × 104 genomic copies/Liter (gc/L), HBoV3 was detected in 35 (74%) of the positive samples with a mean concentration of 4.1 × 106 gc/L and subtypes 2 and/or 4 were detected in 39 (83%) of the positive samples with a mean concentration of 7.8 × 106 gc/L. After the phylogenetic analysis performed by a Bayesian approach, the four HBoV subtypes were confirmed. This is the first study determining a high frequency of HBoV and the presence of the four HBoV subtypes in aquatic matrices in Latin America, mainly in sewage. Although HBoV was scarcely detected in surface water, a waterborne transmission is likely to occur if people enter in contact with polluted surface waters for recreational activities such as fishing or swimming since an elevated frequency of HBoV was detected in raw sewage which is usually directly discharged into surface waters. © 2018, Springer Science+Business Media, LLC, part of Springer Nature.","<b>The HIV-1 subtype B epidemic in French guiana and Suriname is driven by ongoing transmissions of pandemic and non-pandemic lineages<\/b><br><br>The HIV-1 subtype B epidemic in French Guiana and Suriname is characterized by the co-circulation of the globally disseminated “B PANDEMIC ” lineage and of non-pandemic subtype B lineages of Caribbean origin (B CAR ). To reconstruct the spatiotemporal pattern of spread of those viral lineages circulating in these two countries, a total of 361 HIV-1 subtype B pol sequences recovered from treatment-naive adult patients from French Guiana and Suriname between 2006 and 2012 were combined with B PANDEMIC and B CAR reference sequences. Major Guianese/Surinamese B PANDEMIC and B CAR lineages were identified by Maximum Likelihood phylogenetic analysis and the spatiotemporal and demographic parameters estimated using a Bayesian coalescent-based method. We detected four B CAR and three B PANDEMIC transmission chains of large size that together comprise most pandemic and non-pandemic subtype B sequences from French Guiana (≥52%) and Suriname (≥70%) here analyzed. These major lineages were probably introduced into French Guiana and Suriname from the Caribbean (B CAR ) and North/South America (B PANDEMIC ) between the middle 1970s and the late 1980s and spread among populations from both countries with roughly comparable demographic growth rates. We detected a significant trend for higher viral loads and higher proportion of homosexual/bisexual men among subjects infected with B PANDEMIC relative to B CAR strains in French Guiana. These results show that the HIV subtype B epidemic in French Guiana and Suriname has been driven by multiple active B CAR and B PANDEMIC transmission chains that arose since the middle 1970s onward and operate in both countries simultaneously. Although no significant differences in the epidemic potential of major B CAR and B PANDEMIC lineages were observed, relevant associations between the infecting subtype B lineage and epidemiological and clinical characteristics were detected in French Guiana. Copyright © 2018 Bello, Nacher, Divino, Darcissac, Mir and Lacoste. This is an open-access article distributed under the terms of the Creative Commons Attribution License (CC BY).","<b>Kelps' long-distance dispersal: Role of Ecological/Oceanographic processes and implications to marine forest conservation<\/b><br><br>Long-distance dispersal is one of the main drivers structuring the distribution of marine biodiversity. This study reports the first occurrence of Macrocystis pyrifera and Durvillaea antarctica rafts on the southwestern warm temperate coast of the Atlantic Ocean. Our results indicate that an extreme meteo-oceanographic event, characterized by a northward, displacement of cold sub-Antarctic oceanic waters driven by an extratropical cyclone, could account for these unusual occurrences. A niche model based on known current distribution and maximum entropy principle (MAXENT), revealed the availability of suitable habitats at lower latitudes, outside their actual distribution edges. The distributional boundaries, mainly driven by temperature and irradiance, suggest the existence of environmental suitability in warm temperate areas, as well as in the Northern Hemisphere off Atlantic and Asian coasts. These theoretical edges and respective environmental drivers agree with the physiological affinities of both species, supporting the hypothesis that these variables act as limiting factors for their occurrences in tropical or warmer areas. Emerging regions can function as refuges and stepping-stones, providing substrate with adequate habitat conditions for recruitment of propagules, allowing eventual colonization. Long dispersal events reinforce the need for an extensive discussion on selective management of natural dispersion, biological invasions, refuge mapping and conservation initiatives in a transnational perspective. © 2018 by the authors.","<b>Joint denoising and decompression: A patch-based Bayesian approach<\/b><br><br>JPEG and Wavelet compression artifacts leading to Gibbs effects and loss of texture are well known and many restoration solutions exist in the literature. So is denoising, which has occupied the image processing community for decades. However, when a noisy image is compressed, the noisy wavelet coefficients can be assigned to the 'wrong' quantization interval, generating artifacts that can have dramatic consequences in products derived from satellite image pairs such as sub-pixel stereo vision and digital terrain elevation models. Despite the fact that the importance of such artifacts in very high resolution satellite imaging has recently been recognized, this restoration problem has been rarely addressed in the literature. In this work we present a thorough probabilistic analysis of the wavelet outliers phenomenon, and conclude that their probabilistic nature is characterized by a single parameter related to the ratio q/σ of the compression rate and the instrumental noise. This analysis provides the conditional probability for a Bayesian MAP estimator, whereas a patch-based local Gaussian prior model is learnt from the corrupted image iteratively, like in state-of-the-art patch-based denoising algorithms, albeit with the additional difficulty of dealing with non-Gaussian noise during the learning process. The resulting joint denoising and decompression algorithm is experimentally evaluated under realistic conditions. The results show its ability to simultaneously denoise, decompress and remove wavelet outliers better than the available alternatives, both from a quantitative and a qualitative point of view. As expected, the advantage of our method is more evident for large values of q/σ. © 2017 IEEE.","<b>Geographic distribution modeling of the margay (Leopardus wiedii) and jaguarundi (Puma yagouaroundi): A comparative assessment<\/b><br><br>The Neotropical region currently harbors 8 recognized species of small cats, including the margay (Leopardus wiedii) and jaguarundi (Puma yagouaroundi). These similar-sized species have broad geographic distributions and commonly occur in sympatry. We constructed species occurrence databases and used the maximum entropy (Maxent) modeling approach to develop models of habitat suitability for each of these species. We identified the environmental predictors with the highest importance for species occurrence and evaluated how they influence habitat suitability for both wild cats. Variables related to temperature and precipitation were good predictors of species occurrence, with both cats responding similarly to them. However, margay showed higher variation in habitat suitability depending on local climatic predictors. The estimated models indicated high niche similarity (> 76%) between these cats, but despite this we identified different areas with the most favorable conditions for each species' occurrence. Approximately 50% and 72% of the entire suitable area were estimated as being moderately to highly suitable for margay and jaguarundi, respectively, with the margay model comprising a higher proportion of forest environments in comparison to that of the jaguarundi. These findings will be relevant in the context of conservation planning when indicating the priority areas for their long-term preservation. © 2017 American Society of Mammalogists.","<b>Latent class models for Echinococcus multilocularis diagnosis in foxes in Switzerland in the absence of a gold standard<\/b><br><br>Background: In Europe the principal definitive host for Echinococcus multilocularis, causing alveolar echinococcosis in humans, is the red fox (Vulpes vulpes). Obtaining reliable estimates of the prevalence of E. multilocularis and relevant risk factors for infection in foxes can be difficult if diagnostic tests with unknown test accuracies are used. Latent-class analysis can be used to obtain estimates of diagnostic test sensitivities and specificities in the absence of a perfect gold standard. Samples from 300 foxes in Switzerland were assessed by four different diagnostic tests including necropsy followed by sedimentation and counting technique (SCT), an egg-PCR, a monoclonal and a polyclonal copro-antigen ELISA. Information on sex, age and presence of other cestode species was assessed as potential covariates in the Bayesian latent class models. Different Bayesian latent-class models were run, considering dichotomized test results and, additionally, continuous readings resulting in empirical ROC curves. Results: The model without covariates estimated a true parasite prevalence of 59.5% (95% CI: 43.1-66.4%). SCT, assuming a specificity of 100%, performed best among the four tests with a sensitivity of 88.5% (95% CI: 82.7-93.4%). The egg-PCR showed a specificity of 93.4% (95% CI: 87.3-99.1%), although its sensitivity of 54.8% was found moderately low (95% CI: 48.5-61.0%). Relatively higher sensitivity (63.2%, 95% CI: 55.3-70.8%) and specificity (70.0%, 95% CI: 60.1-79.4%) were estimated for the monoclonal ELISA compared to the polyclonal ELISA with a sensitivity and specificity of 56.0% (95% CI: 48.0-63.9%) and 65.9% (95% CI: 55.8-75.6%), respectively. In the Bayesian models, adult foxes were found to be less likely infected than juveniles. Foxes with a concomitant cestode infection had double the odds of an E. multilocularis infection. ROC curves following a Bayesian approach enabled the empirical determination of the best cut-off point. While varying the cut-offs of both ELISAs, sensitivity and specificity of the egg-PCR and SCT remained constant in the Bayesian latent class models. Conclusions: Adoption of a Bayesian latent class approach helps to overcome the absence of a perfectly accurate diagnostic test and gives a more reliable indication of the test performance and the impact of covariates on the prevalence adjusted for diagnostic uncertainty. © 2017 The Author(s).","<b>Historical isolation and contemporary gene flow drive population diversity of the brown alga Sargassum thunbergii along the coast of China<\/b><br><br>CONCLUSIONS: Our results suggest that the deep genetic divergence in S. thunbergii along the China coast may result from long-term geographic isolation during glacial periods. The dispersal of S. thunbergii driven by coastal currents may facilitate the admixture between southern and northern regimes. Our findings exemplify how both historical and contemporary forces are needed to understand phylogeographical patterns in coastal marine species with long-distance dispersal.\nRESULTS: We used 11 microsatellites and the plastid RuBisCo spacer to evaluate the genetic diversity of 22 Sargassum thunbergii populations sampled along the China coast. Population structure and differentiation was inferred based on genotype clustering and pairwise F ST and allele-frequency analyses. Integrated genetic analyses revealed two genetic clusters in S. thunbergii that dominated in the Yellow-Bohai Sea (YBS) and East China Sea (ECS) respectively. Higher levels of genetic diversity and variation were detected among populations in the YBS than in the ECS. Bayesian coalescent theory was used to estimate contemporary and historical gene flow. High levels of contemporary gene flow were detected from the YBS (north) to the ECS (south), whereas low levels of historical gene flow occurred between the two regions.\nBACKGROUND: Long-term survival in isolated marginal seas of the China coast during the late Pleistocene ice ages is widely believed to be an important historical factor contributing to population genetic structure in coastal marine species. Whether or not contemporary factors (e.g. long-distance dispersal via coastal currents) continue to shape diversity gradients in marine organisms with high dispersal capability remains poorly understood. Our aim was to explore how historical and contemporary factors influenced the genetic diversity and distribution of the brown alga Sargassum thunbergii, which can drift on surface water, leading to long-distance dispersal.","<b>Evidence of increasing diversification of Zika virus strains isolated in the American continent<\/b><br><br>Zika virus (ZIKV) is a member of the family Flaviviridae. ZIKV emerged in Brazil in 2015, causing an unprecedented epidemic and since then the virus has rapidly spread throughout the Americas. These facts highlight the need of detailed phylogenetic studies to understand the emergence, spread, and evolution of ZIKV populations. For these reasons, a Bayesian coalescent Markov Chain Monte Carlo analysis of complete genome sequences of ZIKV strains recently isolated in the American continent was performed. The results of these studies revealed an increasing diversification of ZIKV strains in different genetic lineages and co-circulation of distinct genetic lineages in several countries in the region. The time of the most recent common ancestor (tMRCA) was established to be around February 20, 2014 for ZIKV strains circulating in the American region. A mean rate of evolution of 1.55 × 10−3 substitutions/site/year was obtained for ZIKV strains included in this study. A Bayesian skyline plot indicate a sharp increase in population size from February 2014 to July 2015 and a decline during 2016. These results are discussed in terms of the emergence and evolution of ZIKV populations in the American continent. © 2017 Wiley Periodicals, Inc.","<b>Amphitropical disjunctions in new world menthinae: Three pliocene dispersals to South America following late miocene dispersal to North America from the old world<\/b><br><br>PREMISE OF THE STUDY: The subtribe Menthinae (Lamiaceae), with 35 genera and 750 species, is among the largest and most economically important subtribes within the mint family. Most genera of Menthinae are found exclusively in the New World, where the group has a virtually continuous distribution ranging from temperate North America to southern South America. In this study, we explored the presence, timing, and origin of amphitropical disjuncts within Menthinae. METHODS: Our analyses were based on a data set consisting of 89 taxa and the nuclear ribosomal DNA markers ITS and ETS. Phylogenetic relationships were determined under maximum likelihood and Bayesian criteria, divergence times were estimated with the program BEAST, and ancestral range estimated with BioGeoBEARS. KEY RESULTS: A North Atlantic Land Bridge migration event at about 10.6 Ma is inferred from western Eurasia to North America. New World Menthinae spread rapidly across North America, and then into Central and South America. Several of the large speciose genera are not monophyletic with nuclear rDNA, a finding mirrored with previous chloroplast DNA results. Three amphitropical disjunctions involving North and southern South America clades, one including a southeastern South American clade with several genera, were inferred to have occurred within the past 5 Myr. CONCLUSIONS: Although three New World Menthinae genera occur in both North and South America, none exhibit an amphitropical disjunction. However, three clades exhibit amphitropical disjunctions, all dating to the early Pliocene, and all involve jump dispersals to either southeastern or southwestern South America from southeastern North America. © 2017 Botanical Society of America.","<b>The disjunct pattern of the Neotropical harvestman Discocyrtus dilatatus (Gonyleptidae) explained by climate-driven range shifts in the Quaternary: Paleodistributional and molecular evidence<\/b><br><br>The disjunct distribution of the harvestman Discocyrtus dilatatus (Opiliones, Gonyleptidae) is used as a case study to test the hypothesis of a trans-Chaco Pleistocene paleobridge during range expansion stages. This would have temporarily connected humid regions (‘Mesopotamia’ in northeastern Argentina, and the ‘Yungas’ in the northwest, NWA) in the subtropical and temperate South American lowlands. The present study combines two independent approaches: paleodistributional reconstruction, using the Species Distribution Modeling method MaxEnt and projection onto Quaternary paleoclimates (6 kya, 21 kya, 130 kya), and phylogeographic analyses based on the cytochrome oxidase subunit I molecular marker. Models predict a maximal shrinkage during the warm Last Interglacial (130 kya), and the rise of the hypothesized paleobridge in the Last Glacial Maximum (21 kya), revealing that cold-dry stages (not warm-humid ones, as supposed) enabled the range expansion of this species. The disjunction was formed in the mid-Holocene (6 kya) and is intensified under current conditions. The median-joining network shows that NWA haplotypes are peripherally related to different Mesopotamian lineages; haplotypes from Santa Fe and Córdoba Provinces consistently occupy central positions in the network. According to the dated phylogeny, Mesopotamia-NWA expansion events would have occurred in the last glacial period, in many cases closely associated to the Last Glacial Maximum, with most divergence events occurring shortly thereafter. Only two (out of nine) NWA haplotypes are shared with Mesopotamian localities. A single, presumably relictual NWA haplotype was found to have diverged much earlier, suggesting an ancient expansion event not recoverable by the paleodistributional models. Different measures of sequence statistics, genetic diversity, population structure and history of demographic changes are provided. This research offers the first available evidence for the historical origin of NWA disjunct populations of a Mesopotamian harvestman. © 2017 Vergara et al. This is an open access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.","<b>Tracing the origin of the NS1 A188V substitution responsible for recent enhancement of ZIKA virus asian genotype infectivity<\/b><br><br>A recent study showed that infectivity of Zika virus (ZIKV) Asian genotype was enhanced by an alanine-to-valine amino acid substitution at residue 188 of the NS1 protein, but the precise time and location of origin of this mutation were not formally estimated. Here, we applied a Bayesian coalescent-based framework to estimate the age and location of the ancestral viral strain carrying the A188V substitution. Our results support that the ancestral ZIKV strain carrying the A188V substitution arose in Southeastern Asia at the early 2000s and circulated in that region for some time (5-10 years) before being disseminated to Southern Pacific islands and the Americas. © 2017, Fundacao Oswaldo Cruz. All rights reserved.","<b>Evolutionarily significant units of the critically endangered leaf frog Pithecopus ayeaye (Anura, Phyllomedusidae) are not effectively preserved by the Brazilian protected areas network<\/b><br><br>Protected areas (PAs) are essential for biodiversity conservation, but their coverage is considered inefficient for the preservation of all species. Many species are subdivided into evolutionarily significant units (ESUs) and the effectiveness of PAs in protecting them needs to be investigated. We evaluated the usefulness of the Brazilian PAs network in protecting ESUs of the critically endangered Pithecopus ayeaye through ongo-ing climate change. This species occurs in a threatened mountaintop ecosystem known as campos rupestres. We used multilocus DNA sequences to delimit geographic clus-ters, which were further validated as ESUs with a coalescent approach. Ecological niche modeling was used to estimate spatial changes in ESUs’ potential distributions, and a gap analysis was carried out to evaluate the effectiveness of the Brazilian PAs network to protect P. ayeaye in the face of climate changes. We tested the niche overlap between ESUs to gain insights for potential management alternatives for the spe-cies. Pithecopus ayeaye contains at least three ESUs isolated in distinct mountain regions, and one of them is not protected by any PA. There are no climatic niche differences between the units, and only 4% of the suitable potential area of the species is protected in present and future projections. The current PAs are not effective in pre-serving the intraspecific diversity of P. ayeaye in its present and future range distribu-tions. The genetic structure of P. ayeaye could represent a typical pattern in campos rupestres endemics, which should be considered for evaluating its conservation status. © 2017 The Authors. Ecology and Evolution published by John Wiley & Sons Ltd.","<b>Molecular adaptive convergence in the α-globin in subterranean octodontid rodents<\/b><br><br>Tuco-tucos (Ctenomys) and related coruros (Spalacopus) are South American subterranean rodents. An energetically demanding lifestyle within the hypoxic/hypercapnic underground atmosphere may change the selective regime on genes involved in O2 transport in blood. In addition, some species of tuco-tucos may be found at high altitude, thus facing additional reductions in changes O2 availabily. We examined sequence variation in the alpha globin subunit gene of hemoglobine in these lineages, within a robust phylogenetic context. Using different approaches (classical and Bayesian maximum likelihood (PAML/Datamonkey) and alternatives methods (TreeSAAP)) we found at least 2 sites with evidence of positive selection in the basal branch of Octodontidae, but not in tuco-tucos. These results suggest some adaptive changes associated to fossoriality, but not strictly to life underground. © 2017 Elsevier B.V.","<b>Description of a new soft tick species (Acari: Argasidae: Ornithodoros) associated with stream-breeding frogs (Anura: Cycloramphidae: Cycloramphus) in Brazil<\/b><br><br>In this study, we present a morphological description of immature and adult specimens of Ornithodoros saraivai n. sp., a tick associated with the frog Cycloramphus boraceiensis (Cycloramphidae) at São Sebastião island, located in the São Paulo state seaboard, Brazil. While larvae of O. saraivai are ecologically related to Ornithodoros faccinii, another soft tick associated with cycloramphids, the combination of 7 sternal pairs, 16 dorsal pairs, a pyriform dorsal plate and a partially toothed hypostome constitute unique characters of the O. saraivai larvae. One undetermined nymphal instar and adults of O. saraivai are similar to mature specimens of the Ornithodoros talaje species group; however, the O. saraivai specimens can be recognized by the presence of a robust bean-shaped spiracle with a large spiracular plate and more than two long seta in palpal article I. Identical partial sequences of the mitochondrial 16S rDNA gene confirmed the identity for all collected stages and for two cohorts of laboratory-reared larvae of O. saraivai. A Bayesian and Maximum Parsimony inferred phylogenetic trees support the position of O. saraivai in a clade with O. faccinii, suggesting the existence of an Ornithodoros lineage that evolved in association with amphibians. © 2017 Elsevier GmbH","<b>A new species of Deltamys Thomas, 1917 (Rodentia: Cricetidae) endemic to the southern Brazilian Araucaria Forest and notes on the expanded phylogeographic scenario of D. kempi<\/b><br><br>Deltamys is a monotypic sigmodontine rodent from the Pampas of South America. In addition to the formally recognized D. kempi that inhabits lowlands, an undescribed form Deltamys sp. 2n=40 was recently found in the highlands of south-eastern Brazil. In the present study, we perform a phylogeographic reassessment of Deltamys and describe a third form of the genus, endemic to the Brazilian Araucaria Forest. We describe this new species based on an integrative analysis, using complete cytochrome b DNA sequences, karyology and morphology. Bayesian tree recovered two allopatric clades (low-lands vs. highlands) and three lineages: (i) the lowland D. kempi, (ii) the highland Deltamys sp. 2n=40, and (iii) Deltamys araucaria sp. n. Deltamys araucaria sp. n. is karyotypically (2n=34) and morphologically distinguishable from D. kempi (2n=37-38), showing a tawnier dorsum/flank pelage, presence of a protostyle, M1 alveolus positioned anteriorly to the posterior margin of the zygomatic plate, and several other distinguishing characteristics. A phylogeographic assessment of D. kempi recovered two haplogroups with significant differences in skull measurements. This phylogeographic break seems to have been shaped by the Patos Lagoon estuarine channel. The diversification in Deltamys might have been trig-gered by dispersal of older lineages over different altitudinal ranges in the Paraná geological basin. Copyright © 2017 Magnolia Press.","<b>Uncertainty Quantification in a Patient-Specific One-Dimensional Arterial Network Model: EnKF-Based Inflow Estimator<\/b><br><br>Successful clinical use of patient-specific models for cardiovascular dynamics depends on the reliability of the model output in the presence of input uncertainties. For 1D fluid dynamics models of arterial networks, input uncertainties associated with the model output are related to the specification of vessel and network geometry, parameters within the fluid and wall equations, and parameters used to specify inlet and outlet boundary conditions. This study investigates how uncertainty in the flow profile applied at the inlet boundary of a 1D model affects area and pressure predictions at the center of a single vessel. More specifically, this study develops an iterative scheme based on the ensemble Kalman filter (EnKF) to estimate the temporal inflow profile from a prior distribution of curves. The EnKF-based inflow estimator provides a measure of uncertainty in the size and shape of the estimated inflow, which is propagated through the model to determine the corresponding uncertainty in model predictions of area and pressure. Model predictions are compared to ex vivo area and blood pressure measurements in the ascending aorta, the carotid artery, and the femoral artery of a healthy male Merino sheep. Results discuss dynamics obtained using a linear and a nonlinear viscoelastic wall model. Copyright © 2017 by ASME","<b>Trophic relationships in apex predators in an estuary system: A multiple-method approximation<\/b><br><br>The trophic level and ecological role of species in the ecosystem are determined by their feeding habits. While the trophic level and niche width are characteristic at population level, trophic overlap indicates the interaction between populations of a community. Three apex predators coexist in Uruguayan waters: Franciscana dolphin (Pontoporia blainvillei), South American sea lion (SASL, Otaria flavescens) and South American fur seal (SAFS, Arctocephalus australis). The two otariids consume different main prey species and have different isotopic niche, whereas feeding habits of Franciscana dolphin in Uruguayan waters remain practically unknown. This study aimed to determine Franciscana diet composition by stomach content and stable isotope (δ13C and δ15N) analyses, and to assess trophic overlap in diet composition as well as in isotopic niche space among the three apex predators. Diet composition of each species was estimated by Bayesian mixing models with prior information on the diet of predators. Moreover, Bayesian ellipses were used to define the isotopic niche space and overlap among the three species. Diets among predators varied: diet of SAFS was composed mainly by a few pelagic prey species, diet of SASL was more diverse and included pelagic and benthic preys, and for Franciscana dolphin the diet was mainly composed by coastal prey species. The SAFS had the largest isotopic niche area compared to the Franciscana and the SASL. The isotopic niche of the SASL did not overlap with those of the other two species. Although the Franciscana and the SAFS overlap in isotopic space, they do not overlap in feeding areas; this is because particulate organic matter δ13C values increase from inner Rio de la Plata to the estuary and the Atlantic Coast. Here, the relevance of using different sources of information to reach strong conclusions about feeding habits of marine predators and about co-use of resources is shown, mainly in environments where variation in δ13C can be related to diverse reasons. © 2016 Elsevier B.V.","<b>Phylodynamics of the major HIV-1 CRF02_AG African lineages and its global dissemination<\/b><br><br>The HIV-1 CRF02_AG clade is the most prevalent HIV variant in West and West-Central Africa and its detection outside Africa is increasingly common. Little is known, however, about the number and phylodynamics of major CRF02_AG lineages circulating worldwide. To this end, a total of 3170 HIV-1 CRF02_AG-like pol sequences isolated around the world, over a period of 25 years (1989 to 2013), were analyzed using Maximum Likelihood and Bayesian coalescent-based methods. Our results suggest that most of the current CRF02_AG diversity comes from the dissemination of a few founder strains out of Central Africa into West Africa and Cameroon between the late 1960s and the middle 1980s. The CRF02_AG strain introduced into West Africa established a large regional epidemic with low phylogeographic structure. This strain was also successfully disseminated out of the West African region and originated at least three large secondary outbreaks in Cameroon at around the late 1970s, in the former Soviet Union (FSU) countries at around the late 1990s, and in Bulgaria/Germany at around the early 2000s. The CRF02_AG African lineages introduced into Cameroon remained mostly restricted to this country and its neighbors. Demographic reconstructions indicate that major CRF02_AG clades circulating in Africa exhibited a decline in growth rate since the middle 1980s/1990s, whereas CRF02_AG clades in Europe and the FSU countries continue to grow exponentially until the middle to late 2000s. Substantial differences in the median estimated growth rate of the same CRF02_AG clade circulating in different regions (0.63–2.00 year− 1), and of different CRF02_AG clades circulating in the same country (0.41–0.75 year− 1) were observed. Thus, the cause of the epidemic outcome of the different HIV-1 CRF02_AG lineages is probably multifactorial. © 2016 Elsevier B.V.","<b>Trophic niche shifts driven by phytoplankton in sandy beach ecosystems<\/b><br><br>Stable isotopes (δ13C and δ15N) together with chlorophyll a and densities of surf diatoms were used to analyze changes in trophic niches of species in two sandy beaches of Uruguay with contrasting morphodynamics (i.e. dissipative vs. reflective). Consumers and food sources were collected over four seasons, including sediment organic matter (SOM), suspended particulate organic matter (POM) and the surf zone diatom Asterionellopsis guyunusae. Circular statistics and a Bayesian isotope mixing model were used to quantify food web differences between beaches. Consumers changed their trophic niche between beaches in the same direction of the food web space towards higher reliance on surf diatoms in the dissipative beach. Mixing models indicated that A. guyunusae was the primary nutrition source for suspension feeders in the dissipative beach, explaining their change in dietary niche compared to the reflective beach where the proportional contribution of surf diatoms was low. The high C/N ratios in A. guyunusae indicated its high nutritional value and N content, and may help to explain the high assimilation by suspension feeders at the dissipative beach. Furthermore, density of A. guyunusae was higher in the dissipative than in the reflective beach, and cell density was positively correlated with chlorophyll a only in the dissipative beach. Therefore, surf diatoms are important drivers in the dynamics of sandy beach food webs, determining the trophic niche space and productivity. Our study provides valuable insights on shifting foraging behavior by beach fauna in response to changes in resource availability. © 2016 Elsevier Ltd","<b>Modeling a production scale milk drying process: Parameter estimation, uncertainty and sensitivity analysis<\/b><br><br>A steady state model for a production scale milk drying process was built to help process understanding and optimization studies. It involves a spray chamber and also internal/external fluid beds. The model was subjected to a comprehensive statistical analysis for quality assurance using sensitivity analysis of inputs/parameters, and uncertainty analysis to estimate confidence intervals on parameters and model predictions (error propagation). Variance based sensitivity analysis (Sobol's method) was used to quantify the influence of inputs on the final powder moisture as the model output. Bayesian Inference using Markov Chain Monte Carlo sampling was used to quantify the uncertainty on the estimated parameters using available process data. In a full scale process the inputs with major range of variation are: moisture content at concentrate chamber feed (variation around 4%), and humidity at chamber inlet air (variation > 100%). The sensitivity analysis results suggest exploring improvements in the current control (Proportional Integral Derivative) for moisture content at concentrate chamber feed in order to reduce the output variance. It is also confirmed that humidity control at chamber inlet air stream would not be necessary because, despite its wide range of variation (air taken from outside), its impact on output variance is low. The uncertainty analysis results showed that confidence intervals obtained for parameters were reasonable, although some of them were found significantly correlated. For model applications, this means that model simulations should be performed using not only parameter values but also their correlation matrix by means of non-linear error propagation methods such as Monte Carlo techniques. The separate effects on model prediction uncertainties due to parameter estimation and measurement errors were studied. The results indicate that the error in measurements is the main responsible for the uncertainty in output predictions. Therefore using proper filtering of measurements, the comprehensively tested model is ready to support simulation based efforts for further process optimization. © 2016 Elsevier Ltd.","<b>Bayesian coalescent inference reveals high evolutionary rates and diversification of Zika virus populations<\/b><br><br>Zika virus (ZIKV) is a member of the family Flaviviridae. In 2015, ZIKV triggered an epidemic in Brazil and spread across Latin America. By May of 2016, the World Health Organization warns over spread of ZIKV beyond this region. Detailed studies on the mode of evolution of ZIKV strains are extremely important for our understanding of the emergence and spread of ZIKV populations. In order to gain insight into these matters, a Bayesian coalescent Markov Chain Monte Carlo analysis of complete genome sequences of recently isolated ZIKV strains was performed. The results of these studies revealed a mean rate of evolution of 1.20 × 10−3 nucleotide substitutions per site per year (s/s/y) for ZIKV strains enrolled in this study. Several variants isolated in China are grouped together with all strains isolated in Latin America. Another genetic group composed exclusively by Chinese strains were also observed, suggesting the co-circulation of different genetic lineages in China. These findings indicate a high level of diversification of ZIKV populations. Strains isolated from microcephaly cases do not share amino acid substitutions, suggesting that other factors besides viral genetic differences may play a role for the proposed pathogenesis caused by ZIKV infection. J. Med. Virol. 88:1672–1676, 2016. © 2016 Wiley Periodicals, Inc. © 2016 Wiley Periodicals, Inc.","<b>New insights into the hepatitis E virus genotype 3 phylodynamics and evolutionary history<\/b><br><br>Hepatitis E virus (HEV) is an emergent hepatotropic virus endemic mainly in Asia and other developing areas. However, in the last decade it has been increasingly reported in high-income countries. Human infecting HEV strains are currently classified into four genotypes (1-4). Genotype 3 (HEV-3) is the prevalent virus genotype and the mostly associated with autochthonous and sporadic cases of HEV in developed areas. The evolutionary history of HEV worldwide remains largely unknown. In this study we reconstructed the spatiotemporal and population dynamics of HEV-3 at global scale, but with particular emphasis in South America, where case reports have increased dramatically in the last years. To achieve this, we applied a Bayesian coalescent-based approach to a comprehensive data set comprising 97 GenBank HEV-3 sequences for which the location and sampling date was documented. Our phylogenetic analyses suggest that the worldwide genetic diversity of HEV-3 can be grouped into two main Clades (I and II) with a Tmrca dated in approximately 320 years ago (95% HPD: 420-236 years) and that a unique independent introduction of HEV-3 seems to have occurred in Uruguay, where most of the human HEV cases in South America have been described. The phylodynamic inference indicates that the population size of this virus suffered substantial temporal variations after the second half of the 20th century. In this sense and conversely to what is postulated to date, we suggest that the worldwide effective population size of HEV-3 is not decreasing and that frequently sources of error in its estimates stem from assumptions that the analyzed sequences are derived from a single panmictic population. Novel insights on the global population dynamics of HEV are given. Additionally, this work constitutes an attempt to further describe in a Bayesian coalescent framework, the phylodynamics and evolutionary history of HEV-3 in the South American region. © 2016 Elsevier B.V.","<b>Weighting strategies for single-step genomic BLUP: An iterative approach for accurate calculation of GEBV and GWAS<\/b><br><br>Genomic Best Linear Unbiased Predictor (GBLUP) assumes equal variance for all single nucleotide polymorphisms (SNP). When traits are influenced by major SNP, Bayesian methods have the advantage of SNP selection. To overcome the limitation of GBLUP, unequal variance or weights for all SNP are applied in a method called weighted GBLUP (WGBLUP). If only a fraction of animals is genotyped, single-step WGBLUP (WssGBLUP) can be used. Default weights in WGBLUP or WssGBLUP are obtained iteratively based on single SNP effect squared (u2) and/or heterozygosity. When the weights are optimal, prediction accuracy, and ability to detect major SNP are maximized. The objective was to develop optimal weights for WGBLUP-based methods. We evaluated 5 new procedures that accounted for locus-specific or windows-specific variance to maximize accuracy of predicting genomic estimated breeding value (GEBV) and SNP effect. Simulated datasets consisted of phenotypes for 13,000 animals, including 1540 animals genotyped for 45,000 SNP. Scenarios with 5, 100, and 500 simulated quantitative trait loci (QTL) were considered. The 5 new procedures for SNP weighting were: (1) u2 plus a constant equal to the weight of the top SNP; (2) from a heavy-tailed distribution (similar to BayesA); (3) for every 20 SNP in a window along the whole genome, the largest effect (u2) among them; (4) the mean effect of every 20 SNP; and (5) the summation of every 20 SNP. Those methods were compared to the default WssGBLUP, GBLUP, BayesB, and BayesC. WssGBLUP methods were evaluated over 10 iterations. The accuracy of predicting GEBV was the correlation between true and estimated genomic breeding values for 300 genotyped animals from the last generation. The ability to detect the simulated QTL was also investigated. For most of the QTL scenarios, the accuracies obtained with all WssGBLUP procedures were higher compared to those from BayesB and BayesC, partly due to automatic inclusion of parent average in the former. Manhattan plots had higher resolution with 5 and 100 QTL. Using a common weight for a window of 20 SNP that sums or averages the SNP variance enhances accuracy of predicting GEBV and provides accurate estimation of marker effects. © 2016 Zhang, Lourenco, Aguilar, Legarra and Misztal.","<b>Threatened amphibians of Uruguay: Effectiveness of protected areas face of climate change<\/b><br><br>Climate change (CC) constitutes one of the main biodiversity threats. The effects of CC on species are apparent worldwide, with primarily poleward and upward shifts in species distribution. Due to their narrow distribution, threatened species are highly vulnerable to CC. In this context, protected areas (PA) could be key tools for adaptation to CC. Our aims were to study the effects of CC on the distribution and richness of the threatened and near threatened amphibians of Uruguay; and to evaluate the effectiveness of the National Network of Protected Areas (NNPA) at present and under future climate scenarios. To model the distribution of nine species, we obtained records from herpetological collections, scientific publications and GBIF, as well as current data and future climate projections of the General Circulation Model HadCM3 under the A2 and B2 IPCC scenarios from Worldclim. To model species distribution we applied maximum entropy techniques (MAXENT). To evaluate the effectiveness of NNPA we conducted a gap analysis by overlaying PA with the distribution of species. The models indicate that the distribution of most amphibian species could expand in Uruguay in the future; except for Pleurodema bibroni y Melanophryniscus montevidensis. Local amphibian richness is predicted to increase, mainly in the northwest and east of the country. While amphibians studied are included in at least one PA, less than 2% of the distribution is actually covered by the NNPA, both currently and under CC scenarios; which it is a strong indicator of the inefficiency of the system. Although projected CC for this region would not constitute a major threat to amphibians studied, the scarce protection by the NNPA represents a risk to the conservation of herpetofauna facing the other components of global change. © 2016, Asociacion Argentina de Ecologia. All Rights Reserved.","<b>Hidden Markov models for gene sequence classification: Classifying the VSG gene in the Trypanosoma brucei genome<\/b><br><br>The article presents an application of hidden Markov models (HMMs) for pattern recognition on genome sequences. We apply HMM for identifying genes encoding the variant surface glycoprotein (VSG) in the genomes of Trypanosoma brucei (T. brucei) and other African trypanosomes. These are parasitic protozoa causative agents of sleeping sickness and several diseases in domestic and wild animals. These parasites have a peculiar strategy to evade the host’s immune system that consists in periodically changing their predominant cellular surface protein (VSG). The motivation for using patterns recognition methods to identify these genes, instead of traditional homology based ones, is that the levels of sequence identity (amino acid and DNA sequence) amongst these genes is often below of what is considered reliable in these methods. Among pattern recognition approaches, HMM are particularly suitable to tackle this problem because they can handle more naturally the determination of gene edges. We evaluate the performance of the model using different number of states in the Markov model, as well as several performance metrics. The model is applied using public genomic data. Our empirical results show that the VSG genes on T. brucei can be safely identified (high sensitivity and low rate of false positives) using HMM. © 2015, Springer-Verlag London.","<b>Genetic structure of Colletotrichum fructicola associated to apple bitter rot and Glomerella leaf spot in Southern Brazil and Uruguay<\/b><br><br>Colletotrichum fructicola is the main species causing apple bitter rot (ABR) and Glomerella leaf spot (GLS) in southern Brazil, and ABR in Uruguay where GLS remains unnoticed. Thus, this work aimed to determine the genetic structure of C. fructicola isolates of both the countries. A total of 28 out of 31 Brazilian isolates (90.3%) caused typical symptoms of GLS, while only 6 of 25 Uruguayan isolates (24.0%) originating from fruits were able to infect leaves, but causing atypical symptoms. Both populations showed similar levels of Nei's gene diversity (h = 0.088 and 0.079, for Brazilian and Uruguayan populations, respectively), and Bayesian cluster analysis inferred two genetic clusters correlated with the geographical origin of isolates. A principal coordinates analysis scatter plot and an unweighted pair group method with arithmetic mean-based dendrogram also grouped Brazilian and Uruguayan isolates into two groups. By pairwise comparison of nitrate-nonutilizing (nit) mutants with a proposed set of testers, all Uruguayan isolates were grouped into a unique vegetative compatibility group (namely VCG 1), while Brazilian isolates were grouped into four VCGs (VCG 1 to 4). Brazilian and Uruguayan populations of C. fructicola were found to be genetically distinct. Our results suggest that isolates of C. fructicola from Brazil capable of causing GLS and ABR arose independently of those from Uruguay. Possible causes leading to the evolutionary differences between populations are discussed. © 2016 The American Phytopathological Society.","<b>Trophic Dynamics of the Cape Stumpnose (Rhabdosargus holubi, Sparidae) Across Three Adjacent Aquatic Habitats<\/b><br><br>Migratory fish species are major vectors of connectivity among aquatic habitats. In this study, conventional stomach contents and stable isotope methods (δ13C and δ15N) were combined to understand how fish of different sizes feed across contrasting aquatic habitats. The Cape stumpnose Rhabdosargus holubi (Sparidae, Perciformes) was selected as an abundant estuarine-dependent species in the permanently open Kowie system, South Africa. Three different habitats were sampled in the region, namely, river, estuary, and sea. Fish entered the estuary as post-larvae from the marine environment, resided in the estuary and lower part of the river as juveniles, and then returned to the sea as sub-adults. The diet varied among habitats, seasons, and fish sizes. “Stable Isotope Analysis with R” (SIAR) Bayesian mixing models mostly supported the results from the stomach content analyses, but also revealed the importance of some prey (e.g., insects) that were underestimated in the consumed diet. Rhabdosargus holubi δ13C values indicated a clear spatial gradient in the origin of food sources assimilated across the habitats, with increasing δ13C along the freshwater-marine continuum. The δ13C ranges of sources and fish also overlapped within each habitat along this continuum, thus illustrating the fidelity of R. holubi to specific habitats at different life stages. By consuming prey in a particular habitat before migrating, either permanently or temporarily to another habitat, R. holubi participates in allochthonous fluxes among riverine, estuarine, and coastal marine environments, with approximately 7 tonnes of Cape stumpnose productivity being exported from the 142-ha Kowie Estuary to the sea each year. © 2016, Coastal and Estuarine Research Federation.","<b>Genetic parameters for carcass traits and body weight using a Bayesian approach in the Canchim cattle<\/b><br><br>Correlation between genetic parameters and factors such as backfat thickness (BFT), rib eye area (REA), and body weight (BW) were estimated for Canchim beef cattle raised in natural pastures of Brazil. Data from 1648 animals were analyzed using multi-trait (BFT, REA, and BW) animal models by the Bayesian approach. This model included the effects of contemporary group, age, and individual heterozygosity as covariates. In addition, direct additive genetic and random residual effects were also analyzed. Heritability estimated for BFT (0.16), REA (0.50), and BW (0.44) indicated their potential for genetic improvements and response to selection processes. Furthermore, genetic correlations between BW and the remaining traits were high (P > 0.50), suggesting that selection for BW could improve REA and BFT. On the other hand, genetic correlation between BFT and REA was low (P = 0.39 ± 0.17), and included considerable variations, suggesting that these traits can be jointly included as selection criteria without influencing each other. We found that REA and BFT responded to the selection processes, as measured by ultrasound. Therefore, selection for yearling weight results in changes in REA and BFT. © FUNPEC-RP.","<b>Imprints of multiple glacial refugia in the Pyrenees revealed by phylogeography and palaeodistribution modelling of an endemic spider<\/b><br><br>Mediterranean mountain ranges harbour highly endemic biota in islandlike habitats. Their topographic diversity offered the opportunity for mountain species to persist in refugial areas during episodes of major climatic change. We investigate the role of Quaternary climatic oscillations in shaping the demographic history and distribution ranges in the spider Harpactocrates ravastellus, endemic to the Pyrenees. Gene trees and multispecies coalescent analyses on mitochondrial and nuclear DNA sequences unveiled two distinct lineages with a hybrid zone around the northwestern area of the Catalan Pyrenees. The lineages were further supported by morphological differences. Climatic niche-based species distribution models (SDMs) identified two lowland refugia at the western and eastern extremes of the mountain range, which would suggest secondary contact following postglacial expansion of populations from both refugia. Neutrality test and approximate Bayesian computation (ABC) analyses indicated that several local populations underwent severe bottlenecks followed by population expansions, which in combination with the deep population differentiation provided evidence for population survival during glacial periods in microrefugia across the mountain range, in addition to the main Atlantic and Mediterranean (western and eastern) refugia. This study sheds light on the complexities of Quaternary climatic oscillations in building up genetic diversity and local endemicity in the southern Europe mountain ranges. © 2016 John Wiley & Sons Ltd.","<b>Genetic relationships among American donkey populations: Insights into the process of colonization<\/b><br><br>This study presents the first insights into the genetic diversity and structure of the American donkey metapopulation. The primary objectives were to detect the main structural features underlying variability among American donkey populations, identify boundaries between differentiated gene pools, and draw the main colonization pathways since the introduction of donkeys into America in the 15th century. A panel of 14 microsatellite markers was applied for genotyping 350 American donkeys from 13 countries. The genetic structure of this metapopulation was analysed using descriptive statistics and Bayesian model-based methods. These populations were then compared to a database containing information on 476 individuals from 11 European breeds to identify the most likely ancestral donor populations. Results showed the presence of two distinct genetic pools, with confluence of the two in Colombia. The southern pool showed a unique genetic signature subsequent to an older founder event, but lacked any significant influence of modern gene flow from Europe. The northern pool, conversely, may have retained more ancestral polymorphisms and/or have experienced modern gene flow from Spanish breeds. The Andalusian and, to a lesser extent, the Catalan breeds have left a more pronounced footprint in some of the American donkey populations analysed. © 2016 Blackwell Verlag GmbH.","<b>Genetic relationships between Atlantic and Pacific populations of the notothenioid fish Eleginops maclovinus: The footprints of Quaternary glaciations in Patagonia<\/b><br><br>The genetic relationships between the Pacific and the Atlantic populations of marine coastal biota in Southern South America have been analyzed in few studies, most of them relying on a single mitochondrial locus. We analyzed 10 polymorphic microsatellite loci, isolated from a dinucleotide-enriched Eleginops maclovinus genomic library, in a total of 240 individuals (48 from each of 5 sampled sites: 2 Atlantic, 2 Pacific and 1 in Beagle Channel). The results were contrasted against a previous work on the same species with mitochondrial DNA (mtDNA). Observed heterozygosity within localities ranged from 0.85 to 0.88 with the highest overall number of alleles observed at the northernmost locality on the Pacific side (Concepción), but no clear geographic pattern arose from the data. On the other hand, the number of private alleles was negatively correlated with latitude (Spearman's r s test, P=0.017). Among-population variance was low but significant (1.35%; P<0.0001, analysis of molecular variance (AMOVA)) and low genetic differentiation between populations was observed (pairwise F ST values ranged from 0 to 0.021). A Mantel test revealed a significant correlation between geographic distances and F ST (r=0.56, P=0.047). This could be partially accounted by the Atlantic versus Pacific population differentiation detected in three different analyses (STRUCTURE, SAMOVA (Spatial Analysis of MOlecular VAriance) and a population phylogeny). The observed pattern is compatible with a history of separation into two glacial refugia that was better captured by the multilocus microsatellite data than by the mtDNA analysis. © 2016 Macmillan Publishers Limited.","<b>Genotype by environment interaction for tick resistance of Hereford and Braford beef cattle using reaction norm models<\/b><br><br>Background: The cattle tick is a parasite that adversely affects livestock performance in tropical areas. Although countries such as Australia and Brazil have developed genetic evaluations for tick resistance, these evaluations have not considered genotype by environment (G∗E) interactions. Genetic gains could be adversely affected, since breedstock comparisons are environmentally dependent on the presence of G∗E interactions, particularly if residual variability is also heterogeneous across environments. The objective of this study was to infer upon the existence of G∗E interactions for tick resistance of cattle based on various models with different assumptions of genetic and residual variability. Methods: Data were collected by the Delta G Connection Improvement program and included 10,673 records of tick counts on 4363 animals. Twelve models, including three traditional animal models (AM) and nine different hierarchical Bayesian reaction norm models (HBRNM), were investigated. One-step models that jointly estimate environmental covariates and reaction norms and two-step models based on previously estimated environmental covariates were used to infer upon G∗E interactions. Model choice was based on the deviance criterion information. Results: The best-fitting model specified heterogeneous residual variances across 10 subclasses that were bounded by every decile of the contemporary group (CG) estimates of tick count effects. One-step models generally had the highest estimated genetic variances. Heritability estimates were normally higher for HBRNM than for AM. One-step models based on heterogeneous residual variances also usually led to higher heritability estimates. Estimates of repeatability varied along the environmental gradient (ranging from 0.18 to 0.45), which implies that the relative importance of additive and permanent environmental effects for tick resistance is influenced by the environment. Estimated genetic correlations decreased as the tick infestation level increased, with negative correlations between extreme environmental levels, i.e., between more favorable (low infestation) and harsh environments (high infestation). Conclusions: HBRNM can be used to describe the presence of G∗E interactions for tick resistance in Hereford and Braford beef cattle. The preferred model for the genetic evaluation of this population for tick counts in Brazilian climates was a one-step model that considered heteroscedastic residual variance. Reaction norm models are a powerful tool to identify and quantify G∗E interactions and represent a promising alternative for genetic evaluation of tick resistance, since they are expected to lead to greater selection efficiency and genetic progress. © 2016 Mota et al.","<b>Genetic parameters of objectionable fibers and of their associations with fleece traits in Corriedale sheep<\/b><br><br>The aim of this research was to assess the variability and genetic relationships among binary traits denoting the presence or absence of objectionable fibers, namely pigmented (BINPPF), medullated (BINPMED), and kemp fibers (BINPK), and of fleece traits (fiber diameter [FD] and clean fleece weight [CFW]) in Corriedale sheep. Additionally, the total response to selection against objectionable fibers and indirect responses when selecting for fleece traits were evaluated. Fiber records from 679 animals and fleece records from 795 animals obtained from 2 experimental flocks (from 2005 to 2007) were used; the pedigree file included a total of 3,792 animals. Heritability and genetic correlations among the traits were estimated with a multivariate animal model under a Bayesian setting. Heritability estimates (posterior SD) for BINPPF, BINPMED, and BINPK were 0.35 (0.08), 0.37 (0.10), and 0.63 (0.09), respectively; for CFW and FD, estimates were 0.42 (0.09) and 0.43 (0.08), respectively. The genetic correlations between CFW and the 3 types of objectionable fibers were very low (i.e., <0.2). Thus selection for CFW is not expected to affect the number of such fibers in any direction. The same occurred for the genetic correlation between BINPK and FD. Genetic correlations between FD and BINPMED and FD and BINPPF were positive and favorable (0.50 and 0.56, respectively). Selecting for lower FD would decrease the numbers of objectionable fibers. The expected correlated responses in BINPPF, BINPMED, and BINPK when selecting for CFW were −0.03 (0.11), 0.03 (0.11), and −0.05 (0.18), respectively; when selecting for FD, the correlated responses were −0.26 (0.11), −0.26 (0.11), and −0.14 (0.16). Overall, this study reports novel information on genetic parameters for the presence of objectionable fibers and their associations with fleece traits in sheep. Our findings suggest that it could be possible to improve FD while at the same time reducing the content of BINPMED and BINPPF in wool; however, improvements in CFW are likely to have no effect on the numbers of objectionable fibers. © 2016 American Society of Animal Science. All rights reserved.","<b>Automatic calibration of a wave model with an evolutionary Bayesian method<\/b><br><br>Bayesian Inference has been widely applied with success in science and engineering. One of its main uses is the inference of model parameters in order to reconcile model outputs with evidence provided by measures. In this article we propose this application for coastal engineering problems. Specifically, it is proposed to infer the parameters of a numerical wave model used to downscale wave reanalysis data to a coastal site. The proposed method is applied to a case study on the Uruguayan Atlantic coast, where a few month wave measure data series is available and needs to be extended in order to be used on an engineering project. The wave model used is SWAN and the data in deep waters and the wind data were obtained from the ERA-Interim reanalysis. At first, the method was tested with one and two parameters, since in these cases it is possible to compare the obtained results with a plot of the target function. Finally it was used to calibrate four parameters of the wave model and assess the uncertainty introduced by the selection of a set of parameters.","<b>Phylogenetic relationships between tuco-tucos (Ctenomys, Rodentia) of the Corrientes group and the C. pearsoni complex<\/b><br><br>Lineage delimitation is of extreme importance in evolutionary biology and constitutes an essential tool in basic and applied fields including conservation. We addressed the delimitation of two closely related species complexes of South American rodents of the genus Ctenomys (tuco-tucos), namely the C. pearsoni and Corrientes groups, whose relationships have been unclear or conflicting in previous phylogenetic studies. We performed a molecular phylogenetic analysis, increasing the number of representatives of each group and including sequences of three mitochondrial loci (the complete cytochrome b coding sequence, and partial regions of the cytochrome oxydase I gene and the D-loop of the control region). The trees obtained using Bayesian Inference and Maximum Parsimony methods show the Corrientes group and the C. pearsoni complex as reciprocally monophyletic sister clades, with higher values of intergroup compared to intragroup genetic distances. These results indicate that the Corrientes group and the C. pearsoni complex are differentiated lineages. The resulting tree topology is in agreement with a scenario of independent chromosomal rearrangements from the ancestral karyomorph (2n = 70 FN = 84), leading to the considerable chromosomal diversity that characterizes both groups. © SAREM, 2016.","<b>Stable isotopes reveal long-Term fidelity to foraging grounds in the galapagos sea lion zalophus wollebaeki)<\/b><br><br>Most otariids have colony-specific foraging areas during the breeding season, when they behave as central place foragers. However, they may disperse over broad areas after the breeding season and individuals from different colonies may share foraging grounds at that time. Here, stable isotope ratios in the skull bone of adult Galapagos sea lions (Zalophus wollebaeki) were used to assess the long-Term fidelity of both sexes to foraging grounds across the different regions of the Galapagos archipelago. Results indicated that the stable isotope ratios (δ13C and δ15N) of sea lion bone significantly differed among regions of the archipelago, without any significant difference between sexes and with a non significant interaction between sex and region. Moreover, standard ellipses, estimated by Bayesian inference and used as a measure of the isotopic resource use area at the population level, overlapped widely for the sea lions from the southern and central regions, whereas the overlap of the ellipses for sea lions from the central and western regions was small and nonexisting for those from the western and southern regions. These results suggest that males and females from the same region within the archipelago use similar foraging grounds and have similar diets. Furthermore, they indicate that the exchange of adults between regions is limited, thus revealing a certain degree of foraging philopatry at a regional scale within the archipelago. The constraints imposed on males by an expanded reproductive season (~ 6 months), resulting from the weak reproductive synchrony among females, and those imposed on females by a very long lactation period (at least one year but up to three years), may explain the limited mobility of adult Galapagos sea lions of both sexes across the archipelago. © 2016 Drago et al. This is an open access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.","<b>Genetic diversity of the swamp rat in South America: Population expansion after transgressive-regressive marine events in the Late Quaternary<\/b><br><br>We examined the phylogeography of the South American swamp rat Scapteromys tumidus using complete mitochondrial DNA cytochrome b sequences. This species is endemic to the Pampas biome and lives near the coastal plain but also in inland continental areas. The coastal domain of such region experienced a highly dynamic geological history. The inland fraction is part of Precambrian and Mesozoic spills and sedimentary basins, while most of the coastal plain was shaped by transgressive-regressive marine events in the Late Quaternary. As fluctuations in the sea level during this period produced large lateral displacements of the shoreline, originating four barrier-lagoon systems that shaped the present coastline, continental area significantly increased from this time. We hypothesized that the S. tumidus have expanded its range from that period, ramped by its adaptive ability to humid areas. Thus, we examined whether historical Pleistocene events had affected genetic variation of this species along its distributional range. Bayesian phylogenetic analysis and the haplotype network inferred two major genetic clusters along the distribution of S. tumidus. Neutrality tests suggest that populations experienced demographic changes and Bayesian skyline plot evidenced a marked recent demographic expansion pattern, intensified from 350 to 300 kya. Such date coincides with the formation of the lagunar barrier II (dated to ca 325 kya) and the concurrent initial formation of the coastal plain and increase in continental area. © 2015 Deutsche Gesellschaft fü Säugetierkunde.","<b>More on the mesopotamian-yungas disjunction in subtropical and temperate argentina: Bioclimatic distribution models of the harvestman discocyrtus dilatatus (opiliones: Gonyleptidae)<\/b><br><br>In this paper, the potential distribution of the Mesopotamian harvestman Discocyrtus dilatatus Sørensen, 1884 is modeled, and the species’ bioclimatic profile is described. Models were built with the presence-only methods MAXENT and BIOCLIM, using 85 unique records (of which 49 are new) and 11 non-correlated bioclimatic variables as predictors. Both MAXENT and BIOCLIM supported the Mesopotamian-Yungas disjunct pattern observed in D. dilatatus, and confirmed the hypothesis that the sub-xeric Dry Chaco is an effective barrier for the two portions of the range. Similarly to results of previous studies on other Mesopotamian harvestmen, temperature variables proved more relevant than precipitation variables in the final models. In the combined overall score obtained with MAXENT, bc4-temperature seasonality ranked as the most relevant, and only one precipitation variable (bc18-precipitation of warmest quarter, in second place) ranked among the top five. In the Most Limiting Factor analysis, which identifies the relevant variables in a local scale, temperature variables were again more determining than precipitation variables in most of the range. One single variable, bc5-maximal temperature of warmest month, proved critical near the boundaries of the modeled range and the Dry Chaco, suggesting that extremely high temperatures (and not the supposed aridity) are responsible for the 450 km distribution gap. © 2015, Sociedade Brasileira de Zoologia. All rights reserved.","<b>Genetic diversification of local onion populations under different production systems in Uruguay<\/b><br><br>In Uruguay, onion (Allium cepa L.) germplasm is mainly derived from the genetic material introduced by several waves of European immigrants and subsequently multiplied by household farmers, resulting in a wealth of locally adapted populations. This study examined the genetic diversity in a collection of 27 local onion populations and two cultivars derived from them. A total of 843 onion plants were fingerprinted, and 83 inter-simple sequence repeat polymorphic bands were generated. Analysis of molecular variance showed high diversity within the populations (66% of the total variation). Some short-day populations from different geographical origins were grouped together by the unweighted pair-group method using arithmetic averages, principal coordinate analysis and cluster analysis, while the more extensively sampled long-and intermediate-day populations showed a widespread distribution, with no significant subgrouping among them. This weakly structured gene pool is probably the consequence of seed and bulb exchange between farmers and natural inter-pollination. Nevertheless, a Bayesian analysis allowed the distinction of four genetic backgrounds of alleles in the whole collection, and populations were predominately assigned to each genetic background. In addition, mitochondrial variants determining normal (N) pollen fertility or the sterile S or T types were analysed for the same set of plants using specific primers. Most accessions showed a proportion of male-sterile individuals. Cytoplasm type T was the most represented (26 out of 29 accessions), and cytoplasm type S was found in a low proportion of individuals in seven populations. Uruguayan onion germplasm maintains a low degree of genetic differentiation despite the small cultivated area and intense seed exchange, probably due in part to different market purposes based on the growing cycle. © 2014 NIAB.","<b>Molecular phylogeography of canine distemper virus: Geographic origin and global spreading<\/b><br><br>Canine distemper virus (CDV) (Paramyxoviridae-. Morbillivirus) is a worldwide spread virus causing a fatal systemic disease in a broad range of carnivore hosts. In this study we performed Bayesian inferences using 208 full-length hemagglutinin gene nucleotide sequences isolated in 16 countries during 37. years (1975-2011). The estimated time to the most recent common ancestor suggested that current CDV strains emerged in the United States in the 1880s. This ancestor diversified through time into two ancestral clades, the current America 1 lineage that recently spread to Asia, and other ancestral clade that diversified and spread worldwide to originate the remaining eight lineages characterized to date. The spreading of CDV was characterized by several migratory events with posterior local differentiation, and expansion of the virus host range. A significant genetic flow between domestic and wildlife hosts is displayed; being domestic hosts the main viral reservoirs worldwide. This study is an extensive and integrative description of spatio/temporal population dynamics of CDV lineages that provides a novel evolutionary paradigm about the origin and dissemination of the current strains of the virus. © 2015.","<b>Description of the growth curve for Angus pasture-fed cows under extensive systems<\/b><br><br>Growth curves are useful for describing the sigmoid shape of an animal’s weight pattern over time. The purpose of the present study was to describe, for the first time, a complete growth curve for pasture-fed Angus cows in Uruguay and to analyze the major fixed effects that affect the estimation of mature cow weight (CW). A total of 22,743 records from 5,284 cows belonging to 73 herds were used in the statistical analysis. Five nonlinear models (Brody, Gompertz, Von Bertalanffy, logistic, and Richards) and a 3-knot cubic spline function were fitted to model weight change across age. Body weights were recorded at birth, weaning, and 18 mo and as an adult (at 2.3–3.6, 3.6–4.6, 4.6 to 5.6, 5.6 to 6.6, and 6.6 to 8.1 yr). After preliminary analyses, the fixed effects included in the model to estimate CW were BCS, herd, year and month of measurement, and age of dam. The herd and BCS were the major significant sources of variation. The Richards model had the lowest values for Akaike information criterion, Bayesian information criterion, and –2 log likelihood, with the asymptotic weight being 542 kg. The spline function was the model that fitted the data most closely to the observed weights followed by the Richards function. The Richards model gave better predictions of weights from birth to maturity than the other 4 nonlinear models, because these functions were shown to over- or underestimate weights at different ages in this beef cattle data set. The predicted curve showed that cows reach mature CW after 5 yr of age, which is the age commonly assumed as adult weight in beef cattle. Results form this analysis allowed the estimation of CW and rate of maturing and to take knowledge of the shape of growth curve for Angus females in Uruguay under extensive conditions, helping breeders to take selection decisions. In this way, breeders can optimize the management and efficiency for each production system. © 2015 American Society of Animal Science. All rights reserved.","<b>An evolutionary insight into Newcastle disease viruses isolated in Antarctica<\/b><br><br>The disease caused by Newcastle disease virus (NDV) is a severe threat to the poultry industry worldwide. Recently, NDV has been isolated in the Antarctic region. Detailed studies on the mode of evolution of NDV strains isolated worldwide are relevant for our understanding of the evolutionary history of NDV. For this reason, we have performed Bayesian coalescent analysis of NDV strains isolated in Antarctica to study evolutionary rates, population dynamics, and patterns of evolution. Analysis of F protein cleavage-site sequences of NDV isolates from Antarctica suggested that these strains are lentogenic. Strains isolated in Antarctica and genotype I reference strain Ulster/67 diverged from ancestors that existed around 1958. The time of the most recent common ancestor (MRCA) was established to be around 1883 for all class II viruses. A mean rate of evolution of 1.78 × 10-3 substitutions per site per year (s/s/y) was obtained for the F gene sequences of NDV strains examined in this study. A Bayesian skyline plot indicated a decline in NDV population size in the last 25 years. The results are discussed in terms of the possible role of Antarctica in emerging or re-emerging viruses and the evolution of NDV populations worldwide. © 2015, Springer-Verlag Wien.","<b>Phylogenetic analysis of Hepatitis B virus genotypes circulating in different risk groups of Panama, evidence of the introduction of genotype A2 in the country<\/b><br><br>The Hepatitis B Virus (HBV) can cause acute or chronic infection it is also associated with the development of liver cancer, thousands of new infections occur on a yearly basis, and many of these cases are located in certain areas of the Caribbean and Latin America. In these areas, the HBV prevalence is still high which makes this virus a serious public health concern to the entire region. Studies performed in Panama suggest a complex pattern in the distribution of HBV among the country's different risk groups. We use phylogenetic analysis in order to determine which HBV genotypes were circulating in these specific groups; for this we used a fragment of the PreS2/2 region of the HBV genome. Subsequently whole HBV genome sequences were used for Bayesian analysis of phylodynamics and phylogeography. Two main genotypes were found: genotype A (54.5%) and genotype F (45.5%). There was a difference in the distribution of genotypes according to risk groups: 72.9% of high risk groups were associated to genotype A, and 55.0% of samples of genotype F were associated to the low risk group (p<0.002). The Bayesian analysis of phylogeny-traits association revealed a statistically significant geographical association (p<0.0001) with both genotypes and different regions of the country. The Bayesian time of most recent common ancestor analysis (tMRCA) revealed a recent tMRCA for genotype A2 circulating in Panama (1997, 95% HPD: 1986-2005), when it is compared with Panamanian genotype F1c sequences (1930, 95% HPD: 1810-2005). These results suggest a possible change in the distribution of HBV genotypes in Panama and Latin America as a whole. They also serve to encourage the implementation of vaccination programs in high-risk groups, in order to prevent an increase in the number of new HBV cases in Latin America and worldwide. © 2015 Martínez et al. This is an open access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.","<b>Phylogenetic studies of the three rna silencing suppressor genes of south american ctv isolates reveal the circulation of a novel genetic lineage<\/b><br><br>Citrus Tristeza Virus (CTV) is the most economically important virus of citrus worldwide. Genetic diversity and population structure of CTV isolates from all citrus growing areas from Uruguay were analyzed by RT-PCR and cloning of the three RNA silencing suppressor genes (p25, p20 and p23). Bayesian phylogenetic analysis revealed the circulation of three known genotypes (VT, T3, T36) in the country, and the presence of a new genetic lineage composed by isolates from around the world, mainly from South America. Nucleotide and amino acid identity values for this new genetic lineage were both higher than 97% for the three analyzed regions. Due to incongruent phylogenetic relationships, recombination analysis was performed using Genetic Algorithms for Recombination Detection (GARD) and SimPlot software. Recombination events between previously described CTV isolates were detected. High intra-sample variation was found, confirming the co-existence of different genotypes into the same plant. This is the first report describing: (1) the genetic diversity of Uruguayan CTV isolates circulating in the country and (2) the circulation of a novel CTV genetic lineage, highly present in the South American region. This information may provide assistance to develop an effective cross-protection program. © 2015 by the authors; licensee MDPI, Basel, Switzerland.","<b>Status, population trend and genetic structure of South American fur seals, Arctocephalus australis, in southwestern Atlantic waters<\/b><br><br>The South American fur seal (SAFS), Arctocephalus australis, was severely depleted during the 18th century. This work updates information on distribution, abundance, seasonal movements, and genetic structure along the southwestern South Atlantic. Its distribution in Argentine waters includes nineteen rookeries on the Patagonian coast, Isla de los Estados, and islets of the Beagle Channel. Censuses were carried out in rookeries from Isla Escondida to Isla de los Estados from 1996 to 2014. The largest concentrations are found in Chubut, (4,500-15,500 seals) and Isla de los Estados (4,500). Pups were recorded on Isla Escondida, Rasa, and Isla de los Estados. An increasing number of SAFS were recorded in northern Patagonia, including seals marked in Uruguay. Skin samples were taken from southern Brazil to Tierra del Fuego in order to evaluate population structure and demographic trends. Genetic results indicated no population subdivisions (Φst = -0.0292, Fst = -0.00716, P > 0.05). Bayesian skyline plots constructed for the entire data set indicated evidence of rapid population expansion discernible in Nef between the last 20,000 and 17,000 yr. Genetics results and observation data from marked fur seals support the hypothesis of ancient gene flow and a single Atlantic population. Therefore, SAFS should be managed under international and integrated conservation policies. © 2015 Society for Marine Mammalogy.","<b>Genomic prediction for tick resistance in braford and hereford cattle<\/b><br><br>One of the main animal health problems in tropical and subtropical cattle production is the bovine tick, which causes decreased performance, hide devaluation, increased production costs with acaricide treatments, and transmission of infectious diseases. This study investigated the utility of genomic prediction as a tool to select Braford (BO) and Hereford (HH) cattle resistant to ticks. The accuracy and bias of different methods for direct and blended genomic prediction was assessed using 10,673 tick counts obtained from 3,435 BO and 928 HH cattle belonging to the Delta G Connection breeding program. A subset of 2,803 BO and 652 HH samples were genotyped and 41,045 markers remained after quality control. Log transformed records were adjusted by a pedigree repeatability model to estimate variance components, genetic parameters, and breeding values (EBV) and subsequently used to obtain deregressed EBV. Estimated heritability and repeatability for tick counts were 0.19 ± 0.03 and 0.29 ± 0.01, respectively. Data were split into 5 subsets using k-means and random clustering for cross-validation of genomic predictions. Depending on the method, direct genomic value (DGV) prediction accuracies ranged from 0.35 with Bayes least absolute shrinkage and selection operator (LASSO) to 0.39 with BayesB for k-means clustering and between 0.42 with BayesLASSO and 0.45 with BayesC for random clustering. All genomic methods were superior to pedigree BLUP (PBLUP) accuracies of 0.26 for k-means and 0.29 for random groups, with highest accuracy gains obtained with BayesB (39%) for k-means and BayesC (55%) for random groups. Blending of historical phenotypic and pedigree information by different methods further increased DGV accuracies by values between 0.03 and 0.05 for direct prediction methods. However, highest accuracy was observed with single-step genomic BLUP with values of 0.48 for k-means and 0.56, which represent, respectively, 84 and 93% improvement over PBLUP. Observed random clustering cross-validation breedspecific accuracies ranged between 0.29 and 0.36 for HH and between 0.55 and 0.61 for BO, depending on the blending method. These moderately high values for BO demonstrate that genomic predictions could be used as a practical tool to improve genetic resistance to ticks and in the development of resistant lines of this breed. For HH, accuracies are still in the low to moderate side and this breed training population needs to be increased before genomic selection could be reliably applied to improve tick resistance. © 2015 American Society of Animal Science. All rights reserved.","<b>Sexual foraging segregation in South American sea lions increases during the pre-breeding period in the Río de la Plata plume<\/b><br><br>Stable carbon and nitrogen isotopes in skin and bone of South American sea lions from Brazil and Uruguay were analysed to test the hypothesis that trophic overlap between the sexes is lower during the pre-breeding season than throughout the rest of the year. The isotopic values of skin and bone were used to infer the trophic relationships between the sexes during the pre-breeding period and year round, respectively. Prey species were also analysed to establish a baseline necessary for interpreting the stable isotope ratios of skin and bone. Standard ellipse areas, estimated using Bayesian inference in the SIBER routine of the SIAR package in R, suggested that males and females used a wide diversity of foraging strategies throughout the year and that no differences existed between the sexes. However, the diversity of foraging strategies was largely reduced during the pre-breeding period, with all the individuals of each sex adopting similar strategies, but with the two sexes differing considerably in stable isotope values and the ellipse areas of males and females not overlapping at all. Nevertheless, the results revealed a general increase in the consumption of pelagic prey by both sexes during the pre-breeding period. The progressive crowding of individuals in the areas surrounding the breeding rookeries during the pre-breeding period could lead to an increase in the local population density, which could explain the above reported changes. © Inter-Research 2015 · www.int-res.com.","<b>Does the better-than-average effect show that people are overconfident?: Two experiments<\/b><br><br>We conduct two experimental tests of the claim that people are overconfident, using new tests of overplacement that are based on a formal Bayesian model. Our two experiments, on easy quizzes, find that people overplace themselves. More precisely, we find apparently overconfident data that cannot be accounted for by a rational population of expected utility maximizers who care only about money. The finding represents new evidence of overconfidence that is robust to the Bayesian critique offered by Benoît and Dubra (Jean-Pierre Benoît and Juan Dubra (2011). \"Apparent Overconfidence.\" Econometrica, 79, 1591-1625). We discuss possible limitations of our results. © 2014 by the European Economic Association.","<b>Temperament and its heritability in Corriedale and Merino lambs<\/b><br><br>Temperament can be defined as the fearfulness and reactivity of an animal in response to humans and strange, novel or threatening environments. The productive performance of an animal is affected by its temperament, and selection of calm animals might improve their adaptation to the farming environment and handling, as well as improve productivity. The temperament was measured in lambs of two breeds of sheep in Uruguay. The effects of dam's age, type of birth, age of the lamb and contemporary group (CG; lambs belonging to the same year, flock, sex and rearing group) on the temperament of the lambs and the heritability of temperament were estimated with a Bayesian analysis using Gibbs sampling. Overall, 4962 Corriedale lambs and 2952 Merino lambs from 13 farms were tested. Temperament was measured using the isolation box test, isolating a lamb inside the box for 30 s, and recording the vibrations produced by its movements. The average temperament score (±s.e.m.) of the Corriedale lambs was 24.7 (±0.23) and that of the Merino was 36.8 (±0.45). Temperament was not associated with dam's age, type of birth or lamb's age. There were no relevant differences in the agitation score between lambs born in 2010 and 2011. The mean of the distribution of possible values of heritability (±s.d.) was 0.18 (±0.05) for the Corriedale and 0.31 (±0.06) for the Merino. The likelihood of heritability values to be greater than 0.15 exceeded 70% in the Corriedale and 90% in the Merino. The temperament of Merino and Corriedale sheep in Uruguay is moderately heritable. It is not related to dam's age, type of birth or age of the lambs; however, it is affected by some aspect of the CG. © The Animal Consortium 2014.","<b>Bayesian age and growth analysis of the shortfin mako shark Isurus oxyrinchus in the Western South Atlantic Ocean using a flexible model<\/b><br><br>Age and growth estimates of shortfin mako sharks Isurus oxyrinchus from the western South Atlantic Ocean were obtained through the analysis of vertebral sections of 245 specimens (126 females, 116 males and 3 with undetermined sex), ranging in size from 78 to 330 cm fork length (FL), using a flexible growth model (Schnute model) and a Bayesian approach. A significant linear relationship was found between FL and vertebral radius for sexes combined. Marginal increment analyses were inconclusive about periodicity of growth band deposition and an annual periodicity was assumed to make age estimations. Specimens were estimated to be between 0 and 28 years of age. The Schnute model provided a good description of the individual growth for both sexes up to 15 years of age. Shortfin mako growth during the first year of life was 33.9 cm (ICr95% = 19.9–40.8) for females and 30.5 cm (ICr95% = 25.6–35.4) for males. Until approximately 15 years of age, both sexes showed similar growth and reached ~217 cm FL. Sigmoid shaped growth curves obtained for both sexes indicated a change in the growth pattern close to 7 years of age. Inconclusive results about periodicity of growth band deposition in the study area make necessary the application of more robust validation techniques in the future. Meanwhile, a precautionary approach that assumes an annual deposition pattern in the western South Atlantic can be used for the assessment and management of stocks of this species, characterized by low fecundity and late maturity. © 2014, Springer Science+Business Media Dordrecht.","<b>Phylodynamics of major HIV-1 subtype B pandemic clades circuLating in Latin America<\/b><br><br>Objective: To obtain a comprehensive description of the evolutionary and demographic history of major HIV-1 subtype B pandemic (BPANDEMIC) clades circulating in Latin America. Design: A total of 6789 HIV-1 subtype B pol sequences collected from seven different Latin American countries between 1990 and 2011 were combined with BPANDEMIC reference sequences (n=500) from the United States and France. Methods: Major BPANDEMIC clades were identified by maximum likelihood phylogenetic analysis with sequential pruning of ambiguously positioned taxa. Time scale and demographic reconstructions were performed using a Bayesian coalescent-based method. Results: We identified 12 major BPANDEMIC monophyletic lineages mainly composed by Latin American sequences and that together comprise 36% of all subtype B sequences from the region here included. Four clades belong to two major regional lineages that comprise sequences from at least two neighboring countries, whereas the other eight clades were country-specific. The median age of major Latin American BPANDEMIC clades encompass a period of two decades (1968-1988), although most of them probably arose before the early 1980s. All major clades seem to have experienced an initial period of exponential growth, with median epidemic growth rates that range from 0.50 year-1to 0.94 year-1, followed by a recent decline in growth rate. Conclusion: About one-Third of HIV-1 subtype B infections in Latin America originated from the spread of a few BPANDEMIC founder strains probably introduced in the region since the late 1960s. Despite their initial successful dissemination, all major BPANDEMIC clades showed signs of subsequent epidemic stabilization. Copyright © 2015 Wolters Kluwer Health, Inc. All rights reserved.","<b>Endemism and evolution in the littoral woodlouse Halophiloscia Verhoeff, 1908 (Crustacea, Isopoda, Oniscidea) from the Canary Islands: implications for conservation policies<\/b><br><br>1. The woodlouse genus Halophiloscia Verhoeff, 1908 includes littoral halophilous species distributed along the coasts of the Mediterranean Sea, the Atlantic coasts of Europe and northern Africa. The species Halophiloscia couchii has been introduced to the Americas, Hawaii and Australia. The Canary Islands harbour four species, two of which are included in catalogues of endangered species. 2. One mitochondrial and one nuclear gene sequenced from over 100 specimens, including outgroups, were analysed to infer the colonisation and diversification processes of Halophiloscia in the Canary Islands and shed light on their conservation status. 3. The Canary Islands were independently colonised by three distinct Halophiloscia lineages: (i) the endemic species H. rodriguezi, collected in several western Canary Islands, and the troglobiomorphic H. microphthalma from La Palma, (ii) the endemic H. canariensis, found throughout the eastern Canaries, and (iii) the cosmopolitan H. couchii, present on Gran Canaria and Tenerife. 4. Halophiloscia microphthalma is not sister to the extant epigean Halophiloscia species present on the same island, which together with the low levels of genetic variability detected in the species may warrant a higher conservation status than presently given. 5. The mitochondrial marker revealed a fine-scale phylogeographic structure in H. canariensis, and rejected recent sharp declines in its population size, which in addition to its widespread distribution challenge its consideration as an endangered species. 6. The presence of H. couchii in the archipelago is interpreted as a recent introduction. Although no evidence of introgression was detected, more thorough sampling and monitoring of introduced populations are recommended to discard either hybridisation or competitive displacement of native species. © 2014 The Royal Entomological Society.","<b>A phylogeny of the Gochnatieae: Understanding a critically placed tribe in the Compositae<\/b><br><br>Subfamily Gochnatioideae is the sister group of ~96% of the species in Compositae (Asteraceae). It is of particular interest not only because of its position in the phylogeny, but also because, in recent molecular studies, the node it occupies is not strongly supported making difficult any inferences on the direction of character evolution in the family. The recognition of tribe Gochnatieae was one of the results of a comprehensive molecular analysis of the family that showed the traditional circumscription of the Mutisieae to be non-monophyletic. The four genera of Gochnatieae (Cnicothamnus, Cyclolepis, Gochnatia, Richterago) were defined by the presence of apiculate anther appendages and dorsally smooth style branches. Gochnatia, which contained about 70 species, was the largest and most complex genus and in the last decade some of its sections have been moved (or returned) to the rank of genus. This study includes a large selection of potential outgroups and over 60% of all species in the tribe, including all the genera and all but one of the sections of Gochnatia, to examine evolutionary relationships among the taxa. Both cpDNA and nrDNA were used in a phylogenetic analysis using parsimony, likelihood, and Bayesian approaches. The results suggest a non-monophyletic Gochnatia that is here resolved by the recognition of segregate genera. Morphological characters support these new genera and allow the adoption of a new classification for the Gochnatieae. A biogeographic analysis shows a possible southern South American/Andean origin followed by movement in three directions: into the Central Andes, into central and northern Brazil, and into Mexico and the Caribbean. The dating analysis gives an age of the split of the core Gochnatieae from the Wunderlichieae-Cyclolepis clade, and hence the age of the tribe, of 36-45 Ma and an age of 23-25 Ma for the first split within the core Gochnatieae (Andean vs. Brazil-Mexico-Caribbean). Cnicothamnus remains in Gochnatieae but Cyclolepis is designated incertae sedis. © International Association for Plant Taxonomy (IAPT) 2014.","<b>Spatiotemporal dynamics of the HIV-1 subtype G epidemic in West and Central Africa<\/b><br><br>The human immunodeficiency virus type 1 (HIV-1) subtype G is the second most prevalent HIV-1 clade in West Africa, accounting for nearly 30% of infections in the region. There is no information about the spatiotemporal dynamics of dissemination of this HIV-1 clade in Africa. To this end, we analyzed a total of 305 HIV-1 subtype G pol sequences isolated from 11 different countries from West and Central Africa over a period of 20 years (1992 to 2011). Evolutionary, phylogeographic and demographic parameters were jointly estimated from sequence data using a Bayesian coalescentbased method. Our analyses indicate that subtype G most probably emerged in Central Africa in 1968 (1956-1976). From Central Africa, the virus was disseminated to West and West Central Africa at multiple times from the middle 1970s onwards. Two subtype G strains probably introduced into Nigeria and Togo between the middle and the late 1970s were disseminated locally and to neighboring countries, leading to the origin of two major western African clades (GWA-I and G WAII). Subtype G clades circulating in western and central African regions displayed an initial phase of exponential growth followed by a decline in growth rate since the early/middle 1990s; but the mean epidemic growth rate of GWA-I (0.75 year-1) and GWA-II (0.95 year-1) clades was about two times higher than that estimated for central African lineages (0.47 year-1). Notably, the overall evolutionary and demographic history of GWA-I and GWA-II clades was very similar to that estimated for the CRF06-cpx clade circulating in the same region. These results support the notion that the spatiotemporal dissemination dynamics of major HIV-1 clades circulating in western Africa have probably been shaped by the same ecological factors. © 2011 Delatorre et al.","<b>Climate change and the distribution of neotropical red-bellied toads (Melanophryniscus, Anura, Amphibia): How to prioritize species and populations?<\/b><br><br>We used species distribution modeling to investigate the potential effects of climate change on 24 species of Neotropical anurans of the genus Melanophryniscus. These toads are small, have limited mobility, and a high percentage are endangered or present restricted geographical distributions. We looked at the changes in the size of suitable climatic regions and in the numbers of known occurrence sites within the distribution limits of all species. We used the MaxEnt algorithm to project current and future suitable climatic areas (a consensus of IPCC scenarios A2a and B2a for 2020 and 2080) for each species. 40% of the species may lose over 50% of their potential distribution area by 2080, whereas 28% of species may lose less than 10%. Four species had over 40% of the currently known occurrence sites outside the predicted 2080 areas. The effect of climate change (decrease in climatic suitable areas) did not differ according to the present distribution area, major habitat type or phylogenetic group of the studied species. We used the estimated decrease in specific suitable climatic range to set a conservation priority rank for Melanophryniscus species. Four species were set to high conservation priority: M. montevidensis, (100% of its original suitable range and all known occurrence points potentially lost by 2080), M. sp.2, M. cambaraensis, and M. tumifrons. Three species (M. spectabilis, M. stelzneri, and M. sp.3) were set between high to intermediate priority (more than 60% decrease in area predicted by 2080); nine species were ranked as intermediate priority, while eight species were ranked as low conservation priority. We suggest that monitoring and conservation actions should be focused primarily on those species and populations that are likely to lose the largest area of suitable climate and the largest number of known populations in the short-term. © 2014 Zank et al.","<b>Temporal dynamics of norovirus GII.4 variants in Brazil between 2004 and 2012<\/b><br><br>Noroviruses (NoVs) are the major cause of acute gastroenteritis outbreaks, and, despite a wide genetic diversity, genotype II.4 is the most prevalent strain worldwide. Mutations and homologous recombination have been proposed as mechanisms driving the epochal evolution of the GII.4, with the emergence of new variants in 1-3-year intervals causing global epidemics. There are no data reporting the dynamics of GII.4 variants along a specific period in Brazil. Therefore, to improve the understanding of the comportment of these variants in the country, the aim of this study was to evaluate the circulation of NoV GII.4 variants during a 9-year period in 3 out of 5 Brazilian regions. A total of 147 samples were sequenced, and a phylogenetic analysis of subdomain P2 demonstrated the circulation of six GII.4 variants, Asia-2003, Hunter-2004, Den Haag-2006b, Yerseke-2006a, New Orleans-2009, and Sydney-2012, during this period. The most prevalent variant was Den Haag-2006b, circulating in different Brazilian regions from 2006 to 2011. A Bayesian coalescent analysis was used to calculate the mean evolutionary rate of subdomain P2 as 7.3×10-3 (5.85×10-3-8.82×10-3) subst./site/year. These analyses also demonstrated that clade Den Haag-2006b experienced a rapid expansion in 2005 and another in 2008 after a period of decay. The evaluation of the temporal dynamics of NoV GII.4 in Brazil revealed a similar pattern, with few exceptions, to the worldwide observation. These data highlight the importance of surveillance for monitoring the emergence of new strains of NoV GII.4 and its impact on cases of acute gastroenteritis. © 2014 Fioretti et al.","<b>Methods for genomic evaluation of a relatively small genotyped dairy population and effect of genotyped cow information in multiparity analyses<\/b><br><br>Methods for genomic prediction were evaluated for an Israeli Holstein dairy population of 713,686 cows and 1,305 progeny-tested bulls with genotypes. Inclusion of genotypes of 343 elite cows in an evaluation method that considers pedigree, phenotypes, and genotypes simultaneously was also evaluated. Two data sets were available: a complete data set with production records from 1985 through 2011, and a reduced data set with records after 2006 deleted. For each production trait, a multitrait animal model was used to compute traditional genetic evaluations for parities 1 through 3 as separate traits. Evaluations were calculated for the reduced and complete data sets. The evaluations from the reduced data set were used to calculate parent average for validation bulls, which was the benchmark for comparing gain in predictive ability from genomics. Genomic predictions for bulls in 2006 were calculated using a Bayesian regression method (BayesC), genomic BLUP (GBLUP), single-step GBLUP (ssGBLUP), and weighted ssGBLUP (WssGBLUP). Predictions using BayesC and GBLUP were calculated either with or without an index that included parent average. Genomic predictions that included elite cow genotypes were calculated using ssGBLUP and WssGBLUP. Predictive ability was assessed by coefficients of determination (R2) and regressions of predictions of 135 validation bulls with no daughters in 2006 on deregressed evaluations of those bulls in 2011. A reduction in R2 and regression coefficients was observed from parities 1 through 3. Fat and protein yields had the lowest R2 for all the methods. On average, R2 was lowest for parent averages, followed by GBLUP, BayesC, ssGBLUP, and WssGBLUP. For some traits, R2 for direct genomic values from BayesC and GBLUP were lower than those for parent averages. Genomic estimated breeding values using ssGBLUP were the least biased, and this method appears to be a suitable tool for genomic evaluation of a small genotyped population, as it automatically accounts for parental index, allows for inclusion of female genomic information without preadjustments in evaluations, and uses the same model as in traditional evaluations. Weighted ssGBLUP has the potential for higher evaluation accuracy. © 2014 American Dairy Science Association.","<b>Thermal tolerances in rodents: Species that evolved in cold climates exhibit a wider thermoneutral zone<\/b><br><br>Background: Thermal constraints are often invoked to explain animal distributions. Maximum temperatures are less variable in different biomes around the globe than are minimum temperatures. Considerable information is available for mammals about basal metabolic rate and thermal conductance.\nAims: Evaluate the correlation of lower critical temperature (TLC), upper critical temperature (TUC) or TNZ breadth (TUC - TLC TNZb) with three ambient temperatures in rodent species.\nHypotheses: TLC, TUC and TNZb should be adjusted by selective processes to the ambient temperature that is most usually experienced by mammal species. TNZb should be greater in species inhabiting colder habitats.\nMethods: We used TLC, TUC data from published studies of 85 species of rodents. We determined the average annual mean, minimum and maximum temperatures across the distribution of each species. Then, using standard least squares regression with body mass as a covariate, we determined the statistical relationships between the physiological variables and the temperatures. We evaluated the effect of phylogeny using a Bayesian Phylogenetic Mixed Model in addition to Bayesian Model Averaging.\nResults: Ambient temperatures correlate positively with TLC and TUC, and negatively with TNZb. Species that evolved in cold climates exhibited a greater mass-independent TNZb than species from warmer climates. Species that evolved in cold climates exhibited lower TLC and TUC than species from warmer climates. Phylogenetic as well as conventional statistics indicated that there are thermoregulatory constraints across geographic gradients. © 2014 Francisco Bozinovic.","<b>Diversification in the south american pampas: The genetic and morphological variation of the widespread petunia axillaris complex (solanaceae)<\/b><br><br>Understanding the spatiotemporal distribution of genetic variation and the ways in which this distribution is connected to the ecological context of natural populations is fundamental for understanding the nature and mode of intraspecific and, ultimately, interspecific differentiation. The Petunia axillaris complex is endemic to the grasslands of southern South America and includes three subspecies: P. a. axillaris, P. a. parodii and P. a. subandina. These subspecies are traditionally delimited based on both geography and floral morphology, although the latter is highly variable. Here, we determined the patterns of genetic (nuclear and cpDNA), morphological and ecological (bioclimatic) variation of a large number of P. axillaris populations and found that they are mostly coincident with subspecies delimitation. The nuclear data suggest that the subspecies are likely independent evolutionary units, and their morphological differences may be associated with local adaptations to diverse climatic and/or edaphic conditions and population isolation. The demographic dynamics over time estimated by skyline plot analyses showed different patterns for each subspecies in the last 100 000 years, which is compatible with a divergence time between 35 000 and 107 000 years ago between P. a. axillaris and P. a. parodii, as estimated with the IMa program. Coalescent simulation tests using Approximate Bayesian Computation do not support previous suggestions of extensive gene flow between P. a. axillaris and P. a. parodii in their contact zone. © 2013 John Wiley & Sons Ltd.","<b>Comparative population genetics and evolutionary history of two commonly misidentified billfishes of management and conservation concern<\/b><br><br>Background: Misidentifications between exploited species may lead to inaccuracies in population assessments, with potentially irreversible conservation ramifications if overexploitation of either species is occurring. A notable showcase is provided by the realization that the roundscale spearfish (Tetrapturus georgii), a recently validated species, has been historically misidentified as the morphologically very similar and severely overfished white marlin (Kajikia albida) (IUCN listing: Vulnerable). In effect, no information exists on the population status and evolutionary history of the enigmatic roundscale spearfish, a large, highly vagile and broadly distributed pelagic species. We provide the first population genetic evaluation of the roundscale spearfish, utilizing nuclear microsatellite and mitochondrial DNA sequence markers. Furthermore, we re-evaluated existing white marlin mitochondrial genetic data and present our findings in a comparative context to the roundscale spearfish. Results: Microsatellite and mitochondrial (control region) DNA markers provided mixed evidence for roundscale spearfish population differentiation between the western north and south Atlantic regions, depending on marker-statistical analysis combination used. Mitochondrial DNA analyses provided strong signals of historical population growth for both white marlin and roundscale spearfish, but higher genetic diversity and effective female population size (1.5-1.9X) for white marlin. Conclusions: The equivocal indications of roundscale spearfish population structure, combined with a smaller effective female population size compared to the white marlin, already a species of concern, suggests that a species-specific and precautionary management strategy recognizing two management units is prudent for this newly validated billfish. © 2014 Bernard et al.","<b>Genetic diversity and population structure of the New World screwworm fly from the Amazon region of Brazil<\/b><br><br>Cochliomyia hominivorax (Coquerel) is a myiasis fly that causes economic losses to livestock farmers in warmer American regions. Previous studies of this pest had found population structure at north and south of the Amazon Basin, which was considered to be a barrier to dispersal. The present study analyzed three mitochondrial DNA (mtDNA) markers and eight nuclear microsatellite loci to investigate for the first time the genetic diversity and population structure across the Brazilian Amazon region (Amazonia). Both mtDNA and microsatellite data supported the existence of much diversity and significant population structure among nine regional populations of C. hominivorax, which was found to be surprisingly common in Amazonia. Forty-six mtDNA haplotypes were identified, of which 39 were novel and seven had previously been found only at south of Amazonia. Seventy microsatellite alleles were identified by size, moderate to high values of heterozygosity were discovered in all regions, and a Bayesian clustering analysis identified four genetic groups that were not geographically distributed. Reproductive compatibility was also investigated by laboratory crossing, but no evidence of hybrid dysgenesis was found between an Amazonian colony and one each of from Northeast and Southeast Brazil. The results have important implications for area-wide control by the Sterile Insect Technique. © 2014 International Atomic Energy Agency 2014.","<b>Applying spatial analysis of genetic and environmental data to predict connection corridors to the New World screwworm populations in South America<\/b><br><br>The myiasis causing New World screwworm (NWS) fly is responsible for substantial losses to livestock breeders in the Americas. Due to the negative impact of the NWS fly in animal health, expansion of successful NWS fly eradication programmes is under discussion. However, the effects of geography and environmental diversity on NWS population structure and migration patterns need to be assessed before any political decision is made to implement such a programme. We present a GIS tool to construct potential connection corridors among sampling localities based on genetic and environmental data. We integrate, through a home-made python script, a friction raster based on a Maxent niche model and the pairwise ΦST statistic. Among 38 NWS fly sampling localities from South America, we find a high population connectivity among the sampling localities from the south of the Amazon region. The region along the Atlantic Ocean was identified as the most probable migration corridor between the north (NAG) and the south (SAG) of the Amazon region. The approach highlighted previously undetected population structure within NAG showing low to medium connectivity through the Andes, correlating with current understanding of NWS fly migration in South America. Also, the approach is flexible, allowing future research to incorporate other niche simulations and genetic differentiation metrics. With this flexibility, the tool could become part of any AW-IPM by helping to target regions for control. © 2014 International Atomic Energy Agency 2014.","<b>A multilocus perspective on the phylogenetic relationships and diversification of rodents of the tribe Abrotrichini (Cricetidae: Sigmodontinae)<\/b><br><br>Abrotrichini is a recently defined and diagnosed tribe of Sigmodontinae with a complex taxonomy. Abrotrichine genera, Abrothrix (including Chroeomys), Chelemys, Geoxus, Notiomys and Pearsonomys, are mostly distributed in the central and southern Andes and adjacent lowlands and show terrestrial and fossorial habits. Recent studies have evidenced some incongruence between current taxonomy and abrotrichine phylogeny, such as the polyphyly of Chelemys and paraphyly of Geoxus respect to Pearsonomys. We used DNA sequence data of six loci (one mitochondrial and five nuclear) to resolve the relationships within the tribe. Independent and combined analyses of these loci were carried out using parsimony, maximum likelihood and Bayesian inference. Estimates of divergence time of the main lineages of abrotrichines were calculated with a molecular clock using as calibration, a fossil recently found. The concatenated data set increased the resolution and defined the relationships within the tribe. Our phylogenetic analyses corroborate that Abrothrix is sister of a clade containing the long-clawed abrotrichines. We recovered two main clades within Abrothrix that match morphologic variation and geographic distribution of its species. In addition, we corroborated the lack of monophyly of Chelemys and the lack of monophyly of Geoxus. We discuss different taxonomic scenarios to abrotrichine classification reflects the phylogenetic relationships obtained in this study. Our molecular clock estimated the Abrotrichini crown age to be around the early Pliocene (4.4 Ma) and suggest that the tribe diversified over a short period of time. © 2014 Royal Swedish Academy of Sciences.","<b>Single Step, a general approach for genomic selection<\/b><br><br>Genomic evaluation methods assume that the reference population is genotyped and phenotyped. This is most often false and the generation of pseudo-phenotypes is uncertain and inaccurate. However, markers obey transmission rules and therefore the covariances of marker genotypes across individuals can be modelled using pedigree relationships. Based on this, an extension of the genomic relationship matrix can be constructed in which genomic relationships are propagated to all individuals, resulting in a combined relationship matrix, which can be used in a BLUP procedure called the Single Step Genomic BLUP. This procedure provides so far the most comprehensive option for genomic evaluation. Several extensions, options and details are described: compatibility of genomic and pedigree relationships, Bayesian regressions, multiple trait models, computational aspects, etc. Many details scattered through a series of papers are put together into this paper. © 2014 Elsevier B.V.","<b>Functional diversification within a predatory species<\/b><br><br>Ecological speciation is well-known from adaptive radiations in cichlid fishes inhabiting lentic ecosystems throughout the African rift valley and Central America. Here, we investigate the ecological and morphological diversification of a recently discovered lotic predatory Neotropical cichlid species flock in subtropical South America. We document morphological and functional diversification using geometric morphometrics, stable C and N isotopes, stomach contents and character evolution. This species flock displays species-specific diets and skull and pharyngeal jaw morphology. Moreover, this lineage appears to have independently evolved away from piscivory multiple times and derived forms are highly specialized morphologically and functionally relative to ancestral states. Ecological speciation played a fundamental role in this radiation and our data reveal novel conditions of ecological speciation including a species flock that evolved: 1) in a piscivorous lineage, 2) under lotic conditions and 3) with pronounced morphological novelties, including hypertrophied lips that appear to have evolved rapidly. © 2013 Burress et al.","<b>New records and distribution modeling of Gryne orensis (Sørensen) (Opiliones: Cosmetidae) support the Mesopotamian-Yungas disjunction in subtropical Argentina<\/b><br><br>The presence of Gryne orensis (Sørensen) (Opiliones: Cosmetidae) in a Yungas locality (northwestern Argentina) is reported for the first time, providing new evidence for the Mesopotamian-Yungas disjunct pattern. Combining a total of 19 new Mesopotamian records with previous, reliable citations from the literature, a dataset of 45 points was used to model the potential distribution of the species, using the presence-only methods BIOCLIM and MAXENT. Models supported the existence of a distributional gap across the Semiarid Chaco. The imprecise literature record from \"El Impenetrable\", province of Chaco, is assigned to three tentative locations to evaluate if models are affected by their inclusion; in all cases, the disjunction was maintained. It was thereby estimated that the actual record might have originated in a site closer to the Humid Chaco and/or associated to streams. This paper also provides a statement of the bioclimatic profile and identification of major environmental constraints that define the range of G. orensis. Copyright © 2013 Magnolia Press.","<b>Bias in diet determination: Incorporating traditional methods in Bayesian mixing models<\/b><br><br>There are not \"universal methods\" to determine diet composition of predators. Most traditional methods are biased because of their reliance on differential digestibility and the recovery of hard items. By relying on assimilated food, stable isotope and Bayesian mixing models (SIMMs) resolve many biases of traditional methods. SIMMs can incorporate prior information (i.e. proportional diet composition) that may improve the precision in the estimated dietary composition. However few studies have assessed the performance of traditional methods and SIMMs with and without informative priors to study the predators' diets. Here we compare the diet compositions of the South American fur seal and sea lions obtained by scats analysis and by SIMMs-UP (uninformative priors) and assess whether informative priors (SIMMs-IP) from the scat analysis improved the estimated diet composition compared to SIMMs-UP. According to the SIMM-UP, while pelagic species dominated the fur seal's diet the sea lion's did not have a clear dominance of any prey. In contrast, SIMM-IP's diets compositions were dominated by the same preys as in scat analyses. When prior information influenced SIMMs' estimates, incorporating informative priors improved the precision in the estimated diet composition at the risk of inducing biases in the estimates. If preys isotopic data allow discriminating preys' contributions to diets, informative priors should lead to more precise but unbiased estimated diet composition. Just as estimates of diet composition obtained from traditional methods are critically interpreted because of their biases, care must be exercised when interpreting diet composition obtained by SIMMs-IP. The best approach to obtain a near-complete view of predators' diet composition should involve the simultaneous consideration of different sources of partial evidence (traditional methods, SIMM-UP and SIMM-IP) in the light of natural history of the predator species so as to reliably ascertain and weight the information yielded by each method. © 2013 Franco-Trecu et al.","<b>The Phylogeographic History of the New World Screwworm Fly, Inferred by Approximate Bayesian Computation Analysis<\/b><br><br>Insect pest phylogeography might be shaped both by biogeographic events and by human influence. Here, we conducted an approximate Bayesian computation (ABC) analysis to investigate the phylogeography of the New World screwworm fly, Cochliomyia hominivorax, with the aim of understanding its population history and its order and time of divergence. Our ABC analysis supports that populations spread from North to South in the Americas, in at least two different moments. The first split occurred between the North/Central American and South American populations in the end of the Last Glacial Maximum (15,300-19,000 YBP). The second split occurred between the North and South Amazonian populations in the transition between the Pleistocene and the Holocene eras (9,100-11,000 YBP). The species also experienced population expansion. Phylogenetic analysis likewise suggests this north to south colonization and Maxent models suggest an increase in the number of suitable areas in South America from the past to present. We found that the phylogeographic patterns observed in C. hominivorax cannot be explained only by climatic oscillations and can be connected to host population histories. Interestingly we found these patterns are very coincident with general patterns of ancient human movements in the Americas, suggesting that humans might have played a crucial role in shaping the distribution and population structure of this insect pest. This work presents the first hypothesis test regarding the processes that shaped the current phylogeographic structure of C. hominivorax and represents an alternate perspective on investigating the problem of insect pests. © 2013 Fresia et al.","<b>Estimates of genetic parameters for worm resistance, wool and growth traits in Merino sheep of Uruguay<\/b><br><br>The genotype of an individual and the environment as the maternal ability of its dam have substantial effects on the phenotype expression of many production traits. The aim of the present study was to estimate the (co)variance components for worm resistance, wool and growth traits in Merino sheep, testing the importance of maternal effects and to determine the most appropriate model for each trait. The traits analyzed were Greasy Fleece Weight (GFW), Clean Fleece Weight (CFW), average Fibre Diameter (FD), Coefficient of Variation of FD (CVFD), Staple Length (SL), Comfort Factor (CF30), Weaning Weight (WWT), Yearling Body Weight (YWT) and Faecal worm Egg Count (FEC). The data were recorded during a 15-year period from 1995 to 2010, from Uruguayan Merino stud flocks. A Bayesian analysis was performed to estimate (co)variance components and genetic parameters. By ignoring or including maternal genetic or environmental effects, five different univariate models were fitted in order to determine the most effective for each trait. For CVFD and YWT, the model fitting the data best included direct additive effects as the only significant random source of variation. For GFW, CFW, FD, SL and CF30 the most appropriate model included direct-maternal covariance; while for FEC included maternal genetics effects with a zero direct-maternal covariance. The most suitable model for WWT included correlated maternal genetic plus maternal permanent environmental effects. The estimates of direct heritability were moderate to high and ranged from 0.15 for log transformed FEC to 0.74 for FD. Most of the direct additive genetic correlation (rg) estimations were in the expected range for Merino breed. However, the estimate of rg between FEC and FD was unfavourable (-0.18±0.03). In conclusion, there is considerable genetic variation in the traits analyzed, indicating the potential to make genetic progress on these traits. This study showed that maternal effects are influencing most of traits analyzed, thus these effects should be considered in Uruguayan Merino breeding programs; since the implementation of an appropriate model of analysis is critical to obtain accurate estimates. © 2013 Elsevier B.V.","<b>Tracing the Origin and Northward Dissemination Dynamics of HIV-1 Subtype C in Brazil<\/b><br><br>Previous studies indicate that the HIV-1 subtype C epidemic in southern Brazil was initiated by the introduction of a single founder strain probably originating from east Africa. However, the exact country of origin of such a founder strain as well as the origin of the subtype C viruses detected outside the Brazilian southern region remains unknown. HIV-1 subtype C pol sequences isolated in the southern, southeastern and central-western Brazilian regions (n = 209) were compared with a large number (n ~ 2,000) of subtype C pol sequences of African origin. Maximum-likelihood analyses revealed that most HIV-1 subtype C Brazilian sequences branched in a single monophyletic clade (CBR-I), nested within a larger monophyletic lineage characteristic of east Africa. Bayesian analyses indicate that the CBR-I clade most probably originated in Burundi and was introduced into the Paraná state (southern region) around the middle 1970s, after which it rapidly disseminated to neighboring regions. The states of Paraná and Santa Catarina have been the most important hubs of subtype C dissemination, and routine travel and spatial accessibility seems to have been the major driving forces of this process. Five additional introductions of HIV-1 subtype C strains probably originated in eastern (n = 2), southern (n = 2) and central (n = 1) African countries were detected in the Rio de Janeiro state (southeastern region). These results indicate a continuous influx of HIV-1 subtype C strains of African origin into Brazil and also unveil the existence of unrecognized transmission networks linking this country to east Africa. © 2013 Delatorre et al.","<b>Quaternary range and demographic expansion of Liolaemus darwinii (Squamata: Liolaemidae) in the Monte Desert of Central Argentina using Bayesian phylogeography and ecological niche modelling<\/b><br><br>Until recently, most phylogeographic approaches have been unable to distinguish between demographic and range expansion processes, making it difficult to test for the possibility of range expansion without population growth and vice versa. In this study, we applied a Bayesian phylogeographic approach to reconstruct both demographic and range expansion in the lizard Liolaemus darwinii of the Monte Desert in Central Argentina, during the Late Quaternary. Based on analysis of 14 anonymous nuclear loci and the cytochrome b mitochondrial DNA gene, we detected signals of demographic expansion starting at ~55 ka based on Bayesian Skyline and Skyride Plots. In contrast, Bayesian relaxed models of spatial diffusion suggested that range expansion occurred only between ~95 and 55 ka, and more recently, diffusion rates were very low during demographic expansion. The possibility of population growth without substantial range expansion could account for the shared patterns of demographic expansion during the Last Glacial Maxima (OIS 2 and 4) in fish, small mammals and other lizards of the Monte Desert. We found substantial variation in diffusion rates over time, and very high rates during the range expansion phase, consistent with a rapidly advancing expansion front towards the southeast shown by palaeo-distribution models. Furthermore, the estimated diffusion rates are congruent with observed dispersal rates of lizards in field conditions and therefore provide additional confidence to the temporal scale of inferred phylogeographic patterns. Our study highlights how the integration of phylogeography with palaeo-distribution models can shed light on both demographic and range expansion processes and their potential causes. © 2013 John Wiley & Sons Ltd.","<b>Molecular characterisation of hepatitis B virus in the resident Chinese population in Panama City<\/b><br><br>Despite the effectiveness of current hepatitis B virus (HBV) vaccines, it is estimated that 350 million individuals suffer from chronic HBV infection and more than 50% of these affected individuals live on the Asian continent. Panama is a country with a great diversity of foreign groups; the Chinese community is a large example of this phenomenon. There is an urgent need to perform studies that evaluate the prevalence and the genetic diversity of HBV in this community. This study aimed to evaluate the prevalence of HBV and its genotypes and mutant variants in the Chinese population residing in Panama. In total, 320 subjects were enrolled in the study. Forty-two subjects (13.1%) were positive for HBsAg and HBV-DNA from 18 subjects revealed the presence of genotypes B2 and C1. Secondary mutations associated with drug resistance at positions rtV207L and rtN239T of the reverse transcriptase gene were identified. Additionally, the mutation pair A1762T/G1764A was found in three samples and the mutation G1896A was detected in an HBeAg-negative subject. In conclusion, to our knowledge, this is the first study to report high HBV prevalence rates in resident ethnic Chinese in Central America and the presence of genotypes B2 and C1 in this region.","<b>Evolution of optimal codon choices in the family Enterobacteriaceae<\/b><br><br>The Enterobacteriaceae are a large family of Proteobacteria that include many well-known prokaryotic genera, such as Escherichia, Yersinia and Salmonella. The main ideas of synonymous codon usage (CU) evolution and translational selection have been deeply influenced by studies with these bacterial groups. In this work we report the analysis of the CU pattern of completely sequenced bacterial genomes that belong to the Enterobacteriaceae. The effect of selection in translation acting at the levels of speed and accuracy, and phylogenetic trends within this group are described. Preferred (optimal) codons were identified. The evolutionary dynamics of these codons were studied and following a Bayesian approach these preferences were traced back to the common ancestor of the family. We found that there is some level of variation in selection among the analysed micro-organisms that is probably associated with lineage-specific trends. The codon bias was largely conserved across the evolutionary time of the family in highly expressed genes and protein conserved regions, suggesting a major role of negative selection. In this sense, the results support the idea that the extant CU bias is finely tuned over the ancestral well-conserved pool of tRNAs. © 2013 SGM.","<b>Fine-Scale Population Structure of Blue Whale Wintering Aggregations in the Gulf of California<\/b><br><br>Population differentiation in environments without well-defined geographical barriers represents a challenge for wildlife management. Based on a comprehensive database of individual sighting records (1988-2009) of blue whales from the winter/calving Gulf of California, we assessed the fine-scale genetic and spatial structure of the population using individual-based approaches. Skin samples of 187 individuals were analyzed for nine microsatellite loci. A single population with no divergence among years and months and no isolation by distance (Rxy = 0.1-0.001, p>0.05) were found. We ran two Bayesian clustering methods using Structure and Geneland softwares in two different ways: 1) a general analysis including all individuals in which a single cluster was identified with both softwares; 2) a specific analysis of females only in which two main clusters (Loreto Bay and northern areas, and San Jose-La Paz Bay area) were revealed by Geneland program. This study provides information indicating that blue whales wintering in the Gulf of California are part of a single population unit and showed a fine-scale structure among females, possibly associated with their high site fidelity, particularly when attending calves. It is likely that the loss of genetic variation is minimized by male mediated gene flow, which may reduce the genetic drift effect. Opportunities for kin selection may also influence calf survival and, in consequence, have a positive impact on population demography in this small and endangered population. © 2013 Costa-Urrutia et al.","<b>Dating an impressive Neotropical radiation: Molecular time estimates for the Sigmodontinae (Rodentia) provide insights into its historical biogeography<\/b><br><br>With about 400 living species and 82 genera, rodents of the subfamily Sigmodontinae comprise one of the most diverse and more broadly distributed Neotropical mammalian clades. There has been much debate on the origin of the lineage or the lineages of sigmodontines that entered South America, the timing of entrance and different aspects of further diversification within South America. The ages of divergence of the main lineages and the crown age of the subfamily were estimated by using sequences of the interphotoreceptor retinoid binding protein and cytochrome b genes for a dense sigmodontine and muroid sampling. Bayesian inference using three fossil calibration points and a relaxed molecular clock estimated a middle Miocene origin for Sigmodontinae (∼12. Ma), with most tribes diversifying throughout the Late Miocene (6.9-9.4. Ma). These estimates together results of analyses of ancestral area reconstructions suggest a distribution for the most recent common ancestor of Sigmodontinae in Central-South America and a South American distribution for the most recent common ancestor of Oryzomyalia. © 2012 Elsevier Inc.","<b>Bayesian coalescent analysis of pandemic H1N1 influenza A virus circulating in the South American region<\/b><br><br>The first influenza pandemic of this century was declared in April of 2009, with the emergence of a novel H1N1 influenza A virus strain (H1N1pdm). Understanding the evolution of H1N1pdm populations within the South American region is essential for studying global diversification, emergence, resistance and vaccine efficacy. In order to gain insight into these matters, we have performed a Bayesian coalescent Markov Chain Monte Carlo analysis of hemagglutinin (HA) and neuraminidase (NA) gene sequences of all available and comparable HA and NA sequences obtained from H1N1pdm IAV circulating in the South American region. High evolutionary rates and fast population growths characterize the population dynamics of H1N1pdm strains in this region of the world. A significant contribution of first codon position to the mean evolutionary rate was found for both genes studied, revealing a high contribution of non-synonymous substitutions to the mean substitution rate. In the 178. days period covered by these studies, substitutions in all HA epitope regions can be observed. HA substitutions D239G/N and Q310H have been observed only in Brazilian patients. While substitution D239G/N is not particularly associated to a specific genetic lineage, all strains bearing substitution Q310H were assigned to clade 6, suggesting a founder effect. None of the substitutions found in the NA proteins of H1N1pdm strains isolated in South America appears sufficiently close to affect the drug binding pocket for the three NA inhibitor antivirals tested. A more detailed analysis of NA proteins revealed epitope differences among 2010 vaccine and H1N1pdm IAV strains circulating in the South American region. © 2012 Elsevier B.V.","<b>Exploring polymorphisms in B-DNA helical conformations<\/b><br><br>The traditional mesoscopic paradigm represents DNA as a series of base-pair steps whose energy response to equilibrium perturbations is elastic, with harmonic oscillations (defining local stiffness) around a single equilibrium conformation. In addition, base sequence effects are often analysed as a succession of independent XpY base-pair steps (i.e. a nearest-neighbour (NN) model with only 10 unique cases). Unfortunately, recent massive simulations carried out by the ABC consortium suggest that the real picture of DNA flexibility may be much more complex. The paradigm of DNA flexibility therefore needs to be revisited. In this article, we explore in detail one of the most obvious violations of the elastic NN model of flexibility: the bimodal distributions of some helical parameters. We perform here an in-depth statistical analysis of a very large set of MD trajectories and also of experimental structures, which lead to very solid evidence of bimodality. We then suggest ways to improve mesoscopic models to account for this deviation from the elastic regime. © 2012 The Author(s).","<b>Species delimitation with abc and other coalescent-based methods: A test of accuracy with simulations and an empirical example with lizards of the liolaemus darwinii complex (Squamata: Liolaemidae)<\/b><br><br>Species delimitation is a major research focus in evolutionary biology because accurate species boundaries are a prerequisite for the study of speciation. New species delimitation methods (SDMs) can accommodate nonmonophyletic species and gene tree discordance as a result of incomplete lineage sorting via the coalescent model, but do not explicitly accommodate gene flow after divergence. Approximate Bayesian computation (ABC) can incorporate gene flow and estimate other relevant parameters of the speciation process while testing alternative species delimitation hypotheses. We evaluated the accuracy of BPP, SpeDeSTEM, and ABC for delimiting species using simulated data and applied these methods to empirical data from lizards of the Liolaemus darwinii complex. Overall, BPP was the most accurate, ABC showed an intermediate accuracy, and SpeDeSTEM was the least accurate under most simulated conditions. All three SDMs showed lower accuracy when speciation occurred despite gene flow, as found in previous studies, but ABC was the method with the smallest decrease in accuracy. All three SDMs consistently supported the distinctness of southern and northern lineages within L. darwinii. These SDMs based on genetic data should be complemented with novel SDMs based on morphological and ecological data to achieve truly integrative and statistically robust approaches to species discovery. © 2012 The Author(s). Evolution© 2012 The Society for the Study of Evolution.","<b>Patterns of population differentiation in annual killifishes from the Paraná-Uruguay-La Plata Basin: The role of vicariance and dispersal<\/b><br><br>Aim To elucidate the role of vicariance versus dispersal at the microevolutionary scale in annual killifish populations belonging to the Austrolebias bellottii species complex (Rivulidae). Within this complex, A. bellottii and A. apaii have low vagility and occur widely within the study area, making them excellent models for testing biogeographic hypotheses of differentiation. Location South America, in the Paraná-Uruguay-La Plata river basin. Methods Molecular data and morphometric analyses were used to reconstruct the phylogeographic history and morphological variation of 24 populations of two taxa of the A. bellottii species complex. Phylogenetic analyses using maximum likelihood (ML) and Bayesian inference (BI) model-based methods, estimates of clade divergence times implemented in beast, non-metric multidimensional scaling, analysis of molecular variance results, and morphological analyses elucidated the role of vicariance versus dispersal hypotheses in population differentiation in the aforementioned river basin. Results In the A. bellottii species complex from the Paraná-Uruguay-La Plata river basin, past allopatric fragmentation from vicariance events seems to be the most plausible scenario for diversification since the Late Miocene and more recently since the Plio-Pleistocene. The Plio-Pleistocene vicariance produced the differentiation of three major clades in A. bellottii populations. One clade from the eastern Uruguay River drainage was separated from another in western Uruguay and the Paraná-La Plata River drainages. A later vicariance event split populations to the south (lower Paraná-La Plata Basin) and north (middle Paraná) of the western Paraná River drainage. However, our results do not exclude the possibility of dispersal events among A. bellottii populations from both the Uruguay and Paraná river drainages, which could occur in these river basins during hypothesized connectivity cycles of the Late Pliocene and Pleistocene. Main conclusions Past allopatric fragmentation caused by different vicariance events seems to be the main driver of diversification in the A. bellottii species complex since the Plio-Pleistocene. However, the current molecular data suggest that populations from both drainages of the Paraná-Uruguay rivers may have experienced cycles of connectivity during the Pleistocene, perhaps including multiple vicariance or dispersal events from populations located in the western lower Uruguay River drainage, which encompassed climatic and geological changes in the Paraná-Uruguay-La Plata Basin. © 2012 Blackwell Publishing Ltd.","<b>Genetic parameters for nematode resistance in periparturient ewes and post-weaning lambs in Uruguayan Merino sheep<\/b><br><br>Gastrointestinal (GI) parasites are one of the main sanitary and economic constraints for sheep production worldwide. In Uruguay, resistance to GI parasites has been included in genetic evaluations of Merino and Corriedale breeds since 1994, using Faecal worm Egg Count measured in post-weaning lambs (lambFEC) as a selection criterion. Although adult categories are more resistant to GI parasites, a temporary loss of acquired immunity is present in the periparturient period (=periparturient rise). The purpose of the present study is to estimate the genetic parameters of resistance to GI parasites in periparturient ewes and post-weaning Merino lambs. A total 2110 faecal samples of 748 periparturient ewes (eweFEC), the progeny of 107 sires, were collected in 2009 and 2010. 9458 lambFEC records from 7506 lambs born between 2001 and 2009 were analysed as well. (Co)variance components and systematic effects were estimated using a multi-trait animal model, with a Bayesian analysis using the Gibbs sampler algorithm. Direct and correlated responses (DR and CR) of eweFEC to selection using eweFEC and lambFEC respectively, were also estimated. Posterior medians (posterior standard deviation) for heritability (h2) and repeatability were 0.25 (0.03) and 0.34 (0.02) for lambFEC, and 0.08 (0.03) and 0.18 (0.03) for eweFEC, respectively. Posterior median for genetic correlation between both traits was 0.81 (0.11). CR obtained by selecting for lambFEC was two times more efficient than DR from selection by eweFEC. In conclusion, eweFEC has a lower h2 than lambFEC while their genetic correlation is moderate to high. Therefore, indirect selection by using lambFEC will be more effective than direct selection on eweFEC, and will produce ewes that eliminate less worm eggs, resulting in less pasture contamination. © 2012 Elsevier B.V..","<b>Genetic variability and demographic history of Heliothis virescens (Lepidoptera: Noctuidae) populations from Brazil inferred by mtDNA sequences<\/b><br><br>Intra- and inter-population genetic variability and the demographic history of Heliothis virescens (F.) populations were evaluated by using mtDNA markers (coxI, coxII and nad6) with samples from the major cotton- and soybean-producing regions in Brazil in the growing seasons 2007/08, 2008/09 and 2009/10. AMOVA indicated low and non-significant genetic structure, regardless of geographical scale, growing season or crop, with most of genetic variation occurring within populations. Clustering analyzes also indicated low genetic differentiation. The haplotype network obtained with combined datasets resulted in 35 haplotypes, with 28 exclusive occurrences, four of them sampled only from soybean fields. The minimum spanning network showed star-shaped structures typical of populations that underwent a recent demographic expansion. The recent expansion was supported by other demographic analyzes, such as the Bayesian skyline plot, the unimodal distribution of paired differences among mitochondrial sequences, and negative and significant values of neutrality tests for the Tajima's D and Fu's FS parameters. In addition, high values of haplotype diversity (Ĥ) and low values of nucleotide diversity (π), combined with a high number of low frequency haplotypes and values of θπ < θw suggested a recent demographic expansion of H. virescens populations in Brazil. This demographic event could be responsible for the low genetic structure currently found; however, haplotypes present uniquely at the same geographic regions and from one specific host plant suggest an initial differentiation among H. virescens populations within Brazil. © 2011 Cambridge University Press.","<b>Phylogeography of the sub-Antarctic notothenioid fish Eleginops maclovinus: Evidence of population expansion<\/b><br><br>Phylogeography studies add insights into the geographic and evolutionary processes that underline the genetic divergence of populations. This work examines the geographic genetic structure of the Patagonian blennie, Eleginops maclovinus, a notothenioid (Perciformes) endemic to South American temperate and sub-Antarctic waters, using mitochondrial DNA cytochrome b sequences. We found 58 haplotypes in the analysis of 261 individual sequences of 833 base pairs in length. Among-population variance was very low (1. 62%) and many haplotypes were shared between several populations across the species geographic range. Genetic differentiation was not consistent with a simple model of isolation by distance, possibly suggesting a lack of equilibrium between gene flow and local genetic drift. The analysis of mismatch distributions, neutrality tests, and the Bayesian Skyline Plot showed a pattern consistent with a recent population expansion event that may have taken place during the Middle Pleistocene. © 2011 Springer-Verlag.","<b>Diversification of the American bulb-bearing Oxalis (Oxalidaceae): Dispersal to North America and modification of the tristylous breeding system<\/b><br><br>Premise of the study: The American bulb-bearing Oxalis (Oxalidaceae) have diverse heterostylous breeding systems and are distributed in mountainous areas from Patagonia to the northeastern United States. To study the evolutionary processes leading to this diversity, we constructed the first molecular phylogeny for the American bulb bearing Oxalis and used it to infer biogeographic history and breeding system evolution. Methods: We used DNA sequence data (nuclear ribosomal internal transcribed spacer, trnL-trnL-trnF, trnT-trnL, and psbJpetA) to infer phylogenetic history via parsimony, likelihood, and Bayesian analyses. We used Bayes Multistate to infer ancestral geographic distributions at well-supported nodes of the phylogeny. The Shimodaira-Hasegawa (SH) test distinguished among hypotheses of single or multiple transitions from South America to North America, and tristyly to distyly. Key results: The American bulb-bearing Oxalis include sampled members of sections Ionoxalis and Pseudobulbosae and are derived from a larger clade that includes members of sections Palmatifoliae, Articulatae, and the African species. The American bulb-bearing Oxalis comprise two clades: one distributed in SE South America and the other in the Andes and North America. An SH test supports multiple dispersals to North America. Most sampled distylous species form a single clade, but at least two other independent distylous lineages are supported by the topologies and SH tests. Conclusions: Phylogenetic results suggest the American bulb-bearing Oxalis originated in southern South America, dispersed repeatedly to North America, and had multiple transitions from tristyly to distyly. This study adds to our understanding of biogeographic history and breeding system evolution and provides a foundation for more precise inferences about the study group. © 2012 Botanical Society of America.","<b>Phylogenetics of the chiliotrichum group (Compositae: Astereae): The story of the fascinating radiation in the paleate Astereae genera from southern South America<\/b><br><br>The Chiliotrichum group (Compositae: Astereae) is an informal taxonomic group consisting of shrubby daisies that grow mainly in the Andean and Patagonian regions of South America; it is composed of 31 species in 11 genera: Aylacophora, Cabreraea, Chiliotrichiopsis, Chiliophyllum, Chiliotrichum, Haroldia, Katinasia, Lepidophyllum, Llerasia, Nardophyllum, and Ocyroe. We analyzed phylogenetic relationships in the Chiliotrichum group using five DNA regions from two plant genomes: the plastid trnL-trnF and rpl32-trnL(UAG) spacers, the rpl16 intron, and the nuclear ribosomal internal and external transcribed spacers, ITS and ETS respectively. The dataset includes 21 ingroup species representing all the genera, and 10 outgroup species belonging to six genera. Maximum parsimony, maximum likelihood, and Bayesian inference analyses performed on the chloroplast and nuclear data support the monophyly of the group. The four major clades recovered on the basis of the combined nuclear evidence (ITS + ETS) are: (1) Llerasia clade, (2) Nardophyllum clade, (3) Chiliotrichum clade, and (4) a clade formed by Nardophyllum + Llerasia clades. As regards the status of the four non-monospecific genera, Chiliotrichum (2 spp.) and Llerasia (14 spp.) were recovered as monophyletic, whereas the other two, Nardophyllum (5 spp.) and Chiliotrichiopsis (3 spp.), were not recovered as monophyletic. These two are nonetheless recognized as distinct genera because of robust morphological support despite the lack of resolution in the phylogeny based solely on sequence data. The rest of the genera composing the group are monospecific, and they are scattered among the major clades in the group. The only genus with no clear position is Haroldia. An analysis of the morphological data shows that the presence of paleae, believed to function as a defensive structure for the protection of the achenes, is well correlated with other derived head and floret traits such as the number of florets per head and the type of pappus.","<b>Protected areas do not fulfil the wintering habitat needs of the trans-Saharan migratory Montagu's harrier<\/b><br><br>Populations of migratory birds can be affected by events happening at both breeding and wintering grounds. The Sahel is a vast region holding a large number of wintering trans-Saharan migratory European birds, and current land-use changes there may represent a threat for these species. We used satellite tracking data from the migratory Montagu's harrier to evaluate habitat use of the species during the wintering season, and whether the current network of protected areas is effective to provide their habitat needs during that season. We also developed an ecological niche model for the species in Western-Central Sahel to check if the most suitable sites are included within current protected areas. Tracked harriers occupied a large region encompassing a total of eight countries. The most preferred habitats during winter were croplands and some natural vegetation habitats, especially grasslands. Protected areas only covered a small proportion of the overall wintering grounds of tracked harriers and the most suitable areas for the species in Western-Central Sahel. Increasing the extent of preferred natural habitats within protected areas should benefit the conservation of this and probably other insect-eating raptors. However, substantial increases in extent and number of protected areas in sub-Saharan Africa are very unlike to occur. Conservation actions in the region should therefore be mostly focused on improving land use planning and management outside protected areas, specially enhancing agricultural practices to make biodiversity conservation compatible with poverty alleviation. These can be chiefly targeted at an area of <20,000km 2 of very suitable habitat for these species. © 2011 Elsevier Ltd.","<b>Population genetic structure and social kin associations of franciscana dolphin, pontoporia blainvillei<\/b><br><br>We investigated population and social structure of the franciscana dolphin, Pontoporia blainvillei, an endemic and the most endangered cetacean of the southwestern Atlantic Ocean. We analyzed samples from the Rio de la Plata estuary obtained in Uruguayan waters and from the Atlantic Ocean obtained in both Uruguayan and Brazilian waters. Mitochondrial and microsatellite DNA markers were used to study differentiation between the estuary and the ocean and the association between kinship and social group structure. Although multilocus analyses suggested that franciscanas are structured into 2 subpopulations (K = 2, divergence among clusters: FST = 0.06, P = 0.002; RST = 0.3, P = 0.001), mitochondrial markers did not support such divergence (FST = 0.02, P = 0.12; ST = 0.06, P =0.06). However, these units are not entirely segregated geographically. Regarding social structure, some groups are composed by first-order related individuals (R ≥ 0.5, P < 0.5). Overall, the data suggest that matrilines could be the social unit in this species. We argue that the divergence found could be associated to local adaptation and social structure, resulting from either feature leading to a recent divergence or reflecting equilibrium between local differentiation and gene flow. This evidence supports considering franciscanas from the Rio de la Plata estuary a discrete management unit. © The American Genetic Association. 2011. All rights reserved.","<b>Serological monitoring of selected avian pathogens of commercial poultry farms in Uruguay<\/b><br><br>Poultry production is a growing industry in Uruguay, South America. Mycoplasma gallisepticum (MG) and Mycoplasma synoviae (MS) are the etiological agent of chronic respiratory disease in chickens and may be horizontally and vertically infected. These economically important diseases can influence commercial poultry operations in Uruguay. Chickens infected with MG and/or MS become lessening in egg productivity, hatchability and feed efficiency and grow in mortality, waste carcass and veterinary costs. Increasingly, these possible performance and production deprivations require the strict control of MG and MS amongst all the poultry sectors. The objective of this study was to estimate the seroprevalence of the two pathogens mentioned above in relation to Mycoplasmosis in commercial chickens in Uruguay. Sera randomly selected from flocks of Uruguay were analysed using a commercial enzyme-linked immunosorbent assay (ELISA). Test prevalence were adjusted for the imperfect test characteristics using the Rogan-Gladen estimator (deterministic and stochastic functions) and Bayesian inference. Overall, 0.4% and 0.9% of the tested 1861 chickens were classified as test-positive for antibodies against MG and MS, respectively. Considering the simulated ELISA specificities of 99.5% for MG and 99.2% for MS and in the absence of other reported diagnoses from the area, these results were therefore considered to be \"false-positives\". The ELISA in combination with testing sera and statistical methods for true prevalence estimation provided a good survey tool at lower costs and time when compared to other approaches, and can be adapted to many other issues in animal health and production. © 2010 Nova Science Publishers, Inc. All rights reserved.","<b>Assessing the seroprevalence for abortigenic diseases of ovine herds based on diagnostic tests applied to individual-animals in Uruguay<\/b><br><br>The objective of this study was to estimate the true prevalence of seropositive individual sheep against Chlamydophila abortus and Brucella ovis using the Rogan- Gladen estimator in combination with Bayesian inference in Uruguay, South America. Ten ovine herds were kept under observation in 2008. Each study herd was randomly selected at different farms recruited from the Artigas (no. of the study herds = 1), Salto (no. of the study herds = 8) and Canelones (no. of the study herds = 1) Departments. The required total sample size of 1385 was determined by power analysis. Blood samples collected were used for diagnostic investigations. Sera were analysed using a commercial ELISA for the detection of antibody to Chlamydophila abortus and Brucella ovis in sheep serum. Adjusted outcomes were required for precise comparison of seroprevalence estimates. The overall antibody seroprevalence of Chlamydophila abortus and Brucella ovis would be extremely low. Considering the simulated ELISA specificities of 99.3% for Chlamydophila abortus and 97.0% for Brucella ovis and in the absence of other reported diagnoses from the area, these results were therefore considered to be \"false-positives\". The methodology was useful for obtaining estimates of Chlamydophila abortus and Brucella ovis prevalence and for establishing prevalence distributions which could be used as input parameters in risk assessment and decision models. © 2010 Nova Science Publishers, Inc. All rights reserved.","<b>The evolution of mitochondrial genomes in subterranean caviomorph rodents: Adaptation against a background of purifying selection<\/b><br><br>South American tuco-tucos (Ctenomys) and the related coruro (Spalacopus) are two rodent lineages that have independently colonised the subterranean niche. The energetically demanding lifestyles of these species, coupled with the hypoxic atmospheres characteristic of subterranean environments, may have altered the selective regimes on genes encoding proteins related to cellular respiration. Here, we examined the molecular evolution of 13 protein-coding genes in the mitochondrial genome of seven caviomorph rodents, including these two subterranean genera and their above-ground relatives. Using maximum-likelihood and Bayesian approaches, we estimated rates of synonymous (dS) and nonsynonymous (dN) substitutions. We found a significantly higher ω ratio (d. N/d. S) in subterranean groups as compared to their non-subterranean counterparts in 11 of 13 genes, although no ω ratio was larger than 1. Additionally, we applied a method based on quantitative physicochemical properties to test for positive selection. Amino acid changes implicated in radical structural or functional shifts in the protein property were found to be ubiquitous across the phylogeny, but concentrated in the subterranean lineages. Convergent changes were also found between the subterranean genera used in this study and other mammals adapted to hypoxia. The results of this study suggest a link between niche shifts and weak directional (or episodic) selection at the molecular level against a background of purifying selection. © 2011 Elsevier Inc.","<b>Apparent overconfidence<\/b><br><br>It is common for a majority of people to rank themselves as better than average on simple tasks and worse than average on difficult tasks. The literature takes for granted that this apparent misconfidence is problematic. We argue, however, that this behavior is consistent with purely rational Bayesian updaters. In fact, better-than-average data alone cannot be used to show overconfidence; we indicate which type of data can be used. Our theory is consistent with empirical patterns found in the literature. © 2011 The Econometric Society.","<b>Comparing Bayesian inference and case-based reasoning as support techniques in the diagnosis of Acute Bacterial Meningitis<\/b><br><br>The amount of information available for physicians has dramatically increased in the recent past. In contrast, the specialist's ability to understand, synthesize and take into account such information is severely constrained by the short time available for the appointments. Therefore, systems reusing available knowledge and implementing reasoning processes become critical to support the tasks of the doctors. As a number of different techniques for building such systems are available, contrasting their effectiveness becomes a major concern. This is especially important in the case of infectious diseases that can be lethal within hours such as the Acute Bacterial Meningitis (ABM) for which implementing and contrasting different techniques allows for an increased reliability and speed in supporting the process of diagnosis. This work focuses on the construction of diagnosis support tools for ABM, reporting a comparative assessment of the quality of a Clinical Decision Support System (CDSS) resulting from the application of Case Based Reasoning (CBR), to that of an existing CDSS system developed using a Bayesian expert system. Although both approaches proved to be useful, the one based in CBR techniques show some interesting capabilities as higher precision, automatic learning or experience capturing, and also a better response to lack of input data. The three developed systems perform with high levels of accuracy- e.g. propose correct diagnostics based on a certain set of symptoms - but the one based on CBR present some additional capabilities that look very promising for implementing these kind of systems in a real world scenario. © 2011 Published by Elsevier Ltd.","<b>Species groups and the evolutionary diversification of tuco-tucos, genus Ctenomys (Rodentia: Ctenomyidae)<\/b><br><br>We present the most comprehensive study to date of species groups in Ctenomys (tuco-tucos), a species-rich genus of Neotropical rodents. To explore phylogenetic relationships among 38 species and 12 undescribed forms we sequenced the complete mitochondrial cytochrome-b genes of 34 specimens and incorporated 50 previously published sequences. Parsimony, likelihood, and Bayesian phylogenetic analyses were performed using additional hystricognath rodents as outgroup taxa. The basal dichotomy of Ctenomys splits C. sociabilis from the remaining tuco-tucos, within which 8 main species groups were identified: boliviensis, frater, mendocinus, opimus, magellanicus, talarum, torquatus, and tucumanus. Whereas most of these groups refer to previous clades proposed on the basis of chromosomes or morphology, the torquatus and magellanicus species groups are novel taxonomic hypotheses. However, relationships among species groups are poorly resolved. Furthmore, the positions of C. leucodon, C. maulinus, and C. tuconax are conflicting or unresolved, and they might represent additional independent lineages. On the basis of molecular dating, we estimate that most species groups originated approximately 3 million years ago. © American 2011 Society of Mammalogists.","<b>Genome-wide marker-assisted selection combining all pedigree phenotypic information with genotypic data in one step: An example using broiler chickens<\/b><br><br>Data of broiler chickens for 2 pure lines across 3 generations were used for genomic evaluation. A complete population (full data set; FDS) consisted of 183,784 and 164,246 broilers for the 2 lines. The genotyped subsets (SUB) consisted of 3,284 and 3,098 broilers with 57,636 SNP. Genotyped animals were preselected based on more than 20 traits with different index applied to each line. Three traits were analyzed: BW at 6 wk (BW6), ultrasound measurement of breast meat (BM), and leg score (LS) coded 1 = no and 2 = yes for leg defect. Some phenotypes were missing for BM. The training population consisted of the first 2 generations including all animals in FDS or only genotyped animals in SUB. The validation data set contained only genotyped animals in the third generation. Genetic evaluations were performed using 3 approaches: 1) phenotypic BLUP, 2) extending BLUP methodologies to utilize pedigree and genomic information in a single step (ssGBLUP), and 3) Bayes A. Whereas BLUP and ssGBLUP utilized all phenotypic data, Bayes A could use only those of the genotyped subset. Heritabilities were 0.17 to 0.20 for BW6, 0.30 to 0.35 for BM, and 0.09 to 0.11 for LS. The average accuracies of the validation population with BLUP for BW6, BM, and LS were 0.46, 0.30, and <0 with SUB and 0.51, 0.34, and 0.28 with FDS. With ssGBLUP, those accuracies were 0.60, 0.34, and 0.06 with SUB and 0.61, 0.40, and 0.37 with FDS, respectively. With Bayes A, the accuracies were 0.60, 0.36, and 0.09 with SUB. With SUB, Bayes A and ssGBLUP had similar accuracies. For traits of high heritability, the accuracy of Bayes A/SUB and ssGBLUP/FDS were similar, and up to 50% better than BLUP/FDS. However, with low heritability, ssGBLUP/ FDS was 4 to 6 times more accurate than Bayes A/SUB and 50% better than BLUP/FDS. An optimal genomic evaluation would be multi-trait and involve all traits and records on which selection is based. ©2011 American Society of Animal Science.","<b>Evaluation of case based reasoning for clinical decision support systems applied to acute meningitis diagnose<\/b><br><br>This work presents a research about the applicability of Case Based Reasoning to Clinical Decision Support Systems (CDSS), particularly applied to the diagnosis of the disease known as Acute Bacterial Meningitis. In the last few years, the amount of information available to the medical doctor, who usually finds himself in the situation of making a diagnosis of one or more diseases, has dramatically increased. However, the specialist's ability to understand, synthesize and take advantage of such information in the always-little time during the medical act remains to be developed. Many contributions have been made by the computer sciences, especially those by Artificial intelligence, in order to solve these problems. This work focuses on the diagnose of the Acute Bacterial Meningitis, and carries out a comparative assessment of the quality of a Clinical Decision Support System made through Case Based Reasoning, in contrast to an already existing CDSS applied to the same task, but developed using a technique called Bayesian expert system. © Springer Science+Business Media B.V. 2010.","<b>Phenotypic and genetic characterization of novel somatic cell count traits from weekly or monthly observations<\/b><br><br>The objectives of this study were (1) to explore traits that better capture weekly or monthly changes in somatic cell counts (SCC) than does the commonly used lactation-average SCC, (2) to estimate their heritabilities and relationships to clinical mastitis (CM), and (3) to determine if these traits are feasible for use in monthly testing schemes. Clinical mastitis and weekly test-day (TD) records of SCC and milk production traits from 1,006 lactations of Swedish Red and Holstein cows collected from 1989 to 2004 were used (data set W). A data subset was also created to mimic monthly recording (data set M, 980 lactations). Twenty SCC traits were defined, taking into account SCC general levels and variation along the lactation curve, time and level of infection, and time of recovery. To reduce dimensionality, cluster and stepwise logistic regression procedures were applied. In data set W, 3 traits, \" standard deviation of SCC over the lactation,\" a discrete (0/1) indicator of \" at least one TD with SCC >500,000 cells/mL\" , and \" number of days sick in the widest SCC peak\" (DWidest) were the variables kept both with cluster procedures and a stepwise logistic regression with the logit of CM as dependent variable. In data set M, DWidest was replaced by \" number of SCC peaks\" and \" average number of days sick per peak\" (ADSick). Lactation-average SCC (in the first 150 d or between 150 and 305 d) did not enter into the logistic regression. Heritability estimates obtained for these new traits under a Bayesian setting and a Gibbs sampling approach were 10 to 16% (except for ADSick: 5%). Heritabilities were at least as high in the monthly data set as in the weekly data set. Thus, these SCC traits seem promising for use in breeding programs based on monthly milk recording. © 2010 American Dairy Science Association.","<b>Evolution of Dengue virus type 3 genotype III in Venezuela: Diversification, rates and population dynamics<\/b><br><br>Background. Dengue virus (DENV) is a member of the genus Flavivirus of the family Flaviviridae. DENV are comprised of four distinct serotypes (DENV-1 through DENV-4) and each serotype can be divided in different genotypes. Currently, there is a dramatic emergence of DENV-3 genotype III in Latin America. Nevertheless, we still have an incomplete understanding of the evolutionary forces underlying the evolution of this genotype in this region of the world. In order to gain insight into the degree of genetic variability, rates and patterns of evolution of this genotype in Venezuela and the South American region, phylogenetic analysis, based on a large number (n = 119) of envelope gene sequences from DENV-3 genotype III strains isolated in Venezuela from 2001 to 2008, were performed. Results. Phylogenetic analysis revealed an in situ evolution of DENV-3 genotype III following its introduction in the Latin American region, where three different genetic clusters (A to C) can be observed among the DENV-3 genotype III strains circulating in this region. Bayesian coalescent inference analyses revealed an evolutionary rate of 8.48 × 10-4 substitutions/site/year (s/s/y) for strains of cluster A, composed entirely of strains isolated in Venezuela. Amino acid substitution at position 329 of domain III of the E protein (AV) was found in almost all E proteins from Cluster A strains. Conclusions. A significant evolutionary change between DENV-3 genotype III strains that circulated in the initial years of the introduction in the continent and strains isolated in the Latin American region in recent years was observed. The presence of DENV-3 genotype III strains belonging to different clusters was observed in Venezuela, revealing several introduction events into this country. The evolutionary rate found for Cluster A strains circulating in Venezuela is similar to the others previously established for this genotype in other regions of the world. This suggests a lack of correlation among DENV genotype III substitution rate and ecological pattern of virus spread. © 2010 Ramírez et al; licensee BioMed Central Ltd.","<b>Change in the foraging strategy of female South American sea lions (Carnivora: Pinnipedia) after parturition<\/b><br><br>This study tests the hypothesis that female South American sea lions shift from off-shore, pelagic prey to coastal, benthic prey after parturition in order to reduce the foraging trip duration and hence the time pups remain unattended on the beach during early lactation. The δ13C and δ15N values of the serum and blood cells of 26 South American sea lion suckling pups from northern Patagonia were used to track the dietary changes of their mothers from late pregnancy to early lactation, after correction for differential isotopic fractionation between tissues. Primary producers and potential prey species were also analysed to establish a baseline for interpreting the stable isotope concentration of serum and blood cells. Isotopic ratios revealed a generalized increase in the consumption of coastal-benthic prey after parturition. Such a generalized post-partum shift will allow females to spend more time on land and look after their pups. The effects of this foraging strategy on the nutritional quality of the female's diet are discussed.","<b>Phylodynamics of HIV-1 Circulating Recombinant Forms 12_BF and 38_BF in Argentina and Uruguay<\/b><br><br>Background: Although HIV-1 CRF12_BF and CRF38_BF are two epidemiologically important recombinant lineages circulating in Argentina and Uruguay, little is known about their population dynamics.Methods: A total of 120 \"CRF12_BF-like\" and 20 \"CRF38_BF-like\" pol recombinant sequences collected in Argentina and Uruguay from 1997 to 2009 were subjected to phylogenetic and Bayesian coalescent-based analyses to estimate evolutionary and demographic parameters.Results: Phylogenetic analyses revealed that CRF12_BF viruses from Argentina and Uruguay constitute a single epidemic with multiple genetic exchanges among countries; whereas circulation of the CRF38_BF seems to be confined to Uruguay. The mean estimated substitution rate of CRF12_BF at pol gene (2.5 × 10-3 substitutions/site/year) was similar to that previously described for subtype B. According to our estimates, CRF12_BF and CRF38_BF originated at 1983 (1978-1988) and 1986 (1981-1990), respectively. After their emergence, the CRF12_BF and CRF38_BF epidemics seem to have experienced a period of rapid expansion with initial growth rates of around 1.2 year-1 and 0.9 year-1, respectively. Later, the rate of spread of these CRFs_BF seems to have slowed down since the mid-1990s.Conclusions: Our results suggest that CRF12_BF and CRF38_BF viruses were generated during the 1980s, shortly after the estimated introduction of subtype F1 in South America (~1975-1980). After an initial phase of fast exponential expansion, the rate of spread of both CRFs_BF epidemics seems to have slowed down, thereby following a demographic pattern that resembles those previously reported for the HIV-1 epidemics in Brazil, USA, and Western Europe. © 2010 Bello et al; licensee BioMed Central Ltd.","<b>Dispersal and population structure at different spatial scales in the subterranean rodent Ctenomys australis<\/b><br><br>Background: The population genetic structure of subterranean rodent species is strongly affected by demographic (e.g. rates of dispersal and social structure) and stochastic factors (e.g. random genetic drift among subpopulations and habitat fragmentation). In particular, gene flow estimates at different spatial scales are essential to understand genetic differentiation among populations of a species living in a highly fragmented landscape. Ctenomys australis (the sand dune tuco-tuco) is a territorial subterranean rodent that inhabits a relatively secure, permanently sealed burrow system, occurring in sand dune habitats on the coastal landscape in the south-east of Buenos Aires province, Argentina. Currently, this habitat is threatened by urban development and forestry and, therefore, the survival of this endemic species is at risk. Here, we assess population genetic structure and patterns of dispersal among individuals of this species at different spatial scales using 8 polymorphic microsatellite loci. Furthermore, we evaluate the relative importance of sex and habitat configuration in modulating the dispersal patterns at these geographical scales.Results: Our results show that dispersal in C. australis is not restricted at regional spatial scales (~ 4 km). Assignment tests revealed significant population substructure within the study area, providing support for the presence of two subpopulations from three original sampling sites. Finally, male-biased dispersal was found in the Western side of our study area, but in the Eastern side no apparent philopatric pattern was found, suggesting that in a more continuous habitat males might move longer distances than females.Conclusions: Overall, the assignment-based approaches were able to detect population substructure at fine geographical scales. Additionally, the maintenance of a significant genetic structure at regional (~ 4 km) and small (less than 1 km) spatial scales despite apparently moderate to high levels of gene flow between local sampling sites could not be explained simply by the linear distance among them. On the whole, our results support the hypothesis that males disperse more frequently than females; however they do not provide support for strict philopatry within females. © 2010 Mora et al; licensee BioMed Central Ltd.","<b>Genetic history of hepatitis C virus in Venezuela: High diversity and long time of evolution of HCV genotype 2<\/b><br><br>Background: The subtype diversity of the hepatitis C virus (HCV) genotypes is unknown in Venezuela. Methodology/Principal Findings: Partial sequencing of the NS5B region was performed in 310 isolates circulating in patients from 1995 to 2007. In the samples collected between 2005 and 2007, HCV genotype 1 (G1) was the most common genotype (63%), composed as expected of mainly G1a and G1b. G2 was the second most common genotype (33%), being G2a almost absent and G2j the most frequent subtype. Sequence analysis of the core region confirmed the subtype assignment performed within the NS5b region in 63 isolates. The complete genome sequence of G2j was obtained. G2j has been described in France, Canada and Burkina Fasso, but it was not found in Martinique, where several subtypes of G2 circulate in the general population. Bayesian coalescence analysis indicated a most recent common ancestor (MRCA) of G2j around 1785, before the introduction of G1b (1869) and G1a (1922). While HCV G1a and G1b experienced a growth reduction since 1990, coincident with the time when blood testing was implemented in Venezuela, HCV G2j did not seem to reach growth equilibrium during this period. Conclusions/Significance: Assuming the introduction of G2j from Africa during the slave trade, the high frequency of G2j found in Venezuela could suggest: 1- the introduction of African ethnic groups different from the ones introduced to Martinique or 2- the occurrence of a founder effect. This study represents an in-depth analysis of the subtype diversity of HCV in Venezuela, which is still unexplored in the Americas and deserves further studies. © 2010 Sulbaran et al.","<b>L2-Boosting algorithm applied to high-dimensional problems in genomic selection<\/b><br><br>The L2-Boosting algorithm is one of the most promising machine-learning techniques that has appeared in recent decades. It may be applied to high-dimensional problems such as whole-genome studies, and it is relatively simple from a computational point of view. In this study, we used this algorithm in a genomic selection context to make predictions of yet to be observed outcomes. Two data sets were used: (1) productive lifetime predicted transmitting abilities from 4702 Holstein sires genotyped for 32 611 single nucleotide polymorphisms (SNPs) derived from the Illumina BovineSNP50 BeadChip, and (2) progeny averages of food conversion rate, pre-corrected by environmental and mate effects, in 394 broilers genotyped for 3481 SNPs. Each of these data sets was split into training and testing sets, the latter comprising dairy or broiler sires whose ancestors were in the training set. Two weak learners, ordinary least squares (OLS) and non-parametric (NP) regression were used for the L2-Boosting algorithm, to provide a stringent evaluation of the procedure. This algorithm was compared with BL [Bayesian LASSO (least absolute shrinkage and selection operator)] and BayesA regression. Learning tasks were carried out in the training set, whereas validation of the models was performed in the testing set. Pearson correlations between predicted and observed responses in the dairy cattle (broiler) data set were 065 (033), 053 (037), 066 (026) and 063 (027) for OLS-Boosting, NP-Boosting, BL and BayesA, respectively. The smallest bias and mean-squared errors (MSEs) were obtained with OLS-Boosting in both the dairy cattle (008 and 108, respectively) and broiler (0011 and 0006) data sets, respectively. In the dairy cattle data set, the BL was more accurate (bias=010 and MSE=110) than BayesA (bias=126 and MSE=281), whereas no differences between these two methods were found in the broiler data set. L2-Boosting with a suitable learner was found to be a competitive alternative for genomic selection applications, providing high accuracy and low bias in genomic-assisted evaluations with a relatively short computational time. © 2010 Cambridge University Press.","<b>Assessing the seroprevalence against avian pneumovirus and Ornithobacterium rhinotracheale in broilers in Uruguay<\/b><br><br>The objective of this study was to estimate the true prevalence of individual chickens serologically test-positive against avian pneumovirus and Ornithobacterium rhinotracheale in Uruguay. Seventeen different broiler farms existed in three different provinces in Uruguay were recruited and the 1861 broilers were investigated. Individual-chicken sera were analyzed using a commercial enzyme-linked immunosorbent assay. The overall true seroprevalence was 1.9% [95% Bayesian Credible Interval (BCI): <1-7.4%] and less than 0.1% (95% BCI: 0-<0.1%) against avian pneumovirus and Ornithobacterium rhinotracheale, respectively. The result seroprevalence was relatively lower than that reported by other authors in a neighbouring country Argentina where the infection of these diseases was recently observed. This difference was discussed. © Asian Network for Scientific Information, 2010.","<b>Flock-level seroprevalence against avian pneumovirus amongst uruguayan broiler chickens<\/b><br><br>The objective of this study was to estimate the true prevalence of seropositive broiler chickens against avian pneumovirus at flock-level in Uruguay, using the Rogan-Gladen estimator in conjunction with Bayesian inference. A total of 181 pooled samples (consisting of 10 individual-chicken sera each) from the study area were examined with the enzyme-linked immunosorbent assay. All individual-chicken samples in the pools were also examined with the same assay. Forty-four pools were classified as test positive, because they included at least one individual-chicken classified as positive. The estimates for the deterministic (Rogan-Gladen approach) and stochastic (Bayesian approach) true prevalence were 30.9% [95% confidence interval (CI): 26.8-35.0%] and 31.4% (95% CI: 15.4-49.5%), respectively. © Asian Network for Scientific Information, 2010.","<b>Flock-level seroprevalence against Ornithobacterium rhinotracheale among broilers in Uruguay<\/b><br><br>The objective of this study was to estimate the true prevalence of seropositive broiler flocks against Ornithobacterium rhinotracheale in Uruguay, South America. Seventeen farms of broiler chickens greater than 35 days of age were studied. The field investigation was conducted between October 2008 and April 2009. Individual-chicken sera and pooled sera (containing 10 individual-chicken sera each) were analyzed using a commercial ELISA for the detection of antibody against Ornithobacterium rhinotracheale in chicken serum. A total of 181 pooled samples from the study area representing 17 farms were examined. Fifty-four pools were classified as test positive, because they included at least one individual-chicken classified as positive. On the basis of deterministic approach, the estimates for the apparent prevalence and true prevalence at flock-level were 30% and 17%, respectively. The true prevalence estimate with the Bayesian model (stochastic approach) was slightly lower and having wider confidence intervals [11% (95% CI: 0%-32%)]. © Asian Network for Scientific Information, 2010.","<b>Modeling gene sequences over time in 2009 H1N1 Influenza A Virus populations<\/b><br><br>Background. A sudden emergence of Influenza A Virus (IAV) infections with a new pandemic H1N1 IAV is taking place since April of 2009. In order to gain insight into the mode of evolution of these new H1N1 strains, we performed a Bayesian coalescent Markov chain Monte Carlo (MCMC) analysis of full-length neuraminidase (NA) gene sequences of 62 H1N1 IAV strains (isolated from March 30th to by July 28th, 2009). Results. The results of these studies revealed that the expansion population growth model was the best to fit the sequence data. A mean of evolutionary change of 7.84 × 10 -3 nucleotide substitutions per site per year (s/s/y) was obtained for the NA gene. A significant contribution of first codon position to this mean rate was observed. Maximum clade credibility trees revealed a rapid diversification of NA genes in different genetic lineages, all of them containing Oseltamivir-resistant viruses of very recent emergence. Mapping of naturally occurring amino acid substitutions in the NA protein from 2009 H1N1 IAV circulating in 62 different patients revealed that substitutions are distributed all around the surface of the molecule, leaving the hydrophobic core and the catalytic site essentially untouched. Conclusion. High evolutionary rates and fast population growth have contributed to the initial transmission dynamics of 2009 H1N1 IAV. Naturally occurring substitutions are preferentially located at the protein surface and do not interfere with the NA active site. Antigenic regions relevant for vaccine development can differ from previous vaccine strains and vary among patients. © 2009 Gõi et al; licensee BioMed Central Ltd.","<b>Bayesian coalescent inference reveals high evolutionary rates and expansion of Norovirus populations<\/b><br><br>Noroviruses (NoV) are a leading cause of outbreaks of nonbacterial acute gastroenteritis in humans worldwide and have become an important cause of hospitalization of children in South America. NoV belong to the family Caliciviridae and are non-enveloped single stranded, positive sense, RNA viruses. NoV of genotype GII/4 have emerged worldwide, causing four epidemic seasons of viral gastroenteritis during which four novel variants emerged. Despite the importance of NoV outbreaks, little is known about the evolutionary rates, viral spread and population dynamics of NoV populations. In order to gain insight into these matters, a Bayesian Markov chain Monte Carlo (MCMC) approach was used to analyze region D or full-length VP1 gene sequences of GII/4 NoV populations isolated in Brazil or Japan, respectively. The results of these studies revealed that the expansion population growth model was the best to fit the data in both datasets. The dates of the most common recent ancestors revealed that these viruses can quickly emerge in a geographical location. A mean evolutionary rate of 1.21 × 10-2 nucleotide substitution/site/year (s/s/y) was obtained for the VP1 gene using full-length sequences. This rate is higher than the rates reported for other rapidly evolving RNA. Roughly similar rates (1.44 × 10-2 s/s/y) were found using region D sequences, revealing the suitability of this region for evolutionary studies, in agreement with previous reports. High evolutionary rates and fast population growth may have contributed to the vigorous initial transmission dynamics of the GII/4 NoV populations studied. © 2009 Elsevier B.V. All rights reserved.","<b>Predicting quantitative traits with regression models for dense molecular markers and pedigree<\/b><br><br>The availability of genomewide dense markers brings opportunities and challenges to breeding programs. An important question concerns the ways in which dense markers and pedigrees, together with phenotypic records, should be used to arrive at predictions of genetic values for complex traits. If a large number of markers are included in a regression model, marker-specific shrinkage of regression coefficients may be needed. For this reason, the Bayesian least absolute shrinkage and selection operator (LASSO) (BL) appears to be an interesting approach for fitting marker effects in a regression model. This article adapts the BL to arrive at a regression model where markers, pedigrees, and covariates other than markers are considered jointly. Connections between BL and other marker-based regression models are discussed, and the sensitivity of BL with respect to the choice of prior distributions assigned to key parameters is evaluated using simulation. The proposed model was fitted to two data sets from wheat and mouse populations, and evaluated using cross-validation methods. Results indicate that inclusion of markers in the regression further improved the predictive ability of models. An R program that implements the proposed model is freely available. Copyright © 2009 by the Genetics Society of America.","<b>Modeling gene sequence changes over time in type 3 dengue viruses from Ecuador<\/b><br><br>Dengue virus (DENV) is a member of the genus Flavivirus of the family Flaviviridae. DENV-3 re-emerged in Central America in 1994, and continues to expand into the South American region. Little is known about the evolutionary rates, viral spread and population dynamics of this genotype in the Latin American region. In order to gain insight into these matters, we used a Bayesian Markov chain Monte Carlo (MCMC) approach, to analyze envelope (E) gene sequences of the DENV-3 genotype III of strains included in a monophyletic cluster composed by Ecuadorian as well as strains from Cuba, Puerto Rico and Peru. The results of these studies revealed that the expansion population growth model was the best fit to the data. The most common recent ancestor (MRCA) was placed around 1989, in agreement with the first reports of the emergence of this new DENV-3 type. A mean rate 1.033 × 10-3 nucleotide substitution per site per year was obtained. This rate is comparatively higher than the ones obtained for DENV-2 and DENV-4 in the same region. Faster population growth and greater population dispersal may have contributed to the vigorous initial transmission dynamics of this genotype in the Latin American region. © 2009 Elsevier B.V. All rights reserved.","<b>Predictive ability of direct genomic values for lifetime net merit of Holstein sires using selected subsets of single nucleotide polymorphism markers<\/b><br><br>The objective of the present study was to assess the predictive ability of subsets of single nucleotide polymorphism (SNP) markers for development of low-cost, low-density genotyping assays in dairy cattle. Dense SNP genotypes of 4,703 Holstein bulls were provided by the USDA Agricultural Research Service. A subset of 3,305 bulls born from 1952 to 1998 was used to fit various models (training set), and a subset of 1,398 bulls born from 1999 to 2002 was used to evaluate their predictive ability (testing set). After editing, data included genotypes for 32,518 SNP and August 2003 and April 2008 predicted transmitting abilities (PTA) for lifetime net merit (LNM$), the latter resulting from progeny testing. The Bayesian least absolute shrinkage and selection operator method was used to regress August 2003 PTA on marker covariates in the training set to arrive at estimates of marker effects and direct genomic PTA. The coefficient of determination (R2) from regressing the April 2008 progeny test PTA of bulls in the testing set on their August 2003 direct genomic PTA was 0.375. Subsets of 300, 500, 750, 1,000, 1,250, 1,500, and 2,000 SNP were created by choosing equally spaced and highly ranked SNP, with the latter based on the absolute value of their estimated effects obtained from the training set. The SNP effects were re-estimated from the training set for each subset of SNP, and the 2008 progeny test PTA of bulls in the testing set were regressed on corresponding direct genomic PTA. The R2 values for subsets of 300, 500, 750, 1,000, 1,250, 1,500, and 2,000 SNP with largest effects (evenly spaced SNP) were 0.184 (0.064), 0.236 (0.111), 0.269 (0.190), 0.289 (0.179), 0.307 (0.228), 0.313 (0.268), and 0.322 (0.291), respectively. These results indicate that a low-density assay comprising selected SNP could be a cost-effective alternative for selection decisions and that significant gains in predictive ability may be achieved by increasing the number of SNP allocated to such an assay from 300 or fewer to 1,000 or more. © American Dairy Science Association, 2009.","<b>Monitoring the infectious disease status of dairy cattle herds in Uruguay: A random effects hierarchical model analysis<\/b><br><br>Dairy industry in Uruguay has grown steadily over the last three decades, positioning it as the sector with greatest growth amongst exporting countries of dairy products. Milk production expanded from 765 million metric tonnes in 1976 to 1770 million metric tonnes in 2006 at a 4.3% annual growth rate. Meanwhile, infectious diseases can play an important role in the economic feasibility of a dairy industry. An infectious agent circulating in a dairy herd, as well as the costs associated with an outbreak, can contribute to fertility problems, abortions, decreased milk production and loss of bodyweight. Prevented by meticulous attention with knowledge of the disease status in a dairy herd, dairy farmers and veterinarians can collectively draw up an action plan to manage the diseases in question. In Uruguay, there are diagnostic testing facilities for most infectious diseases in dairy cattle in relation to the problems mentioned above, however, dairy cattle health schemes covering infectious bovine rhinotracheitis (IBR), bovine viral diarrhoea (BVD), leptospirosis, Johne's disease, neosporosis and campylobacteriosis are not relatively organised. This chapter discusses how to monitor a number of infectious diseases that are of particular economic importance to the dairy cattle industry in Uruguay, namely, IBR and BVD, using one of Bayesian analytic approaches \"a random effects hierarchical model analysis\". © 2009 Nova Science Publishers, Inc. All rights reserved.","<b>Phylogeography of the Southwestern Atlantic menhaden genus Brevoortia (Clupeidae, Alosinae)<\/b><br><br>Among pelagic fish, the Southwestern Atlantic menhaden genus Brevoortia (Clupeidae, Alosinae) constitutes an important species model to investigate the patterns of genetic differentiation. It is abundant in the Río de la Plata estuary and in the Atlantic coastal lagoons system from Uruguay and Southern Brazil. To access in the taxa discrimination and population structure in Brevoortia we perform a phylogeographic approach based on mitochondrial cytochrome b (cyt-b) sequences including 240 individuals from 16 collecting sites. Among the 720 bp cyt-b sequenced, 199 correspond to variables and 88 to phylogenetically informative sites. High values of haplotype diversity (h = 1.000) and nucleotide diversity (π = 0.061), as well as an average of 0.084 polymorphic segregating sites and 46 different haplotypes were found. Maximum likelihood analysis based on the GTR + I + G model and Bayesian inference strongly support the idea that B. aurea is the only species of the genus inhabiting the Southwestern Atlantic region. Our analyses revealed a complex population pattern characterized by the existence of long-term highly structured genetic assemblages of mixed stocks. Each monophyletic entity included individuals from different collecting sites, different age groups and collected in different years. Our data also suggest that the recruitment of unrelated mtDNA haplotypes carried out by individuals within schools could be occurring in the same nursery areas revealing the existence of many different maternal lineages. A scenario where different simultaneously and successively mixed mtDNA lineages remain historically connected through basal haplotypes among different clades could explains more accurately the complex and ordered metapopulation dynamic found in this pelagic fish. © 2008 Springer-Verlag.","<b>Phylogeography of the Neotropical catfish Pimelodus albicans (Siluriformes: Pimelodidae) from río de la Plata basin, South America, and conservation remarks<\/b><br><br>Pimelodus albicans Valenciennes, 1840 (common name \"moncholo\" or \"bagre blanco\") is an endemic species of the family Pimelodidae in the río de la Plata basin. Phylogenetic approach based on cytochrome b sequences was performed to test the existence of a unique evolutionary lineage in P. albicans and to discriminate populations units or subpopulations related to a migration behavior of this taxon in the río de la Plata basin. This study included 34 samples of P. albicans of different collecting sites in the río de la Plata estuary and in the río Arrecifes belonging to the río Paraná basin. Among 614 base pairs in the cytochrome b sequence data set, 203 were variable and 120 were phylogenetically informative sites in P. albicans. A total of twenty haplotypes, nucleotide diversity (π) = 0.032 and haplotype diversity = 0.941 were found. Tajima's test showed significant value D= -1.88 (p<0.05) rejecting the neutral mutation hypothesis for the P. albicans data set. All phylogenetic approaches showed that P. albicans included four monophyletic assemblages that were supported by high bootstrap and Bayesian posterior probability values. Minimum spanning network corroborated these groups for P. albicans haplotypes. High genetic structure was found in P. albicans by means of AMOVA analysis showing that the río Arrecifes samples constitute an isolated lineage. Moreover, the high value of genetic divergence (10%) between the río de la Plata and the río Arrecifes populations could suggest that P. albicans may be conformed by a sibling species complex. On the other hand, a degree of genetic structuring was detected among different sites of the río de la Plata. A partial isolation of the 760 site may suggest that P. albicans could migrates to different tributaries for reproduction, generating different schools of haplotypes which could mix in the río de la Plata estuary. The high nucleotide diversity found in the 765 site and the existence of gene flow with the remaining collecting sites would be concordant with the outlined hypothetic scenarios of the mixing populations in the middle of the río de la Plata estuary. Copyright © 2008 Sociedade Brasileira de Ictiologia.","<b>A comparison between Poisson and zero-inflated Poisson regression models with an application to number of black spots in Corriedale sheep<\/b><br><br>Dark spots in the fleece area are often associated with dark fibres in wool, which limits its competitiveness with other textile fibres. Field data from a sheep experiment in Uruguay revealed an excess number of zeros for dark spots. We compared the performance of four Poisson and zero-inflated Poisson (ZIP) models under four simulation scenarios. All models performed reasonably well under the same scenario for which the data were simulated. The deviance information criterion favoured a Poisson model with residual, while the ZIP model with a residual gave estimates closer to their true values under all simulation scenarios. Both Poisson and ZIP models with an error term at the regression level performed better than their counterparts without such an error. Field data from Corriedale sheep were analysed with Poisson and ZIP models with residuals. Parameter estimates were similar for both models. Although the posterior distribution of the sire variance was skewed due to a small number of rams in the dataset, the median of this variance suggested a scope for genetic selection. The main environmental factor was the age of the sheep at shearing. In summary, age related processes seem to drive the number of dark spots in this breed of sheep. © 2008 INRA EDP Sciences.","<b>Bayesian coalescent inference of hepatitis A virus populations: Evolutionary rates and patterns<\/b><br><br>Hepatitis A virus (HAV) is a hepatotropic member of the family Picornaviridae. Previous studies suggested that HAV may evolve more slowly than other members of the family. To estimate HAV substitution rates precisely, we used a Bayesian Markov chain Monte Carlo (MCMC) approach on temporally sampled HAV VP1 full-length sequences from strains isolated in France. A mean rate of evolutionary change of 9.76×10-4 nucleotide substitution per site per year was found. The results also revealed that the synonymous rate found for HAV is lower than that of other members of the family. Bayesian skyline plots revealed a sharp decline in the effective number of infections in 1996, coinciding with the introduction of HAV vaccine. © 2007 SGM.","<b>Genetic variability in calving success in Aberdeen Angus cows under extensive recording<\/b><br><br>Data from 2032 Uruguayan Aberdeen Angus cows under extensive management and recording practices were analysed with Bayesian threshold-liability sire models, to assess genetic variability in calving success (CS), defined as a different binary trait for each of the second (CS2), third (CS3) and fourth (CS4) calving opportunities. Sire (herd) variances ranged from 0.08 to 0.11 (0.10 to 0.20) and heritability from 0.27 to 0.35, with large credibility intervals. Correlations between herd effects on CS at different calving opportunities were positive. Genetic correlation between CS2 and CS4 was positive (0.68), whereas those involving adjacent calving opportunities (CS2-CS3 and CS3-CS4) were negative, at -0.39 and -0.54, respectively. The residual correlation CS2-CS3 was negative (-0.32). The extent of uncertainty associated with the posterior estimates of the parameters was further evaluated through simulation, assuming different true values (-0.4, -0.2, +0.2 and +0.4) for the genetic correlations and changes in the degree of belief parameters of the inverse Wishart priors for the sire covariance matrix. Although inferences were not sharp enough, CS appears to be moderately heritable. The quality of data recording should be improved, in order to effect genetic improvement in female fertility. © 2007 The Animal Consortium.","<b>Genetic and environment factors on probability of early pregnancy of Caracu cattle<\/b><br><br>Selection criteria associated to sexual precocity have increased their importance and in order to reduce the age at first calving they have been including in selection programs with positive consequences to the profit of the herds. The present study estimated the heritability for probability of early pregnancy (PPP), a trait associated with puberty, that could be considered as indicative of the beginning of the reproductive activity in females, using records on 5273 females. In order to verify the significance of the effects of milk yield in first lactation, age at calving of the dam and weaning age of the heifer on PPP logistic regression was applied. Heritability for PPP was estimated by Bayesian method, using a sire model. This model included the effects of contemporary group, milk yield in first lactation (in classes), sire and residual. The \"posteriori\" mean of heritability was 0.13. This value, although not high, showed the presence of genetic additive action on the determination of this trait.","<b>Phylogeography of the deer mouse (Peromyscus maniculatus) provides a predictive framework for research on hantaviruses<\/b><br><br>Phylogeographical partitioning of Sin Nombre and Monongahela viruses (hantaviruses) may reflect that of their primary rodent host, the deer mouse (Peromyscus maniculatus). Lack of a comprehensive assessment of phylogeographical variation of the host has precluded the possibility of predicting spatial limits of existing strains of these viruses or geographical regions where novel viral strains might emerge. The complete cytochrome b gene was sequenced for 206 deer mice collected from sites throughout North America to provide a foundation for future studies of spatial structure and evolution of this ubiquitous host. Bayesian analyses of these sequences partitioned deer mice into six largely allopatric lineages, some of which may represent unrecognized species. The geographical distributions of these lineages were probably shaped by Quaternary climatic events. Populations of mice were apparently restricted to refugia during glacial advances, where they experienced genetic divergence. Expansion of these populations, following climatic amelioration, brought genetically distinctive forms into contact. Occurrence of parallel changes in virus strains can now be explored in appropriate regions. In New Mexico, for example, near the location where Sin Nombre virus was first discovered, there are three genetically distinctive lineages of deer mice whose geographical ranges need to be delineated precisely. The phylogeography of P. maniculatus provides a framework for interpreting geographical variability, not only in hosts, but also in associated viral variants and disease transmission, and an opportunity to predict the potential geographical distribution of newly emerging viral strains. © 2006 SGM.","<b>Inferring parameters shaping amino acid usage in prokaryotic genomes via Bayesian MCMC methods<\/b><br><br>Molar content of guanine plus cytosine (G + C) and optimal growth temperature (OGT) are main factors characterizing the frequency distribution of amino acids in prokaryotes. Previous work, using multivariate exploratory methods, has emphasized ascertainment of biological factors underlying variability between genomes, but the strength of each identified factor on amino acid content has not been quantified. We combine the flexibility of the phylogenetic mixed model (PMM) with the power of Bayesian inference via Markov Chain Monte Carlo (MCMC) methods, to obtain a novel evolutionary picture of amino acid usage in prokaryotic genomes. We implement a Bayesian PMM which incorporates the feature that evolutionary history makes observed data interdependent. As in previous studies with PMM, we present a variance partition; however, attention is also given to the posterior distribution of \"systematic effects\" that may shed light about the relative importance of and relationships between evolutionary forces acting at the genomic level. In particular, we analyzed influences of G + C, OGT, and respiratory metabolism. Estimates of G + C effects were significant for amino acids coded by G + C or molar content of adenine plus thymine (A + T) in first and second bases. OGT had an important effect on 12 amino acids, probably reflecting complex patterns of protein modifications, to cope with varying environments. The effect of respiratory metabolism was less clear, probably due to the already reported association of G + C with aerobic metabolism. A \"heritability\" parameter was always high and significant, reinforcing the importance of accommodating phylogenetic relationships in these analyses. \" Heritable\" component correlations displayed a pattern that tended to cluster \"pure\" G + C (A + T) in first and second codon positions, suggesting an inherited departure from linear regression on G + C. © The Author 2005. Published by Oxford University Press on behalf of the Society for Molecular Biology and Evolution. All rights reserved.","<b>Model comparison for genetic evaluation of milk yield in Uruguayan holsteins<\/b><br><br>Three models for genetic evaluation of milk yield of Uruguayan Holstein cattle were compared using 159 169 lactation records from 81 928 cows calving between 1989 and 1998. Model I included the effects of herd-year-season, parity by age group, additive genetic merit, permanent environment, and residual. Model II included all effects in Model I, as well as number of days open and length of the dry period. Model III included all factors in Model II, and it accommodated heterogeneity of variance within contemporary groups (CG) through a pre-adjustment of the data based on empirical Bayes estimates of the CG variance. Estimates of heritability for milk yield were 0.23, 0.24 and 0.25, and estimates of repeatability were 0.55, 0.56 and 0.57 for Models I, II and III, respectively. Models were contrasted by examining changes in sire ranking and by a cross-validation procedure, based on the ability of the models to predict first, second and later lactations. Data were divided into two subsets, and records from one subset were predicted using location parameters estimated from the other subset. A resampling procedure was used to minimise the dependency on the sample structure. Correspondence between observed and predicted values was assessed in terms of square root of empirical mean square errors of prediction, percentage squared bias and the coefficient of determination 'R2'. Adjustment for heterogeneous CG variance had a marked effect on rankings of animals, especially elite cows, where correlations between solutions from Models I and II versus Model III ranged from 0.53 to 0.80. The percentage of animals selected in common by each pair of models decreased when selection intensity increased. Cross-validation analyses suggested that an assumption of heterogeneity of CG variance is tenable, especially in later lactations, whereas some doubts arise in first lactations, most probably due to the data structure used in the analyses. © 2003 Elsevier B.V. All rights reserved.","<b>Evaluation of extent and amount of heterogeneous variance for milk yield in Uruguayan Holsteins<\/b><br><br>The extent and amount of heterogeneous phenotypic variance for milk yield in the Uruguayan Holstein population were evaluated and a simple method of accounting for heterogeneity was developed. Lactation records (159 169) collected between 1989 and 1998 by two recording schemes were used to form 8955 herd-year-season-parity-lactation length contemporary groups. A log-linear model was used to identify factors affecting heterogeneity of phenotypic variance. The model included effects of production level, contemporary group size, recording scheme, herd, season of calving, parity number, calving year period and length of lactation and accounted for 50% of the variation in log variances. Estimates from this model were used in a Bayesian manner, to obtain posterior mean estimates of within-contemporary-group variances, which were then used to standardize records to a baseline variance. Effects of the adjustment were assessed by comparing coefficients of variation before and after correction, by correlation and regression between mean and standard deviations, and by using Gini coefficients and Lorenz curves. The adjustment procedure reduced heteroscedasticity primarily by decreasing the frequency of low-variance contemporary groups. Lorenz curves and Gini coefficients indicated that the largest impact of the standardization procedure was related to the size of the contemporary group. Some differences in the effect of the correction were found between recording schemes. The method for adjusting records is simple and easy to adapt to current genetic evaluation procedures. © 2001 British Society of Animal Science.","<b>Assessing relationships between genetic evaluations using robust regression with an application to holsteins in uruguay<\/b><br><br>Globalization of dairy cattle breeding has created a need for international sire proofs. Some early methods for converting proofs from one population to another are based on simple linear regression. An alternative robust regression method based on the t-distribution is presented, and maximum likelihood and Bayesian techniques for analysis are described, including the situation in which some proofs are missing. Procedures were used to investigate the relationship between Holstein sire proofs obtained by two Uruguayan genetic evaluation programs. The results suggest that conversion equations developed from data including only sires having proofs in both populations can lead to distorted results, relative to estimates obtained using techniques for incomplete data. There was evidence of non-normality of regression residuals, which constitutes an additional source of bias. A robust estimator may not solve all problems, but can provide simple conversion equations that are less sensitive to outlying proofs and to departures from assumptions. © 2001 Taylor & Francis.","<b>Bioequivalence study of theophylline considering Michaelis-Menten kinetics<\/b><br><br>A bioequivalence study between two controlled-release forms of theophylline was performed, taking into account its non linear elimination kinetics. Three hundred mg of drug were given to twelve human volunteers with a randomized and crossover design. The elimination parameters were estimated from the oral administration of a solution of Theophylline. Area under the curve (AUC), experimental peak concentration (Cmax), bioavailable dose (FD/Vd) and half value duration (HVD) were used as bioavailability parameters. Analysis of variance, bayesian approach for confidence intervals and bioequivalence two-one sided test, were performed."],"shape":["dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot","dot"],"color":["#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#3366CC","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633","#CC6633"],"doi":[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"10.1186/s12879-022-07262-0","10.1016/j.virusres.2022.198858","10.1016/j.smallrumres.2022.106692","10.3390/pathogens11050489","10.1111/jbi.14320","10.1007/s11250-021-03039-w","10.1007/s13127-022-00586-4","10.1186/s12864-020-07316-0","10.1093/sysbio/syab023","10.3390/microorganisms9102165","10.1007/s00436-021-07305-6","10.1111/jbi.14231","10.1016/j.coastaleng.2021.103944","10.1016/j.envpol.2021.117308","10.3390/v13091801","10.1016/j.ympev.2021.107113","10.3389/fsufs.2021.547301","10.1093/jmammal/gyab043","10.3390/su13116318","10.1016/j.ecolind.2021.107506","10.1016/j.jsames.2021.103189","10.1016/j.ecolmodel.2021.109504","10.1093/comnet/cnab022","10.1111/eff.12575","10.1016/j.gene.2020.145352","10.5713/ajas.19.0960","10.1109/URUCON53396.2021.9647280","10.1016/j.crpvbd.2021.100022","10.1155/2021/3108749","10.3390/ani11010128","10.1177/1040638720975742","10.1016/j.vetpar.2020.109272","10.1002/jmv.26018","10.15232/aas.2019-01965","10.1002/jmv.25627","10.1534/g3.120.401194","10.3390/genes11070790","10.1007/s00705-020-04624-4","10.1016/j.ympev.2020.106819","10.1016/j.watres.2020.115710","10.1590/1984-70332020v20n1s9","10.7717/peerj.8452","10.1109/TPWRS.2019.2928276","10.1136/vr.105336","10.1111/zsc.12377","10.1111/iere.12400","10.1016/j.scitotenv.2019.05.485","10.1080/01650521.2019.1669421","10.4490/algae.2019.34.7.22","10.1016/j.ympev.2019.05.024","10.3389/fvets.2019.00261","10.5194/esd-10-347-2019","10.1007/s11783-019-1128-1","10.1016/j.ttbdis.2019.04.009","10.1093/sysbio/syy058","10.1016/j.ympev.2018.11.022","10.1007/s10922-018-9464-1","10.17660/ActaHortic.2019.1261.13","10.1590/0001-3765201920190325","10.3934/mbe.2019389","10.1071/WR17188","10.1093/botlinnean/boy062","10.3390/v10120698","10.1038/s41598-018-34100-8","10.1038/s41598-018-26824-4","10.1016/j.ympev.2018.04.025","10.3389/fpsyg.2018.01194","10.2135/cropsci2017.12.0752","10.3389/fgene.2018.00195","10.1007/s12560-017-9334-0","10.3389/fmicb.2018.01738","10.3390/d10010011","10.1109/ICIP.2017.8296482","10.1093/jmammal/gyx152","10.1186/s13071-017-2562-1","10.1186/s12862-017-1089-6","10.1002/jmv.24910","10.3732/ajb.1700225","10.1371/journal.pone.0187983","10.1590/0074-02760170299","10.1002/ece3.3261","10.1016/j.gene.2017.07.057","10.1016/j.ttbdis.2017.04.015","10.11646/zootaxa.4294.1.3","10.1115/1.4035918","10.1016/j.jembe.2016.10.017","10.1016/j.meegid.2016.05.017","10.1016/j.ecss.2016.06.023","10.1016/j.ces.2016.06.019","10.1002/jmv.24596","10.1016/j.meegid.2016.06.003","10.3389/fgene.2016.00151","","10.1007/s10044-015-0508-9","10.1094/PHYTO-09-15-0222-R","10.1007/s12237-016-0075-3","10.4238/gmr.15027471","10.1111/mec.13585","10.1111/jbg.12180","10.1038/hdy.2015.106","10.1186/s12711-015-0178-5","10.2527/jas.2015-9619","","","10.1371/journal.pone.0147857","10.1016/j.mambio.2015.08.003","10.1590/S1984-46702015000600003","10.1017/S1479262114000963","10.1016/j.ympev.2015.06.015","10.2527/jas.2015-9208","10.1007/s00705-015-2434-y","10.1371/journal.pone.0134850","10.3390/v7072814","10.1111/mms.12199","10.2527/jas.2014-8832","10.3354/meps11236","10.1111/jeea.12116","10.1017/S1751731114002833","10.1007/s10641-014-0284-1","10.1097/QAD.0000000000000770","10.1111/ICAD.12079","10.12705/634.27","10.1371/journal.pone.0098908","10.1371/journal.pone.0094625","10.1371/journal.pone.0092988","10.3168/jds.2013-6916","","10.1111/mec.12632","10.1186/s12863-014-0141-4","10.1016/j.actatropica.2014.04.002","10.1016/j.actatropica.2014.04.003","10.1111/zsc.12069","10.1016/j.livsci.2014.04.029","10.1371/journal.pone.0080929","10.11646/zootaxa.3736.2.3","10.1371/journal.pone.0080019","10.1371/journal.pone.0076168","10.1016/j.livsci.2013.07.011","10.1371/journal.pone.0074072","10.1111/mec.12369","10.1590/0074-0276108052013002","10.1099/mic.0.061952-0","10.1371/journal.pone.0058315","10.1016/j.ympev.2012.12.001","10.1016/j.virusres.2012.09.002","10.1093/nar/gks884","10.1111/j.1558-5646.2012.01640.x","10.1111/j.1365-2699.2012.02722.x","10.1016/j.livsci.2012.05.003","10.1017/S0007485311000678","10.1007/s00227-011-1830-4","10.3732/ajb.1100152","10.1002/tax.611013","10.1016/j.biocon.2011.10.009","10.1093/jhered/esr103","","","10.1016/j.ympev.2011.06.014","10.3982/ECTA8583","10.1016/j.eswa.2011.02.055","10.1644/10-MAMM-A-121.1","10.2527/jas.2010-3071","10.1007/978-90-481-3658-2_45","10.3168/jds.2010-3301","10.1186/1743-422X-7-329","10.3989/scimar.2010.74n3589","10.1186/1742-4690-7-22","10.1186/1471-2156-11-9","10.1371/journal.pone.0014315","10.1017/S0016672310000261","10.3923/ijps.2010.490.494","10.3923/ijps.2010.217.220","10.3923/ijps.2010.167.170","10.1186/1743-422X-6-215","10.1016/j.meegid.2009.06.014","10.1534/genetics.109.101501","10.1016/j.virusres.2009.01.003","10.3168/jds.2009-2092","","10.1007/s00227-008-1030-z","10.1590/s1679-62252008000100009","10.1051/gse:2008010","10.1099/vir.0.83038-0","10.1017/S1751731107000390","10.1590/s0103-84782007000500029","10.1099/vir.0.81576-0","10.1093/molbev/msj023","10.1016/S0301-6226(03)00051-4","10.1017/s1357729800055752","10.1080/090647001300004763a",""],"issn":[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"14712334","01681702","09214488","20760817","03050270","00494747","14396092","14712164","10635157","20762607","09320113","03050270","03783839","02697491","19994915","10557903","2571581X","00222372","20711050","1470160X","08959811","03043800","20511310","09066691","03781119","27650189","","2667114X","23146133","20762615","10406387","03044017","01466615","25902873","01466615","21601836","20734425","03048608","10557903","00431354","15187853","21678359","08858950","00424900","03003256","00206598","00489697","01650521","12262617","10557903","22971769","21904979","20952201","1877959X","10635157","10557903","10647570","05677572","00013765","15471063","10353712","00244074","19994915","20452322","20452322","10557903","16641078","0011183X","16648021","18670334","1664302X","14242818","15224880","00222372","17563305","14712148","01466615","00029122","19326203","00740276","20457758","03781119","1877959X","11755326","23772158","00220981","15671348","02727714","00092509","01466615","15671348","16648021","03275477","14337541","0031949X","15592723","16765680","09621083","09312668","0018067X","0999193X","00218812","01613782","03279383","19326203","16165047","19844670","14792621","10557903","00218812","03048608","19326203","19994915","08240469","00218812","01718630","15424766","17517311","03781909","02699370","1752458X","00400262","19326203","19326203","19326203","15253198","15220613","1365294X","14712156","0001706X","0001706X","03003256","18711413","19326203","11755334","19326203","19326203","18711413","19326203","1365294X","16788060","14652080","19326203","10959513","18727492","13624962","15585646","13652699","18711413","14752670","00253162","00029122","00400262","00063207","14657333","","","10959513","14680262","09574174","15451542","15253163","","15253198","1743422X","18868134","17424690","14712156","19326203","00166723","16828356","16828356","16828356","1743422X","15671348","00166731","01681702","00220302","","00253162","16796225","0999193X","00221317","17517311","01038478","00221317","15371719","03016226","13577298","09064702","03262383"],"abstract":[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"Background: Knowing the age-specific rates at which individuals infected with SARS-CoV-2 develop severe and critical disease is essential for designing public policy, for infectious disease modeling, and for individual risk evaluation. Methods: In this study, we present the first estimates of these rates using multi-country serology studies, and public data on hospital admissions and mortality from early to mid-2020. We combine these under a Bayesian framework that accounts for the high heterogeneity between data sources and their respective uncertainties. We also validate our results using an indirect method based on infection fatality rates and hospital mortality data. Results: Our results show that the risk of severe and critical disease increases exponentially with age, but much less steeply than the risk of fatal illness. We also show that our results are consistent across several robustness checks. Conclusion: A complete evaluation of the risks of SARS-CoV-2 for health must take non-fatal disease outcomes into account, particularly in young populations where they can be 2 orders of magnitude more frequent than deaths. © 2022, The Author(s).","Canine distemper virus (CDV) is a Morbillivirus (Canine morbillivirus) that greatly impacts domestic and wildlife carnivores worldwide. The CDV RNA genome has high genetic variability, evidenced by several lineages that follow a global geographic pattern. The evolutionary trajectories and population dynamics of CDV lineages are still unclear and debatable, particularly in South America, where relatively few sequences are available. We performed phylogenetic and Bayesian analyses using an updated dataset of the highly variable hemagglutinin (H) gene, including seven South American countries. The time to the most recent common ancestor (tMRCA) of the current CDV lineages was dated to the early 1900s in North America. Maximum likelihood and Bayesian maximum clade credibility phylogenies showed similar topologies with two main branches (L1 and L2) corresponding to the NA1 lineage (L1) and the remaining lineages worldwide (L2). The four circulating lineages in South America (EU1/SA1, SA2, SA3, NA4/SA4) arose from independent migration events from North America and Europe. North American strains colonized most northern South American countries via Ecuador and then Colombia and Peru, originating the SA3 and NA4/SA4 lineages during their spread. The entry and expansion in the southern part of South America (Argentina, Brazil, Chile, and Uruguay) occurred through three independent migration events and gave rise to the EU1/SA1 and SA2 lineages. South American lineages have specific combinations of amino acids under positive selection that constitute signatures of taxonomic and evolutionary relevance. Our findings provide a comprehensive scenario for the origin and migration routes of Canine morbillivirus in South America and highlight the importance of phylodynamics in understanding the geographic patterns of modern genetic variability. © 2022 Elsevier B.V.","Scoured wool colour determines the potenti al colour range and dye-ability of wool tops. Improvement of scoured wool colour of fleeces from Corriedale sheep through multi-trait breeding programmes is currently not possible due to limited existing information on its heritability and genetic correlations with other fleece traits. The present study aimed to determine genetic parameters of scoured wool colour traits, with emphasis on their genetic and phenotypic correlations with objectively and visually assessed fleece traits. Data from 1181 hoggets of two experimental flocks were used. Heritabilities were estimated for scoured wool yellowness (Y-Z) and scoured wool brightness (Y), greasy fleece weight (GFW) and clean fleece weight (CFW), wool yield (WY), mean fibre diameter (MFD), coefficient of variation of fibre diameter (CVD), staple length (SL), staple strength (SS) and a range of visual wool traits (greasy wool colour, wool character, handle, staple formation, dust penetration and cross-linking). Trivariate statistical models under a Bayesian setting were used, including the environmental effects of flock, sex, litter size and climatic variables (accumulated precipitation, maximum temperature and relative humidity) and the animal genetic effect, Y-Z was moderately heritable (0.22 ± 0.06) and should respond to selection. Its genetic correlations with GFW, CFW and SL were significant, positive (unfavourable) and moderate (0.43 ± 0.15, 0.56 ± 0.14, and 0.44 ± 0.14, respectively). There was a positive (favourable) association between MFD and Y-Z (0.25 ± 0.17). The genetic correlation between Y-Z and greasy wool colour was 0.70 ± 0.13, although this trait showed a low heritability value (0.13 ± 0.04). Wool brightness showed low heritability (0.10 ± 0.04) and only appeared to be genetically correlated with SL and wool character (0.56 ± 0.17 and 0.39 ± 0.22, respectively). Phenotypic correlations of Y-Z and Y with wool production and quality traits were all less than 0.20 in size. Overall, this study reports new genetic parameters for wool colour and their associations with fleece traits in Corriedale sheep. Our findings suggest that it could be possible to select against wool yellowness, with some caution, given the possible detrimental effects on economically important wool traits, especially fleece weight and staple length. More investigation is warranted. Indirect selection via greasy wool colour is an alternative approach, but the responses would be slower. Easy and cheap measurement should be balanced against the size of the response from direct selection versus indirect selection. © 2022 Elsevier B.V.","Chronic wasting disease (CWD) is a transmissible prion disease of the cervidae family. ELISA and IHC tests performed postmortem on the medial retropharyngeal lymph nodes (RPLN) or obex are considered diagnostic gold standards for prion detection. However, differences in CWD transmission, stage of infection, pathogenesis, and strain can limit performance. To overcome these uncertainties, we used Bayesian statistics to assess the accuracy of RT-QuIC, an increasingly used prion amplification assay, to diagnose CWD on tonsil (TLN), parotid (PLN) and submandibular lymph nodes (SMLN), and ELISA/IHC on RPLN of white-tailed deer (WTD) sampled from Minnesota. Dichotomous RT-QuIC and ELISA/IHC results from wild (n = 61) and captive (n = 46) WTD were analyzed with two-dependent-test, one-population models. RT-QuIC performed on TLN and SMLN of the wild WTD population had similar sensitivity (median range (MR): 92.2–95.1) to ELISA/IHC on RPLN (MR: 91.1–92.3). Slightly lower (4–7%) sensitivity estimates were obtained from farmed animal and PLN models. RT-QuIC specificity estimates were high (MR: 94.5–98.5%) and similar to ELISA/IHC estimates (MR: 95.7–97.6%) in all models. This study offers new insights on RT-QuIC and ELISA/IHC performance at the population level and under field conditions, an important step in CWD diagnosis and management. © 2022 by the authors. Licensee MDPI, Basel, Switzerland.","Aim: Geographical patterns of montane biodiversity worldwide are related to biotic and abiotic factors, such as historical climate dynamics and species dispersal capabilities, which affect the biota from population to community levels. Understanding of processes related to population diversification in extra-Andean Neotropical mountains remains largely unknown. Here, we tested how colonization of new areas as opposed to fragmentation of geographical ranges influenced the distribution and diversification of frogs from Brazilian mountains. Location: Espinhaço Range, the largest extra-Andean mountain range in South America. Taxon: Ledge treefrog Bokermannohyla saxicola. Methods: We used multilocus DNA and morphometric data throughout the species' distribution to delimit lineages, infer phylogenetic relationships and estimate divergence times. We used ecological niche modelling (ENM) and approximate Bayesian computation (ABC) to reconstruct changes of geographical distribution and population sizes for testing alternative hypotheses of diversification concerning fragmentation and colonization. Results: We found four evolutionary significant lineages that diverged from the Pliocene to the Early Pleistocene. All lineages were validated by DNA and morphometric data by independent and joint analyses. ENM showed that climatic fluctuations might have influenced the species' distribution, whereas ABC model selection further supports associated demographic changes. Taken together, a Pleistocene jump-dispersal scenario best explains the diversification of the Bokermannohyla saxicola lineages. Main conclusions: Endemism in the Espinhaço Range is usually explained by ancient diversification, associated with long-term isolation of climatic stable areas following climate-driven habitat fragmentation. Our results challenge this general view, indicating recent diversification, habitat expansion and colonization of new areas as important processes explaining the current distribution and genetic diversity of B. saxicola. © 2022 John Wiley & Sons Ltd.","The sensitivity (Se) and specificity (Sp) of three diagnostic tests for the detection of Campylobacter fetus venerealis (Cfv) using field samples were estimated using a Bayesian latent class model (BLCM), accounting for the absence of a gold standard. The tests included in this study were direct immunofluorescence antibody test (IFAT), polymerase chain reaction (PCR), and real-time PCR (RT-PCR). Twelve farms from two different populations were selected and bull prepuce samples were collected. The IFAT was performed according to the OIE Manual. The conventional PCR was performed as multiplex, targeting the gene nahE for C. fetus species identification and insertion element ISCfe1 for Cfv identification. The RT-PCR was performed as uniplex: one targeting the gene nahE for C. fetus and the other targeting the insertion ISCfe1 (ISC2) for Cfv. Results from the BLCM showed a median Se of 11.7% (Bayesian credibility interval (BCI): 1.93–29.79%), 53.7% (BCI: 23.1–95.0%), and 36.1% (BCI: 14.5–71.7%) for IFAT, PCR, and RT-PCR respectively. The Sp were 94.5% (BCI: 90.1–97.9%), 97.0% (BCI: 92.9–99.3%), and 98.4% (BCI: 95.3–99.7%) for IFAT, PCR, and RT-PCR respectively. The correlation between PCR and RT-PCR was positive and low in samples from both sampled population (0.63% vs 8.47%). These results suggest that diagnostic sensitivity of the studied tests is lower using field samples than using pure Cfv strains. © 2021, The Author(s), under exclusive licence to Springer Nature B.V.","Species are the fundamental category and the key to formulate conservation efforts. DNA and ecological niche modeling have become valuable tools for species delimitation. Wolf spiders include few web-living species, such as Aglaoctenus lagotis (Holmberg, 1876), a priority species for conservation in Uruguay. Behavioral and body coloration patterns of this species have allowed us to distinguish two groups (forms I and II). Here, we combine information from gene trees and multispecies coalescent analyses on mitochondrial (cox1, 12S, 16S + L1 + nad1) and nuclear (intron tif5A) DNA sequences, as well as from ecological niches comparisons, in order to clarify their taxonomic identity. We worked with localities in Uruguay and Argentina, including sympatric and allopatric areas. Gene trees were inferred with Maximum Likelihood, Bayesian, and statistical parsimony analyses. Molecular species delimitation analyses were conducted, and the species tree and divergence times were co-estimated. Characterization and comparison of the climatic requirements of both forms throughout annual and sexual periods were analyzed. Species delimitation and species tree analyses recovered three main lineages (Form I, Form IIa, and Form IIb). Form I is restricted to Uruguay and is closely related and sympatric with Form IIa. Form IIb is located in Argentina and in the Uruguayan west coast, generating a sympatric area of the three forms. Regarding to the sexual climatic niche, the three main lineages differ and do not overlap. Our results support the existence of more than one lineage within what is nowadays Aglaoctenus lagotis. Possible evolving processes explaining this scenario and the conservation consequences are discussed. © 2022, Gesellschaft für Biologische Systematik.","Background: Streptococcus pneumoniae serotype 1 remains a leading cause of invasive pneumococcal diseases, even in countries with PCV-10/PCV-13 vaccine implementation. The main objective of this study, which is part of the Pneumococcal African Genome project (PAGe), was to determine the phylogenetic relationships of serotype 1 isolates recovered from children patients in Casablanca (Morocco), compared to these from other African countries; and to investigate the contribution of accessory genes and recombination events to the genetic diversity of this serotype. Results: The genome average size of the six-pneumococcus serotype 1 from Casablanca was 2,227,119 bp, and the average content of coding sequences was 2113, ranging from 2041 to 2161. Pangenome analysis of the 80 genomes used in this study revealed 1685 core genes and 1805 accessory genes. The phylogenetic tree based on core genes and the hierarchical bayesian clustering analysis revealed five sublineages with a phylogeographic structure by country. The Moroccan strains cluster in two different lineages, the five invasive strains clusters altogether in a divergent clade distantly related to the non-invasive strain, that cluster with all the serotype 1 genomes from Africa. Conclusions: The whole genome sequencing provides increased resolution analysis of the highly virulent serotype 1 in Casablanca, Morocco. Our results are concordant with previous works, showing that the phylogeography of S. pneumoniae serotype 1 is structured by country, and despite the small size (six isolates) of the Moroccan sample, our analysis shows the genetic cohesion of the Moroccan invasive isolates. © 2021, The Author(s).","Sigmodontine rodents (Cricetidae, Sigmodontinae) represent the second largest muroid subfamily and the most species-rich group of New World mammals, encompassing above 410 living species and ca. 87 genera. Even with advances on the clarification of sigmodontine phylogenetic relationships that have been made recently, the phylogenetic relationships among the 12 main groups of genera (i.e., tribes) remain poorly resolved, in particular among those forming the large clade Oryzomyalia. This pattern has been interpreted as consequence of a rapid radiation upon the group entrance into South America. Here, we attempted to resolve phylogenetic relationships within Sigmodontinae using target capture and high-throughput sequencing of ultraconserved elements (UCEs). We enriched and sequenced UCEs for 56 individuals and collected data from four already available genomes. Analyses of distinct data sets, based on the capture of 4634 loci, resulted in a highly resolved phylogeny consistent across different methods. Coalescent species-tree-based approaches, concatenated matrices, and Bayesian analyses recovered similar topologies that were congruent at the resolution of difficult nodes. We recovered good support for the intertribal relationships within Oryzomyalia; for instance, the tribe Oryzomyini appears as the sister taxa of the remaining oryzomyalid tribes. The estimates of divergence times agree with the results of previous studies. We inferred the crown age of the sigmodontine rodents at the end of the Middle Miocene, while the main lineages of Oryzomyalia appear to have radiated in a short interval during the Late Miocene. Thus, the collection of a genomic-scale data set with a wide taxonomic sampling provided resolution for the first time of the relationships among the main lineages of Sigmodontinae. We expect the phylogeny presented here will become the backbone for future systematic and evolutionary studies of the group. © 2021 The Author(s). Published by Oxford University Press, on behalf of the Society of Systematic Biologists. All rights reserved.","Human ehrlichiosis are scantily documented in Uruguay. The aim of this study was to investigate the presence of Ehrlichia spp. in Haemaphysalis juxtakochi and in a gray brocket deer (Mazama gouazoubira) from Uruguay. The presence of Ehrlichia DNA was investigated in free-living H. juxtakochi in five localities of southeast and northeast Uruguay, as well as blood, spleen, and ticks retrieved from a M. gouazoubira. Ehrlichia spp. DNA was detected in six out of 99 tick pools from vegetation, in the spleen of M. gouazoubira, and in one out of five pools of ticks feeding on this cervid. Bayesian inference analyses for three loci (16S rRNA, dsb, and groEL) revealed the presence of a new rickettsial organism, named herein as “Candidatus Ehrlichia pampeana”. This new detected Ehrlichia is phylogenetically related to those found in ticks from Asia, as well as Ehrlichia ewingii from USA and Cameroon. Although the potential pathogenicity of “Ca. E. pampeana” for humans is currently unknown, some eco-epidemiological factors may be relevant to its possible pathogenic role, namely: (i) the phylogenetic closeness with the zoonotic agent E. ewingii, (ii) the evidence of H. juxtakochi parasitizing humans, and (iii) the importance of cervids as reservoirs for zoonotic Ehrlichia spp. The molecular detection of “Ca. E. pampeana” represents the third Ehrlichia genotype described in Uruguay. © 2021 by the authors. Licensee MDPI, Basel, Switzerland.","In South America, apicomplexan parasites of the genus Hepatozoon have been sporadically detected in mammals. Previous studies in wild canids from Brazil and Argentina demonstrated infections by species genetically related to Hepatozoon americanum. The aim of the present work was to detect the presence of Hepatozoon in road-killed foxes encountered in Uruguayan highways. Blood samples from 45 crab-eating (Cerdocyon thous) and 32 grey pampean (Lycalopex gymnocercus) foxes were analyzed by PCR for Hepatozoon 18S rRNA gene. Eight foxes (10.4%) were found to be infected with an H. americanum-like protozoan, an Hepatozoon closely related to H. americanum. Bayesian and maximum-likelihood phylogenetic analyses revealed that the sequences obtained in this study cluster with H. americanum from the United States, and with an H. americanum-like species from dog and foxes from Brazil and Argentina. In the Unites States, H. americanum causes severe disease in dogs. In addition to this, an increasing habitat overlap between dogs and foxes makes the presence of H. americanum-like protozoan in foxes acquires veterinary relevance. This work represents the first report of L. gymnocercus infected with an H. americanum-like protozoan, and of wild canids infected with Hepatozoon in Uruguay. © 2021, The Author(s), under exclusive licence to Springer-Verlag GmbH Germany, part of Springer Nature.","Aim: During the glacial phases, the Andean slopes of southern South America were covered by ice sheets, causing the isolation of many species in glacial refugia. This work aimed to test the number and putative location of glacial refugia in southern South America and establish the sources, timing and direction of post-glacial colonization events. Location: Southern South America, Patagonia. Taxon: Olive mouse (Abrothrix olivacea). Methods: Using exome capture, we genotyped 7339 SNPs in 172 specimens of A. olivacea collected at 21 localities distributed across the steppe, grasslands, and forests in the Patagonian-Fuegian region and seven specimens from two other species of Abrothrix. Based on the effective migration rate, principal component analysis, and admixture composition, we first study the population structure of A. olivacea. The source and direction of colonization events were inferred using the directionality index and standard population genetic analyses. To study the role of the glacial phases, we performed a bayesian demographic analysis to infer divergences times and ancestral population sizes. Results: Population genomic analyses based on SNPs of more than 7000 exons identified three Patagonian genetic clusters: Valdivian forests, continental Magellan forests, and a combination of continental open biomes and Tierra del Fuego. The directionality index supports a single source in northwestern forests, in line with a Valdivian forest refugium, from which the post-glacial colonization may have started. The effective diversity rate, the onset of population expansions and divergence times also support a Valdivian forest refugium and rule out a separate southern refugium in Tierra del Fuego. Main conclusions: Our results support a coastal refugium located in or near the previously proposed Valdivian refugium and reject in situ differentiation in Tierra del Fuego. More in general, our demographical analysis reveals the role of the Last Glacial Maximum in affecting the population dynamics (i.e., contraction and expansion) of A. olivacea and in fostering the geographical diversification of populations. © 2021 John Wiley & Sons Ltd","Automatic calibration of numerical models and systematic quantification of the uncertainties associated with their use is not yet a widespread practice in coastal engineering. This paper proposes the use of a Bayesian algorithm for the automatic calibration and uncertainty quantification of a wave model, in the framework of the dynamical downscaling of off-shore waves to a nearshore project site, where a set of wave measurements is available. A spectral error was defined and used for the definition of the likelihood function used by the algorithm; this allows for improvements in terms of the wave spectra and not only in terms of a restricted set of wave parameter. In addition to the calibration of the model parameters, the methodology also addresses errors coming from boundary conditions; to this end it distinguishes between different wave systems and a set of parameters are defined for the correction of each wave system, reducing the errors introduced into the model by the boundary conditions. A case study in the South Atlantic coast showed the ability of the methodology to calibrate the model, resulting in simulations that properly fit the available measurement, while also providing an estimation of the uncertainties associated with the model results that can be straightforwardly used for probabilistic analysis in the coastal environment. © 2021 Elsevier B.V.","Despite the global occurrence of microplastic contamination on sandy beaches, evidence of microplastic distribution within beaches remains contradictory. When conflicting evidence is used to inform sampling surveys, it increases uncertainty in resulting data. Moreover, it hampers spatially explicit risk characterization of microplastic pollution to intertidal fauna. We aimed to guide sampling designs for microplastic monitoring on beaches, and to quantify macroinfauna exposure to microplastics. Microplastic abundance, quantified between 5 mm–66 μm, lacked a significant zonation across the top sediment layer of sub-terrestrial, upper and lower midlittoral, and swash zones at two sites with varying anthropogenic influence on a microtidal dissipative beach in Uruguay. Microplastic abundance decreased exponentially with increasing grain size, as revealed by Bayesian Poisson regression, although the decrease was less steep compared to prior knowledge regarding sediment – plastic interactions obtained for large (millimeter-sized) industrial pellets. Significant differences in microplastic contamination between the two sites with varying anthropogenic influence likely related to their proximity to a freshwater canal. Corresponding field measurements of body burdens of fibers and irregular particles were significantly lower for the polychaete Euzonus (Thoracophelia) furcifera, despite its preference for finer sediments with higher microplastic loads, compared to the isopods Excirolana braziliensis and Excirolana armata. Results provide critical insights toward representative sampling of microplastics within beach sites. Specifically, we caution against sampling limited to the drift line, and instead recommend: 1) reporting beach morphodynamic characteristics; 2) using clearly defined, ecologically-informed zonation schemes; and 3) accounting for sediment grain size as a covariate to normalize among reported contamination levels. The results contribute valuable baseline data toward realistic exposure landscapes relative to the sediment grain size preferences of macroinfauna, needed to inform laboratory experiments. © 2021 The Author(s)","Uruguay controlled the viral dissemination during the first nine months of the SARS-CoV-2 pandemic. Unfortunately, towards the end of 2020, the number of daily new cases exponentially in-creased. Herein, we analyzed the country-wide genetic diversity of SARS-CoV-2 between November 2020 and April 2021. We identified that the most prevalent viral variant during the first epidemic wave in Uruguay (December 2020–February 2021) was a B.1.1.28 sublineage carrying Spike mutations Q675H + Q677H, now designated as P.6, followed by lineages P.2 and P.7. P.6 probably arose around November 2020, in Montevideo, Uruguay’s capital department, and rapidly spread to other depart-ments, with evidence of further local transmission clusters; it also spread sporadically to the USA and Spain. The more efficient dissemination of lineage P.6 with respect to P.2 and P.7 and the presence of mutations (Q675H and Q677H) in the proximity of the key cleavage site at the S1/S2 boundary suggest that P.6 may be more transmissible than other lineages co-circulating in Uruguay. Although P.6 was replaced by the variant of concern (VOC) P.1 as the predominant lineage in Uruguay since April 2021, the monitoring of the concurrent emergence of Q675H + Q677H in VOCs should be of worldwide interest. © 2021 by the authors. Licensee MDPI, Basel, Switzerland.","We evaluated the role of Quaternary climatic fluctuations on the demographic history and population structure of amphibian species endemic to the ‘campo rupestre’ in the Neotropics, evaluating their distributional shifts, demographic changes, and lineage formation from the end of Pleistocene to present. We chose two anurans endemic to the high-elevation ‘campo rupestre’ in the Espinhaço Range (ER) in northeastern and southeastern Brazil (Bokermannohyla alvarengai and Bokermannohyla oxente), as models to test the role of Quaternary climatic fluctuations over their distribution range in this region. We collected tissue samples throughout their distribution range and used statistical phylogeography to examine processes of divergence and population demography. We generated spatial–temporal reconstructions using Bayesian inference in a coalescent framework in combination with hind-cast projections of species distribution models (SDMs). We also used the results and literature information to test alternative diversification scenarios via approximate Bayesian computation (ABC). Our results show that Quaternary climatic fluctuations influenced the geographic ranges of both species showing population expansion during the last glacial maximum (LGM) and range contraction during interglacial periods, as inferred from selected ABC models and from past projections of SDMs. We recovered Pleistocene diversification for both species occuring in distinctly unique periods for each taxon. An older and range-restricted lineage was recovered in a geographically isolated geological massif, deserving conservation and further taxonomic study. The diversification and distribution of these amphibian species endemic to the Neotropical ‘campo rupestre’ were influenced by Quaternary climatic fluctuations. The expansion of cold adapted species restricted to higher elevations during glacial periods and their concomitant retraction during interglacial periods may have been crucial for producing patterns of species richness and endemism along elevation gradients in tropical and subtropical domains. Such processes may influence the evolution of the biota distributed in heterogeneous landscapes with varied topography. © 2021 Elsevier Inc.","To feed the rising population whilst also preserving ecosystem functions, creative solutions are needed for the ecological intensification of natural grassland-based livestock systems. In Uruguay, natural grasslands are the main nutritional resource for livestock production. In these ecosystems, cattle and sheep graze together all the year round, and grasslands are frequently heavily grazed. Considerable research has been generated concerning grassland management, but there is still no knowledge about the impact of decision rules that supports management actions on long-term ecosystem functioning, at the system level. To meet this deficit, a participatory working group of farmers, researchers, and consultants have developed the GLENCOE platform. This platform is a large-scale facility, supported by INIA-Uruguay, designed to answer the following question: How to intensify the grazing management to improve the sustainability of livestock systems based on natural grasslands? To build the platform three steps were followed: (I) definition of the research problem using a problem tree analysis; (ii) conceptualization of the platform and the design of the grazing systems to be evaluated; and, (iii) spatial allocation of the grazing systems according to the variability of soil, slopes, and seasonal dynamic of vegetation indexes. These criteria were considered across farmlets that were equivalent in the initial stage, allowing causal inferences for the systems trajectories on productive and environmental traits. The platform is composed of three independent farmlets of 50 ha each, where multiparous Hereford cows and Merinos wethers co-graze under three grazing management systems. Each farmlet is managed according to different spatio-temporal decisions of the specific management of vegetation communities, grazing methods, and the stockpile of forage that is allowed by the number of the existing paddocks. Farmlet-1; comprises less decisions (2 paddocks), Farmlet-2; intermediate (8 paddocks), and Farmlet-3; high level of decisions (32 paddocks). This innovative platform will be used as a participatory and interdisciplinary space for research and co-learning of management on processes that can only be observed in long-term evaluations, and at farmlet scale. We expect that this new approach will contribute to the developement and implemention of sustainable grazing management systems in Uruguay. © Copyright © 2021 Devincenzi, Jaurena, Durante, Savian, Ciappesoni, Navajas, Ciganda, Lattanzi and Paruelo.","Felids are among the species most threatened by habitat fragmentation resulting from land-use change. In the Uruguayan Savanna ecoregion, about 30% of natural habitats have been lost, large felids have been eradicated from most of the region, and the impact of anthropogenic threats over the smaller species that remain is unknown. To develop management strategies, it is important to enhance knowledge about species population structure and landscape connectivity, particularly when land-use change will continue and intensify in the next years. In this study, we evaluate the population structure and gene flow of Geoffroy's cat in the Uruguayan Savanna ecoregion. We generated a matrix of 11 microsatellite loci for 70 individuals. Based on Bayesian approaches we found that within the Uruguayan Savanna, Geoffroy's cat shows high levels of genetic variability and no population structure. However, we observed genetic differences between individuals from the Uruguayan Savanna and those from the contiguous ecoregion, the Argentinian Humid Pampa. Four first-generation migrants from Humid Pampa were identified in the Uruguayan Savanna, suggesting a stronger gene flow in the west-east direction. We detected a past bottleneck followed by a subsequent recovery in Geoffroy's cat populations in both ecoregions. These results lay the groundwork to understand the population dynamics and conservation status of Geoffroy's cat in the Uruguayan Savanna ecoregion, and provide baseline data to establish population monitoring. © 2021 The Author(s).","The monitoring of surface-water quality followed by water-quality modeling and analysis are essential for generating effective strategies in surface-water-resource management. However, worldwide, particularly in developing countries, water-quality studies are limited due to the lack of a complete and reliable dataset of surface-water-quality variables. In this context, several statistical and machine-learning models were assessed for imputing water-quality data at six monitoring stations located in the Santa Lucía Chico river (Uruguay), a mixed lotic and lentic river system. The challenge of this study is represented by the high percentage of missing data (between 50% and 70%) and the high temporal and spatial variability that characterizes the water-quality variables. The competing algorithms implement univariate and multivariate imputation methods (inverse distance weighting (IDW), Random Forest Regressor (RFR), Ridge (R), Bayesian Ridge (BR), AdaBoost (AB), Hubber Regressor (HR), Support Vector Regressor (SVR) and K-nearest neighbors Regressor (KNNR)). According to the results, more than 76% of the imputation outcomes are considered ‘satisfactory’ (NSE > 0.45). The imputation performance shows better results at the monitoring stations located inside the reservoir than those positioned along the mainstream. IDW was the model with the best imputation results, followed by RFR, HR and SVR. The approach proposed in this study is expected to aid water-resource researchers and managers in augmenting water-quality datasets and overcoming the missing data issue to increase the number of future studies related to the water-quality matter. © 2021 by the authors. Licensee MDPI, Basel, Switzerland.","The vulnerability of species richness to several factors like, climate change, habitat fragmentation, resource exploitation, etc., poses a challenge to conservation biologists and agencies working to sustain the ecosystem services. Hence, there is a clear need for early warning indicators of species loss generated from empirical data. The tree community of the long-term 50-hectare plot on Barro Colorado Island (BCI), Panama, is one of the most intensively studied in the world. This plot was established in 1981 and fully censused in 1982 then every 5 years from 1985 through 2015. This extensive dataset reveals that some tree species suffered steep population declines. Here we propose an early warning indicator of such tree population crashes and test it against the BCI dataset. The spatial covariance matrices, Cij, of the 20 most abundant tree species in BCI allow us to compute, via MaxEnt, the effective interaction matrices, Jij, among these species for the eight censuses available from 1982 to 2015. For each species i and each census c, the absolute value of the intraspecific competition coefficients Jii(c) are much larger than those of the interspecific interaction coefficients Jij(c) with i ≠ j. We show that this result can be derived from a similar empirical relationship observed for the matrices Cii(c). Our main finding is that for those tree species that suffered steep population declines (of at least 50%), across the eight tree censuses, the drop of Jii is always steeper and occurs before the drop of the corresponding species abundance Ni. Indeed, such sharp declines in Jii occur between 5 and 15 years in advance than comparable declines for Ni, and thus they serve as early warnings of impending population busts. Furthermore, this drop of Jii is linked to the anomalous variance, which is a known early warning of incoming catastrophic shifts. © 2021 The Author(s)","In recent years, the increase in studies on the inner ear anatomy of xenarthrans provided new insights regarding some locomotor and phylogenetic aspects. These works have begun to include fossil specimens, although so far, only two extinct representatives of the suborder Folivora (ground sloths) have been analyzed. In the present study, we present the first reconstruction and description of the bony labyrinth of five extinct specimens of the suborder Cingulata, the glyptodonts Glyptodon, Doedicurus, Panochthus, and Pseudoplohophorus, as well as the pampathere Holmesina, in order to analyze their morphology and evaluate functional and phylogenetic aspects. The anatomical and morphometric comparison shows that the inner ear of glyptodonts is largely similar to that of modern armadillos, particularly sharing many characteristics with Chlamyphorus. While the inner ear of Holmesina is similar to that of glyptodonts, it presents some characteristics that make it more similar to extant armadillos. Regarding some functional aspects inferred from the anatomy of the semicircular canals, glyptodonts would have had low levels of agility, although similar to most extant armadillos, and low head angular velocity. On the other hand, Holmesina would have had an agility level similar to that of the giant armadillo Priodontes, although it would also present low head angular velocity as seen in most extant armadillos. Finally, preliminary parsimony and Bayesian phylogenetic inference analyses based on a reduced set of inner ear characters showed interesting results, with glyptodonts and Holmesina recovered as closely related to Chlamyphorus. These results closely resemble the recent topologies obtained using ancient DNA and markedly contrast with the topologies obtained using more traditional morphological characters, highlighting the importance of inner ear data for the inference of phylogenetic relationships in cases where DNA preservation in fossil taxa is unaccessible. © 2021 Elsevier Ltd","The ability to predict changes in the abundances of the species in ecological communities is essential for sustainable management, biodiversity conservation, and community restoration. We propose a framework to predict such changes. We test our method, which uses the linear Lotka-Volterra equations (LLVE) as well as other empirical predictors (linear least squares regression, quadratic extrapolation, simple exponential smoothing), against the measured abundances of trees from the long-term 50-ha plot on Barro Colorado Island (BCI), along eight censuses. To obtain the parameters of the LLVE -the intrinsic growth rate r and the carrying capacity K of each species and the interspecific interaction matrix A- we first estimate A through the Maximum Entropy (MaxEnt) method. Next, using A as input, we fit r and K. Then, feeding the LLVE with these parameters, we obtain predicted species trajectories along censuses. Since for this particular community the interspecific interaction coefficients are much smaller than the intraspecific ones, keeping only intraspecific competition is enough to predict the evolution of the abundances of several tree species, i.e. the LLVE reduce to a set of uncoupled logistic equations. However, this simplification is not a requirement of the method. We define P-values to establish when the predicted trajectory for a species is statistically significant; this is crucial in determining the set of species over which a particular predictor can be meaningfully applied. To illustrate a possible application of the method, we present our predictions for the abundances of tree species for the currently underway BCI 2020 census, which provide warnings regarding species that are likely to experience important population loss. © 2021 Elsevier B.V.","Financial markets are paradigmatic examples of complex systems and have been compared to ecological networks in which different species (firms) interact and co-evolve. A central object governing species dynamics in ecology is the community matrix, whose elements are closely related to pairwise interspecific interaction coefficients. Using this ecological analogy we propose a method, based on the Maximum Entropy (MaxEnt) principle, that allows us to infer candidates for an economic community matrix from time series data of market values. To assess the usefulness of this picture, we construct community matrices for a set of companies belonging to the Fortune 500 list and perform a community analysis on the resultant networks. This analysis shows these networks to strongly reflect the known industry groupings of the firms. We conclude therefore that our community matrices capture non-Trivial information about the interaction of firms, not immediately apparent from the covariance of market values. We anticipate our approach being useful in elucidating further aspects of market structure, as well as forming the basis of forecasting market dynamics. © 2021 The Author(s). Published by Oxford University Press.","Riparian forest may represent a key subsidy to aquatic food webs. While most research has been done in running waters, the origin of resources subsidising biomass and the role of riparian coverage in food webs remains largely unstudied in reservoirs. This research evaluated the role of forest riparian zones as food web subsidies in a recently formed reservoir of Ijuí River, Brazil. The diet of fish and the origin of carbon and nitrogen fuelling fish biomass was compared between littoral habitats with riparian forests and of open canopy. Sampling for stable isotopes of fish and basal resources and the application of bayesian mixing models showed that most of the fish production originates from terrestrial carbon in this recently formed reservoir. Moreover, an increasing terrestrial support to the biomass was found towards the riparian forest areas (subsidising c.a. 70% vs. around 57% of fish biomass in littoral areas of riparian forest and open canopy, respectively).This study remarked the role of riparian forest areas as energetic subsidises to aquatic ecosystems. Management actions in newly formed reservoirs should include preserving riparian forests to avoid losing natural ecosystem subsidies. © 2020 John Wiley & Sons A/S. Published by John Wiley & Sons Ltd","The convergent evolution of subterranean rodents is an excellent model to study how natural selection operates and the genetic bases of these adaptations, but the study on the different taxa has been very uneven and still insufficient. In the octodontoid caviomorph rodent superfamily there are two independent lineages where they have recently evolved into totally underground lifestyles: the genera Ctenomys (tuco-tucos) and Spalacopus (coruro). The underground habitat is characterized by an hypoxic and hypercapnic atmosphere, thus gas exchange is one of the most important challenges for these animals. The invasion of the underground niche could have modified the selective regimes of proteins involved in the respiration and transport of O2 of these rodents, positively selecting mutations of higher affinity for O2. Here we examine the sequence variation in the beta globin gene in these two lineages, within a robust phylogenetic context. Using different approaches (classical and Bayesian maximum likelihood (PAML/Datamonkey) and alternatives methods (TreeSAAP)) we found at least three sites with evidence of positive selection in underground lineages, especially the basal branch that leads to the Octodontidae family and the branch that leads to the coruro, suggesting some adaptive changes to the underground life. We also found a convergence with another underground rodent, which cannot be identified by the above methods. © 2020","Objective: Considering the importance of dairy farming and the negative effects of heat stress, more tolerant genotypes need to be identified. The objective of this study was to investigate the effect of heat stress via temperature-humidity index (THI) and diurnal temperature variation (DTV) in the genetic evaluations for daily milk yield of Holstein dairy cattle, using random regression models. Methods: The data comprised 94,549 test-day records of 11,294 first parity Holstein cows from Brazil, collected from 1997 to 2013, and bioclimatic data (THI and DTV) from 18 weather stations. Least square linear regression models were used to determine the THI and DTV thresholds for milk yield losses caused by heat stress. In addition to the standard model (SM, without bioclimatic variables), THI and DTV were combined in various ways and tested for different days, totaling 41 models. Results: The THI and DTV thresholds for milk yield losses was THI = 74 (-0.106 kg/d/THI) and DTV = 13 (-0.045 kg/d/DTV). The model that included THI and DTV as fixed effects, considering the two-day average, presented better fit (-2logL, Akaike information criterion, and Bayesian information criterion). The estimated breeding values (EBVs) and the reliabilities of the EBVs improved when using this model. Conclusion: Sires are re-ranking when heat stress indicators are included in the model. Genetic evaluation using the mean of two days of THI and DTV as fixed effect, improved EBVs and EBVs reliability. © 2021 by Animal Bioscience.","The differential diagnosis of respiratory diseases is usually a challenge for medical specialists in the first line of care, increased under the current COVID-19 pandemic. A Clinical Decision Support System-CDSS-is being developed using Bayesian Networks-BNs-to help physicians diagnose respiratory diseases, including those related to COVID-19. Network structure has been elicited from expert physicians, and network parameters (diseases prevalence, symptoms, findings, and lab results conditional probabilities) were extracted from relevant bibliography or currently standard global information sources. The CDSS is being tested using case studies taken from real situations, provided and validated by physicians. The resulting system demonstrates the suitability and flexibility of BNs for diagnosis support and healthcare training. © 2021 IEEE.","Ehrlichia are small intracellular Gram-negative bacteria transmitted by ticks. These microorganisms cause ehrlichiosis, a complex of life-threatening emerging zoonoses and diseases of global veterinary relevance. The aim of this study was to investigate the presence of Ehrlichia in free-living Ixodes auritulus collected in Uruguay. Ticks were collected from vegetation in five localities from the southeast and northeast of the country between 2014 and 2017. Detection of Ehrlichia DNA was performed in pools of adults or nymphs grouped according to the collection site and date. A total of 1,548 I. auritulus ticks were collected in four of the five locations sampled. Fragments of three loci (16S rRNA, dsb and groEL) were obtained by PCR, and phylogenies inferred using Bayesian inference analysis for each gene independently. DNA of Ehrlichia spp. was found in 15 out of 42 tick pools. Based on the topology of the phylogenetic trees, our sequences represent two novel genotypes for the genus named as Ehrlichia sp. Serrana and Ehrlichia sp. Laguna Negra. Both genotypes were closely related to Ehrlichia sp. Magellanica, a species detected in Ixodes uriae and Magellanic penguins. Considering that all stages of I. auritulus and I. uriae are parasites of birds, their phylogenetic relationships, and common eco–epidemiological profiles, it is reasonable to state that these genotypes of Ehrlichia spp. may represent a natural group likely associated with birds. Our results constitute the first characterization of Ehrlichia spp. in Uruguay. Future studies on birds reported as hosts for I. auritulus are needed to further understand the epidemiological cycles of both Ehrlichia genotypes in the country. Finally, I. auritulus does not feed on humans, so the two Ehrlichia species reported herein might have no implications in human health. © 2021 The Author(s)","Aim. To develop a population pharmacokinetic model for Uruguayan patients under treatment with cyclosporine (CsA) that can be applied to TDM. Patients and Methods. A total of 53 patients under treatment with CsA were included. 37 patients with at least one pharmacokinetic profile described with four samples were considered for model building, while the remaining 16 were considered for the assessments of predictive performances. Pharmacokinetic parameter estimation was performed using a nonlinear mixed effect modelling implemented in the Monolix® software (version 2019R1, Lixoft, France); meanwhile, simulations were performed in R v.3.6.0 with the mlxR package. Results. A two-compartment model with a first-order disposition model including lag time was used as a structural model. The final model was internally validated using prediction corrected visual predictive check (pcVPC) and other graphical diagnostics. A total of 621 CsA steady-state concentrations were analyzed for model development. Population estimates for the absorption constant (ka) and lag time were 0.523 h-1 and 0.512 h, respectively; apparent clearance (CL/F) was 30.3 L/h (relative standard error RSE±8.25%) with an interindividual variability of 39.8% and interoccasion variability of 38.0%; meanwhile, apparent clearance of distribution (Q/F) was 17.0 L/h (RSE±12.1%) with and interindividual variability of 53.2%. The covariate analysis identified creatinine clearance (ClCrea) as an individual factor influencing the Cl of CsA. The predictive capacity of the population model was demonstrated to be effective since predictions made for new patients were accurate for C1 and C2 (MPPEs below 50%). Bayesian forecasting improved significantly in the second and third occasions. Conclusion. A population pharmacokinetic model was developed to reasonably estimate the individual cyclosporine clearance for patients. Hence, it can be utilized to individualize CsA doses for prompt and adequate achievement of target blood concentrations of CsA. © 2021 Martín Umpiérrez et al.","Intense selection for milk yield has increased environmental sensitivity in animals, and currently, heat stress is an expensive problem in dairy farming. The objectives were to identify the best model for characterizing environmental sensitivity in Holstein cattle, using the test-day milk yield (TDMY) combined with the temperature–humidity index (THI), and identify sires genetically superior for heat-stress (HS) tolerance and milk yield, through random regression. The data comprised 94,549 TDMYs of 11,294 first-parity Holstein cows in Brazil, collected from 1997 to 2013. The yield data were fitted to Legendre orthogonal polynomials, linear splines and the Wilmink function. The THI (the average of two days before the dairy control) was used as an environmental gradient. An animal model that fitted production using a Legendre polynomials of quartic order for the days in milk and quadratic equations for the THI presented a better quality of fit (Akaike’s information criterion (AIC) and Bayesian information criterion (BIC)). The Spearman correlation coefficient of greatest impact was 0.54, between the top 1% for TDMY and top 1% for HS. Only 9% of the sires showed plasticity and an aptitude for joint selection. Thus, despite the small population fraction allowed for joint selection, sufficient genetic variability for selecting more resilient sires was found, which promoted concomitant genetic gains in milk yield and thermotolerance. © 2021 by the authors. Licensee MDPI, Basel, Switzerland.","Detection of bovine Babesia spp. and Anaplasma marginale is based on the reading of Giemsa-stained blood or organ smears, which can have low sensitivity. Our aim was to improve the detection of bovine Babesia spp. and A. marginale by validating a multiplex PCR (mPCR). We used 466 samples of blood and/or organs of animals with signs and presumptive autopsy findings of babesiosis or anaplasmosis. The primers in our mPCR amplified the rap-1a gene region of Babesia bovis and B. bigemina, and the msp-5 region of A. marginale. We used a Bayesian model with a non-informative priori distribution for the prevalence estimate and informative priori distribution for estimation of sensitivity and specificity. The sensitivity and specificity for smear detection of Babesia spp. were 68.6% and 99.1%, and for A. marginale 85.6% and 98.8%, respectively. Sensitivity and specificity for mPCR detection for Babesia spp. were 94.2% and 97.1%, and for A. marginale 95.2% and 92.7%, respectively. Our mPCR had good accuracy in detecting Babesia spp. and A. marginale, and would be a reliable test for veterinarians to choose the correct treatment for each agent. © 2020 The Author(s).","Characterized as the most important gastrointestinal parasite (GIP) in the world, Haemonchus contortus is also the most predominant and pathogenic genus in Uruguay, causing high morbidity and mortality rates in sheep flocks. For approximately 26 years resistance to GIP has been included in the National Genetic Evaluation (NGE) of Corriedale and Merino breeds, using Faecal Egg Count (FEC) as the selection criterion. The aim of the present study was to estimate genetic parameters for FEC, post-weaning Body Weight (BW), Packed Cell Volume (PCV), FAMACHA© and Body Condition Score (BCS), to determine the possibility to include these traits in the NGE of GIP resistance and resilience, as complementary selection criteria and to investigate the sensitivity of breeding values to different environmental worm burden challenges. Data were collected on 19,510 lambs from 2000 to 2009 in 24 Corriedale studs that participated in the NGE of sheep in Uruguay. (Co)variances components were estimated using a multi-trait animal model performing a Bayesian analysis. Posterior means estimated for heritability (h2) were of low to moderate magnitude (between 0.10 and 0.33) for the traits analysed. Genetic correlations (rg) were not significantly different from zero for BW- Loge (FEC + 100), BW-FAMACHA©, BW-PCV, Loge (FEC + 100)-PCV, BCS-FAMACHA©, moderate favourable rg for Loge (FEC + 100)-FAMACHA and PCV-BCS (0.55, 0.41) and high favourable rg for BW-BCS (0.66). These results indicate the possibility to complement selection programs incorporating BW, Loge (FEC + 100), FAMACHA©, PCV and BCS in order to select resistant and resilient animal to GIP. Furthermore, genetic parameters for BW and Loge (FEC + 100) to different environmental worm burden challenge was investigated. Two contemporary groups of low worm environment (Loge (FEC + 100) below 6.40) and a high worm environment (Loge (FEC + 100) above 6.75) were considered. In this model, h2 estimated for BW and Loge (FEC + 100) were found to be not statistically different between low and high environmental worm burden, furthermore, high correlations between traits at both environments were also presented. © 2020 Elsevier B.V.","On 30th January 2020, an outbreak of atypical pneumonia caused by a novel betacoronavirus, named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), was declared a public health emergency of international concern by the World Health Organization. For this reason, a detailed evolutionary analysis of SARS-CoV-2 strains currently circulating in different geographic regions of the world was performed. A compositional analysis as well as a Bayesian coalescent analysis of complete genome sequences of SARS-CoV-2 strains recently isolated in Europe, North America, South America, and Asia was performed. The results of these studies revealed a diversification of SARS-CoV-2 strains in three different genetic clades. Co-circulation of different clades in different countries, as well as different genetic lineages within different clades were observed. The time of the most recent common ancestor was established to be around 1st November 2019. A mean rate of evolution of 6.57 × 10−4 substitutions per site per year was found. A significant migration rate per genetic lineage per year from Europe to South America was also observed. The results of these studies revealed an increasing diversification of SARS-CoV-2 strains. High evolutionary rates and fast population growth characterizes the population dynamics of SARS-CoV-2 strains. © 2020 Wiley Periodicals LLC","Objective: The objective was to characterize the lactation curve applying 3 different models using multiparous grazing beef cows. Materials and Methods: Milk production data from 99 British crossbred multiparous cows grazing native pastures were analyzed. Lactation was assessed 15 d postpartum and then monthly until weaning (180 d postpartum) using a milking machine after an oxytocin injection. Total milk production and lactation curve were characterized using Wood (WD) and Wilmink (WIL) models, and both were compared with spline functions. Comparison was made applying adjusted coefficient of determination (R2adj) and MSE. Results and Discussion: Cubic splines with 5 equally spaced knots (CS5) presented the best adjustment (lowest Akaike information criterion and Bayesian information criterion). The R2adj values were 0.55, 0.54, and 0.53 (the greater the better) and MSE values were 2.54, 2.59, and 2.47 (the lower the better) for WD, WIL, and CS5, respectively. Estimated milk production for the lactation period was 1,277, 1,255, and 1,195 kg for WD, WIL, and CS5, respectively. Milk peak was predicted to happen at 32, 25, and 36 d postpartum, with a production of 8.74, 8.21, and 8.40 kg for WD, WIL, and CS5, respectively. No differences were evident in the lactation curves (95% CI). Implications and Applications: The proposed method and frequency used to assess grazing beef cattle milk production accurately estimate the lactation curve. The Wood model, used worldwide, was a precise estimator of the lactation curve, which in our case was verified applying splines. These results provide key information to calculate grazing beef cow requirements. © 2020 American Registry of Professional Animal Scientists","Abstract: On July 19, 2019, the World Health Organization declared the current Ebolavirus (EBOV) outbreak in Congo Democratic Republic (COD) a public health emergency of international concern. To address the potential threat of EBOV evolution outpacing antibody treatment and vaccine efforts, a detailed evolutionary analysis of EBOV strains circulating in different African countries was performed. Genome composition of EBOV strains was studied using multivariate statistical analysis. To investigate the patterns of evolution of EBOV strains, a Bayesian Markov Chain Monte Carlo approach was used. Two different genetic lineages, with a distinct genome composition gave rise to the recent EBOV outbreaks in central and western Africa. Strains isolated in COD in 2018 fall into two different genetic clusters, according to their geographical location of isolation. Different amino acid substitutions among strains from these two clusters have been found, particularly in NP, GP, and L proteins. Significant differences in codon and amino acid usage among clusters were found. Strains isolated in COD in 2018 belong to two distinct genetic clusters, with distinct codon and amino acid usage. Geographical diversity plays an important role in shaping the molecular evolution of EBOV populations. © 2019 Wiley Periodicals, Inc.","Quantitative traits are important targets of both natural and artificial selection. The genetic architecture of these traits and its change during the adaptive process is thus of fundamental interest. The fate of the additive effects of variants underlying a trait receives particular attention because they constitute the genetic variation component that is transferred from parents to offspring and thus governs the response to selection. While estimation of this component of phenotypic variation is challenging, the increasing availability of dense molecular markers puts it within reach. Inbred plant species offer an additional advantage because phenotypes of genetically identical individuals can be measured in replicate. This makes it possible to estimate marker effects separately from the contribution of the genetic background not captured by genotyped loci. We focused on root growth in domesticated rice, Oryza sativa, under normal and aluminum (Al) stress conditions, a trait under recent selection because it correlates with survival under drought. A dense single nucleotide polymorphism (SNP) map is available for all accessions studied. Taking advantage of this map and a set of Bayesian models, we assessed additive marker effects. While total genetic variation accounted for a large proportion of phenotypic variance, marker effects contributed little information, particularly in the Al-tolerant tropical japonica population of rice. We were unable to identify any loci associated with root growth in this population. Models estimating the aggregate effects of all measured genotypes likewise produced low estimates of marker heritability and were unable to predict total genetic values accurately. Our results support the long-standing conjecture that additive genetic variation is depleted in traits under selection. We further provide evidence that this depletion is due to the prevalence of low-frequency alleles that underlie the trait. Copyright © 2020 Karavolias et al.","Single-step genomic evaluation became a standard procedure in livestock breeding, and the main reason is the ability to combine all pedigree, phenotypes, and genotypes available into one single evaluation, without the need of post-analysis processing. Therefore, the incorporation of data on genotyped and non-genotyped animals in this method is straightforward. Since 2009, two main implementations of single-step were proposed. One is called single-step genomic best linear unbiased prediction (ssGBLUP) and uses single nucleotide polymorphism (SNP) to construct the genomic relationship matrix; the other is the single-step Bayesian regression (ssBR), which is a marker effect model. Under the same assumptions, both models are equivalent. In this review, we focus solely on ssGBLUP. The implementation of ssGBLUP into the BLUPF90 software suite was done in 2009, and since then, several changes were made to make ssGBLUP flexible to any model, number of traits, number of phenotypes, and number of genotyped animals. Single-step GBLUP from the BLUPF90 software suite has been used for genomic evaluations worldwide. In this review, we will show theoretical developments and numerical examples of ssGBLUP using SNP data from regular chips to sequence data. © 2020 by the authors. Licensee MDPI, Basel, Switzerland.","During 2009–2012, several outbreaks of avian influenza virus H9N2 were reported in Tunisian poultry. The circulating strains carried in their hemagglutinins the human-like marker 226L, which is known to be important for avian-to-human viral transmission. To investigate the origins and zoonotic potential of the Tunisian H9N2 viruses, five new isolates were identified during 2012–2016 and their whole genomes were sequenced. Bayesian-based phylogeny showed that the HA, NA, M and NP segments belong to the G1-like lineage. The PB1, PB2, PA and NS segments appeared to have undergone multiple intersubtype reassortments and to be only distantly related to all of the Eurasian lineages (G1-like, Y280-like and Korean-like). The spatiotemporal dynamic of virus spread revealed that the H9N2 virus was transferred to Tunisia from the UAE through Asian and European pathways. As indicated by Bayesian analysis of host traits, ducks and terrestrial birds played an important role in virus transmission to Tunisia. The subtype phylodynamics showed that the history of the PB1 and PB2 segments was marked by intersubtype reassortments with H4N6, H10N4 and H2N2 subtypes. Most of these transitions between locations, hosts and subtypes were statistically supported (BF > 3) and not influenced by sampling bias. Evidence of genetic evolution was observed in the predicted amino acid sequences of the viral proteins of recent Tunisian H9N2 viruses, which were characterized by the acquisition of new mutations involved in virus adaptation to avian and mammalian hosts and amantadine resistance. This study is the first comprehensive analysis of the evolutionary history of Tunisian H9N2 viruses and highlights the zoonotic risk associated with their circulation in poultry, indicating the need for continuous surveillance of their molecular evolution. © 2020, Springer-Verlag GmbH Austria, part of Springer Nature.","The Brazilian Atlantic Forest harbors high levels of anuran diversity and endemism, including several taxa restricted to small geographic ranges. Here, we provide a multilocus phylogeny for Paratelmatobiinae, a leptodactylid subfamily composed of small-ranged species distributed in the Brazilian Atlantic Forest and in the campo rupestre ecosystem. We performed Bayesian inference and maximum likelihood analyses using three mitochondrial and five nuclear markers, and a matrix comprising a broad taxonomic sampling. We then delimitated independently evolving lineages within the group. We recovered Paratelmatobiinae and each of its four genera as monophyletic and robustly supported. Five putatively new species included in our analyses were unambiguously supported in the phylogenetic trees and delimitation analyses. We also recovered other deeply divergent and geographically structured lineages within the four genera of Paratelmatobiinae. Our estimation of divergence times indicates that diversification in the subfamily began in the Eocene and continued until the Pleistocene. We discuss possible scenarios of diversification for the four genera of Paratelmatobiinae, and outline the implications of our findings for taxonomy and conservation. © 2020 Elsevier Inc.","Eutrophication and climate change scenarios engender the need to develop good predictive models for harmful cyanobacterial blooms (CyanoHABs). Nevertheless, modeling cyanobacterial biomass is a challenging task due to strongly skewed distributions that include many absences as well as extreme values (dense blooms). Most modeling approaches alter the natural distribution of the data by splitting them into zeros (absences) and positive values, assuming that different processes underlie these two components. Our objectives were (1) to develop a probabilistic model relating cyanobacterial biovolume to environmental variables in the Río de la Plata Estuary (35°S, 56°W, n = 205 observations) considering all biovolume values (zeros and positive biomass) as part of the same process; and (2) to use the model to predict cyanobacterial biovolume under different risk level scenarios using water temperature and conductivity as explanatory variables. We developed a compound Poisson-Gamma (CPG) regression model, an approach that has not previously been used for modeling phytoplankton biovolume, within a Bayesian hierarchical framework. Posterior predictive checks showed that the fitted model had a good overall fit to the observed cyanobacterial biovolume and to more specific features of the data, such as the proportion of samples crossing three threshold risk levels (0.2, 1 and 2 mm³ L−1) at different water temperatures and conductivities. The CPG model highlights the strong control of cyanobacterial biovolume by nonlinear and interactive effects of water temperature and conductivity. The highest probability of crossing the three biovolume levels occurred at 22.2 °C and at the lowest observed conductivity (∼0.1 mS cm−1). Cross-validation of the fitted model using out-of-sample observations (n = 72) showed the model's potential to be used in situ, as it enabled prediction of cyanobacterial biomass based on two readily measured variables (temperature and conductivity), making it an interesting tool for early alert systems and management strategies. Furthermore, this novel application demonstrates the potential of the Bayesian CPG approach for predicting cyanobacterial dynamics in response to environmental change. © 2020 Elsevier Ltd","Bulk sampling and subsequent DNA fingerprinting are applied to obtain estimated allelic frequency data and unveil population structure for outbreeding species. We developed ‘Gametes Simulator’, a routine to simulate gametes from allelic frequencies of unlinked loci, which enables the analysis of population structure through Bayesian analysis. Based on the allelic frequencies in the populations, the software simulates the alleles per population in the right proportions and assigns one allele per marker to each gamete following Mendel’s laws that govern gametogenesis. © 2020, Brazilian Society of Plant Breeding. All rights reserved.","The study of animal movement is challenging because movement is a process modulated by many factors acting at different spatial and temporal scales. In order to describe and analyse animal movement, several models have been proposed which differ primarily in the temporal conceptualization, namely continuous and discrete time formulations. Naturally, animal movement occurs in continuous time but we tend to observe it at fixed time intervals. To account for the temporal mismatch between observations and movement decisions, we used a state-space model where movement decisions (steps and turns) are made in continuous time. That is, at any time there is a nonzero probability of making a change in movement direction. The movement process is then observed at regular time intervals. As the likelihood function of this state-space model turned out to be intractable yet simulating data is straightforward, we conduct inference using different variations of Approximate Bayesian Computation (ABC). We explore the applicability of this approach as a function of the discrepancy between the temporal scale of the observations and that of the movement process in a simulation study. Simulation results suggest that the model parameters can be recovered if the observation time scale is moderately close to the average time between changes in movement direction. Good estimates were obtained when the scale of observation was up to five times that of the scale of changes in direction. We demonstrate the application of this model to a trajectory of a sheep that was reconstructed in high resolution using information from magnetometer and GPS devices. The state-space model used here allowed us to connect the scales of the observations and movement decisions in an intuitive and easy to interpret way. Our findings underscore the idea that the time scale at which animal movement decisions are made needs to be considered when designing data collection protocols. In principle, ABC methods allow to make inferences about movement processes defined in continuous time but in terms of easily interpreted steps and turns. © 2020 Ruiz-Suarez et al.","The detection of non-technical losses (NTL) is a very important economic issue for power utilities. Diverse machine learning strategies have been proposed to support electric power companies tackling this problem. Methods performance is often measured using standard cost-insensitive metrics, such as the accuracy, true positive ratio, AUC, or F1. In contrast, we propose to design a NTL detection solution that maximizes the effective economic return. To that end, both the income recovered and the inspection cost are considered. Furthermore, the proposed framework can be used to design the infrastructure of the division in charge of performing customers inspections. Then, assisting not only short term decisions, e.g., which customer should be inspected first, but also the elaboration of long term strategies, e.g., planning of NTL company budget. The problem is formulated in a Bayesian risk framework. Experimental validation is presented using a large dataset of real users from the Uruguayan utility. The results obtained show that the proposed method can boost companies profit and provide a highly efficient and realistic countermeasure to NTL. Moreover, the proposed pipeline is general and can be easily adapted to other practical problems. © 1969-2012 IEEE.","Background Bovine tuberculosis (bTB) diagnosis is impaired by numerous factors including cross-reactivity with Mycobacterium avium subspecies paratuberculosis, which causes Johne's disease (JD). In addition, the effect of repeated bTB-intradermal testing on the performance of JD diagnostic tests is not fully understood. This study aimed to evaluate the impact of repeated bTB-intradermal tests under field conditions in Spain on the JD serological status of cattle. Methods bTB-positive herds (n=264) from Castilla-y-Leon region were selected and matched with officially tuberculosis-free control herds. The association between JD and bTB status at the herd level was assessed using conditional logistic regression and, in herds with both JD-positive and bTB-positive animals, a Bayesian hierarchical mixed-effect model was used for individual-level analysis. Results A significantly higher risk of being JD positive (OR: 1.48; 95 per cent CI: 1.01 to 2.15) was found for bTB-positive herds compared with controls. Individual results indicated that cattle tested more than three times per year, within the last 90 days and more than 12 months were more likely to be JD positive. A skin test-related boost in antibody response could be the cause of an apparent increase of the sensitivity of the JD-absorbed ELISA. Conclusion The results demonstrate the interaction between bTB repeated testing and JD individual and herd-level results and this improved knowledge will facilitate the design of more effective control programmes in herds coinfected with two of the most important endemic diseases affecting cattle in Spain. © 2019 British Veterinary Association. No commercial re-use. See rights and permissions. Published by BMJ.","We herein present the first reliable record of Lepidosira from Neotropical Region. Lepidosira neotropicalis sp. n. from Brazil is described and illustrated in detail, including its complete mitochondrial genome. We perform a Bayesian phylogenetic analysis to place the new species within the Entomobryidae, and at the same time to test previous contrasting hypotheses on Lepidosira position within the Entomobryinae versus Seirinae for the first time. Phylogenetic analyses were based on one mitochondrial and two nuclear genes, Cytochrome Oxidase subunit I, 18S ribosomal RNA and 28S ribosomal RNA, respectively. Lepidosira neotropicalis sp. n. resembles L. sundana Yoshii and Suhardjono and L. nigropunctata (Nguyen) in dorsal chaetotaxy of abdominal segments I and II, but differs from all other species by the combination of head (dorsally and ventrally) and dorsal trunk chaetotaxy, plus empodial complex morphology. Our phylogenetic analyses support the placement of Lepidosira within Entomobryinae, as the sister group of Lepidocyrtoides. Overall, our revision enables a more objective diagnosis to Lepidosira and suggests that the genus is in need of a full revision due to its variable morphology, and lack of data needed to evaluate its monophyly. Finally, we provide an identification key for Neotropical genera of Entomobryinae. © 2019 Royal Swedish Academy of Sciences","Many, though not all, experiments have found that exposing groups of subjects who disagree to the same evidence may cause their initial attitudes to strengthen and move further apart, or polarize. Some have concluded that findings of attitude polarization show that people process information in a biased manner so as to support their initial views. We argue that, on the contrary, polarization is often what we should expect to find in an unbiased Bayesian population, in the context of experiments that find polarization. © (2019) by the Economics Department of the University of Pennsylvania and the Osaka University Institute of Social and Economic Research Association","Anthropogenic food subsidies, such as refuse, are an important driver of animal population changes and gulls heavily forage on this food source. Foraging on refuse during the rearing period could affect the acquisition of resources with potential demographic consequences. Using conventional diet analysis and stable isotopes of δ13C and δ15N of blood of Kelp Gull (Larus dominicanus) nestlings, we studied the variation of the chick growth in response to foraging on refuse on a reproductive colony in the Rio de la Plata Estuary in Uruguay. Using Bayesian mixing models on isotopic data, we estimated the proportion and variation of natural food and refuse in the diet of nestlings. Then, we modelled the variation between the mean posterior densities of the food sources and their standard deviation with the nestling morphometric measurements of different sizes. We found that refuse was gradually delivered to Kelp Gull nestlings during the chick rearing period. Additionally, variation of refuse incorporated into nestling tissues increased with nestlings' size. We propose that parents use more isotopically unique food sources during the nestling growth thereby increasing isotopic diversity. This study highlights the need to improve the current waste management system, which is being reviewed in Uruguay. We believe that decision makers should consider the results of this study, which show that refuse is directly impacting coastal ecosystems through mechanisms poorly explored by the environmental sciences. © 2019 Elsevier B.V.","Felids biogeographic patterns in Uruguay are poorly understood despite the importance of fringe populations of species in conservation. We determined suitability areas, analyzed species richness and assessed the role of Uruguayan protected areas in the conservation of small felids: margay, pampas cat, and Geoffroy’s cat. We applied umbrella-species approach to select conservation areas based on habitat suitability for these species. Habitat suitability models were developed with Maxent, 218 presence records and 21 bioclimatic and topographic variables. Maps show margay in the east, pampas cat in the entire country except the north and Geoffroy’s cat in the entire country. Highest richness was found in the east and the lowest in the north. Protected areas cover 1% of each species distribution range and although most of them provide suitable habitat for at least one species, none of them has adequate surface alone to sustain viable populations. This is the first study that addresses Uruguayan felid distribution based on objective methodologies and a broad set of presence records. Based on information generated, we selected six zones to focus efforts in research and biodiversity conservation. The approach proved useful to guide management plans and conservation decisions and can be extrapolated to other species and countries. © 2019, © 2019 Informa UK Limited, trading as Taylor & Francis Group.","The marine brown algal genus Colpomenia has a worldwide distribution, with five species reported in Korea and Japan. However, no studies to date attempted to identify the number of species and geographical distribution of Colpomenia along Chinese coast. To fill the biodiversity knowledge gap, we analyzed 63 mitochondrial cox3 and 62 mitochondrial atp6 sequences of Colpomenia specimens collected from 30 localities along the Chinese coast. Maximum likelihood and Bayesian inference trees suggest the presence of at least three Colpomenia species (i.e., C. peregrina, C. claytoniae, and C. sinuosa) in China. C. peregrina and C. claytoniae are documented for the first time. C. sinuosa was only found in the South China Sea and its distribution didn’t overlap with that of C. peregrina which was found in the Yellow-Bohai Sea and the East China Sea. C. claytoniae appears to be confined to three isolated islands in the East and the South China Sea, where it occurs in sympatry with, respectively, C. peregrina and C. sinuosa. Future study can focus on comparing eco-physiological differences of Colpomenia species in response to environmental variables and exploring possible genetic hybridization / introgression at inter-specific contact zones. © 2019 The Korean Society of Phycology.","Coalescent-based algorithms coupled with the access to genome-wide data have become powerful tools for assessing questions on recent or rapid diversification, as well as delineating species boundaries in the absence of reciprocal monophyly. In southern South America, the diversification of Liolaemus lizards during the Pleistocene is well documented and has been attributed to the climatic changes that characterized this recent period of time. Past climatic changes had harsh effects at extreme latitudes, including Patagonia, but habitat changes at intermediate latitudes of South America have also been recorded, including expansion of sand fields over northern Patagonia and Pampas). In this work, we apply a coalescent-based approach to study the diversification of the Liolaemus wiegmannii species complex, a morphologically conservative clade that inhabits sandy soils across northwest and south-central Argentina, and the south shores of Uruguay. Using four standard sequence markers (mitochondrial DNA and three nuclear loci) along with ddRADseq data we inferred species limits and a time-calibrated species tree for the L. wiegmannii complex in order to evaluate the influence of Quaternary sand expansion/retraction cycles on diversification. We also evaluated the evolutionary independence of the recently described L. gardeli and inferred its phylogenetic position relative to L. wiegmannii. We find strong evidence for six allopatric candidate species within L. wiegmannii, which diversified during the Pleistocene. The Great Patagonian Glaciation (∼1 million years before present) likely split the species complex into two main groups: one composed of lineages associated with sub-Andean sedimentary formations, and the other mostly related to sand fields in the Pampas and northern Patagonia. We hypothesize that early speciation within L. wiegmannii was influenced by the expansion of sand dunes throughout central Argentina and Pampas. Finally, L. gardeli is supported as a distinct lineage nested within the L. wiegmannii complex. © 2019 Elsevier Inc.","Accuracy of new or alternative diagnostic tests is typically estimated in relation to a well-standardized reference test referred to as a gold standard. However, for bovine tuberculosis (bTB), a chronic disease of cattle, affecting animal and public health, no reliable gold standard is available. In this context, latent-class models implemented using a Bayesian approach can help to assess the accuracy of diagnostic tests incorporating previous knowledge on test performance and disease prevalence. In Uruguay, bTB-prevalence has increased in the past decades partially because of the limited accuracy of the diagnostic strategy in place, based on intradermal testing (caudal fold test, CFT, for screening and comparative cervical test, CCT, for confirmation) and slaughter of reactors. Here, we evaluated the performance of two alternative bTB-diagnostic tools, the interferon-gamma assay, IGRA, and the enzyme-linked immunosorbent assay (ELISA), which had never been used in Uruguay in the absence of a gold standard. In order to do so animals from two heavily infected dairy herds and tested with CFT-CCT were also analyzed with the IGRA using two antigens (study 1) and the ELISA (study 2). The accuracy of the IGRA and ELISA was assessed fitting two latent-class models: a two test-one population model (LCA-a) based on the analysis of CFT/CFT-CCT test results and one in-vitro test (IGRA/ELISA), and a one test-one population model (LCA-b) using the IGRA or ELISA information in which the prevalence was modeled using information from the skin tests. Posterior estimates for model LCA-a suggested that IGRA was as sensitive (75–78%) as the CFT and more sensitive than the serial use of CFT-CCT. Its specificity (90–96%) was superior to the one for the CFT and equivalent to the use of CFT-CCT. Estimates from LCA-b models consistently yielded lower posterior Se estimates for the IGRA but similar results for its Sp. Estimates for the Se (52% 95%PPI:44.41-71.28) and the Sp (92% 95%PPI:78.63–98.76) of the ELISA were however similar regardless of the model used. These results suggest that the incorporation of IGRA for detection of bTB in highly infected herds could be a useful tool to improve the sensitivity of the bTB-control in Uruguay. © Copyright © 2019 Picasso-Risso, Perez, Gil, Nunez, Salaberry, Suanes and Alvarez.","Quantifying precipitation variability beyond the instrumental period is essential for putting current and future fluctuations into long-Term perspective and providing a test bed for evaluating climate simulations. For south-eastern Asia such quantifications are scarce and millennium-long attempts are still missing. In this study we take a pseudo-proxy approach to evaluate the potential for generating summer precipitation reconstructions over south-eastern Asia during the past millennium. The ability of a series of novel Bayesian approaches to generate reconstructions at either annual or decadal resolutions and under diverse scenarios of pseudo-proxy records noise is analysed and compared to the classic analogue method. We find that for all the algorithms and resolutions a high density of pseudo-proxy information is a necessary but not sufficient condition for a successful reconstruction. Among the selected algorithms, the Bayesian techniques perform generally better than the analogue method, the difference in abilities being highest over the semi-Arid areas and in the decadal-resolution framework. The superiority of the Bayesian schemes indicates that directly modelling the space and time precipitation field variability is more appropriate than just relying on a pool of observational-based analogues in which certain precipitation regimes might be absent. Using a pseudo-proxy network with locations and noise levels similar to the ones found in the real world, we conclude that performing a millennium-long precipitation reconstruction over south-eastern Asia is feasible as the Bayesian schemes provide skilful results over most of the target area. © Author(s) 2019. This work is distributed under the Creative Commons Attribution 4.0 License.","Analysis of environmental samples for bacterial antibiotic resistance genes may have different objectives and analysis strategies. In some cases, the purpose was to study diversity and evolution of genes that could be grouped within a mechanism of antibiotic resistance. Different protocols have been designed for detection and confirmation that a functional gene was found. In this study, we present a sequence-based screening of candidate genes encoding beta-lactamases in 14 metagenomes of Antarctic microbial mats. The samples were obtained from different sites, representing diverse biogeographic regions of maritime and continental Antarctica. A protocol was designed based on generation of Hidden Markov Models from the four beta-lactamase classes by Ambler classification, using sequences from the Comprehensive Antibiotic Resistance Database (CARD). The models were used as queries for metagenome analysis and recovered contigs were subsequently annotated using RAST. According to our analysis, 14 metagenomes analyzed contain A, B and C beta-lactamase genes. Class D genes, however, were identified in 11 metagenomes. The most abundant was class C (46.8%), followed by classes B (35.5%), A (14.2%) and D (3.5%). A considerable number of sequences formed clusters which included, in some cases, contigs from different metagenomes. These assemblies are clearly separated from reference clusters, previously identified using CARD beta-lactamase sequences. While bacterial antibiotic resistance is a major challenge of public health worldwide, our results suggest that environmental diversity of beta-lactamase genes is higher than that currently reported, although this should be complemented with gene function analysis. [Figure not available: see fulltext.]. © 2019, Higher Education Press and Springer-Verlag GmbH Germany, part of Springer Nature.","Attempts to eliminate Rhipicephalus microplus from Uruguay have been unsuccessful, and, currently, the country is divided into two areas: a tick-free area and a tick-infested area. In the tick-infested area, different farms face different situations. Some farms are in regions where, due to environmental conditions or a lack of infrastructure, it is difficult to eliminate R. microplus, and the only option is to control it. In contrast, other farms can attempt complete removal. Before deciding whether a farmer should attempt to eliminate R. microplus, the probability of reintroduction must be evaluated. The objective of this study was to develop a probabilistic model based on a Bayesian Belief Network (BBN) to assess the likelihood of a farm becoming infested with R. microplus via the introduction of tick-infested cattle. Only the tick-infested area was considered in the development of this model. Nine variables related to environmental conditions and biosecurity measures, with a focus on cattle movement, were considered. Three different sources of data were used to populate the BBN model: data from the literature; a representative national survey from 2016; and a survey developed to identify biosecurity practices on farms. Model sensitivity and specificity were assessed, and an overall accuracy of 92% was obtained. The model was applied to 33 farms located in the tick-infested area. For one farm, the probability of introduction of R. microplus was 1%; for three farms, the probability was between 21% and 34%; for seven farms, it was between 66% and 76%; and for 22 farms, the probability was greater than 83%. This model was useful for estimating the probability of the introduction of R. microplus into farms, making it possible to assess the impact that the evaluated biosecurity measures have on the probability of introduction and, thus, guiding more objective decision making about the control or elimination of R. microplus from farms. © 2019 Elsevier GmbH","Sloths, like other xenarthrans, are an extremely interesting group of mammals that, after a long history of evolution and diversification in South America, became established on islands in the Caribbean and later reached North America during the Great American Biotic Interchange. In all three regions, they were part of the impressive Pleistocene megafauna. Most taxa became extinct and only two small, distantly related tree-dwelling genera survived. Here, we incorporate several recently described genera of sloths into an assembled morphological data supermatrix and apply Bayesian inference, using phylogenetic and morphological clock methods, to 64 sloth genera. Thus, we investigate the evolution of the group in terms of the timing of divergence of different lineages and their diversity, morphological disparity and biogeographical history. The phylogeny obtained supports the existence of the commonly recognized clades for the group. Our results provide divergence time estimates for the major clades within Folivora that could not be dated with molecular methods. Lineage diversity shows an early increase, reaching a peak in the Early Miocene followed by a major drop at the end of the Santacrucian (Early Miocene). A second peak in the Late Miocene was also followed by a major drop at the end of the Huayquerian (Late Miocene). Both events show differential impact at the family level. After that, a slight Plio-Pleistocene decline was observed before the marked drop with the extinction at the end of the Pleistocene. Phenotypic evolutionary rates were high during the early history of the clade, mainly associated with Mylodontidae, but rapidly decreased to lower values around 25 Ma, whereas Megalonychidae had lower rates at the beginning followed by a steady increase, peaking during the Late Miocene and the Pliocene. Morphological disparity showed a similar trend, with an early increase, followed by a slowly increasing phase through the Late Oligocene and Early Miocene, and ending with another increase beginning at the middle of the Miocene. Biogeographic analysis showed southern South America as the most probable area of origin of the clade and the main region in which the early diversification events took place. Both Megatheriinae and Nothrotheriinae basal nodes were strongly correlated with Andean uplift events, whereas the early history of Mylodontidae is closely associated with southern South America and also shows an early occupation of the northern regions. Within Megalonychidae, our results show Choloepus as a descendant of an island dispersing ancestor and a probable re-ingression to South America by a clade that originated in Central or North America. © The Author(s) 2018.","Even though Brazil is the world leader in amphibian diversity, a significant part of its richness remains unknown or hidden under cryptic taxa. Here, we used model-based species delimitation in an integrative taxonomic approach, by gathering molecular and morphometric data to assess cryptic taxa within the monkey frogs Pithecopus rohdei, from the Atlantic Forest, and P. megacephalus, from campos rupestres ecosystem. We sampled one mitochondrial, five nuclear loci, and 18 morphometric variables. Using species-delimitation methods with genetic and morphometric data, we recovered five divergent lineages within P. rohdei and no cryptic lineages were recovered for P. megacephalus. Morphometric data show differentiation only for one of the candidate species revealed by the delimitation approaches, suggesting that individuals from Doce River basin constitute a putative species for formal taxonomic description. The time-calibrated mtDNA tree shows that P. rohdei complex lineages began to diverge in late Miocene. However, dates from the multilocus species tree are more recent, occurring in Pleistocene, and suggesting their persistence in refuges of forest and sky islands within the Atlantic Forest biome. The existence of cryptic taxa within P. rohdei is, therefore, relevant for planning conservation strategies for this species complex in the Atlantic Forest biodiversity hotspot. © 2018 Elsevier Inc.","Continuous monitoring of network resources enables to make more-informed resource allocation decisions but incurs overheads. We investigate the trade-off between monitoring costs and benefits of accurate state information for a routing problem. In our approach link delays are modeled by Markov chains or hidden Markov models. The current delay information on a link can be obtained by actively monitoring this link at a fixed cost. At each time slot, the decision maker chooses to monitor a subset of links with the objective of minimizing a linear combination of long-run average delay and monitoring costs. This decision problem is modeled as a Markov decision process whose solution is computed numerically. In addition, in simple settings we prove that immediate monitoring cost and delay minimization leads to a threshold policy on a filter which sums up information from past measurements. The lightweight method as well as the optimal policy are tested on several use-cases. We demonstrate on an overlay of 30 nodes of RIPE Atlas that we obtain delay values close to the performance of the always best path with an extremely low monitoring effort when delays between nodes are modeled with hierarchical Dirichlet process hidden Markov models. © 2018, Springer Science+Business Media, LLC, part of Springer Nature.","Two distinct diseases can be caused by Colletotrichum spp. on apple (Malus domestica); apple bitter rot (ABR) and Glomerella leaf spot (GLS). ABR is an old fruit disease occurring in most countries where apple is commercially cultivated. In contrast, GLS was first observed in the USA in 1970, and since then, it has been reported as an emerging leaf disease in different countries, such as Brazil in 1988 and China in 2012. In Brazil, GLS-outbreaks have become more frequent in the last years, making this disease one of the most important constraints for further cultivation of susceptible apples descending from 'Golden Delicious', like 'Gala'. Symptoms of GLS begin with small reddish-purple leaf spots about two days after pathogen infection, evolving into larger irregular necrotic lesions a few days later. Leaves usually turn yellow and fall off resulting in severe tree defoliation. Interestingly, whereas most isolates show organ specialization causing either ABR or GLS, only a few of them can provoke both diseases. Both ABR and GLS occur in Brazil, but only the former in Uruguay. Multilocus sequence analysis allowed the identification of three species causing ABR and GLS in Brazil, Colletotrichum fructicola, C. karstii, and C. nymphaeae, and four species causing ABR in Uruguay, C. fructicola, C. theobromicola, C. melonis, and C. paranaense. As C. fructicola is the most frequent species in both countries, genetic structure of isolates was determined using ISSR-based analysis and vegetative compatibility groups (VCGs). Both populations showed similar levels of Nei's diversity, and Bayesian cluster analysis inferred two genetic clusters correlated with the geographical origin of isolates. Moreover, all Uruguayan isolates were grouped into a unique VCG, while Brazilian isolates were grouped into four VCGs. C. fructicola isolates causing ABR and GLS in southern Brazil revealed to be genetically distinct and arose independently from the Uruguayan population. © 2019 International Society for Horticultural Science. All rights reserved.","The West Indian (Trichechus manatus) and Amazonian (T. inunguis) manatees have a sympatric occurrence at the mouth of the Amazon River. A result of this interspecific encounter is the occurrence of hybrids, which are frequently found along the coasts of Amapá state in Brazil, French Guiana and Guyana. Here we present new genetic evidence indicating the occurrence of a hybrid swarm along the Guianas Shield coastline, which is an interspecific hybrid zone that also separates T. manatus populations located east (Brazil) and west (Caribbean, Gulf of Mexico, Florida and Antilles). In addition, we suggest that this hybrid population occupies a peculiar mangrove-rich environment under strong influence of the Amazon River plume, which requires an independent management and should be considered a special conservation area. © 2019, Academia Brasileira de Ciencias. All rights reserved.","Diploid organisms have two copies of each gene, called alleles, that can be separately transcribed. The RNA abundance associated to any particular allele is known as allele-specific expression (ASE). When two alleles have polymorphisms in transcribed regions, ASE can be studied using RNA-seq read count data. ASE has characteristics different from the regular RNA-seq expression: ASE cannot be assessed for every gene, measures of ASE can be biased towards one of the alleles (reference allele), and ASE provides two measures of expression for a single gene for each biological samples with leads to additional complications for single-gene models. We present statistical methods for modeling ASE and detecting genes with differential allelic expression. We propose a hierarchical, overdispersed, count regression model to deal with ASE counts. The model accommodates gene-specific overdispersion, has an internal measure of the reference allele bias, and uses random effects to model the gene-specific regression parameters. Fully Bayesian inference is obtained using the fbseq package that implements a parallel strategy to make the computational times reasonable. Simulation and real data analysis suggest the proposed model is a practical and powerful tool for the study of differential ASE. © 2019 the Author(s),","Context: As top predators, marine mammals play a key role consuming in different trophic levels and the trophic niche characterization may help to understand how species utilize and share resources. On the coast of the Río de la Plata and the South-west Atlantic, the South American sea lion (Otaria flavescens) and the franciscana dolphin (Pontoporia blainvillei) are two important predators. Aims: The present study investigated potential trophic overlap of both species by measuring stable carbon (δ13C) and nitrogen (δ15N) isotopes over two periods: historical (1959-79) and recent (2002-15) on the Uruguayan coast. Methods: Bone samples of P. blainvillei and O. flavescens were used to determine the isotopic niche using the Stable Isotope Bayesian Ellipses in R (SIBER) analysis. Key results: The isotopic niche did not overlap between species in any period. δ15N was higher in O. flavescens in both periods (20.29‰ ± 0.73 in the historical and 19.95‰ ± 1.0 in the recent period), indicating that it feeds at a higher trophic level than P. blainvillei. The δ13C was also significantly higher in O. flavescens than in P. blainvillei during the two periods (O. flavescens: -11.43 ± 0.6‰ historic, -12.72 ± 0.4‰ recent, and P. blainvillei: -12.69 ± 1.1‰ historic, -13.84 ± 1.3‰ recent). The isotopic niche areas of P. blainvillei in recent and historic periods confirmed they forage in 2 distinct environments, marine and estuarine, with low isotopic overlap. This overlap was higher in the recent period. Conclusions and Implications: O. flavescens and both P. blainvillei groups were segregated in both periods, with a higher overlap in the recent. These species appear to reduce competition by using different resources in the same coastal habitat. O. flavescens preferentially feeds on benthic fish and showed wider trophic amplitude in both periods, whereas P. blainvillei has a more coastal-pelagic diet and included a greater variability of resources in its diet. The differences between species trophic niches can still be detected after both marine mammals species abundance has declined and after the development of fisheries. © 2019 CSIRO.","Aiming to shed light on the influence of Pleistocene geoclimatic events on the current genetic patterns in the flora of the South American Chaqueño Phytogeograpic Domain, we performed phylogeographic and Bayesian ancestral area reconstruction analyses based on plastid DNA sequences of Turnera sidoides. Lineages started to diversify 2.02 Mya, forming two major clades, one comprising extant populations from the western boundary and the other including populations from the eastern boundary of the distribution of T. sidoides. Major divergences within clades occurred at two years 1.44 and 1.72 Mya. Ancestral area reconstruction analyses suggest a western origin of T. sidoides in the Peripampasic Arc. Our results support the hypothesis that Pleistocene geoclimatic events in the Chaqueño Domain played an important role in shaping genetic variation. The establishment of the Arid Diagonal acted as a barrier to gene flow between the western and eastern populations during the Pleistocene Patagonian Glaciations. After this period, the orogeny of the region and Pleistocene climatic oscillations played a role in the divergence of isolated ancestral populations followed by migrations to the Chaco-Pampa plain. © 2018 The Linnean Society of London","Zika virus (ZIKV) is an emerging pathogen member of the Flaviviridae family. ZIKV has spread rapidly in the Latin American region, causing hundreds of thousands of cases of ZIKV disease, as well as microcephaly in congenital infections. Detailed studies on the pattern of evolution of ZIKV strains have been extremely important to our understanding of viral survival, fitness, and evasion of the host’s immune system. For these reasons, we performed a comprehensive phylogenetic analysis of ZIKV strains recently isolated in the Americas. The results of these studies revealed evidence of diversification of ZIKV strains circulating in the Latin American region into at least five different genetic clusters. This diversification was also reflected in the different trends in dinucleotide bias and codon usage variation. Amino acid substitutions were found in E and prM proteins of the ZIKV strains isolated in this region, revealing the presence of novel genetic variants circulating in Latin America. © 2018 by the authors. Licensee MDPI, Basel, Switzerland.","Here, we analyse changes throughout time in the isotopic niche of the Franciscana dolphin (Pontoporia blainvillei), the South American fur seal (Arctocephalus australis) and the South American sea lion (Otaria flavescens) from the Río de la Plata estuary and adjacent Atlantic Ocean to test the hypothesis that fishing may modify the diet of small-gape predators by reducing the average size of prey. The overall evidence, from stable isotope and stomach contents analyses, reveals major changes in resource partitioning between the three predators considered, mainly because of an increased access of Franciscana dolphins to juvenile demersal fishes. These results are consistent with the changes in the length distribution of demersal fish species resulting from fishing and suggest that Franciscana dolphin has been the most benefited species of the three marine mammal species considered because of its intermediate mouth gape. In conclusion, the impact of fishing on marine mammals goes beyond the simple reduction in prey biomass and is highly dependent on the mouth gape of the species involved. © 2018, The Author(s).","The subtype C Eastern Africa clade (CEA), a particularly successful HIV-1 subtype C lineage, has seeded several sub-epidemics in Eastern African countries and Southern Brazil during the 1960s and 1970s. Here, we characterized the past population dynamics of the major CEA sub-epidemics in Eastern Africa and Brazil by using Bayesian phylodynamic approaches based on coalescent and birth-death models. All phylodynamic models support similar epidemic dynamics and exponential growth rates until roughly the mid-1980s for all the CEA sub-epidemics. Divergent growth patterns, however, were supported afterwards. The Bayesian skygrid coalescent model (BSKG) and the birth-death skyline model (BDSKY) supported longer exponential growth phases than the Bayesian skyline coalescent model (BSKL). The BDSKY model uncovers patterns of a recent decline for the CEA sub-epidemics in Burundi/Rwanda and Tanzania (Re < 1) and a recent growth for Southern Brazil (Re > 1); whereas coalescent models infer an epidemic stabilization. To the contrary, the BSKG model captured a decline of Ethiopian CEA sub-epidemic between the mid-1990s and mid-2000s that was not uncovered by the BDSKY model. These results underscore that the joint use of different phylodynamic approaches may yield complementary insights into the past HIV population dynamics. © 2018 The Author(s).","New human polyomaviruses have been described in the last years, including the Merkel-cell polyomavirus (MCPyV; Human polyomavirus 5) and the Human polyomavirus 6 (HPyV6). Although their infection is usually asymptomatic, in immunocompromised host can cause life-threatening pathologies, such as the Merkel cell carcinoma, an aggressive skin neoplasia associated to the MCPyV. Despite being prevalent viruses in population, epidemiological data from South America are scarce, as well as the characterization of the viral types circulating and their origin. The aims of this work were to describe MCPyV and HPyV6 from environmental samples with different geographical origin and to analyze their phylogenetic and evolutionary histories, particularly for MCPyV. Partial and complete genome sequences were obtained from sewage samples from Argentina, Uruguay and Spain. A total number of 87 sequences were obtained for MCPyV and 33 for HPyV6. Phylogenetic analysis showed that MCPyV sequences distributed according to their geographic origin in Europe/North America, Africa, Asia, South America and Oceania groups, suggesting that viral diversification might have followed human migrations across the globe. In particular, viruses from Argentina associated with Europe/North America and South America genotypes, whereas those from Uruguay and Spain also grouped with Africa genotype, reflecting the origin of the current population in each country, which could arrive not only during ancient human migration but also during recent migratory events. In addition, the South American group presented a high level of clusterization, showing internal clusters that could be related to specific locations, such as French Guiana and Brazil or the Southern region into South America, such as Argentina and Uruguay, suggesting a long term evolutionary process in the region. Additionally, in this work, we carried out the first analysis about the evolutionary history of MCPyV trough the integration of phylogenetic, epidemiological and historical data. Since a strong association is observed between the phylogenetic relationships and the origin of the sampled population, this analysis was based on the hypothesis of co-divergence between the virus and human populations. This analysis resulted in a substitution rate of 5.1 × 10−8 s/s/y (∼5.1% of divergence per million years) for the complete genome of MCPyV, which is in the range of those estimated for other double-stranded DNA viruses. Regarding HPyV6, a South American group with clusterization was observed (sequences from Uruguay). Meanwhile, sequences from Argentina grouped with European ones (France and Spain) and remained separated from those isolated in China, USA or Australia. The analysis of viruses from the environment allowed us to deep characterize prevalent infections in different geographic regions, reveling that viruses circulating in each population reflected its origin and that there are specific lineages associated with South America. © 2018 Elsevier Inc.","Recent embodied cognition research shows that access to action verbs in shallow-processing tasks becomes selectively compromised upon atrophy of the cerebellum, a critical motor region. Here we assessed whether cerebellar damage also disturbs explicit semantic processing of action pictures and its integration with ongoing motor responses. We evaluated a cognitively preserved 33-year-old man with severe dysplastic cerebellar gangliocytoma (Lhermitte-Duclos disease), encompassing most of the right cerebellum and the posterior part of the left cerebellum. The patient and eight healthy controls completed two semantic association tasks (involving pictures of objects and actions, respectively) that required motor responses. Accuracy results via Crawford's modified t-tests revealed that the patient was selectively impaired in action association. Moreover, reaction-time analysis through Crawford's Revised Standardized Difference Test showed that, while processing of action concepts involved slower manual responses in controls, no such effect was observed in the patient, suggesting that motor-semantic integration dynamics may be compromised following cerebellar damage. Notably, a Bayesian Test for a Deficit allowing for Covariates revealed that these patterns remained after covarying for executive performance, indicating that they were not secondary to extra-linguistic impairments. Taken together, our results extend incipient findings on the embodied functions of the cerebellum, offering unprecedented evidence of its crucial role in processing non-verbal action meanings and integrating them with concomitant movements. These findings illuminate the relatively unexplored semantic functions of this region while calling for extensions of motor cognition models. © 2018 Cervetto, Abrevaya, Martorell Caro, Kozono, Muñoz, Ferrari, Sedeño, Ibáñez and García.","Genetic diversity and structure of plant germplasm collections, frequently determined by molecular markers, can be used to assist breeding programs, to validate core collections determined by other methodologies, to identify priority accessions for conservation, and to confirm genetic integrity after regeneration. This research aimed to study the genetic diversity and structure of the Uruguayan white dent maize collection, to evaluate the genetic representativeness of its core collection (previously defined by phenotypic traits), and to confirm the genetic integrity of seven regenerations made in Mexico and Uruguay in comparison with the original accessions. Ninety accessions were fingerprinted using 26 simple sequence repeat (SSR) markers. Genetic structure was analyzed by Ward clustering, canonical analysis, and a Bayesian approach based on allelic frequencies. All SSR markers were polymorphic with a mean number of alleles (A) of 7.43, an effective allele number (Ae) of 3.04, and expected heterozygosity of 0.579. The genetic variation between accessions was 0.251, and variation within accessions was 0.749. Four genetic groups were obtained using the three approaches. The core collection represented the structure of the whole collection because the four genetic groups were proportionally represented. The genetic diversity in the core collection did not differ from the entire collection in A, Ae, expected heterozygosity (He) percentage of polymorphic loci (%P), and expected heterozygosity within accessions (Hs). Most regenerations (9 out of 14) preserved the genetic integrity of original accessions, whereas in other cases, either new or lost alleles caused genetic differences. The analysis of genetic structure and diversity of germplasm collections, in combination with morphological characterizations, helps to define ex situ conservation strategies and usage in breeding programs. © Crop Science Society of America | 5585 Guilford Rd., Madison, WI 53711 USA All rights reserved.","A widely used method for prediction of complex traits in animal and plant breeding is \"genomic best linear unbiased prediction\" (GBLUP). In a quantitative genetics setting, BLUP is a linear regression of phenotypes on a pedigree or on a genomic relationship matrix, depending on the type of input information available. Normality of the distributions of random effects and of model residuals is not required for BLUP but a Gaussian assumption is made implicitly. A potential downside is that Gaussian linear regressions are sensitive to outliers, genetic or environmental in origin. We present simple (relative to a fully Bayesian analysis) to implement robust alternatives to BLUP using a linear model with residual t or Laplace distributions instead of a Gaussian one, and evaluate the methods with milk yield records on Italian Brown Swiss cattle, grain yield data in inbred wheat lines, and using three traits measured on accessions of Arabidopsis thaliana. The methods do not use Markov chain Monte Carlo sampling and model hyper-parameters, viewed here as regularization \"knobs,\" are tuned via some cross-validation. Uncertainty of predictions are evaluated by employing bootstrapping or by random reconstruction of training and testing sets. It was found (e.g., test-day milk yield in cows, flowering time and FRIGIDA expression in Arabidopsis) that the best predictions were often those obtained with the robust methods. The results obtained are encouraging and stimulate further investigation and generalization. © 2018 Gianola, Cecchinato, Naya and Schön.","Human bocavirus (HBoV) infections are related to respiratory and gastroenteric diseases. The aim of this study was to investigate the presence of HBoV in both sewage and surface waters in Uruguay. Sixty-eight sewage samples from the cities of Salto, Paysandú, Bella Unión, Fray Bentos, Treinta y Tres and Melo and 36 surface water samples from the cities of Salto, Florida and Santa Lucía were studied. HBoV was screened by multiplex qPCR for the detection of the four subtypes, followed by monoplex qPCRs for the independent quantification of each subtype. A qualitative PCR followed by DNA sequencing and phylogenetic analysis was carried out for molecular characterization of HBoV strains. HBoV was present in a high frequency (69%) in sewage and only one positive sample (3%) was found in surface water. Concerning sewage samples, HBoV1 was detected in 11 (23%) out of the 47 positives samples, with a mean concentration of 8.2 × 104 genomic copies/Liter (gc/L), HBoV3 was detected in 35 (74%) of the positive samples with a mean concentration of 4.1 × 106 gc/L and subtypes 2 and/or 4 were detected in 39 (83%) of the positive samples with a mean concentration of 7.8 × 106 gc/L. After the phylogenetic analysis performed by a Bayesian approach, the four HBoV subtypes were confirmed. This is the first study determining a high frequency of HBoV and the presence of the four HBoV subtypes in aquatic matrices in Latin America, mainly in sewage. Although HBoV was scarcely detected in surface water, a waterborne transmission is likely to occur if people enter in contact with polluted surface waters for recreational activities such as fishing or swimming since an elevated frequency of HBoV was detected in raw sewage which is usually directly discharged into surface waters. © 2018, Springer Science+Business Media, LLC, part of Springer Nature.","The HIV-1 subtype B epidemic in French Guiana and Suriname is characterized by the co-circulation of the globally disseminated “B PANDEMIC ” lineage and of non-pandemic subtype B lineages of Caribbean origin (B CAR ). To reconstruct the spatiotemporal pattern of spread of those viral lineages circulating in these two countries, a total of 361 HIV-1 subtype B pol sequences recovered from treatment-naive adult patients from French Guiana and Suriname between 2006 and 2012 were combined with B PANDEMIC and B CAR reference sequences. Major Guianese/Surinamese B PANDEMIC and B CAR lineages were identified by Maximum Likelihood phylogenetic analysis and the spatiotemporal and demographic parameters estimated using a Bayesian coalescent-based method. We detected four B CAR and three B PANDEMIC transmission chains of large size that together comprise most pandemic and non-pandemic subtype B sequences from French Guiana (≥52%) and Suriname (≥70%) here analyzed. These major lineages were probably introduced into French Guiana and Suriname from the Caribbean (B CAR ) and North/South America (B PANDEMIC ) between the middle 1970s and the late 1980s and spread among populations from both countries with roughly comparable demographic growth rates. We detected a significant trend for higher viral loads and higher proportion of homosexual/bisexual men among subjects infected with B PANDEMIC relative to B CAR strains in French Guiana. These results show that the HIV subtype B epidemic in French Guiana and Suriname has been driven by multiple active B CAR and B PANDEMIC transmission chains that arose since the middle 1970s onward and operate in both countries simultaneously. Although no significant differences in the epidemic potential of major B CAR and B PANDEMIC lineages were observed, relevant associations between the infecting subtype B lineage and epidemiological and clinical characteristics were detected in French Guiana. Copyright © 2018 Bello, Nacher, Divino, Darcissac, Mir and Lacoste. This is an open-access article distributed under the terms of the Creative Commons Attribution License (CC BY).","Long-distance dispersal is one of the main drivers structuring the distribution of marine biodiversity. This study reports the first occurrence of Macrocystis pyrifera and Durvillaea antarctica rafts on the southwestern warm temperate coast of the Atlantic Ocean. Our results indicate that an extreme meteo-oceanographic event, characterized by a northward, displacement of cold sub-Antarctic oceanic waters driven by an extratropical cyclone, could account for these unusual occurrences. A niche model based on known current distribution and maximum entropy principle (MAXENT), revealed the availability of suitable habitats at lower latitudes, outside their actual distribution edges. The distributional boundaries, mainly driven by temperature and irradiance, suggest the existence of environmental suitability in warm temperate areas, as well as in the Northern Hemisphere off Atlantic and Asian coasts. These theoretical edges and respective environmental drivers agree with the physiological affinities of both species, supporting the hypothesis that these variables act as limiting factors for their occurrences in tropical or warmer areas. Emerging regions can function as refuges and stepping-stones, providing substrate with adequate habitat conditions for recruitment of propagules, allowing eventual colonization. Long dispersal events reinforce the need for an extensive discussion on selective management of natural dispersion, biological invasions, refuge mapping and conservation initiatives in a transnational perspective. © 2018 by the authors.","JPEG and Wavelet compression artifacts leading to Gibbs effects and loss of texture are well known and many restoration solutions exist in the literature. So is denoising, which has occupied the image processing community for decades. However, when a noisy image is compressed, the noisy wavelet coefficients can be assigned to the 'wrong' quantization interval, generating artifacts that can have dramatic consequences in products derived from satellite image pairs such as sub-pixel stereo vision and digital terrain elevation models. Despite the fact that the importance of such artifacts in very high resolution satellite imaging has recently been recognized, this restoration problem has been rarely addressed in the literature. In this work we present a thorough probabilistic analysis of the wavelet outliers phenomenon, and conclude that their probabilistic nature is characterized by a single parameter related to the ratio q/σ of the compression rate and the instrumental noise. This analysis provides the conditional probability for a Bayesian MAP estimator, whereas a patch-based local Gaussian prior model is learnt from the corrupted image iteratively, like in state-of-the-art patch-based denoising algorithms, albeit with the additional difficulty of dealing with non-Gaussian noise during the learning process. The resulting joint denoising and decompression algorithm is experimentally evaluated under realistic conditions. The results show its ability to simultaneously denoise, decompress and remove wavelet outliers better than the available alternatives, both from a quantitative and a qualitative point of view. As expected, the advantage of our method is more evident for large values of q/σ. © 2017 IEEE.","The Neotropical region currently harbors 8 recognized species of small cats, including the margay (Leopardus wiedii) and jaguarundi (Puma yagouaroundi). These similar-sized species have broad geographic distributions and commonly occur in sympatry. We constructed species occurrence databases and used the maximum entropy (Maxent) modeling approach to develop models of habitat suitability for each of these species. We identified the environmental predictors with the highest importance for species occurrence and evaluated how they influence habitat suitability for both wild cats. Variables related to temperature and precipitation were good predictors of species occurrence, with both cats responding similarly to them. However, margay showed higher variation in habitat suitability depending on local climatic predictors. The estimated models indicated high niche similarity (> 76%) between these cats, but despite this we identified different areas with the most favorable conditions for each species' occurrence. Approximately 50% and 72% of the entire suitable area were estimated as being moderately to highly suitable for margay and jaguarundi, respectively, with the margay model comprising a higher proportion of forest environments in comparison to that of the jaguarundi. These findings will be relevant in the context of conservation planning when indicating the priority areas for their long-term preservation. © 2017 American Society of Mammalogists.","Background: In Europe the principal definitive host for Echinococcus multilocularis, causing alveolar echinococcosis in humans, is the red fox (Vulpes vulpes). Obtaining reliable estimates of the prevalence of E. multilocularis and relevant risk factors for infection in foxes can be difficult if diagnostic tests with unknown test accuracies are used. Latent-class analysis can be used to obtain estimates of diagnostic test sensitivities and specificities in the absence of a perfect gold standard. Samples from 300 foxes in Switzerland were assessed by four different diagnostic tests including necropsy followed by sedimentation and counting technique (SCT), an egg-PCR, a monoclonal and a polyclonal copro-antigen ELISA. Information on sex, age and presence of other cestode species was assessed as potential covariates in the Bayesian latent class models. Different Bayesian latent-class models were run, considering dichotomized test results and, additionally, continuous readings resulting in empirical ROC curves. Results: The model without covariates estimated a true parasite prevalence of 59.5% (95% CI: 43.1-66.4%). SCT, assuming a specificity of 100%, performed best among the four tests with a sensitivity of 88.5% (95% CI: 82.7-93.4%). The egg-PCR showed a specificity of 93.4% (95% CI: 87.3-99.1%), although its sensitivity of 54.8% was found moderately low (95% CI: 48.5-61.0%). Relatively higher sensitivity (63.2%, 95% CI: 55.3-70.8%) and specificity (70.0%, 95% CI: 60.1-79.4%) were estimated for the monoclonal ELISA compared to the polyclonal ELISA with a sensitivity and specificity of 56.0% (95% CI: 48.0-63.9%) and 65.9% (95% CI: 55.8-75.6%), respectively. In the Bayesian models, adult foxes were found to be less likely infected than juveniles. Foxes with a concomitant cestode infection had double the odds of an E. multilocularis infection. ROC curves following a Bayesian approach enabled the empirical determination of the best cut-off point. While varying the cut-offs of both ELISAs, sensitivity and specificity of the egg-PCR and SCT remained constant in the Bayesian latent class models. Conclusions: Adoption of a Bayesian latent class approach helps to overcome the absence of a perfectly accurate diagnostic test and gives a more reliable indication of the test performance and the impact of covariates on the prevalence adjusted for diagnostic uncertainty. © 2017 The Author(s).","CONCLUSIONS: Our results suggest that the deep genetic divergence in S. thunbergii along the China coast may result from long-term geographic isolation during glacial periods. The dispersal of S. thunbergii driven by coastal currents may facilitate the admixture between southern and northern regimes. Our findings exemplify how both historical and contemporary forces are needed to understand phylogeographical patterns in coastal marine species with long-distance dispersal.\nRESULTS: We used 11 microsatellites and the plastid RuBisCo spacer to evaluate the genetic diversity of 22 Sargassum thunbergii populations sampled along the China coast. Population structure and differentiation was inferred based on genotype clustering and pairwise F ST and allele-frequency analyses. Integrated genetic analyses revealed two genetic clusters in S. thunbergii that dominated in the Yellow-Bohai Sea (YBS) and East China Sea (ECS) respectively. Higher levels of genetic diversity and variation were detected among populations in the YBS than in the ECS. Bayesian coalescent theory was used to estimate contemporary and historical gene flow. High levels of contemporary gene flow were detected from the YBS (north) to the ECS (south), whereas low levels of historical gene flow occurred between the two regions.\nBACKGROUND: Long-term survival in isolated marginal seas of the China coast during the late Pleistocene ice ages is widely believed to be an important historical factor contributing to population genetic structure in coastal marine species. Whether or not contemporary factors (e.g. long-distance dispersal via coastal currents) continue to shape diversity gradients in marine organisms with high dispersal capability remains poorly understood. Our aim was to explore how historical and contemporary factors influenced the genetic diversity and distribution of the brown alga Sargassum thunbergii, which can drift on surface water, leading to long-distance dispersal.","Zika virus (ZIKV) is a member of the family Flaviviridae. ZIKV emerged in Brazil in 2015, causing an unprecedented epidemic and since then the virus has rapidly spread throughout the Americas. These facts highlight the need of detailed phylogenetic studies to understand the emergence, spread, and evolution of ZIKV populations. For these reasons, a Bayesian coalescent Markov Chain Monte Carlo analysis of complete genome sequences of ZIKV strains recently isolated in the American continent was performed. The results of these studies revealed an increasing diversification of ZIKV strains in different genetic lineages and co-circulation of distinct genetic lineages in several countries in the region. The time of the most recent common ancestor (tMRCA) was established to be around February 20, 2014 for ZIKV strains circulating in the American region. A mean rate of evolution of 1.55 × 10−3 substitutions/site/year was obtained for ZIKV strains included in this study. A Bayesian skyline plot indicate a sharp increase in population size from February 2014 to July 2015 and a decline during 2016. These results are discussed in terms of the emergence and evolution of ZIKV populations in the American continent. © 2017 Wiley Periodicals, Inc.","PREMISE OF THE STUDY: The subtribe Menthinae (Lamiaceae), with 35 genera and 750 species, is among the largest and most economically important subtribes within the mint family. Most genera of Menthinae are found exclusively in the New World, where the group has a virtually continuous distribution ranging from temperate North America to southern South America. In this study, we explored the presence, timing, and origin of amphitropical disjuncts within Menthinae. METHODS: Our analyses were based on a data set consisting of 89 taxa and the nuclear ribosomal DNA markers ITS and ETS. Phylogenetic relationships were determined under maximum likelihood and Bayesian criteria, divergence times were estimated with the program BEAST, and ancestral range estimated with BioGeoBEARS. KEY RESULTS: A North Atlantic Land Bridge migration event at about 10.6 Ma is inferred from western Eurasia to North America. New World Menthinae spread rapidly across North America, and then into Central and South America. Several of the large speciose genera are not monophyletic with nuclear rDNA, a finding mirrored with previous chloroplast DNA results. Three amphitropical disjunctions involving North and southern South America clades, one including a southeastern South American clade with several genera, were inferred to have occurred within the past 5 Myr. CONCLUSIONS: Although three New World Menthinae genera occur in both North and South America, none exhibit an amphitropical disjunction. However, three clades exhibit amphitropical disjunctions, all dating to the early Pliocene, and all involve jump dispersals to either southeastern or southwestern South America from southeastern North America. © 2017 Botanical Society of America.","The disjunct distribution of the harvestman Discocyrtus dilatatus (Opiliones, Gonyleptidae) is used as a case study to test the hypothesis of a trans-Chaco Pleistocene paleobridge during range expansion stages. This would have temporarily connected humid regions (‘Mesopotamia’ in northeastern Argentina, and the ‘Yungas’ in the northwest, NWA) in the subtropical and temperate South American lowlands. The present study combines two independent approaches: paleodistributional reconstruction, using the Species Distribution Modeling method MaxEnt and projection onto Quaternary paleoclimates (6 kya, 21 kya, 130 kya), and phylogeographic analyses based on the cytochrome oxidase subunit I molecular marker. Models predict a maximal shrinkage during the warm Last Interglacial (130 kya), and the rise of the hypothesized paleobridge in the Last Glacial Maximum (21 kya), revealing that cold-dry stages (not warm-humid ones, as supposed) enabled the range expansion of this species. The disjunction was formed in the mid-Holocene (6 kya) and is intensified under current conditions. The median-joining network shows that NWA haplotypes are peripherally related to different Mesopotamian lineages; haplotypes from Santa Fe and Córdoba Provinces consistently occupy central positions in the network. According to the dated phylogeny, Mesopotamia-NWA expansion events would have occurred in the last glacial period, in many cases closely associated to the Last Glacial Maximum, with most divergence events occurring shortly thereafter. Only two (out of nine) NWA haplotypes are shared with Mesopotamian localities. A single, presumably relictual NWA haplotype was found to have diverged much earlier, suggesting an ancient expansion event not recoverable by the paleodistributional models. Different measures of sequence statistics, genetic diversity, population structure and history of demographic changes are provided. This research offers the first available evidence for the historical origin of NWA disjunct populations of a Mesopotamian harvestman. © 2017 Vergara et al. This is an open access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.","A recent study showed that infectivity of Zika virus (ZIKV) Asian genotype was enhanced by an alanine-to-valine amino acid substitution at residue 188 of the NS1 protein, but the precise time and location of origin of this mutation were not formally estimated. Here, we applied a Bayesian coalescent-based framework to estimate the age and location of the ancestral viral strain carrying the A188V substitution. Our results support that the ancestral ZIKV strain carrying the A188V substitution arose in Southeastern Asia at the early 2000s and circulated in that region for some time (5-10 years) before being disseminated to Southern Pacific islands and the Americas. © 2017, Fundacao Oswaldo Cruz. All rights reserved.","Protected areas (PAs) are essential for biodiversity conservation, but their coverage is considered inefficient for the preservation of all species. Many species are subdivided into evolutionarily significant units (ESUs) and the effectiveness of PAs in protecting them needs to be investigated. We evaluated the usefulness of the Brazilian PAs network in protecting ESUs of the critically endangered Pithecopus ayeaye through ongo-ing climate change. This species occurs in a threatened mountaintop ecosystem known as campos rupestres. We used multilocus DNA sequences to delimit geographic clus-ters, which were further validated as ESUs with a coalescent approach. Ecological niche modeling was used to estimate spatial changes in ESUs’ potential distributions, and a gap analysis was carried out to evaluate the effectiveness of the Brazilian PAs network to protect P. ayeaye in the face of climate changes. We tested the niche overlap between ESUs to gain insights for potential management alternatives for the spe-cies. Pithecopus ayeaye contains at least three ESUs isolated in distinct mountain regions, and one of them is not protected by any PA. There are no climatic niche differences between the units, and only 4% of the suitable potential area of the species is protected in present and future projections. The current PAs are not effective in pre-serving the intraspecific diversity of P. ayeaye in its present and future range distribu-tions. The genetic structure of P. ayeaye could represent a typical pattern in campos rupestres endemics, which should be considered for evaluating its conservation status. © 2017 The Authors. Ecology and Evolution published by John Wiley & Sons Ltd.","Tuco-tucos (Ctenomys) and related coruros (Spalacopus) are South American subterranean rodents. An energetically demanding lifestyle within the hypoxic/hypercapnic underground atmosphere may change the selective regime on genes involved in O2 transport in blood. In addition, some species of tuco-tucos may be found at high altitude, thus facing additional reductions in changes O2 availabily. We examined sequence variation in the alpha globin subunit gene of hemoglobine in these lineages, within a robust phylogenetic context. Using different approaches (classical and Bayesian maximum likelihood (PAML/Datamonkey) and alternatives methods (TreeSAAP)) we found at least 2 sites with evidence of positive selection in the basal branch of Octodontidae, but not in tuco-tucos. These results suggest some adaptive changes associated to fossoriality, but not strictly to life underground. © 2017 Elsevier B.V.","In this study, we present a morphological description of immature and adult specimens of Ornithodoros saraivai n. sp., a tick associated with the frog Cycloramphus boraceiensis (Cycloramphidae) at São Sebastião island, located in the São Paulo state seaboard, Brazil. While larvae of O. saraivai are ecologically related to Ornithodoros faccinii, another soft tick associated with cycloramphids, the combination of 7 sternal pairs, 16 dorsal pairs, a pyriform dorsal plate and a partially toothed hypostome constitute unique characters of the O. saraivai larvae. One undetermined nymphal instar and adults of O. saraivai are similar to mature specimens of the Ornithodoros talaje species group; however, the O. saraivai specimens can be recognized by the presence of a robust bean-shaped spiracle with a large spiracular plate and more than two long seta in palpal article I. Identical partial sequences of the mitochondrial 16S rDNA gene confirmed the identity for all collected stages and for two cohorts of laboratory-reared larvae of O. saraivai. A Bayesian and Maximum Parsimony inferred phylogenetic trees support the position of O. saraivai in a clade with O. faccinii, suggesting the existence of an Ornithodoros lineage that evolved in association with amphibians. © 2017 Elsevier GmbH","Deltamys is a monotypic sigmodontine rodent from the Pampas of South America. In addition to the formally recognized D. kempi that inhabits lowlands, an undescribed form Deltamys sp. 2n=40 was recently found in the highlands of south-eastern Brazil. In the present study, we perform a phylogeographic reassessment of Deltamys and describe a third form of the genus, endemic to the Brazilian Araucaria Forest. We describe this new species based on an integrative analysis, using complete cytochrome b DNA sequences, karyology and morphology. Bayesian tree recovered two allopatric clades (low-lands vs. highlands) and three lineages: (i) the lowland D. kempi, (ii) the highland Deltamys sp. 2n=40, and (iii) Deltamys araucaria sp. n. Deltamys araucaria sp. n. is karyotypically (2n=34) and morphologically distinguishable from D. kempi (2n=37-38), showing a tawnier dorsum/flank pelage, presence of a protostyle, M1 alveolus positioned anteriorly to the posterior margin of the zygomatic plate, and several other distinguishing characteristics. A phylogeographic assessment of D. kempi recovered two haplogroups with significant differences in skull measurements. This phylogeographic break seems to have been shaped by the Patos Lagoon estuarine channel. The diversification in Deltamys might have been trig-gered by dispersal of older lineages over different altitudinal ranges in the Paraná geological basin. Copyright © 2017 Magnolia Press.","Successful clinical use of patient-specific models for cardiovascular dynamics depends on the reliability of the model output in the presence of input uncertainties. For 1D fluid dynamics models of arterial networks, input uncertainties associated with the model output are related to the specification of vessel and network geometry, parameters within the fluid and wall equations, and parameters used to specify inlet and outlet boundary conditions. This study investigates how uncertainty in the flow profile applied at the inlet boundary of a 1D model affects area and pressure predictions at the center of a single vessel. More specifically, this study develops an iterative scheme based on the ensemble Kalman filter (EnKF) to estimate the temporal inflow profile from a prior distribution of curves. The EnKF-based inflow estimator provides a measure of uncertainty in the size and shape of the estimated inflow, which is propagated through the model to determine the corresponding uncertainty in model predictions of area and pressure. Model predictions are compared to ex vivo area and blood pressure measurements in the ascending aorta, the carotid artery, and the femoral artery of a healthy male Merino sheep. Results discuss dynamics obtained using a linear and a nonlinear viscoelastic wall model. Copyright © 2017 by ASME","The trophic level and ecological role of species in the ecosystem are determined by their feeding habits. While the trophic level and niche width are characteristic at population level, trophic overlap indicates the interaction between populations of a community. Three apex predators coexist in Uruguayan waters: Franciscana dolphin (Pontoporia blainvillei), South American sea lion (SASL, Otaria flavescens) and South American fur seal (SAFS, Arctocephalus australis). The two otariids consume different main prey species and have different isotopic niche, whereas feeding habits of Franciscana dolphin in Uruguayan waters remain practically unknown. This study aimed to determine Franciscana diet composition by stomach content and stable isotope (δ13C and δ15N) analyses, and to assess trophic overlap in diet composition as well as in isotopic niche space among the three apex predators. Diet composition of each species was estimated by Bayesian mixing models with prior information on the diet of predators. Moreover, Bayesian ellipses were used to define the isotopic niche space and overlap among the three species. Diets among predators varied: diet of SAFS was composed mainly by a few pelagic prey species, diet of SASL was more diverse and included pelagic and benthic preys, and for Franciscana dolphin the diet was mainly composed by coastal prey species. The SAFS had the largest isotopic niche area compared to the Franciscana and the SASL. The isotopic niche of the SASL did not overlap with those of the other two species. Although the Franciscana and the SAFS overlap in isotopic space, they do not overlap in feeding areas; this is because particulate organic matter δ13C values increase from inner Rio de la Plata to the estuary and the Atlantic Coast. Here, the relevance of using different sources of information to reach strong conclusions about feeding habits of marine predators and about co-use of resources is shown, mainly in environments where variation in δ13C can be related to diverse reasons. © 2016 Elsevier B.V.","The HIV-1 CRF02_AG clade is the most prevalent HIV variant in West and West-Central Africa and its detection outside Africa is increasingly common. Little is known, however, about the number and phylodynamics of major CRF02_AG lineages circulating worldwide. To this end, a total of 3170 HIV-1 CRF02_AG-like pol sequences isolated around the world, over a period of 25 years (1989 to 2013), were analyzed using Maximum Likelihood and Bayesian coalescent-based methods. Our results suggest that most of the current CRF02_AG diversity comes from the dissemination of a few founder strains out of Central Africa into West Africa and Cameroon between the late 1960s and the middle 1980s. The CRF02_AG strain introduced into West Africa established a large regional epidemic with low phylogeographic structure. This strain was also successfully disseminated out of the West African region and originated at least three large secondary outbreaks in Cameroon at around the late 1970s, in the former Soviet Union (FSU) countries at around the late 1990s, and in Bulgaria/Germany at around the early 2000s. The CRF02_AG African lineages introduced into Cameroon remained mostly restricted to this country and its neighbors. Demographic reconstructions indicate that major CRF02_AG clades circulating in Africa exhibited a decline in growth rate since the middle 1980s/1990s, whereas CRF02_AG clades in Europe and the FSU countries continue to grow exponentially until the middle to late 2000s. Substantial differences in the median estimated growth rate of the same CRF02_AG clade circulating in different regions (0.63–2.00 year− 1), and of different CRF02_AG clades circulating in the same country (0.41–0.75 year− 1) were observed. Thus, the cause of the epidemic outcome of the different HIV-1 CRF02_AG lineages is probably multifactorial. © 2016 Elsevier B.V.","Stable isotopes (δ13C and δ15N) together with chlorophyll a and densities of surf diatoms were used to analyze changes in trophic niches of species in two sandy beaches of Uruguay with contrasting morphodynamics (i.e. dissipative vs. reflective). Consumers and food sources were collected over four seasons, including sediment organic matter (SOM), suspended particulate organic matter (POM) and the surf zone diatom Asterionellopsis guyunusae. Circular statistics and a Bayesian isotope mixing model were used to quantify food web differences between beaches. Consumers changed their trophic niche between beaches in the same direction of the food web space towards higher reliance on surf diatoms in the dissipative beach. Mixing models indicated that A. guyunusae was the primary nutrition source for suspension feeders in the dissipative beach, explaining their change in dietary niche compared to the reflective beach where the proportional contribution of surf diatoms was low. The high C/N ratios in A. guyunusae indicated its high nutritional value and N content, and may help to explain the high assimilation by suspension feeders at the dissipative beach. Furthermore, density of A. guyunusae was higher in the dissipative than in the reflective beach, and cell density was positively correlated with chlorophyll a only in the dissipative beach. Therefore, surf diatoms are important drivers in the dynamics of sandy beach food webs, determining the trophic niche space and productivity. Our study provides valuable insights on shifting foraging behavior by beach fauna in response to changes in resource availability. © 2016 Elsevier Ltd","A steady state model for a production scale milk drying process was built to help process understanding and optimization studies. It involves a spray chamber and also internal/external fluid beds. The model was subjected to a comprehensive statistical analysis for quality assurance using sensitivity analysis of inputs/parameters, and uncertainty analysis to estimate confidence intervals on parameters and model predictions (error propagation). Variance based sensitivity analysis (Sobol's method) was used to quantify the influence of inputs on the final powder moisture as the model output. Bayesian Inference using Markov Chain Monte Carlo sampling was used to quantify the uncertainty on the estimated parameters using available process data. In a full scale process the inputs with major range of variation are: moisture content at concentrate chamber feed (variation around 4%), and humidity at chamber inlet air (variation > 100%). The sensitivity analysis results suggest exploring improvements in the current control (Proportional Integral Derivative) for moisture content at concentrate chamber feed in order to reduce the output variance. It is also confirmed that humidity control at chamber inlet air stream would not be necessary because, despite its wide range of variation (air taken from outside), its impact on output variance is low. The uncertainty analysis results showed that confidence intervals obtained for parameters were reasonable, although some of them were found significantly correlated. For model applications, this means that model simulations should be performed using not only parameter values but also their correlation matrix by means of non-linear error propagation methods such as Monte Carlo techniques. The separate effects on model prediction uncertainties due to parameter estimation and measurement errors were studied. The results indicate that the error in measurements is the main responsible for the uncertainty in output predictions. Therefore using proper filtering of measurements, the comprehensively tested model is ready to support simulation based efforts for further process optimization. © 2016 Elsevier Ltd.","Zika virus (ZIKV) is a member of the family Flaviviridae. In 2015, ZIKV triggered an epidemic in Brazil and spread across Latin America. By May of 2016, the World Health Organization warns over spread of ZIKV beyond this region. Detailed studies on the mode of evolution of ZIKV strains are extremely important for our understanding of the emergence and spread of ZIKV populations. In order to gain insight into these matters, a Bayesian coalescent Markov Chain Monte Carlo analysis of complete genome sequences of recently isolated ZIKV strains was performed. The results of these studies revealed a mean rate of evolution of 1.20 × 10−3 nucleotide substitutions per site per year (s/s/y) for ZIKV strains enrolled in this study. Several variants isolated in China are grouped together with all strains isolated in Latin America. Another genetic group composed exclusively by Chinese strains were also observed, suggesting the co-circulation of different genetic lineages in China. These findings indicate a high level of diversification of ZIKV populations. Strains isolated from microcephaly cases do not share amino acid substitutions, suggesting that other factors besides viral genetic differences may play a role for the proposed pathogenesis caused by ZIKV infection. J. Med. Virol. 88:1672–1676, 2016. © 2016 Wiley Periodicals, Inc. © 2016 Wiley Periodicals, Inc.","Hepatitis E virus (HEV) is an emergent hepatotropic virus endemic mainly in Asia and other developing areas. However, in the last decade it has been increasingly reported in high-income countries. Human infecting HEV strains are currently classified into four genotypes (1-4). Genotype 3 (HEV-3) is the prevalent virus genotype and the mostly associated with autochthonous and sporadic cases of HEV in developed areas. The evolutionary history of HEV worldwide remains largely unknown. In this study we reconstructed the spatiotemporal and population dynamics of HEV-3 at global scale, but with particular emphasis in South America, where case reports have increased dramatically in the last years. To achieve this, we applied a Bayesian coalescent-based approach to a comprehensive data set comprising 97 GenBank HEV-3 sequences for which the location and sampling date was documented. Our phylogenetic analyses suggest that the worldwide genetic diversity of HEV-3 can be grouped into two main Clades (I and II) with a Tmrca dated in approximately 320 years ago (95% HPD: 420-236 years) and that a unique independent introduction of HEV-3 seems to have occurred in Uruguay, where most of the human HEV cases in South America have been described. The phylodynamic inference indicates that the population size of this virus suffered substantial temporal variations after the second half of the 20th century. In this sense and conversely to what is postulated to date, we suggest that the worldwide effective population size of HEV-3 is not decreasing and that frequently sources of error in its estimates stem from assumptions that the analyzed sequences are derived from a single panmictic population. Novel insights on the global population dynamics of HEV are given. Additionally, this work constitutes an attempt to further describe in a Bayesian coalescent framework, the phylodynamics and evolutionary history of HEV-3 in the South American region. © 2016 Elsevier B.V.","Genomic Best Linear Unbiased Predictor (GBLUP) assumes equal variance for all single nucleotide polymorphisms (SNP). When traits are influenced by major SNP, Bayesian methods have the advantage of SNP selection. To overcome the limitation of GBLUP, unequal variance or weights for all SNP are applied in a method called weighted GBLUP (WGBLUP). If only a fraction of animals is genotyped, single-step WGBLUP (WssGBLUP) can be used. Default weights in WGBLUP or WssGBLUP are obtained iteratively based on single SNP effect squared (u2) and/or heterozygosity. When the weights are optimal, prediction accuracy, and ability to detect major SNP are maximized. The objective was to develop optimal weights for WGBLUP-based methods. We evaluated 5 new procedures that accounted for locus-specific or windows-specific variance to maximize accuracy of predicting genomic estimated breeding value (GEBV) and SNP effect. Simulated datasets consisted of phenotypes for 13,000 animals, including 1540 animals genotyped for 45,000 SNP. Scenarios with 5, 100, and 500 simulated quantitative trait loci (QTL) were considered. The 5 new procedures for SNP weighting were: (1) u2 plus a constant equal to the weight of the top SNP; (2) from a heavy-tailed distribution (similar to BayesA); (3) for every 20 SNP in a window along the whole genome, the largest effect (u2) among them; (4) the mean effect of every 20 SNP; and (5) the summation of every 20 SNP. Those methods were compared to the default WssGBLUP, GBLUP, BayesB, and BayesC. WssGBLUP methods were evaluated over 10 iterations. The accuracy of predicting GEBV was the correlation between true and estimated genomic breeding values for 300 genotyped animals from the last generation. The ability to detect the simulated QTL was also investigated. For most of the QTL scenarios, the accuracies obtained with all WssGBLUP procedures were higher compared to those from BayesB and BayesC, partly due to automatic inclusion of parent average in the former. Manhattan plots had higher resolution with 5 and 100 QTL. Using a common weight for a window of 20 SNP that sums or averages the SNP variance enhances accuracy of predicting GEBV and provides accurate estimation of marker effects. © 2016 Zhang, Lourenco, Aguilar, Legarra and Misztal.","Climate change (CC) constitutes one of the main biodiversity threats. The effects of CC on species are apparent worldwide, with primarily poleward and upward shifts in species distribution. Due to their narrow distribution, threatened species are highly vulnerable to CC. In this context, protected areas (PA) could be key tools for adaptation to CC. Our aims were to study the effects of CC on the distribution and richness of the threatened and near threatened amphibians of Uruguay; and to evaluate the effectiveness of the National Network of Protected Areas (NNPA) at present and under future climate scenarios. To model the distribution of nine species, we obtained records from herpetological collections, scientific publications and GBIF, as well as current data and future climate projections of the General Circulation Model HadCM3 under the A2 and B2 IPCC scenarios from Worldclim. To model species distribution we applied maximum entropy techniques (MAXENT). To evaluate the effectiveness of NNPA we conducted a gap analysis by overlaying PA with the distribution of species. The models indicate that the distribution of most amphibian species could expand in Uruguay in the future; except for Pleurodema bibroni y Melanophryniscus montevidensis. Local amphibian richness is predicted to increase, mainly in the northwest and east of the country. While amphibians studied are included in at least one PA, less than 2% of the distribution is actually covered by the NNPA, both currently and under CC scenarios; which it is a strong indicator of the inefficiency of the system. Although projected CC for this region would not constitute a major threat to amphibians studied, the scarce protection by the NNPA represents a risk to the conservation of herpetofauna facing the other components of global change. © 2016, Asociacion Argentina de Ecologia. All Rights Reserved.","The article presents an application of hidden Markov models (HMMs) for pattern recognition on genome sequences. We apply HMM for identifying genes encoding the variant surface glycoprotein (VSG) in the genomes of Trypanosoma brucei (T. brucei) and other African trypanosomes. These are parasitic protozoa causative agents of sleeping sickness and several diseases in domestic and wild animals. These parasites have a peculiar strategy to evade the host’s immune system that consists in periodically changing their predominant cellular surface protein (VSG). The motivation for using patterns recognition methods to identify these genes, instead of traditional homology based ones, is that the levels of sequence identity (amino acid and DNA sequence) amongst these genes is often below of what is considered reliable in these methods. Among pattern recognition approaches, HMM are particularly suitable to tackle this problem because they can handle more naturally the determination of gene edges. We evaluate the performance of the model using different number of states in the Markov model, as well as several performance metrics. The model is applied using public genomic data. Our empirical results show that the VSG genes on T. brucei can be safely identified (high sensitivity and low rate of false positives) using HMM. © 2015, Springer-Verlag London.","Colletotrichum fructicola is the main species causing apple bitter rot (ABR) and Glomerella leaf spot (GLS) in southern Brazil, and ABR in Uruguay where GLS remains unnoticed. Thus, this work aimed to determine the genetic structure of C. fructicola isolates of both the countries. A total of 28 out of 31 Brazilian isolates (90.3%) caused typical symptoms of GLS, while only 6 of 25 Uruguayan isolates (24.0%) originating from fruits were able to infect leaves, but causing atypical symptoms. Both populations showed similar levels of Nei's gene diversity (h = 0.088 and 0.079, for Brazilian and Uruguayan populations, respectively), and Bayesian cluster analysis inferred two genetic clusters correlated with the geographical origin of isolates. A principal coordinates analysis scatter plot and an unweighted pair group method with arithmetic mean-based dendrogram also grouped Brazilian and Uruguayan isolates into two groups. By pairwise comparison of nitrate-nonutilizing (nit) mutants with a proposed set of testers, all Uruguayan isolates were grouped into a unique vegetative compatibility group (namely VCG 1), while Brazilian isolates were grouped into four VCGs (VCG 1 to 4). Brazilian and Uruguayan populations of C. fructicola were found to be genetically distinct. Our results suggest that isolates of C. fructicola from Brazil capable of causing GLS and ABR arose independently of those from Uruguay. Possible causes leading to the evolutionary differences between populations are discussed. © 2016 The American Phytopathological Society.","Migratory fish species are major vectors of connectivity among aquatic habitats. In this study, conventional stomach contents and stable isotope methods (δ13C and δ15N) were combined to understand how fish of different sizes feed across contrasting aquatic habitats. The Cape stumpnose Rhabdosargus holubi (Sparidae, Perciformes) was selected as an abundant estuarine-dependent species in the permanently open Kowie system, South Africa. Three different habitats were sampled in the region, namely, river, estuary, and sea. Fish entered the estuary as post-larvae from the marine environment, resided in the estuary and lower part of the river as juveniles, and then returned to the sea as sub-adults. The diet varied among habitats, seasons, and fish sizes. “Stable Isotope Analysis with R” (SIAR) Bayesian mixing models mostly supported the results from the stomach content analyses, but also revealed the importance of some prey (e.g., insects) that were underestimated in the consumed diet. Rhabdosargus holubi δ13C values indicated a clear spatial gradient in the origin of food sources assimilated across the habitats, with increasing δ13C along the freshwater-marine continuum. The δ13C ranges of sources and fish also overlapped within each habitat along this continuum, thus illustrating the fidelity of R. holubi to specific habitats at different life stages. By consuming prey in a particular habitat before migrating, either permanently or temporarily to another habitat, R. holubi participates in allochthonous fluxes among riverine, estuarine, and coastal marine environments, with approximately 7 tonnes of Cape stumpnose productivity being exported from the 142-ha Kowie Estuary to the sea each year. © 2016, Coastal and Estuarine Research Federation.","Correlation between genetic parameters and factors such as backfat thickness (BFT), rib eye area (REA), and body weight (BW) were estimated for Canchim beef cattle raised in natural pastures of Brazil. Data from 1648 animals were analyzed using multi-trait (BFT, REA, and BW) animal models by the Bayesian approach. This model included the effects of contemporary group, age, and individual heterozygosity as covariates. In addition, direct additive genetic and random residual effects were also analyzed. Heritability estimated for BFT (0.16), REA (0.50), and BW (0.44) indicated their potential for genetic improvements and response to selection processes. Furthermore, genetic correlations between BW and the remaining traits were high (P > 0.50), suggesting that selection for BW could improve REA and BFT. On the other hand, genetic correlation between BFT and REA was low (P = 0.39 ± 0.17), and included considerable variations, suggesting that these traits can be jointly included as selection criteria without influencing each other. We found that REA and BFT responded to the selection processes, as measured by ultrasound. Therefore, selection for yearling weight results in changes in REA and BFT. © FUNPEC-RP.","Mediterranean mountain ranges harbour highly endemic biota in islandlike habitats. Their topographic diversity offered the opportunity for mountain species to persist in refugial areas during episodes of major climatic change. We investigate the role of Quaternary climatic oscillations in shaping the demographic history and distribution ranges in the spider Harpactocrates ravastellus, endemic to the Pyrenees. Gene trees and multispecies coalescent analyses on mitochondrial and nuclear DNA sequences unveiled two distinct lineages with a hybrid zone around the northwestern area of the Catalan Pyrenees. The lineages were further supported by morphological differences. Climatic niche-based species distribution models (SDMs) identified two lowland refugia at the western and eastern extremes of the mountain range, which would suggest secondary contact following postglacial expansion of populations from both refugia. Neutrality test and approximate Bayesian computation (ABC) analyses indicated that several local populations underwent severe bottlenecks followed by population expansions, which in combination with the deep population differentiation provided evidence for population survival during glacial periods in microrefugia across the mountain range, in addition to the main Atlantic and Mediterranean (western and eastern) refugia. This study sheds light on the complexities of Quaternary climatic oscillations in building up genetic diversity and local endemicity in the southern Europe mountain ranges. © 2016 John Wiley & Sons Ltd.","This study presents the first insights into the genetic diversity and structure of the American donkey metapopulation. The primary objectives were to detect the main structural features underlying variability among American donkey populations, identify boundaries between differentiated gene pools, and draw the main colonization pathways since the introduction of donkeys into America in the 15th century. A panel of 14 microsatellite markers was applied for genotyping 350 American donkeys from 13 countries. The genetic structure of this metapopulation was analysed using descriptive statistics and Bayesian model-based methods. These populations were then compared to a database containing information on 476 individuals from 11 European breeds to identify the most likely ancestral donor populations. Results showed the presence of two distinct genetic pools, with confluence of the two in Colombia. The southern pool showed a unique genetic signature subsequent to an older founder event, but lacked any significant influence of modern gene flow from Europe. The northern pool, conversely, may have retained more ancestral polymorphisms and/or have experienced modern gene flow from Spanish breeds. The Andalusian and, to a lesser extent, the Catalan breeds have left a more pronounced footprint in some of the American donkey populations analysed. © 2016 Blackwell Verlag GmbH.","The genetic relationships between the Pacific and the Atlantic populations of marine coastal biota in Southern South America have been analyzed in few studies, most of them relying on a single mitochondrial locus. We analyzed 10 polymorphic microsatellite loci, isolated from a dinucleotide-enriched Eleginops maclovinus genomic library, in a total of 240 individuals (48 from each of 5 sampled sites: 2 Atlantic, 2 Pacific and 1 in Beagle Channel). The results were contrasted against a previous work on the same species with mitochondrial DNA (mtDNA). Observed heterozygosity within localities ranged from 0.85 to 0.88 with the highest overall number of alleles observed at the northernmost locality on the Pacific side (Concepción), but no clear geographic pattern arose from the data. On the other hand, the number of private alleles was negatively correlated with latitude (Spearman's r s test, P=0.017). Among-population variance was low but significant (1.35%; P<0.0001, analysis of molecular variance (AMOVA)) and low genetic differentiation between populations was observed (pairwise F ST values ranged from 0 to 0.021). A Mantel test revealed a significant correlation between geographic distances and F ST (r=0.56, P=0.047). This could be partially accounted by the Atlantic versus Pacific population differentiation detected in three different analyses (STRUCTURE, SAMOVA (Spatial Analysis of MOlecular VAriance) and a population phylogeny). The observed pattern is compatible with a history of separation into two glacial refugia that was better captured by the multilocus microsatellite data than by the mtDNA analysis. © 2016 Macmillan Publishers Limited.","Background: The cattle tick is a parasite that adversely affects livestock performance in tropical areas. Although countries such as Australia and Brazil have developed genetic evaluations for tick resistance, these evaluations have not considered genotype by environment (G∗E) interactions. Genetic gains could be adversely affected, since breedstock comparisons are environmentally dependent on the presence of G∗E interactions, particularly if residual variability is also heterogeneous across environments. The objective of this study was to infer upon the existence of G∗E interactions for tick resistance of cattle based on various models with different assumptions of genetic and residual variability. Methods: Data were collected by the Delta G Connection Improvement program and included 10,673 records of tick counts on 4363 animals. Twelve models, including three traditional animal models (AM) and nine different hierarchical Bayesian reaction norm models (HBRNM), were investigated. One-step models that jointly estimate environmental covariates and reaction norms and two-step models based on previously estimated environmental covariates were used to infer upon G∗E interactions. Model choice was based on the deviance criterion information. Results: The best-fitting model specified heterogeneous residual variances across 10 subclasses that were bounded by every decile of the contemporary group (CG) estimates of tick count effects. One-step models generally had the highest estimated genetic variances. Heritability estimates were normally higher for HBRNM than for AM. One-step models based on heterogeneous residual variances also usually led to higher heritability estimates. Estimates of repeatability varied along the environmental gradient (ranging from 0.18 to 0.45), which implies that the relative importance of additive and permanent environmental effects for tick resistance is influenced by the environment. Estimated genetic correlations decreased as the tick infestation level increased, with negative correlations between extreme environmental levels, i.e., between more favorable (low infestation) and harsh environments (high infestation). Conclusions: HBRNM can be used to describe the presence of G∗E interactions for tick resistance in Hereford and Braford beef cattle. The preferred model for the genetic evaluation of this population for tick counts in Brazilian climates was a one-step model that considered heteroscedastic residual variance. Reaction norm models are a powerful tool to identify and quantify G∗E interactions and represent a promising alternative for genetic evaluation of tick resistance, since they are expected to lead to greater selection efficiency and genetic progress. © 2016 Mota et al.","The aim of this research was to assess the variability and genetic relationships among binary traits denoting the presence or absence of objectionable fibers, namely pigmented (BINPPF), medullated (BINPMED), and kemp fibers (BINPK), and of fleece traits (fiber diameter [FD] and clean fleece weight [CFW]) in Corriedale sheep. Additionally, the total response to selection against objectionable fibers and indirect responses when selecting for fleece traits were evaluated. Fiber records from 679 animals and fleece records from 795 animals obtained from 2 experimental flocks (from 2005 to 2007) were used; the pedigree file included a total of 3,792 animals. Heritability and genetic correlations among the traits were estimated with a multivariate animal model under a Bayesian setting. Heritability estimates (posterior SD) for BINPPF, BINPMED, and BINPK were 0.35 (0.08), 0.37 (0.10), and 0.63 (0.09), respectively; for CFW and FD, estimates were 0.42 (0.09) and 0.43 (0.08), respectively. The genetic correlations between CFW and the 3 types of objectionable fibers were very low (i.e., <0.2). Thus selection for CFW is not expected to affect the number of such fibers in any direction. The same occurred for the genetic correlation between BINPK and FD. Genetic correlations between FD and BINPMED and FD and BINPPF were positive and favorable (0.50 and 0.56, respectively). Selecting for lower FD would decrease the numbers of objectionable fibers. The expected correlated responses in BINPPF, BINPMED, and BINPK when selecting for CFW were −0.03 (0.11), 0.03 (0.11), and −0.05 (0.18), respectively; when selecting for FD, the correlated responses were −0.26 (0.11), −0.26 (0.11), and −0.14 (0.16). Overall, this study reports novel information on genetic parameters for the presence of objectionable fibers and their associations with fleece traits in sheep. Our findings suggest that it could be possible to improve FD while at the same time reducing the content of BINPMED and BINPPF in wool; however, improvements in CFW are likely to have no effect on the numbers of objectionable fibers. © 2016 American Society of Animal Science. All rights reserved.","Bayesian Inference has been widely applied with success in science and engineering. One of its main uses is the inference of model parameters in order to reconcile model outputs with evidence provided by measures. In this article we propose this application for coastal engineering problems. Specifically, it is proposed to infer the parameters of a numerical wave model used to downscale wave reanalysis data to a coastal site. The proposed method is applied to a case study on the Uruguayan Atlantic coast, where a few month wave measure data series is available and needs to be extended in order to be used on an engineering project. The wave model used is SWAN and the data in deep waters and the wind data were obtained from the ERA-Interim reanalysis. At first, the method was tested with one and two parameters, since in these cases it is possible to compare the obtained results with a plot of the target function. Finally it was used to calibrate four parameters of the wave model and assess the uncertainty introduced by the selection of a set of parameters.","Lineage delimitation is of extreme importance in evolutionary biology and constitutes an essential tool in basic and applied fields including conservation. We addressed the delimitation of two closely related species complexes of South American rodents of the genus Ctenomys (tuco-tucos), namely the C. pearsoni and Corrientes groups, whose relationships have been unclear or conflicting in previous phylogenetic studies. We performed a molecular phylogenetic analysis, increasing the number of representatives of each group and including sequences of three mitochondrial loci (the complete cytochrome b coding sequence, and partial regions of the cytochrome oxydase I gene and the D-loop of the control region). The trees obtained using Bayesian Inference and Maximum Parsimony methods show the Corrientes group and the C. pearsoni complex as reciprocally monophyletic sister clades, with higher values of intergroup compared to intragroup genetic distances. These results indicate that the Corrientes group and the C. pearsoni complex are differentiated lineages. The resulting tree topology is in agreement with a scenario of independent chromosomal rearrangements from the ancestral karyomorph (2n = 70 FN = 84), leading to the considerable chromosomal diversity that characterizes both groups. © SAREM, 2016.","Most otariids have colony-specific foraging areas during the breeding season, when they behave as central place foragers. However, they may disperse over broad areas after the breeding season and individuals from different colonies may share foraging grounds at that time. Here, stable isotope ratios in the skull bone of adult Galapagos sea lions (Zalophus wollebaeki) were used to assess the long-Term fidelity of both sexes to foraging grounds across the different regions of the Galapagos archipelago. Results indicated that the stable isotope ratios (δ13C and δ15N) of sea lion bone significantly differed among regions of the archipelago, without any significant difference between sexes and with a non significant interaction between sex and region. Moreover, standard ellipses, estimated by Bayesian inference and used as a measure of the isotopic resource use area at the population level, overlapped widely for the sea lions from the southern and central regions, whereas the overlap of the ellipses for sea lions from the central and western regions was small and nonexisting for those from the western and southern regions. These results suggest that males and females from the same region within the archipelago use similar foraging grounds and have similar diets. Furthermore, they indicate that the exchange of adults between regions is limited, thus revealing a certain degree of foraging philopatry at a regional scale within the archipelago. The constraints imposed on males by an expanded reproductive season (~ 6 months), resulting from the weak reproductive synchrony among females, and those imposed on females by a very long lactation period (at least one year but up to three years), may explain the limited mobility of adult Galapagos sea lions of both sexes across the archipelago. © 2016 Drago et al. This is an open access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.","We examined the phylogeography of the South American swamp rat Scapteromys tumidus using complete mitochondrial DNA cytochrome b sequences. This species is endemic to the Pampas biome and lives near the coastal plain but also in inland continental areas. The coastal domain of such region experienced a highly dynamic geological history. The inland fraction is part of Precambrian and Mesozoic spills and sedimentary basins, while most of the coastal plain was shaped by transgressive-regressive marine events in the Late Quaternary. As fluctuations in the sea level during this period produced large lateral displacements of the shoreline, originating four barrier-lagoon systems that shaped the present coastline, continental area significantly increased from this time. We hypothesized that the S. tumidus have expanded its range from that period, ramped by its adaptive ability to humid areas. Thus, we examined whether historical Pleistocene events had affected genetic variation of this species along its distributional range. Bayesian phylogenetic analysis and the haplotype network inferred two major genetic clusters along the distribution of S. tumidus. Neutrality tests suggest that populations experienced demographic changes and Bayesian skyline plot evidenced a marked recent demographic expansion pattern, intensified from 350 to 300 kya. Such date coincides with the formation of the lagunar barrier II (dated to ca 325 kya) and the concurrent initial formation of the coastal plain and increase in continental area. © 2015 Deutsche Gesellschaft fü Säugetierkunde.","In this paper, the potential distribution of the Mesopotamian harvestman Discocyrtus dilatatus Sørensen, 1884 is modeled, and the species’ bioclimatic profile is described. Models were built with the presence-only methods MAXENT and BIOCLIM, using 85 unique records (of which 49 are new) and 11 non-correlated bioclimatic variables as predictors. Both MAXENT and BIOCLIM supported the Mesopotamian-Yungas disjunct pattern observed in D. dilatatus, and confirmed the hypothesis that the sub-xeric Dry Chaco is an effective barrier for the two portions of the range. Similarly to results of previous studies on other Mesopotamian harvestmen, temperature variables proved more relevant than precipitation variables in the final models. In the combined overall score obtained with MAXENT, bc4-temperature seasonality ranked as the most relevant, and only one precipitation variable (bc18-precipitation of warmest quarter, in second place) ranked among the top five. In the Most Limiting Factor analysis, which identifies the relevant variables in a local scale, temperature variables were again more determining than precipitation variables in most of the range. One single variable, bc5-maximal temperature of warmest month, proved critical near the boundaries of the modeled range and the Dry Chaco, suggesting that extremely high temperatures (and not the supposed aridity) are responsible for the 450 km distribution gap. © 2015, Sociedade Brasileira de Zoologia. All rights reserved.","In Uruguay, onion (Allium cepa L.) germplasm is mainly derived from the genetic material introduced by several waves of European immigrants and subsequently multiplied by household farmers, resulting in a wealth of locally adapted populations. This study examined the genetic diversity in a collection of 27 local onion populations and two cultivars derived from them. A total of 843 onion plants were fingerprinted, and 83 inter-simple sequence repeat polymorphic bands were generated. Analysis of molecular variance showed high diversity within the populations (66% of the total variation). Some short-day populations from different geographical origins were grouped together by the unweighted pair-group method using arithmetic averages, principal coordinate analysis and cluster analysis, while the more extensively sampled long-and intermediate-day populations showed a widespread distribution, with no significant subgrouping among them. This weakly structured gene pool is probably the consequence of seed and bulb exchange between farmers and natural inter-pollination. Nevertheless, a Bayesian analysis allowed the distinction of four genetic backgrounds of alleles in the whole collection, and populations were predominately assigned to each genetic background. In addition, mitochondrial variants determining normal (N) pollen fertility or the sterile S or T types were analysed for the same set of plants using specific primers. Most accessions showed a proportion of male-sterile individuals. Cytoplasm type T was the most represented (26 out of 29 accessions), and cytoplasm type S was found in a low proportion of individuals in seven populations. Uruguayan onion germplasm maintains a low degree of genetic differentiation despite the small cultivated area and intense seed exchange, probably due in part to different market purposes based on the growing cycle. © 2014 NIAB.","Canine distemper virus (CDV) (Paramyxoviridae-. Morbillivirus) is a worldwide spread virus causing a fatal systemic disease in a broad range of carnivore hosts. In this study we performed Bayesian inferences using 208 full-length hemagglutinin gene nucleotide sequences isolated in 16 countries during 37. years (1975-2011). The estimated time to the most recent common ancestor suggested that current CDV strains emerged in the United States in the 1880s. This ancestor diversified through time into two ancestral clades, the current America 1 lineage that recently spread to Asia, and other ancestral clade that diversified and spread worldwide to originate the remaining eight lineages characterized to date. The spreading of CDV was characterized by several migratory events with posterior local differentiation, and expansion of the virus host range. A significant genetic flow between domestic and wildlife hosts is displayed; being domestic hosts the main viral reservoirs worldwide. This study is an extensive and integrative description of spatio/temporal population dynamics of CDV lineages that provides a novel evolutionary paradigm about the origin and dissemination of the current strains of the virus. © 2015.","Growth curves are useful for describing the sigmoid shape of an animal’s weight pattern over time. The purpose of the present study was to describe, for the first time, a complete growth curve for pasture-fed Angus cows in Uruguay and to analyze the major fixed effects that affect the estimation of mature cow weight (CW). A total of 22,743 records from 5,284 cows belonging to 73 herds were used in the statistical analysis. Five nonlinear models (Brody, Gompertz, Von Bertalanffy, logistic, and Richards) and a 3-knot cubic spline function were fitted to model weight change across age. Body weights were recorded at birth, weaning, and 18 mo and as an adult (at 2.3–3.6, 3.6–4.6, 4.6 to 5.6, 5.6 to 6.6, and 6.6 to 8.1 yr). After preliminary analyses, the fixed effects included in the model to estimate CW were BCS, herd, year and month of measurement, and age of dam. The herd and BCS were the major significant sources of variation. The Richards model had the lowest values for Akaike information criterion, Bayesian information criterion, and –2 log likelihood, with the asymptotic weight being 542 kg. The spline function was the model that fitted the data most closely to the observed weights followed by the Richards function. The Richards model gave better predictions of weights from birth to maturity than the other 4 nonlinear models, because these functions were shown to over- or underestimate weights at different ages in this beef cattle data set. The predicted curve showed that cows reach mature CW after 5 yr of age, which is the age commonly assumed as adult weight in beef cattle. Results form this analysis allowed the estimation of CW and rate of maturing and to take knowledge of the shape of growth curve for Angus females in Uruguay under extensive conditions, helping breeders to take selection decisions. In this way, breeders can optimize the management and efficiency for each production system. © 2015 American Society of Animal Science. All rights reserved.","The disease caused by Newcastle disease virus (NDV) is a severe threat to the poultry industry worldwide. Recently, NDV has been isolated in the Antarctic region. Detailed studies on the mode of evolution of NDV strains isolated worldwide are relevant for our understanding of the evolutionary history of NDV. For this reason, we have performed Bayesian coalescent analysis of NDV strains isolated in Antarctica to study evolutionary rates, population dynamics, and patterns of evolution. Analysis of F protein cleavage-site sequences of NDV isolates from Antarctica suggested that these strains are lentogenic. Strains isolated in Antarctica and genotype I reference strain Ulster/67 diverged from ancestors that existed around 1958. The time of the most recent common ancestor (MRCA) was established to be around 1883 for all class II viruses. A mean rate of evolution of 1.78 × 10-3 substitutions per site per year (s/s/y) was obtained for the F gene sequences of NDV strains examined in this study. A Bayesian skyline plot indicated a decline in NDV population size in the last 25 years. The results are discussed in terms of the possible role of Antarctica in emerging or re-emerging viruses and the evolution of NDV populations worldwide. © 2015, Springer-Verlag Wien.","The Hepatitis B Virus (HBV) can cause acute or chronic infection it is also associated with the development of liver cancer, thousands of new infections occur on a yearly basis, and many of these cases are located in certain areas of the Caribbean and Latin America. In these areas, the HBV prevalence is still high which makes this virus a serious public health concern to the entire region. Studies performed in Panama suggest a complex pattern in the distribution of HBV among the country's different risk groups. We use phylogenetic analysis in order to determine which HBV genotypes were circulating in these specific groups; for this we used a fragment of the PreS2/2 region of the HBV genome. Subsequently whole HBV genome sequences were used for Bayesian analysis of phylodynamics and phylogeography. Two main genotypes were found: genotype A (54.5%) and genotype F (45.5%). There was a difference in the distribution of genotypes according to risk groups: 72.9% of high risk groups were associated to genotype A, and 55.0% of samples of genotype F were associated to the low risk group (p<0.002). The Bayesian analysis of phylogeny-traits association revealed a statistically significant geographical association (p<0.0001) with both genotypes and different regions of the country. The Bayesian time of most recent common ancestor analysis (tMRCA) revealed a recent tMRCA for genotype A2 circulating in Panama (1997, 95% HPD: 1986-2005), when it is compared with Panamanian genotype F1c sequences (1930, 95% HPD: 1810-2005). These results suggest a possible change in the distribution of HBV genotypes in Panama and Latin America as a whole. They also serve to encourage the implementation of vaccination programs in high-risk groups, in order to prevent an increase in the number of new HBV cases in Latin America and worldwide. © 2015 Martínez et al. This is an open access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.","Citrus Tristeza Virus (CTV) is the most economically important virus of citrus worldwide. Genetic diversity and population structure of CTV isolates from all citrus growing areas from Uruguay were analyzed by RT-PCR and cloning of the three RNA silencing suppressor genes (p25, p20 and p23). Bayesian phylogenetic analysis revealed the circulation of three known genotypes (VT, T3, T36) in the country, and the presence of a new genetic lineage composed by isolates from around the world, mainly from South America. Nucleotide and amino acid identity values for this new genetic lineage were both higher than 97% for the three analyzed regions. Due to incongruent phylogenetic relationships, recombination analysis was performed using Genetic Algorithms for Recombination Detection (GARD) and SimPlot software. Recombination events between previously described CTV isolates were detected. High intra-sample variation was found, confirming the co-existence of different genotypes into the same plant. This is the first report describing: (1) the genetic diversity of Uruguayan CTV isolates circulating in the country and (2) the circulation of a novel CTV genetic lineage, highly present in the South American region. This information may provide assistance to develop an effective cross-protection program. © 2015 by the authors; licensee MDPI, Basel, Switzerland.","The South American fur seal (SAFS), Arctocephalus australis, was severely depleted during the 18th century. This work updates information on distribution, abundance, seasonal movements, and genetic structure along the southwestern South Atlantic. Its distribution in Argentine waters includes nineteen rookeries on the Patagonian coast, Isla de los Estados, and islets of the Beagle Channel. Censuses were carried out in rookeries from Isla Escondida to Isla de los Estados from 1996 to 2014. The largest concentrations are found in Chubut, (4,500-15,500 seals) and Isla de los Estados (4,500). Pups were recorded on Isla Escondida, Rasa, and Isla de los Estados. An increasing number of SAFS were recorded in northern Patagonia, including seals marked in Uruguay. Skin samples were taken from southern Brazil to Tierra del Fuego in order to evaluate population structure and demographic trends. Genetic results indicated no population subdivisions (Φst = -0.0292, Fst = -0.00716, P > 0.05). Bayesian skyline plots constructed for the entire data set indicated evidence of rapid population expansion discernible in Nef between the last 20,000 and 17,000 yr. Genetics results and observation data from marked fur seals support the hypothesis of ancient gene flow and a single Atlantic population. Therefore, SAFS should be managed under international and integrated conservation policies. © 2015 Society for Marine Mammalogy.","One of the main animal health problems in tropical and subtropical cattle production is the bovine tick, which causes decreased performance, hide devaluation, increased production costs with acaricide treatments, and transmission of infectious diseases. This study investigated the utility of genomic prediction as a tool to select Braford (BO) and Hereford (HH) cattle resistant to ticks. The accuracy and bias of different methods for direct and blended genomic prediction was assessed using 10,673 tick counts obtained from 3,435 BO and 928 HH cattle belonging to the Delta G Connection breeding program. A subset of 2,803 BO and 652 HH samples were genotyped and 41,045 markers remained after quality control. Log transformed records were adjusted by a pedigree repeatability model to estimate variance components, genetic parameters, and breeding values (EBV) and subsequently used to obtain deregressed EBV. Estimated heritability and repeatability for tick counts were 0.19 ± 0.03 and 0.29 ± 0.01, respectively. Data were split into 5 subsets using k-means and random clustering for cross-validation of genomic predictions. Depending on the method, direct genomic value (DGV) prediction accuracies ranged from 0.35 with Bayes least absolute shrinkage and selection operator (LASSO) to 0.39 with BayesB for k-means clustering and between 0.42 with BayesLASSO and 0.45 with BayesC for random clustering. All genomic methods were superior to pedigree BLUP (PBLUP) accuracies of 0.26 for k-means and 0.29 for random groups, with highest accuracy gains obtained with BayesB (39%) for k-means and BayesC (55%) for random groups. Blending of historical phenotypic and pedigree information by different methods further increased DGV accuracies by values between 0.03 and 0.05 for direct prediction methods. However, highest accuracy was observed with single-step genomic BLUP with values of 0.48 for k-means and 0.56, which represent, respectively, 84 and 93% improvement over PBLUP. Observed random clustering cross-validation breedspecific accuracies ranged between 0.29 and 0.36 for HH and between 0.55 and 0.61 for BO, depending on the blending method. These moderately high values for BO demonstrate that genomic predictions could be used as a practical tool to improve genetic resistance to ticks and in the development of resistant lines of this breed. For HH, accuracies are still in the low to moderate side and this breed training population needs to be increased before genomic selection could be reliably applied to improve tick resistance. © 2015 American Society of Animal Science. All rights reserved.","Stable carbon and nitrogen isotopes in skin and bone of South American sea lions from Brazil and Uruguay were analysed to test the hypothesis that trophic overlap between the sexes is lower during the pre-breeding season than throughout the rest of the year. The isotopic values of skin and bone were used to infer the trophic relationships between the sexes during the pre-breeding period and year round, respectively. Prey species were also analysed to establish a baseline necessary for interpreting the stable isotope ratios of skin and bone. Standard ellipse areas, estimated using Bayesian inference in the SIBER routine of the SIAR package in R, suggested that males and females used a wide diversity of foraging strategies throughout the year and that no differences existed between the sexes. However, the diversity of foraging strategies was largely reduced during the pre-breeding period, with all the individuals of each sex adopting similar strategies, but with the two sexes differing considerably in stable isotope values and the ellipse areas of males and females not overlapping at all. Nevertheless, the results revealed a general increase in the consumption of pelagic prey by both sexes during the pre-breeding period. The progressive crowding of individuals in the areas surrounding the breeding rookeries during the pre-breeding period could lead to an increase in the local population density, which could explain the above reported changes. © Inter-Research 2015 · www.int-res.com.","We conduct two experimental tests of the claim that people are overconfident, using new tests of overplacement that are based on a formal Bayesian model. Our two experiments, on easy quizzes, find that people overplace themselves. More precisely, we find apparently overconfident data that cannot be accounted for by a rational population of expected utility maximizers who care only about money. The finding represents new evidence of overconfidence that is robust to the Bayesian critique offered by Benoît and Dubra (Jean-Pierre Benoît and Juan Dubra (2011). \"Apparent Overconfidence.\" Econometrica, 79, 1591-1625). We discuss possible limitations of our results. © 2014 by the European Economic Association.","Temperament can be defined as the fearfulness and reactivity of an animal in response to humans and strange, novel or threatening environments. The productive performance of an animal is affected by its temperament, and selection of calm animals might improve their adaptation to the farming environment and handling, as well as improve productivity. The temperament was measured in lambs of two breeds of sheep in Uruguay. The effects of dam's age, type of birth, age of the lamb and contemporary group (CG; lambs belonging to the same year, flock, sex and rearing group) on the temperament of the lambs and the heritability of temperament were estimated with a Bayesian analysis using Gibbs sampling. Overall, 4962 Corriedale lambs and 2952 Merino lambs from 13 farms were tested. Temperament was measured using the isolation box test, isolating a lamb inside the box for 30 s, and recording the vibrations produced by its movements. The average temperament score (±s.e.m.) of the Corriedale lambs was 24.7 (±0.23) and that of the Merino was 36.8 (±0.45). Temperament was not associated with dam's age, type of birth or lamb's age. There were no relevant differences in the agitation score between lambs born in 2010 and 2011. The mean of the distribution of possible values of heritability (±s.d.) was 0.18 (±0.05) for the Corriedale and 0.31 (±0.06) for the Merino. The likelihood of heritability values to be greater than 0.15 exceeded 70% in the Corriedale and 90% in the Merino. The temperament of Merino and Corriedale sheep in Uruguay is moderately heritable. It is not related to dam's age, type of birth or age of the lambs; however, it is affected by some aspect of the CG. © The Animal Consortium 2014.","Age and growth estimates of shortfin mako sharks Isurus oxyrinchus from the western South Atlantic Ocean were obtained through the analysis of vertebral sections of 245 specimens (126 females, 116 males and 3 with undetermined sex), ranging in size from 78 to 330 cm fork length (FL), using a flexible growth model (Schnute model) and a Bayesian approach. A significant linear relationship was found between FL and vertebral radius for sexes combined. Marginal increment analyses were inconclusive about periodicity of growth band deposition and an annual periodicity was assumed to make age estimations. Specimens were estimated to be between 0 and 28 years of age. The Schnute model provided a good description of the individual growth for both sexes up to 15 years of age. Shortfin mako growth during the first year of life was 33.9 cm (ICr95% = 19.9–40.8) for females and 30.5 cm (ICr95% = 25.6–35.4) for males. Until approximately 15 years of age, both sexes showed similar growth and reached ~217 cm FL. Sigmoid shaped growth curves obtained for both sexes indicated a change in the growth pattern close to 7 years of age. Inconclusive results about periodicity of growth band deposition in the study area make necessary the application of more robust validation techniques in the future. Meanwhile, a precautionary approach that assumes an annual deposition pattern in the western South Atlantic can be used for the assessment and management of stocks of this species, characterized by low fecundity and late maturity. © 2014, Springer Science+Business Media Dordrecht.","Objective: To obtain a comprehensive description of the evolutionary and demographic history of major HIV-1 subtype B pandemic (BPANDEMIC) clades circulating in Latin America. Design: A total of 6789 HIV-1 subtype B pol sequences collected from seven different Latin American countries between 1990 and 2011 were combined with BPANDEMIC reference sequences (n=500) from the United States and France. Methods: Major BPANDEMIC clades were identified by maximum likelihood phylogenetic analysis with sequential pruning of ambiguously positioned taxa. Time scale and demographic reconstructions were performed using a Bayesian coalescent-based method. Results: We identified 12 major BPANDEMIC monophyletic lineages mainly composed by Latin American sequences and that together comprise 36% of all subtype B sequences from the region here included. Four clades belong to two major regional lineages that comprise sequences from at least two neighboring countries, whereas the other eight clades were country-specific. The median age of major Latin American BPANDEMIC clades encompass a period of two decades (1968-1988), although most of them probably arose before the early 1980s. All major clades seem to have experienced an initial period of exponential growth, with median epidemic growth rates that range from 0.50 year-1to 0.94 year-1, followed by a recent decline in growth rate. Conclusion: About one-Third of HIV-1 subtype B infections in Latin America originated from the spread of a few BPANDEMIC founder strains probably introduced in the region since the late 1960s. Despite their initial successful dissemination, all major BPANDEMIC clades showed signs of subsequent epidemic stabilization. Copyright © 2015 Wolters Kluwer Health, Inc. All rights reserved.","1. The woodlouse genus Halophiloscia Verhoeff, 1908 includes littoral halophilous species distributed along the coasts of the Mediterranean Sea, the Atlantic coasts of Europe and northern Africa. The species Halophiloscia couchii has been introduced to the Americas, Hawaii and Australia. The Canary Islands harbour four species, two of which are included in catalogues of endangered species. 2. One mitochondrial and one nuclear gene sequenced from over 100 specimens, including outgroups, were analysed to infer the colonisation and diversification processes of Halophiloscia in the Canary Islands and shed light on their conservation status. 3. The Canary Islands were independently colonised by three distinct Halophiloscia lineages: (i) the endemic species H. rodriguezi, collected in several western Canary Islands, and the troglobiomorphic H. microphthalma from La Palma, (ii) the endemic H. canariensis, found throughout the eastern Canaries, and (iii) the cosmopolitan H. couchii, present on Gran Canaria and Tenerife. 4. Halophiloscia microphthalma is not sister to the extant epigean Halophiloscia species present on the same island, which together with the low levels of genetic variability detected in the species may warrant a higher conservation status than presently given. 5. The mitochondrial marker revealed a fine-scale phylogeographic structure in H. canariensis, and rejected recent sharp declines in its population size, which in addition to its widespread distribution challenge its consideration as an endangered species. 6. The presence of H. couchii in the archipelago is interpreted as a recent introduction. Although no evidence of introgression was detected, more thorough sampling and monitoring of introduced populations are recommended to discard either hybridisation or competitive displacement of native species. © 2014 The Royal Entomological Society.","Subfamily Gochnatioideae is the sister group of ~96% of the species in Compositae (Asteraceae). It is of particular interest not only because of its position in the phylogeny, but also because, in recent molecular studies, the node it occupies is not strongly supported making difficult any inferences on the direction of character evolution in the family. The recognition of tribe Gochnatieae was one of the results of a comprehensive molecular analysis of the family that showed the traditional circumscription of the Mutisieae to be non-monophyletic. The four genera of Gochnatieae (Cnicothamnus, Cyclolepis, Gochnatia, Richterago) were defined by the presence of apiculate anther appendages and dorsally smooth style branches. Gochnatia, which contained about 70 species, was the largest and most complex genus and in the last decade some of its sections have been moved (or returned) to the rank of genus. This study includes a large selection of potential outgroups and over 60% of all species in the tribe, including all the genera and all but one of the sections of Gochnatia, to examine evolutionary relationships among the taxa. Both cpDNA and nrDNA were used in a phylogenetic analysis using parsimony, likelihood, and Bayesian approaches. The results suggest a non-monophyletic Gochnatia that is here resolved by the recognition of segregate genera. Morphological characters support these new genera and allow the adoption of a new classification for the Gochnatieae. A biogeographic analysis shows a possible southern South American/Andean origin followed by movement in three directions: into the Central Andes, into central and northern Brazil, and into Mexico and the Caribbean. The dating analysis gives an age of the split of the core Gochnatieae from the Wunderlichieae-Cyclolepis clade, and hence the age of the tribe, of 36-45 Ma and an age of 23-25 Ma for the first split within the core Gochnatieae (Andean vs. Brazil-Mexico-Caribbean). Cnicothamnus remains in Gochnatieae but Cyclolepis is designated incertae sedis. © International Association for Plant Taxonomy (IAPT) 2014.","The human immunodeficiency virus type 1 (HIV-1) subtype G is the second most prevalent HIV-1 clade in West Africa, accounting for nearly 30% of infections in the region. There is no information about the spatiotemporal dynamics of dissemination of this HIV-1 clade in Africa. To this end, we analyzed a total of 305 HIV-1 subtype G pol sequences isolated from 11 different countries from West and Central Africa over a period of 20 years (1992 to 2011). Evolutionary, phylogeographic and demographic parameters were jointly estimated from sequence data using a Bayesian coalescentbased method. Our analyses indicate that subtype G most probably emerged in Central Africa in 1968 (1956-1976). From Central Africa, the virus was disseminated to West and West Central Africa at multiple times from the middle 1970s onwards. Two subtype G strains probably introduced into Nigeria and Togo between the middle and the late 1970s were disseminated locally and to neighboring countries, leading to the origin of two major western African clades (GWA-I and G WAII). Subtype G clades circulating in western and central African regions displayed an initial phase of exponential growth followed by a decline in growth rate since the early/middle 1990s; but the mean epidemic growth rate of GWA-I (0.75 year-1) and GWA-II (0.95 year-1) clades was about two times higher than that estimated for central African lineages (0.47 year-1). Notably, the overall evolutionary and demographic history of GWA-I and GWA-II clades was very similar to that estimated for the CRF06-cpx clade circulating in the same region. These results support the notion that the spatiotemporal dissemination dynamics of major HIV-1 clades circulating in western Africa have probably been shaped by the same ecological factors. © 2011 Delatorre et al.","We used species distribution modeling to investigate the potential effects of climate change on 24 species of Neotropical anurans of the genus Melanophryniscus. These toads are small, have limited mobility, and a high percentage are endangered or present restricted geographical distributions. We looked at the changes in the size of suitable climatic regions and in the numbers of known occurrence sites within the distribution limits of all species. We used the MaxEnt algorithm to project current and future suitable climatic areas (a consensus of IPCC scenarios A2a and B2a for 2020 and 2080) for each species. 40% of the species may lose over 50% of their potential distribution area by 2080, whereas 28% of species may lose less than 10%. Four species had over 40% of the currently known occurrence sites outside the predicted 2080 areas. The effect of climate change (decrease in climatic suitable areas) did not differ according to the present distribution area, major habitat type or phylogenetic group of the studied species. We used the estimated decrease in specific suitable climatic range to set a conservation priority rank for Melanophryniscus species. Four species were set to high conservation priority: M. montevidensis, (100% of its original suitable range and all known occurrence points potentially lost by 2080), M. sp.2, M. cambaraensis, and M. tumifrons. Three species (M. spectabilis, M. stelzneri, and M. sp.3) were set between high to intermediate priority (more than 60% decrease in area predicted by 2080); nine species were ranked as intermediate priority, while eight species were ranked as low conservation priority. We suggest that monitoring and conservation actions should be focused primarily on those species and populations that are likely to lose the largest area of suitable climate and the largest number of known populations in the short-term. © 2014 Zank et al.","Noroviruses (NoVs) are the major cause of acute gastroenteritis outbreaks, and, despite a wide genetic diversity, genotype II.4 is the most prevalent strain worldwide. Mutations and homologous recombination have been proposed as mechanisms driving the epochal evolution of the GII.4, with the emergence of new variants in 1-3-year intervals causing global epidemics. There are no data reporting the dynamics of GII.4 variants along a specific period in Brazil. Therefore, to improve the understanding of the comportment of these variants in the country, the aim of this study was to evaluate the circulation of NoV GII.4 variants during a 9-year period in 3 out of 5 Brazilian regions. A total of 147 samples were sequenced, and a phylogenetic analysis of subdomain P2 demonstrated the circulation of six GII.4 variants, Asia-2003, Hunter-2004, Den Haag-2006b, Yerseke-2006a, New Orleans-2009, and Sydney-2012, during this period. The most prevalent variant was Den Haag-2006b, circulating in different Brazilian regions from 2006 to 2011. A Bayesian coalescent analysis was used to calculate the mean evolutionary rate of subdomain P2 as 7.3×10-3 (5.85×10-3-8.82×10-3) subst./site/year. These analyses also demonstrated that clade Den Haag-2006b experienced a rapid expansion in 2005 and another in 2008 after a period of decay. The evaluation of the temporal dynamics of NoV GII.4 in Brazil revealed a similar pattern, with few exceptions, to the worldwide observation. These data highlight the importance of surveillance for monitoring the emergence of new strains of NoV GII.4 and its impact on cases of acute gastroenteritis. © 2014 Fioretti et al.","Methods for genomic prediction were evaluated for an Israeli Holstein dairy population of 713,686 cows and 1,305 progeny-tested bulls with genotypes. Inclusion of genotypes of 343 elite cows in an evaluation method that considers pedigree, phenotypes, and genotypes simultaneously was also evaluated. Two data sets were available: a complete data set with production records from 1985 through 2011, and a reduced data set with records after 2006 deleted. For each production trait, a multitrait animal model was used to compute traditional genetic evaluations for parities 1 through 3 as separate traits. Evaluations were calculated for the reduced and complete data sets. The evaluations from the reduced data set were used to calculate parent average for validation bulls, which was the benchmark for comparing gain in predictive ability from genomics. Genomic predictions for bulls in 2006 were calculated using a Bayesian regression method (BayesC), genomic BLUP (GBLUP), single-step GBLUP (ssGBLUP), and weighted ssGBLUP (WssGBLUP). Predictions using BayesC and GBLUP were calculated either with or without an index that included parent average. Genomic predictions that included elite cow genotypes were calculated using ssGBLUP and WssGBLUP. Predictive ability was assessed by coefficients of determination (R2) and regressions of predictions of 135 validation bulls with no daughters in 2006 on deregressed evaluations of those bulls in 2011. A reduction in R2 and regression coefficients was observed from parities 1 through 3. Fat and protein yields had the lowest R2 for all the methods. On average, R2 was lowest for parent averages, followed by GBLUP, BayesC, ssGBLUP, and WssGBLUP. For some traits, R2 for direct genomic values from BayesC and GBLUP were lower than those for parent averages. Genomic estimated breeding values using ssGBLUP were the least biased, and this method appears to be a suitable tool for genomic evaluation of a small genotyped population, as it automatically accounts for parental index, allows for inclusion of female genomic information without preadjustments in evaluations, and uses the same model as in traditional evaluations. Weighted ssGBLUP has the potential for higher evaluation accuracy. © 2014 American Dairy Science Association.","Background: Thermal constraints are often invoked to explain animal distributions. Maximum temperatures are less variable in different biomes around the globe than are minimum temperatures. Considerable information is available for mammals about basal metabolic rate and thermal conductance.\nAims: Evaluate the correlation of lower critical temperature (TLC), upper critical temperature (TUC) or TNZ breadth (TUC - TLC TNZb) with three ambient temperatures in rodent species.\nHypotheses: TLC, TUC and TNZb should be adjusted by selective processes to the ambient temperature that is most usually experienced by mammal species. TNZb should be greater in species inhabiting colder habitats.\nMethods: We used TLC, TUC data from published studies of 85 species of rodents. We determined the average annual mean, minimum and maximum temperatures across the distribution of each species. Then, using standard least squares regression with body mass as a covariate, we determined the statistical relationships between the physiological variables and the temperatures. We evaluated the effect of phylogeny using a Bayesian Phylogenetic Mixed Model in addition to Bayesian Model Averaging.\nResults: Ambient temperatures correlate positively with TLC and TUC, and negatively with TNZb. Species that evolved in cold climates exhibited a greater mass-independent TNZb than species from warmer climates. Species that evolved in cold climates exhibited lower TLC and TUC than species from warmer climates. Phylogenetic as well as conventional statistics indicated that there are thermoregulatory constraints across geographic gradients. © 2014 Francisco Bozinovic.","Understanding the spatiotemporal distribution of genetic variation and the ways in which this distribution is connected to the ecological context of natural populations is fundamental for understanding the nature and mode of intraspecific and, ultimately, interspecific differentiation. The Petunia axillaris complex is endemic to the grasslands of southern South America and includes three subspecies: P. a. axillaris, P. a. parodii and P. a. subandina. These subspecies are traditionally delimited based on both geography and floral morphology, although the latter is highly variable. Here, we determined the patterns of genetic (nuclear and cpDNA), morphological and ecological (bioclimatic) variation of a large number of P. axillaris populations and found that they are mostly coincident with subspecies delimitation. The nuclear data suggest that the subspecies are likely independent evolutionary units, and their morphological differences may be associated with local adaptations to diverse climatic and/or edaphic conditions and population isolation. The demographic dynamics over time estimated by skyline plot analyses showed different patterns for each subspecies in the last 100 000 years, which is compatible with a divergence time between 35 000 and 107 000 years ago between P. a. axillaris and P. a. parodii, as estimated with the IMa program. Coalescent simulation tests using Approximate Bayesian Computation do not support previous suggestions of extensive gene flow between P. a. axillaris and P. a. parodii in their contact zone. © 2013 John Wiley & Sons Ltd.","Background: Misidentifications between exploited species may lead to inaccuracies in population assessments, with potentially irreversible conservation ramifications if overexploitation of either species is occurring. A notable showcase is provided by the realization that the roundscale spearfish (Tetrapturus georgii), a recently validated species, has been historically misidentified as the morphologically very similar and severely overfished white marlin (Kajikia albida) (IUCN listing: Vulnerable). In effect, no information exists on the population status and evolutionary history of the enigmatic roundscale spearfish, a large, highly vagile and broadly distributed pelagic species. We provide the first population genetic evaluation of the roundscale spearfish, utilizing nuclear microsatellite and mitochondrial DNA sequence markers. Furthermore, we re-evaluated existing white marlin mitochondrial genetic data and present our findings in a comparative context to the roundscale spearfish. Results: Microsatellite and mitochondrial (control region) DNA markers provided mixed evidence for roundscale spearfish population differentiation between the western north and south Atlantic regions, depending on marker-statistical analysis combination used. Mitochondrial DNA analyses provided strong signals of historical population growth for both white marlin and roundscale spearfish, but higher genetic diversity and effective female population size (1.5-1.9X) for white marlin. Conclusions: The equivocal indications of roundscale spearfish population structure, combined with a smaller effective female population size compared to the white marlin, already a species of concern, suggests that a species-specific and precautionary management strategy recognizing two management units is prudent for this newly validated billfish. © 2014 Bernard et al.","Cochliomyia hominivorax (Coquerel) is a myiasis fly that causes economic losses to livestock farmers in warmer American regions. Previous studies of this pest had found population structure at north and south of the Amazon Basin, which was considered to be a barrier to dispersal. The present study analyzed three mitochondrial DNA (mtDNA) markers and eight nuclear microsatellite loci to investigate for the first time the genetic diversity and population structure across the Brazilian Amazon region (Amazonia). Both mtDNA and microsatellite data supported the existence of much diversity and significant population structure among nine regional populations of C. hominivorax, which was found to be surprisingly common in Amazonia. Forty-six mtDNA haplotypes were identified, of which 39 were novel and seven had previously been found only at south of Amazonia. Seventy microsatellite alleles were identified by size, moderate to high values of heterozygosity were discovered in all regions, and a Bayesian clustering analysis identified four genetic groups that were not geographically distributed. Reproductive compatibility was also investigated by laboratory crossing, but no evidence of hybrid dysgenesis was found between an Amazonian colony and one each of from Northeast and Southeast Brazil. The results have important implications for area-wide control by the Sterile Insect Technique. © 2014 International Atomic Energy Agency 2014.","The myiasis causing New World screwworm (NWS) fly is responsible for substantial losses to livestock breeders in the Americas. Due to the negative impact of the NWS fly in animal health, expansion of successful NWS fly eradication programmes is under discussion. However, the effects of geography and environmental diversity on NWS population structure and migration patterns need to be assessed before any political decision is made to implement such a programme. We present a GIS tool to construct potential connection corridors among sampling localities based on genetic and environmental data. We integrate, through a home-made python script, a friction raster based on a Maxent niche model and the pairwise ΦST statistic. Among 38 NWS fly sampling localities from South America, we find a high population connectivity among the sampling localities from the south of the Amazon region. The region along the Atlantic Ocean was identified as the most probable migration corridor between the north (NAG) and the south (SAG) of the Amazon region. The approach highlighted previously undetected population structure within NAG showing low to medium connectivity through the Andes, correlating with current understanding of NWS fly migration in South America. Also, the approach is flexible, allowing future research to incorporate other niche simulations and genetic differentiation metrics. With this flexibility, the tool could become part of any AW-IPM by helping to target regions for control. © 2014 International Atomic Energy Agency 2014.","Abrotrichini is a recently defined and diagnosed tribe of Sigmodontinae with a complex taxonomy. Abrotrichine genera, Abrothrix (including Chroeomys), Chelemys, Geoxus, Notiomys and Pearsonomys, are mostly distributed in the central and southern Andes and adjacent lowlands and show terrestrial and fossorial habits. Recent studies have evidenced some incongruence between current taxonomy and abrotrichine phylogeny, such as the polyphyly of Chelemys and paraphyly of Geoxus respect to Pearsonomys. We used DNA sequence data of six loci (one mitochondrial and five nuclear) to resolve the relationships within the tribe. Independent and combined analyses of these loci were carried out using parsimony, maximum likelihood and Bayesian inference. Estimates of divergence time of the main lineages of abrotrichines were calculated with a molecular clock using as calibration, a fossil recently found. The concatenated data set increased the resolution and defined the relationships within the tribe. Our phylogenetic analyses corroborate that Abrothrix is sister of a clade containing the long-clawed abrotrichines. We recovered two main clades within Abrothrix that match morphologic variation and geographic distribution of its species. In addition, we corroborated the lack of monophyly of Chelemys and the lack of monophyly of Geoxus. We discuss different taxonomic scenarios to abrotrichine classification reflects the phylogenetic relationships obtained in this study. Our molecular clock estimated the Abrotrichini crown age to be around the early Pliocene (4.4 Ma) and suggest that the tribe diversified over a short period of time. © 2014 Royal Swedish Academy of Sciences.","Genomic evaluation methods assume that the reference population is genotyped and phenotyped. This is most often false and the generation of pseudo-phenotypes is uncertain and inaccurate. However, markers obey transmission rules and therefore the covariances of marker genotypes across individuals can be modelled using pedigree relationships. Based on this, an extension of the genomic relationship matrix can be constructed in which genomic relationships are propagated to all individuals, resulting in a combined relationship matrix, which can be used in a BLUP procedure called the Single Step Genomic BLUP. This procedure provides so far the most comprehensive option for genomic evaluation. Several extensions, options and details are described: compatibility of genomic and pedigree relationships, Bayesian regressions, multiple trait models, computational aspects, etc. Many details scattered through a series of papers are put together into this paper. © 2014 Elsevier B.V.","Ecological speciation is well-known from adaptive radiations in cichlid fishes inhabiting lentic ecosystems throughout the African rift valley and Central America. Here, we investigate the ecological and morphological diversification of a recently discovered lotic predatory Neotropical cichlid species flock in subtropical South America. We document morphological and functional diversification using geometric morphometrics, stable C and N isotopes, stomach contents and character evolution. This species flock displays species-specific diets and skull and pharyngeal jaw morphology. Moreover, this lineage appears to have independently evolved away from piscivory multiple times and derived forms are highly specialized morphologically and functionally relative to ancestral states. Ecological speciation played a fundamental role in this radiation and our data reveal novel conditions of ecological speciation including a species flock that evolved: 1) in a piscivorous lineage, 2) under lotic conditions and 3) with pronounced morphological novelties, including hypertrophied lips that appear to have evolved rapidly. © 2013 Burress et al.","The presence of Gryne orensis (Sørensen) (Opiliones: Cosmetidae) in a Yungas locality (northwestern Argentina) is reported for the first time, providing new evidence for the Mesopotamian-Yungas disjunct pattern. Combining a total of 19 new Mesopotamian records with previous, reliable citations from the literature, a dataset of 45 points was used to model the potential distribution of the species, using the presence-only methods BIOCLIM and MAXENT. Models supported the existence of a distributional gap across the Semiarid Chaco. The imprecise literature record from \"El Impenetrable\", province of Chaco, is assigned to three tentative locations to evaluate if models are affected by their inclusion; in all cases, the disjunction was maintained. It was thereby estimated that the actual record might have originated in a site closer to the Humid Chaco and/or associated to streams. This paper also provides a statement of the bioclimatic profile and identification of major environmental constraints that define the range of G. orensis. Copyright © 2013 Magnolia Press.","There are not \"universal methods\" to determine diet composition of predators. Most traditional methods are biased because of their reliance on differential digestibility and the recovery of hard items. By relying on assimilated food, stable isotope and Bayesian mixing models (SIMMs) resolve many biases of traditional methods. SIMMs can incorporate prior information (i.e. proportional diet composition) that may improve the precision in the estimated dietary composition. However few studies have assessed the performance of traditional methods and SIMMs with and without informative priors to study the predators' diets. Here we compare the diet compositions of the South American fur seal and sea lions obtained by scats analysis and by SIMMs-UP (uninformative priors) and assess whether informative priors (SIMMs-IP) from the scat analysis improved the estimated diet composition compared to SIMMs-UP. According to the SIMM-UP, while pelagic species dominated the fur seal's diet the sea lion's did not have a clear dominance of any prey. In contrast, SIMM-IP's diets compositions were dominated by the same preys as in scat analyses. When prior information influenced SIMMs' estimates, incorporating informative priors improved the precision in the estimated diet composition at the risk of inducing biases in the estimates. If preys isotopic data allow discriminating preys' contributions to diets, informative priors should lead to more precise but unbiased estimated diet composition. Just as estimates of diet composition obtained from traditional methods are critically interpreted because of their biases, care must be exercised when interpreting diet composition obtained by SIMMs-IP. The best approach to obtain a near-complete view of predators' diet composition should involve the simultaneous consideration of different sources of partial evidence (traditional methods, SIMM-UP and SIMM-IP) in the light of natural history of the predator species so as to reliably ascertain and weight the information yielded by each method. © 2013 Franco-Trecu et al.","Insect pest phylogeography might be shaped both by biogeographic events and by human influence. Here, we conducted an approximate Bayesian computation (ABC) analysis to investigate the phylogeography of the New World screwworm fly, Cochliomyia hominivorax, with the aim of understanding its population history and its order and time of divergence. Our ABC analysis supports that populations spread from North to South in the Americas, in at least two different moments. The first split occurred between the North/Central American and South American populations in the end of the Last Glacial Maximum (15,300-19,000 YBP). The second split occurred between the North and South Amazonian populations in the transition between the Pleistocene and the Holocene eras (9,100-11,000 YBP). The species also experienced population expansion. Phylogenetic analysis likewise suggests this north to south colonization and Maxent models suggest an increase in the number of suitable areas in South America from the past to present. We found that the phylogeographic patterns observed in C. hominivorax cannot be explained only by climatic oscillations and can be connected to host population histories. Interestingly we found these patterns are very coincident with general patterns of ancient human movements in the Americas, suggesting that humans might have played a crucial role in shaping the distribution and population structure of this insect pest. This work presents the first hypothesis test regarding the processes that shaped the current phylogeographic structure of C. hominivorax and represents an alternate perspective on investigating the problem of insect pests. © 2013 Fresia et al.","The genotype of an individual and the environment as the maternal ability of its dam have substantial effects on the phenotype expression of many production traits. The aim of the present study was to estimate the (co)variance components for worm resistance, wool and growth traits in Merino sheep, testing the importance of maternal effects and to determine the most appropriate model for each trait. The traits analyzed were Greasy Fleece Weight (GFW), Clean Fleece Weight (CFW), average Fibre Diameter (FD), Coefficient of Variation of FD (CVFD), Staple Length (SL), Comfort Factor (CF30), Weaning Weight (WWT), Yearling Body Weight (YWT) and Faecal worm Egg Count (FEC). The data were recorded during a 15-year period from 1995 to 2010, from Uruguayan Merino stud flocks. A Bayesian analysis was performed to estimate (co)variance components and genetic parameters. By ignoring or including maternal genetic or environmental effects, five different univariate models were fitted in order to determine the most effective for each trait. For CVFD and YWT, the model fitting the data best included direct additive effects as the only significant random source of variation. For GFW, CFW, FD, SL and CF30 the most appropriate model included direct-maternal covariance; while for FEC included maternal genetics effects with a zero direct-maternal covariance. The most suitable model for WWT included correlated maternal genetic plus maternal permanent environmental effects. The estimates of direct heritability were moderate to high and ranged from 0.15 for log transformed FEC to 0.74 for FD. Most of the direct additive genetic correlation (rg) estimations were in the expected range for Merino breed. However, the estimate of rg between FEC and FD was unfavourable (-0.18±0.03). In conclusion, there is considerable genetic variation in the traits analyzed, indicating the potential to make genetic progress on these traits. This study showed that maternal effects are influencing most of traits analyzed, thus these effects should be considered in Uruguayan Merino breeding programs; since the implementation of an appropriate model of analysis is critical to obtain accurate estimates. © 2013 Elsevier B.V.","Previous studies indicate that the HIV-1 subtype C epidemic in southern Brazil was initiated by the introduction of a single founder strain probably originating from east Africa. However, the exact country of origin of such a founder strain as well as the origin of the subtype C viruses detected outside the Brazilian southern region remains unknown. HIV-1 subtype C pol sequences isolated in the southern, southeastern and central-western Brazilian regions (n = 209) were compared with a large number (n ~ 2,000) of subtype C pol sequences of African origin. Maximum-likelihood analyses revealed that most HIV-1 subtype C Brazilian sequences branched in a single monophyletic clade (CBR-I), nested within a larger monophyletic lineage characteristic of east Africa. Bayesian analyses indicate that the CBR-I clade most probably originated in Burundi and was introduced into the Paraná state (southern region) around the middle 1970s, after which it rapidly disseminated to neighboring regions. The states of Paraná and Santa Catarina have been the most important hubs of subtype C dissemination, and routine travel and spatial accessibility seems to have been the major driving forces of this process. Five additional introductions of HIV-1 subtype C strains probably originated in eastern (n = 2), southern (n = 2) and central (n = 1) African countries were detected in the Rio de Janeiro state (southeastern region). These results indicate a continuous influx of HIV-1 subtype C strains of African origin into Brazil and also unveil the existence of unrecognized transmission networks linking this country to east Africa. © 2013 Delatorre et al.","Until recently, most phylogeographic approaches have been unable to distinguish between demographic and range expansion processes, making it difficult to test for the possibility of range expansion without population growth and vice versa. In this study, we applied a Bayesian phylogeographic approach to reconstruct both demographic and range expansion in the lizard Liolaemus darwinii of the Monte Desert in Central Argentina, during the Late Quaternary. Based on analysis of 14 anonymous nuclear loci and the cytochrome b mitochondrial DNA gene, we detected signals of demographic expansion starting at ~55 ka based on Bayesian Skyline and Skyride Plots. In contrast, Bayesian relaxed models of spatial diffusion suggested that range expansion occurred only between ~95 and 55 ka, and more recently, diffusion rates were very low during demographic expansion. The possibility of population growth without substantial range expansion could account for the shared patterns of demographic expansion during the Last Glacial Maxima (OIS 2 and 4) in fish, small mammals and other lizards of the Monte Desert. We found substantial variation in diffusion rates over time, and very high rates during the range expansion phase, consistent with a rapidly advancing expansion front towards the southeast shown by palaeo-distribution models. Furthermore, the estimated diffusion rates are congruent with observed dispersal rates of lizards in field conditions and therefore provide additional confidence to the temporal scale of inferred phylogeographic patterns. Our study highlights how the integration of phylogeography with palaeo-distribution models can shed light on both demographic and range expansion processes and their potential causes. © 2013 John Wiley & Sons Ltd.","Despite the effectiveness of current hepatitis B virus (HBV) vaccines, it is estimated that 350 million individuals suffer from chronic HBV infection and more than 50% of these affected individuals live on the Asian continent. Panama is a country with a great diversity of foreign groups; the Chinese community is a large example of this phenomenon. There is an urgent need to perform studies that evaluate the prevalence and the genetic diversity of HBV in this community. This study aimed to evaluate the prevalence of HBV and its genotypes and mutant variants in the Chinese population residing in Panama. In total, 320 subjects were enrolled in the study. Forty-two subjects (13.1%) were positive for HBsAg and HBV-DNA from 18 subjects revealed the presence of genotypes B2 and C1. Secondary mutations associated with drug resistance at positions rtV207L and rtN239T of the reverse transcriptase gene were identified. Additionally, the mutation pair A1762T/G1764A was found in three samples and the mutation G1896A was detected in an HBeAg-negative subject. In conclusion, to our knowledge, this is the first study to report high HBV prevalence rates in resident ethnic Chinese in Central America and the presence of genotypes B2 and C1 in this region.","The Enterobacteriaceae are a large family of Proteobacteria that include many well-known prokaryotic genera, such as Escherichia, Yersinia and Salmonella. The main ideas of synonymous codon usage (CU) evolution and translational selection have been deeply influenced by studies with these bacterial groups. In this work we report the analysis of the CU pattern of completely sequenced bacterial genomes that belong to the Enterobacteriaceae. The effect of selection in translation acting at the levels of speed and accuracy, and phylogenetic trends within this group are described. Preferred (optimal) codons were identified. The evolutionary dynamics of these codons were studied and following a Bayesian approach these preferences were traced back to the common ancestor of the family. We found that there is some level of variation in selection among the analysed micro-organisms that is probably associated with lineage-specific trends. The codon bias was largely conserved across the evolutionary time of the family in highly expressed genes and protein conserved regions, suggesting a major role of negative selection. In this sense, the results support the idea that the extant CU bias is finely tuned over the ancestral well-conserved pool of tRNAs. © 2013 SGM.","Population differentiation in environments without well-defined geographical barriers represents a challenge for wildlife management. Based on a comprehensive database of individual sighting records (1988-2009) of blue whales from the winter/calving Gulf of California, we assessed the fine-scale genetic and spatial structure of the population using individual-based approaches. Skin samples of 187 individuals were analyzed for nine microsatellite loci. A single population with no divergence among years and months and no isolation by distance (Rxy = 0.1-0.001, p>0.05) were found. We ran two Bayesian clustering methods using Structure and Geneland softwares in two different ways: 1) a general analysis including all individuals in which a single cluster was identified with both softwares; 2) a specific analysis of females only in which two main clusters (Loreto Bay and northern areas, and San Jose-La Paz Bay area) were revealed by Geneland program. This study provides information indicating that blue whales wintering in the Gulf of California are part of a single population unit and showed a fine-scale structure among females, possibly associated with their high site fidelity, particularly when attending calves. It is likely that the loss of genetic variation is minimized by male mediated gene flow, which may reduce the genetic drift effect. Opportunities for kin selection may also influence calf survival and, in consequence, have a positive impact on population demography in this small and endangered population. © 2013 Costa-Urrutia et al.","With about 400 living species and 82 genera, rodents of the subfamily Sigmodontinae comprise one of the most diverse and more broadly distributed Neotropical mammalian clades. There has been much debate on the origin of the lineage or the lineages of sigmodontines that entered South America, the timing of entrance and different aspects of further diversification within South America. The ages of divergence of the main lineages and the crown age of the subfamily were estimated by using sequences of the interphotoreceptor retinoid binding protein and cytochrome b genes for a dense sigmodontine and muroid sampling. Bayesian inference using three fossil calibration points and a relaxed molecular clock estimated a middle Miocene origin for Sigmodontinae (∼12. Ma), with most tribes diversifying throughout the Late Miocene (6.9-9.4. Ma). These estimates together results of analyses of ancestral area reconstructions suggest a distribution for the most recent common ancestor of Sigmodontinae in Central-South America and a South American distribution for the most recent common ancestor of Oryzomyalia. © 2012 Elsevier Inc.","The first influenza pandemic of this century was declared in April of 2009, with the emergence of a novel H1N1 influenza A virus strain (H1N1pdm). Understanding the evolution of H1N1pdm populations within the South American region is essential for studying global diversification, emergence, resistance and vaccine efficacy. In order to gain insight into these matters, we have performed a Bayesian coalescent Markov Chain Monte Carlo analysis of hemagglutinin (HA) and neuraminidase (NA) gene sequences of all available and comparable HA and NA sequences obtained from H1N1pdm IAV circulating in the South American region. High evolutionary rates and fast population growths characterize the population dynamics of H1N1pdm strains in this region of the world. A significant contribution of first codon position to the mean evolutionary rate was found for both genes studied, revealing a high contribution of non-synonymous substitutions to the mean substitution rate. In the 178. days period covered by these studies, substitutions in all HA epitope regions can be observed. HA substitutions D239G/N and Q310H have been observed only in Brazilian patients. While substitution D239G/N is not particularly associated to a specific genetic lineage, all strains bearing substitution Q310H were assigned to clade 6, suggesting a founder effect. None of the substitutions found in the NA proteins of H1N1pdm strains isolated in South America appears sufficiently close to affect the drug binding pocket for the three NA inhibitor antivirals tested. A more detailed analysis of NA proteins revealed epitope differences among 2010 vaccine and H1N1pdm IAV strains circulating in the South American region. © 2012 Elsevier B.V.","The traditional mesoscopic paradigm represents DNA as a series of base-pair steps whose energy response to equilibrium perturbations is elastic, with harmonic oscillations (defining local stiffness) around a single equilibrium conformation. In addition, base sequence effects are often analysed as a succession of independent XpY base-pair steps (i.e. a nearest-neighbour (NN) model with only 10 unique cases). Unfortunately, recent massive simulations carried out by the ABC consortium suggest that the real picture of DNA flexibility may be much more complex. The paradigm of DNA flexibility therefore needs to be revisited. In this article, we explore in detail one of the most obvious violations of the elastic NN model of flexibility: the bimodal distributions of some helical parameters. We perform here an in-depth statistical analysis of a very large set of MD trajectories and also of experimental structures, which lead to very solid evidence of bimodality. We then suggest ways to improve mesoscopic models to account for this deviation from the elastic regime. © 2012 The Author(s).","Species delimitation is a major research focus in evolutionary biology because accurate species boundaries are a prerequisite for the study of speciation. New species delimitation methods (SDMs) can accommodate nonmonophyletic species and gene tree discordance as a result of incomplete lineage sorting via the coalescent model, but do not explicitly accommodate gene flow after divergence. Approximate Bayesian computation (ABC) can incorporate gene flow and estimate other relevant parameters of the speciation process while testing alternative species delimitation hypotheses. We evaluated the accuracy of BPP, SpeDeSTEM, and ABC for delimiting species using simulated data and applied these methods to empirical data from lizards of the Liolaemus darwinii complex. Overall, BPP was the most accurate, ABC showed an intermediate accuracy, and SpeDeSTEM was the least accurate under most simulated conditions. All three SDMs showed lower accuracy when speciation occurred despite gene flow, as found in previous studies, but ABC was the method with the smallest decrease in accuracy. All three SDMs consistently supported the distinctness of southern and northern lineages within L. darwinii. These SDMs based on genetic data should be complemented with novel SDMs based on morphological and ecological data to achieve truly integrative and statistically robust approaches to species discovery. © 2012 The Author(s). Evolution© 2012 The Society for the Study of Evolution.","Aim To elucidate the role of vicariance versus dispersal at the microevolutionary scale in annual killifish populations belonging to the Austrolebias bellottii species complex (Rivulidae). Within this complex, A. bellottii and A. apaii have low vagility and occur widely within the study area, making them excellent models for testing biogeographic hypotheses of differentiation. Location South America, in the Paraná-Uruguay-La Plata river basin. Methods Molecular data and morphometric analyses were used to reconstruct the phylogeographic history and morphological variation of 24 populations of two taxa of the A. bellottii species complex. Phylogenetic analyses using maximum likelihood (ML) and Bayesian inference (BI) model-based methods, estimates of clade divergence times implemented in beast, non-metric multidimensional scaling, analysis of molecular variance results, and morphological analyses elucidated the role of vicariance versus dispersal hypotheses in population differentiation in the aforementioned river basin. Results In the A. bellottii species complex from the Paraná-Uruguay-La Plata river basin, past allopatric fragmentation from vicariance events seems to be the most plausible scenario for diversification since the Late Miocene and more recently since the Plio-Pleistocene. The Plio-Pleistocene vicariance produced the differentiation of three major clades in A. bellottii populations. One clade from the eastern Uruguay River drainage was separated from another in western Uruguay and the Paraná-La Plata River drainages. A later vicariance event split populations to the south (lower Paraná-La Plata Basin) and north (middle Paraná) of the western Paraná River drainage. However, our results do not exclude the possibility of dispersal events among A. bellottii populations from both the Uruguay and Paraná river drainages, which could occur in these river basins during hypothesized connectivity cycles of the Late Pliocene and Pleistocene. Main conclusions Past allopatric fragmentation caused by different vicariance events seems to be the main driver of diversification in the A. bellottii species complex since the Plio-Pleistocene. However, the current molecular data suggest that populations from both drainages of the Paraná-Uruguay rivers may have experienced cycles of connectivity during the Pleistocene, perhaps including multiple vicariance or dispersal events from populations located in the western lower Uruguay River drainage, which encompassed climatic and geological changes in the Paraná-Uruguay-La Plata Basin. © 2012 Blackwell Publishing Ltd.","Gastrointestinal (GI) parasites are one of the main sanitary and economic constraints for sheep production worldwide. In Uruguay, resistance to GI parasites has been included in genetic evaluations of Merino and Corriedale breeds since 1994, using Faecal worm Egg Count measured in post-weaning lambs (lambFEC) as a selection criterion. Although adult categories are more resistant to GI parasites, a temporary loss of acquired immunity is present in the periparturient period (=periparturient rise). The purpose of the present study is to estimate the genetic parameters of resistance to GI parasites in periparturient ewes and post-weaning Merino lambs. A total 2110 faecal samples of 748 periparturient ewes (eweFEC), the progeny of 107 sires, were collected in 2009 and 2010. 9458 lambFEC records from 7506 lambs born between 2001 and 2009 were analysed as well. (Co)variance components and systematic effects were estimated using a multi-trait animal model, with a Bayesian analysis using the Gibbs sampler algorithm. Direct and correlated responses (DR and CR) of eweFEC to selection using eweFEC and lambFEC respectively, were also estimated. Posterior medians (posterior standard deviation) for heritability (h2) and repeatability were 0.25 (0.03) and 0.34 (0.02) for lambFEC, and 0.08 (0.03) and 0.18 (0.03) for eweFEC, respectively. Posterior median for genetic correlation between both traits was 0.81 (0.11). CR obtained by selecting for lambFEC was two times more efficient than DR from selection by eweFEC. In conclusion, eweFEC has a lower h2 than lambFEC while their genetic correlation is moderate to high. Therefore, indirect selection by using lambFEC will be more effective than direct selection on eweFEC, and will produce ewes that eliminate less worm eggs, resulting in less pasture contamination. © 2012 Elsevier B.V..","Intra- and inter-population genetic variability and the demographic history of Heliothis virescens (F.) populations were evaluated by using mtDNA markers (coxI, coxII and nad6) with samples from the major cotton- and soybean-producing regions in Brazil in the growing seasons 2007/08, 2008/09 and 2009/10. AMOVA indicated low and non-significant genetic structure, regardless of geographical scale, growing season or crop, with most of genetic variation occurring within populations. Clustering analyzes also indicated low genetic differentiation. The haplotype network obtained with combined datasets resulted in 35 haplotypes, with 28 exclusive occurrences, four of them sampled only from soybean fields. The minimum spanning network showed star-shaped structures typical of populations that underwent a recent demographic expansion. The recent expansion was supported by other demographic analyzes, such as the Bayesian skyline plot, the unimodal distribution of paired differences among mitochondrial sequences, and negative and significant values of neutrality tests for the Tajima's D and Fu's FS parameters. In addition, high values of haplotype diversity (Ĥ) and low values of nucleotide diversity (π), combined with a high number of low frequency haplotypes and values of θπ < θw suggested a recent demographic expansion of H. virescens populations in Brazil. This demographic event could be responsible for the low genetic structure currently found; however, haplotypes present uniquely at the same geographic regions and from one specific host plant suggest an initial differentiation among H. virescens populations within Brazil. © 2011 Cambridge University Press.","Phylogeography studies add insights into the geographic and evolutionary processes that underline the genetic divergence of populations. This work examines the geographic genetic structure of the Patagonian blennie, Eleginops maclovinus, a notothenioid (Perciformes) endemic to South American temperate and sub-Antarctic waters, using mitochondrial DNA cytochrome b sequences. We found 58 haplotypes in the analysis of 261 individual sequences of 833 base pairs in length. Among-population variance was very low (1. 62%) and many haplotypes were shared between several populations across the species geographic range. Genetic differentiation was not consistent with a simple model of isolation by distance, possibly suggesting a lack of equilibrium between gene flow and local genetic drift. The analysis of mismatch distributions, neutrality tests, and the Bayesian Skyline Plot showed a pattern consistent with a recent population expansion event that may have taken place during the Middle Pleistocene. © 2011 Springer-Verlag.","Premise of the study: The American bulb-bearing Oxalis (Oxalidaceae) have diverse heterostylous breeding systems and are distributed in mountainous areas from Patagonia to the northeastern United States. To study the evolutionary processes leading to this diversity, we constructed the first molecular phylogeny for the American bulb bearing Oxalis and used it to infer biogeographic history and breeding system evolution. Methods: We used DNA sequence data (nuclear ribosomal internal transcribed spacer, trnL-trnL-trnF, trnT-trnL, and psbJpetA) to infer phylogenetic history via parsimony, likelihood, and Bayesian analyses. We used Bayes Multistate to infer ancestral geographic distributions at well-supported nodes of the phylogeny. The Shimodaira-Hasegawa (SH) test distinguished among hypotheses of single or multiple transitions from South America to North America, and tristyly to distyly. Key results: The American bulb-bearing Oxalis include sampled members of sections Ionoxalis and Pseudobulbosae and are derived from a larger clade that includes members of sections Palmatifoliae, Articulatae, and the African species. The American bulb-bearing Oxalis comprise two clades: one distributed in SE South America and the other in the Andes and North America. An SH test supports multiple dispersals to North America. Most sampled distylous species form a single clade, but at least two other independent distylous lineages are supported by the topologies and SH tests. Conclusions: Phylogenetic results suggest the American bulb-bearing Oxalis originated in southern South America, dispersed repeatedly to North America, and had multiple transitions from tristyly to distyly. This study adds to our understanding of biogeographic history and breeding system evolution and provides a foundation for more precise inferences about the study group. © 2012 Botanical Society of America.","The Chiliotrichum group (Compositae: Astereae) is an informal taxonomic group consisting of shrubby daisies that grow mainly in the Andean and Patagonian regions of South America; it is composed of 31 species in 11 genera: Aylacophora, Cabreraea, Chiliotrichiopsis, Chiliophyllum, Chiliotrichum, Haroldia, Katinasia, Lepidophyllum, Llerasia, Nardophyllum, and Ocyroe. We analyzed phylogenetic relationships in the Chiliotrichum group using five DNA regions from two plant genomes: the plastid trnL-trnF and rpl32-trnL(UAG) spacers, the rpl16 intron, and the nuclear ribosomal internal and external transcribed spacers, ITS and ETS respectively. The dataset includes 21 ingroup species representing all the genera, and 10 outgroup species belonging to six genera. Maximum parsimony, maximum likelihood, and Bayesian inference analyses performed on the chloroplast and nuclear data support the monophyly of the group. The four major clades recovered on the basis of the combined nuclear evidence (ITS + ETS) are: (1) Llerasia clade, (2) Nardophyllum clade, (3) Chiliotrichum clade, and (4) a clade formed by Nardophyllum + Llerasia clades. As regards the status of the four non-monospecific genera, Chiliotrichum (2 spp.) and Llerasia (14 spp.) were recovered as monophyletic, whereas the other two, Nardophyllum (5 spp.) and Chiliotrichiopsis (3 spp.), were not recovered as monophyletic. These two are nonetheless recognized as distinct genera because of robust morphological support despite the lack of resolution in the phylogeny based solely on sequence data. The rest of the genera composing the group are monospecific, and they are scattered among the major clades in the group. The only genus with no clear position is Haroldia. An analysis of the morphological data shows that the presence of paleae, believed to function as a defensive structure for the protection of the achenes, is well correlated with other derived head and floret traits such as the number of florets per head and the type of pappus.","Populations of migratory birds can be affected by events happening at both breeding and wintering grounds. The Sahel is a vast region holding a large number of wintering trans-Saharan migratory European birds, and current land-use changes there may represent a threat for these species. We used satellite tracking data from the migratory Montagu's harrier to evaluate habitat use of the species during the wintering season, and whether the current network of protected areas is effective to provide their habitat needs during that season. We also developed an ecological niche model for the species in Western-Central Sahel to check if the most suitable sites are included within current protected areas. Tracked harriers occupied a large region encompassing a total of eight countries. The most preferred habitats during winter were croplands and some natural vegetation habitats, especially grasslands. Protected areas only covered a small proportion of the overall wintering grounds of tracked harriers and the most suitable areas for the species in Western-Central Sahel. Increasing the extent of preferred natural habitats within protected areas should benefit the conservation of this and probably other insect-eating raptors. However, substantial increases in extent and number of protected areas in sub-Saharan Africa are very unlike to occur. Conservation actions in the region should therefore be mostly focused on improving land use planning and management outside protected areas, specially enhancing agricultural practices to make biodiversity conservation compatible with poverty alleviation. These can be chiefly targeted at an area of <20,000km 2 of very suitable habitat for these species. © 2011 Elsevier Ltd.","We investigated population and social structure of the franciscana dolphin, Pontoporia blainvillei, an endemic and the most endangered cetacean of the southwestern Atlantic Ocean. We analyzed samples from the Rio de la Plata estuary obtained in Uruguayan waters and from the Atlantic Ocean obtained in both Uruguayan and Brazilian waters. Mitochondrial and microsatellite DNA markers were used to study differentiation between the estuary and the ocean and the association between kinship and social group structure. Although multilocus analyses suggested that franciscanas are structured into 2 subpopulations (K = 2, divergence among clusters: FST = 0.06, P = 0.002; RST = 0.3, P = 0.001), mitochondrial markers did not support such divergence (FST = 0.02, P = 0.12; ST = 0.06, P =0.06). However, these units are not entirely segregated geographically. Regarding social structure, some groups are composed by first-order related individuals (R ≥ 0.5, P < 0.5). Overall, the data suggest that matrilines could be the social unit in this species. We argue that the divergence found could be associated to local adaptation and social structure, resulting from either feature leading to a recent divergence or reflecting equilibrium between local differentiation and gene flow. This evidence supports considering franciscanas from the Rio de la Plata estuary a discrete management unit. © The American Genetic Association. 2011. All rights reserved.","Poultry production is a growing industry in Uruguay, South America. Mycoplasma gallisepticum (MG) and Mycoplasma synoviae (MS) are the etiological agent of chronic respiratory disease in chickens and may be horizontally and vertically infected. These economically important diseases can influence commercial poultry operations in Uruguay. Chickens infected with MG and/or MS become lessening in egg productivity, hatchability and feed efficiency and grow in mortality, waste carcass and veterinary costs. Increasingly, these possible performance and production deprivations require the strict control of MG and MS amongst all the poultry sectors. The objective of this study was to estimate the seroprevalence of the two pathogens mentioned above in relation to Mycoplasmosis in commercial chickens in Uruguay. Sera randomly selected from flocks of Uruguay were analysed using a commercial enzyme-linked immunosorbent assay (ELISA). Test prevalence were adjusted for the imperfect test characteristics using the Rogan-Gladen estimator (deterministic and stochastic functions) and Bayesian inference. Overall, 0.4% and 0.9% of the tested 1861 chickens were classified as test-positive for antibodies against MG and MS, respectively. Considering the simulated ELISA specificities of 99.5% for MG and 99.2% for MS and in the absence of other reported diagnoses from the area, these results were therefore considered to be \"false-positives\". The ELISA in combination with testing sera and statistical methods for true prevalence estimation provided a good survey tool at lower costs and time when compared to other approaches, and can be adapted to many other issues in animal health and production. © 2010 Nova Science Publishers, Inc. All rights reserved.","The objective of this study was to estimate the true prevalence of seropositive individual sheep against Chlamydophila abortus and Brucella ovis using the Rogan- Gladen estimator in combination with Bayesian inference in Uruguay, South America. Ten ovine herds were kept under observation in 2008. Each study herd was randomly selected at different farms recruited from the Artigas (no. of the study herds = 1), Salto (no. of the study herds = 8) and Canelones (no. of the study herds = 1) Departments. The required total sample size of 1385 was determined by power analysis. Blood samples collected were used for diagnostic investigations. Sera were analysed using a commercial ELISA for the detection of antibody to Chlamydophila abortus and Brucella ovis in sheep serum. Adjusted outcomes were required for precise comparison of seroprevalence estimates. The overall antibody seroprevalence of Chlamydophila abortus and Brucella ovis would be extremely low. Considering the simulated ELISA specificities of 99.3% for Chlamydophila abortus and 97.0% for Brucella ovis and in the absence of other reported diagnoses from the area, these results were therefore considered to be \"false-positives\". The methodology was useful for obtaining estimates of Chlamydophila abortus and Brucella ovis prevalence and for establishing prevalence distributions which could be used as input parameters in risk assessment and decision models. © 2010 Nova Science Publishers, Inc. All rights reserved.","South American tuco-tucos (Ctenomys) and the related coruro (Spalacopus) are two rodent lineages that have independently colonised the subterranean niche. The energetically demanding lifestyles of these species, coupled with the hypoxic atmospheres characteristic of subterranean environments, may have altered the selective regimes on genes encoding proteins related to cellular respiration. Here, we examined the molecular evolution of 13 protein-coding genes in the mitochondrial genome of seven caviomorph rodents, including these two subterranean genera and their above-ground relatives. Using maximum-likelihood and Bayesian approaches, we estimated rates of synonymous (dS) and nonsynonymous (dN) substitutions. We found a significantly higher ω ratio (d. N/d. S) in subterranean groups as compared to their non-subterranean counterparts in 11 of 13 genes, although no ω ratio was larger than 1. Additionally, we applied a method based on quantitative physicochemical properties to test for positive selection. Amino acid changes implicated in radical structural or functional shifts in the protein property were found to be ubiquitous across the phylogeny, but concentrated in the subterranean lineages. Convergent changes were also found between the subterranean genera used in this study and other mammals adapted to hypoxia. The results of this study suggest a link between niche shifts and weak directional (or episodic) selection at the molecular level against a background of purifying selection. © 2011 Elsevier Inc.","It is common for a majority of people to rank themselves as better than average on simple tasks and worse than average on difficult tasks. The literature takes for granted that this apparent misconfidence is problematic. We argue, however, that this behavior is consistent with purely rational Bayesian updaters. In fact, better-than-average data alone cannot be used to show overconfidence; we indicate which type of data can be used. Our theory is consistent with empirical patterns found in the literature. © 2011 The Econometric Society.","The amount of information available for physicians has dramatically increased in the recent past. In contrast, the specialist's ability to understand, synthesize and take into account such information is severely constrained by the short time available for the appointments. Therefore, systems reusing available knowledge and implementing reasoning processes become critical to support the tasks of the doctors. As a number of different techniques for building such systems are available, contrasting their effectiveness becomes a major concern. This is especially important in the case of infectious diseases that can be lethal within hours such as the Acute Bacterial Meningitis (ABM) for which implementing and contrasting different techniques allows for an increased reliability and speed in supporting the process of diagnosis. This work focuses on the construction of diagnosis support tools for ABM, reporting a comparative assessment of the quality of a Clinical Decision Support System (CDSS) resulting from the application of Case Based Reasoning (CBR), to that of an existing CDSS system developed using a Bayesian expert system. Although both approaches proved to be useful, the one based in CBR techniques show some interesting capabilities as higher precision, automatic learning or experience capturing, and also a better response to lack of input data. The three developed systems perform with high levels of accuracy- e.g. propose correct diagnostics based on a certain set of symptoms - but the one based on CBR present some additional capabilities that look very promising for implementing these kind of systems in a real world scenario. © 2011 Published by Elsevier Ltd.","We present the most comprehensive study to date of species groups in Ctenomys (tuco-tucos), a species-rich genus of Neotropical rodents. To explore phylogenetic relationships among 38 species and 12 undescribed forms we sequenced the complete mitochondrial cytochrome-b genes of 34 specimens and incorporated 50 previously published sequences. Parsimony, likelihood, and Bayesian phylogenetic analyses were performed using additional hystricognath rodents as outgroup taxa. The basal dichotomy of Ctenomys splits C. sociabilis from the remaining tuco-tucos, within which 8 main species groups were identified: boliviensis, frater, mendocinus, opimus, magellanicus, talarum, torquatus, and tucumanus. Whereas most of these groups refer to previous clades proposed on the basis of chromosomes or morphology, the torquatus and magellanicus species groups are novel taxonomic hypotheses. However, relationships among species groups are poorly resolved. Furthmore, the positions of C. leucodon, C. maulinus, and C. tuconax are conflicting or unresolved, and they might represent additional independent lineages. On the basis of molecular dating, we estimate that most species groups originated approximately 3 million years ago. © American 2011 Society of Mammalogists.","Data of broiler chickens for 2 pure lines across 3 generations were used for genomic evaluation. A complete population (full data set; FDS) consisted of 183,784 and 164,246 broilers for the 2 lines. The genotyped subsets (SUB) consisted of 3,284 and 3,098 broilers with 57,636 SNP. Genotyped animals were preselected based on more than 20 traits with different index applied to each line. Three traits were analyzed: BW at 6 wk (BW6), ultrasound measurement of breast meat (BM), and leg score (LS) coded 1 = no and 2 = yes for leg defect. Some phenotypes were missing for BM. The training population consisted of the first 2 generations including all animals in FDS or only genotyped animals in SUB. The validation data set contained only genotyped animals in the third generation. Genetic evaluations were performed using 3 approaches: 1) phenotypic BLUP, 2) extending BLUP methodologies to utilize pedigree and genomic information in a single step (ssGBLUP), and 3) Bayes A. Whereas BLUP and ssGBLUP utilized all phenotypic data, Bayes A could use only those of the genotyped subset. Heritabilities were 0.17 to 0.20 for BW6, 0.30 to 0.35 for BM, and 0.09 to 0.11 for LS. The average accuracies of the validation population with BLUP for BW6, BM, and LS were 0.46, 0.30, and <0 with SUB and 0.51, 0.34, and 0.28 with FDS. With ssGBLUP, those accuracies were 0.60, 0.34, and 0.06 with SUB and 0.61, 0.40, and 0.37 with FDS, respectively. With Bayes A, the accuracies were 0.60, 0.36, and 0.09 with SUB. With SUB, Bayes A and ssGBLUP had similar accuracies. For traits of high heritability, the accuracy of Bayes A/SUB and ssGBLUP/FDS were similar, and up to 50% better than BLUP/FDS. However, with low heritability, ssGBLUP/ FDS was 4 to 6 times more accurate than Bayes A/SUB and 50% better than BLUP/FDS. An optimal genomic evaluation would be multi-trait and involve all traits and records on which selection is based. ©2011 American Society of Animal Science.","This work presents a research about the applicability of Case Based Reasoning to Clinical Decision Support Systems (CDSS), particularly applied to the diagnosis of the disease known as Acute Bacterial Meningitis. In the last few years, the amount of information available to the medical doctor, who usually finds himself in the situation of making a diagnosis of one or more diseases, has dramatically increased. However, the specialist's ability to understand, synthesize and take advantage of such information in the always-little time during the medical act remains to be developed. Many contributions have been made by the computer sciences, especially those by Artificial intelligence, in order to solve these problems. This work focuses on the diagnose of the Acute Bacterial Meningitis, and carries out a comparative assessment of the quality of a Clinical Decision Support System made through Case Based Reasoning, in contrast to an already existing CDSS applied to the same task, but developed using a technique called Bayesian expert system. © Springer Science+Business Media B.V. 2010.","The objectives of this study were (1) to explore traits that better capture weekly or monthly changes in somatic cell counts (SCC) than does the commonly used lactation-average SCC, (2) to estimate their heritabilities and relationships to clinical mastitis (CM), and (3) to determine if these traits are feasible for use in monthly testing schemes. Clinical mastitis and weekly test-day (TD) records of SCC and milk production traits from 1,006 lactations of Swedish Red and Holstein cows collected from 1989 to 2004 were used (data set W). A data subset was also created to mimic monthly recording (data set M, 980 lactations). Twenty SCC traits were defined, taking into account SCC general levels and variation along the lactation curve, time and level of infection, and time of recovery. To reduce dimensionality, cluster and stepwise logistic regression procedures were applied. In data set W, 3 traits, \" standard deviation of SCC over the lactation,\" a discrete (0/1) indicator of \" at least one TD with SCC >500,000 cells/mL\" , and \" number of days sick in the widest SCC peak\" (DWidest) were the variables kept both with cluster procedures and a stepwise logistic regression with the logit of CM as dependent variable. In data set M, DWidest was replaced by \" number of SCC peaks\" and \" average number of days sick per peak\" (ADSick). Lactation-average SCC (in the first 150 d or between 150 and 305 d) did not enter into the logistic regression. Heritability estimates obtained for these new traits under a Bayesian setting and a Gibbs sampling approach were 10 to 16% (except for ADSick: 5%). Heritabilities were at least as high in the monthly data set as in the weekly data set. Thus, these SCC traits seem promising for use in breeding programs based on monthly milk recording. © 2010 American Dairy Science Association.","Background. Dengue virus (DENV) is a member of the genus Flavivirus of the family Flaviviridae. DENV are comprised of four distinct serotypes (DENV-1 through DENV-4) and each serotype can be divided in different genotypes. Currently, there is a dramatic emergence of DENV-3 genotype III in Latin America. Nevertheless, we still have an incomplete understanding of the evolutionary forces underlying the evolution of this genotype in this region of the world. In order to gain insight into the degree of genetic variability, rates and patterns of evolution of this genotype in Venezuela and the South American region, phylogenetic analysis, based on a large number (n = 119) of envelope gene sequences from DENV-3 genotype III strains isolated in Venezuela from 2001 to 2008, were performed. Results. Phylogenetic analysis revealed an in situ evolution of DENV-3 genotype III following its introduction in the Latin American region, where three different genetic clusters (A to C) can be observed among the DENV-3 genotype III strains circulating in this region. Bayesian coalescent inference analyses revealed an evolutionary rate of 8.48 × 10-4 substitutions/site/year (s/s/y) for strains of cluster A, composed entirely of strains isolated in Venezuela. Amino acid substitution at position 329 of domain III of the E protein (AV) was found in almost all E proteins from Cluster A strains. Conclusions. A significant evolutionary change between DENV-3 genotype III strains that circulated in the initial years of the introduction in the continent and strains isolated in the Latin American region in recent years was observed. The presence of DENV-3 genotype III strains belonging to different clusters was observed in Venezuela, revealing several introduction events into this country. The evolutionary rate found for Cluster A strains circulating in Venezuela is similar to the others previously established for this genotype in other regions of the world. This suggests a lack of correlation among DENV genotype III substitution rate and ecological pattern of virus spread. © 2010 Ramírez et al; licensee BioMed Central Ltd.","This study tests the hypothesis that female South American sea lions shift from off-shore, pelagic prey to coastal, benthic prey after parturition in order to reduce the foraging trip duration and hence the time pups remain unattended on the beach during early lactation. The δ13C and δ15N values of the serum and blood cells of 26 South American sea lion suckling pups from northern Patagonia were used to track the dietary changes of their mothers from late pregnancy to early lactation, after correction for differential isotopic fractionation between tissues. Primary producers and potential prey species were also analysed to establish a baseline for interpreting the stable isotope concentration of serum and blood cells. Isotopic ratios revealed a generalized increase in the consumption of coastal-benthic prey after parturition. Such a generalized post-partum shift will allow females to spend more time on land and look after their pups. The effects of this foraging strategy on the nutritional quality of the female's diet are discussed.","Background: Although HIV-1 CRF12_BF and CRF38_BF are two epidemiologically important recombinant lineages circulating in Argentina and Uruguay, little is known about their population dynamics.Methods: A total of 120 \"CRF12_BF-like\" and 20 \"CRF38_BF-like\" pol recombinant sequences collected in Argentina and Uruguay from 1997 to 2009 were subjected to phylogenetic and Bayesian coalescent-based analyses to estimate evolutionary and demographic parameters.Results: Phylogenetic analyses revealed that CRF12_BF viruses from Argentina and Uruguay constitute a single epidemic with multiple genetic exchanges among countries; whereas circulation of the CRF38_BF seems to be confined to Uruguay. The mean estimated substitution rate of CRF12_BF at pol gene (2.5 × 10-3 substitutions/site/year) was similar to that previously described for subtype B. According to our estimates, CRF12_BF and CRF38_BF originated at 1983 (1978-1988) and 1986 (1981-1990), respectively. After their emergence, the CRF12_BF and CRF38_BF epidemics seem to have experienced a period of rapid expansion with initial growth rates of around 1.2 year-1 and 0.9 year-1, respectively. Later, the rate of spread of these CRFs_BF seems to have slowed down since the mid-1990s.Conclusions: Our results suggest that CRF12_BF and CRF38_BF viruses were generated during the 1980s, shortly after the estimated introduction of subtype F1 in South America (~1975-1980). After an initial phase of fast exponential expansion, the rate of spread of both CRFs_BF epidemics seems to have slowed down, thereby following a demographic pattern that resembles those previously reported for the HIV-1 epidemics in Brazil, USA, and Western Europe. © 2010 Bello et al; licensee BioMed Central Ltd.","Background: The population genetic structure of subterranean rodent species is strongly affected by demographic (e.g. rates of dispersal and social structure) and stochastic factors (e.g. random genetic drift among subpopulations and habitat fragmentation). In particular, gene flow estimates at different spatial scales are essential to understand genetic differentiation among populations of a species living in a highly fragmented landscape. Ctenomys australis (the sand dune tuco-tuco) is a territorial subterranean rodent that inhabits a relatively secure, permanently sealed burrow system, occurring in sand dune habitats on the coastal landscape in the south-east of Buenos Aires province, Argentina. Currently, this habitat is threatened by urban development and forestry and, therefore, the survival of this endemic species is at risk. Here, we assess population genetic structure and patterns of dispersal among individuals of this species at different spatial scales using 8 polymorphic microsatellite loci. Furthermore, we evaluate the relative importance of sex and habitat configuration in modulating the dispersal patterns at these geographical scales.Results: Our results show that dispersal in C. australis is not restricted at regional spatial scales (~ 4 km). Assignment tests revealed significant population substructure within the study area, providing support for the presence of two subpopulations from three original sampling sites. Finally, male-biased dispersal was found in the Western side of our study area, but in the Eastern side no apparent philopatric pattern was found, suggesting that in a more continuous habitat males might move longer distances than females.Conclusions: Overall, the assignment-based approaches were able to detect population substructure at fine geographical scales. Additionally, the maintenance of a significant genetic structure at regional (~ 4 km) and small (less than 1 km) spatial scales despite apparently moderate to high levels of gene flow between local sampling sites could not be explained simply by the linear distance among them. On the whole, our results support the hypothesis that males disperse more frequently than females; however they do not provide support for strict philopatry within females. © 2010 Mora et al; licensee BioMed Central Ltd.","Background: The subtype diversity of the hepatitis C virus (HCV) genotypes is unknown in Venezuela. Methodology/Principal Findings: Partial sequencing of the NS5B region was performed in 310 isolates circulating in patients from 1995 to 2007. In the samples collected between 2005 and 2007, HCV genotype 1 (G1) was the most common genotype (63%), composed as expected of mainly G1a and G1b. G2 was the second most common genotype (33%), being G2a almost absent and G2j the most frequent subtype. Sequence analysis of the core region confirmed the subtype assignment performed within the NS5b region in 63 isolates. The complete genome sequence of G2j was obtained. G2j has been described in France, Canada and Burkina Fasso, but it was not found in Martinique, where several subtypes of G2 circulate in the general population. Bayesian coalescence analysis indicated a most recent common ancestor (MRCA) of G2j around 1785, before the introduction of G1b (1869) and G1a (1922). While HCV G1a and G1b experienced a growth reduction since 1990, coincident with the time when blood testing was implemented in Venezuela, HCV G2j did not seem to reach growth equilibrium during this period. Conclusions/Significance: Assuming the introduction of G2j from Africa during the slave trade, the high frequency of G2j found in Venezuela could suggest: 1- the introduction of African ethnic groups different from the ones introduced to Martinique or 2- the occurrence of a founder effect. This study represents an in-depth analysis of the subtype diversity of HCV in Venezuela, which is still unexplored in the Americas and deserves further studies. © 2010 Sulbaran et al.","The L2-Boosting algorithm is one of the most promising machine-learning techniques that has appeared in recent decades. It may be applied to high-dimensional problems such as whole-genome studies, and it is relatively simple from a computational point of view. In this study, we used this algorithm in a genomic selection context to make predictions of yet to be observed outcomes. Two data sets were used: (1) productive lifetime predicted transmitting abilities from 4702 Holstein sires genotyped for 32 611 single nucleotide polymorphisms (SNPs) derived from the Illumina BovineSNP50 BeadChip, and (2) progeny averages of food conversion rate, pre-corrected by environmental and mate effects, in 394 broilers genotyped for 3481 SNPs. Each of these data sets was split into training and testing sets, the latter comprising dairy or broiler sires whose ancestors were in the training set. Two weak learners, ordinary least squares (OLS) and non-parametric (NP) regression were used for the L2-Boosting algorithm, to provide a stringent evaluation of the procedure. This algorithm was compared with BL [Bayesian LASSO (least absolute shrinkage and selection operator)] and BayesA regression. Learning tasks were carried out in the training set, whereas validation of the models was performed in the testing set. Pearson correlations between predicted and observed responses in the dairy cattle (broiler) data set were 065 (033), 053 (037), 066 (026) and 063 (027) for OLS-Boosting, NP-Boosting, BL and BayesA, respectively. The smallest bias and mean-squared errors (MSEs) were obtained with OLS-Boosting in both the dairy cattle (008 and 108, respectively) and broiler (0011 and 0006) data sets, respectively. In the dairy cattle data set, the BL was more accurate (bias=010 and MSE=110) than BayesA (bias=126 and MSE=281), whereas no differences between these two methods were found in the broiler data set. L2-Boosting with a suitable learner was found to be a competitive alternative for genomic selection applications, providing high accuracy and low bias in genomic-assisted evaluations with a relatively short computational time. © 2010 Cambridge University Press.","The objective of this study was to estimate the true prevalence of individual chickens serologically test-positive against avian pneumovirus and Ornithobacterium rhinotracheale in Uruguay. Seventeen different broiler farms existed in three different provinces in Uruguay were recruited and the 1861 broilers were investigated. Individual-chicken sera were analyzed using a commercial enzyme-linked immunosorbent assay. The overall true seroprevalence was 1.9% [95% Bayesian Credible Interval (BCI): <1-7.4%] and less than 0.1% (95% BCI: 0-<0.1%) against avian pneumovirus and Ornithobacterium rhinotracheale, respectively. The result seroprevalence was relatively lower than that reported by other authors in a neighbouring country Argentina where the infection of these diseases was recently observed. This difference was discussed. © Asian Network for Scientific Information, 2010.","The objective of this study was to estimate the true prevalence of seropositive broiler chickens against avian pneumovirus at flock-level in Uruguay, using the Rogan-Gladen estimator in conjunction with Bayesian inference. A total of 181 pooled samples (consisting of 10 individual-chicken sera each) from the study area were examined with the enzyme-linked immunosorbent assay. All individual-chicken samples in the pools were also examined with the same assay. Forty-four pools were classified as test positive, because they included at least one individual-chicken classified as positive. The estimates for the deterministic (Rogan-Gladen approach) and stochastic (Bayesian approach) true prevalence were 30.9% [95% confidence interval (CI): 26.8-35.0%] and 31.4% (95% CI: 15.4-49.5%), respectively. © Asian Network for Scientific Information, 2010.","The objective of this study was to estimate the true prevalence of seropositive broiler flocks against Ornithobacterium rhinotracheale in Uruguay, South America. Seventeen farms of broiler chickens greater than 35 days of age were studied. The field investigation was conducted between October 2008 and April 2009. Individual-chicken sera and pooled sera (containing 10 individual-chicken sera each) were analyzed using a commercial ELISA for the detection of antibody against Ornithobacterium rhinotracheale in chicken serum. A total of 181 pooled samples from the study area representing 17 farms were examined. Fifty-four pools were classified as test positive, because they included at least one individual-chicken classified as positive. On the basis of deterministic approach, the estimates for the apparent prevalence and true prevalence at flock-level were 30% and 17%, respectively. The true prevalence estimate with the Bayesian model (stochastic approach) was slightly lower and having wider confidence intervals [11% (95% CI: 0%-32%)]. © Asian Network for Scientific Information, 2010.","Background. A sudden emergence of Influenza A Virus (IAV) infections with a new pandemic H1N1 IAV is taking place since April of 2009. In order to gain insight into the mode of evolution of these new H1N1 strains, we performed a Bayesian coalescent Markov chain Monte Carlo (MCMC) analysis of full-length neuraminidase (NA) gene sequences of 62 H1N1 IAV strains (isolated from March 30th to by July 28th, 2009). Results. The results of these studies revealed that the expansion population growth model was the best to fit the sequence data. A mean of evolutionary change of 7.84 × 10 -3 nucleotide substitutions per site per year (s/s/y) was obtained for the NA gene. A significant contribution of first codon position to this mean rate was observed. Maximum clade credibility trees revealed a rapid diversification of NA genes in different genetic lineages, all of them containing Oseltamivir-resistant viruses of very recent emergence. Mapping of naturally occurring amino acid substitutions in the NA protein from 2009 H1N1 IAV circulating in 62 different patients revealed that substitutions are distributed all around the surface of the molecule, leaving the hydrophobic core and the catalytic site essentially untouched. Conclusion. High evolutionary rates and fast population growth have contributed to the initial transmission dynamics of 2009 H1N1 IAV. Naturally occurring substitutions are preferentially located at the protein surface and do not interfere with the NA active site. Antigenic regions relevant for vaccine development can differ from previous vaccine strains and vary among patients. © 2009 Gõi et al; licensee BioMed Central Ltd.","Noroviruses (NoV) are a leading cause of outbreaks of nonbacterial acute gastroenteritis in humans worldwide and have become an important cause of hospitalization of children in South America. NoV belong to the family Caliciviridae and are non-enveloped single stranded, positive sense, RNA viruses. NoV of genotype GII/4 have emerged worldwide, causing four epidemic seasons of viral gastroenteritis during which four novel variants emerged. Despite the importance of NoV outbreaks, little is known about the evolutionary rates, viral spread and population dynamics of NoV populations. In order to gain insight into these matters, a Bayesian Markov chain Monte Carlo (MCMC) approach was used to analyze region D or full-length VP1 gene sequences of GII/4 NoV populations isolated in Brazil or Japan, respectively. The results of these studies revealed that the expansion population growth model was the best to fit the data in both datasets. The dates of the most common recent ancestors revealed that these viruses can quickly emerge in a geographical location. A mean evolutionary rate of 1.21 × 10-2 nucleotide substitution/site/year (s/s/y) was obtained for the VP1 gene using full-length sequences. This rate is higher than the rates reported for other rapidly evolving RNA. Roughly similar rates (1.44 × 10-2 s/s/y) were found using region D sequences, revealing the suitability of this region for evolutionary studies, in agreement with previous reports. High evolutionary rates and fast population growth may have contributed to the vigorous initial transmission dynamics of the GII/4 NoV populations studied. © 2009 Elsevier B.V. All rights reserved.","The availability of genomewide dense markers brings opportunities and challenges to breeding programs. An important question concerns the ways in which dense markers and pedigrees, together with phenotypic records, should be used to arrive at predictions of genetic values for complex traits. If a large number of markers are included in a regression model, marker-specific shrinkage of regression coefficients may be needed. For this reason, the Bayesian least absolute shrinkage and selection operator (LASSO) (BL) appears to be an interesting approach for fitting marker effects in a regression model. This article adapts the BL to arrive at a regression model where markers, pedigrees, and covariates other than markers are considered jointly. Connections between BL and other marker-based regression models are discussed, and the sensitivity of BL with respect to the choice of prior distributions assigned to key parameters is evaluated using simulation. The proposed model was fitted to two data sets from wheat and mouse populations, and evaluated using cross-validation methods. Results indicate that inclusion of markers in the regression further improved the predictive ability of models. An R program that implements the proposed model is freely available. Copyright © 2009 by the Genetics Society of America.","Dengue virus (DENV) is a member of the genus Flavivirus of the family Flaviviridae. DENV-3 re-emerged in Central America in 1994, and continues to expand into the South American region. Little is known about the evolutionary rates, viral spread and population dynamics of this genotype in the Latin American region. In order to gain insight into these matters, we used a Bayesian Markov chain Monte Carlo (MCMC) approach, to analyze envelope (E) gene sequences of the DENV-3 genotype III of strains included in a monophyletic cluster composed by Ecuadorian as well as strains from Cuba, Puerto Rico and Peru. The results of these studies revealed that the expansion population growth model was the best fit to the data. The most common recent ancestor (MRCA) was placed around 1989, in agreement with the first reports of the emergence of this new DENV-3 type. A mean rate 1.033 × 10-3 nucleotide substitution per site per year was obtained. This rate is comparatively higher than the ones obtained for DENV-2 and DENV-4 in the same region. Faster population growth and greater population dispersal may have contributed to the vigorous initial transmission dynamics of this genotype in the Latin American region. © 2009 Elsevier B.V. All rights reserved.","The objective of the present study was to assess the predictive ability of subsets of single nucleotide polymorphism (SNP) markers for development of low-cost, low-density genotyping assays in dairy cattle. Dense SNP genotypes of 4,703 Holstein bulls were provided by the USDA Agricultural Research Service. A subset of 3,305 bulls born from 1952 to 1998 was used to fit various models (training set), and a subset of 1,398 bulls born from 1999 to 2002 was used to evaluate their predictive ability (testing set). After editing, data included genotypes for 32,518 SNP and August 2003 and April 2008 predicted transmitting abilities (PTA) for lifetime net merit (LNM$), the latter resulting from progeny testing. The Bayesian least absolute shrinkage and selection operator method was used to regress August 2003 PTA on marker covariates in the training set to arrive at estimates of marker effects and direct genomic PTA. The coefficient of determination (R2) from regressing the April 2008 progeny test PTA of bulls in the testing set on their August 2003 direct genomic PTA was 0.375. Subsets of 300, 500, 750, 1,000, 1,250, 1,500, and 2,000 SNP were created by choosing equally spaced and highly ranked SNP, with the latter based on the absolute value of their estimated effects obtained from the training set. The SNP effects were re-estimated from the training set for each subset of SNP, and the 2008 progeny test PTA of bulls in the testing set were regressed on corresponding direct genomic PTA. The R2 values for subsets of 300, 500, 750, 1,000, 1,250, 1,500, and 2,000 SNP with largest effects (evenly spaced SNP) were 0.184 (0.064), 0.236 (0.111), 0.269 (0.190), 0.289 (0.179), 0.307 (0.228), 0.313 (0.268), and 0.322 (0.291), respectively. These results indicate that a low-density assay comprising selected SNP could be a cost-effective alternative for selection decisions and that significant gains in predictive ability may be achieved by increasing the number of SNP allocated to such an assay from 300 or fewer to 1,000 or more. © American Dairy Science Association, 2009.","Dairy industry in Uruguay has grown steadily over the last three decades, positioning it as the sector with greatest growth amongst exporting countries of dairy products. Milk production expanded from 765 million metric tonnes in 1976 to 1770 million metric tonnes in 2006 at a 4.3% annual growth rate. Meanwhile, infectious diseases can play an important role in the economic feasibility of a dairy industry. An infectious agent circulating in a dairy herd, as well as the costs associated with an outbreak, can contribute to fertility problems, abortions, decreased milk production and loss of bodyweight. Prevented by meticulous attention with knowledge of the disease status in a dairy herd, dairy farmers and veterinarians can collectively draw up an action plan to manage the diseases in question. In Uruguay, there are diagnostic testing facilities for most infectious diseases in dairy cattle in relation to the problems mentioned above, however, dairy cattle health schemes covering infectious bovine rhinotracheitis (IBR), bovine viral diarrhoea (BVD), leptospirosis, Johne's disease, neosporosis and campylobacteriosis are not relatively organised. This chapter discusses how to monitor a number of infectious diseases that are of particular economic importance to the dairy cattle industry in Uruguay, namely, IBR and BVD, using one of Bayesian analytic approaches \"a random effects hierarchical model analysis\". © 2009 Nova Science Publishers, Inc. All rights reserved.","Among pelagic fish, the Southwestern Atlantic menhaden genus Brevoortia (Clupeidae, Alosinae) constitutes an important species model to investigate the patterns of genetic differentiation. It is abundant in the Río de la Plata estuary and in the Atlantic coastal lagoons system from Uruguay and Southern Brazil. To access in the taxa discrimination and population structure in Brevoortia we perform a phylogeographic approach based on mitochondrial cytochrome b (cyt-b) sequences including 240 individuals from 16 collecting sites. Among the 720 bp cyt-b sequenced, 199 correspond to variables and 88 to phylogenetically informative sites. High values of haplotype diversity (h = 1.000) and nucleotide diversity (π = 0.061), as well as an average of 0.084 polymorphic segregating sites and 46 different haplotypes were found. Maximum likelihood analysis based on the GTR + I + G model and Bayesian inference strongly support the idea that B. aurea is the only species of the genus inhabiting the Southwestern Atlantic region. Our analyses revealed a complex population pattern characterized by the existence of long-term highly structured genetic assemblages of mixed stocks. Each monophyletic entity included individuals from different collecting sites, different age groups and collected in different years. Our data also suggest that the recruitment of unrelated mtDNA haplotypes carried out by individuals within schools could be occurring in the same nursery areas revealing the existence of many different maternal lineages. A scenario where different simultaneously and successively mixed mtDNA lineages remain historically connected through basal haplotypes among different clades could explains more accurately the complex and ordered metapopulation dynamic found in this pelagic fish. © 2008 Springer-Verlag.","Pimelodus albicans Valenciennes, 1840 (common name \"moncholo\" or \"bagre blanco\") is an endemic species of the family Pimelodidae in the río de la Plata basin. Phylogenetic approach based on cytochrome b sequences was performed to test the existence of a unique evolutionary lineage in P. albicans and to discriminate populations units or subpopulations related to a migration behavior of this taxon in the río de la Plata basin. This study included 34 samples of P. albicans of different collecting sites in the río de la Plata estuary and in the río Arrecifes belonging to the río Paraná basin. Among 614 base pairs in the cytochrome b sequence data set, 203 were variable and 120 were phylogenetically informative sites in P. albicans. A total of twenty haplotypes, nucleotide diversity (π) = 0.032 and haplotype diversity = 0.941 were found. Tajima's test showed significant value D= -1.88 (p<0.05) rejecting the neutral mutation hypothesis for the P. albicans data set. All phylogenetic approaches showed that P. albicans included four monophyletic assemblages that were supported by high bootstrap and Bayesian posterior probability values. Minimum spanning network corroborated these groups for P. albicans haplotypes. High genetic structure was found in P. albicans by means of AMOVA analysis showing that the río Arrecifes samples constitute an isolated lineage. Moreover, the high value of genetic divergence (10%) between the río de la Plata and the río Arrecifes populations could suggest that P. albicans may be conformed by a sibling species complex. On the other hand, a degree of genetic structuring was detected among different sites of the río de la Plata. A partial isolation of the 760 site may suggest that P. albicans could migrates to different tributaries for reproduction, generating different schools of haplotypes which could mix in the río de la Plata estuary. The high nucleotide diversity found in the 765 site and the existence of gene flow with the remaining collecting sites would be concordant with the outlined hypothetic scenarios of the mixing populations in the middle of the río de la Plata estuary. Copyright © 2008 Sociedade Brasileira de Ictiologia.","Dark spots in the fleece area are often associated with dark fibres in wool, which limits its competitiveness with other textile fibres. Field data from a sheep experiment in Uruguay revealed an excess number of zeros for dark spots. We compared the performance of four Poisson and zero-inflated Poisson (ZIP) models under four simulation scenarios. All models performed reasonably well under the same scenario for which the data were simulated. The deviance information criterion favoured a Poisson model with residual, while the ZIP model with a residual gave estimates closer to their true values under all simulation scenarios. Both Poisson and ZIP models with an error term at the regression level performed better than their counterparts without such an error. Field data from Corriedale sheep were analysed with Poisson and ZIP models with residuals. Parameter estimates were similar for both models. Although the posterior distribution of the sire variance was skewed due to a small number of rams in the dataset, the median of this variance suggested a scope for genetic selection. The main environmental factor was the age of the sheep at shearing. In summary, age related processes seem to drive the number of dark spots in this breed of sheep. © 2008 INRA EDP Sciences.","Hepatitis A virus (HAV) is a hepatotropic member of the family Picornaviridae. Previous studies suggested that HAV may evolve more slowly than other members of the family. To estimate HAV substitution rates precisely, we used a Bayesian Markov chain Monte Carlo (MCMC) approach on temporally sampled HAV VP1 full-length sequences from strains isolated in France. A mean rate of evolutionary change of 9.76×10-4 nucleotide substitution per site per year was found. The results also revealed that the synonymous rate found for HAV is lower than that of other members of the family. Bayesian skyline plots revealed a sharp decline in the effective number of infections in 1996, coinciding with the introduction of HAV vaccine. © 2007 SGM.","Data from 2032 Uruguayan Aberdeen Angus cows under extensive management and recording practices were analysed with Bayesian threshold-liability sire models, to assess genetic variability in calving success (CS), defined as a different binary trait for each of the second (CS2), third (CS3) and fourth (CS4) calving opportunities. Sire (herd) variances ranged from 0.08 to 0.11 (0.10 to 0.20) and heritability from 0.27 to 0.35, with large credibility intervals. Correlations between herd effects on CS at different calving opportunities were positive. Genetic correlation between CS2 and CS4 was positive (0.68), whereas those involving adjacent calving opportunities (CS2-CS3 and CS3-CS4) were negative, at -0.39 and -0.54, respectively. The residual correlation CS2-CS3 was negative (-0.32). The extent of uncertainty associated with the posterior estimates of the parameters was further evaluated through simulation, assuming different true values (-0.4, -0.2, +0.2 and +0.4) for the genetic correlations and changes in the degree of belief parameters of the inverse Wishart priors for the sire covariance matrix. Although inferences were not sharp enough, CS appears to be moderately heritable. The quality of data recording should be improved, in order to effect genetic improvement in female fertility. © 2007 The Animal Consortium.","Selection criteria associated to sexual precocity have increased their importance and in order to reduce the age at first calving they have been including in selection programs with positive consequences to the profit of the herds. The present study estimated the heritability for probability of early pregnancy (PPP), a trait associated with puberty, that could be considered as indicative of the beginning of the reproductive activity in females, using records on 5273 females. In order to verify the significance of the effects of milk yield in first lactation, age at calving of the dam and weaning age of the heifer on PPP logistic regression was applied. Heritability for PPP was estimated by Bayesian method, using a sire model. This model included the effects of contemporary group, milk yield in first lactation (in classes), sire and residual. The \"posteriori\" mean of heritability was 0.13. This value, although not high, showed the presence of genetic additive action on the determination of this trait.","Phylogeographical partitioning of Sin Nombre and Monongahela viruses (hantaviruses) may reflect that of their primary rodent host, the deer mouse (Peromyscus maniculatus). Lack of a comprehensive assessment of phylogeographical variation of the host has precluded the possibility of predicting spatial limits of existing strains of these viruses or geographical regions where novel viral strains might emerge. The complete cytochrome b gene was sequenced for 206 deer mice collected from sites throughout North America to provide a foundation for future studies of spatial structure and evolution of this ubiquitous host. Bayesian analyses of these sequences partitioned deer mice into six largely allopatric lineages, some of which may represent unrecognized species. The geographical distributions of these lineages were probably shaped by Quaternary climatic events. Populations of mice were apparently restricted to refugia during glacial advances, where they experienced genetic divergence. Expansion of these populations, following climatic amelioration, brought genetically distinctive forms into contact. Occurrence of parallel changes in virus strains can now be explored in appropriate regions. In New Mexico, for example, near the location where Sin Nombre virus was first discovered, there are three genetically distinctive lineages of deer mice whose geographical ranges need to be delineated precisely. The phylogeography of P. maniculatus provides a framework for interpreting geographical variability, not only in hosts, but also in associated viral variants and disease transmission, and an opportunity to predict the potential geographical distribution of newly emerging viral strains. © 2006 SGM.","Molar content of guanine plus cytosine (G + C) and optimal growth temperature (OGT) are main factors characterizing the frequency distribution of amino acids in prokaryotes. Previous work, using multivariate exploratory methods, has emphasized ascertainment of biological factors underlying variability between genomes, but the strength of each identified factor on amino acid content has not been quantified. We combine the flexibility of the phylogenetic mixed model (PMM) with the power of Bayesian inference via Markov Chain Monte Carlo (MCMC) methods, to obtain a novel evolutionary picture of amino acid usage in prokaryotic genomes. We implement a Bayesian PMM which incorporates the feature that evolutionary history makes observed data interdependent. As in previous studies with PMM, we present a variance partition; however, attention is also given to the posterior distribution of \"systematic effects\" that may shed light about the relative importance of and relationships between evolutionary forces acting at the genomic level. In particular, we analyzed influences of G + C, OGT, and respiratory metabolism. Estimates of G + C effects were significant for amino acids coded by G + C or molar content of adenine plus thymine (A + T) in first and second bases. OGT had an important effect on 12 amino acids, probably reflecting complex patterns of protein modifications, to cope with varying environments. The effect of respiratory metabolism was less clear, probably due to the already reported association of G + C with aerobic metabolism. A \"heritability\" parameter was always high and significant, reinforcing the importance of accommodating phylogenetic relationships in these analyses. \" Heritable\" component correlations displayed a pattern that tended to cluster \"pure\" G + C (A + T) in first and second codon positions, suggesting an inherited departure from linear regression on G + C. © The Author 2005. Published by Oxford University Press on behalf of the Society for Molecular Biology and Evolution. All rights reserved.","Three models for genetic evaluation of milk yield of Uruguayan Holstein cattle were compared using 159 169 lactation records from 81 928 cows calving between 1989 and 1998. Model I included the effects of herd-year-season, parity by age group, additive genetic merit, permanent environment, and residual. Model II included all effects in Model I, as well as number of days open and length of the dry period. Model III included all factors in Model II, and it accommodated heterogeneity of variance within contemporary groups (CG) through a pre-adjustment of the data based on empirical Bayes estimates of the CG variance. Estimates of heritability for milk yield were 0.23, 0.24 and 0.25, and estimates of repeatability were 0.55, 0.56 and 0.57 for Models I, II and III, respectively. Models were contrasted by examining changes in sire ranking and by a cross-validation procedure, based on the ability of the models to predict first, second and later lactations. Data were divided into two subsets, and records from one subset were predicted using location parameters estimated from the other subset. A resampling procedure was used to minimise the dependency on the sample structure. Correspondence between observed and predicted values was assessed in terms of square root of empirical mean square errors of prediction, percentage squared bias and the coefficient of determination 'R2'. Adjustment for heterogeneous CG variance had a marked effect on rankings of animals, especially elite cows, where correlations between solutions from Models I and II versus Model III ranged from 0.53 to 0.80. The percentage of animals selected in common by each pair of models decreased when selection intensity increased. Cross-validation analyses suggested that an assumption of heterogeneity of CG variance is tenable, especially in later lactations, whereas some doubts arise in first lactations, most probably due to the data structure used in the analyses. © 2003 Elsevier B.V. All rights reserved.","The extent and amount of heterogeneous phenotypic variance for milk yield in the Uruguayan Holstein population were evaluated and a simple method of accounting for heterogeneity was developed. Lactation records (159 169) collected between 1989 and 1998 by two recording schemes were used to form 8955 herd-year-season-parity-lactation length contemporary groups. A log-linear model was used to identify factors affecting heterogeneity of phenotypic variance. The model included effects of production level, contemporary group size, recording scheme, herd, season of calving, parity number, calving year period and length of lactation and accounted for 50% of the variation in log variances. Estimates from this model were used in a Bayesian manner, to obtain posterior mean estimates of within-contemporary-group variances, which were then used to standardize records to a baseline variance. Effects of the adjustment were assessed by comparing coefficients of variation before and after correction, by correlation and regression between mean and standard deviations, and by using Gini coefficients and Lorenz curves. The adjustment procedure reduced heteroscedasticity primarily by decreasing the frequency of low-variance contemporary groups. Lorenz curves and Gini coefficients indicated that the largest impact of the standardization procedure was related to the size of the contemporary group. Some differences in the effect of the correction were found between recording schemes. The method for adjusting records is simple and easy to adapt to current genetic evaluation procedures. © 2001 British Society of Animal Science.","Globalization of dairy cattle breeding has created a need for international sire proofs. Some early methods for converting proofs from one population to another are based on simple linear regression. An alternative robust regression method based on the t-distribution is presented, and maximum likelihood and Bayesian techniques for analysis are described, including the situation in which some proofs are missing. Procedures were used to investigate the relationship between Holstein sire proofs obtained by two Uruguayan genetic evaluation programs. The results suggest that conversion equations developed from data including only sires having proofs in both populations can lead to distorted results, relative to estimates obtained using techniques for incomplete data. There was evidence of non-normality of regression residuals, which constitutes an additional source of bias. A robust estimator may not solve all problems, but can provide simple conversion equations that are less sensitive to outlying proofs and to departures from assumptions. © 2001 Taylor & Francis.","A bioequivalence study between two controlled-release forms of theophylline was performed, taking into account its non linear elimination kinetics. Three hundred mg of drug were given to twelve human volunteers with a randomized and crossover design. The elimination parameters were estimated from the oral administration of a solution of Theophylline. Area under the curve (AUC), experimental peak concentration (Cmax), bioavailable dose (FD/Vd) and half value duration (HVD) were used as bioavailability parameters. Analysis of variance, bayesian approach for confidence intervals and bioequivalence two-one sided test, were performed."],"palabras":[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"COVID-19; Critical disease; Meta-analysis; SARS-CoV-2; Serology; Severity","Canine morbillivirus (CDV); Hemagglutinin, South America; Phylogenetic","Genetic correlations; Heritability; Wool quality traits; Wool yellowing","diagnosis; ELISA; gold standard; immunoassays; immunohistochemistry; latent class analysis; prion; real-time quaking-induced conversion; sensitivity; specificity","approximate Bayesian computation; Bokermannohyla saxicola; campo rupestre; ecological niche modelling; Espinhaço Range; flickering connectivity system; OCBIL; species delimitation","Beef cattle; Campylobacter fetus subspecies venerealis; Field samples; Latent class model","Lycosid; Sexual niche; Species delimitation; Species tree; Web spider","Comparative genomic analysis; Serotype 1; Streptococcus pneumoniae; Whole genome sequencing","Coalescent; Muroidea; Oryzomyalia; phylogenomics; polytomy; Rodentia; Sigmodontalia; species tree; UCEs","Anaplasmataceae; Ehrlichia; Gray brocket deer; Haemaphysalis juxtakochi; Molecular characterization; Rickettsiales; Ticks; Uruguay","Cerdocyon thous; Hepatozoon; Lycalopex gymnocercus; Molecular detection; Uruguay","exon-capture; last glacial maximum; olive mouse; patagonia; population genetics; quaternary colonization","Automatic calibration; Bayesian inference; DREAM(ZS); Markov chain Monte Carlo (MCMC); Models uncertainties; Spectral error; SWAN wave Model; Wave hindcast downscaling; Wave systems","Bioaccumulation; Isopoda; Microplastic deposition; Microplastic ingestion; Morphodynamics; Polychaeta; Sandy beach granulometry","B.1.1.28; Phylogenetics; Phylogeography; Q675H; Q677H; SARS-CoV-2; Spike; Uruguay","ABC; Climate change; Diversification; Elevation shifts; Statistical phylogeography; ‘Campo rupestre’","beef-cattle; campos grasslands; mixed-grazing; rangelands; sustainable intensification; wool","ecología molecular; fragmentación del paisaje; félidos neotropicales; genética de poblaciones; landscape fragmentation; microsatellite loci; microsatellites; molecular ecology; Neotropical felids; population genetics","Data scarcity; Hydroinformatics; Machine learning; Missing data; Multivariate imputation; Univariate imputation; Water quality","Early warning indicators, Species loss; Ecosystem collapse; Maximum entropy method; Tropical forest dynamics","Agility; Bony labyrinth; Cochlea; Morphometry; Semicircular canals","Prediction of changes in species abundances; Quantitative community ecology; Time series forecasting; Tropical forest dynamics","business ecosystem; community detection; ecological networks; MaxEnt; modularity","Allochthonous resources; reservoir food webs; stable isotopes; terrestrial subsidies","Hemoglobin; Hemoglobin beta subunit; Hypoxia; Molecular adaptation; Underground","Diurnal Temperature Variation; Heat Stress; Random Regression; Temperature-humidity Index","Artificial Intelligence; Bayesian Networks; Clinical Decision Support Systems; COVID-19; respiratory diseases","Anaplasmataceae; Birds; Ehrlichia; Ixodes auritulus; Novel genotypes; Phylogeny","","Animal resilience; Genotype-by-environment interaction; Longitudinal data","Anaplasma marginale; Babesia spp; cattle; multiplex PCR; sensitivity; specificity; tick-borne diseases","Faecal egg count; Genetic correlation; Haemonchus contortus; Heritability","coalescent; coronavirus; evolution; SARS-CoV-2","beef cattle; energy requirements; milk production; native pastures; Wood model","codon usage; Ebolavirus; evolution; GP protein","Abiotic stress; Heritability GWAS; Stress tolerance","Genome-wide association; Genomic prediction; Genomic selection; Single-step genomic BLUP","","Amphibia; Campo rupestre; Endemism; Genetic diversity; Molecular systematics; Species tree","Alert levels; Bayesian modeling; Brackish ecosystems; HAB management; Risk thresholds; Salinity","Allelic frequencies; Bulk sampling; Unlinked loci","Animal Movement; Approximate Bayesian Computation; Movement Ecology; Observation Time-Scale; Random walk; Simulated Trajectories","automatic fraud detection; Economic return; electricity theft; example-cost-sensitive; non-technical losses","Bayesian regression; Bovine Tuberculosis; diagnostic interaction; Johne's disease; Spain","Caatinga; Entomobryomorpha; genome; integrative taxonomy; mitochondrial; phylogeny","","Forging ecology; Global change; Human impacts; Seabirds; Stable isotopes; Waste management","effective population size; habitat suitability; Leopardus; Maxent; protected areas","Distribution range; Genetic variation; New record species; Phylogenetic diversity","Bayes factor species delimitation; ddRADseq; SNPs; Species tree","chronically infected; diagnosis; elisa; interferon-gamma release assay; latent class analysis; Uruguay","","Antarctic microbial mats; Antibiotic resistance coding genes; Beta-lactamases; Metagenomes","Bayesian Belief Network; Cattle tick elimination; Rhipicephalus microplus; Risk assessment","Disparity; diversity; Folivora; historical biogeography; morphological clock; phylogeny; sloth; tip-dating","Atlantic Forest; Bayesian species delimitation; Campos rupestres; Conservation; Pithecopus megacephalus; Pithecopus rohdei","Active monitoring; HDP-HMM; Hidden Markov models; Markov chains; Markov decision processes; RIPE Atlas; Round trip times; Routing overlays; Sparse monitoring","Apple bitter rot; Colletotrichum fructicola; Glomerella leaf spot; Pathogenicity","Amazon River plume; Conservation Genetics; Guianas coastline; Hybrid zone; Introgression; Trichechus","Allele-specific expression; GPU; Hierarchical model; Markov chain Monte Carlo; RNA-seq; Shrinkage priors","Otaria flavescens; Pontoporia blainvillei; SIBER analysis; stable isotopes","Chaqueño Phytogeographical Domain; Demographic history; Genetic diversity; Genetic patterns; Phylogeography; RASP","Bayesian analyses; Genetic variability; Viral evolution; Zika","","","Evolution; Human polyomavirus 6; Merkel Cell polyomavirus; phylodynamic analysis, South America","Action pictures; Cerebellar atrophy; Embodied cognition; Neurodegeneration; Object pictures","","Complex traits; Genome-enabled prediction; Genomic selection; Prediction; Quantitative genetics","Human bocavirus; Molecular characterization; qPCR; Sewage; Surface water; Uruguay","French Guiana; HIV-1; Non-pandemic; Pandemic; Phylodynamics; Subtype B; Suriname","Biogeography; Extratropical cyclone; Kelp rafts; Marine migration; Niche model; Upwelling","Bayesian estimation; Inverse problems in Imaging; Joint Denoising and Decompression; Patch-based non-local methods; Remote Sensing; Satellite Imaging","environmental predictors; jaguarundi; margay; Maxent; niche modeling","Diagnostic sensitivity; Diagnostic specificity; Diagnostic test; Echinococcus multilocularis; Foxes","Gene flow; Historical isolation; Long-distance dispersal; Microsatellite; Plastid RuBisCo spacer; Population genetic diversity; Sargassum thunbergii","Bayesian; coalescent; evolution; Zika","BEAST; BioGeoBEARS; Biogeography; Disjunct; Jump dispersal; Lamiaceae; Mint family","","Evolution; Infectivity; NS1; Zika virus","approximate Bayesian computation; campos rupestres; conservation genetics; ecological niche modeling; niche overlap; statistical phylogeography","Caviomorphs; Fossoriality; Hemoglobine; Molecular evolution","Amphibians; Atlantic forest; Ectoparasites; Ixodida; New species","Akodontini; Altitudinal gradient; Cytochrome b; Dispersal; Evolution","Bayesian inference; cardiovascular dynamics; ensemble Kalman filter (EnKF); fluid mechanics; inverse problems; uncertainty quantification","Apex predators; Arctocephalus australis; Estuary; Isotopic niche; Otaria flavescens; Pontoporia blainvillei; Stable isotope; Trophic overlap","Africa; CRF02_AG; HIV-1; Phylodynamics; Worldwide","Asterionellopsis guyunusae; Diatoms; Food webs; sandy beaches; Stable isotopes; Uruguay","Bayesian inference; Milk drying process; Sensitivity analysis; Uncertainty analysis","bayesian; coalescent; evolution; microcephaly; Zika","Evolutionary history; Hepatitis E virus genotype 3 phylodynamics; South America","BayesB; BayesC; Genome-wide association; SNP window; WssGBLUP","Anurans; Conservation; Niche modeling; Red lists","Classification; Gene sequence classification; Hidden Markov model; Trypanosoma brucei; Variant surface glycoprotein","","Connectivity; Fish; SIAR; Stable isotopes; Stomach contents","Animal selection; Backfat thickness; Cattle; Composite breed; Rib eye area","Climatic oscillations; glacial refugia; mitochondrial genes; mountain species; nuclear intron","Bayesian inference; Creole livestock; Endangered breeds; Genetic resources; Microsatellites","","","Genetic correlations; Heritability; Medullated fibers; Pigmented fibers; Response to selection; Wool","Bayesian inference; Markov chain Monte Carlo; Wave modelling; Wave reanalysis downscaling","Corrientes group; Ctenomys pearsoni; Mitochondrial phylogeny","","Genetic differentiation; Geological evolution; Pampas; Phylogeography; Scapteromys tumidus","BIOCLIM; Bioclimatic profile; MAXENT; Most limiting factor; Species distribution models","Allium cepa; farmer practices; genetic structure; ISSR; landraces; seed exchange","CDV; Hemagglutinin; Paramyxovirus; Phylogeographic analysis","Aberdeen Angus; Beef cattle; Growth models; Mature cow weight","","","Citrus tristeza virus; Novel genetic lineage; Phylogeny; Recombination; RNA silencing suppressor genes; RT-PCR; Uruguay","Abundance; Arctocephalus australis; Distribution; Genetic structure; Population trend; Seasonal movements; South American fur seal; Southwestern South Atlantic","Beef cattle; Genomic selection; Health; Tick resistance","Bone; Otaria flavescens; Sexual foraging segregation; Skin; Stable isotope Bayesian ellipses","","emotional reactivity; Gibb's sampling; sheep temperament","Age and growth; Bayesian approach; Isurus oxyrinchus; Schnute growth model","Demographic history; Evolutionary history; HIV-1; Latin america; Subtype B","28S rRNA; alien species; arthropod conservation; Bayesian skyline plots; cave life; cytochrome c oxidase (COI); molecular phylogenetics; phylogeography","Anastraphia; Asteraceae; Biogeography; Character evolution; Gochnatia; Gochnatieae; Molecular phylogenetics; Moquiniastrum; Neotropics; Pentaphorus; Richterago; Systematics; Taxon age","","","","Few genotyped animals; Genomic selection; Multitrait model; Single-step method","Climate change; Energetics; Environmental temperature; Global ecology; Macrophysiology; Small mammals","CAPS; Diversification; Floral morphology; Grasslands; Petunia axillaris; Plastid markers","Effective population size; Genetic diversity; Genetic population structure; Kajikia albida; Roundscale spearfish; Tetrapturus georgii; White marlin","Cochliomyia hominivorax; Microsatellites; MtDNA; Obligate myiasis","Cochliomyia hominivorax; Ecological niche Modelling; GIS; Myiasis; Pest control; Phylogeography","","BLUP; Genetic evaluation; Genomic evaluation; Marker genotypes; Relationship","","BIOCLIM; Bioclimatic profile; Disjunction; MAXENT; Neotropical Region; Potential distribution modeling","","","(Co)variance components; FEC; Growth traits; Maternal effects; Wool traits","","Bayesian phylogeography; demographic expansion; dispersal; Liolaemus; Monte Desert; range expansion","Bayesian analysis; Genotype B2; Genotype C1; Hepatitis B; Panama","","","Cricetidae; Diversification; Great American Biotic Interchange; Muroidea; Relaxed molecular clock; South America","Bayesian coalescent analysis; Evolution; Influenza A virus; Pandemic; Vaccine","","Divergence; Gene flow; Gene trees; Simulation; Speciation","Annual killifishes; Austrolebias; Dispersal; Past fragmentation; Phylogeography; Rivulidae; South America; Vicariance","FEC; Gastrointestinal parasites; Genetic correlation; Genetic progress; Heritability; Periparturient rise","cotton; cytochrome c oxidase I; demographic expansion; insect resistance management; NADH subunit 6; soybean","","Biogeography; Bulb; Heterostyly; NrITS; Oxalis; Psbj-peta; Tristyly; TrnL-trnL-trnF; Trnt-trnL","Astereae; Chiliotrichum group; Compositae; Molecular systematics; South America","Circus pygargus; Ecological niche modelling; Land uses; MaxEnt; Raptors; Sahel","Fine-scale population structure; Local adaptation; Matrilines; Social structure","","","Caviomorpha; MtDNA; Positive selection; Purifying selection; Subterranean niche","Better than average; Experimental economics; Irrationality; Overconfidence; Signalling models","Acute Bacterial Meningitis; Case Based Reasoning; Clinical Decision Support Systems; Diagnosis support; Expert systems","Ctenomys; cytochrome b; molecular dating; species groups; subterranean rodents; systematics; tuco-tucos","Chicken; Genetic evaluation; Genomic prediction","Artificial intelligence; Case based reasoning; Clinical decision support systems; Clinical diagnose; Expert systems; Intelligent systems","Genetic parameter; Novel trait; Somatic cell count","","Bayesian mixing models; Diet; Feeding strategy; Lactating females; Otaria flavescens; South American sea lion; Stable isotopes; Suckling pup","","","","","Apv; Bayesian model; Ort","Bayesian model; Poultry-health prevalence study; Swollen head syndrome","Bayesian inference; Pooling; Rogan-gladen estimator","","Bayesian inference; Evolution; Norovirus; VP1","","Bayesian inference; Dengue virus; Ecuador; Evolution","Genotyping; Lifetime net merit; Predicted transmitting ability; Single nucleotide polymorphism","","","Cytochrome b gene; Population structure","Bayesian hierarchical model; Posterior predictive ability; Sheep; Spot; Zero-inflated Poisson","","Bayesian theory; Beef cows; Calving success; Genetic parameters; Threshold models","Bayesian statistic; Caracu cattle; Probability of precocious pregnancy","","Amino acid usage; Bayesian methods; GC content; Genome evolution; Linear models; MCMC; Optimal growth temperature","Dairy cattle; Genetic evaluation; Heterogeneous variance; Milk yield; Predictive ability","Gini coefficient; Heterogeneity; Holstein (Uruguayan); Milk yield; Variance","Genetic Evaluation; Holstein; Robust Regression; Uruguay","bioequivalence; non linear kinetics; theophylline"],"Año":[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,2022,2022,2022,2022,2022,2022,2022,2021,2021,2021,2021,2021,2021,2021,2021,2021,2021,2021,2021,2021,2021,2021,2021,2021,2021,2021,2021,2021,2021,2021,2021,2020,2020,2020,2020,2020,2020,2020,2020,2020,2020,2020,2020,2019,2019,2019,2019,2019,2019,2019,2019,2019,2019,2019,2019,2019,2019,2019,2019,2019,2019,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2018,2017,2017,2017,2017,2017,2017,2017,2017,2017,2017,2017,2017,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2016,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2015,2014,2014,2014,2014,2014,2014,2014,2014,2014,2014,2014,2014,2013,2013,2013,2013,2013,2013,2013,2013,2013,2013,2013,2012,2012,2012,2012,2012,2012,2012,2012,2012,2012,2012,2011,2011,2011,2011,2011,2011,2011,2010,2010,2010,2010,2010,2010,2010,2010,2010,2010,2010,2009,2009,2009,2009,2009,2008,2008,2008,2008,2007,2007,2007,2006,2006,2003,2001,2001,1994],"documento":[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"['Article• Gold Open Access• Green Open Access']","['Article']","['Article']","['Article• Gold Open Access• Green Open Access']","['Article']","['Article• Bronze Open Access• Green Open Access']","['Article']","['Article• Gold Open Access• Green Open Access']","['Article']","['Article• Gold Open Access• Green Open Access']","['Article']","['Article']","['Article']","['Article• Hybrid Gold Open Access• Green Open Access']","['Article• Gold Open Access• Green Open Access']","['Article• Bronze Open Access']","['Article• Gold Open Access']","['Article']","['Article• Gold Open Access• Green Open Access']","['Article• Gold Open Access• Green Open Access']","['Article']","['Article']","['Article• Hybrid Gold Open Access• Green Open Access']","['Article']","['Article']","['Article• Gold Open Access']","['Conference Paper']","['Article• Gold Open Access• Green Open Access']","['Article• Gold Open Access• Green Open Access']","['Article• Gold Open Access• Green Open Access']","['Article• Green Open Access']","['Article']","['Article• Green Open Access']","['Article• Hybrid Gold Open Access']","['Article']","['Article• Gold Open Access• Green Open Access']","['Article• Gold Open Access• Green Open Access']","['Article']","['Article• Bronze Open Access']","['Article']","['Article• Gold Open Access']","['Article• Gold Open Access• Green Open Access']","['Article']","['Article']","['Article']","['Article• Green Open Access']","['Article']","['Article• Green Open Access']","['Article• Bronze Open Access']","['Article• Green Open Access']","['Article• Gold Open Access• Green Open Access']","['Article• Gold Open Access• Green Open Access']","['Article• Green Open Access']","['Article']","['Article']","['Article• Bronze Open Access']","['Article• Green Open Access']","['Conference Paper']","['Article• Gold Open Access• Green Open Access']","['Article• Gold Open Access• Green Open Access']","['Article']","['Article']","['Article• Gold Open Access• Green Open Access']","['Article• Gold Open Access• Green Open Access']","['Article• Gold Open Access• Green Open Access']","['Article']","['Article• Gold Open Access• Green Open Access']","['Article']","['Article• Gold Open Access• Green Open Access']","['Article']","['Article• Gold Open Access• Green Open Access']","['Article• Gold Open Access• Green Open Access']","['Conference Paper• Green Open Access']","['Article• Bronze Open Access']","['Article• Gold Open Access• Green Open Access']","['Article• Gold Open Access• Green Open Access']","['Article']","['Article• Bronze Open Access']","['Article• Gold Open Access• Green Open Access']","['Article• Gold Open Access• Green Open Access']","['Article• Gold Open Access• Green Open Access']","['Article']","['Article']","['Article']","['Article• Bronze Open Access• Green Open Access']","['Article']","['Article• Green Open Access']","['Article']","['Article']","['Article']","['Article']","['Article• Gold Open Access• Green Open Access']","['Article']","['Article• Green Open Access']","['Article• Bronze Open Access']","['Article']","['Article• Bronze Open Access']","['Article']","['Article']","['Article• Bronze Open Access• Green Open Access']","['Article• Gold Open Access• Green Open Access']","['Article']","['Conference Paper']","['Article']","['Article• Gold Open Access• Green Open Access']","['Article']","['Article• Gold Open Access• Green Open Access']","['Article']","['Article']","['Article']","['Article']","['Article• Gold Open Access• Green Open Access']","['Article• Gold Open Access• Green Open Access']","['Article']","['Article']","['Article• Bronze Open Access• Green Open Access']","['Article• Green Open Access']","['Article• Hybrid Gold Open Access']","['Article']","['Article']","['Article']","['Article']","['Article• Gold Open Access• Green Open Access']","['Article• Gold Open Access• Green Open Access']","['Article• Gold Open Access• Green Open Access']","['Article• Hybrid Gold Open Access']","['Article']","['Article']","['Article• Gold Open Access• Green Open Access']","['Article']","['Article']","['Article']","['Article']","['Article• Gold Open Access• Green Open Access']","['Article']","['Article• Gold Open Access• Green Open Access']","['Article• Gold Open Access• Green Open Access']","['Article']","['Article• Gold Open Access• Green Open Access']","['Article• Green Open Access']","['Article']","['Article']","['Article• Gold Open Access• Green Open Access']","['Article']","['Article']","['Article• Green Open Access']","['Article• Bronze Open Access• Green Open Access']","['Article']","['Article']","['Article']","['Article']","['Article• Bronze Open Access']","['Article']","['Article• Green Open Access']","['Article• Bronze Open Access• Green Open Access']","['Book Chapter']","['Book Chapter']","['Article']","['Article• Green Open Access']","['Article']","['Article• Bronze Open Access']","['Article']","['Conference Paper']","['Article• Bronze Open Access']","['Article• Gold Open Access• Green Open Access']","['Article• Gold Open Access• Green Open Access']","['Article• Gold Open Access• Green Open Access']","['Article• Gold Open Access• Green Open Access']","['Article• Gold Open Access• Green Open Access']","['Article• Bronze Open Access• Green Open Access']","['Article• Green Open Access']","['Article• Green Open Access']","['Article• Green Open Access']","['Article• Gold Open Access• Green Open Access']","['Article• Bronze Open Access']","['Article• Green Open Access']","['Article']","['Article• Bronze Open Access']","['Book Chapter']","['Article']","['Article• Gold Open Access• Green Open Access']","['Article• Bronze Open Access• Green Open Access']","['Article• Bronze Open Access']","['Article• Bronze Open Access• Green Open Access']","['Article• Gold Open Access• Green Open Access']","['Article• Bronze Open Access']","['Article• Green Open Access']","['Article']","['Article']","['Article']","['Article']"],"fuente":[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Conference Proceedings']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Book Series']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Conference Proceedings']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Conference Proceedings']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Book']","['Book']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Conference Proceedings']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Book']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']","['Journal']"],"revista":[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"BMC Infectious Diseases","Virus Research","Small Ruminant Research","Pathogens","Journal of Biogeography","Tropical Animal Health and Production","Organisms Diversity and Evolution","BMC Genomics","Systematic Biology","Microorganisms","Parasitology Research","Journal of Biogeography","Coastal Engineering","Environmental Pollution","Viruses","Molecular Phylogenetics and Evolution","Frontiers in Sustainable Food Systems","Journal of Mammalogy","Sustainability (Switzerland)","Ecological Indicators","Journal of South American Earth Sciences","Ecological Modelling","Journal of Complex Networks","Ecology of Freshwater Fish","Gene","Animal Bioscience","2021 IEEE URUCON, URUCON 2021","Current Research in Parasitology and Vector-Borne Diseases","BioMed Research International","Animals","Journal of Veterinary Diagnostic Investigation","Veterinary Parasitology","Journal of Medical Virology","Applied Animal Science","Journal of Medical Virology","G3: Genes, Genomes, Genetics","Genes","Archives of Virology","Molecular Phylogenetics and Evolution","Water Research","Crop Breeding and Applied Biotechnology","PeerJ","IEEE Transactions on Power Systems","Veterinary Record","Zoologica Scripta","International Economic Review","Science of the Total Environment","Studies on Neotropical Fauna and Environment","Algae","Molecular Phylogenetics and Evolution","Frontiers in Veterinary Science","Earth System Dynamics","Frontiers of Environmental Science and Engineering","Ticks and Tick-borne Diseases","Systematic Biology","Molecular Phylogenetics and Evolution","Journal of Network and Systems Management","Acta Horticulturae","Anais da Academia Brasileira de Ciencias","Mathematical Biosciences and Engineering","Wildlife Research","Botanical Journal of the Linnean Society","Viruses","Scientific Reports","Scientific Reports","Molecular Phylogenetics and Evolution","Frontiers in Psychology","Crop Science","Frontiers in Genetics","Food and Environmental Virology","Frontiers in Microbiology","Diversity","Proceedings - International Conference on Image Processing, ICIP","Journal of Mammalogy","Parasites and Vectors","BMC evolutionary biology","Journal of Medical Virology","American Journal of Botany","PLoS ONE","Memorias do Instituto Oswaldo Cruz","Ecology and Evolution","Gene","Ticks and Tick-borne Diseases","Zootaxa","Journal of Verification, Validation and Uncertainty Quantification","Journal of Experimental Marine Biology and Ecology","Infection, Genetics and Evolution","Estuarine, Coastal and Shelf Science","Chemical Engineering Science","Journal of Medical Virology","Infection, Genetics and Evolution","Frontiers in Genetics","Ecologia Austral","Pattern Analysis and Applications","Phytopathology","Estuaries and Coasts","Genetics and Molecular Research","Molecular Ecology","Journal of Animal Breeding and Genetics","Heredity","Genetics Selection Evolution","Journal of Animal Science","Proceedings of the Coastal Engineering Conference","Mastozoologia Neotropical","PLoS ONE","Mammalian Biology","Zoologia","Plant Genetic Resources: Characterisation and Utilisation","Molecular Phylogenetics and Evolution","Journal of Animal Science","Archives of Virology","PLoS ONE","Viruses","Marine Mammal Science","Journal of Animal Science","Marine Ecology Progress Series","Journal of the European Economic Association","Animal","Environmental Biology of Fishes","AIDS","Insect Conservation and Diversity","Taxon","PLoS ONE","PLoS ONE","PLoS ONE","Journal of Dairy Science","Evolutionary Ecology Research","Molecular Ecology","BMC Genetics","Acta Tropica","Acta Tropica","Zoologica Scripta","Livestock Science","PLoS ONE","Zootaxa","PLoS ONE","PLoS ONE","Livestock Science","PLoS ONE","Molecular Ecology","Memorias do Instituto Oswaldo Cruz","Microbiology (United Kingdom)","PLoS ONE","Molecular Phylogenetics and Evolution","Virus Research","Nucleic Acids Research","Evolution","Journal of Biogeography","Livestock Science","Bulletin of Entomological Research","Marine Biology","American Journal of Botany","Taxon","Biological Conservation","Journal of Heredity","Agricultural Production","Agricultural Production","Molecular Phylogenetics and Evolution","Econometrica","Expert Systems with Applications","Journal of Mammalogy","Journal of Animal Science","Innovations and Advances in Computer Sciences and Engineering","Journal of Dairy Science","Virology Journal","Scientia Marina","Retrovirology","BMC Genetics","PLoS ONE","Genetics Research","International Journal of Poultry Science","International Journal of Poultry Science","International Journal of Poultry Science","Virology Journal","Infection, Genetics and Evolution","Genetics","Virus Research","Journal of Dairy Science","New Research on Livestock Science and Dairy Farming","Marine Biology","Neotropical Ichthyology","Genetics Selection Evolution","Journal of General Virology","Animal","Ciencia Rural","Journal of General Virology","Molecular Biology and Evolution","Livestock Production Science","Animal Science","Acta Agriculturae Scandinavica A: Animal Sciences","Acta Farmaceutica Bonaerense"],"citas":[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,8,-1,0,-1,0,-1,0,0,4,4,0,3,0,7,-1,6,0,2,14,-1,2,2,-1,2,0,3,0,4,2,7,6,3,18,2,0,5,42,4,14,13,0,3,33,8,11,4,10,6,4,15,5,5,2,5,35,15,12,-1,9,-1,3,14,7,7,9,9,11,3,13,8,9,21,-1,21,10,13,4,18,8,14,12,10,41,12,12,13,18,12,11,8,12,86,4,5,24,14,2,26,18,13,16,8,0,7,14,5,7,7,45,17,6,3,11,40,39,19,44,9,8,10,5,32,23,54,31,48,16,44,4,9,8,15,186,28,7,46,19,14,21,25,4,6,15,94,3,68,151,28,22,22,34,21,8,32,35,0,0,64,111,29,74,86,-1,25,17,27,38,32,25,18,-1,4,7,10,27,397,6,97,3,11,14,24,31,2,-1,55,16,11,8,2,0],"paper_name":[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,"Age-specific rate of severe and critical SARS-CoV-2 infections estimated with multi-country seroprevalence studies","Origin and spreading of canine morbillivirus in South America","Wool scoured colour: Heritability, genetic and phenotypic correlations with wool traits in Corriedale sheep","Assessment of Real-Time Quaking-Induced Conversion (RT-QuIC) Assay, Immunohistochemistry and ELISA for Detection of Chronic Wasting Disease under Field Conditions in White-Tailed Deer: A Bayesian Approach","Colonization rather than fragmentation explains the geographical distribution and diversification of treefrogs endemic to Brazilian shield sky islands","Performance of bovine genital campylobacteriosis diagnostic tests in bulls from Uruguay: a Bayesian latent class model approach","Not the same: phylogenetic relationships and ecological niche comparisons between two different forms of Aglaoctenus lagotis from Argentina and Uruguay","Whole genomic comparative analysis of Streptococcus pneumoniae serotype 1 isolates causing invasive and non-invasive infections among children under 5 years in Casablanca, Morocco","Ultraconserved Elements Improve the Resolution of Difficult Nodes within the Rapid Radiation of Neotropical Sigmodontine Rodents (Cricetidae: Sigmodontinae)","Characterization of “candidatus ehrlichia pampeana” in haemaphysalis juxtakochi ticks and gray brocket deer (Mazama gouazoubira) from uruguay","An Hepatozoon americanum-like protozoan in crab-eating (Cerdocyon thous) and grey pampean (Lycalopex gymnocercus) foxes from Uruguay","Genomic footprints of Quaternary colonization and population expansion in the Patagonian-Fuegian region rules out a separate southern refugium in Tierra del Fuego","Automatic calibration and uncertainty quantification in waves dynamical downscaling","Sediment grain size determines microplastic exposure landscapes for sandy beach macroinfauna","Emergence and spread of a b.1.1.28-derived p.6 lineage with q675h and q677h spike mutations in uruguay","Quaternary climatic fluctuations influence the demographic history of two species of sky-island endemic amphibians in the Neotropics","Building the GLENCOE Platform -Grasslands LENding eConomic and ecOsystems sErvices","Population structure and gene flow of Geoffroy's cat (Leopardus geoffroyi) in the Uruguayan Savanna ecoregion","Water-quality data imputation with a high percentage of missing values: A machine learning approach","A new early warning indicator of tree species crashes from effective intraspecific interactions in tropical forests","The inner ear anatomy of glyptodonts and pampatheres (Xenarthra, Cingulata): Functional and phylogenetic implications","A method for predicting species trajectories tested with trees in barro colorado tropical forest","Markets as ecological networks: Inferring interactions and identifying communities","Riparian forest subsidises the biomass of fish in a recently formed subtropical reservoir","Adaptive evolution of β-globin gene in subterranean in South America octodontid rodents","Inclusion of bioclimatic variables in genetic evaluations of dairy cattle","Applying Bayesian Networks to help Physicians Diagnose Respiratory Diseases in the context of COVID-19 Pandemic","Molecular characterization of novel Ehrlichia genotypes in Ixodes auritulus from Uruguay","Development of a Population Pharmacokinetic Model for Cyclosporine from Therapeutic Drug Monitoring Data","Selection for test-day milk yield and thermotolerance in brazilian holstein cattle","Validation of a multiplex PCR assay to detect Babesia spp. and Anaplasma marginale in cattle in Uruguay in the absence of a gold standard test","Genetic parameters for production traits, resistance and resilience to Nematode parasites under different worm burden challenges in Corriedale sheep","Evidence of increasing diversification of emerging Severe Acute Respiratory Syndrome Coronavirus 2 strains","Is it possible to accurately estimate lactation curve parameters in extensive beef production systems?","An evolutionary insight into emerging Ebolavirus strains isolated in Africa","Low additive genetic variation in a trait under selection in domesticated rice","Single-step genomic evaluations from theory to practice: using snp chips and sequence data in blupf90","Historical origins and zoonotic potential of avian influenza virus H9N2 in Tunisia revealed by Bayesian analysis and molecular characterization","Multilocus phylogeny of Paratelmatobiinae (Anura: Leptodactylidae) reveals strong spatial structure and previously unknown diversity in the Atlantic Forest hotspot","Predicting cyanobacterial biovolume from water temperature and conductivity using a Bayesian compound Poisson-Gamma model","Gametes Simulator: A multilocus genotype simulator to analyze genetic structure in outbreeding diploid species","Using approximate Bayesian inference for a \"steps and turns\" continuous-time random walk observed at regular time intervals","Fraud Detection in Electric Power Distribution: An Approach That Maximizes the Economic Return","Association between results of diagnostic tests for bovine tuberculosis and Johne's disease in cattle","The discovery of Neotropical Lepidosira (Collembola, Entomobryidae) and its systematic position","APPARENT BIAS: WHAT DOES ATTITUDE POLARIZATION SHOW?","The impact of anthropogenic food subsidies on a generalist seabird during nestling growth","Conservation planning in Uruguay based on small felids (Leopardus spp.) as umbrella species","Species diversity and distribution of the genus Colpomenia (Scytosiphonaceae, Phaeophyceae) along the coast of China","Coalescent-based species delimitation in the sand lizards of the Liolaemus wiegmannii complex (Squamata: Liolaemidae)","Modeling the Accuracy of Two in-vitro Bovine Tuberculosis Tests Using a Bayesian Approach","Millennium-length precipitation reconstruction over south-eastern asia: A pseudo-proxy approach","Detection of presumed genes encoding beta-lactamases by sequence based screening of metagenomes derived from Antarctic microbial mats","Probability of Rhipicephalus microplus introduction into farms by cattle movement using a Bayesian Belief Network","Phylogeny, Macroevolutionary Trends and Historical Biogeography of Sloths: Insights From a Bayesian Morphological Clock Analysis","Cryptic diversity in Brazilian endemic monkey frogs (Hylidae, Phyllomedusinae, Pithecopus) revealed by multispecies coalescent and integrative approaches","Joint Minimization of Monitoring Cost and Delay in Overlay Networks: Optimal Policies with a Markovian Approach","Genetic and pathogenic diversity of Colletotrichum species associated with apple diseases in southern Brazil and Uruguay","A hybrid swarm of manatees along the guianas coastline, a peculiar environment under the influence of the Amazon River plume","Fully Bayesian analysis of allele-specific RNA-seq data","A historical perspective of niche differentiation between two top predators in the Uruguayan coastal area","Impact of Pleistocene geoclimatic events on the genetic structure in mid-latitude South American plants: Insights from the phylogeography of Turnera sidoides complex (Passifloraceae, Turneroideae)","An evolutionary insight into zika virus strains isolated in the Latin American region","Mouth gape determines the response of marine top predators to long-term fishery-induced changes in food web structure","Inferring population dynamics of HIV-1 subtype C epidemics in Eastern Africa and Southern Brazil applying different Bayesian phylodynamics approaches","Phylodynamics of Merkel-cell polyomavirus and human polyomavirus 6: A long-term history with humans","Action semantics at the bottom of the brain: Insights from dysplastic cerebellar gangliocytoma","Genetic structure, core collection, and regeneration quality in white dent corn landraces","Prediction of complex traits: Robust alternatives to best linear unbiased prediction","Human Bocavirus: Detection, Quantification and Molecular Characterization in Sewage and Surface Waters in Uruguay","The HIV-1 subtype B epidemic in French guiana and Suriname is driven by ongoing transmissions of pandemic and non-pandemic lineages","Kelps' long-distance dispersal: Role of Ecological/Oceanographic processes and implications to marine forest conservation","Joint denoising and decompression: A patch-based Bayesian approach","Geographic distribution modeling of the margay (Leopardus wiedii) and jaguarundi (Puma yagouaroundi): A comparative assessment","Latent class models for Echinococcus multilocularis diagnosis in foxes in Switzerland in the absence of a gold standard","Historical isolation and contemporary gene flow drive population diversity of the brown alga Sargassum thunbergii along the coast of China","Evidence of increasing diversification of Zika virus strains isolated in the American continent","Amphitropical disjunctions in new world menthinae: Three pliocene dispersals to South America following late miocene dispersal to North America from the old world","The disjunct pattern of the Neotropical harvestman Discocyrtus dilatatus (Gonyleptidae) explained by climate-driven range shifts in the Quaternary: Paleodistributional and molecular evidence","Tracing the origin of the NS1 A188V substitution responsible for recent enhancement of ZIKA virus asian genotype infectivity","Evolutionarily significant units of the critically endangered leaf frog Pithecopus ayeaye (Anura, Phyllomedusidae) are not effectively preserved by the Brazilian protected areas network","Molecular adaptive convergence in the α-globin in subterranean octodontid rodents","Description of a new soft tick species (Acari: Argasidae: Ornithodoros) associated with stream-breeding frogs (Anura: Cycloramphidae: Cycloramphus) in Brazil","A new species of Deltamys Thomas, 1917 (Rodentia: Cricetidae) endemic to the southern Brazilian Araucaria Forest and notes on the expanded phylogeographic scenario of D. kempi","Uncertainty Quantification in a Patient-Specific One-Dimensional Arterial Network Model: EnKF-Based Inflow Estimator","Trophic relationships in apex predators in an estuary system: A multiple-method approximation","Phylodynamics of the major HIV-1 CRF02_AG African lineages and its global dissemination","Trophic niche shifts driven by phytoplankton in sandy beach ecosystems","Modeling a production scale milk drying process: Parameter estimation, uncertainty and sensitivity analysis","Bayesian coalescent inference reveals high evolutionary rates and diversification of Zika virus populations","New insights into the hepatitis E virus genotype 3 phylodynamics and evolutionary history","Weighting strategies for single-step genomic BLUP: An iterative approach for accurate calculation of GEBV and GWAS","Threatened amphibians of Uruguay: Effectiveness of protected areas face of climate change","Hidden Markov models for gene sequence classification: Classifying the VSG gene in the Trypanosoma brucei genome","Genetic structure of Colletotrichum fructicola associated to apple bitter rot and Glomerella leaf spot in Southern Brazil and Uruguay","Trophic Dynamics of the Cape Stumpnose (Rhabdosargus holubi, Sparidae) Across Three Adjacent Aquatic Habitats","Genetic parameters for carcass traits and body weight using a Bayesian approach in the Canchim cattle","Imprints of multiple glacial refugia in the Pyrenees revealed by phylogeography and palaeodistribution modelling of an endemic spider","Genetic relationships among American donkey populations: Insights into the process of colonization","Genetic relationships between Atlantic and Pacific populations of the notothenioid fish Eleginops maclovinus: The footprints of Quaternary glaciations in Patagonia","Genotype by environment interaction for tick resistance of Hereford and Braford beef cattle using reaction norm models","Genetic parameters of objectionable fibers and of their associations with fleece traits in Corriedale sheep","Automatic calibration of a wave model with an evolutionary Bayesian method","Phylogenetic relationships between tuco-tucos (Ctenomys, Rodentia) of the Corrientes group and the C. pearsoni complex","Stable isotopes reveal long-Term fidelity to foraging grounds in the galapagos sea lion zalophus wollebaeki)","Genetic diversity of the swamp rat in South America: Population expansion after transgressive-regressive marine events in the Late Quaternary","More on the mesopotamian-yungas disjunction in subtropical and temperate argentina: Bioclimatic distribution models of the harvestman discocyrtus dilatatus (opiliones: Gonyleptidae)","Genetic diversification of local onion populations under different production systems in Uruguay","Molecular phylogeography of canine distemper virus: Geographic origin and global spreading","Description of the growth curve for Angus pasture-fed cows under extensive systems","An evolutionary insight into Newcastle disease viruses isolated in Antarctica","Phylogenetic analysis of Hepatitis B virus genotypes circulating in different risk groups of Panama, evidence of the introduction of genotype A2 in the country","Phylogenetic studies of the three rna silencing suppressor genes of south american ctv isolates reveal the circulation of a novel genetic lineage","Status, population trend and genetic structure of South American fur seals, Arctocephalus australis, in southwestern Atlantic waters","Genomic prediction for tick resistance in braford and hereford cattle","Sexual foraging segregation in South American sea lions increases during the pre-breeding period in the Río de la Plata plume","Does the better-than-average effect show that people are overconfident?: Two experiments","Temperament and its heritability in Corriedale and Merino lambs","Bayesian age and growth analysis of the shortfin mako shark Isurus oxyrinchus in the Western South Atlantic Ocean using a flexible model","Phylodynamics of major HIV-1 subtype B pandemic clades circuLating in Latin America","Endemism and evolution in the littoral woodlouse Halophiloscia Verhoeff, 1908 (Crustacea, Isopoda, Oniscidea) from the Canary Islands: implications for conservation policies","A phylogeny of the Gochnatieae: Understanding a critically placed tribe in the Compositae","Spatiotemporal dynamics of the HIV-1 subtype G epidemic in West and Central Africa","Climate change and the distribution of neotropical red-bellied toads (Melanophryniscus, Anura, Amphibia): How to prioritize species and populations?","Temporal dynamics of norovirus GII.4 variants in Brazil between 2004 and 2012","Methods for genomic evaluation of a relatively small genotyped dairy population and effect of genotyped cow information in multiparity analyses","Thermal tolerances in rodents: Species that evolved in cold climates exhibit a wider thermoneutral zone","Diversification in the south american pampas: The genetic and morphological variation of the widespread petunia axillaris complex (solanaceae)","Comparative population genetics and evolutionary history of two commonly misidentified billfishes of management and conservation concern","Genetic diversity and population structure of the New World screwworm fly from the Amazon region of Brazil","Applying spatial analysis of genetic and environmental data to predict connection corridors to the New World screwworm populations in South America","A multilocus perspective on the phylogenetic relationships and diversification of rodents of the tribe Abrotrichini (Cricetidae: Sigmodontinae)","Single Step, a general approach for genomic selection","Functional diversification within a predatory species","New records and distribution modeling of Gryne orensis (Sørensen) (Opiliones: Cosmetidae) support the Mesopotamian-Yungas disjunction in subtropical Argentina","Bias in diet determination: Incorporating traditional methods in Bayesian mixing models","The Phylogeographic History of the New World Screwworm Fly, Inferred by Approximate Bayesian Computation Analysis","Estimates of genetic parameters for worm resistance, wool and growth traits in Merino sheep of Uruguay","Tracing the Origin and Northward Dissemination Dynamics of HIV-1 Subtype C in Brazil","Quaternary range and demographic expansion of Liolaemus darwinii (Squamata: Liolaemidae) in the Monte Desert of Central Argentina using Bayesian phylogeography and ecological niche modelling","Molecular characterisation of hepatitis B virus in the resident Chinese population in Panama City","Evolution of optimal codon choices in the family Enterobacteriaceae","Fine-Scale Population Structure of Blue Whale Wintering Aggregations in the Gulf of California","Dating an impressive Neotropical radiation: Molecular time estimates for the Sigmodontinae (Rodentia) provide insights into its historical biogeography","Bayesian coalescent analysis of pandemic H1N1 influenza A virus circulating in the South American region","Exploring polymorphisms in B-DNA helical conformations","Species delimitation with abc and other coalescent-based methods: A test of accuracy with simulations and an empirical example with lizards of the liolaemus darwinii complex (Squamata: Liolaemidae)","Patterns of population differentiation in annual killifishes from the Paraná-Uruguay-La Plata Basin: The role of vicariance and dispersal","Genetic parameters for nematode resistance in periparturient ewes and post-weaning lambs in Uruguayan Merino sheep","Genetic variability and demographic history of Heliothis virescens (Lepidoptera: Noctuidae) populations from Brazil inferred by mtDNA sequences","Phylogeography of the sub-Antarctic notothenioid fish Eleginops maclovinus: Evidence of population expansion","Diversification of the American bulb-bearing Oxalis (Oxalidaceae): Dispersal to North America and modification of the tristylous breeding system","Phylogenetics of the chiliotrichum group (Compositae: Astereae): The story of the fascinating radiation in the paleate Astereae genera from southern South America","Protected areas do not fulfil the wintering habitat needs of the trans-Saharan migratory Montagu's harrier","Population genetic structure and social kin associations of franciscana dolphin, pontoporia blainvillei","Serological monitoring of selected avian pathogens of commercial poultry farms in Uruguay","Assessing the seroprevalence for abortigenic diseases of ovine herds based on diagnostic tests applied to individual-animals in Uruguay","The evolution of mitochondrial genomes in subterranean caviomorph rodents: Adaptation against a background of purifying selection","Apparent overconfidence","Comparing Bayesian inference and case-based reasoning as support techniques in the diagnosis of Acute Bacterial Meningitis","Species groups and the evolutionary diversification of tuco-tucos, genus Ctenomys (Rodentia: Ctenomyidae)","Genome-wide marker-assisted selection combining all pedigree phenotypic information with genotypic data in one step: An example using broiler chickens","Evaluation of case based reasoning for clinical decision support systems applied to acute meningitis diagnose","Phenotypic and genetic characterization of novel somatic cell count traits from weekly or monthly observations","Evolution of Dengue virus type 3 genotype III in Venezuela: Diversification, rates and population dynamics","Change in the foraging strategy of female South American sea lions (Carnivora: Pinnipedia) after parturition","Phylodynamics of HIV-1 Circulating Recombinant Forms 12_BF and 38_BF in Argentina and Uruguay","Dispersal and population structure at different spatial scales in the subterranean rodent Ctenomys australis","Genetic history of hepatitis C virus in Venezuela: High diversity and long time of evolution of HCV genotype 2","L2-Boosting algorithm applied to high-dimensional problems in genomic selection","Assessing the seroprevalence against avian pneumovirus and Ornithobacterium rhinotracheale in broilers in Uruguay","Flock-level seroprevalence against avian pneumovirus amongst uruguayan broiler chickens","Flock-level seroprevalence against Ornithobacterium rhinotracheale among broilers in Uruguay","Modeling gene sequences over time in 2009 H1N1 Influenza A Virus populations","Bayesian coalescent inference reveals high evolutionary rates and expansion of Norovirus populations","Predicting quantitative traits with regression models for dense molecular markers and pedigree","Modeling gene sequence changes over time in type 3 dengue viruses from Ecuador","Predictive ability of direct genomic values for lifetime net merit of Holstein sires using selected subsets of single nucleotide polymorphism markers","Monitoring the infectious disease status of dairy cattle herds in Uruguay: A random effects hierarchical model analysis","Phylogeography of the Southwestern Atlantic menhaden genus Brevoortia (Clupeidae, Alosinae)","Phylogeography of the Neotropical catfish Pimelodus albicans (Siluriformes: Pimelodidae) from río de la Plata basin, South America, and conservation remarks","A comparison between Poisson and zero-inflated Poisson regression models with an application to number of black spots in Corriedale sheep","Bayesian coalescent inference of hepatitis A virus populations: Evolutionary rates and patterns","Genetic variability in calving success in Aberdeen Angus cows under extensive recording","Genetic and environment factors on probability of early pregnancy of Caracu cattle","Phylogeography of the deer mouse (Peromyscus maniculatus) provides a predictive framework for research on hantaviruses","Inferring parameters shaping amino acid usage in prokaryotic genomes via Bayesian MCMC methods","Model comparison for genetic evaluation of milk yield in Uruguayan holsteins","Evaluation of extent and amount of heterogeneous variance for milk yield in Uruguayan Holsteins","Assessing relationships between genetic evaluations using robust regression with an application to holsteins in uruguay","Bioequivalence study of theophylline considering Michaelis-Menten kinetics"],"x":[-1,-0.364015190212571,-0.309282011806261,-0.35771449324717,-0.337764980784839,-0.358955254925635,-0.335714973587102,-0.274063983333706,-0.311378847439834,-0.274541930320281,-0.273313164690543,-0.183364488159146,-0.14107456960794,-0.145341865357172,-0.15810039432982,-0.201773483700306,-0.0333517043953641,0.0939992237675045,-0.918630679620809,0.181588117398441,0.183590199526892,0.109282811106933,-0.907607890788414,-0.416537422786278,0.186530420993051,0.182434652376062,0.203788655906823,0.255182295611373,0.258564508375005,0.210744797200192,0.17786263602419,-0.438809068770838,-0.389881022020713,0.739291176234216,0.739959678026259,-0.655705533199836,-0.587971569397855,-0.673650302159226,-0.0498953047968297,-0.0342875372400485,0.0408085186880027,0.0303635011239589,0.0374954627333364,0.0189389120208014,0.00363184002174877,0.606065000258965,0.310800267467057,0.228911378909704,0.267346110714004,0.224274859716487,0.274957378159098,0.809589820107933,0.78357579905903,0.742428019406586,0.717915270857352,0.727251723207179,0.801568220705098,0.764926160691469,0.847192384336148,-0.318546653209763,-0.325804016940907,-0.251263549213361,-0.169238017167667,-0.54728690920826,-0.476422348328599,0.507950112904263,0.419835844055203,0.435234321725504,0.601138921485062,0.554259315240683,0.634665919809116,0.547560748477672,0.589674015048634,0.344583191672267,0.305730997722832,0.344784897762793,0.344929641385771,0.599710469060364,-0.0183318519172307,-0.0898693494944965,0.947208191047205,1,0.927220679801959,-0.0534818632657575,-0.768313526268597,-0.981897957492411,-0.927783789079458,-1,-0.614885423116738,-0.698136354572625,-0.638566938313913,0.0171163439002546,-0.837261359241618,0.139030482152161,-0.852642208524603,-0.101767538173216,-0.102009623270795,-0.737939159137488,-0.0623780730668124,-0.0826722957238385,0.00661966520696078,0.0397340774393606,0.0403508854086514,-0.755892038862427,-0.181608527585831,0.420055576115378,0.421410089584839,-0.674522718483236,-0.625676561331556,-0.625873326828428,-0.775710653583163,-0.737800458832939,-0.828604831281051,-0.777874753547258,-0.022395498948239,-0.0145865985988425,-0.107735039612385,-0.928800696418573,-0.916681803217787,-0.976813169465921,-0.906302654608661,-0.944915011581384,-0.285607317841657,-0.0456595630354508,-0.593153398104045,-0.535908136476806,-0.481696274387988,-0.554001473597116,-0.511784077724854,-1,-0.0540922976645684,-0.40659345681486,-0.525647918128617,0.301662980777935,0.22039692711847,-1,-0.962840301747204,-0.430414757345663,-0.180118298178278,-0.117080134856221,-0.157342807235419,0.382998052415664,-0.349045436966472,-0.198027467425014,-0.198484460383282,0.61586622443032,0.00799906863512434,-0.0462543819163133,0.0767248675601202,-0.864915045576107,0.77193931795008,0.691397873195945,0.76069092263993,0.505018465297112,-0.204807997549805,-0.442784458532877,-0.229821861515712,-0.217170852610751,-0.45858724499834,-0.132119876443759,-0.114159666577967,-0.0949282717637436,-0.267676116587281,-0.500482130384214,-0.474833249311217,-0.434237443516211,-0.18630679620809,-0.474694256823184,0.892746117517315,0.88189188259188,0.888660454375335,0.878139248586905,-0.104937475828899,0.506876088288077,0.477204922096092,-0.0894871943916329,0.0210130999139502,0.0100493251914926,0.0172731684198761,0.667660995472069,0.58635316970052,-0.166877318552942,-0.959315339810405,-0.318093328808749,-0.172569490803067,-0.060953709785504,0.136336685657718,-0.877946019431214,0.154799680624439,-0.93184278294631,-0.421399302834578,0.225470690924579,0.216156007551033,-0.407715506273102,0.708345655661665,-0.629438740647344,-0.0112284946810688,0.112406077926596,0.618217579498464,0.26481521242199,0.763947522821224,0.811589858787963,-0.286425584731139,0.843247152251877,0.886536019612364,-0.210578889023766,-0.507870320984348,0.462094749774434,0.394073182866563,0.2189485891053,0.588876661516419,0.516710412206032,0.302921846757058,0.625578486120775,-0.0705773511828611,0.958231354431261,-0.0769640085534589,-0.78903550745052,0.518391821533947,-0.0962029663045079,0.11122471300507,-0.969960012319622,-0.65703308478544,0.0576642899929325,-0.796576699052022,-0.0610828779836207,0.179712266928874,-0.887099128889862,-0.125845770180426,-0.771960156969466,-0.865489204078217,-0.0409318131657243,0.0121717728684072,-0.715207378672831,-0.20972773972987,0.385140725578772,-0.29179520614353,0.137177975834594,-0.63383805829364,-0.656869163788849,0.0786427082023771,-0.0235681554497079,-0.783611152305618,-0.82311002743773,-0.0636202980230517,-0.929449013772403,-0.285235637405286,0.00477365415548436,-0.55246873791445,-0.681245328293018,-0.115229591419237,-0.0070696216713616,-0.295287318654158,-0.523777291139794,-0.471099417535258,0.178511648619467,0.264118468146932,-0.956107169088601,-0.0756536450483456,-0.441011614198393,-0.553714205123461,-0.309688126271238,0.0476015269448076,-0.43838737358266,0.277561385233645,-0.744064623183551,0.260972684963396,-0.953173156563747,-0.307835044632598,-0.701063643266663,-0.389730097156067,-0.0624659125977922,-0.240630787351774,0.462878809385783,-0.111335145744462,0.0983489501825279,-0.594686133786712,-0.714120440210846,0.423661517574285,0.0810355455982468,-0.308360776776876,-0.380730490330723,0.472504864981183,-0.162089772725515,0.770905198598368,-0.520584772866707,-0.958931489912679,-0.697553823978444,-0.574200762927142,-0.745974664333839,-0.233031307597196,0.608150772117009,-0.045456440674107,-0.121746055894465,0.0352790939602097,-0.910757876467242,0.496632456126357,-0.876024592146273,-0.857932009801961,0.735040572548167,0.545668509510925,-0.238461696627754,0.0821605348827585,-0.412550080178575,-0.301772524123558,-0.11913771354363,-0.0393947754573166,0.530752414879345,-0.188430072278442,-0.782682633241893,0.46434617776388,-0.895406227872982,-0.891334798549793,-0.339312723818706,0.475448761264211,-0.404699850402166,-0.516763657246363,-0.940556485539537,-0.92849651851827,0.645862539648163,-0.199783775970198,-0.00248907005809851,-0.122819266507702,-0.202019954654897,-0.4981120602271,-0.455577492716916,-0.0985044247654197,-0.226991456397685,0.0282410819296792,-0.47668226738741,0.561711568759777,-0.863794687627325,-0.410784614756897,-0.145622136018494,-0.366488677130502,0.353388522676967,-0.429842006082167,0.850274480297529,-0.0642528156393034,-0.431641926012091,-0.812644817159061,0.47068818708653,-0.398385152479503,0.492089413772424,0.433376595707381,-0.247608304255446,-0.0886151440753689,-0.981519214224213,-0.0982448650907568,-0.400749631987149,-0.111145172303233,-0.120824529990389,0.920414988771061,0.916069325499366,0.854630573708919,-0.0549898584127825,-0.0681152977474667,-0.158150393985125,-0.0289629510429955,-0.106728076965046,0.626976910622093,-0.525278149040915,-0.55019340069765,-0.1621973762235,-0.141568845877338,-0.17872455727385,0.342347640967531,-0.392551529080759,-0.177641538073425,-0.240827433744508,-0.228243748508092,-0.240969096635113,0.677230610667757],"y":[1,0.373518025114293,0.272605087273482,0.315879697881651,0.47128648815779,0.431744494698171,0.278622589720582,0.429807696831425,0.480085342653905,0.3780499005601,0.325534648279581,-0.0253426498574287,-0.047165545236754,-0.218202205024212,-0.0173447329971513,-0.221186622627867,0.685821528962925,0.293700428216996,1,0.760325442462034,0.613171752922682,0.682999757195622,0.38199573075588,-0.785528988252336,0.24846037063091,0.28287486351737,0.40335645792776,0.240762737472114,0.268294332928469,0.0900538002561964,0.132858388556303,-0.359689451149015,-0.557862465061058,0.868411549576266,1,-0.197999703108389,-0.0999236211260174,-0.0987165962525512,-0.278078180632424,-0.477244609334181,-0.378720203708784,-0.313333148903423,-0.34543222956801,-0.286349234709858,-0.273178411868025,0.382064049912267,0.656801469141516,0.725800126413701,0.566380733057115,0.618383451188943,0.760325442462034,0.694018095272393,0.760325442462034,0.758503646948959,0.68984302016009,0.60595736946228,0.608726286559888,0.568992720640435,0.94740925590885,0.760325442462034,0.660147463587401,0.687124380810023,0.682163052615666,-0.332276574010407,-0.453082803155528,0.354338088747386,0.689100871560569,0.736736386163239,0.567436163239689,0.597521898619551,0.674370442592396,0.704946597398927,0.760325442462034,0.219313576456322,0.393471973597937,0.274049964435028,0.334617154139905,0.262253281033038,-0.678103524045067,-0.579124720286265,0.846620832865244,0.96038840832807,1,-0.389849702659438,0.398745503631865,0.605500063305637,0.715587508589687,0.760325442462034,0.304448195381304,0.314848571439742,0.180866572280735,0.998360092364091,1,1,0.704439118788088,0.604047763023194,0.760325442462034,0.703281830975228,0.450052645035314,0.310392135546614,0.381207151860224,0.760325442462034,0.61335103206269,1,-0.451097630228755,0.340920685045192,0.215741262894117,1,0.628042619788502,0.760325442462034,-0.359596184135239,-0.236487021706495,-0.224723008290818,0.236375072273154,-0.430871098204306,-0.516451483739753,-0.561445363585078,-0.209650295578792,-0.22880294718395,-0.0629175447523498,-0.0276527880276149,-0.0173447329971513,-0.514941800972525,-0.787289116531699,1,0.605934279467058,0.716541898622024,0.760325442462034,1,0.37697614448271,-0.642020230471365,0.704279996314305,0.383089685244686,0.995334982673961,1,-0.338054081052444,-0.405937129834686,1,-0.436966601703296,0.138090590096522,0.233113326884516,1,1,0.314538845540893,0.443196826158992,0.068950657406059,-0.572714339881607,-0.694082755004924,-0.752586244648846,-0.3351336486115,0.313757247841737,0.346114451176462,0.449714347319678,0.997832616990481,-0.022614621098785,0.420704273073723,-0.584503185789721,-0.623657273630291,-0.7018967199424,-0.758625559551521,-0.774449771699663,-0.800006568357612,1,0.270341176858047,0.250664405144313,0.308908661015591,1,-0.616755950913742,0.616584948722254,0.725439070359174,0.651265117373554,0.760325442462034,1,0.758854928803206,0.608680646799889,-1,-0.580259892717711,-0.62288053789105,-0.5312831952119,0.997831912028802,0.994825588022286,-0.585553389752954,1,0.375930800961773,-0.119061204216217,0.611923595927823,0.340666701394698,1,0.684954928406564,0.465123147275877,-0.885343094455872,0.32409322489718,0.193965106991851,-0.444418758654973,0.934920614399836,-0.121186543325097,-0.363554386857247,0.200368181399109,0.480085342653905,0.664489929596002,0.669133133732098,1,0.736519830700423,0.85575452434619,0.986115605136197,0.678775483956435,-0.376045436819144,0.374880383560734,0.760325442462034,0.303745630562109,0.662080710201018,0.461540913309692,0.28636828729778,0.294678786713718,-0.655232367451606,0.935616975929461,-0.469764991390013,0.480085342653905,0.273026106434739,-0.240443477168404,0.386687914118737,0.693984754480538,0.267406850831863,0.991107889167997,1,0.759581604066463,0.998734390643744,0.760325442462034,0.682034121477535,0.760325442462034,0.427378500831339,0.358159182774425,0.686262883933524,1,-0.42788417999753,0.276336417241976,0.662399557748339,0.2605753940222,1,0.693994213625867,0.200421596064216,1,-0.256361943611918,0.234495660322375,-0.494763753414806,-0.110730046150258,-0.631640838172942,-0.762160699401941,1,0.406568568067381,-0.182124994477674,-0.783624849692183,-0.607886336617043,0.694292142242449,1,0.480085342653905,0.133895744019197,0.378271182913594,-0.530262684499152,0.760325442462034,0.46140602393598,-0.460300505420209,0.327231736127822,-0.520434047929016,0.166688318826272,0.612170117294332,0.993179115622959,-0.307508368166589,-0.422758712159236,-0.176517886978228,1,-0.535979768248619,-0.462597526152281,0.321284962620697,0.232341687684489,0.998698344319947,0.694372351003439,-0.327508290024131,0.996425846340024,0.755904216237633,1,-0.639316271538648,0.423505529366868,-0.120677346011383,0.933490934060359,-0.483409342489951,-0.247310160462127,0.736929939045396,0.386918297058789,0.310807398937927,0.3785112675674,0.16142515744478,-0.562245834795844,-0.648848053218861,-0.706801407549863,-0.324730363917217,0.249029931230988,-0.234738508275065,0.613125786627136,0.369694578167172,0.994250457206643,-0.492312957884996,0.608331454380421,0.613126335053329,-0.567624859116531,0.376046725570933,-0.888659815746668,0.417571225199567,-0.140025085635933,0.141371944105397,1,0.480085342653905,0.287451074592368,-0.534681010891657,0.276092718867486,0.466506232247261,0.480085342653905,-0.158582522776583,0.29377810386874,0.331606475025038,-0.396810644969325,0.480085342653905,-0.603778399598383,-0.514040779210342,-0.752114054624889,-0.844262450595752,-0.684791796950014,1,0.469619435672367,0.351381828534895,0.332665536410693,0.344474985082409,-0.533973922839674,1,0.739293043664543,0.653693895875488,-0.62036652399335,0.696509934529407,1,-0.482429681623576,0.740146529098009,0.704345791679721,-0.683894678066488,0.455166343688198,0.621265202925222,-0.19100596669766,-0.49591149945457,-0.184877944796576,-0.904363070316195,-0.620802550742024,-0.63181170436784,-0.130642462515088,0.680415510661366,0.725474543042472,0.65124436320273,-0.587077113137967,-0.687784548928713,-0.243002087761108,-0.57751219275503,-0.149488094804552,1,0.280170126548647,0.302513275301649,-0.160580138556489,-0.513268377632983,-0.204156054338563,0.99581072750404,-0.484747010303342,-0.324984422601163,-0.157146213066398,-0.133717117493149,-0.224700524123274,0.679539636632415]},"edges":{"from":["per_57219793915","per_57212764664","per_36464380700","per_56436897300","per_57456534500","per_57189387436","per_55280003800","per_53985251000","per_10440626500","per_55743415000","per_6507604844","per_54403612000","per_7103353204","per_6603463277","per_53985361900","per_55979915700","per_56741425700","per_57190007914","per_35739135100","per_24512267300","per_57211533183","per_17342030700","per_8543458200","per_39962198000","per_8542323900","per_56190665700","per_22954214400","per_36665830700","per_55940741800","per_56190665700","per_8542323900","per_57212211925","per_57250988600","per_36665830700","per_55940741800","per_56194926500","per_6603929800","per_55726786800","per_52564389200","per_6602636501","per_14423907700","per_7004144747","per_37034834100","per_23024872800","per_55276541900","per_57299589400","per_35107535800","per_57222077797","per_57224946340","per_57190007914","per_8647872300","per_11839066900","per_57225087680","per_18634327300","per_57192692601","per_23093709400","per_57224577809","per_57206898463","per_36550241400","per_25027075800","per_15057981500","per_35267867000","per_55832083700","per_6701767937","per_55654724000","per_56210777300","per_7006077782","per_6701767937","per_6701767937","per_26433451900","per_57221371525","per_6508090491","per_23488048300","per_36997206000","per_36997017700","per_8542323900","per_56190665700","per_22954214400","per_36665830700","per_55940741800","per_57202985610","per_7101928929","per_7004526393","per_57193317190","per_7005713737","per_23488048300","per_57208315990","per_36925959500","per_57196479751","per_7006691882","per_55940741800","per_36665830700","per_55250349200","per_8647872300","per_6602160256","per_6701907779","per_57218268471","per_50262130700","per_6602356437","per_37034854100","per_23024872800","per_6701907779","per_56400915100","per_23488048300","per_37034834100","per_6603463277","per_57190007914","per_56685832700","per_9432632800","per_36785334500","per_57216353981","per_23767166000","per_57216338926","per_57021871900","per_57205611884","per_56741425700","per_6603223065","per_6602981109","per_36562060300","per_26433451900","per_55841158700","per_55166465600","per_8543458200","per_56123759300","per_15127074300","per_36823419900","per_56741425700","per_57212795362","per_57204186756","per_55540217900","per_7005952768","per_57196479751","per_7006691882","per_57203635258","per_57208318231","per_57208315990","per_56210777300","per_55654724000","per_7006077782","per_57190007914","per_6504054963","per_24331023000","per_56162108500","per_57190007914","per_57021871900","per_57207945523","per_24723823500","per_6507184057","per_8855704500","per_57194586621","per_23024872800","per_16751030800","per_6504749273","per_6701907779","per_28367582700","per_36674748500","per_41862438300","per_57204391106","per_57204393725","per_56205553600","per_16426432300","per_6602160256","per_57202946021","per_23767166000","per_6603463277","per_6602160256","per_16426432300","per_56205553600","per_23502406500","per_16031671800","per_56417516200","per_55891701000","per_22954214400","per_36665830700","per_41661641800","per_8543458200","per_23024872800","per_55375454200","per_6701907779","per_7005385351","per_24068102500","per_56205553600","per_57190007914","per_36823419900","per_6508090491","per_6603929800","per_55940741800","per_55166465600","per_6507277353","per_57793531800","per_36674748500","per_28367582700","per_8982534700","per_21743627000","per_56205553600","per_24723823500","per_6602636501","per_7004144747","per_16444345400","per_23024872800","per_55375454200","per_16751030800","per_6504749273","per_6701907779","per_56205553600","per_55954974200","per_23488048300","per_36244466900","per_6602445638","per_15127074300","per_6603223065","per_56162108500","per_24331023000","per_24723823500","per_21740696500","per_17342030700","per_6507071133","per_6603929800","per_23488048300","per_55979915700","per_54403612000","per_53985361900","per_6603463277","per_7103353204","per_55726786800","per_52564389200","per_6508090491","per_28367582700","per_36674748500","per_55166465600","per_24068102500","per_56400915100","per_23767166000","per_8855704500","per_6507604844","per_53985251000","per_55328949500","per_7401971740","per_55743415000","per_55250349200","per_6602674368","per_55375454200","per_16751030800","per_24070523500","per_55375874000","per_23024872800","per_6701907779","per_6701907779","per_56748010200","per_6602160256","per_36674748500","per_23488048300","per_28367582700","per_36674748500","per_6507184057","per_35099343500","per_6602981109","per_56433512900","per_54406796800","per_6603171524","per_13407411800","per_56205553600","per_7005952768","per_17342030700","per_7005385351","per_56205553600","per_15127074300","per_16426432300","per_23488048300","per_6603463277","per_6506385182","per_8855704500","per_13407411800","per_8543458200","per_8543458200","per_56205553600","per_6603929800","per_23488048300","per_55342418800","per_24068102500","per_36674748500","per_28367582700","per_8543458200","per_8647872300","per_55250349200","per_54406796800","per_7005952768","per_36823419900","per_6701907779","per_36128589400","per_7005952768","per_6506958322","per_55954974200","per_54584939400","per_39962198000","per_6504749273","per_15070249500","per_36521591600","per_55375874000","per_55375454200","per_6602843376","per_6701907779","per_25931363700","per_36823419900","per_7202755185","per_18041904100","per_24068102500","per_55342418800","per_55394472800","per_55250349200","per_8647872300","per_23488048300","per_8543458200","per_6603929800","per_8862840800","per_7005385351","per_11839066900","per_54584939400","per_35195259300","per_6603929800","per_36104233000","per_55851946982","per_36104759600","per_36104785900","per_55996060000","per_6508090491","per_6603929800","per_6602981109","per_36997206000","per_36997190700","per_36997017700","per_36997336200","per_39962198000","per_6603929800","per_23488048300","per_36997336200","per_36997206000","per_36997017700","per_55979915700","per_23024872800","per_6504749273","per_6701907779","per_28367582700","per_6507291476","per_6602843376","per_6603929800","per_6504749273","per_6701907779","per_6603463277","per_55851946982","per_36104759600","per_36104233000","per_36104785900","per_36104233000","per_55851946982","per_36104759600","per_36104785900","per_36104759600","per_36104233000","per_55851946982","per_36104785900","per_23024872800","per_6504749273","per_6602160256","per_6701907779","per_16426432300","per_6602160256","per_6701907779","per_6603463277","per_23024872800","per_22981409500","per_23024205100","per_23024124800","per_6602160256","per_6701907779","per_6603463277","per_56055484200","per_57192872701","per_7202755185","per_24068102500","per_18041904100","per_24068102500","per_7202755185","per_6603463277","per_55979915700","per_7103353204","per_6504749273","per_8976045400","per_7005952768","per_55954974200","per_6701907779","per_55979915700","per_6603463277","per_21740696500","per_6603929800","per_6603463277","per_7005952768","per_55979915700","per_55954974200","per_55979915700","per_55979915700","per_55979915700","per_7004526393"],"to":["pap_57219793915::Age-specific rate of severe and critical SARS-CoV-2 infections estimated with multi-country seroprevalence studies","pap_57212764664::Origin and spreading of canine morbillivirus in South America","pap_57212764664::Origin and spreading of canine morbillivirus in South America","pap_57212764664::Origin and spreading of canine morbillivirus in South America","pap_57212764664::Origin and spreading of canine morbillivirus in South America","pap_57212764664::Origin and spreading of canine morbillivirus in South America","pap_57212764664::Origin and spreading of canine morbillivirus in South America","pap_57212764664::Origin and spreading of canine morbillivirus in South America","pap_57212764664::Origin and spreading of canine morbillivirus in South America","pap_57212764664::Origin and spreading of canine morbillivirus in South America","pap_57212764664::Origin and spreading of canine morbillivirus in South America","pap_54403612000::Wool scoured colour: Heritability, genetic and phenotypic correlations with wool traits in Corriedale sheep","pap_54403612000::Wool scoured colour: Heritability, genetic and phenotypic correlations with wool traits in Corriedale sheep","pap_54403612000::Wool scoured colour: Heritability, genetic and phenotypic correlations with wool traits in Corriedale sheep","pap_54403612000::Wool scoured colour: Heritability, genetic and phenotypic correlations with wool traits in Corriedale sheep","pap_54403612000::Wool scoured colour: Heritability, genetic and phenotypic correlations with wool traits in Corriedale sheep","pap_56741425700::Assessment of Real-Time Quaking-Induced Conversion (RT-QuIC) Assay, Immunohistochemistry and ELISA for Detection of Chronic Wasting Disease under Field Conditions in White-Tailed Deer: A Bayesian Approach","pap_57224533812::Colonization rather than fragmentation explains the geographical distribution and diversification of treefrogs endemic to Brazilian shield sky islands","pap_35739135100::Performance of bovine genital campylobacteriosis diagnostic tests in bulls from Uruguay: a Bayesian latent class model approach","pap_24512267300::Not the same: phylogenetic relationships and ecological niche comparisons between two different forms of Aglaoctenus lagotis from Argentina and Uruguay","pap_24512267300::Not the same: phylogenetic relationships and ecological niche comparisons between two different forms of Aglaoctenus lagotis from Argentina and Uruguay","pap_24512267300::Not the same: phylogenetic relationships and ecological niche comparisons between two different forms of Aglaoctenus lagotis from Argentina and Uruguay","pap_57217136035::Whole genomic comparative analysis of Streptococcus pneumoniae serotype 1 isolates causing invasive and non-invasive infections among children under 5 years in Casablanca, Morocco","pap_39962198000::Ultraconserved Elements Improve the Resolution of Difficult Nodes within the Rapid Radiation of Neotropical Sigmodontine Rodents (Cricetidae: Sigmodontinae)","pap_8542323900::Characterization of “candidatus ehrlichia pampeana” in haemaphysalis juxtakochi ticks and gray brocket deer (Mazama gouazoubira) from uruguay","pap_8542323900::Characterization of “candidatus ehrlichia pampeana” in haemaphysalis juxtakochi ticks and gray brocket deer (Mazama gouazoubira) from uruguay","pap_8542323900::Characterization of “candidatus ehrlichia pampeana” in haemaphysalis juxtakochi ticks and gray brocket deer (Mazama gouazoubira) from uruguay","pap_8542323900::Characterization of “candidatus ehrlichia pampeana” in haemaphysalis juxtakochi ticks and gray brocket deer (Mazama gouazoubira) from uruguay","pap_8542323900::Characterization of “candidatus ehrlichia pampeana” in haemaphysalis juxtakochi ticks and gray brocket deer (Mazama gouazoubira) from uruguay","pap_56190665700::An Hepatozoon americanum-like protozoan in crab-eating (Cerdocyon thous) and grey pampean (Lycalopex gymnocercus) foxes from Uruguay","pap_56190665700::An Hepatozoon americanum-like protozoan in crab-eating (Cerdocyon thous) and grey pampean (Lycalopex gymnocercus) foxes from Uruguay","pap_56190665700::An Hepatozoon americanum-like protozoan in crab-eating (Cerdocyon thous) and grey pampean (Lycalopex gymnocercus) foxes from Uruguay","pap_56190665700::An Hepatozoon americanum-like protozoan in crab-eating (Cerdocyon thous) and grey pampean (Lycalopex gymnocercus) foxes from Uruguay","pap_56190665700::An Hepatozoon americanum-like protozoan in crab-eating (Cerdocyon thous) and grey pampean (Lycalopex gymnocercus) foxes from Uruguay","pap_56190665700::An Hepatozoon americanum-like protozoan in crab-eating (Cerdocyon thous) and grey pampean (Lycalopex gymnocercus) foxes from Uruguay","pap_56194926500::Genomic footprints of Quaternary colonization and population expansion in the Patagonian-Fuegian region rules out a separate southern refugium in Tierra del Fuego","pap_56194926500::Genomic footprints of Quaternary colonization and population expansion in the Patagonian-Fuegian region rules out a separate southern refugium in Tierra del Fuego","pap_55726786800::Automatic calibration and uncertainty quantification in waves dynamical downscaling","pap_55726786800::Automatic calibration and uncertainty quantification in waves dynamical downscaling","pap_55909075500::Sediment grain size determines microplastic exposure landscapes for sandy beach macroinfauna","pap_55909075500::Sediment grain size determines microplastic exposure landscapes for sandy beach macroinfauna","pap_55909075500::Sediment grain size determines microplastic exposure landscapes for sandy beach macroinfauna","pap_37034834100::Emergence and spread of a b.1.1.28-derived p.6 lineage with q675h and q677h spike mutations in uruguay","pap_37034834100::Emergence and spread of a b.1.1.28-derived p.6 lineage with q675h and q677h spike mutations in uruguay","pap_37034834100::Emergence and spread of a b.1.1.28-derived p.6 lineage with q675h and q677h spike mutations in uruguay","pap_37034834100::Emergence and spread of a b.1.1.28-derived p.6 lineage with q675h and q677h spike mutations in uruguay","pap_37034834100::Emergence and spread of a b.1.1.28-derived p.6 lineage with q675h and q677h spike mutations in uruguay","pap_37034834100::Emergence and spread of a b.1.1.28-derived p.6 lineage with q675h and q677h spike mutations in uruguay","pap_37034834100::Emergence and spread of a b.1.1.28-derived p.6 lineage with q675h and q677h spike mutations in uruguay","pap_25824960900::Quaternary climatic fluctuations influence the demographic history of two species of sky-island endemic amphibians in the Neotropics","pap_25821559500::Building the GLENCOE Platform -Grasslands LENding eConomic and ecOsystems sErvices","pap_57193920068::Population structure and gene flow of Geoffroy's cat (Leopardus geoffroyi) in the Uruguayan Savanna ecoregion","pap_57193920068::Population structure and gene flow of Geoffroy's cat (Leopardus geoffroyi) in the Uruguayan Savanna ecoregion","pap_57193920068::Population structure and gene flow of Geoffroy's cat (Leopardus geoffroyi) in the Uruguayan Savanna ecoregion","pap_57193920068::Population structure and gene flow of Geoffroy's cat (Leopardus geoffroyi) in the Uruguayan Savanna ecoregion","pap_57193920068::Population structure and gene flow of Geoffroy's cat (Leopardus geoffroyi) in the Uruguayan Savanna ecoregion","pap_57224577809::Water-quality data imputation with a high percentage of missing values: A machine learning approach","pap_57224577809::Water-quality data imputation with a high percentage of missing values: A machine learning approach","pap_57224577809::Water-quality data imputation with a high percentage of missing values: A machine learning approach","pap_57224577809::Water-quality data imputation with a high percentage of missing values: A machine learning approach","pap_57224577809::Water-quality data imputation with a high percentage of missing values: A machine learning approach","pap_57224577809::Water-quality data imputation with a high percentage of missing values: A machine learning approach","pap_57224577809::Water-quality data imputation with a high percentage of missing values: A machine learning approach","pap_6701767937::A new early warning indicator of tree species crashes from effective intraspecific interactions in tropical forests","pap_55654724000::The inner ear anatomy of glyptodonts and pampatheres (Xenarthra, Cingulata): Functional and phylogenetic implications","pap_55654724000::The inner ear anatomy of glyptodonts and pampatheres (Xenarthra, Cingulata): Functional and phylogenetic implications","pap_55654724000::The inner ear anatomy of glyptodonts and pampatheres (Xenarthra, Cingulata): Functional and phylogenetic implications","pap_6701767937::A method for predicting species trajectories tested with trees in barro colorado tropical forest","pap_6603547554::Markets as ecological networks: Inferring interactions and identifying communities","pap_57216650158::Riparian forest subsidises the biomass of fish in a recently formed subtropical reservoir","pap_57221371525::Adaptive evolution of β-globin gene in subterranean in South America octodontid rodents","pap_57221371525::Adaptive evolution of β-globin gene in subterranean in South America octodontid rodents","pap_56927960700::Inclusion of bioclimatic variables in genetic evaluations of dairy cattle","pap_36997206000::Applying Bayesian Networks to help Physicians Diagnose Respiratory Diseases in the context of COVID-19 Pandemic","pap_36997206000::Applying Bayesian Networks to help Physicians Diagnose Respiratory Diseases in the context of COVID-19 Pandemic","pap_8542323900::Molecular characterization of novel Ehrlichia genotypes in Ixodes auritulus from Uruguay","pap_8542323900::Molecular characterization of novel Ehrlichia genotypes in Ixodes auritulus from Uruguay","pap_8542323900::Molecular characterization of novel Ehrlichia genotypes in Ixodes auritulus from Uruguay","pap_8542323900::Molecular characterization of novel Ehrlichia genotypes in Ixodes auritulus from Uruguay","pap_8542323900::Molecular characterization of novel Ehrlichia genotypes in Ixodes auritulus from Uruguay","pap_57202985610::Development of a Population Pharmacokinetic Model for Cyclosporine from Therapeutic Drug Monitoring Data","pap_57202985610::Development of a Population Pharmacokinetic Model for Cyclosporine from Therapeutic Drug Monitoring Data","pap_57202985610::Development of a Population Pharmacokinetic Model for Cyclosporine from Therapeutic Drug Monitoring Data","pap_57202985610::Development of a Population Pharmacokinetic Model for Cyclosporine from Therapeutic Drug Monitoring Data","pap_57202985610::Development of a Population Pharmacokinetic Model for Cyclosporine from Therapeutic Drug Monitoring Data","pap_56927960700::Selection for test-day milk yield and thermotolerance in brazilian holstein cattle","pap_57208315990::Validation of a multiplex PCR assay to detect Babesia spp. and Anaplasma marginale in cattle in Uruguay in the absence of a gold standard test","pap_57208315990::Validation of a multiplex PCR assay to detect Babesia spp. and Anaplasma marginale in cattle in Uruguay in the absence of a gold standard test","pap_57208315990::Validation of a multiplex PCR assay to detect Babesia spp. and Anaplasma marginale in cattle in Uruguay in the absence of a gold standard test","pap_57208315990::Validation of a multiplex PCR assay to detect Babesia spp. and Anaplasma marginale in cattle in Uruguay in the absence of a gold standard test","pap_57208315990::Validation of a multiplex PCR assay to detect Babesia spp. and Anaplasma marginale in cattle in Uruguay in the absence of a gold standard test","pap_57208315990::Validation of a multiplex PCR assay to detect Babesia spp. and Anaplasma marginale in cattle in Uruguay in the absence of a gold standard test","pap_57219778546::Genetic parameters for production traits, resistance and resilience to Nematode parasites under different worm burden challenges in Corriedale sheep","pap_57219778546::Genetic parameters for production traits, resistance and resilience to Nematode parasites under different worm burden challenges in Corriedale sheep","pap_35228674100::Evidence of increasing diversification of emerging Severe Acute Respiratory Syndrome Coronavirus 2 strains","pap_35228674100::Evidence of increasing diversification of emerging Severe Acute Respiratory Syndrome Coronavirus 2 strains","pap_57218268471::Is it possible to accurately estimate lactation curve parameters in extensive beef production systems?","pap_57218268471::Is it possible to accurately estimate lactation curve parameters in extensive beef production systems?","pap_57218268471::Is it possible to accurately estimate lactation curve parameters in extensive beef production systems?","pap_37034854100::An evolutionary insight into emerging Ebolavirus strains isolated in Africa","pap_37034854100::An evolutionary insight into emerging Ebolavirus strains isolated in Africa","pap_37034854100::An evolutionary insight into emerging Ebolavirus strains isolated in Africa","pap_57218192116::Low additive genetic variation in a trait under selection in domesticated rice","pap_55841211100::Single-step genomic evaluations from theory to practice: using snp chips and sequence data in blupf90","pap_57193259331::Historical origins and zoonotic potential of avian influenza virus H9N2 in Tunisia revealed by Bayesian analysis and molecular characterization","pap_57193259331::Historical origins and zoonotic potential of avian influenza virus H9N2 in Tunisia revealed by Bayesian analysis and molecular characterization","pap_45861549800::Multilocus phylogeny of Paratelmatobiinae (Anura: Leptodactylidae) reveals strong spatial structure and previously unknown diversity in the Atlantic Forest hotspot","pap_56685832700::Predicting cyanobacterial biovolume from water temperature and conductivity using a Bayesian compound Poisson-Gamma model","pap_56685832700::Predicting cyanobacterial biovolume from water temperature and conductivity using a Bayesian compound Poisson-Gamma model","pap_56685832700::Predicting cyanobacterial biovolume from water temperature and conductivity using a Bayesian compound Poisson-Gamma model","pap_56805256800::Gametes Simulator: A multilocus genotype simulator to analyze genetic structure in outbreeding diploid species","pap_56805256800::Gametes Simulator: A multilocus genotype simulator to analyze genetic structure in outbreeding diploid species","pap_56805256800::Gametes Simulator: A multilocus genotype simulator to analyze genetic structure in outbreeding diploid species","pap_57215014126::Using approximate Bayesian inference for a \"steps and turns\" continuous-time random walk observed at regular time intervals","pap_57205611884::Fraud Detection in Electric Power Distribution: An Approach That Maximizes the Economic Return","pap_56741425700::Association between results of diagnostic tests for bovine tuberculosis and Johne's disease in cattle","pap_57202058591::The discovery of Neotropical Lepidosira (Collembola, Entomobryidae) and its systematic position","pap_7202016735::APPARENT BIAS: WHAT DOES ATTITUDE POLARIZATION SHOW?","pap_36562060300::The impact of anthropogenic food subsidies on a generalist seabird during nestling growth","pap_36562060300::The impact of anthropogenic food subsidies on a generalist seabird during nestling growth","pap_36562060300::The impact of anthropogenic food subsidies on a generalist seabird during nestling growth","pap_57193920068::Conservation planning in Uruguay based on small felids (Leopardus spp.) as umbrella species","pap_57211321476::Species diversity and distribution of the genus Colpomenia (Scytosiphonaceae, Phaeophyceae) along the coast of China","pap_56123759300::Coalescent-based species delimitation in the sand lizards of the Liolaemus wiegmannii complex (Squamata: Liolaemidae)","pap_56123759300::Coalescent-based species delimitation in the sand lizards of the Liolaemus wiegmannii complex (Squamata: Liolaemidae)","pap_56123759300::Coalescent-based species delimitation in the sand lizards of the Liolaemus wiegmannii complex (Squamata: Liolaemidae)","pap_56741425700::Modeling the Accuracy of Two in-vitro Bovine Tuberculosis Tests Using a Bayesian Approach","pap_56741425700::Modeling the Accuracy of Two in-vitro Bovine Tuberculosis Tests Using a Bayesian Approach","pap_56741425700::Modeling the Accuracy of Two in-vitro Bovine Tuberculosis Tests Using a Bayesian Approach","pap_55540217900::Millennium-length precipitation reconstruction over south-eastern asia: A pseudo-proxy approach","pap_26657911800::Detection of presumed genes encoding beta-lactamases by sequence based screening of metagenomes derived from Antarctic microbial mats","pap_57196479751::Probability of Rhipicephalus microplus introduction into farms by cattle movement using a Bayesian Belief Network","pap_57196479751::Probability of Rhipicephalus microplus introduction into farms by cattle movement using a Bayesian Belief Network","pap_57196479751::Probability of Rhipicephalus microplus introduction into farms by cattle movement using a Bayesian Belief Network","pap_57196479751::Probability of Rhipicephalus microplus introduction into farms by cattle movement using a Bayesian Belief Network","pap_57196479751::Probability of Rhipicephalus microplus introduction into farms by cattle movement using a Bayesian Belief Network","pap_56210777300::Phylogeny, Macroevolutionary Trends and Historical Biogeography of Sloths: Insights From a Bayesian Morphological Clock Analysis","pap_56210777300::Phylogeny, Macroevolutionary Trends and Historical Biogeography of Sloths: Insights From a Bayesian Morphological Clock Analysis","pap_56210777300::Phylogeny, Macroevolutionary Trends and Historical Biogeography of Sloths: Insights From a Bayesian Morphological Clock Analysis","pap_57195593253::Cryptic diversity in Brazilian endemic monkey frogs (Hylidae, Phyllomedusinae, Pithecopus) revealed by multispecies coalescent and integrative approaches","pap_6602206179::Joint Minimization of Monitoring Cost and Delay in Overlay Networks: Optimal Policies with a Markovian Approach","pap_7003534815::Genetic and pathogenic diversity of Colletotrichum species associated with apple diseases in southern Brazil and Uruguay","pap_7003534815::Genetic and pathogenic diversity of Colletotrichum species associated with apple diseases in southern Brazil and Uruguay","pap_57210218727::A hybrid swarm of manatees along the guianas coastline, a peculiar environment under the influence of the Amazon River plume","pap_57021871900::Fully Bayesian analysis of allele-specific RNA-seq data","pap_57207945523::A historical perspective of niche differentiation between two top predators in the Uruguayan coastal area","pap_57207945523::A historical perspective of niche differentiation between two top predators in the Uruguayan coastal area","pap_57207945523::A historical perspective of niche differentiation between two top predators in the Uruguayan coastal area","pap_57197386412::Impact of Pleistocene geoclimatic events on the genetic structure in mid-latitude South American plants: Insights from the phylogeography of Turnera sidoides complex (Passifloraceae, Turneroideae)","pap_57194586621::An evolutionary insight into zika virus strains isolated in the Latin American region","pap_57194586621::An evolutionary insight into zika virus strains isolated in the Latin American region","pap_57194586621::An evolutionary insight into zika virus strains isolated in the Latin American region","pap_57194586621::An evolutionary insight into zika virus strains isolated in the Latin American region","pap_57194586621::An evolutionary insight into zika virus strains isolated in the Latin American region","pap_28367582700::Mouth gape determines the response of marine top predators to long-term fishery-induced changes in food web structure","pap_28367582700::Mouth gape determines the response of marine top predators to long-term fishery-induced changes in food web structure","pap_28367582700::Mouth gape determines the response of marine top predators to long-term fishery-induced changes in food web structure","pap_28367582700::Mouth gape determines the response of marine top predators to long-term fishery-induced changes in food web structure","pap_28367582700::Mouth gape determines the response of marine top predators to long-term fishery-induced changes in food web structure","pap_56205553600::Inferring population dynamics of HIV-1 subtype C epidemics in Eastern Africa and Southern Brazil applying different Bayesian phylodynamics approaches","pap_57042629400::Phylodynamics of Merkel-cell polyomavirus and human polyomavirus 6: A long-term history with humans","pap_57042629400::Phylodynamics of Merkel-cell polyomavirus and human polyomavirus 6: A long-term history with humans","pap_57202946021::Action semantics at the bottom of the brain: Insights from dysplastic cerebellar gangliocytoma","pap_56805256800::Genetic structure, core collection, and regeneration quality in white dent corn landraces","pap_7006290311::Prediction of complex traits: Robust alternatives to best linear unbiased prediction","pap_57195056016::Human Bocavirus: Detection, Quantification and Molecular Characterization in Sewage and Surface Waters in Uruguay","pap_57195056016::Human Bocavirus: Detection, Quantification and Molecular Characterization in Sewage and Surface Waters in Uruguay","pap_14015150800::The HIV-1 subtype B epidemic in French guiana and Suriname is driven by ongoing transmissions of pandemic and non-pandemic lineages","pap_54987316200::Kelps' long-distance dispersal: Role of Ecological/Oceanographic processes and implications to marine forest conservation","pap_54987316200::Kelps' long-distance dispersal: Role of Ecological/Oceanographic processes and implications to marine forest conservation","pap_54987316200::Kelps' long-distance dispersal: Role of Ecological/Oceanographic processes and implications to marine forest conservation","pap_36718043400::Joint denoising and decompression: A patch-based Bayesian approach","pap_57206396817::Geographic distribution modeling of the margay (Leopardus wiedii) and jaguarundi (Puma yagouaroundi): A comparative assessment","pap_55777005500::Latent class models for Echinococcus multilocularis diagnosis in foxes in Switzerland in the absence of a gold standard","pap_41661641800::Historical isolation and contemporary gene flow drive population diversity of the brown alga Sargassum thunbergii along the coast of China","pap_41661641800::Historical isolation and contemporary gene flow drive population diversity of the brown alga Sargassum thunbergii along the coast of China","pap_43861064400::Evidence of increasing diversification of Zika virus strains isolated in the American continent","pap_43861064400::Evidence of increasing diversification of Zika virus strains isolated in the American continent","pap_43861064400::Evidence of increasing diversification of Zika virus strains isolated in the American continent","pap_23484882900::Amphitropical disjunctions in new world menthinae: Three pliocene dispersals to South America following late miocene dispersal to North America from the old world","pap_24068102500::The disjunct pattern of the Neotropical harvestman Discocyrtus dilatatus (Gonyleptidae) explained by climate-driven range shifts in the Quaternary: Paleodistributional and molecular evidence","pap_32067654500::Tracing the origin of the NS1 A188V substitution responsible for recent enhancement of ZIKA virus asian genotype infectivity","pap_57190007914::Evolutionarily significant units of the critically endangered leaf frog Pithecopus ayeaye (Anura, Phyllomedusidae) are not effectively preserved by the Brazilian protected areas network","pap_57190007914::Evolutionarily significant units of the critically endangered leaf frog Pithecopus ayeaye (Anura, Phyllomedusidae) are not effectively preserved by the Brazilian protected areas network","pap_6508090491::Molecular adaptive convergence in the α-globin in subterranean octodontid rodents","pap_6508090491::Molecular adaptive convergence in the α-globin in subterranean octodontid rodents","pap_55485032800::Description of a new soft tick species (Acari: Argasidae: Ornithodoros) associated with stream-breeding frogs (Anura: Cycloramphidae: Cycloramphus) in Brazil","pap_45861485800::A new species of Deltamys Thomas, 1917 (Rodentia: Cricetidae) endemic to the southern Brazilian Araucaria Forest and notes on the expanded phylogeographic scenario of D. kempi","pap_55819830400::Uncertainty Quantification in a Patient-Specific One-Dimensional Arterial Network Model: EnKF-Based Inflow Estimator","pap_55819830400::Uncertainty Quantification in a Patient-Specific One-Dimensional Arterial Network Model: EnKF-Based Inflow Estimator","pap_36674748500::Trophic relationships in apex predators in an estuary system: A multiple-method approximation","pap_36674748500::Trophic relationships in apex predators in an estuary system: A multiple-method approximation","pap_36674748500::Trophic relationships in apex predators in an estuary system: A multiple-method approximation","pap_36674748500::Trophic relationships in apex predators in an estuary system: A multiple-method approximation","pap_56205553600::Phylodynamics of the major HIV-1 CRF02_AG African lineages and its global dissemination","pap_24723823500::Trophic niche shifts driven by phytoplankton in sandy beach ecosystems","pap_24723823500::Trophic niche shifts driven by phytoplankton in sandy beach ecosystems","pap_24723823500::Trophic niche shifts driven by phytoplankton in sandy beach ecosystems","pap_16444345400::Modeling a production scale milk drying process: Parameter estimation, uncertainty and sensitivity analysis","pap_23024872800::Bayesian coalescent inference reveals high evolutionary rates and diversification of Zika virus populations","pap_23024872800::Bayesian coalescent inference reveals high evolutionary rates and diversification of Zika virus populations","pap_23024872800::Bayesian coalescent inference reveals high evolutionary rates and diversification of Zika virus populations","pap_23024872800::Bayesian coalescent inference reveals high evolutionary rates and diversification of Zika virus populations","pap_23024872800::Bayesian coalescent inference reveals high evolutionary rates and diversification of Zika virus populations","pap_12042289100::New insights into the hepatitis E virus genotype 3 phylodynamics and evolutionary history","pap_12042289100::New insights into the hepatitis E virus genotype 3 phylodynamics and evolutionary history","pap_56452472600::Weighting strategies for single-step genomic BLUP: An iterative approach for accurate calculation of GEBV and GWAS","pap_36244466900::Threatened amphibians of Uruguay: Effectiveness of protected areas face of climate change","pap_36244466900::Threatened amphibians of Uruguay: Effectiveness of protected areas face of climate change","pap_36244466900::Threatened amphibians of Uruguay: Effectiveness of protected areas face of climate change","pap_56340718000::Hidden Markov models for gene sequence classification: Classifying the VSG gene in the Trypanosoma brucei genome","pap_56520190900::Genetic structure of Colletotrichum fructicola associated to apple bitter rot and Glomerella leaf spot in Southern Brazil and Uruguay","pap_56520190900::Genetic structure of Colletotrichum fructicola associated to apple bitter rot and Glomerella leaf spot in Southern Brazil and Uruguay","pap_15724490400::Trophic Dynamics of the Cape Stumpnose (Rhabdosargus holubi, Sparidae) Across Three Adjacent Aquatic Habitats","pap_21741325600::Genetic parameters for carcass traits and body weight using a Bayesian approach in the Canchim cattle","pap_17342030700::Imprints of multiple glacial refugia in the Pyrenees revealed by phylogeography and palaeodistribution modelling of an endemic spider","pap_7004217649::Genetic relationships among American donkey populations: Insights into the process of colonization","pap_36778318500::Genetic relationships between Atlantic and Pacific populations of the notothenioid fish Eleginops maclovinus: The footprints of Quaternary glaciations in Patagonia","pap_55620407900::Genotype by environment interaction for tick resistance of Hereford and Braford beef cattle using reaction norm models","pap_57189904568::Genetic parameters of objectionable fibers and of their associations with fleece traits in Corriedale sheep","pap_57189904568::Genetic parameters of objectionable fibers and of their associations with fleece traits in Corriedale sheep","pap_57189904568::Genetic parameters of objectionable fibers and of their associations with fleece traits in Corriedale sheep","pap_57189904568::Genetic parameters of objectionable fibers and of their associations with fleece traits in Corriedale sheep","pap_57189904568::Genetic parameters of objectionable fibers and of their associations with fleece traits in Corriedale sheep","pap_55726786800::Automatic calibration of a wave model with an evolutionary Bayesian method","pap_55726786800::Automatic calibration of a wave model with an evolutionary Bayesian method","pap_37086878000::Phylogenetic relationships between tuco-tucos (Ctenomys, Rodentia) of the Corrientes group and the C. pearsoni complex","pap_28367582700::Stable isotopes reveal long-Term fidelity to foraging grounds in the galapagos sea lion zalophus wollebaeki)","pap_28367582700::Stable isotopes reveal long-Term fidelity to foraging grounds in the galapagos sea lion zalophus wollebaeki)","pap_45861485800::Genetic diversity of the swamp rat in South America: Population expansion after transgressive-regressive marine events in the Late Quaternary","pap_24068102500::More on the mesopotamian-yungas disjunction in subtropical and temperate argentina: Bioclimatic distribution models of the harvestman discocyrtus dilatatus (opiliones: Gonyleptidae)","pap_56400915100::Genetic diversification of local onion populations under different production systems in Uruguay","pap_56400915100::Genetic diversification of local onion populations under different production systems in Uruguay","pap_56400915100::Genetic diversification of local onion populations under different production systems in Uruguay","pap_6507604844::Molecular phylogeography of canine distemper virus: Geographic origin and global spreading","pap_6507604844::Molecular phylogeography of canine distemper virus: Geographic origin and global spreading","pap_6507604844::Molecular phylogeography of canine distemper virus: Geographic origin and global spreading","pap_6507604844::Molecular phylogeography of canine distemper virus: Geographic origin and global spreading","pap_6507604844::Molecular phylogeography of canine distemper virus: Geographic origin and global spreading","pap_55250349200::Description of the growth curve for Angus pasture-fed cows under extensive systems","pap_55250349200::Description of the growth curve for Angus pasture-fed cows under extensive systems","pap_55375454200::An evolutionary insight into Newcastle disease viruses isolated in Antarctica","pap_55375454200::An evolutionary insight into Newcastle disease viruses isolated in Antarctica","pap_55375454200::An evolutionary insight into Newcastle disease viruses isolated in Antarctica","pap_55375454200::An evolutionary insight into Newcastle disease viruses isolated in Antarctica","pap_55375454200::An evolutionary insight into Newcastle disease viruses isolated in Antarctica","pap_55375454200::An evolutionary insight into Newcastle disease viruses isolated in Antarctica","pap_55192958000::Phylogenetic analysis of Hepatitis B virus genotypes circulating in different risk groups of Panama, evidence of the introduction of genotype A2 in the country","pap_56748010200::Phylogenetic studies of the three rna silencing suppressor genes of south american ctv isolates reveal the circulation of a novel genetic lineage","pap_56748010200::Phylogenetic studies of the three rna silencing suppressor genes of south american ctv isolates reveal the circulation of a novel genetic lineage","pap_7004290258::Status, population trend and genetic structure of South American fur seals, Arctocephalus australis, in southwestern Atlantic waters","pap_7005873546::Genomic prediction for tick resistance in braford and hereford cattle","pap_28367582700::Sexual foraging segregation in South American sea lions increases during the pre-breeding period in the Río de la Plata plume","pap_28367582700::Sexual foraging segregation in South American sea lions increases during the pre-breeding period in the Río de la Plata plume","pap_28367582700::Sexual foraging segregation in South American sea lions increases during the pre-breeding period in the Río de la Plata plume","pap_28367582700::Sexual foraging segregation in South American sea lions increases during the pre-breeding period in the Río de la Plata plume","pap_7202016735::Does the better-than-average effect show that people are overconfident?: Two experiments","pap_56433512900::Temperament and its heritability in Corriedale and Merino lambs","pap_56433512900::Temperament and its heritability in Corriedale and Merino lambs","pap_56433512900::Temperament and its heritability in Corriedale and Merino lambs","pap_55308459300::Bayesian age and growth analysis of the shortfin mako shark Isurus oxyrinchus in the Western South Atlantic Ocean using a flexible model","pap_56205553600::Phylodynamics of major HIV-1 subtype B pandemic clades circuLating in Latin America","pap_56205553600::Phylodynamics of major HIV-1 subtype B pandemic clades circuLating in Latin America","pap_17342030700::Endemism and evolution in the littoral woodlouse Halophiloscia Verhoeff, 1908 (Crustacea, Isopoda, Oniscidea) from the Canary Islands: implications for conservation policies","pap_7003385168::A phylogeny of the Gochnatieae: Understanding a critically placed tribe in the Compositae","pap_32067654500::Spatiotemporal dynamics of the HIV-1 subtype G epidemic in West and Central Africa","pap_26026180300::Climate change and the distribution of neotropical red-bellied toads (Melanophryniscus, Anura, Amphibia): How to prioritize species and populations?","pap_27067620300::Temporal dynamics of norovirus GII.4 variants in Brazil between 2004 and 2012","pap_55841211100::Methods for genomic evaluation of a relatively small genotyped dairy population and effect of genotyped cow information in multiparity analyses","pap_7006293432::Thermal tolerances in rodents: Species that evolved in cold climates exhibit a wider thermoneutral zone","pap_7006293432::Thermal tolerances in rodents: Species that evolved in cold climates exhibit a wider thermoneutral zone","pap_55413829400::Diversification in the south american pampas: The genetic and morphological variation of the widespread petunia axillaris complex (solanaceae)","pap_30067502200::Comparative population genetics and evolutionary history of two commonly misidentified billfishes of management and conservation concern","pap_36100371800::Genetic diversity and population structure of the New World screwworm fly from the Amazon region of Brazil","pap_8543458200::Applying spatial analysis of genetic and environmental data to predict connection corridors to the New World screwworm populations in South America","pap_37119917600::A multilocus perspective on the phylogenetic relationships and diversification of rodents of the tribe Abrotrichini (Cricetidae: Sigmodontinae)","pap_37119917600::A multilocus perspective on the phylogenetic relationships and diversification of rodents of the tribe Abrotrichini (Cricetidae: Sigmodontinae)","pap_55945072500::Single Step, a general approach for genomic selection","pap_55342533900::Functional diversification within a predatory species","pap_7005722174::New records and distribution modeling of Gryne orensis (Sørensen) (Opiliones: Cosmetidae) support the Mesopotamian-Yungas disjunction in subtropical Argentina","pap_36674748500::Bias in diet determination: Incorporating traditional methods in Bayesian mixing models","pap_36674748500::Bias in diet determination: Incorporating traditional methods in Bayesian mixing models","pap_8543458200::The Phylogeographic History of the New World Screwworm Fly, Inferred by Approximate Bayesian Computation Analysis","pap_8647872300::Estimates of genetic parameters for worm resistance, wool and growth traits in Merino sheep of Uruguay","pap_8647872300::Estimates of genetic parameters for worm resistance, wool and growth traits in Merino sheep of Uruguay","pap_8647872300::Estimates of genetic parameters for worm resistance, wool and growth traits in Merino sheep of Uruguay","pap_32067654500::Tracing the Origin and Northward Dissemination Dynamics of HIV-1 Subtype C in Brazil","pap_36823419900::Quaternary range and demographic expansion of Liolaemus darwinii (Squamata: Liolaemidae) in the Monte Desert of Central Argentina using Bayesian phylogeography and ecological niche modelling","pap_55778646000::Molecular characterisation of hepatitis B virus in the resident Chinese population in Panama City","pap_36128589400::Evolution of optimal codon choices in the family Enterobacteriaceae","pap_36128589400::Evolution of optimal codon choices in the family Enterobacteriaceae","pap_36128589400::Evolution of optimal codon choices in the family Enterobacteriaceae","pap_36128589400::Evolution of optimal codon choices in the family Enterobacteriaceae","pap_54584939400::Fine-Scale Population Structure of Blue Whale Wintering Aggregations in the Gulf of California","pap_39962198000::Dating an impressive Neotropical radiation: Molecular time estimates for the Sigmodontinae (Rodentia) provide insights into its historical biogeography","pap_15070187100::Bayesian coalescent analysis of pandemic H1N1 influenza A virus circulating in the South American region","pap_15070187100::Bayesian coalescent analysis of pandemic H1N1 influenza A virus circulating in the South American region","pap_15070187100::Bayesian coalescent analysis of pandemic H1N1 influenza A virus circulating in the South American region","pap_15070187100::Bayesian coalescent analysis of pandemic H1N1 influenza A virus circulating in the South American region","pap_15070187100::Bayesian coalescent analysis of pandemic H1N1 influenza A virus circulating in the South American region","pap_15070187100::Bayesian coalescent analysis of pandemic H1N1 influenza A virus circulating in the South American region","pap_15070187100::Bayesian coalescent analysis of pandemic H1N1 influenza A virus circulating in the South American region","pap_25931363700::Exploring polymorphisms in B-DNA helical conformations","pap_36823419900::Species delimitation with abc and other coalescent-based methods: A test of accuracy with simulations and an empirical example with lizards of the liolaemus darwinii complex (Squamata: Liolaemidae)","pap_7202755185::Patterns of population differentiation in annual killifishes from the Paraná-Uruguay-La Plata Basin: The role of vicariance and dispersal","pap_7202755185::Patterns of population differentiation in annual killifishes from the Paraná-Uruguay-La Plata Basin: The role of vicariance and dispersal","pap_7202755185::Patterns of population differentiation in annual killifishes from the Paraná-Uruguay-La Plata Basin: The role of vicariance and dispersal","pap_7202755185::Patterns of population differentiation in annual killifishes from the Paraná-Uruguay-La Plata Basin: The role of vicariance and dispersal","pap_7202755185::Patterns of population differentiation in annual killifishes from the Paraná-Uruguay-La Plata Basin: The role of vicariance and dispersal","pap_55250349200::Genetic parameters for nematode resistance in periparturient ewes and post-weaning lambs in Uruguayan Merino sheep","pap_55250349200::Genetic parameters for nematode resistance in periparturient ewes and post-weaning lambs in Uruguayan Merino sheep","pap_55250349200::Genetic parameters for nematode resistance in periparturient ewes and post-weaning lambs in Uruguayan Merino sheep","pap_54410107400::Genetic variability and demographic history of Heliothis virescens (Lepidoptera: Noctuidae) populations from Brazil inferred by mtDNA sequences","pap_36778318500::Phylogeography of the sub-Antarctic notothenioid fish Eleginops maclovinus: Evidence of population expansion","pap_57548899500::Diversification of the American bulb-bearing Oxalis (Oxalidaceae): Dispersal to North America and modification of the tristylous breeding system","pap_7005385351::Phylogenetics of the chiliotrichum group (Compositae: Astereae): The story of the fascinating radiation in the paleate Astereae genera from southern South America","pap_14033086500::Protected areas do not fulfil the wintering habitat needs of the trans-Saharan migratory Montagu's harrier","pap_54584939400::Population genetic structure and social kin associations of franciscana dolphin, pontoporia blainvillei","pap_54584939400::Population genetic structure and social kin associations of franciscana dolphin, pontoporia blainvillei","pap_54584939400::Population genetic structure and social kin associations of franciscana dolphin, pontoporia blainvillei","pap_56172545600::Serological monitoring of selected avian pathogens of commercial poultry farms in Uruguay","pap_56172545600::Serological monitoring of selected avian pathogens of commercial poultry farms in Uruguay","pap_56172545600::Serological monitoring of selected avian pathogens of commercial poultry farms in Uruguay","pap_56172545600::Serological monitoring of selected avian pathogens of commercial poultry farms in Uruguay","pap_56172545600::Assessing the seroprevalence for abortigenic diseases of ovine herds based on diagnostic tests applied to individual-animals in Uruguay","pap_6508090491::The evolution of mitochondrial genomes in subterranean caviomorph rodents: Adaptation against a background of purifying selection","pap_6508090491::The evolution of mitochondrial genomes in subterranean caviomorph rodents: Adaptation against a background of purifying selection","pap_7202016735::Apparent overconfidence","pap_36997206000::Comparing Bayesian inference and case-based reasoning as support techniques in the diagnosis of Acute Bacterial Meningitis","pap_36997206000::Comparing Bayesian inference and case-based reasoning as support techniques in the diagnosis of Acute Bacterial Meningitis","pap_36997206000::Comparing Bayesian inference and case-based reasoning as support techniques in the diagnosis of Acute Bacterial Meningitis","pap_36997206000::Comparing Bayesian inference and case-based reasoning as support techniques in the diagnosis of Acute Bacterial Meningitis","pap_39962198000::Species groups and the evolutionary diversification of tuco-tucos, genus Ctenomys (Rodentia: Ctenomyidae)","pap_39962198000::Species groups and the evolutionary diversification of tuco-tucos, genus Ctenomys (Rodentia: Ctenomyidae)","pap_57218505091::Genome-wide marker-assisted selection combining all pedigree phenotypic information with genotypic data in one step: An example using broiler chickens","pap_36997336200::Evaluation of case based reasoning for clinical decision support systems applied to acute meningitis diagnose","pap_36997336200::Evaluation of case based reasoning for clinical decision support systems applied to acute meningitis diagnose","pap_36997336200::Evaluation of case based reasoning for clinical decision support systems applied to acute meningitis diagnose","pap_55979915700::Phenotypic and genetic characterization of novel somatic cell count traits from weekly or monthly observations","pap_20535566900::Evolution of Dengue virus type 3 genotype III in Venezuela: Diversification, rates and population dynamics","pap_20535566900::Evolution of Dengue virus type 3 genotype III in Venezuela: Diversification, rates and population dynamics","pap_20535566900::Evolution of Dengue virus type 3 genotype III in Venezuela: Diversification, rates and population dynamics","pap_28367582700::Change in the foraging strategy of female South American sea lions (Carnivora: Pinnipedia) after parturition","pap_14015150800::Phylodynamics of HIV-1 Circulating Recombinant Forms 12_BF and 38_BF in Argentina and Uruguay","pap_14015150800::Phylodynamics of HIV-1 Circulating Recombinant Forms 12_BF and 38_BF in Argentina and Uruguay","pap_15760008700::Dispersal and population structure at different spatial scales in the subterranean rodent Ctenomys australis","pap_37079657700::Genetic history of hepatitis C virus in Venezuela: High diversity and long time of evolution of HCV genotype 2","pap_37079657700::Genetic history of hepatitis C virus in Venezuela: High diversity and long time of evolution of HCV genotype 2","pap_55921976700::L2-Boosting algorithm applied to high-dimensional problems in genomic selection","pap_56172545600::Assessing the seroprevalence against avian pneumovirus and Ornithobacterium rhinotracheale in broilers in Uruguay","pap_56172545600::Assessing the seroprevalence against avian pneumovirus and Ornithobacterium rhinotracheale in broilers in Uruguay","pap_56172545600::Assessing the seroprevalence against avian pneumovirus and Ornithobacterium rhinotracheale in broilers in Uruguay","pap_56172545600::Assessing the seroprevalence against avian pneumovirus and Ornithobacterium rhinotracheale in broilers in Uruguay","pap_36104233000::Flock-level seroprevalence against avian pneumovirus amongst uruguayan broiler chickens","pap_36104233000::Flock-level seroprevalence against avian pneumovirus amongst uruguayan broiler chickens","pap_36104233000::Flock-level seroprevalence against avian pneumovirus amongst uruguayan broiler chickens","pap_36104233000::Flock-level seroprevalence against avian pneumovirus amongst uruguayan broiler chickens","pap_56172545600::Flock-level seroprevalence against Ornithobacterium rhinotracheale among broilers in Uruguay","pap_56172545600::Flock-level seroprevalence against Ornithobacterium rhinotracheale among broilers in Uruguay","pap_56172545600::Flock-level seroprevalence against Ornithobacterium rhinotracheale among broilers in Uruguay","pap_56172545600::Flock-level seroprevalence against Ornithobacterium rhinotracheale among broilers in Uruguay","pap_15070187100::Modeling gene sequences over time in 2009 H1N1 Influenza A Virus populations","pap_15070187100::Modeling gene sequences over time in 2009 H1N1 Influenza A Virus populations","pap_15070187100::Modeling gene sequences over time in 2009 H1N1 Influenza A Virus populations","pap_15070187100::Modeling gene sequences over time in 2009 H1N1 Influenza A Virus populations","pap_16426432300::Bayesian coalescent inference reveals high evolutionary rates and expansion of Norovirus populations","pap_16426432300::Bayesian coalescent inference reveals high evolutionary rates and expansion of Norovirus populations","pap_16426432300::Bayesian coalescent inference reveals high evolutionary rates and expansion of Norovirus populations","pap_55893081600::Predicting quantitative traits with regression models for dense molecular markers and pedigree","pap_23024872800::Modeling gene sequence changes over time in type 3 dengue viruses from Ecuador","pap_23024872800::Modeling gene sequence changes over time in type 3 dengue viruses from Ecuador","pap_23024872800::Modeling gene sequence changes over time in type 3 dengue viruses from Ecuador","pap_23024872800::Modeling gene sequence changes over time in type 3 dengue viruses from Ecuador","pap_23024872800::Modeling gene sequence changes over time in type 3 dengue viruses from Ecuador","pap_23024872800::Modeling gene sequence changes over time in type 3 dengue viruses from Ecuador","pap_57204215827::Predictive ability of direct genomic values for lifetime net merit of Holstein sires using selected subsets of single nucleotide polymorphism markers","pap_56172545600::Monitoring the infectious disease status of dairy cattle herds in Uruguay: A random effects hierarchical model analysis","pap_56172545600::Monitoring the infectious disease status of dairy cattle herds in Uruguay: A random effects hierarchical model analysis","pap_7202755185::Phylogeography of the Southwestern Atlantic menhaden genus Brevoortia (Clupeidae, Alosinae)","pap_7202755185::Phylogeography of the Southwestern Atlantic menhaden genus Brevoortia (Clupeidae, Alosinae)","pap_7202755185::Phylogeography of the Southwestern Atlantic menhaden genus Brevoortia (Clupeidae, Alosinae)","pap_24068102500::Phylogeography of the Neotropical catfish Pimelodus albicans (Siluriformes: Pimelodidae) from río de la Plata basin, South America, and conservation remarks","pap_24068102500::Phylogeography of the Neotropical catfish Pimelodus albicans (Siluriformes: Pimelodidae) from río de la Plata basin, South America, and conservation remarks","pap_6603463277::A comparison between Poisson and zero-inflated Poisson regression models with an application to number of black spots in Corriedale sheep","pap_6603463277::A comparison between Poisson and zero-inflated Poisson regression models with an application to number of black spots in Corriedale sheep","pap_6603463277::A comparison between Poisson and zero-inflated Poisson regression models with an application to number of black spots in Corriedale sheep","pap_6504749273::Bayesian coalescent inference of hepatitis A virus populations: Evolutionary rates and patterns","pap_6504749273::Bayesian coalescent inference of hepatitis A virus populations: Evolutionary rates and patterns","pap_6504749273::Bayesian coalescent inference of hepatitis A virus populations: Evolutionary rates and patterns","pap_6504749273::Bayesian coalescent inference of hepatitis A virus populations: Evolutionary rates and patterns","pap_6504749273::Bayesian coalescent inference of hepatitis A virus populations: Evolutionary rates and patterns","pap_55979915700::Genetic variability in calving success in Aberdeen Angus cows under extensive recording","pap_55979915700::Genetic variability in calving success in Aberdeen Angus cows under extensive recording","pap_23492809900::Genetic and environment factors on probability of early pregnancy of Caracu cattle","pap_6603663336::Phylogeography of the deer mouse (Peromyscus maniculatus) provides a predictive framework for research on hantaviruses","pap_6603463277::Inferring parameters shaping amino acid usage in prokaryotic genomes via Bayesian MCMC methods","pap_6603463277::Inferring parameters shaping amino acid usage in prokaryotic genomes via Bayesian MCMC methods","pap_6603463277::Inferring parameters shaping amino acid usage in prokaryotic genomes via Bayesian MCMC methods","pap_6603463277::Inferring parameters shaping amino acid usage in prokaryotic genomes via Bayesian MCMC methods","pap_55979915700::Model comparison for genetic evaluation of milk yield in Uruguayan holsteins","pap_55979915700::Evaluation of extent and amount of heterogeneous variance for milk yield in Uruguayan Holsteins","pap_35581971400::Assessing relationships between genetic evaluations using robust regression with an application to holsteins in uruguay","pap_7004526393::Bioequivalence study of theophylline considering Michaelis-Menten kinetics"],"posicion":[0,0,1,2,3,4,5,6,9,10,11,0,1,2,3,4,0,7,0,0,1,4,2,0,0,2,3,5,6,0,1,2,3,4,6,0,2,0,1,1,4,6,0,4,7,8,9,10,11,6,4,1,2,4,5,10,0,1,2,3,4,5,6,0,0,1,5,0,1,1,0,1,1,0,6,0,2,3,6,7,0,2,3,4,5,1,0,3,4,5,6,7,1,2,2,3,0,1,3,0,2,3,5,4,2,4,1,0,3,5,1,2,3,2,0,0,5,1,0,1,2,2,4,0,5,6,0,3,5,0,2,0,1,2,3,4,0,1,3,1,3,3,4,1,0,0,2,3,2,0,1,2,3,4,0,1,2,3,4,0,3,6,0,7,2,5,7,4,6,10,11,3,5,1,0,5,2,3,4,2,0,1,0,2,0,3,2,2,2,3,0,1,3,4,0,0,3,4,0,0,1,2,3,4,1,4,2,0,1,2,3,3,4,2,2,0,11,1,3,1,3,4,5,6,0,1,1,0,1,3,0,0,1,2,0,1,2,3,4,0,1,0,1,2,4,5,6,8,0,5,3,9,0,1,3,5,1,0,1,3,2,0,2,0,6,1,4,3,3,2,4,5,5,1,0,1,3,2,1,1,0,1,0,0,1,2,6,0,7,0,2,3,4,0,0,1,2,3,4,5,6,7,0,0,0,1,2,4,5,0,1,2,2,1,1,0,1,0,1,3,2,3,4,5,10,0,1,1,0,1,2,3,0,3,2,0,1,2,0,1,10,11,0,2,6,4,8,9,3,2,3,4,5,0,3,4,5,2,3,4,5,1,2,3,4,0,6,7,1,0,1,3,4,6,8,3,1,2,0,1,2,0,2,0,1,4,0,2,3,4,5,0,2,3,3,0,2,3,4,0,0,2,0]},"nodesToDataframe":true,"edgesToDataframe":true,"options":{"width":"100%","height":"100%","nodes":{"shape":"dot","physics":false},"manipulation":{"enabled":false},"edges":{"smooth":false},"physics":{"stabilization":false}},"groups":null,"width":null,"height":null,"idselection":{"enabled":false},"byselection":{"enabled":false},"main":{"text":"Uruguay","style":"font-family:Georgia, Times New Roman, Times, serif;font-weight:bold;font-size:20px;text-align:center;"},"submain":null,"footer":null,"background":"rgba(0, 0, 0, 0)","igraphlayout":{"type":"square"},"legend":{"width":0.2,"useGroups":true,"position":"left","ncol":1,"stepX":100,"stepY":100,"zoom":true,"nodes":{"label":["Author","Paper"],"shape":["dot","dot"],"color":["#3366CC","#CC6633"],"title":["Nodes","Nodes"]},"nodesToDataframe":true}},"evals":[],"jsHooks":[]}</script>
<script type="application/htmlwidget-sizing" data-for="htmlwidget-40b5eba84db10dfbcbe9">{"viewer":{"width":450,"height":350,"padding":15,"fill":true},"browser":{"width":960,"height":500,"padding":40,"fill":true}}</script>
</body>
</html>
